{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "from bioservices import UniProt\n",
    "import pandas as pd\n",
    "from Bio import SeqIO\n",
    "import urllib\n",
    "import re\n",
    "import ast"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Get UniProt entry id and feature info for EC numbers for top hundred organisms"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Get cleaned data from BRENDA\n",
    "df_brenda = pd.read_csv('BRENDA_interactions_intracellular.txt', index_col=0, header=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>EC</th>\n",
       "      <th>Enz</th>\n",
       "      <th>Met</th>\n",
       "      <th>Org</th>\n",
       "      <th>Mode</th>\n",
       "      <th>ChEBI</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1.1.1.1</td>\n",
       "      <td>alcohol dehydrogenase</td>\n",
       "      <td>2-propanol</td>\n",
       "      <td>Sulfolobus acidocaldarius</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:17824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1.1.1.1</td>\n",
       "      <td>alcohol dehydrogenase</td>\n",
       "      <td>ethanol</td>\n",
       "      <td>Saccharomyces cerevisiae</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:16236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1.1.1.1</td>\n",
       "      <td>alcohol dehydrogenase</td>\n",
       "      <td>Isopropanol</td>\n",
       "      <td>Saccharomyces cerevisiae</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:17824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1.1.1.1</td>\n",
       "      <td>alcohol dehydrogenase</td>\n",
       "      <td>Urea</td>\n",
       "      <td>Thermus sp.</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:16199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1.1.1.101</td>\n",
       "      <td>acylglycerone-phosphate reductase</td>\n",
       "      <td>dihydroxyacetone</td>\n",
       "      <td>Saccharomyces cerevisiae</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:16016</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33238</th>\n",
       "      <td>7.6.2.9</td>\n",
       "      <td>ABC-type quaternary amine transporter</td>\n",
       "      <td>Betaine aldehyde</td>\n",
       "      <td>Aphanothece halophytica</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:15710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33239</th>\n",
       "      <td>7.6.2.9</td>\n",
       "      <td>ABC-type quaternary amine transporter</td>\n",
       "      <td>carnitine</td>\n",
       "      <td>Lactococcus lactis</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:3424</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33240</th>\n",
       "      <td>7.6.2.9</td>\n",
       "      <td>ABC-type quaternary amine transporter</td>\n",
       "      <td>choline</td>\n",
       "      <td>Aphanothece halophytica</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:15354</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33241</th>\n",
       "      <td>7.6.2.9</td>\n",
       "      <td>ABC-type quaternary amine transporter</td>\n",
       "      <td>proline</td>\n",
       "      <td>Lactococcus lactis</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:60039</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33242</th>\n",
       "      <td>7.6.2.9</td>\n",
       "      <td>ABC-type quaternary amine transporter</td>\n",
       "      <td>proline betaine</td>\n",
       "      <td>Lactococcus lactis</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:44813</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>33243 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              EC                                    Enz               Met  \\\n",
       "0        1.1.1.1                  alcohol dehydrogenase        2-propanol   \n",
       "1        1.1.1.1                  alcohol dehydrogenase           ethanol   \n",
       "2        1.1.1.1                  alcohol dehydrogenase       Isopropanol   \n",
       "3        1.1.1.1                  alcohol dehydrogenase              Urea   \n",
       "4      1.1.1.101      acylglycerone-phosphate reductase  dihydroxyacetone   \n",
       "...          ...                                    ...               ...   \n",
       "33238    7.6.2.9  ABC-type quaternary amine transporter  Betaine aldehyde   \n",
       "33239    7.6.2.9  ABC-type quaternary amine transporter         carnitine   \n",
       "33240    7.6.2.9  ABC-type quaternary amine transporter           choline   \n",
       "33241    7.6.2.9  ABC-type quaternary amine transporter           proline   \n",
       "33242    7.6.2.9  ABC-type quaternary amine transporter   proline betaine   \n",
       "\n",
       "                             Org Mode        ChEBI  \n",
       "0      Sulfolobus acidocaldarius    +  CHEBI:17824  \n",
       "1       Saccharomyces cerevisiae    +  CHEBI:16236  \n",
       "2       Saccharomyces cerevisiae    +  CHEBI:17824  \n",
       "3                    Thermus sp.    +  CHEBI:16199  \n",
       "4       Saccharomyces cerevisiae    +  CHEBI:16016  \n",
       "...                          ...  ...          ...  \n",
       "33238    Aphanothece halophytica    -  CHEBI:15710  \n",
       "33239         Lactococcus lactis    -   CHEBI:3424  \n",
       "33240    Aphanothece halophytica    -  CHEBI:15354  \n",
       "33241         Lactococcus lactis    -  CHEBI:60039  \n",
       "33242         Lactococcus lactis    -  CHEBI:44813  \n",
       "\n",
       "[33243 rows x 6 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_brenda"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Get the top hundred organisms from previously created file\n",
    "top_hundred_list = pd.read_csv('top_hundred_organisms.txt', sep='\\t', index_col=0)['Org'].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Filter dataframe for interactions in the top hundred organisms\n",
    "df_brenda_top_hundred = df_brenda[df_brenda['Org'].isin(top_hundred_list)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>EC</th>\n",
       "      <th>Enz</th>\n",
       "      <th>Met</th>\n",
       "      <th>Org</th>\n",
       "      <th>Mode</th>\n",
       "      <th>ChEBI</th>\n",
       "      <th>Entry_string</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1.1.1.1</td>\n",
       "      <td>alcohol dehydrogenase</td>\n",
       "      <td>2-propanol</td>\n",
       "      <td>Sulfolobus acidocaldarius</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:17824</td>\n",
       "      <td>EC:1.1.1.1+AND+Sulfolobus acidocaldarius+AND+r...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1.1.1.1</td>\n",
       "      <td>alcohol dehydrogenase</td>\n",
       "      <td>ethanol</td>\n",
       "      <td>Saccharomyces cerevisiae</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:16236</td>\n",
       "      <td>EC:1.1.1.1+AND+Saccharomyces cerevisiae+AND+re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1.1.1.1</td>\n",
       "      <td>alcohol dehydrogenase</td>\n",
       "      <td>Isopropanol</td>\n",
       "      <td>Saccharomyces cerevisiae</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:17824</td>\n",
       "      <td>EC:1.1.1.1+AND+Saccharomyces cerevisiae+AND+re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1.1.1.101</td>\n",
       "      <td>acylglycerone-phosphate reductase</td>\n",
       "      <td>dihydroxyacetone</td>\n",
       "      <td>Saccharomyces cerevisiae</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:16016</td>\n",
       "      <td>EC:1.1.1.101+AND+Saccharomyces cerevisiae+AND+...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1.1.1.101</td>\n",
       "      <td>acylglycerone-phosphate reductase</td>\n",
       "      <td>ethanol</td>\n",
       "      <td>Saccharomyces cerevisiae</td>\n",
       "      <td>+</td>\n",
       "      <td>CHEBI:16236</td>\n",
       "      <td>EC:1.1.1.101+AND+Saccharomyces cerevisiae+AND+...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33234</th>\n",
       "      <td>7.6.2.3</td>\n",
       "      <td>ABC-type glutathione-S-conjugate transporter</td>\n",
       "      <td>leukotriene C4</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:16978</td>\n",
       "      <td>EC:7.6.2.3+AND+Homo sapiens+AND+reviewed:true</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33237</th>\n",
       "      <td>7.6.2.9</td>\n",
       "      <td>ABC-type quaternary amine transporter</td>\n",
       "      <td>ADP</td>\n",
       "      <td>Lactococcus lactis</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:16761</td>\n",
       "      <td>EC:7.6.2.9+AND+Lactococcus lactis+AND+reviewed...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33239</th>\n",
       "      <td>7.6.2.9</td>\n",
       "      <td>ABC-type quaternary amine transporter</td>\n",
       "      <td>carnitine</td>\n",
       "      <td>Lactococcus lactis</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:3424</td>\n",
       "      <td>EC:7.6.2.9+AND+Lactococcus lactis+AND+reviewed...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33241</th>\n",
       "      <td>7.6.2.9</td>\n",
       "      <td>ABC-type quaternary amine transporter</td>\n",
       "      <td>proline</td>\n",
       "      <td>Lactococcus lactis</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:60039</td>\n",
       "      <td>EC:7.6.2.9+AND+Lactococcus lactis+AND+reviewed...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33242</th>\n",
       "      <td>7.6.2.9</td>\n",
       "      <td>ABC-type quaternary amine transporter</td>\n",
       "      <td>proline betaine</td>\n",
       "      <td>Lactococcus lactis</td>\n",
       "      <td>-</td>\n",
       "      <td>CHEBI:44813</td>\n",
       "      <td>EC:7.6.2.9+AND+Lactococcus lactis+AND+reviewed...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>20825 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              EC                                           Enz  \\\n",
       "0        1.1.1.1                         alcohol dehydrogenase   \n",
       "1        1.1.1.1                         alcohol dehydrogenase   \n",
       "2        1.1.1.1                         alcohol dehydrogenase   \n",
       "4      1.1.1.101             acylglycerone-phosphate reductase   \n",
       "5      1.1.1.101             acylglycerone-phosphate reductase   \n",
       "...          ...                                           ...   \n",
       "33234    7.6.2.3  ABC-type glutathione-S-conjugate transporter   \n",
       "33237    7.6.2.9         ABC-type quaternary amine transporter   \n",
       "33239    7.6.2.9         ABC-type quaternary amine transporter   \n",
       "33241    7.6.2.9         ABC-type quaternary amine transporter   \n",
       "33242    7.6.2.9         ABC-type quaternary amine transporter   \n",
       "\n",
       "                    Met                        Org Mode        ChEBI  \\\n",
       "0            2-propanol  Sulfolobus acidocaldarius    +  CHEBI:17824   \n",
       "1               ethanol   Saccharomyces cerevisiae    +  CHEBI:16236   \n",
       "2           Isopropanol   Saccharomyces cerevisiae    +  CHEBI:17824   \n",
       "4      dihydroxyacetone   Saccharomyces cerevisiae    +  CHEBI:16016   \n",
       "5               ethanol   Saccharomyces cerevisiae    +  CHEBI:16236   \n",
       "...                 ...                        ...  ...          ...   \n",
       "33234    leukotriene C4               Homo sapiens    -  CHEBI:16978   \n",
       "33237               ADP         Lactococcus lactis    -  CHEBI:16761   \n",
       "33239         carnitine         Lactococcus lactis    -   CHEBI:3424   \n",
       "33241           proline         Lactococcus lactis    -  CHEBI:60039   \n",
       "33242   proline betaine         Lactococcus lactis    -  CHEBI:44813   \n",
       "\n",
       "                                            Entry_string  \n",
       "0      EC:1.1.1.1+AND+Sulfolobus acidocaldarius+AND+r...  \n",
       "1      EC:1.1.1.1+AND+Saccharomyces cerevisiae+AND+re...  \n",
       "2      EC:1.1.1.1+AND+Saccharomyces cerevisiae+AND+re...  \n",
       "4      EC:1.1.1.101+AND+Saccharomyces cerevisiae+AND+...  \n",
       "5      EC:1.1.1.101+AND+Saccharomyces cerevisiae+AND+...  \n",
       "...                                                  ...  \n",
       "33234      EC:7.6.2.3+AND+Homo sapiens+AND+reviewed:true  \n",
       "33237  EC:7.6.2.9+AND+Lactococcus lactis+AND+reviewed...  \n",
       "33239  EC:7.6.2.9+AND+Lactococcus lactis+AND+reviewed...  \n",
       "33241  EC:7.6.2.9+AND+Lactococcus lactis+AND+reviewed...  \n",
       "33242  EC:7.6.2.9+AND+Lactococcus lactis+AND+reviewed...  \n",
       "\n",
       "[20825 rows x 7 columns]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_brenda_top_hundred"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Add column of entry-string for search in UniProt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Define formula for making entry-string\n",
    "def entry_formula(EC, Org):\n",
    "    return f'EC:{EC}+AND+{Org}+AND+reviewed:true'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\Elin\\AppData\\Local\\Temp\\ipykernel_2644\\2498833545.py:2: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df_brenda_top_hundred['Entry_string'] = df_brenda_top_hundred.apply(lambda x: entry_formula(x['EC'], x['Org']), axis=1)\n"
     ]
    }
   ],
   "source": [
    "#apply formula to dataframe\n",
    "df_brenda_top_hundred['Entry_string'] = df_brenda_top_hundred.apply(lambda x: entry_formula(x['EC'], x['Org']), axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Make dataframe of only enzymes (EC numbers), organisms and entry-strings\n",
    "df_ec_enz_org = df_brenda_top_hundred[['EC', 'Enz', 'Org', 'Entry_string']].drop_duplicates()"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### Search in UniProt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "u = UniProt(verbose=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Get info for every EC number+Organism combination\n",
    "#The get_df() function has a limit of 10 entries per search for the sake of efficiency, \n",
    "# since the number of entries is quite large I decided to not change that.\n",
    "#Also: the UniProt database changes all the time, and a new search would therefore probably not give the same results\n",
    "\n",
    "entries = set(df_ec_enz_org['Entry_string'])\n",
    "\n",
    "df = pd.DataFrame([])\n",
    "\n",
    "for entry in entries:\n",
    "    df = pd.concat([df,u.get_df(entry)])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry   Entry Name                                         Gene Names  \\\n",
       " 0   Q7YRC6  AURKB_BOVIN         AURKB AIK2 AIM1 AIRK2 ARK2 STK1 STK12 STK5   \n",
       " 1   Q32PI1   VRK1_BOVIN                                               VRK1   \n",
       " 2   F1MH24   AAK1_BOVIN                                               AAK1   \n",
       " 3   P35508   KC1D_BOVIN                                       CSNK1D HCKID   \n",
       " 4   Q01314   AKT1_BOVIN                                           AKT1 PKB   \n",
       " 5   Q3SWY2    ILK_BOVIN                                      ILK ILK1 ILK2   \n",
       " 6   Q95KV0   IKKB_BOVIN                                              IKBKB   \n",
       " 7   P67827   KC1A_BOVIN                                            CSNK1A1   \n",
       " 8   P68399  CSK21_BOVIN                                      CSNK2A1 CK2A1   \n",
       " 9   Q2LGB3  IRAK1_BOVIN                                              IRAK1   \n",
       " 10  Q8MIT6  ROCK1_BOVIN                                              ROCK1   \n",
       " 11  Q9N0P9   PIM1_BOVIN                                               PIM1   \n",
       " 12  E1BFR5    GWL_BOVIN                                       MASTL GW GWL   \n",
       " 13  Q28021  ROCK2_BOVIN                                              ROCK2   \n",
       " 14  Q2KIP2   HASP_BOVIN                                        HASPIN GSG2   \n",
       " 15  Q3SWY6  STK25_BOVIN                                              STK25   \n",
       " 16  Q3SYZ2  MAPK3_BOVIN                                           MAPKAPK3   \n",
       " 17  Q5E9L6   STK4_BOVIN                                          STK4 MST1   \n",
       " 18  Q08E52   PAK1_BOVIN                                               PAK1   \n",
       " 19  Q0V7M1  KPSH1_BOVIN                                              PSKH1   \n",
       " 20  Q1RMT7  RIOK3_BOVIN                                              RIOK3   \n",
       " 21  Q2TA06  AURKA_BOVIN  AURKA AIK AIRK1 ARK1 AURA AYK1 BTAK IAK1 STK15...   \n",
       " 22  A7E3S4   RAF1_BOVIN                                               RAF1   \n",
       " 23  A7MB74   SGK1_BOVIN                                           SGK1 SGK   \n",
       " 24  Q02399   CDK5_BOVIN                                 CDK5 CDKN5 PSSALRE   \n",
       " 25  Q17R13   ACK1_BOVIN                                     TNK2 ACK1 ACK2   \n",
       " 26  Q3SZJ2  RIPK2_BOVIN                                              RIPK2   \n",
       " 27  Q6TJY3  KS6B1_BOVIN                                            RPS6KB1   \n",
       " 28  P32871  PK3CA_BOVIN                                             PIK3CA   \n",
       " 29  A2VDV2  STK38_BOVIN                                              STK38   \n",
       " 30  P20427  CSK22_BOVIN                                            CSNK2A2   \n",
       " 31  Q08DZ2  PRP4B_BOVIN                                             PRPF4B   \n",
       " 32  Q32L23  LIMK2_BOVIN                                              LIMK2   \n",
       " 33  Q3SZW1  TSSK1_BOVIN                                       TSSK1B TSSK1   \n",
       " 34  E1BK52  STK10_BOVIN                                              STK10   \n",
       " 35  A7E3X2  KC1G1_BOVIN                                            CSNK1G1   \n",
       " 36  P35507   KC1B_BOVIN                                             CSNK1B   \n",
       " 37  Q0VD22  STK33_BOVIN                                              STK33   \n",
       " 38  Q17QV9  STK40_BOVIN                                              STK40   \n",
       " 39  Q1RMT8  IRAK4_BOVIN                                              IRAK4   \n",
       " 40  P35509  KC1G3_BOVIN                                            CSNK1G3   \n",
       " 41  Q58D94  MKNK1_BOVIN                                              MKNK1   \n",
       " 42  P16913     B1_VACCW                                  VPK1 VACWR183 B1R   \n",
       " 43  Q89121   VPK2_VACCW                                 VPK2 VACWR049 F10L   \n",
       " 44  Q08097   US03_BHV1S                                                NaN   \n",
       " \n",
       "    Gene Names (primary)                           Gene Names (synonym)  \\\n",
       " 0                 AURKB           AIK2 AIM1 AIRK2 ARK2 STK1 STK12 STK5   \n",
       " 1                  VRK1                                            NaN   \n",
       " 2                  AAK1                                            NaN   \n",
       " 3                CSNK1D                                          HCKID   \n",
       " 4                  AKT1                                            PKB   \n",
       " 5                   ILK                                      ILK1 ILK2   \n",
       " 6                 IKBKB                                            NaN   \n",
       " 7               CSNK1A1                                            NaN   \n",
       " 8               CSNK2A1                                          CK2A1   \n",
       " 9                 IRAK1                                            NaN   \n",
       " 10                ROCK1                                            NaN   \n",
       " 11                 PIM1                                            NaN   \n",
       " 12                MASTL                                         GW GWL   \n",
       " 13                ROCK2                                            NaN   \n",
       " 14               HASPIN                                           GSG2   \n",
       " 15                STK25                                            NaN   \n",
       " 16             MAPKAPK3                                            NaN   \n",
       " 17                 STK4                                           MST1   \n",
       " 18                 PAK1                                            NaN   \n",
       " 19                PSKH1                                            NaN   \n",
       " 20                RIOK3                                            NaN   \n",
       " 21                AURKA  AIK AIRK1 ARK1 AURA AYK1 BTAK IAK1 STK15 STK6   \n",
       " 22                 RAF1                                            NaN   \n",
       " 23                 SGK1                                            SGK   \n",
       " 24                 CDK5                                  CDKN5 PSSALRE   \n",
       " 25                 TNK2                                      ACK1 ACK2   \n",
       " 26                RIPK2                                            NaN   \n",
       " 27              RPS6KB1                                            NaN   \n",
       " 28               PIK3CA                                            NaN   \n",
       " 29                STK38                                            NaN   \n",
       " 30              CSNK2A2                                            NaN   \n",
       " 31               PRPF4B                                            NaN   \n",
       " 32                LIMK2                                            NaN   \n",
       " 33               TSSK1B                                          TSSK1   \n",
       " 34                STK10                                            NaN   \n",
       " 35              CSNK1G1                                            NaN   \n",
       " 36               CSNK1B                                            NaN   \n",
       " 37                STK33                                            NaN   \n",
       " 38                STK40                                            NaN   \n",
       " 39                IRAK4                                            NaN   \n",
       " 40              CSNK1G3                                            NaN   \n",
       " 41                MKNK1                                            NaN   \n",
       " 42                 VPK1                                            NaN   \n",
       " 43                 VPK2                                            NaN   \n",
       " 44                  NaN                                            NaN   \n",
       " \n",
       "    Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                         NaN              NaN   \n",
       " 1                         NaN              NaN   \n",
       " 2                         NaN              NaN   \n",
       " 3                         NaN              NaN   \n",
       " 4                         NaN              NaN   \n",
       " 5                         NaN              NaN   \n",
       " 6                         NaN              NaN   \n",
       " 7                         NaN              NaN   \n",
       " 8                         NaN              NaN   \n",
       " 9                         NaN              NaN   \n",
       " 10                        NaN              NaN   \n",
       " 11                        NaN              NaN   \n",
       " 12                        NaN              NaN   \n",
       " 13                        NaN              NaN   \n",
       " 14                        NaN              NaN   \n",
       " 15                        NaN              NaN   \n",
       " 16                        NaN              NaN   \n",
       " 17                        NaN              NaN   \n",
       " 18                        NaN              NaN   \n",
       " 19                        NaN              NaN   \n",
       " 20                        NaN              NaN   \n",
       " 21                        NaN              NaN   \n",
       " 22                        NaN              NaN   \n",
       " 23                        NaN              NaN   \n",
       " 24                        NaN              NaN   \n",
       " 25                        NaN              NaN   \n",
       " 26                        NaN              NaN   \n",
       " 27                        NaN              NaN   \n",
       " 28                        NaN              NaN   \n",
       " 29                        NaN              NaN   \n",
       " 30                        NaN              NaN   \n",
       " 31                        NaN              NaN   \n",
       " 32                        NaN              NaN   \n",
       " 33                        NaN              NaN   \n",
       " 34                        NaN              NaN   \n",
       " 35                        NaN              NaN   \n",
       " 36                        NaN              NaN   \n",
       " 37                        NaN              NaN   \n",
       " 38                        NaN              NaN   \n",
       " 39                        NaN              NaN   \n",
       " 40                        NaN              NaN   \n",
       " 41                        NaN              NaN   \n",
       " 42                   VACWR183              B1R   \n",
       " 43                   VACWR049             F10L   \n",
       " 44                        NaN              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                 Bos taurus (Bovine)           9913   \n",
       " 1                                 Bos taurus (Bovine)           9913   \n",
       " 2                                 Bos taurus (Bovine)           9913   \n",
       " 3                                 Bos taurus (Bovine)           9913   \n",
       " 4                                 Bos taurus (Bovine)           9913   \n",
       " 5                                 Bos taurus (Bovine)           9913   \n",
       " 6                                 Bos taurus (Bovine)           9913   \n",
       " 7                                 Bos taurus (Bovine)           9913   \n",
       " 8                                 Bos taurus (Bovine)           9913   \n",
       " 9                                 Bos taurus (Bovine)           9913   \n",
       " 10                                Bos taurus (Bovine)           9913   \n",
       " 11                                Bos taurus (Bovine)           9913   \n",
       " 12                                Bos taurus (Bovine)           9913   \n",
       " 13                                Bos taurus (Bovine)           9913   \n",
       " 14                                Bos taurus (Bovine)           9913   \n",
       " 15                                Bos taurus (Bovine)           9913   \n",
       " 16                                Bos taurus (Bovine)           9913   \n",
       " 17                                Bos taurus (Bovine)           9913   \n",
       " 18                                Bos taurus (Bovine)           9913   \n",
       " 19                                Bos taurus (Bovine)           9913   \n",
       " 20                                Bos taurus (Bovine)           9913   \n",
       " 21                                Bos taurus (Bovine)           9913   \n",
       " 22                                Bos taurus (Bovine)           9913   \n",
       " 23                                Bos taurus (Bovine)           9913   \n",
       " 24                                Bos taurus (Bovine)           9913   \n",
       " 25                                Bos taurus (Bovine)           9913   \n",
       " 26                                Bos taurus (Bovine)           9913   \n",
       " 27                                Bos taurus (Bovine)           9913   \n",
       " 28                                Bos taurus (Bovine)           9913   \n",
       " 29                                Bos taurus (Bovine)           9913   \n",
       " 30                                Bos taurus (Bovine)           9913   \n",
       " 31                                Bos taurus (Bovine)           9913   \n",
       " 32                                Bos taurus (Bovine)           9913   \n",
       " 33                                Bos taurus (Bovine)           9913   \n",
       " 34                                Bos taurus (Bovine)           9913   \n",
       " 35                                Bos taurus (Bovine)           9913   \n",
       " 36                                Bos taurus (Bovine)           9913   \n",
       " 37                                Bos taurus (Bovine)           9913   \n",
       " 38                                Bos taurus (Bovine)           9913   \n",
       " 39                                Bos taurus (Bovine)           9913   \n",
       " 40                                Bos taurus (Bovine)           9913   \n",
       " 41                                Bos taurus (Bovine)           9913   \n",
       " 42  Vaccinia virus (strain Western Reserve) (VACV)...          10254   \n",
       " 43  Vaccinia virus (strain Western Reserve) (VACV)...          10254   \n",
       " 44  Bovine herpesvirus 1.2 (strain ST) (BoHV-1) (I...          45407   \n",
       " \n",
       "                                         Protein names  ...  \\\n",
       " 0   Aurora kinase B (EC 2.7.11.1) (Aurora 1) (Auro...  ...   \n",
       " 1   Serine/threonine-protein kinase VRK1 (EC 2.7.1...  ...   \n",
       " 2   AP2-associated protein kinase 1 (EC 2.7.11.1) ...  ...   \n",
       " 3   Casein kinase I isoform delta (CKI-delta) (CKI...  ...   \n",
       " 4   RAC-alpha serine/threonine-protein kinase (EC ...  ...   \n",
       " 5   Integrin-linked protein kinase (EC 2.7.11.1) (...  ...   \n",
       " 6   Inhibitor of nuclear factor kappa-B kinase sub...  ...   \n",
       " 7   Casein kinase I isoform alpha (CKI-alpha) (EC ...  ...   \n",
       " 8   Casein kinase II subunit alpha (CK II alpha) (...  ...   \n",
       " 9   Interleukin-1 receptor-associated kinase 1 (IR...  ...   \n",
       " 10  Rho-associated protein kinase 1 (EC 2.7.11.1) ...  ...   \n",
       " 11  Serine/threonine-protein kinase pim-1 (EC 2.7....  ...   \n",
       " 12  Serine/threonine-protein kinase greatwall (GW)...  ...   \n",
       " 13  Rho-associated protein kinase 2 (EC 2.7.11.1) ...  ...   \n",
       " 14  Serine/threonine-protein kinase haspin (EC 2.7...  ...   \n",
       " 15   Serine/threonine-protein kinase 25 (EC 2.7.11.1)  ...   \n",
       " 16  MAP kinase-activated protein kinase 3 (MAPK-ac...  ...   \n",
       " 17  Serine/threonine-protein kinase 4 (EC 2.7.11.1...  ...   \n",
       " 18  Serine/threonine-protein kinase PAK 1 (EC 2.7....  ...   \n",
       " 19  Serine/threonine-protein kinase H1 (EC 2.7.11....  ...   \n",
       " 20  Serine/threonine-protein kinase RIO3 (EC 2.7.1...  ...   \n",
       " 21  Aurora kinase A (EC 2.7.11.1) (Aurora 2) (Auro...  ...   \n",
       " 22  RAF proto-oncogene serine/threonine-protein ki...  ...   \n",
       " 23  Serine/threonine-protein kinase Sgk1 (EC 2.7.1...  ...   \n",
       " 24  Cyclin-dependent kinase 5 (EC 2.7.11.1) (Cell ...  ...   \n",
       " 25  Activated CDC42 kinase 1 (ACK-1) (EC 2.7.10.2)...  ...   \n",
       " 26  Receptor-interacting serine/threonine-protein ...  ...   \n",
       " 27  Ribosomal protein S6 kinase beta-1 (S6K-beta-1...  ...   \n",
       " 28  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...   \n",
       " 29   Serine/threonine-protein kinase 38 (EC 2.7.11.1)  ...   \n",
       " 30  Casein kinase II subunit alpha' (CK II alpha')...  ...   \n",
       " 31  Serine/threonine-protein kinase PRP4 homolog (...  ...   \n",
       " 32         LIM domain kinase 2 (LIMK-2) (EC 2.7.11.1)  ...   \n",
       " 33  Testis-specific serine/threonine-protein kinas...  ...   \n",
       " 34   Serine/threonine-protein kinase 10 (EC 2.7.11.1)  ...   \n",
       " 35  Casein kinase I isoform gamma-1 (CKI-gamma 1) ...  ...   \n",
       " 36  Casein kinase I isoform beta (CKI-beta) (EC 2....  ...   \n",
       " 37   Serine/threonine-protein kinase 33 (EC 2.7.11.1)  ...   \n",
       " 38   Serine/threonine-protein kinase 40 (EC 2.7.11.1)  ...   \n",
       " 39  Interleukin-1 receptor-associated kinase 4 (IR...  ...   \n",
       " 40  Casein kinase I isoform gamma-3 (CKI-gamma 3) ...  ...   \n",
       " 41  MAP kinase-interacting serine/threonine-protei...  ...   \n",
       " 42  B1 kinase (Serine/threonine-protein kinase 1) ...  ...   \n",
       " 43  Serine/threonine-protein kinase 2 (EC 2.7.11.1...  ...   \n",
       " 44  Serine/threonine-protein kinase US3 homolog (E...  ...   \n",
       " \n",
       "                                         Glycosylation  \\\n",
       " 0                                                 NaN   \n",
       " 1                                                 NaN   \n",
       " 2                                                 NaN   \n",
       " 3                                                 NaN   \n",
       " 4   CARBOHYD 129; /note=\"O-linked (GlcNAc) serine;...   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7                                                 NaN   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " 13                                                NaN   \n",
       " 14                                                NaN   \n",
       " 15                                                NaN   \n",
       " 16                                                NaN   \n",
       " 17                                                NaN   \n",
       " 18                                                NaN   \n",
       " 19                                                NaN   \n",
       " 20                                                NaN   \n",
       " 21                                                NaN   \n",
       " 22                                                NaN   \n",
       " 23                                                NaN   \n",
       " 24                                                NaN   \n",
       " 25                                                NaN   \n",
       " 26                                                NaN   \n",
       " 27                                                NaN   \n",
       " 28                                                NaN   \n",
       " 29                                                NaN   \n",
       " 30                                                NaN   \n",
       " 31                                                NaN   \n",
       " 32                                                NaN   \n",
       " 33                                                NaN   \n",
       " 34                                                NaN   \n",
       " 35                                                NaN   \n",
       " 36                                                NaN   \n",
       " 37                                                NaN   \n",
       " 38                                                NaN   \n",
       " 39                                                NaN   \n",
       " 40                                                NaN   \n",
       " 41                                                NaN   \n",
       " 42                                                NaN   \n",
       " 43                                                NaN   \n",
       " 44                                                NaN   \n",
       " \n",
       "                                  Initiator methionine  \\\n",
       " 0                                                 NaN   \n",
       " 1                                                 NaN   \n",
       " 2                                                 NaN   \n",
       " 3                                                 NaN   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " 13                                                NaN   \n",
       " 14                                                NaN   \n",
       " 15                                                NaN   \n",
       " 16                                                NaN   \n",
       " 17                                                NaN   \n",
       " 18  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 19                        INIT_MET 1; /note=\"Removed\"   \n",
       " 20                                                NaN   \n",
       " 21                                                NaN   \n",
       " 22                                                NaN   \n",
       " 23                                                NaN   \n",
       " 24                                                NaN   \n",
       " 25                                                NaN   \n",
       " 26                                                NaN   \n",
       " 27                                                NaN   \n",
       " 28                                                NaN   \n",
       " 29  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 30                                                NaN   \n",
       " 31  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 32                                                NaN   \n",
       " 33                                                NaN   \n",
       " 34                                                NaN   \n",
       " 35                                                NaN   \n",
       " 36                                                NaN   \n",
       " 37                                                NaN   \n",
       " 38                                                NaN   \n",
       " 39                                                NaN   \n",
       " 40                                                NaN   \n",
       " 41                                                NaN   \n",
       " 42                                                NaN   \n",
       " 43                                                NaN   \n",
       " 44                                                NaN   \n",
       " \n",
       "                                            Lipidation  \\\n",
       " 0                                                 NaN   \n",
       " 1                                                 NaN   \n",
       " 2                                                 NaN   \n",
       " 3                                                 NaN   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7                                                 NaN   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " 13                                                NaN   \n",
       " 14                                                NaN   \n",
       " 15                                                NaN   \n",
       " 16                                                NaN   \n",
       " 17                                                NaN   \n",
       " 18                                                NaN   \n",
       " 19  LIPID 2; /note=\"N-myristoyl glycine\"; /evidenc...   \n",
       " 20                                                NaN   \n",
       " 21                                                NaN   \n",
       " 22                                                NaN   \n",
       " 23                                                NaN   \n",
       " 24                                                NaN   \n",
       " 25                                                NaN   \n",
       " 26                                                NaN   \n",
       " 27                                                NaN   \n",
       " 28                                                NaN   \n",
       " 29                                                NaN   \n",
       " 30                                                NaN   \n",
       " 31                                                NaN   \n",
       " 32                                                NaN   \n",
       " 33                                                NaN   \n",
       " 34                                                NaN   \n",
       " 35                                                NaN   \n",
       " 36                                                NaN   \n",
       " 37                                                NaN   \n",
       " 38                                                NaN   \n",
       " 39                                                NaN   \n",
       " 40                                                NaN   \n",
       " 41                                                NaN   \n",
       " 42                                                NaN   \n",
       " 43                                                NaN   \n",
       " 44                                                NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 35; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 1   MOD_RES 376; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 2   MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 3   MOD_RES 328; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 4   MOD_RES 14; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 5   MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 6   MOD_RES 177; /note=\"Phosphoserine; by TBK1 and...     NaN   \n",
       " 7   MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 8   MOD_RES 344; /note=\"Phosphothreonine; by CDK1\"...     NaN   \n",
       " 9   MOD_RES 66; /note=\"Phosphothreonine; by PKC/PR...     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11  MOD_RES 8; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " 12  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 13  MOD_RES 414; /note=\"Phosphothreonine; by ROCK2...     NaN   \n",
       " 14  MOD_RES 58; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 15  MOD_RES 174; /note=\"Phosphothreonine; by autoc...     NaN   \n",
       " 16  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 17  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 18  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 19  MOD_RES 380; /note=\"Phosphoserine; by autocata...     NaN   \n",
       " 20  MOD_RES 8; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " 21  MOD_RES 41; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 22  MOD_RES 29; /note=\"Phosphoserine; by MAPK1\"; /...     NaN   \n",
       " 23  MOD_RES 74; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 24  MOD_RES 15; /note=\"Phosphotyrosine; by ABL1, E...     NaN   \n",
       " 25  MOD_RES 284; /note=\"Phosphotyrosine; by SRC an...     NaN   \n",
       " 26  MOD_RES 168; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 27  MOD_RES 252; /note=\"Phosphothreonine; by PDPK1...     NaN   \n",
       " 28                                                NaN     NaN   \n",
       " 29  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 30  MOD_RES 13; /note=\"Phosphotyrosine\"; /evidence...     NaN   \n",
       " 31  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 32  MOD_RES 293; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 33  MOD_RES 174; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 34  MOD_RES 13; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 35  MOD_RES 343; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 36                                                NaN     NaN   \n",
       " 37  MOD_RES 407; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 38                                                NaN     NaN   \n",
       " 39  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 40                                                NaN     NaN   \n",
       " 41  MOD_RES 27; /note=\"Phosphoserine; by PAK2\"; /e...     NaN   \n",
       " 42                                                NaN     NaN   \n",
       " 43                                                NaN     NaN   \n",
       " 44                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0   PTM: The phosphorylation of Thr-232 requires t...         NaN   \n",
       " 1   PTM: Autophosphorylated at various serine and ...         NaN   \n",
       " 2   PTM: Autophosphorylated. {ECO:0000250|UniProtK...         NaN   \n",
       " 3   PTM: Autophosphorylated on serine and threonin...         NaN   \n",
       " 4   PTM: O-GlcNAcylation at Thr-305 and Thr-312 in...         NaN   \n",
       " 5   PTM: Autophosphorylated on serine residues. {E...         NaN   \n",
       " 6   PTM: Upon cytokine stimulation, phosphorylated...         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8   PTM: Phosphorylated at Thr-344, Thr-360, Ser-3...         NaN   \n",
       " 9   PTM: Following recruitment on the activated re...         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11  PTM: Autophosphorylated (By similarity). Phosp...         NaN   \n",
       " 12  PTM: Phosphorylation at Thr-745 by CDK1 during...         NaN   \n",
       " 13  PTM: Autophosphorylated. Phosphorylation at Ty...         NaN   \n",
       " 14  PTM: Autophosphorylated on both serine and thr...         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16  PTM: Phosphorylated and activated by MAPK1/ERK...         NaN   \n",
       " 17  PTM: Autophosphorylated on serine and threonin...         NaN   \n",
       " 18  PTM: Autophosphorylated in trans, meaning that...         NaN   \n",
       " 19  PTM: Autophosphorylated on serine residues. {E...         NaN   \n",
       " 20  PTM: Autophosphorylated (in vitro). {ECO:00002...         NaN   \n",
       " 21  PTM: Activated by phosphorylation at Thr-288; ...         NaN   \n",
       " 22  PTM: Phosphorylation at Thr-269, Ser-338, Tyr-...         NaN   \n",
       " 23  PTM: Regulated by phosphorylation. Activated b...         NaN   \n",
       " 24  PTM: Phosphorylation on Tyr-15 by ABL1 and FYN...         NaN   \n",
       " 25  PTM: Autophosphorylation regulates kinase acti...         NaN   \n",
       " 26  PTM: Autophosphorylated. Autophosphorylation a...         NaN   \n",
       " 27  PTM: Phosphorylation at Thr-412 is regulated b...         NaN   \n",
       " 28                                                NaN         NaN   \n",
       " 29  PTM: ISGylated. {ECO:0000250}.; PTM: Phosphory...         NaN   \n",
       " 30                                                NaN         NaN   \n",
       " 31        PTM: Phosphorylated by Clk1. {ECO:0000250}.         NaN   \n",
       " 32  PTM: Phosphorylated on serine and/or threonine...         NaN   \n",
       " 33  PTM: Autophosphorylated. {ECO:0000250|UniProtK...         NaN   \n",
       " 34  PTM: Autophosphorylates following homodimeriza...         NaN   \n",
       " 35            PTM: Autophosphorylated. {ECO:0000250}.         NaN   \n",
       " 36                                                NaN         NaN   \n",
       " 37            PTM: Autophosphorylated. {ECO:0000250}.         NaN   \n",
       " 38                                                NaN         NaN   \n",
       " 39                PTM: Phosphorylated. {ECO:0000250}.         NaN   \n",
       " 40                                                NaN         NaN   \n",
       " 41  PTM: Dual phosphorylation of Thr-197 and Thr-2...         NaN   \n",
       " 42                           PTM: Autophosphorylated.         NaN   \n",
       " 43                           PTM: Autophosphorylated.         NaN   \n",
       " 44  PTM: Phosphorylated by UL13 homolog; this phos...         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide              PDB  \n",
       " 0              NaN             NaN              NaN  \n",
       " 1              NaN             NaN              NaN  \n",
       " 2              NaN             NaN              NaN  \n",
       " 3              NaN             NaN              NaN  \n",
       " 4              NaN             NaN              NaN  \n",
       " 5              NaN             NaN              NaN  \n",
       " 6              NaN             NaN              NaN  \n",
       " 7              NaN             NaN              NaN  \n",
       " 8              NaN             NaN              NaN  \n",
       " 9              NaN             NaN              NaN  \n",
       " 10             NaN             NaN              NaN  \n",
       " 11             NaN             NaN              NaN  \n",
       " 12             NaN             NaN              NaN  \n",
       " 13             NaN             NaN  1UIX;2F2U;2H9V;  \n",
       " 14             NaN             NaN              NaN  \n",
       " 15             NaN             NaN              NaN  \n",
       " 16             NaN             NaN              NaN  \n",
       " 17             NaN             NaN              NaN  \n",
       " 18             NaN             NaN              NaN  \n",
       " 19             NaN             NaN              NaN  \n",
       " 20             NaN             NaN              NaN  \n",
       " 21             NaN             NaN              NaN  \n",
       " 22             NaN             NaN              NaN  \n",
       " 23             NaN             NaN              NaN  \n",
       " 24             NaN             NaN              NaN  \n",
       " 25             NaN             NaN              NaN  \n",
       " 26             NaN             NaN              NaN  \n",
       " 27             NaN             NaN              NaN  \n",
       " 28             NaN             NaN            2V1Y;  \n",
       " 29             NaN             NaN              NaN  \n",
       " 30             NaN             NaN              NaN  \n",
       " 31             NaN             NaN              NaN  \n",
       " 32             NaN             NaN              NaN  \n",
       " 33             NaN             NaN              NaN  \n",
       " 34             NaN             NaN              NaN  \n",
       " 35             NaN             NaN              NaN  \n",
       " 36             NaN             NaN              NaN  \n",
       " 37             NaN             NaN              NaN  \n",
       " 38             NaN             NaN              NaN  \n",
       " 39             NaN             NaN              NaN  \n",
       " 40             NaN             NaN              NaN  \n",
       " 41             NaN             NaN              NaN  \n",
       " 42             NaN             NaN              NaN  \n",
       " 43             NaN             NaN              NaN  \n",
       " 44             NaN             NaN              NaN  \n",
       " \n",
       " [45 rows x 110 columns],\n",
       "      Entry  Entry Name                                         Gene Names  \\\n",
       " 0   Q9N0X0   AURKB_PIG         AURKB AIK2 AIM1 AIRK2 ARK2 STK1 STK12 STK5   \n",
       " 1   A5GFW1   AURKA_PIG  AURKA AIK AIRK1 ARK1 AURA AYK1 BTAK IAK1 STK15...   \n",
       " 2   Q6R2V0    WNK1_PIG                                          WNK1 HSN2   \n",
       " 3   A2BD05    NEK6_PIG                                               NEK6   \n",
       " 4   Q09136   AAPK1_PIG                                       PRKAA1 AMPK1   \n",
       " 5   Q6PQD5     ATM_PIG                                                ATM   \n",
       " 6   F1SPM8    AAK1_PIG                                               AAK1   \n",
       " 7   O19175    KC1A_PIG                                            CSNK1A1   \n",
       " 8   P50118     MOS_PIG                                                MOS   \n",
       " 9   M3TYT0   ROCK2_PIG                                              ROCK2   \n",
       " 10  Q28948   AAPK2_PIG                                       PRKAA2 AMPK2   \n",
       " 11  Q863I2   OXSR1_PIG                                         OXSR1 OSR1   \n",
       " 12  O02697   PK3CG_PIG                                             PIK3CG   \n",
       " 13  B8Y466   SRPK3_PIG                                  SRPK3 MSSK1 STK23   \n",
       " 14  O19004    ARAF_PIG                                         ARAF ARAF1   \n",
       " 15  P24381  US03_SUHVN                                                 PK   \n",
       " 16  P32216  VPK2_SWPVK                                               C20L   \n",
       " 17  P17613   KR1_SUHVK                                                 PK   \n",
       " 18  P0C8F3   PK1_ASFWA                                            War-131   \n",
       " 19  P42493   PK1_ASFB7                                    Ba71V-121 R298L   \n",
       " 20  P34206   PK1_ASFM2                              Mal-129 j8L j9L L19IL   \n",
       " 21  P0C8F2   PK1_ASFP4                                           Pret-133   \n",
       " 22  P0C8F4   PK1_ASFK5                                            Ken-133   \n",
       " 23  P30662  UL13_SUHVN                                          UL13 ULPK   \n",
       " \n",
       "    Gene Names (primary)                           Gene Names (synonym)  \\\n",
       " 0                 AURKB           AIK2 AIM1 AIRK2 ARK2 STK1 STK12 STK5   \n",
       " 1                 AURKA  AIK AIRK1 ARK1 AURA AYK1 BTAK IAK1 STK15 STK6   \n",
       " 2                  WNK1                                           HSN2   \n",
       " 3                  NEK6                                            NaN   \n",
       " 4                PRKAA1                                          AMPK1   \n",
       " 5                   ATM                                            NaN   \n",
       " 6                  AAK1                                            NaN   \n",
       " 7               CSNK1A1                                            NaN   \n",
       " 8                   MOS                                            NaN   \n",
       " 9                 ROCK2                                            NaN   \n",
       " 10               PRKAA2                                          AMPK2   \n",
       " 11                OXSR1                                           OSR1   \n",
       " 12               PIK3CG                                            NaN   \n",
       " 13                SRPK3                                    MSSK1 STK23   \n",
       " 14                 ARAF                                          ARAF1   \n",
       " 15                   PK                                            NaN   \n",
       " 16                  NaN                                            NaN   \n",
       " 17                   PK                                            NaN   \n",
       " 18                  NaN                                            NaN   \n",
       " 19                  NaN                                            NaN   \n",
       " 20                  NaN                                            NaN   \n",
       " 21                  NaN                                            NaN   \n",
       " 22                  NaN                                            NaN   \n",
       " 23                 UL13                                           ULPK   \n",
       " \n",
       "    Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                         NaN              NaN   \n",
       " 1                         NaN              NaN   \n",
       " 2                         NaN              NaN   \n",
       " 3                         NaN              NaN   \n",
       " 4                         NaN              NaN   \n",
       " 5                         NaN              NaN   \n",
       " 6                         NaN              NaN   \n",
       " 7                         NaN              NaN   \n",
       " 8                         NaN              NaN   \n",
       " 9                         NaN              NaN   \n",
       " 10                        NaN              NaN   \n",
       " 11                        NaN              NaN   \n",
       " 12                        NaN              NaN   \n",
       " 13                        NaN              NaN   \n",
       " 14                        NaN              NaN   \n",
       " 15                        NaN              NaN   \n",
       " 16                        NaN             C20L   \n",
       " 17                        NaN              NaN   \n",
       " 18                    War-131              NaN   \n",
       " 19                  Ba71V-121            R298L   \n",
       " 20                    Mal-129    j8L j9L L19IL   \n",
       " 21                   Pret-133              NaN   \n",
       " 22                    Ken-133              NaN   \n",
       " 23                        NaN              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                    Sus scrofa (Pig)           9823   \n",
       " 1                                    Sus scrofa (Pig)           9823   \n",
       " 2                                    Sus scrofa (Pig)           9823   \n",
       " 3                                    Sus scrofa (Pig)           9823   \n",
       " 4                                    Sus scrofa (Pig)           9823   \n",
       " 5                                    Sus scrofa (Pig)           9823   \n",
       " 6                                    Sus scrofa (Pig)           9823   \n",
       " 7                                    Sus scrofa (Pig)           9823   \n",
       " 8                                    Sus scrofa (Pig)           9823   \n",
       " 9                                    Sus scrofa (Pig)           9823   \n",
       " 10                                   Sus scrofa (Pig)           9823   \n",
       " 11                                   Sus scrofa (Pig)           9823   \n",
       " 12                                   Sus scrofa (Pig)           9823   \n",
       " 13                                   Sus scrofa (Pig)           9823   \n",
       " 14                                   Sus scrofa (Pig)           9823   \n",
       " 15  Suid herpesvirus 1 (strain NIA-3) (SuHV-1) (Ps...          10349   \n",
       " 16               Swinepox virus (strain Kasza) (SWPV)          10277   \n",
       " 17  Suid herpesvirus 1 (strain Kaplan) (SuHV-1) (P...          33703   \n",
       " 18  African swine fever virus (isolate Warthog/Nam...         561444   \n",
       " 19  African swine fever virus (strain Badajoz 1971...          10498   \n",
       " 20  African swine fever virus (isolate Tick/Malawi...          10500   \n",
       " 21  African swine fever virus (isolate Tick/South ...         561443   \n",
       " 22  African swine fever virus (isolate Pig/Kenya/K...         561445   \n",
       " 23  Suid herpesvirus 1 (strain NIA-3) (SuHV-1) (Ps...          10349   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Aurora kinase B (EC 2.7.11.1) (Aurora 1) (Auro...  ...           NaN   \n",
       " 1   Aurora kinase A (EC 2.7.11.1) (Aurora 2) (Auro...  ...           NaN   \n",
       " 2   Serine/threonine-protein kinase WNK1 (EC 2.7.1...  ...           NaN   \n",
       " 3   Serine/threonine-protein kinase Nek6 (EC 2.7.1...  ...           NaN   \n",
       " 4   5'-AMP-activated protein kinase catalytic subu...  ...           NaN   \n",
       " 5   Serine-protein kinase ATM (EC 2.7.11.1) (Ataxi...  ...           NaN   \n",
       " 6   AP2-associated protein kinase 1 (EC 2.7.11.1) ...  ...           NaN   \n",
       " 7   Casein kinase I isoform alpha (CKI-alpha) (EC ...  ...           NaN   \n",
       " 8   Proto-oncogene serine/threonine-protein kinase...  ...           NaN   \n",
       " 9   Rho-associated protein kinase 2 (EC 2.7.11.1) ...  ...           NaN   \n",
       " 10  5'-AMP-activated protein kinase catalytic subu...  ...           NaN   \n",
       " 11  Serine/threonine-protein kinase OSR1 (EC 2.7.1...  ...           NaN   \n",
       " 12  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...           NaN   \n",
       " 13  SRSF protein kinase 3 (EC 2.7.11.1) (Muscle-sp...  ...           NaN   \n",
       " 14  Serine/threonine-protein kinase A-Raf (EC 2.7....  ...           NaN   \n",
       " 15  Serine/threonine-protein kinase US3 homolog (E...  ...           NaN   \n",
       " 16    Serine/threonine-protein kinase 2 (EC 2.7.11.1)  ...           NaN   \n",
       " 17      Serine/threonine-protein kinase (EC 2.7.11.1)  ...           NaN   \n",
       " 18    Serine/threonine-protein kinase 1 (EC 2.7.11.1)  ...           NaN   \n",
       " 19    Serine/threonine-protein kinase 1 (EC 2.7.11.1)  ...           NaN   \n",
       " 20    Serine/threonine-protein kinase 1 (EC 2.7.11.1)  ...           NaN   \n",
       " 21    Serine/threonine-protein kinase 1 (EC 2.7.11.1)  ...           NaN   \n",
       " 22    Serine/threonine-protein kinase 1 (EC 2.7.11.1)  ...           NaN   \n",
       " 23  Serine/threonine-protein kinase UL13 (EC 2.7.1...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine Lipidation  \\\n",
       " 0                                                 NaN        NaN   \n",
       " 1                                                 NaN        NaN   \n",
       " 2                                                 NaN        NaN   \n",
       " 3                                                 NaN        NaN   \n",
       " 4                                                 NaN        NaN   \n",
       " 5   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...        NaN   \n",
       " 6                                                 NaN        NaN   \n",
       " 7                                                 NaN        NaN   \n",
       " 8                                                 NaN        NaN   \n",
       " 9                                                 NaN        NaN   \n",
       " 10                                                NaN        NaN   \n",
       " 11  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...        NaN   \n",
       " 12                                                NaN        NaN   \n",
       " 13                                                NaN        NaN   \n",
       " 14                                                NaN        NaN   \n",
       " 15                                                NaN        NaN   \n",
       " 16                                                NaN        NaN   \n",
       " 17                                                NaN        NaN   \n",
       " 18                                                NaN        NaN   \n",
       " 19                                                NaN        NaN   \n",
       " 20                                                NaN        NaN   \n",
       " 21                                                NaN        NaN   \n",
       " 22                                                NaN        NaN   \n",
       " 23                                                NaN        NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 35; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 1   MOD_RES 41; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 2   MOD_RES 19; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 3   MOD_RES 37; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 4   MOD_RES 14; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 5   MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 6   MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 7                                                 NaN     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9   MOD_RES 414; /note=\"Phosphothreonine; by ROCK2...     NaN   \n",
       " 10  MOD_RES 172; /note=\"Phosphothreonine; by LKB1 ...     NaN   \n",
       " 11  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 12  MOD_RES 1024; /note=\"Phosphothreonine; by PKA\"...     NaN   \n",
       " 13  MOD_RES 50; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 14  MOD_RES 157; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 15                                                NaN     NaN   \n",
       " 16                                                NaN     NaN   \n",
       " 17                                                NaN     NaN   \n",
       " 18                                                NaN     NaN   \n",
       " 19                                                NaN     NaN   \n",
       " 20                                                NaN     NaN   \n",
       " 21                                                NaN     NaN   \n",
       " 22                                                NaN     NaN   \n",
       " 23                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0   PTM: The phosphorylation of Thr-232 requires t...         NaN   \n",
       " 1   PTM: Activated by phosphorylation at Thr-288; ...         NaN   \n",
       " 2   PTM: Autophosphorylation at Ser-380 is inhibit...         NaN   \n",
       " 3   PTM: Autophosphorylated. Phosphorylation at Se...         NaN   \n",
       " 4   PTM: Ubiquitinated. {ECO:0000250|UniProtKB:Q5E...         NaN   \n",
       " 5   PTM: Phosphorylated by NUAK1/ARK5. Autophospho...         NaN   \n",
       " 6   PTM: Autophosphorylated. {ECO:0000250|UniProtK...         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9   PTM: Autophosphorylated. Phosphorylation at Ty...         NaN   \n",
       " 10  PTM: Ubiquitinated. {ECO:0000250|UniProtKB:Q8B...         NaN   \n",
       " 11  PTM: Autophosphorylated. {ECO:0000250|UniProtK...         NaN   \n",
       " 12  PTM: Autophosphorylation at Ser-1101 has no ef...         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15  PTM: Phosphorylated by UL13 homolog; this phos...         NaN   \n",
       " 16            PTM: Autophosphorylated. {ECO:0000250}.         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19                                                NaN         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22                                                NaN         NaN   \n",
       " 23            PTM: Autophosphorylated. {ECO:0000250}.         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide                        PDB  \n",
       " 0              NaN             NaN                        NaN  \n",
       " 1              NaN             NaN                        NaN  \n",
       " 2              NaN             NaN                        NaN  \n",
       " 3              NaN             NaN                        NaN  \n",
       " 4              NaN             NaN                        NaN  \n",
       " 5              NaN             NaN                        NaN  \n",
       " 6              NaN             NaN                        NaN  \n",
       " 7              NaN             NaN                        NaN  \n",
       " 8              NaN             NaN                        NaN  \n",
       " 9              NaN             NaN                        NaN  \n",
       " 10             NaN             NaN                        NaN  \n",
       " 11             NaN             NaN                        NaN  \n",
       " 12             NaN             NaN  1E7U;1E7V;1E8W;1E8X;1E90;  \n",
       " 13             NaN             NaN                        NaN  \n",
       " 14             NaN             NaN                        NaN  \n",
       " 15             NaN             NaN                        NaN  \n",
       " 16             NaN             NaN                        NaN  \n",
       " 17             NaN             NaN                        NaN  \n",
       " 18             NaN             NaN                        NaN  \n",
       " 19             NaN             NaN                        NaN  \n",
       " 20             NaN             NaN                        NaN  \n",
       " 21             NaN             NaN                        NaN  \n",
       " 22             NaN             NaN                        NaN  \n",
       " 23             NaN             NaN                        NaN  \n",
       " \n",
       " [24 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P17707  DCAM_HUMAN   AMD1 AMD                 AMD1                  AMD   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  S-adenosylmethionine decarboxylase proenzyme (...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 68; /note=\"Pyruvic acid (Ser); by auto...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1I72;1I79;1I7B;1I7C;1I7M;1JEN;1JL0;1MSV;3DZ2;3...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  P00374   DYR_HUMAN                 DHFR                 DHFR   \n",
       " 1  Q86XF0  DYR2_HUMAN  DHFR2 DHFRL1 DHFRP4                DHFR2   \n",
       " 2  Q2HRC6   DYR_HHV8P                 ORF2                 ORF2   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1        DHFRL1 DHFRP4                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                               Homo sapiens (Human)           9606   \n",
       " 1                               Homo sapiens (Human)           9606   \n",
       " 2  Human herpesvirus 8 type P (isolate GK18) (HHV...         868565   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0               Dihydrofolate reductase (EC 1.5.1.3)  ...           NaN   \n",
       " 1  Dihydrofolate reductase 2, mitochondrial (Dihy...  ...           NaN   \n",
       " 2      Putative Dihydrofolate reductase (EC 1.5.1.3)  ...           NaN   \n",
       " \n",
       "   Initiator methionine Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN        NaN               NaN     NaN   \n",
       " 1                  NaN        NaN               NaN     NaN   \n",
       " 2                  NaN        NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1BOZ;1DHF;1DLR;1DLS;1DRF;1HFP;1HFQ;1HFR;1KMS;1...  \n",
       " 1             NaN                                                NaN  \n",
       " 2             NaN                                                NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry   Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  O15269  SPTC1_HUMAN              SPTLC1 LCB1               SPTLC1   \n",
       " 1  O15270  SPTC2_HUMAN     SPTLC2 KIAA0526 LCB2               SPTLC2   \n",
       " 2  Q9NUV7  SPTC3_HUMAN  SPTLC3 C20orf38 SPTLC2L               SPTLC3   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 LCB1                         NaN               NaN   \n",
       " 1        KIAA0526 LCB2                         NaN               NaN   \n",
       " 2     C20orf38 SPTLC2L                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Serine palmitoyltransferase 1 (EC 2.3.1.50) (L...  ...           NaN   \n",
       " 1  Serine palmitoyltransferase 2 (EC 2.3.1.50) (L...  ...           NaN   \n",
       " 2  Serine palmitoyltransferase 3 (EC 2.3.1.50) (L...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 164; /note=\"Phosphotyrosine; by ABL\"; ...     NaN   \n",
       " 1  MOD_RES 379; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 2  MOD_RES 371; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylation at Tyr-164 inhibits activ...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  6M4N;6M4O;7CQI;7CQK;7K0I;7K0J;7K0K;7K0L;7K0M;7...  \n",
       " 1  6M4N;6M4O;7CQI;7CQK;7K0I;7K0J;7K0K;7K0L;7K0M;7...  \n",
       " 2                                                NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P07934  PHKG1_MOUSE  Phkg1 Phkg                Phkg1                 Phkg   \n",
       " 1  Q9DB30  PHKG2_MOUSE       Phkg2                Phkg2                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 1                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Phosphorylase b kinase gamma catalytic chain, ...  ...   \n",
       " 1          10090  Phosphorylase b kinase gamma catalytic chain, ...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " 1           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P46844   BIEA_RAT  Blvra Blvr                Blvra                 Blvr   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Biliverdin reductase A (BVR A) (EC 1.3.1.24) (...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 154; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  \\\n",
       " 0                              NaN   \n",
       " \n",
       "                                           Propeptide  Signal peptide  \\\n",
       " 0  PROPEP 1..2; /evidence=\"ECO:0000269|PubMed:137...             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  1GCU;1LC0;1LC3;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  H9TB17  AOXA_CAVPO           AOX1 AO                 AOX1   \n",
       " 1  H9TB18  AOXD_CAVPO              AOX4                 AOX4   \n",
       " 2  H9TB19  AOXB_CAVPO  AOX2 AOH3 AOX3L1                 AOX2   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   AO                         NaN               NaN   \n",
       " 1                  NaN                         NaN               NaN   \n",
       " 2          AOH3 AOX3L1                         NaN               NaN   \n",
       " \n",
       "                        Organism  Organism (ID)  \\\n",
       " 0  Cavia porcellus (Guinea pig)          10141   \n",
       " 1  Cavia porcellus (Guinea pig)          10141   \n",
       " 2  Cavia porcellus (Guinea pig)          10141   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aldehyde oxidase 1 (EC 1.2.3.1) (Azaheterocycl...  ...           NaN   \n",
       " 1  Aldehyde oxidase 4 (EC 1.2.3.1) (Aldehyde oxid...  ...           NaN   \n",
       " 2  Aldehyde oxidase 2 (EC 1.2.3.1) (Aldehyde oxid...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  P77961  GLN1B_SYNY3  glnA slr1756                 glnA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    slr1756               NaN   \n",
       " \n",
       "                                        Organism  Organism (ID)  \\\n",
       " 0  Synechocystis sp. (strain PCC 6803 / Kazusa)        1111708   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutamine synthetase (GS) (EC 6.3.1.2) (Glutam...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 401; /note=\"O-AMP-tyrosine\"; /evidence...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  3NG0;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0   P0C089  PTPM1_RAT                     Ptpmt1               Ptpmt1   \n",
       " 1   P20651  PP2BB_RAT                     Ppp3cb               Ppp3cb   \n",
       " 2   B4F7B7  DUS15_RAT                     Dusp15               Dusp15   \n",
       " 3   O54857   PTEN_RAT             Pten rCG_47874                 Pten   \n",
       " 4   Q5I0E6  RPAP2_RAT                      Rpap2                Rpap2   \n",
       " 5   Q64346   DUS6_RAT                 Dusp6 Mkp3                Dusp6   \n",
       " 6   Q80Z30  PPM1E_RAT                Ppm1e Camkn                Ppm1e   \n",
       " 7   Q9WVR7  PPM1F_RAT                Ppm1f Popx2                Ppm1f   \n",
       " 8   O54838   DUS5_RAT                      Dusp5                Dusp5   \n",
       " 9   P53042   PPP5_RAT                      Ppp5c                Ppp5c   \n",
       " 10  P62716  PP2AB_RAT                     Ppp2cb               Ppp2cb   \n",
       " 11  P63329  PP2BA_RAT               Ppp3ca Calna               Ppp3ca   \n",
       " 12  Q5BJ92   PP4C_RAT                      Ppp4c                Ppp4c   \n",
       " 13  Q63340   DUS7_RAT                 Dusp7 Mkpx                Dusp7   \n",
       " 14  Q91XQ2  EPM2A_RAT                      Epm2a                Epm2a   \n",
       " 15  Q9JIM4  DUS12_RAT                Dusp12 Gkap               Dusp12   \n",
       " 16  Q9WTR8  PHLP1_RAT  Phlpp1 Phlpp Plekhe1 Scop               Phlpp1   \n",
       " 17  F1LNI5  PPM1G_RAT                      Ppm1g                Ppm1g   \n",
       " 18  P35815  PPM1B_RAT         Ppm1b Pp2c2 Pppm1b                Ppm1b   \n",
       " 19  P62138   PP1A_RAT               Ppp1ca Ppp1a               Ppp1ca   \n",
       " 20  P63331  PP2AA_RAT                     Ppp2ca               Ppp2ca   \n",
       " 21  Q62767   DUS4_RAT                 Dusp4 Mkp2                Dusp4   \n",
       " 22  Q64623   DUS1_RAT    Dusp1 Cl100 Mkp1 Ptpn16                Dusp1   \n",
       " 23  P20650  PPM1A_RAT         Ppm1a Pp2c1 Pppm1a                Ppm1a   \n",
       " 24  P62142   PP1B_RAT                     Ppp1cb               Ppp1cb   \n",
       " 25  P63088   PP1G_RAT                     Ppp1cc               Ppp1cc   \n",
       " 26  Q8VD52   PLPP_RAT     Pdxp Cin Plp Plpp Rbp1                 Pdxp   \n",
       " 27  Q562B5  PGAM5_RAT                      Pgam5                Pgam5   \n",
       " 28  Q5FVI9  DUS26_RAT                     Dusp26               Dusp26   \n",
       " 29  Q64620   PPP6_RAT                  Ppp6c Ppv                Ppp6c   \n",
       " 30  Q6AXW7  DUS18_RAT                     Dusp18               Dusp18   \n",
       " 31  Q3B7T6  CNEP1_RAT            Ctdnep1 Dullard              Ctdnep1   \n",
       " 32  Q5M821  PPM1H_RAT                      Ppm1h                Ppm1h   \n",
       " 33  P0C595  DUS29_RAT               Dusp29 Dupd1               Dusp29   \n",
       " 34  B2RZ50  CDKN3_RAT                  Cdkn3 Kap                Cdkn3   \n",
       " 35  Q5XIS1   SSH3_RAT                       Ssh3                 Ssh3   \n",
       " 36  Q9Z1Z6  ILKAP_RAT                      Ilkap                Ilkap   \n",
       " 37  Q641Y6  PPM1J_RAT               Ppm1j Ppp2cz                Ppm1j   \n",
       " 38  Q3SWT6   PPE1_RAT                      Ppef1                Ppef1   \n",
       " 39  Q4KLK9  SSU72_RAT                      Ssu72                Ssu72   \n",
       " 40  Q5FWT7  UBCP1_RAT                     Ublcp1               Ublcp1   \n",
       " 41  Q66H71  CPPED_RAT               Cpped1 Cstp1               Cpped1   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN              NaN   \n",
       " 1                   NaN                         NaN              NaN   \n",
       " 2                   NaN                         NaN              NaN   \n",
       " 3                   NaN                         NaN        rCG_47874   \n",
       " 4                   NaN                         NaN              NaN   \n",
       " 5                  Mkp3                         NaN              NaN   \n",
       " 6                 Camkn                         NaN              NaN   \n",
       " 7                 Popx2                         NaN              NaN   \n",
       " 8                   NaN                         NaN              NaN   \n",
       " 9                   NaN                         NaN              NaN   \n",
       " 10                  NaN                         NaN              NaN   \n",
       " 11                Calna                         NaN              NaN   \n",
       " 12                  NaN                         NaN              NaN   \n",
       " 13                 Mkpx                         NaN              NaN   \n",
       " 14                  NaN                         NaN              NaN   \n",
       " 15                 Gkap                         NaN              NaN   \n",
       " 16   Phlpp Plekhe1 Scop                         NaN              NaN   \n",
       " 17                  NaN                         NaN              NaN   \n",
       " 18         Pp2c2 Pppm1b                         NaN              NaN   \n",
       " 19                Ppp1a                         NaN              NaN   \n",
       " 20                  NaN                         NaN              NaN   \n",
       " 21                 Mkp2                         NaN              NaN   \n",
       " 22    Cl100 Mkp1 Ptpn16                         NaN              NaN   \n",
       " 23         Pp2c1 Pppm1a                         NaN              NaN   \n",
       " 24                  NaN                         NaN              NaN   \n",
       " 25                  NaN                         NaN              NaN   \n",
       " 26    Cin Plp Plpp Rbp1                         NaN              NaN   \n",
       " 27                  NaN                         NaN              NaN   \n",
       " 28                  NaN                         NaN              NaN   \n",
       " 29                  Ppv                         NaN              NaN   \n",
       " 30                  NaN                         NaN              NaN   \n",
       " 31              Dullard                         NaN              NaN   \n",
       " 32                  NaN                         NaN              NaN   \n",
       " 33                Dupd1                         NaN              NaN   \n",
       " 34                  Kap                         NaN              NaN   \n",
       " 35                  NaN                         NaN              NaN   \n",
       " 36                  NaN                         NaN              NaN   \n",
       " 37               Ppp2cz                         NaN              NaN   \n",
       " 38                  NaN                         NaN              NaN   \n",
       " 39                  NaN                         NaN              NaN   \n",
       " 40                  NaN                         NaN              NaN   \n",
       " 41                Cstp1                         NaN              NaN   \n",
       " \n",
       "                    Organism  Organism (ID)  \\\n",
       " 0   Rattus norvegicus (Rat)          10116   \n",
       " 1   Rattus norvegicus (Rat)          10116   \n",
       " 2   Rattus norvegicus (Rat)          10116   \n",
       " 3   Rattus norvegicus (Rat)          10116   \n",
       " 4   Rattus norvegicus (Rat)          10116   \n",
       " 5   Rattus norvegicus (Rat)          10116   \n",
       " 6   Rattus norvegicus (Rat)          10116   \n",
       " 7   Rattus norvegicus (Rat)          10116   \n",
       " 8   Rattus norvegicus (Rat)          10116   \n",
       " 9   Rattus norvegicus (Rat)          10116   \n",
       " 10  Rattus norvegicus (Rat)          10116   \n",
       " 11  Rattus norvegicus (Rat)          10116   \n",
       " 12  Rattus norvegicus (Rat)          10116   \n",
       " 13  Rattus norvegicus (Rat)          10116   \n",
       " 14  Rattus norvegicus (Rat)          10116   \n",
       " 15  Rattus norvegicus (Rat)          10116   \n",
       " 16  Rattus norvegicus (Rat)          10116   \n",
       " 17  Rattus norvegicus (Rat)          10116   \n",
       " 18  Rattus norvegicus (Rat)          10116   \n",
       " 19  Rattus norvegicus (Rat)          10116   \n",
       " 20  Rattus norvegicus (Rat)          10116   \n",
       " 21  Rattus norvegicus (Rat)          10116   \n",
       " 22  Rattus norvegicus (Rat)          10116   \n",
       " 23  Rattus norvegicus (Rat)          10116   \n",
       " 24  Rattus norvegicus (Rat)          10116   \n",
       " 25  Rattus norvegicus (Rat)          10116   \n",
       " 26  Rattus norvegicus (Rat)          10116   \n",
       " 27  Rattus norvegicus (Rat)          10116   \n",
       " 28  Rattus norvegicus (Rat)          10116   \n",
       " 29  Rattus norvegicus (Rat)          10116   \n",
       " 30  Rattus norvegicus (Rat)          10116   \n",
       " 31  Rattus norvegicus (Rat)          10116   \n",
       " 32  Rattus norvegicus (Rat)          10116   \n",
       " 33  Rattus norvegicus (Rat)          10116   \n",
       " 34  Rattus norvegicus (Rat)          10116   \n",
       " 35  Rattus norvegicus (Rat)          10116   \n",
       " 36  Rattus norvegicus (Rat)          10116   \n",
       " 37  Rattus norvegicus (Rat)          10116   \n",
       " 38  Rattus norvegicus (Rat)          10116   \n",
       " 39  Rattus norvegicus (Rat)          10116   \n",
       " 40  Rattus norvegicus (Rat)          10116   \n",
       " 41  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Phosphatidylglycerophosphatase and protein-tyr...  ...           NaN   \n",
       " 1   Serine/threonine-protein phosphatase 2B cataly...  ...           NaN   \n",
       " 2   Dual specificity protein phosphatase 15 (EC 3....  ...           NaN   \n",
       " 3   Phosphatidylinositol 3,4,5-trisphosphate 3-pho...  ...           NaN   \n",
       " 4   RNA polymerase II subunit B1 CTD phosphatase R...  ...           NaN   \n",
       " 5   Dual specificity protein phosphatase 6 (EC 3.1...  ...           NaN   \n",
       " 6   Protein phosphatase 1E (EC 3.1.3.16) (Ca(2+)/c...  ...           NaN   \n",
       " 7   Protein phosphatase 1F (EC 3.1.3.16) (Ca(2+)/c...  ...           NaN   \n",
       " 8   Dual specificity protein phosphatase 5 (EC 3.1...  ...           NaN   \n",
       " 9   Serine/threonine-protein phosphatase 5 (PP5) (...  ...           NaN   \n",
       " 10  Serine/threonine-protein phosphatase 2A cataly...  ...           NaN   \n",
       " 11  Protein phosphatase 3 catalytic subunit alpha ...  ...           NaN   \n",
       " 12  Serine/threonine-protein phosphatase 4 catalyt...  ...           NaN   \n",
       " 13  Dual specificity protein phosphatase 7 (EC 3.1...  ...           NaN   \n",
       " 14  Laforin (EC 3.1.3.-) (EC 3.1.3.16) (EC 3.1.3.4...  ...           NaN   \n",
       " 15  Dual specificity protein phosphatase 12 (EC 3....  ...           NaN   \n",
       " 16  PH domain leucine-rich repeat protein phosphat...  ...           NaN   \n",
       " 17  Protein phosphatase 1G (EC 3.1.3.16) (Protein ...  ...           NaN   \n",
       " 18  Protein phosphatase 1B (EC 3.1.3.16) (Protein ...  ...           NaN   \n",
       " 19  Serine/threonine-protein phosphatase PP1-alpha...  ...           NaN   \n",
       " 20  Serine/threonine-protein phosphatase 2A cataly...  ...           NaN   \n",
       " 21  Dual specificity protein phosphatase 4 (EC 3.1...  ...           NaN   \n",
       " 22  Dual specificity protein phosphatase 1 (EC 3.1...  ...           NaN   \n",
       " 23  Protein phosphatase 1A (EC 3.1.3.16) (Protein ...  ...           NaN   \n",
       " 24  Serine/threonine-protein phosphatase PP1-beta ...  ...           NaN   \n",
       " 25  Serine/threonine-protein phosphatase PP1-gamma...  ...           NaN   \n",
       " 26  Chronophin (EC 3.1.3.16) (EC 3.1.3.74) (Pyrido...  ...           NaN   \n",
       " 27  Serine/threonine-protein phosphatase PGAM5, mi...  ...           NaN   \n",
       " 28  Dual specificity protein phosphatase 26 (EC 3....  ...           NaN   \n",
       " 29  Serine/threonine-protein phosphatase 6 catalyt...  ...           NaN   \n",
       " 30  Dual specificity protein phosphatase 18 (EC 3....  ...           NaN   \n",
       " 31  CTD nuclear envelope phosphatase 1 (EC 3.1.3.1...  ...           NaN   \n",
       " 32               Protein phosphatase 1H (EC 3.1.3.16)  ...           NaN   \n",
       " 33  Dual specificity phosphatase 29 (Dual specific...  ...           NaN   \n",
       " 34  Cyclin-dependent kinase inhibitor 3 (EC 3.1.3....  ...           NaN   \n",
       " 35  Protein phosphatase Slingshot homolog 3 (EC 3....  ...           NaN   \n",
       " 36  Integrin-linked kinase-associated serine/threo...  ...           NaN   \n",
       " 37  Protein phosphatase 1J (EC 3.1.3.16) (Protein ...  ...           NaN   \n",
       " 38  Serine/threonine-protein phosphatase with EF-h...  ...           NaN   \n",
       " 39  RNA polymerase II subunit A C-terminal domain ...  ...           NaN   \n",
       " 40  Ubiquitin-like domain-containing CTD phosphata...  ...           NaN   \n",
       " 41  Serine/threonine-protein phosphatase CPPED1 (E...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  \\\n",
       " 0                                                 NaN   \n",
       " 1   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 2                         INIT_MET 1; /note=\"Removed\"   \n",
       " 3   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 4   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7                                                 NaN   \n",
       " 8                                                 NaN   \n",
       " 9   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 10                                                NaN   \n",
       " 11  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 12  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 13                                                NaN   \n",
       " 14                                                NaN   \n",
       " 15                                                NaN   \n",
       " 16                                                NaN   \n",
       " 17  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 18  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 19  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 20                                                NaN   \n",
       " 21  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 22                                                NaN   \n",
       " 23  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 24  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 25  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 26                                                NaN   \n",
       " 27                                                NaN   \n",
       " 28                                                NaN   \n",
       " 29                                                NaN   \n",
       " 30                                                NaN   \n",
       " 31                                                NaN   \n",
       " 32                                                NaN   \n",
       " 33                                                NaN   \n",
       " 34                                                NaN   \n",
       " 35  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 36                                                NaN   \n",
       " 37                                                NaN   \n",
       " 38                                                NaN   \n",
       " 39                                                NaN   \n",
       " 40                                                NaN   \n",
       " 41                                                NaN   \n",
       " \n",
       "                                            Lipidation  \\\n",
       " 0                                                 NaN   \n",
       " 1                                                 NaN   \n",
       " 2   LIPID 2; /note=\"N-myristoyl glycine\"; /evidenc...   \n",
       " 3                                                 NaN   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7                                                 NaN   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " 13                                                NaN   \n",
       " 14                                                NaN   \n",
       " 15                                                NaN   \n",
       " 16                                                NaN   \n",
       " 17  LIPID 2; /note=\"N-myristoyl glycine\"; /evidenc...   \n",
       " 18  LIPID 2; /note=\"N-myristoyl glycine\"; /evidenc...   \n",
       " 19                                                NaN   \n",
       " 20                                                NaN   \n",
       " 21                                                NaN   \n",
       " 22                                                NaN   \n",
       " 23  LIPID 2; /note=\"N-myristoyl glycine\"; /evidenc...   \n",
       " 24                                                NaN   \n",
       " 25                                                NaN   \n",
       " 26                                                NaN   \n",
       " 27                                                NaN   \n",
       " 28                                                NaN   \n",
       " 29                                                NaN   \n",
       " 30                                                NaN   \n",
       " 31                                                NaN   \n",
       " 32                                                NaN   \n",
       " 33                                                NaN   \n",
       " 34                                                NaN   \n",
       " 35                                                NaN   \n",
       " 36                                                NaN   \n",
       " 37                                                NaN   \n",
       " 38                                                NaN   \n",
       " 39                                                NaN   \n",
       " 40                                                NaN   \n",
       " 41                                                NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 85; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " 1   MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3   MOD_RES 2; /note=\"N-acetylthreonine\"; /evidenc...     NaN   \n",
       " 4   MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6   MOD_RES 532; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 7   MOD_RES 450; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9   MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 10  MOD_RES 307; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 11  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 12  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 13                                                NaN     NaN   \n",
       " 14  MOD_RES 25; /note=\"Phosphoserine; by AMPK\"; /e...     NaN   \n",
       " 15  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 16  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 17  MOD_RES 22; /note=\"Omega-N-methylarginine\"; /e...     NaN   \n",
       " 18  MOD_RES 386; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 19  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 20  MOD_RES 307; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 21  MOD_RES 2; /note=\"N-acetylvaline\"; /evidence=\"...     NaN   \n",
       " 22  MOD_RES 359; /note=\"Phosphoserine; by MAPK1 an...     NaN   \n",
       " 23  MOD_RES 375; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 24  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 25  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 26                                                NaN     NaN   \n",
       " 27  MOD_RES 86; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 28                                                NaN     NaN   \n",
       " 29  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 30                                                NaN     NaN   \n",
       " 31                                                NaN     NaN   \n",
       " 32  MOD_RES 7; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " 33                                                NaN     NaN   \n",
       " 34                                                NaN     NaN   \n",
       " 35  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 36  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 37  MOD_RES 41; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 38                                                NaN     NaN   \n",
       " 39                                                NaN     NaN   \n",
       " 40  MOD_RES 117; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 41  MOD_RES 2; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3   PTM: Constitutively phosphorylated by CK2 unde...         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9   PTM: Activated by at least two different prote...         NaN   \n",
       " 10  PTM: Reversibly methyl esterified on Leu-309 b...         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12  PTM: Methylation at the C-terminal Leu-307 is ...         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14  PTM: Polyubiquitinated by NHLRC1/malin. {ECO:0...         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18  PTM: Isgylation negatively regulates its activ...         NaN   \n",
       " 19  PTM: Phosphorylated. Dephosphorylated at Thr-3...         NaN   \n",
       " 20  PTM: Reversibly methyl esterified on Leu-309 b...         NaN   \n",
       " 21  PTM: Phosphorylation in the C-terminus by ERK1...         NaN   \n",
       " 22  PTM: Phosphorylation at Ser-359 and Ser-364 by...         NaN   \n",
       " 23  PTM: N-myristoylation is essential for the rec...         NaN   \n",
       " 24                                                NaN         NaN   \n",
       " 25        PTM: Phosphorylated by NEK2. {ECO:0000250}.         NaN   \n",
       " 26                                                NaN         NaN   \n",
       " 27  PTM: Phosphorylated by the RIPK1/RIPK3 complex...         NaN   \n",
       " 28                                                NaN         NaN   \n",
       " 29                                                NaN         NaN   \n",
       " 30                                                NaN         NaN   \n",
       " 31                                                NaN         NaN   \n",
       " 32                                                NaN         NaN   \n",
       " 33                                                NaN         NaN   \n",
       " 34                                                NaN         NaN   \n",
       " 35                                                NaN         NaN   \n",
       " 36                                                NaN         NaN   \n",
       " 37                                                NaN         NaN   \n",
       " 38                                                NaN         NaN   \n",
       " 39                                                NaN         NaN   \n",
       " 40                                                NaN         NaN   \n",
       " 41                                                NaN         NaN   \n",
       " \n",
       "     Signal peptide                                    Transit peptide  \\\n",
       " 0              NaN  TRANSIT 1..31; /note=\"Mitochondrion\"; /evidenc...   \n",
       " 1              NaN                                                NaN   \n",
       " 2              NaN                                                NaN   \n",
       " 3              NaN                                                NaN   \n",
       " 4              NaN                                                NaN   \n",
       " 5              NaN                                                NaN   \n",
       " 6              NaN                                                NaN   \n",
       " 7              NaN                                                NaN   \n",
       " 8              NaN                                                NaN   \n",
       " 9              NaN                                                NaN   \n",
       " 10             NaN                                                NaN   \n",
       " 11             NaN                                                NaN   \n",
       " 12             NaN                                                NaN   \n",
       " 13             NaN                                                NaN   \n",
       " 14             NaN                                                NaN   \n",
       " 15             NaN                                                NaN   \n",
       " 16             NaN                                                NaN   \n",
       " 17             NaN                                                NaN   \n",
       " 18             NaN                                                NaN   \n",
       " 19             NaN                                                NaN   \n",
       " 20             NaN                                                NaN   \n",
       " 21             NaN                                                NaN   \n",
       " 22             NaN                                                NaN   \n",
       " 23             NaN                                                NaN   \n",
       " 24             NaN                                                NaN   \n",
       " 25             NaN                                                NaN   \n",
       " 26             NaN                                                NaN   \n",
       " 27             NaN                                                NaN   \n",
       " 28             NaN                                                NaN   \n",
       " 29             NaN                                                NaN   \n",
       " 30             NaN                                                NaN   \n",
       " 31             NaN                                                NaN   \n",
       " 32             NaN                                                NaN   \n",
       " 33             NaN                                                NaN   \n",
       " 34             NaN                                                NaN   \n",
       " 35             NaN                                                NaN   \n",
       " 36             NaN                                                NaN   \n",
       " 37             NaN                                                NaN   \n",
       " 38             NaN                                                NaN   \n",
       " 39             NaN                                                NaN   \n",
       " 40             NaN                                                NaN   \n",
       " 41             NaN                                                NaN   \n",
       " \n",
       "            PDB  \n",
       " 0          NaN  \n",
       " 1          NaN  \n",
       " 2          NaN  \n",
       " 3        2K20;  \n",
       " 4          NaN  \n",
       " 5        2FYS;  \n",
       " 6          NaN  \n",
       " 7          NaN  \n",
       " 8          NaN  \n",
       " 9   4JA7;4JA9;  \n",
       " 10         NaN  \n",
       " 11       4IL1;  \n",
       " 12         NaN  \n",
       " 13         NaN  \n",
       " 14         NaN  \n",
       " 15         NaN  \n",
       " 16         NaN  \n",
       " 17         NaN  \n",
       " 18         NaN  \n",
       " 19         NaN  \n",
       " 20         NaN  \n",
       " 21         NaN  \n",
       " 22         NaN  \n",
       " 23         NaN  \n",
       " 24         NaN  \n",
       " 25  2O8A;2O8G;  \n",
       " 26         NaN  \n",
       " 27         NaN  \n",
       " 28         NaN  \n",
       " 29         NaN  \n",
       " 30         NaN  \n",
       " 31         NaN  \n",
       " 32         NaN  \n",
       " 33         NaN  \n",
       " 34         NaN  \n",
       " 35         NaN  \n",
       " 36         NaN  \n",
       " 37         NaN  \n",
       " 38         NaN  \n",
       " 39         NaN  \n",
       " 40         NaN  \n",
       " 41         NaN  \n",
       " \n",
       " [42 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P80644  SSUE_ECOLI  ssuE ycbP b0937 JW0920                 ssuE   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 ycbP               b0937 JW0920               NaN   \n",
       " \n",
       "                         Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  FMN reductase (NADPH) (EC 1.5.1.38) (FMN reduc...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  4PTY;4PTZ;4PU0;6DQI;6DQO;6DQP;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  O31749  PYRH_BACSU     pyrH smbA BSU16510                 pyrH   \n",
       " 1  A7Z4S2  PYRH_BACVZ       pyrH RBAM_016350                 pyrH   \n",
       " 2  Q9KA65  PYRH_HALH5       pyrH smbA BH2425                 pyrH   \n",
       " 3  Q65JJ7  PYRH_BACLD  pyrH BLi01872 BL01242                 pyrH   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 smbA                   BSU16510               NaN   \n",
       " 1                  NaN                RBAM_016350               NaN   \n",
       " 2                 smbA                     BH2425               NaN   \n",
       " 3                  NaN           BLi01872 BL01242               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                     Bacillus subtilis (strain 168)         224308   \n",
       " 1  Bacillus velezensis (strain DSM 23117 / BGSC 1...         326423   \n",
       " 2  Halalkalibacterium halodurans (strain ATCC BAA...         272558   \n",
       " 3  Bacillus licheniformis (strain ATCC 14580 / DS...         279010   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Uridylate kinase (UK) (EC 2.7.4.22) (Uridine m...  ...           NaN   \n",
       " 1  Uridylate kinase (UK) (EC 2.7.4.22) (Uridine m...  ...           NaN   \n",
       " 2  Uridylate kinase (UK) (EC 2.7.4.22) (Uridine m...  ...           NaN   \n",
       " 3  Uridylate kinase (UK) (EC 2.7.4.22) (Uridine m...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name   Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P80969  THT10_TOBAC  THT10 THT30                THT10                THT30   \n",
       " 1  Q9SMB8  THT11_TOBAC  THT4; THT11          THT4; THT11                   ;    \n",
       " \n",
       "   Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                        NaN              NaN   \n",
       " 1                         ;                ;    \n",
       " \n",
       "                              Organism  Organism (ID)  \\\n",
       " 0  Nicotiana tabacum (Common tobacco)           4097   \n",
       " 1  Nicotiana tabacum (Common tobacco)           4097   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Tyramine N-feruloyltransferase 10/30 (EC 2.3.1...  ...           NaN   \n",
       " 1  Tyramine N-feruloyltransferase 4/11 (EC 2.3.1....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "      Entry   Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0   Q9NP87  DPOLM_HUMAN             POLM polmu                 POLM   \n",
       " 1   Q9Y253   POLH_HUMAN  POLH RAD30 RAD30A XPV                 POLH   \n",
       " 2   P09884  DPOLA_HUMAN             POLA1 POLA                POLA1   \n",
       " 3   P28340  DPOD1_HUMAN             POLD1 POLD                POLD1   \n",
       " 4   P54098  DPOG1_HUMAN  POLG MDP1 POLG1 POLGA                 POLG   \n",
       " ..     ...          ...                    ...                  ...   \n",
       " 95  Q69605   DPOL_HBVF6                      P                    P   \n",
       " 96  Q80IU4   DPOL_HBVE4                      P                    P   \n",
       " 97  Q80IU7   DPOL_HBVE2                      P                    P   \n",
       " 98  Q8JMZ7   DPOL_HBVH3                      P                    P   \n",
       " 99  Q8JN08   DPOL_HBVH2                      P                    P   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 polmu                         NaN              NaN   \n",
       " 1      RAD30 RAD30A XPV                         NaN              NaN   \n",
       " 2                  POLA                         NaN              NaN   \n",
       " 3                  POLD                         NaN              NaN   \n",
       " 4      MDP1 POLG1 POLGA                         NaN              NaN   \n",
       " ..                  ...                         ...              ...   \n",
       " 95                  NaN                         NaN              NaN   \n",
       " 96                  NaN                         NaN              NaN   \n",
       " 97                  NaN                         NaN              NaN   \n",
       " 98                  NaN                         NaN              NaN   \n",
       " 99                  NaN                         NaN              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                Homo sapiens (Human)           9606   \n",
       " 1                                Homo sapiens (Human)           9606   \n",
       " 2                                Homo sapiens (Human)           9606   \n",
       " 3                                Homo sapiens (Human)           9606   \n",
       " 4                                Homo sapiens (Human)           9606   \n",
       " ..                                                ...            ...   \n",
       " 95  Hepatitis B virus genotype F2 subtype adw4q (i...         489503   \n",
       " 96  Hepatitis B virus genotype E (isolate Cote d'I...         489498   \n",
       " 97  Hepatitis B virus genotype E (isolate Cote d'I...         489496   \n",
       " 98  Hepatitis B virus genotype H subtype adw4 (iso...         489541   \n",
       " 99  Hepatitis B virus genotype H subtype adw4 (iso...         489540   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   DNA-directed DNA/RNA polymerase mu (Pol Mu) (E...  ...           NaN   \n",
       " 1   DNA polymerase eta (EC 2.7.7.7) (RAD30 homolog...  ...           NaN   \n",
       " 2   DNA polymerase alpha catalytic subunit (EC 2.7...  ...           NaN   \n",
       " 3   DNA polymerase delta catalytic subunit (EC 2.7...  ...           NaN   \n",
       " 4   DNA polymerase subunit gamma-1 (EC 2.7.7.7) (M...  ...           NaN   \n",
       " ..                                                ...  ...           ...   \n",
       " 95  Protein P [Includes: DNA-directed DNA polymera...  ...           NaN   \n",
       " 96  Protein P [Includes: DNA-directed DNA polymera...  ...           NaN   \n",
       " 97  Protein P [Includes: DNA-directed DNA polymera...  ...           NaN   \n",
       " 98  Protein P [Includes: DNA-directed DNA polymera...  ...           NaN   \n",
       " 99  Protein P [Includes: DNA-directed DNA polymera...  ...           NaN   \n",
       " \n",
       "    Initiator methionine Lipidation  \\\n",
       " 0                   NaN        NaN   \n",
       " 1                   NaN        NaN   \n",
       " 2                   NaN        NaN   \n",
       " 3                   NaN        NaN   \n",
       " 4                   NaN        NaN   \n",
       " ..                  ...        ...   \n",
       " 95                  NaN        NaN   \n",
       " 96                  NaN        NaN   \n",
       " 97                  NaN        NaN   \n",
       " 98                  NaN        NaN   \n",
       " 99                  NaN        NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 12; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1                                                 NaN     NaN   \n",
       " 2   MOD_RES 174; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 3   MOD_RES 19; /note=\"Omega-N-methylarginine\"; /e...     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " ..                                                ...     ...   \n",
       " 95                                                NaN     NaN   \n",
       " 96                                                NaN     NaN   \n",
       " 97                                                NaN     NaN   \n",
       " 98                                                NaN     NaN   \n",
       " 99                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1   PTM: Monoubiquitinated by RCHY1/PIRH2 (PubMed:...         NaN   \n",
       " 2   PTM: A 165 kDa form is probably produced by pr...         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " ..                                                ...         ...   \n",
       " 95                                                NaN         NaN   \n",
       " 96                                                NaN         NaN   \n",
       " 97                                                NaN         NaN   \n",
       " 98                                                NaN         NaN   \n",
       " 99                                                NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  \\\n",
       " 0              NaN             NaN   \n",
       " 1              NaN             NaN   \n",
       " 2              NaN             NaN   \n",
       " 3              NaN             NaN   \n",
       " 4              NaN             NaN   \n",
       " ..             ...             ...   \n",
       " 95             NaN             NaN   \n",
       " 96             NaN             NaN   \n",
       " 97             NaN             NaN   \n",
       " 98             NaN             NaN   \n",
       " 99             NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0   2DUN;2HTF;4LZD;4LZG;4M04;4M0A;4YCX;4YD1;4YD2;5...  \n",
       " 1   2I5O;2LSK;3JAA;3MR2;3MR3;3MR5;3MR6;3SI8;3TQ1;3...  \n",
       " 2   1K0P;1K18;1N5G;4Q5V;4QCL;4Y97;5EXR;5IUD;6AS7;7...  \n",
       " 3                           6S1M;6S1N;6S1O;6TNY;6TNZ;  \n",
       " 4                      3IKM;4ZTU;4ZTZ;5C51;5C52;5C53;  \n",
       " ..                                                ...  \n",
       " 95                                                NaN  \n",
       " 96                                                NaN  \n",
       " 97                                                NaN  \n",
       " 98                                                NaN  \n",
       " 99                                                NaN  \n",
       " \n",
       " [100 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9M571  PEAMT_SPIOL      PEAMT                PEAMT                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                     Organism  \\\n",
       " 0                         NaN               NaN  Spinacia oleracea (Spinach)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           3562  Phosphoethanolamine N-methyltransferase (EC 2....  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  P9WJJ3  NADE_MYCTU  nadE Rv2438c MTCY428.08                 nadE   \n",
       " 1  P9WJJ2  NADE_MYCTO              nadE MT2513                 nadE   \n",
       " 2  P0A5L7  NADE_MYCBO      nadE BQ2027_MB2464C                 nadE   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    Rv2438c       MTCY428.08   \n",
       " 1                   NaN                     MT2513              NaN   \n",
       " 2                   NaN             BQ2027_MB2464C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 2  Mycobacterium bovis (strain ATCC BAA-935 / AF2...         233413   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutamine-dependent NAD(+) synthetase (EC 6.3....  ...           NaN   \n",
       " 1  Glutamine-dependent NAD(+) synthetase (EC 6.3....  ...           NaN   \n",
       " 2  Glutamine-dependent NAD(+) synthetase (EC 6.3....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  3DLA;3SDB;3SEQ;3SEZ;3SYT;3SZG;  \n",
       " 1             NaN                             NaN  \n",
       " 2             NaN                             NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  Q8I566  GLYA_PLAF7  SHMT PF3D7_1235600                 SHMT   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN    PF3D7_1235600   \n",
       " \n",
       "                               Organism  Organism (ID)  \\\n",
       " 0  Plasmodium falciparum (isolate 3D7)          36329   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Serine hydroxymethyltransferase (EC 2.1.2.1) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 237; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  4O6Z;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                     Gene Names Gene Names (primary)  \\\n",
       " 0  A6T7D6  PYRC_KLEP7  pyrC KPN78578_10460 KPN_01074                 pyrC   \n",
       " 1  B5XXJ5  PYRC_KLEP3                  pyrC KPK_3483                 pyrC   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN             KPN78578_10460        KPN_01074   \n",
       " 1                   NaN                   KPK_3483              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Klebsiella pneumoniae subsp. pneumoniae (strai...         272620   \n",
       " 1                 Klebsiella pneumoniae (strain 342)         507522   \n",
       " \n",
       "                           Protein names  ... Glycosylation  \\\n",
       " 0  Dihydroorotase (DHOase) (EC 3.5.2.3)  ...           NaN   \n",
       " 1  Dihydroorotase (DHOase) (EC 3.5.2.3)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 103; /note=\"N6-carboxylysine\"; /eviden...     NaN   \n",
       " 1  MOD_RES 103; /note=\"N6-carboxylysine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  B7V467  MASZ_PSEA8  glcB PLES_04781                 glcB   \n",
       " 1  A6UYU5  MASZ_PSEA7  glcB PSPA7_0585                 glcB   \n",
       " 2  Q02TT1  MASZ_PSEAB  glcB PA14_06290                 glcB   \n",
       " 3  Q9I636  MASZ_PSEAE      glcB PA0482                 glcB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                 PLES_04781               NaN   \n",
       " 1                   NaN                 PSPA7_0585               NaN   \n",
       " 2                   NaN                 PA14_06290               NaN   \n",
       " 3                   NaN                     PA0482               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0             Pseudomonas aeruginosa (strain LESB58)         557722   \n",
       " 1                Pseudomonas aeruginosa (strain PA7)         381754   \n",
       " 2         Pseudomonas aeruginosa (strain UCBPP-PA14)         208963   \n",
       " 3  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " \n",
       "                     Protein names  ... Glycosylation Initiator methionine  \\\n",
       " 0  Malate synthase G (EC 2.3.3.9)  ...           NaN                  NaN   \n",
       " 1  Malate synthase G (EC 2.3.3.9)  ...           NaN                  NaN   \n",
       " 2  Malate synthase G (EC 2.3.3.9)  ...           NaN                  NaN   \n",
       " 3  Malate synthase G (EC 2.3.3.9)  ...           NaN                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 617; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 1         NaN  MOD_RES 617; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 2         NaN  MOD_RES 617; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 3         NaN  MOD_RES 617; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN         NaN  \n",
       " 1             NaN         NaN  \n",
       " 2             NaN         NaN  \n",
       " 3             NaN  5OAS;5VFB;  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  Q7S332  KYNU1_NEUCR  kyn-1 bna5-1 NCU09183                kyn-1   \n",
       " 1  Q7RXY2  KYNU2_NEUCR  kyn-2 bna5-2 NCU00463                kyn-2   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0               bna5-1                         NaN         NCU09183   \n",
       " 1               bna5-2                         NaN         NCU00463   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Neurospora crassa (strain ATCC 24698 / 74-OR23...         367110   \n",
       " 1  Neurospora crassa (strain ATCC 24698 / 74-OR23...         367110   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Kynureninase 1 (EC 3.7.1.3) (Biosynthesis of n...  ...           NaN   \n",
       " 1  Kynureninase 2 (EC 3.7.1.3) (Biosynthesis of n...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 293; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 1  MOD_RES 273; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P18429  XYNA_BACSU  xynA BSU18840                 xynA   \n",
       " 1  P07528  XYNA_HALH5    xynA BH2120                 xynA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   BSU18840               NaN   \n",
       " 1                   NaN                     BH2120               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                     Bacillus subtilis (strain 168)         224308   \n",
       " 1  Halalkalibacterium halodurans (strain ATCC BAA...         272558   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Endo-1,4-beta-xylanase A (Xylanase A) (EC 3.2....  ...           NaN   \n",
       " 1  Endo-1,4-beta-xylanase A (Xylanase A) (EC 3.2....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " 1                              NaN         NaN   \n",
       " \n",
       "                                 Signal peptide Transit peptide  \\\n",
       " 0                                 SIGNAL 1..28             NaN   \n",
       " 1  SIGNAL 1..28; /evidence=\"ECO:0000269|Ref.1\"             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1AXK;1XXN;2B42;2B45;2B46;2DCY;2DCZ;2QZ3;2Z79;3...  \n",
       " 1                                    2UWF;7CPK;7CPL;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P17256   KIME_RAT        Mvk                  Mvk                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                         Protein names  ... Glycosylation  \\\n",
       " 0          10116  Mevalonate kinase (MK) (EC 2.7.1.36)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1KVK;2R42;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name                        Gene Names Gene Names (primary)  \\\n",
       " 0   P0A9B2  G3P1_ECOLI                 gapA b1779 JW1768                 gapA   \n",
       " 1   P0A9B3   G3P_ECOL6                        gapA c2184                 gapA   \n",
       " 2   P0C0G6   G3P_STRPY                      gap gapA plr                  gap   \n",
       " 3   P04406   G3P_HUMAN  GAPDH GAPD CDABP0047 OK/SW-cl.12                GAPDH   \n",
       " 4   Q9ESV6    G3PT_RAT               Gapdhs Gapd-s Gapds               Gapdhs   \n",
       " 5   P17721   G3P_THEMA                       gap TM_0688                  gap   \n",
       " 6   P00362   G3P_GEOSE                               gap                  gap   \n",
       " 7   P58072  G3P2_ECO57                gapC Z2304 ECs2022                 gapC   \n",
       " 8   P0A9B4  G3P1_ECO57                gapA Z2818 ECs2488                 gapA   \n",
       " 9   B7LQ20   G3P_ESCF3                gapA gap EFER_1293                 gapA   \n",
       " 10  Q59800   G3P_KITAU                               gap                  gap   \n",
       " 11  P24164   G3P_KLEPN                               gap                  gap   \n",
       " 12  P0A9B5   G3P_SHIFL                 gapA SF1444 S1559                 gapA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)       Gene Names (ORF)  \\\n",
       " 0                   NaN               b1779 JW1768                    NaN   \n",
       " 1                   NaN                      c2184                    NaN   \n",
       " 2              gapA plr                        NaN                    NaN   \n",
       " 3                  GAPD                        NaN  CDABP0047 OK/SW-cl.12   \n",
       " 4          Gapd-s Gapds                        NaN                    NaN   \n",
       " 5                   NaN                    TM_0688                    NaN   \n",
       " 6                   NaN                        NaN                    NaN   \n",
       " 7                   NaN              Z2304 ECs2022                    NaN   \n",
       " 8                   NaN              Z2818 ECs2488                    NaN   \n",
       " 9                   gap                  EFER_1293                    NaN   \n",
       " 10                  NaN                        NaN                    NaN   \n",
       " 11                  NaN                        NaN                    NaN   \n",
       " 12                  NaN               SF1444 S1559                    NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1   Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 2                              Streptococcus pyogenes           1314   \n",
       " 3                                Homo sapiens (Human)           9606   \n",
       " 4                             Rattus norvegicus (Rat)          10116   \n",
       " 5   Thermotoga maritima (strain ATCC 43589 / DSM 3...         243274   \n",
       " 6   Geobacillus stearothermophilus (Bacillus stear...           1422   \n",
       " 7                            Escherichia coli O157:H7          83334   \n",
       " 8                            Escherichia coli O157:H7          83334   \n",
       " 9   Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 10  Kitasatospora aureofaciens (Streptomyces aureo...           1894   \n",
       " 11                              Klebsiella pneumoniae            573   \n",
       " 12                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ...  \\\n",
       " 0   Glyceraldehyde-3-phosphate dehydrogenase A (GA...  ...   \n",
       " 1   Glyceraldehyde-3-phosphate dehydrogenase (GAPD...  ...   \n",
       " 2   Glyceraldehyde-3-phosphate dehydrogenase (GAPD...  ...   \n",
       " 3   Glyceraldehyde-3-phosphate dehydrogenase (GAPD...  ...   \n",
       " 4   Glyceraldehyde-3-phosphate dehydrogenase, test...  ...   \n",
       " 5   Glyceraldehyde-3-phosphate dehydrogenase (GAPD...  ...   \n",
       " 6   Glyceraldehyde-3-phosphate dehydrogenase (GAPD...  ...   \n",
       " 7   Glyceraldehyde-3-phosphate dehydrogenase C (GA...  ...   \n",
       " 8   Glyceraldehyde-3-phosphate dehydrogenase A (GA...  ...   \n",
       " 9   Glyceraldehyde-3-phosphate dehydrogenase (GAPD...  ...   \n",
       " 10  Glyceraldehyde-3-phosphate dehydrogenase (GAPD...  ...   \n",
       " 11  Glyceraldehyde-3-phosphate dehydrogenase (GAPD...  ...   \n",
       " 12  Glyceraldehyde-3-phosphate dehydrogenase (GAPD...  ...   \n",
       " \n",
       "                                         Glycosylation  \\\n",
       " 0                                                 NaN   \n",
       " 1                                                 NaN   \n",
       " 2                                                 NaN   \n",
       " 3   CARBOHYD 197; /note=\"(Microbial infection) N-b...   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7                                                 NaN   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 6   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 115; /note=\"N6-succinyllysine\"; /evide...     NaN   \n",
       " 1   MOD_RES 132; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3   MOD_RES 5; /note=\"N6,N6-dimethyllysine\"; /evid...     NaN   \n",
       " 4   MOD_RES 350; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6                                                 NaN     NaN   \n",
       " 7                                                 NaN     NaN   \n",
       " 8   MOD_RES 132; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 9                                                 NaN     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11                                                NaN     NaN   \n",
       " 12  MOD_RES 132; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3   PTM: S-nitrosylation of Cys-152 leads to inter...         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  \\\n",
       " 0              NaN             NaN   \n",
       " 1              NaN             NaN   \n",
       " 2              NaN             NaN   \n",
       " 3              NaN             NaN   \n",
       " 4              NaN             NaN   \n",
       " 5              NaN             NaN   \n",
       " 6              NaN             NaN   \n",
       " 7              NaN             NaN   \n",
       " 8              NaN             NaN   \n",
       " 9              NaN             NaN   \n",
       " 10             NaN             NaN   \n",
       " 11             NaN             NaN   \n",
       " 12             NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0   1DC3;1DC4;1DC5;1DC6;1GAD;1GAE;1S7C;2VYN;2VYV;5...  \n",
       " 1                                                 NaN  \n",
       " 2                                               6ITE;  \n",
       " 3   1U8F;1ZNQ;3GPD;4WNC;4WNI;6ADE;6IQ6;6M61;6YND;6...  \n",
       " 4                                          2VYN;2VYV;  \n",
       " 5                                               1HDG;  \n",
       " 6   1DBV;1GD1;1NPT;1NQ5;1NQA;1NQO;2DBV;2GD1;3CMC;3...  \n",
       " 7                                                 NaN  \n",
       " 8                                                 NaN  \n",
       " 9                                                 NaN  \n",
       " 10                                                NaN  \n",
       " 11                                                NaN  \n",
       " 12                                                NaN  \n",
       " \n",
       " [13 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P11217  PYGM_HUMAN       PYGM                 PYGM                   NaN   \n",
       " 1  P06737  PYGL_HUMAN       PYGL                 PYGL                   NaN   \n",
       " 2  P11216  PYGB_HUMAN       PYGB                 PYGB                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " 2                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Glycogen phosphorylase, muscle form (EC 2.4.1....  ...   \n",
       " 1           9606    Glycogen phosphorylase, liver form (EC 2.4.1.1)  ...   \n",
       " 2           9606    Glycogen phosphorylase, brain form (EC 2.4.1.1)  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 2           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 1         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylation of Ser-15 converts phosph...         NaN   \n",
       " 1  PTM: Acetylation, which is up-regulated by glu...         NaN   \n",
       " 2  PTM: Phosphorylated (PubMed:27402852). Phospho...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                              1Z8D;  \n",
       " 1  1EM6;1EXV;1FA9;1FC0;1L5Q;1L5R;1L5S;1L7X;1XOI;2...  \n",
       " 2                                         5IKO;5IKP;  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P09373  PFLB_ECOLI   pflB pfl b0903 JW0886                 pflB   \n",
       " 1  P42632  TDCE_ECOLI  tdcE yhaS b3114 JW5522                 tdcE   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  pfl               b0903 JW0886               NaN   \n",
       " 1                 yhaS               b3114 JW5522               NaN   \n",
       " \n",
       "                         Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli (strain K12)          83333   \n",
       " 1  Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Formate acetyltransferase 1 (EC 2.3.1.54) (Pyr...  ...           NaN   \n",
       " 1  PFL-like enzyme TdcE (Keto-acid formate acetyl...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 63; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " 1  MOD_RES 739; /note=\"Glycine radical\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                       PDB  \n",
       " 0             NaN  1CM5;1H16;1H17;1H18;1MZO;1QHM;2PFL;3PFL;  \n",
       " 1             NaN                                       NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P95442  ATZB_PSESD       atzB                 atzB                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                      Organism  \\\n",
       " 0                         NaN               NaN  Pseudomonas sp. (strain ADP)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          47660  Hydroxydechloroatrazine ethylaminohydrolase (H...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide Signal peptide Transit peptide  \\\n",
       " 0                              NaN         NaN            NaN             NaN   \n",
       " \n",
       "    PDB  \n",
       " 0  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P21795  LMO_MYCSM         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycolicibacterium smegmatis (Mycobacterium sme...           1772   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  L-lactate 2-monooxygenase (LMO) (EC 1.13.12.4)...  ...            NaN   \n",
       " \n",
       "           Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  6DVH;6DVI;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  Q12382  DGK1_YEAST  DGK1 HSD1 YOR311C O6111                 DGK1   \n",
       " 1  Q753I3  DGK1_ASHGO        DGK1 HSD1 AFR329C                 DGK1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 HSD1                    YOR311C            O6111   \n",
       " 1                 HSD1                    AFR329C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Ashbya gossypii (strain ATCC 10895 / CBS 109.5...         284811   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  CTP-dependent diacylglycerol kinase 1 (EC 2.7....  ...   \n",
       " 1  CTP-dependent diacylglycerol kinase 1 (EC 2.7....  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 11; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 1                                                NaN                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 44; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1         NaN                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: CKII-mediated phosphorylation of Ser-45 a...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name                                         Gene Names  \\\n",
       " 0  Q6Z5I0  CPS2_ORYSJ  CPS2 CYC2 Os02g0571100 LOC_Os02g36210 OsJ_0069...   \n",
       " 1  Q6ET36  CPS1_ORYSJ       CPS1 Os02g0278700 LOC_Os02g17780 P0444A09.11   \n",
       " 2  Q5MQ85  CPS2_ORYSI                               CPS2 CYC2 OsI_007598   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym)   Gene Names (ordered locus)  \\\n",
       " 0                 CPS2                 CYC2  Os02g0571100 LOC_Os02g36210   \n",
       " 1                 CPS1                  NaN  Os02g0278700 LOC_Os02g17780   \n",
       " 2                 CPS2                 CYC2                          NaN   \n",
       " \n",
       "          Gene Names (ORF)                             Organism  Organism (ID)  \\\n",
       " 0  OsJ_006996 P0689H05.16  Oryza sativa subsp. japonica (Rice)          39947   \n",
       " 1             P0444A09.11  Oryza sativa subsp. japonica (Rice)          39947   \n",
       " 2              OsI_007598    Oryza sativa subsp. indica (Rice)          39946   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Ent-copalyl diphosphate synthase 2, chloroplas...  ...           NaN   \n",
       " 1  Ent-copalyl diphosphate synthase 1, chloroplas...  ...           NaN   \n",
       " 2  Ent-copalyl diphosphate synthase 2 (Ent-CDP sy...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide PDB  \n",
       " 0  TRANSIT 1..47; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " 1  TRANSIT 1..35; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " 2                                                NaN NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P17812  PYRG1_HUMAN  CTPS1 CTPS                CTPS1                 CTPS   \n",
       " 1  Q9NRF8  PYRG2_HUMAN       CTPS2                CTPS2                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  CTP synthase 1 (EC 6.3.4.2) (CTP synthetase 1)...  ...   \n",
       " 1           9606  CTP synthase 2 (EC 6.3.4.2) (CTP synthetase 2)...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 100; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 1  MOD_RES 568; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                       PDB  \n",
       " 0             NaN            2VO1;5U03;7MGZ;7MH0;7MIF;7MIG;  \n",
       " 1             NaN  2V4U;2VKT;3IHL;6PK4;6PK7;7MH1;7MIH;7MII;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P31939  PUR9_HUMAN  ATIC PURH OK/SW-cl.86                 ATIC   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 PURH                         NaN      OK/SW-cl.86   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Bifunctional purine biosynthesis protein ATIC ...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed; alternate\"; /evide...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                        PDB  \n",
       " 0             NaN  1P4R;1PKX;1PL0;5UY8;5UZ0;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9BDJ5   VNN1_PIG       VNN1                 VNN1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Sus scrofa (Pig)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9823  Pantetheinase (EC 3.5.1.92) (Pantetheine hydro...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 147; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "                                           Lipidation  Modified residue  \\\n",
       " 0  LIPID 487; /note=\"GPI-anchor amidated glycine\"...               NaN   \n",
       " \n",
       "   Peptide  Post-translational modification  \\\n",
       " 0     NaN                              NaN   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 488..513; /note=\"Removed in mature form...   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..22; /evidence=\"ECO:0000250\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P14168  RFBS_SALTI  rfbS STY2299 t0783                 rfbS   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN              STY2299 t0783               NaN   \n",
       " \n",
       "            Organism  Organism (ID)                         Protein names  ...  \\\n",
       " 0  Salmonella typhi          90370  CDP-paratose synthase (EC 1.1.1.342)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  Q59931  GAPN_STRMU  gapN SMU_676                 gapN   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    SMU_676               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Streptococcus mutans serotype c (strain ATCC 7...         210007   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  NADP-dependent glyceraldehyde-3-phosphate dehy...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                  PDB  \n",
       " 0             NaN  1EUH;1QI1;1QI6;2ESD;2EUH;2ID2;2QE0;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P50213  IDH3A_HUMAN      IDH3A                IDH3A                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Isocitrate dehydrogenase [NAD] subunit alpha, ...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 77; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  \\\n",
       " 0  TRANSIT 1..27; /note=\"Mitochondrion\"; /evidenc...   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  5GRE;5GRF;5GRH;5GRI;5GRL;5YVT;6KDE;6KDF;6KDY;6...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q16719  KYNU_HUMAN       KYNU                 KYNU                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Kynureninase (EC 3.7.1.3) (L-kynurenine hydrol...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  2HZP;3E9K;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name                        Gene Names Gene Names (primary)  \\\n",
       " 0  Q59RA2  PIS_CANAL  PIS1 orf19.6860 CAALFM_C405210WA                 PIS1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                 orf19.6860  CAALFM_C405210WA   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Candida albicans (strain SC5314 / ATCC MYA-287...         237561   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  CDP-diacylglycerol--inositol 3-phosphatidyltra...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q76EQ0    SRR_RAT        Srr                  Srr                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Serine racemase (EC 5.1.1.18) (D-serine ammoni...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 56; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: S-nitrosylated, leading to decrease the e...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide         PDB  \n",
       " 0             NaN             NaN  3HMK;3L6C;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P42357  HUTH_HUMAN    HAL HIS                  HAL                  HIS   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                     Protein names  ...  \\\n",
       " 0           9606  Histidine ammonia-lyase (Histidase) (EC 4.3.1.3)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 254; /note=\"2,3-didehydroalanine (Ser)...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Contains an active site 4-methylidene-imi...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names  Gene Names (primary)  \\\n",
       " 0  Q8TZE8  ASPG_PYRFU              PF2047                   NaN   \n",
       " 1  Q8U4E6  ASGX_PYRFU              PF0142                   NaN   \n",
       " 2  Q9V262  ASGX_PYRAB  PYRAB02120 PAB0145                   NaN   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                     PF2047              NaN   \n",
       " 1                   NaN                     PF0142              NaN   \n",
       " 2                   NaN                 PYRAB02120          PAB0145   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pyrococcus furiosus (strain ATCC 43587 / DSM 3...         186497   \n",
       " 1  Pyrococcus furiosus (strain ATCC 43587 / DSM 3...         186497   \n",
       " 2             Pyrococcus abyssi (strain GE5 / Orsay)         272844   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0                        L-asparaginase (EC 3.5.1.1)  ...           NaN   \n",
       " 1  Putative L-asparaginase (EC 3.5.1.1) (L-aspara...  ...           NaN   \n",
       " 2  Putative L-asparaginase (EC 3.5.1.1) (L-aspara...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: Cleaved into an alpha and beta chain by a...         NaN   \n",
       " 2  PTM: Cleaved into an alpha and beta chain by a...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                                  PDB  \n",
       " 0             NaN             NaN  4NJE;4Q0M;4RA6;4RA9;5B5U;5B74;5CBP;  \n",
       " 1             NaN             NaN                                  NaN  \n",
       " 2             NaN             NaN                                  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  P43409  KADA_METJA  adkA adk MJ0479                 adkA   \n",
       " 1  Q58450  KAD6_METJA           MJ1050                  NaN   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  adk                     MJ0479               NaN   \n",
       " 1                  NaN                     MJ1050               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " 1  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Adenylate kinase (AK) (EC 2.7.4.3) (ATP-AMP tr...  ...           NaN   \n",
       " 1  Putative adenylate kinase (AK) (EC 2.7.4.3) (A...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0  P9WIL5  PANC_MYCTU  panC Rv3602c MTCY07H7B.20                 panC   \n",
       " 1  A5U8S7  PANC_MYCTA              panC MRA_3641                 panC   \n",
       " 2  P9WIL4  PANC_MYCTO                panC MT3707                 panC   \n",
       " 3  A1KPT7  PANC_MYCBP             panC BCG_3666c                 panC   \n",
       " 4  P0A5R1  PANC_MYCBO        panC BQ2027_MB3632C                 panC   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    Rv3602c     MTCY07H7B.20   \n",
       " 1                   NaN                   MRA_3641              NaN   \n",
       " 2                   NaN                     MT3707              NaN   \n",
       " 3                   NaN                  BCG_3666c              NaN   \n",
       " 4                   NaN             BQ2027_MB3632C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain ATCC 25177 ...         419947   \n",
       " 2  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 3  Mycobacterium bovis (strain BCG / Pasteur 1173P2)         410289   \n",
       " 4  Mycobacterium bovis (strain ATCC BAA-935 / AF2...         233413   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Pantothenate synthetase (PS) (EC 6.3.2.1) (Pan...  ...           NaN   \n",
       " 1  Pantothenate synthetase (PS) (EC 6.3.2.1) (Pan...  ...           NaN   \n",
       " 2  Pantothenate synthetase (PS) (EC 6.3.2.1) (Pan...  ...           NaN   \n",
       " 3  Pantothenate synthetase (PS) (EC 6.3.2.1) (Pan...  ...           NaN   \n",
       " 4  Pantothenate synthetase (PS) (EC 6.3.2.1) (Pan...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylthreonine\"; /evidenc...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " 4                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1MOP;1N2B;1N2E;1N2G;1N2H;1N2I;1N2J;1N2O;2A7X;2...  \n",
       " 1             NaN                                                NaN  \n",
       " 2             NaN  4MQ6;4MUE;4MUF;4MUG;4MUH;4MUI;4MUJ;4MUK;4MUL;4...  \n",
       " 3             NaN                                                NaN  \n",
       " 4             NaN                                                NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P49081  MASY_MAIZE        LIP                  LIP                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Zea mays (Maize)   \n",
       " \n",
       "    Organism (ID)                              Protein names  ...  \\\n",
       " 0           4577  Malate synthase, glyoxysomal (EC 2.3.3.9)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  P49419  AL7A1_HUMAN  ALDH7A1 ATQ1              ALDH7A1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 ATQ1                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Alpha-aminoadipic semialdehyde dehydrogenase (...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET P49419-2:1; /note=\"Removed\"; /evidenc...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 94; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  \\\n",
       " 0  TRANSIT 1..26; /note=\"Mitochondrion\"; /evidenc...   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  2J6L;4X0T;4X0U;4ZUK;4ZUL;4ZVW;4ZVX;4ZVY;6O4B;6...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P00743  FA10_BOVIN        F10                  F10                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Coagulation factor X (EC 3.4.21.6) (Stuart fac...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 208; /note=\"O-linked (GalNAc...) thre...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 46; /note=\"4-carboxyglutamate\"; /evide...     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: The vitamin K-dependent, enzymatic carbox...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 24..40; /evidence=\"ECO:0000269|PubMed:6...   \n",
       " \n",
       "                           Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN   \n",
       " \n",
       "                               PDB  \n",
       " 0  1APO;1CCF;1IOD;1KIG;1WHE;1WHF;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                                Gene Names  \\\n",
       " 0  Q9BV86  NTM1A_HUMAN  NTMT1 C9orf32 METTL11A NRMT NRMT1 AD-003   \n",
       " \n",
       "   Gene Names (primary)         Gene Names (synonym)  \\\n",
       " 0                NTMT1  C9orf32 METTL11A NRMT NRMT1   \n",
       " \n",
       "    Gene Names (ordered locus) Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN           AD-003  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  N-terminal Xaa-Pro-Lys N-methyltransferase 1 (...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed; alternate\"; /evide...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  2EX4;5CVD;5CVE;5E1B;5E1D;5E1M;5E1O;5E2A;5E2B;6...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  D3ZG27  UBIA1_RAT     Ubiad1               Ubiad1                   NaN   \n",
       " 1  Q499N4   COQ2_RAT       Coq2                 Coq2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  UbiA prenyltransferase domain-containing prote...  ...   \n",
       " 1          10116  4-hydroxybenzoate polyprenyltransferase, mitoc...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN                                                NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 1         NaN                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0                                                NaN  NaN  \n",
       " 1  TRANSIT 1..63; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9S467  ARAA_GEOSE       araA                 araA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Geobacillus stearothermophilus (Bacillus stear...           1422   \n",
       " \n",
       "                         Protein names  ...  Glycosylation  \\\n",
       " 0  L-arabinose isomerase (EC 5.3.1.4)  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name   Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P34914   HYES_MOUSE   Ephx2 Eph2                Ephx2                 Eph2   \n",
       " 1  Q3V1F8  EPHX3_MOUSE  Ephx3 Abhd9                Ephx3                Abhd9   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 1                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Bifunctional epoxide hydrolase 2 [Includes: Cy...  ...   \n",
       " 1          10090  Epoxide hydrolase 3 (EH3) (EC 3.3.2.10) (Abhyd...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  \\\n",
       " 0           NaN                  NaN   \n",
       " 1           NaN                  NaN   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0  LIPID 521; /note=\"S-(15-deoxy-Delta12,14-prost...   \n",
       " 1                                                NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 55; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: The N-terminus is blocked.; PTM: The cova...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                   PDB  \n",
       " 0             NaN             NaN  1CQZ;1CR6;1EK1;1EK2;  \n",
       " 1             NaN             NaN                   NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  P31373  CYS3_YEAST  CYS3 CYI1 STR1 YAL012W FUN35                 CYS3   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0            CYI1 STR1                    YAL012W            FUN35   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Cystathionine gamma-lyase (gamma-CTLase) (EC 4...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 204; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1N8P;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  P17118  THPS_SULAC  thpS Saci_1714                 thpS   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                  Saci_1714               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Sulfolobus acidocaldarius (strain ATCC 33909 /...         330779   \n",
       " \n",
       "                Protein names  ...  \\\n",
       " 0  Thermopsin (EC 3.4.23.42)  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 31; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 29..41; /evidence=\"ECO:0000269|PubMed:2...   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..28; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P21528   MDHP_PEA         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                    Organism  \\\n",
       " 0                         NaN               NaN  Pisum sativum (Garden pea)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           3888  Malate dehydrogenase [NADP], chloroplastic (EC...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..58; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "      Entry   Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0   D3Z5S8  TET5A_MOUSE      Tent5a Fam46a               Tent5a   \n",
       " 1   Q61183  PAPOA_MOUSE    Papola Pap Plap               Papola   \n",
       " 2   Q9WVP6  PAPOB_MOUSE   Papolb Papt Tpap               Papolb   \n",
       " 3   Q6PB75  PAPD7_MOUSE  Tent4a Papd7 Pols               Tent4a   \n",
       " 4   Q5SSF7  TET5C_MOUSE      Tent5c Fam46c               Tent5c   \n",
       " 5   Q91YI6   GLD2_MOUSE   Tent2 Gld2 Papd4                Tent2   \n",
       " 6   Q9D0D3  PAPD1_MOUSE        Mtpap Papd1                Mtpap   \n",
       " 7   Q68ED3  PAPD5_MOUSE       Tent4b Papd5               Tent4b   \n",
       " 8   Q8R3F9  STPAP_MOUSE         Tut1 Rbm21                 Tut1   \n",
       " 9   Q6PCL9  PAPOG_MOUSE             Papolg               Papolg   \n",
       " 10  Q8C152  TET5B_MOUSE      Tent5b Fam46b               Tent5b   \n",
       " 11  Q8JLG4   PAP1_ECTVM        PAPL EVM041                 PAPL   \n",
       " 12  Q8QMZ9   PAP1_CWPXB       PAPL CPXV067                 PAPL   \n",
       " 13  Q80E12   PAP1_CWPXG           PAPL F1L                 PAPL   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                Fam46a                        NaN              NaN   \n",
       " 1              Pap Plap                        NaN              NaN   \n",
       " 2             Papt Tpap                        NaN              NaN   \n",
       " 3            Papd7 Pols                        NaN              NaN   \n",
       " 4                Fam46c                        NaN              NaN   \n",
       " 5            Gld2 Papd4                        NaN              NaN   \n",
       " 6                 Papd1                        NaN              NaN   \n",
       " 7                 Papd5                        NaN              NaN   \n",
       " 8                 Rbm21                        NaN              NaN   \n",
       " 9                   NaN                        NaN              NaN   \n",
       " 10               Fam46b                        NaN              NaN   \n",
       " 11                  NaN                     EVM041              NaN   \n",
       " 12                  NaN                    CPXV067              NaN   \n",
       " 13                  NaN                        NaN              F1L   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                Mus musculus (Mouse)          10090   \n",
       " 1                                Mus musculus (Mouse)          10090   \n",
       " 2                                Mus musculus (Mouse)          10090   \n",
       " 3                                Mus musculus (Mouse)          10090   \n",
       " 4                                Mus musculus (Mouse)          10090   \n",
       " 5                                Mus musculus (Mouse)          10090   \n",
       " 6                                Mus musculus (Mouse)          10090   \n",
       " 7                                Mus musculus (Mouse)          10090   \n",
       " 8                                Mus musculus (Mouse)          10090   \n",
       " 9                                Mus musculus (Mouse)          10090   \n",
       " 10                               Mus musculus (Mouse)          10090   \n",
       " 11  Ectromelia virus (strain Moscow) (ECTV) (Mouse...         265874   \n",
       " 12           Cowpox virus (strain Brighton Red) (CPV)         265872   \n",
       " 13       Cowpox virus (strain GRI-90 / Grishak) (CPV)         265871   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0    Terminal nucleotidyltransferase 5A (EC 2.7.7.19)  ...           NaN   \n",
       " 1   Poly(A) polymerase alpha (PAP-alpha) (EC 2.7.7...  ...           NaN   \n",
       " 2   Poly(A) polymerase beta (PAP-beta) (EC 2.7.7.1...  ...           NaN   \n",
       " 3   Terminal nucleotidyltransferase 4A (DNA polyme...  ...           NaN   \n",
       " 4    Terminal nucleotidyltransferase 5C (EC 2.7.7.19)  ...           NaN   \n",
       " 5   Poly(A) RNA polymerase GLD2 (mGLD-2) (EC 2.7.7...  ...           NaN   \n",
       " 6   Poly(A) RNA polymerase, mitochondrial (PAP) (E...  ...           NaN   \n",
       " 7   Terminal nucleotidyltransferase 4B (Non-canoni...  ...           NaN   \n",
       " 8   Speckle targeted PIP5K1A-regulated poly(A) pol...  ...           NaN   \n",
       " 9   Poly(A) polymerase gamma (PAP-gamma) (EC 2.7.7...  ...           NaN   \n",
       " 10  Terminal nucleotidyltransferase 5B (EC 2.7.7.1...  ...           NaN   \n",
       " 11  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 12  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 13  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " \n",
       "    Initiator methionine Lipidation  \\\n",
       " 0                   NaN        NaN   \n",
       " 1                   NaN        NaN   \n",
       " 2                   NaN        NaN   \n",
       " 3                   NaN        NaN   \n",
       " 4                   NaN        NaN   \n",
       " 5                   NaN        NaN   \n",
       " 6                   NaN        NaN   \n",
       " 7                   NaN        NaN   \n",
       " 8                   NaN        NaN   \n",
       " 9                   NaN        NaN   \n",
       " 10                  NaN        NaN   \n",
       " 11                  NaN        NaN   \n",
       " 12                  NaN        NaN   \n",
       " 13                  NaN        NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0                                                 NaN     NaN   \n",
       " 1   MOD_RES 10; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3                                                 NaN     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " 5   MOD_RES 62; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 6   MOD_RES 90; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 7   MOD_RES 545; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 8   MOD_RES 688; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 9   MOD_RES 2; /note=\"N6-acetyllysine\"; /evidence=...     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11                                                NaN     NaN   \n",
       " 12                                                NaN     NaN   \n",
       " 13                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1   PTM: Polysumoylated. Varying sumolyation depen...         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8   PTM: Phosphorylated by CK1 in the proline-rich...         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " \n",
       "     Signal peptide                                    Transit peptide    PDB  \n",
       " 0              NaN                                                NaN    NaN  \n",
       " 1              NaN                                                NaN    NaN  \n",
       " 2              NaN                                                NaN    NaN  \n",
       " 3              NaN                                                NaN    NaN  \n",
       " 4              NaN                                                NaN  7CQZ;  \n",
       " 5              NaN                                                NaN  6LBJ;  \n",
       " 6              NaN  TRANSIT 1..37; /note=\"Mitochondrion\"; /evidenc...    NaN  \n",
       " 7              NaN                                                NaN    NaN  \n",
       " 8              NaN                                                NaN    NaN  \n",
       " 9              NaN                                                NaN    NaN  \n",
       " 10             NaN                                                NaN    NaN  \n",
       " 11             NaN                                                NaN    NaN  \n",
       " 12             NaN                                                NaN    NaN  \n",
       " 13             NaN                                                NaN    NaN  \n",
       " \n",
       " [14 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  P06738  PHSG_YEAST  GPH1 YPR160W P9584.1                 GPH1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YPR160W          P9584.1   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                          Protein names  ... Glycosylation  \\\n",
       " 0  Glycogen phosphorylase (EC 2.4.1.1)  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 31; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1YGP;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  O60725  ICMT_HUMAN  ICMT PCCMT                 ICMT                PCCMT   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Protein-S-isoprenylcysteine O-methyltransferas...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  Q57696  CHMU_METJA  aroQ aroQ(f) MJ0246                 aroQ   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0              aroQ(f)                     MJ0246               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Chorismate mutase (CM) (EC 5.4.99.5) (Monofunc...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2GTV;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name                         Gene Names  \\\n",
       " 0  P17516  AK1C4_HUMAN                        AKR1C4 CHDR   \n",
       " 1  P42330  AK1C3_HUMAN  AKR1C3 DDH1 HSD17B5 KIAA0119 PGFS   \n",
       " 2  P52895  AK1C2_HUMAN                        AKR1C2 DDH2   \n",
       " 3  Q04828  AK1C1_HUMAN                    AKR1C1 DDH DDH1   \n",
       " \n",
       "   Gene Names (primary)        Gene Names (synonym)  \\\n",
       " 0               AKR1C4                        CHDR   \n",
       " 1               AKR1C3  DDH1 HSD17B5 KIAA0119 PGFS   \n",
       " 2               AKR1C2                        DDH2   \n",
       " 3               AKR1C1                    DDH DDH1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " 2                         NaN               NaN  Homo sapiens (Human)   \n",
       " 3                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Aldo-keto reductase family 1 member C4 (EC 1.1...  ...   \n",
       " 1           9606  Aldo-keto reductase family 1 member C3 (EC 1.1...  ...   \n",
       " 2           9606  Aldo-keto reductase family 1 member C2 (EC 1.-...  ...   \n",
       " 3           9606  Aldo-keto reductase family 1 member C1 (EC 1.1...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " 1           NaN                  NaN         NaN               NaN     NaN   \n",
       " 2           NaN                  NaN         NaN               NaN     NaN   \n",
       " 3           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0  PTM: The N-terminus is blocked.         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN                                              2FVL;  \n",
       " 1             NaN  1RY0;1RY8;1S1P;1S1R;1S2A;1S2C;1XF0;1ZQ5;2F38;2...  \n",
       " 2             NaN  1IHI;1J96;1XJB;2HDJ;2IPJ;4JQ1;4JQ2;4JQ3;4JQ4;4...  \n",
       " 3             NaN                1MRQ;3C3U;3GUG;3NTY;4YVP;6A7A;6IJX;  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry  Entry Name                                         Gene Names  \\\n",
       " 0  P08432  DCOR_YEAST                                  SPE1 ORD1 YKL184W   \n",
       " 1  P78599  DCOR_CANAL  SPE1 ORD1 CAALFM_C100740CA CaO19.13453 CaO19.6032   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                 SPE1                 ORD1                    YKL184W   \n",
       " 1                 SPE1                 ORD1           CAALFM_C100740CA   \n",
       " \n",
       "          Gene Names (ORF)                                           Organism  \\\n",
       " 0                     NaN  Saccharomyces cerevisiae (strain ATCC 204508 /...   \n",
       " 1  CaO19.13453 CaO19.6032  Candida albicans (strain SC5314 / ATCC MYA-287...   \n",
       " \n",
       "    Organism (ID)                                Protein names  ...  \\\n",
       " 0         559292  Ornithine decarboxylase (ODC) (EC 4.1.1.17)  ...   \n",
       " 1         237561  Ornithine decarboxylase (ODC) (EC 4.1.1.17)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 116; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 1  MOD_RES 106; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q922E4  PCY2_MOUSE      Pcyt2                Pcyt2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Ethanolamine-phosphate cytidylyltransferase (E...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 356; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P49079  AKH1_MAIZE    AKHSDH1              AKHSDH1                   NaN   \n",
       " 1  P49080  AKH2_MAIZE    AKHSDH2              AKHSDH2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Zea mays (Maize)   \n",
       " 1                         NaN               NaN  Zea mays (Maize)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           4577  Bifunctional aspartokinase/homoserine dehydrog...  ...   \n",
       " 1           4577  Bifunctional aspartokinase/homoserine dehydrog...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " 1           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..92; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 1  TRANSIT 1..89; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P06633  HIS7_YEAST       HIS3 YOR202W                 HIS3   \n",
       " 1  Q02986  HIS7_LACK1               HIS3                 HIS3   \n",
       " 2  P34041  HIS7_TRIHA           his3 igh                 his3   \n",
       " 3  Q75B47  HIS7_ASHGO       HIS3 ADL270C                 HIS3   \n",
       " 4  Q12578  HIS7_CANGA  HIS3 CAGL0L02937g                 HIS3   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                    YOR202W               NaN   \n",
       " 1                  NaN                        NaN               NaN   \n",
       " 2                  igh                        NaN               NaN   \n",
       " 3                  NaN                    ADL270C               NaN   \n",
       " 4                  NaN               CAGL0L02937g               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Lachancea kluyveri (strain ATCC 58438 / CBS 30...         226302   \n",
       " 2             Trichoderma harzianum (Hypocrea lixii)           5544   \n",
       " 3  Ashbya gossypii (strain ATCC 10895 / CBS 109.5...         284811   \n",
       " 4  Candida glabrata (strain ATCC 2001 / CBS 138 /...         284593   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Imidazoleglycerol-phosphate dehydratase (IGPD)...  ...           NaN   \n",
       " 1  Imidazoleglycerol-phosphate dehydratase (IGPD)...  ...           NaN   \n",
       " 2  Imidazoleglycerol-phosphate dehydratase (IGPD)...  ...           NaN   \n",
       " 3  Imidazoleglycerol-phosphate dehydratase (IGPD)...  ...           NaN   \n",
       " 4  Imidazoleglycerol-phosphate dehydratase (IGPD)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  6EZM;  \n",
       " 1             NaN    NaN  \n",
       " 2             NaN    NaN  \n",
       " 3             NaN    NaN  \n",
       " 4             NaN    NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  Q9LDY2  ODBB2_ARATH       DIN4 At3g13450 MRP15.11                 DIN4   \n",
       " 1  Q9LPL5  ODBA1_ARATH             At1g21400 F24J8.4                  NaN   \n",
       " 2  Q84JL2  ODBA2_ARATH            At5g09300 T5E8_100                  NaN   \n",
       " 3  Q9SAV3  ODBB1_ARATH  BCDH BETA1 At1g55510 T5A14.9           BCDH BETA1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                  At3g13450         MRP15.11   \n",
       " 1                   NaN                  At1g21400          F24J8.4   \n",
       " 2                   NaN                  At5g09300         T5E8_100   \n",
       " 3                   NaN                  At1g55510          T5A14.9   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 3  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  2-oxoisovalerate dehydrogenase subunit beta 2,...  ...           NaN   \n",
       " 1  2-oxoisovalerate dehydrogenase subunit alpha 1...  ...           NaN   \n",
       " 2  2-oxoisovalerate dehydrogenase subunit alpha 2...  ...           NaN   \n",
       " 3  2-oxoisovalerate dehydrogenase subunit beta 1,...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..16; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " 1  TRANSIT 1..56; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " 2  TRANSIT 1..?; /note=\"Mitochondrion\"; /evidence...  NaN  \n",
       " 3  TRANSIT 1..23; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P37946  ILVA_BACSU  ilvA BSU21770                 ilvA   \n",
       " 1  Q9KC63  ILVA_HALH5    ilvA BH1711                 ilvA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   BSU21770               NaN   \n",
       " 1                   NaN                     BH1711               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                     Bacillus subtilis (strain 168)         224308   \n",
       " 1  Halalkalibacterium halodurans (strain ATCC BAA...         272558   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  L-threonine dehydratase biosynthetic IlvA (EC ...  ...           NaN   \n",
       " 1  L-threonine dehydratase biosynthetic IlvA (EC ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 60; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 1  MOD_RES 53; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q8C0I1  ADAS_MOUSE       Agps                 Agps                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Alkyldihydroxyacetonephosphate synthase, perox...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 52; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..45; /note=\"Peroxisome\"; /evidence=\"...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                                         Gene Names  \\\n",
       " 0  Q10RZ1  BAMY2_ORYSJ        BAMY2 Os03g0141200 LOC_Os03g04770 OsJ_09355   \n",
       " 1  Q9AV88  BAMY1_ORYSJ  BAMY1 Os10g0465700 LOC_Os10g32810 OSJNBa0006L06.1   \n",
       " \n",
       "   Gene Names (primary)  Gene Names (synonym)   Gene Names (ordered locus)  \\\n",
       " 0                BAMY2                   NaN  Os03g0141200 LOC_Os03g04770   \n",
       " 1                BAMY1                   NaN  Os10g0465700 LOC_Os10g32810   \n",
       " \n",
       "   Gene Names (ORF)                             Organism  Organism (ID)  \\\n",
       " 0        OsJ_09355  Oryza sativa subsp. japonica (Rice)          39947   \n",
       " 1  OSJNBa0006L06.1  Oryza sativa subsp. japonica (Rice)          39947   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Beta-amylase 2, chloroplastic (OsBamy2) (EC 3....  ...           NaN   \n",
       " 1  Beta-amylase 1, chloroplastic (OsBamy1) (EC 3....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..38; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 1  TRANSIT 1..36; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "      Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0   P0A9Q7  ADHE_ECOLI   adhE ana b1241 JW1228                 adhE   \n",
       " 1   P0A9Q8  ADHE_ECO57      adhE Z2016 ECs1741                 adhE   \n",
       " 2   P77580  ACDH_ECOLI  mhpF mhpE b0351 JW0342                 mhpF   \n",
       " 3   Q24803  ADH2_ENTHI                    ADH2                 ADH2   \n",
       " 4   P41793  EUTE_SALTY            eutE STM2463                 eutE   \n",
       " 5   P77445  EUTE_ECOLI  eutE yffX b2455 JW2439                 eutE   \n",
       " 6   B6HZX7  ACDH_ECOSE          mhpF ECSE_0376                 mhpF   \n",
       " 7   Q8X5J9  ACDH_ECO57      mhpF Z0450 ECs0406                 mhpF   \n",
       " 8   B7MPB8  ACDH_ECO81         mhpF ECED1_0379                 mhpF   \n",
       " 9   B7M2Z9  ACDH_ECO8A        mhpF ECIAI1_0352                 mhpF   \n",
       " 10  B7L507  ACDH_ECO55       mhpF EC55989_0358                 mhpF   \n",
       " 11  C4ZTB5  ACDH_ECOBW           mhpF BWG_0240                 mhpF   \n",
       " 12  A7ZWZ8  ACDH_ECOHS         mhpF EcHS_A0415                 mhpF   \n",
       " 13  A7ZI98  ACDH_ECO24     mhpF EcE24377A_0375                 mhpF   \n",
       " 14  B5Z2Q6  ACDH_ECO5E      mhpF ECH74115_0426                 mhpF   \n",
       " 15  B1LIN6  ACDH_ECOSM       mhpF EcSMS35_0382                 mhpF   \n",
       " 16  B7NK04  ACDH_ECO7I       mhpF ECIAI39_0327                 mhpF   \n",
       " 17  B7N8Q8  ACDH_ECOLU         mhpF ECUMN_0394                 mhpF   \n",
       " 18  B1J0S8  ACDH_ECOLC         mhpF EcolC_3274                 mhpF   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   ana               b1241 JW1228               NaN   \n",
       " 1                   NaN              Z2016 ECs1741               NaN   \n",
       " 2                  mhpE               b0351 JW0342               NaN   \n",
       " 3                   NaN                        NaN               NaN   \n",
       " 4                   NaN                    STM2463               NaN   \n",
       " 5                  yffX               b2455 JW2439               NaN   \n",
       " 6                   NaN                  ECSE_0376               NaN   \n",
       " 7                   NaN              Z0450 ECs0406               NaN   \n",
       " 8                   NaN                 ECED1_0379               NaN   \n",
       " 9                   NaN                ECIAI1_0352               NaN   \n",
       " 10                  NaN               EC55989_0358               NaN   \n",
       " 11                  NaN                   BWG_0240               NaN   \n",
       " 12                  NaN                 EcHS_A0415               NaN   \n",
       " 13                  NaN             EcE24377A_0375               NaN   \n",
       " 14                  NaN              ECH74115_0426               NaN   \n",
       " 15                  NaN               EcSMS35_0382               NaN   \n",
       " 16                  NaN               ECIAI39_0327               NaN   \n",
       " 17                  NaN                 ECUMN_0394               NaN   \n",
       " 18                  NaN                 EcolC_3274               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                            Escherichia coli O157:H7          83334   \n",
       " 2                       Escherichia coli (strain K12)          83333   \n",
       " 3                               Entamoeba histolytica           5759   \n",
       " 4   Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 5                       Escherichia coli (strain K12)          83333   \n",
       " 6                      Escherichia coli (strain SE11)         409438   \n",
       " 7                            Escherichia coli O157:H7          83334   \n",
       " 8                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 9                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 10             Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 11    Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 12                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 13  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 14    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 15          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 16      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 17  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 18  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Bifunctional aldehyde-alcohol dehydrogenase Ad...  ...           NaN   \n",
       " 1   Bifunctional aldehyde-alcohol dehydrogenase Ad...  ...           NaN   \n",
       " 2   Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 3   Aldehyde-alcohol dehydrogenase 2 [Includes: Al...  ...           NaN   \n",
       " 4   Acetaldehyde dehydrogenase (acetylating) EutE ...  ...           NaN   \n",
       " 5   Acetaldehyde dehydrogenase (acetylating) EutE ...  ...           NaN   \n",
       " 6   Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 7   Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 8   Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 9   Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 10  Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 11  Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 12  Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 13  Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 14  Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 15  Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 16  Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 17  Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " 18  Acetaldehyde dehydrogenase (EC 1.2.1.10) (Acet...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 358; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 1                                                 NaN     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3                                                 NaN     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6                                                 NaN     NaN   \n",
       " 7                                                 NaN     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9                                                 NaN     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11                                                NaN     NaN   \n",
       " 12                                                NaN     NaN   \n",
       " 13                                                NaN     NaN   \n",
       " 14                                                NaN     NaN   \n",
       " 15                                                NaN     NaN   \n",
       " 16                                                NaN     NaN   \n",
       " 17                                                NaN     NaN   \n",
       " 18                                                NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                   PDB  \n",
       " 0              NaN  6AHC;6TQH;6TQM;7BVP;  \n",
       " 1              NaN            6SCG;6SCI;  \n",
       " 2              NaN                   NaN  \n",
       " 3              NaN                   NaN  \n",
       " 4              NaN                   NaN  \n",
       " 5              NaN                   NaN  \n",
       " 6              NaN                   NaN  \n",
       " 7              NaN                   NaN  \n",
       " 8              NaN                   NaN  \n",
       " 9              NaN                   NaN  \n",
       " 10             NaN                   NaN  \n",
       " 11             NaN                   NaN  \n",
       " 12             NaN                   NaN  \n",
       " 13             NaN                   NaN  \n",
       " 14             NaN                   NaN  \n",
       " 15             NaN                   NaN  \n",
       " 16             NaN                   NaN  \n",
       " 17             NaN                   NaN  \n",
       " 18             NaN                   NaN  \n",
       " \n",
       " [19 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P08236  BGLR_HUMAN       GUSB                 GUSB                   NaN   \n",
       " 1  Q9UEF7  KLOT_HUMAN         KL                   KL                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606         Beta-glucuronidase (EC 3.2.1.31) (Beta-G1)  ...   \n",
       " 1           9606  Klotho (EC 3.2.1.31) [Cleaved into: Klotho pep...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 173; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 106; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " 1         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-linked glycosylated with 3 to 4 oligosa...         NaN   \n",
       " 1                PTM: N-glycosylated. {ECO:0000250}.         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..22; /evidence=\"ECO:0000269|PubMed:13...             NaN   \n",
       " 1              SIGNAL 1..33; /evidence=\"ECO:0000255\"             NaN   \n",
       " \n",
       "           PDB  \n",
       " 0  1BHG;3HN3;  \n",
       " 1       5W21;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  Q5T3U5  MRP7_HUMAN            ABCC10 MRP7 SIMRP7               ABCC10   \n",
       " 1  O95255  MRP6_HUMAN                ABCC6 ARA MRP6                ABCC6   \n",
       " 2  Q15311  RBP1_HUMAN      RALBP1 RLIP RLIP1 RLIP76               RALBP1   \n",
       " 3  Q96J66  MRP8_HUMAN                   ABCC11 MRP8               ABCC11   \n",
       " 4  O15438  MRP3_HUMAN        ABCC3 CMOAT2 MLP2 MRP3                ABCC3   \n",
       " 5  O15439  MRP4_HUMAN              ABCC4 MOATB MRP4                ABCC4   \n",
       " 6  P33527  MRP1_HUMAN                ABCC1 MRP MRP1                ABCC1   \n",
       " 7  Q92887  MRP2_HUMAN  ABCC2 CMOAT CMOAT1 CMRP MRP2                ABCC2   \n",
       " \n",
       "      Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0             MRP7 SIMRP7                         NaN               NaN   \n",
       " 1                ARA MRP6                         NaN               NaN   \n",
       " 2       RLIP RLIP1 RLIP76                         NaN               NaN   \n",
       " 3                    MRP8                         NaN               NaN   \n",
       " 4        CMOAT2 MLP2 MRP3                         NaN               NaN   \n",
       " 5              MOATB MRP4                         NaN               NaN   \n",
       " 6                MRP MRP1                         NaN               NaN   \n",
       " 7  CMOAT CMOAT1 CMRP MRP2                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " 3  Homo sapiens (Human)           9606   \n",
       " 4  Homo sapiens (Human)           9606   \n",
       " 5  Homo sapiens (Human)           9606   \n",
       " 6  Homo sapiens (Human)           9606   \n",
       " 7  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  ATP-binding cassette sub-family C member 10 (E...  ...   \n",
       " 1  ATP-binding cassette sub-family C member 6 (EC...  ...   \n",
       " 2  RalA-binding protein 1 (RalBP1) (76 kDa Ral-in...  ...   \n",
       " 3  ATP-binding cassette sub-family C member 11 (E...  ...   \n",
       " 4  ATP-binding cassette sub-family C member 3 (EC...  ...   \n",
       " 5  ATP-binding cassette sub-family C member 4 (EC...  ...   \n",
       " 6  Multidrug resistance-associated protein 1 (EC ...  ...   \n",
       " 7  ATP-binding cassette sub-family C member 2 (EC...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0                                                NaN   \n",
       " 1  CARBOHYD 15; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 2                                                NaN   \n",
       " 3  CARBOHYD 838; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 4  CARBOHYD 18; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 5  CARBOHYD 746; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 6  CARBOHYD 19; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 7  CARBOHYD 7; /note=\"N-linked (GlcNAc...) aspara...   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6                                                NaN         NaN   \n",
       " 7                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 463; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 1  MOD_RES 1286; /note=\"Phosphoserine\"; /evidence...     NaN   \n",
       " 2  MOD_RES 2; /note=\"N-acetylthreonine\"; /evidenc...     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " 4  MOD_RES 908; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 5  MOD_RES 646; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 6  MOD_RES 277; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 7  MOD_RES 281; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: Glycosylated. {ECO:0000269|PubMed:30154241}.         NaN   \n",
       " 2  PTM: Tyrosine-phosphorylated upon stimulation ...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5  PTM: N-glycosylated; leading to substrate-sele...         NaN   \n",
       " 6                                                NaN         NaN   \n",
       " 7                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                        PDB  \n",
       " 0             NaN             NaN                        NaN  \n",
       " 1             NaN             NaN       6BZR;6BZS;6NLO;6P7F;  \n",
       " 2             NaN             NaN  2KWH;2KWI;2MBG;6ZQT;6ZRN;  \n",
       " 3             NaN             NaN                        NaN  \n",
       " 4             NaN             NaN                        NaN  \n",
       " 5             NaN             NaN                        NaN  \n",
       " 6             NaN             NaN                 2CBZ;4C3Z;  \n",
       " 7             NaN             NaN                        NaN  \n",
       " \n",
       " [8 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  Q837U5  AGUA_ENTFA  aguA EF_0734                 aguA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    EF_0734               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Enterococcus faecalis (strain ATCC 700802 / V583)         226185   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Putative agmatine deiminase (EC 3.5.3.12) (Agm...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2JER;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P32871  PK3CA_BOVIN     PIK3CA               PIK3CA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2V1Y;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  Q06128  TRPE_SACS2  trpE SSO0893                 trpE   \n",
       " 1  Q06129  TRPG_SACS2  trpG SSO0894                 trpG   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    SSO0893               NaN   \n",
       " 1                   NaN                    SSO0894               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharolobus solfataricus (strain ATCC 35092 ...         273057   \n",
       " 1  Saccharolobus solfataricus (strain ATCC 35092 ...         273057   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Anthranilate synthase component 1 (AS) (ASI) (...  ...           NaN   \n",
       " 1  Anthranilate synthase component 2 (AS) (ASII) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1QDL;  \n",
       " 1             NaN  1QDL;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P00570   KAD1_BOVIN         AK1                  AK1                  NaN   \n",
       " 1  P52174  NDKA1_BOVIN      NME1-1               NME1-1                  NaN   \n",
       " 2  P52175  NDKA2_BOVIN      NME1-2               NME1-2                  NaN   \n",
       " 3  Q3T0Q4   NDKB_BOVIN        NME2                 NME2                  NaN   \n",
       " 4  Q0VCP1   KAD4_BOVIN   AK4 AK3L1                  AK4                AK3L1   \n",
       " 5  Q2KIW9    KCY_BOVIN  CMPK1 CMPK                CMPK1                 CMPK   \n",
       " 6  Q5E9Y9   NDK7_BOVIN        NME7                 NME7                  NaN   \n",
       " 7  A4IFD0   KAD5_BOVIN         Ak5                  Ak5                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 1                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 2                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 3                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 4                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 5                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 6                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 7                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Adenylate kinase isoenzyme 1 (AK 1) (EC 2.7.4....  ...   \n",
       " 1           9913  Nucleoside diphosphate kinase A 1 (NDK A 1) (N...  ...   \n",
       " 2           9913  Nucleoside diphosphate kinase A 2 (NDK A 2) (N...  ...   \n",
       " 3           9913  Nucleoside diphosphate kinase B (NDK B) (NDP k...  ...   \n",
       " 4           9913  Adenylate kinase 4, mitochondrial (AK 4) (EC 2...  ...   \n",
       " 5           9913  UMP-CMP kinase (EC 2.7.4.14) (Deoxycytidylate ...  ...   \n",
       " 6           9913  Nucleoside diphosphate kinase 7 (NDK 7) (NDP k...  ...   \n",
       " 7           9913  Adenylate kinase isoenzyme 5 (AK 5) (EC 2.7.4....  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN                                                NaN   \n",
       " 1           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 2           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 3           NaN                                                NaN   \n",
       " 4           NaN                                                NaN   \n",
       " 5           NaN                                                NaN   \n",
       " 6           NaN                                                NaN   \n",
       " 7           NaN                                                NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 1         NaN                                                NaN     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " 3         NaN                                                NaN     NaN   \n",
       " 4         NaN  MOD_RES 60; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " 5         NaN  MOD_RES 33; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 6         NaN                                                NaN     NaN   \n",
       " 7         NaN                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1  PTM: The N-terminus is blocked.         NaN             NaN   \n",
       " 2  PTM: The N-terminus is blocked.         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " 7                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN    NaN  \n",
       " 1             NaN  1BHN;  \n",
       " 2             NaN  1BE4;  \n",
       " 3             NaN    NaN  \n",
       " 4             NaN    NaN  \n",
       " 5             NaN    NaN  \n",
       " 6             NaN  7RRO;  \n",
       " 7             NaN    NaN  \n",
       " \n",
       " [8 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P21397  AOFA_HUMAN       MAOA                 MAOA                   NaN   \n",
       " 1  P27338  AOFB_HUMAN       MAOB                 MAOB                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Amine oxidase [flavin-containing] A (EC 1.4.3....  ...   \n",
       " 1           9606  Amine oxidase [flavin-containing] B (EC 1.4.3....  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN                                                NaN   \n",
       " 1           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 1         NaN  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN                               2BXR;2BXS;2Z5X;2Z5Y;  \n",
       " 1             NaN  1GOS;1OJ9;1OJA;1OJC;1OJD;1S2Q;1S2Y;1S3B;1S3E;2...  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "      Entry   Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0   O94806  KPCD3_HUMAN           PRKD3 EPK2 PRKCN                PRKD3   \n",
       " 1   P41743   KPCI_HUMAN             PRKCI DXS1179E                PRKCI   \n",
       " 2   Q02156   KPCE_HUMAN                 PRKCE PKCE                PRKCE   \n",
       " 3   Q05655   KPCD_HUMAN                 PRKCD PKCD                PRKCD   \n",
       " 4   Q16512   PKN1_HUMAN  PKN1 PAK1 PKN PRK1 PRKCL1                 PKN1   \n",
       " 5   P17252   KPCA_HUMAN          PRKCA PKCA PRKACA                PRKCA   \n",
       " 6   Q9BZL6  KPCD2_HUMAN         PRKD2 PKD2 HSPC187                PRKD2   \n",
       " 7   P05129   KPCG_HUMAN                 PRKCG PKCG                PRKCG   \n",
       " 8   P05771   KPCB_HUMAN          PRKCB PKCB PRKCB1                PRKCB   \n",
       " 9   P24723   KPCL_HUMAN           PRKCH PKCL PRKCL                PRKCH   \n",
       " 10  Q16513   PKN2_HUMAN           PKN2 PRK2 PRKCL2                 PKN2   \n",
       " 11  Q6P5Z2   PKN3_HUMAN               PKN3 PKNBETA                 PKN3   \n",
       " 12  Q04759   KPCT_HUMAN                PRKCQ PRKCT                PRKCQ   \n",
       " 13  Q05513   KPCZ_HUMAN                 PRKCZ PKC2                PRKCZ   \n",
       " 14  Q15139  KPCD1_HUMAN       PRKD1 PKD PKD1 PRKCM                PRKD1   \n",
       " \n",
       "     Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0             EPK2 PRKCN                         NaN              NaN   \n",
       " 1               DXS1179E                         NaN              NaN   \n",
       " 2                   PKCE                         NaN              NaN   \n",
       " 3                   PKCD                         NaN              NaN   \n",
       " 4   PAK1 PKN PRK1 PRKCL1                         NaN              NaN   \n",
       " 5            PKCA PRKACA                         NaN              NaN   \n",
       " 6                   PKD2                         NaN          HSPC187   \n",
       " 7                   PKCG                         NaN              NaN   \n",
       " 8            PKCB PRKCB1                         NaN              NaN   \n",
       " 9             PKCL PRKCL                         NaN              NaN   \n",
       " 10           PRK2 PRKCL2                         NaN              NaN   \n",
       " 11               PKNBETA                         NaN              NaN   \n",
       " 12                 PRKCT                         NaN              NaN   \n",
       " 13                  PKC2                         NaN              NaN   \n",
       " 14        PKD PKD1 PRKCM                         NaN              NaN   \n",
       " \n",
       "                 Organism  Organism (ID)  \\\n",
       " 0   Homo sapiens (Human)           9606   \n",
       " 1   Homo sapiens (Human)           9606   \n",
       " 2   Homo sapiens (Human)           9606   \n",
       " 3   Homo sapiens (Human)           9606   \n",
       " 4   Homo sapiens (Human)           9606   \n",
       " 5   Homo sapiens (Human)           9606   \n",
       " 6   Homo sapiens (Human)           9606   \n",
       " 7   Homo sapiens (Human)           9606   \n",
       " 8   Homo sapiens (Human)           9606   \n",
       " 9   Homo sapiens (Human)           9606   \n",
       " 10  Homo sapiens (Human)           9606   \n",
       " 11  Homo sapiens (Human)           9606   \n",
       " 12  Homo sapiens (Human)           9606   \n",
       " 13  Homo sapiens (Human)           9606   \n",
       " 14  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Serine/threonine-protein kinase D3 (EC 2.7.11....  ...           NaN   \n",
       " 1   Protein kinase C iota type (EC 2.7.11.13) (Aty...  ...           NaN   \n",
       " 2   Protein kinase C epsilon type (EC 2.7.11.13) (...  ...           NaN   \n",
       " 3   Protein kinase C delta type (EC 2.7.11.13) (Ty...  ...           NaN   \n",
       " 4   Serine/threonine-protein kinase N1 (EC 2.7.11....  ...           NaN   \n",
       " 5   Protein kinase C alpha type (PKC-A) (PKC-alpha...  ...           NaN   \n",
       " 6   Serine/threonine-protein kinase D2 (EC 2.7.11....  ...           NaN   \n",
       " 7   Protein kinase C gamma type (PKC-gamma) (EC 2....  ...           NaN   \n",
       " 8   Protein kinase C beta type (PKC-B) (PKC-beta) ...  ...           NaN   \n",
       " 9   Protein kinase C eta type (EC 2.7.11.13) (PKC-...  ...           NaN   \n",
       " 10  Serine/threonine-protein kinase N2 (EC 2.7.11....  ...           NaN   \n",
       " 11  Serine/threonine-protein kinase N3 (EC 2.7.11....  ...           NaN   \n",
       " 12  Protein kinase C theta type (EC 2.7.11.13) (nP...  ...           NaN   \n",
       " 13  Protein kinase C zeta type (EC 2.7.11.13) (nPK...  ...           NaN   \n",
       " 14  Serine/threonine-protein kinase D1 (EC 2.7.11....  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 5   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 6; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " 1   MOD_RES 2; /note=\"N-acetylproline\"; /evidence=...     NaN   \n",
       " 2   MOD_RES 62; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 3   MOD_RES 43; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 4   MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 5   MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 6   MOD_RES 30; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 7   MOD_RES 250; /note=\"Phosphothreonine; by autoc...     NaN   \n",
       " 8   MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 9   MOD_RES 28; /note=\"Phosphoserine; by autocatal...     NaN   \n",
       " 10  MOD_RES 21; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 11  MOD_RES 171; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 12  MOD_RES 90; /note=\"Phosphotyrosine; by LCK\"; /...     NaN   \n",
       " 13  MOD_RES 410; /note=\"Phosphothreonine; by PDPK1...     NaN   \n",
       " 14  MOD_RES 95; /note=\"Phosphotyrosine\"; /evidence...     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1   PTM: Phosphorylation at Thr-412 in the activat...         NaN   \n",
       " 2   PTM: Phosphorylation on Thr-566 by PDPK1 trigg...         NaN   \n",
       " 3   PTM: Autophosphorylated and/or phosphorylated ...         NaN   \n",
       " 4   PTM: Autophosphorylated; preferably on serine....         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6   PTM: Phosphorylation of Ser-876 correlates wit...         NaN   \n",
       " 7   PTM: Autophosphorylation on Thr-674 appears to...         NaN   \n",
       " 8   PTM: Phosphorylation on Thr-500 within the act...         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10  PTM: Autophosphorylated. Phosphorylated during...         NaN   \n",
       " 11  PTM: Autophosphorylated. {ECO:0000269|PubMed:1...         NaN   \n",
       " 12  PTM: Autophosphorylation at Thr-219 is require...         NaN   \n",
       " 13  PTM: CDH5 is required for its phosphorylation ...         NaN   \n",
       " 14  PTM: Phosphorylated at Ser-397 and Ser-401 by ...         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  \\\n",
       " 0              NaN             NaN   \n",
       " 1              NaN             NaN   \n",
       " 2              NaN             NaN   \n",
       " 3              NaN             NaN   \n",
       " 4              NaN             NaN   \n",
       " 5              NaN             NaN   \n",
       " 6              NaN             NaN   \n",
       " 7              NaN             NaN   \n",
       " 8              NaN             NaN   \n",
       " 9              NaN             NaN   \n",
       " 10             NaN             NaN   \n",
       " 11             NaN             NaN   \n",
       " 12             NaN             NaN   \n",
       " 13             NaN             NaN   \n",
       " 14             NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0                                               2D9Z;  \n",
       " 1   1VD2;1WMH;1ZRZ;3A8W;3A8X;3ZH8;5LI1;5LI9;5LIH;6...  \n",
       " 2                                          2WH0;5LIH;  \n",
       " 3                                          1YRK;2YUU;  \n",
       " 4            1CXZ;1URF;2RMK;4NKG;4OTD;4OTG;4OTH;4OTI;  \n",
       " 5                                2ELI;3IW4;4DNL;4RA4;  \n",
       " 6                                2COA;3BGM;4NNX;4NNY;  \n",
       " 7                                          2E73;2UZP;  \n",
       " 8                                               2I0E;  \n",
       " 9                                          2FK9;3TXO;  \n",
       " 10                               4CRS;4RRV;6CCY;6GBE;  \n",
       " 11                                                NaN  \n",
       " 12           1XJD;2ENJ;2ENN;2ENZ;2JED;4Q9Z;4RA5;5F9E;  \n",
       " 13                                                NaN  \n",
       " 14                                                NaN  \n",
       " \n",
       " [15 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   P0A749  MURA_ECOLI        murA murZ b3189 JW3156                 murA   \n",
       " 1   B6I1R2  MURA_ECOSE                murA ECSE_3473                 murA   \n",
       " 2   Q8X9J9  MURA_ECO57            murA Z4552 ECs4068                 murA   \n",
       " 3   B7LHP8  MURA_ECO55             murA EC55989_3607                 murA   \n",
       " 4   A1AG91  MURA_ECOK1  murA Ecok1_31870 APECO1_3245                 murA   \n",
       " 5   B7N0X1  MURA_ECO81               murA ECED1_3847                 murA   \n",
       " 6   B7M095  MURA_ECO8A              murA ECIAI1_3337                 murA   \n",
       " 7   Q1R6E7  MURA_ECOUT              murA UTI89_C3623                 murA   \n",
       " 8   C4ZSS8  MURA_ECOBW                 murA BWG_2891                 murA   \n",
       " 9   A8A504  MURA_ECOHS               murA EcHS_A3382                 murA   \n",
       " 10  B5YS78  MURA_ECO5E            murA ECH74115_4511                 murA   \n",
       " 11  A7ZS86  MURA_ECO24           murA EcE24377A_3675                 murA   \n",
       " 12  B7MBV8  MURA_ECO45               murA ECS88_3571                 murA   \n",
       " 13  B7NKQ6  MURA_ECO7I             murA ECIAI39_3684                 murA   \n",
       " 14  B1LGF5  MURA_ECOSM             murA EcSMS35_3485                 murA   \n",
       " 15  B7UJS9  MURA_ECO27              murA E2348C_3468                 murA   \n",
       " 16  B7NDH3  MURA_ECOLU               murA ECUMN_3669                 murA   \n",
       " 17  Q0TCS1  MURA_ECOL5                 murA ECP_3276                 murA   \n",
       " 18  P0A750  MURA_ECOL6               murA murZ c3947                 murA   \n",
       " 19  B1IQT4  MURA_ECOLC               murA EcolC_0511                 murA   \n",
       " 20  B7LR56  MURA_ESCF3                murA EFER_3166                 murA   \n",
       " 21  P0A751  MURA_SHIFL        murA murZ SF3229 S3447                 murA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                  murZ               b3189 JW3156              NaN   \n",
       " 1                   NaN                  ECSE_3473              NaN   \n",
       " 2                   NaN              Z4552 ECs4068              NaN   \n",
       " 3                   NaN               EC55989_3607              NaN   \n",
       " 4                   NaN                Ecok1_31870      APECO1_3245   \n",
       " 5                   NaN                 ECED1_3847              NaN   \n",
       " 6                   NaN                ECIAI1_3337              NaN   \n",
       " 7                   NaN                UTI89_C3623              NaN   \n",
       " 8                   NaN                   BWG_2891              NaN   \n",
       " 9                   NaN                 EcHS_A3382              NaN   \n",
       " 10                  NaN              ECH74115_4511              NaN   \n",
       " 11                  NaN             EcE24377A_3675              NaN   \n",
       " 12                  NaN                 ECS88_3571              NaN   \n",
       " 13                  NaN               ECIAI39_3684              NaN   \n",
       " 14                  NaN               EcSMS35_3485              NaN   \n",
       " 15                  NaN                E2348C_3468              NaN   \n",
       " 16                  NaN                 ECUMN_3669              NaN   \n",
       " 17                  NaN                   ECP_3276              NaN   \n",
       " 18                 murZ                      c3947              NaN   \n",
       " 19                  NaN                 EcolC_0511              NaN   \n",
       " 20                  NaN                  EFER_3166              NaN   \n",
       " 21                 murZ               SF3229 S3447              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain SE11)         409438   \n",
       " 2                            Escherichia coli O157:H7          83334   \n",
       " 3              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 4                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 5                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 6                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 7              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 8     Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 9                  Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 10    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 11  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 12       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 13      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 14          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 15  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 16  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 17    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 18  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 19  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 20  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 21                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 1   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 2   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 3   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 4   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 5   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 6   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 7   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 8   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 9   UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 10  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 11  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 12  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 13  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 14  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 15  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 16  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 17  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 18  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 19  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 20  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " 21  UDP-N-acetylglucosamine 1-carboxyvinyltransfer...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  \\\n",
       " 0                   NaN         NaN   \n",
       " 1                   NaN         NaN   \n",
       " 2                   NaN         NaN   \n",
       " 3                   NaN         NaN   \n",
       " 4                   NaN         NaN   \n",
       " 5                   NaN         NaN   \n",
       " 6                   NaN         NaN   \n",
       " 7                   NaN         NaN   \n",
       " 8                   NaN         NaN   \n",
       " 9                   NaN         NaN   \n",
       " 10                  NaN         NaN   \n",
       " 11                  NaN         NaN   \n",
       " 12                  NaN         NaN   \n",
       " 13                  NaN         NaN   \n",
       " 14                  NaN         NaN   \n",
       " 15                  NaN         NaN   \n",
       " 16                  NaN         NaN   \n",
       " 17                  NaN         NaN   \n",
       " 18                  NaN         NaN   \n",
       " 19                  NaN         NaN   \n",
       " 20                  NaN         NaN   \n",
       " 21                  NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 1   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 2   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 3   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 4   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 5   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 6   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 7   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 8   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 9   MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 10  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 11  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 12  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 13  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 14  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 15  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 16  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 17  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 18  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 19  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 20  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " 21  MOD_RES 115; /note=\"2-(S-cysteinyl)pyruvic aci...     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " 21                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                                  PDB  \n",
       " 0              NaN  1A2N;1UAE;2Z2C;3ISS;3KQJ;3KR6;3SWD;  \n",
       " 1              NaN                                  NaN  \n",
       " 2              NaN                                  NaN  \n",
       " 3              NaN                                  NaN  \n",
       " 4              NaN                                  NaN  \n",
       " 5              NaN                                  NaN  \n",
       " 6              NaN                                  NaN  \n",
       " 7              NaN                                  NaN  \n",
       " 8              NaN                                  NaN  \n",
       " 9              NaN                                  NaN  \n",
       " 10             NaN                                  NaN  \n",
       " 11             NaN                                  NaN  \n",
       " 12             NaN                                  NaN  \n",
       " 13             NaN                                  NaN  \n",
       " 14             NaN                                  NaN  \n",
       " 15             NaN                                  NaN  \n",
       " 16             NaN                                  NaN  \n",
       " 17             NaN                                  NaN  \n",
       " 18             NaN                                  NaN  \n",
       " 19             NaN                                  NaN  \n",
       " 20             NaN                                  NaN  \n",
       " 21             NaN                                  NaN  \n",
       " \n",
       " [22 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P16453   DCHS_RAT        Hdc                  Hdc                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                Protein names  ...  \\\n",
       " 0          10116  Histidine decarboxylase (HDC) (EC 4.1.1.22)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 308; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "                Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0  PTM: May be post-translationally processed.         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  Q8IJV6  KAD1_PLAF7  AK1 AK PF3D7_1008900                  AK1   \n",
       " 1  Q8IB06  KAD2_PLAF7     AK2 PF3D7_0816900                  AK2   \n",
       " 2  Q8I236  KAD6_PLAF7             PFA_0530c                  NaN   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   AK                         NaN    PF3D7_1008900   \n",
       " 1                  NaN                         NaN    PF3D7_0816900   \n",
       " 2                  NaN                         NaN        PFA_0530c   \n",
       " \n",
       "                               Organism  Organism (ID)  \\\n",
       " 0  Plasmodium falciparum (isolate 3D7)          36329   \n",
       " 1  Plasmodium falciparum (isolate 3D7)          36329   \n",
       " 2  Plasmodium falciparum (isolate 3D7)          36329   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0            Adenylate kinase 1 (PfAK1) (EC 2.7.4.3)  ...           NaN   \n",
       " 1            Adenylate kinase 2 (PfAK2) (EC 2.7.4.3)  ...           NaN   \n",
       " 2  Adenylate kinase isoenzyme 6 homolog (AK6) (EC...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  \\\n",
       " 0                                                NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 2                                                NaN   \n",
       " \n",
       "                                           Lipidation  Modified residue  \\\n",
       " 0                                                NaN               NaN   \n",
       " 1  LIPID 2; /note=\"N-myristoyl glycine\"; /evidenc...               NaN   \n",
       " 2                                                NaN               NaN   \n",
       " \n",
       "   Peptide                    Post-translational modification  Propeptide  \\\n",
       " 0     NaN                                                NaN         NaN   \n",
       " 1     NaN  PTM: Myristoylation is required for cell membr...         NaN   \n",
       " 2     NaN                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide    PDB  \n",
       " 0             NaN             NaN  3TLX;  \n",
       " 1             NaN             NaN    NaN  \n",
       " 2             NaN             NaN    NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name                      Gene Names Gene Names (primary)  \\\n",
       " 0  P76373   UDG_ECOLI  ugd pmrE udg yefA b2028 JW2010                  ugd   \n",
       " 1  O33952  UDG8_ECOLX                             ugd                  ugd   \n",
       " 2  Q47329  UDG5_ECOLX                            kfiD                 kfiD   \n",
       " 3  Q04872   UDG_ECO11                             ugd                  ugd   \n",
       " 4  Q7DBF9   UDG_ECO57               ugd Z3190 ECs2829                  ugd   \n",
       " 5  Q8FG45   UDG_ECOL6                       ugd c2555                  ugd   \n",
       " 6  P37791   UDG_SHIFL                udg SF2090 S2211                  udg   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0        pmrE udg yefA               b2028 JW2010               NaN   \n",
       " 1                  NaN                        NaN               NaN   \n",
       " 2                  NaN                        NaN               NaN   \n",
       " 3                  NaN                        NaN               NaN   \n",
       " 4                  NaN              Z3190 ECs2829               NaN   \n",
       " 5                  NaN                      c2555               NaN   \n",
       " 6                  NaN               SF2090 S2211               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                                   Escherichia coli            562   \n",
       " 2                                   Escherichia coli            562   \n",
       " 3                           Escherichia coli O111:H-         168927   \n",
       " 4                           Escherichia coli O157:H7          83334   \n",
       " 5  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 6                                  Shigella flexneri            623   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  UDP-glucose 6-dehydrogenase (UDP-Glc dehydroge...  ...           NaN   \n",
       " 1  UDP-glucose 6-dehydrogenase (UDP-Glc dehydroge...  ...           NaN   \n",
       " 2  UDP-glucose 6-dehydrogenase (UDP-Glc dehydroge...  ...           NaN   \n",
       " 3  UDP-glucose 6-dehydrogenase (UDP-Glc dehydroge...  ...           NaN   \n",
       " 4  UDP-glucose 6-dehydrogenase (UDP-Glc dehydroge...  ...           NaN   \n",
       " 5  UDP-glucose 6-dehydrogenase (UDP-Glc dehydroge...  ...           NaN   \n",
       " 6  Putative UDP-glucose 6-dehydrogenase (UDP-Glc ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " 6                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylated on a tyrosine residue. It ...         NaN   \n",
       " 1  PTM: Phosphorylated on a tyrosine residue. It ...         NaN   \n",
       " 2  PTM: Phosphorylated on a tyrosine residue. It ...         NaN   \n",
       " 3  PTM: Phosphorylated on a tyrosine residue. It ...         NaN   \n",
       " 4  PTM: Phosphorylated on a tyrosine residue. It ...         NaN   \n",
       " 5  PTM: Phosphorylated on a tyrosine residue. It ...         NaN   \n",
       " 6                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " 3             NaN             NaN  NaN  \n",
       " 4             NaN             NaN  NaN  \n",
       " 5             NaN             NaN  NaN  \n",
       " 6             NaN             NaN  NaN  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       "      Entry   Entry Name                         Gene Names  \\\n",
       " 0   P0A7A7   PLSB_ECOLI                  plsB b4041 JW4001   \n",
       " 1   P60782   PLSY_ECOLI             plsY ygiH b3059 JW3031   \n",
       " 2   Q61586  GPAT1_MOUSE                         Gpam Gpat1   \n",
       " 3   B6I430   PLSY_ECOSE                plsY ygiH ECSE_3339   \n",
       " 4   P60784   PLSY_ECO57            plsY ygiH Z4412 ECs3942   \n",
       " 5   B1XG64   PLSY_ECODH             plsY ygiH ECDH10B_3234   \n",
       " 6   B7N0K9   PLSY_ECO81               plsY ygiH ECED1_3728   \n",
       " 7   B7LGZ5   PLSY_ECO55             plsY ygiH EC55989_3474   \n",
       " 8   B7LZK9   PLSY_ECO8A              plsY ygiH ECIAI1_3207   \n",
       " 9   Q1R6S2   PLSY_ECOUT              plsY ygiH UTI89_C3495   \n",
       " 10  A1AFY1   PLSY_ECOK1  plsY ygiH Ecok1_30770 APECO1_3355   \n",
       " 11  C4ZQX6   PLSY_ECOBW                 plsY ygiH BWG_2770   \n",
       " 12  A8A4L4   PLSY_ECOHS               plsY ygiH EcHS_A3237   \n",
       " 13  A7ZRU0   PLSY_ECO24           plsY ygiH EcE24377A_3522   \n",
       " 14  B5YR99   PLSY_ECO5E            plsY ygiH ECH74115_4371   \n",
       " 15  B7MAZ5   PLSY_ECO45               plsY ygiH ECS88_3457   \n",
       " 16  B1LF51   PLSY_ECOSM             plsY ygiH EcSMS35_3352   \n",
       " 17  B7NJS2   PLSY_ECO7I             plsY ygiH ECIAI39_3555   \n",
       " 18  B7UIW7   PLSY_ECO27              plsY ygiH E2348C_3352   \n",
       " 19  B7ND48   PLSY_ECOLU               plsY ygiH ECUMN_3542   \n",
       " 20  Q0TD47   PLSY_ECOL5                 plsY ygiH ECP_3149   \n",
       " 21  P60783   PLSY_ECOL6                    plsY ygiH c3809   \n",
       " 22  B1IRQ7   PLSY_ECOLC               plsY ygiH EcolC_0640   \n",
       " 23  B7LQD7   PLSY_ESCF3                plsY ygiH EFER_3004   \n",
       " 24  P58130   PLSB_ECO57                 plsB Z5640 ECs5024   \n",
       " 25  P59252   PLSY_SHIFL             plsY ygiH SF3100 S3305   \n",
       " 26  B6I5Q6   PLSB_ECOSE                     plsB ECSE_4333   \n",
       " 27  B1XC40   PLSB_ECODH                  plsB ECDH10B_4230   \n",
       " 28  B7LAY9   PLSB_ECO55                  plsB EC55989_4533   \n",
       " 29  A1AIL8   PLSB_ECOK1       plsB Ecok1_40140 APECO1_2428   \n",
       " 30  B7N2P8   PLSB_ECO81                    plsB ECED1_4756   \n",
       " 31  Q1R3P5   PLSB_ECOUT                   plsB UTI89_C4611   \n",
       " 32  B7M7V4   PLSB_ECO8A                   plsB ECIAI1_4271   \n",
       " 33  C5A134   PLSB_ECOBW                      plsB BWG_3754   \n",
       " 34  A8A7E1   PLSB_ECOHS                    plsB EcHS_A4281   \n",
       " 35  B5Z182   PLSB_ECO5E                 plsB ECH74115_5522   \n",
       " 36  A7ZUR3   PLSB_ECO24                plsB EcE24377A_4594   \n",
       " 37  B7MJ31   PLSB_ECO45                    plsB ECS88_4514   \n",
       " 38  B1LPK6   PLSB_ECOSM                  plsB EcSMS35_4503   \n",
       " 39  B7NS05   PLSB_ECO7I                  plsB ECIAI39_4462   \n",
       " 40  B7UPK2   PLSB_ECO27                   plsB E2348C_4356   \n",
       " 41  B7NFY5   PLSB_ECOLU                    plsB ECUMN_4575   \n",
       " 42  P0A7A8   PLSB_ECOL6                         plsB c5011   \n",
       " 43  B1IUL1   PLSB_ECOLC                    plsB EcolC_3987   \n",
       " 44  B7LL13   PLSB_ESCF3                     plsB EFER_4132   \n",
       " 45  Q7UBC6   PLSB_SHIFL                  plsB SF4164 S3567   \n",
       " \n",
       "    Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                  plsB                  NaN               b4041 JW4001   \n",
       " 1                  plsY                 ygiH               b3059 JW3031   \n",
       " 2                  Gpam                Gpat1                        NaN   \n",
       " 3                  plsY                 ygiH                  ECSE_3339   \n",
       " 4                  plsY                 ygiH              Z4412 ECs3942   \n",
       " 5                  plsY                 ygiH               ECDH10B_3234   \n",
       " 6                  plsY                 ygiH                 ECED1_3728   \n",
       " 7                  plsY                 ygiH               EC55989_3474   \n",
       " 8                  plsY                 ygiH                ECIAI1_3207   \n",
       " 9                  plsY                 ygiH                UTI89_C3495   \n",
       " 10                 plsY                 ygiH                Ecok1_30770   \n",
       " 11                 plsY                 ygiH                   BWG_2770   \n",
       " 12                 plsY                 ygiH                 EcHS_A3237   \n",
       " 13                 plsY                 ygiH             EcE24377A_3522   \n",
       " 14                 plsY                 ygiH              ECH74115_4371   \n",
       " 15                 plsY                 ygiH                 ECS88_3457   \n",
       " 16                 plsY                 ygiH               EcSMS35_3352   \n",
       " 17                 plsY                 ygiH               ECIAI39_3555   \n",
       " 18                 plsY                 ygiH                E2348C_3352   \n",
       " 19                 plsY                 ygiH                 ECUMN_3542   \n",
       " 20                 plsY                 ygiH                   ECP_3149   \n",
       " 21                 plsY                 ygiH                      c3809   \n",
       " 22                 plsY                 ygiH                 EcolC_0640   \n",
       " 23                 plsY                 ygiH                  EFER_3004   \n",
       " 24                 plsB                  NaN              Z5640 ECs5024   \n",
       " 25                 plsY                 ygiH               SF3100 S3305   \n",
       " 26                 plsB                  NaN                  ECSE_4333   \n",
       " 27                 plsB                  NaN               ECDH10B_4230   \n",
       " 28                 plsB                  NaN               EC55989_4533   \n",
       " 29                 plsB                  NaN                Ecok1_40140   \n",
       " 30                 plsB                  NaN                 ECED1_4756   \n",
       " 31                 plsB                  NaN                UTI89_C4611   \n",
       " 32                 plsB                  NaN                ECIAI1_4271   \n",
       " 33                 plsB                  NaN                   BWG_3754   \n",
       " 34                 plsB                  NaN                 EcHS_A4281   \n",
       " 35                 plsB                  NaN              ECH74115_5522   \n",
       " 36                 plsB                  NaN             EcE24377A_4594   \n",
       " 37                 plsB                  NaN                 ECS88_4514   \n",
       " 38                 plsB                  NaN               EcSMS35_4503   \n",
       " 39                 plsB                  NaN               ECIAI39_4462   \n",
       " 40                 plsB                  NaN                E2348C_4356   \n",
       " 41                 plsB                  NaN                 ECUMN_4575   \n",
       " 42                 plsB                  NaN                      c5011   \n",
       " 43                 plsB                  NaN                 EcolC_3987   \n",
       " 44                 plsB                  NaN                  EFER_4132   \n",
       " 45                 plsB                  NaN               SF4164 S3567   \n",
       " \n",
       "    Gene Names (ORF)                                           Organism  \\\n",
       " 0               NaN                      Escherichia coli (strain K12)   \n",
       " 1               NaN                      Escherichia coli (strain K12)   \n",
       " 2               NaN                               Mus musculus (Mouse)   \n",
       " 3               NaN                     Escherichia coli (strain SE11)   \n",
       " 4               NaN                           Escherichia coli O157:H7   \n",
       " 5               NaN              Escherichia coli (strain K12 / DH10B)   \n",
       " 6               NaN                 Escherichia coli O81 (strain ED1a)   \n",
       " 7               NaN             Escherichia coli (strain 55989 / EAEC)   \n",
       " 8               NaN                  Escherichia coli O8 (strain IAI1)   \n",
       " 9               NaN             Escherichia coli (strain UTI89 / UPEC)   \n",
       " 10      APECO1_3355                      Escherichia coli O1:K1 / APEC   \n",
       " 11              NaN    Escherichia coli (strain K12 / MC4100 / BW2952)   \n",
       " 12              NaN                 Escherichia coli O9:H4 (strain HS)   \n",
       " 13              NaN  Escherichia coli O139:H28 (strain E24377A / ETEC)   \n",
       " 14              NaN    Escherichia coli O157:H7 (strain EC4115 / EHEC)   \n",
       " 15              NaN       Escherichia coli O45:K1 (strain S88 / ExPEC)   \n",
       " 16              NaN          Escherichia coli (strain SMS-3-5 / SECEC)   \n",
       " 17              NaN      Escherichia coli O7:K1 (strain IAI39 / ExPEC)   \n",
       " 18              NaN  Escherichia coli O127:H6 (strain E2348/69 / EPEC)   \n",
       " 19              NaN  Escherichia coli O17:K52:H18 (strain UMN026 / ...   \n",
       " 20              NaN    Escherichia coli O6:K15:H31 (strain 536 / UPEC)   \n",
       " 21              NaN  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...   \n",
       " 22              NaN  Escherichia coli (strain ATCC 8739 / DSM 1576 ...   \n",
       " 23              NaN  Escherichia fergusonii (strain ATCC 35469 / DS...   \n",
       " 24              NaN                           Escherichia coli O157:H7   \n",
       " 25              NaN                                  Shigella flexneri   \n",
       " 26              NaN                     Escherichia coli (strain SE11)   \n",
       " 27              NaN              Escherichia coli (strain K12 / DH10B)   \n",
       " 28              NaN             Escherichia coli (strain 55989 / EAEC)   \n",
       " 29      APECO1_2428                      Escherichia coli O1:K1 / APEC   \n",
       " 30              NaN                 Escherichia coli O81 (strain ED1a)   \n",
       " 31              NaN             Escherichia coli (strain UTI89 / UPEC)   \n",
       " 32              NaN                  Escherichia coli O8 (strain IAI1)   \n",
       " 33              NaN    Escherichia coli (strain K12 / MC4100 / BW2952)   \n",
       " 34              NaN                 Escherichia coli O9:H4 (strain HS)   \n",
       " 35              NaN    Escherichia coli O157:H7 (strain EC4115 / EHEC)   \n",
       " 36              NaN  Escherichia coli O139:H28 (strain E24377A / ETEC)   \n",
       " 37              NaN       Escherichia coli O45:K1 (strain S88 / ExPEC)   \n",
       " 38              NaN          Escherichia coli (strain SMS-3-5 / SECEC)   \n",
       " 39              NaN      Escherichia coli O7:K1 (strain IAI39 / ExPEC)   \n",
       " 40              NaN  Escherichia coli O127:H6 (strain E2348/69 / EPEC)   \n",
       " 41              NaN  Escherichia coli O17:K52:H18 (strain UMN026 / ...   \n",
       " 42              NaN  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...   \n",
       " 43              NaN  Escherichia coli (strain ATCC 8739 / DSM 1576 ...   \n",
       " 44              NaN  Escherichia fergusonii (strain ATCC 35469 / DS...   \n",
       " 45              NaN                                  Shigella flexneri   \n",
       " \n",
       "     Organism (ID)                                      Protein names  ...  \\\n",
       " 0           83333  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 1           83333  Probable glycerol-3-phosphate acyltransferase ...  ...   \n",
       " 2           10090  Glycerol-3-phosphate acyltransferase 1, mitoch...  ...   \n",
       " 3          409438  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 4           83334  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 5          316385  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 6          585397  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 7          585055  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 8          585034  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 9          364106  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 10         405955  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 11         595496  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 12         331112  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 13         331111  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 14         444450  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 15         585035  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 16         439855  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 17         585057  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 18         574521  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 19         585056  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 20         362663  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 21         199310  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 22         481805  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 23         585054  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 24          83334  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 25            623  Glycerol-3-phosphate acyltransferase (G3P acyl...  ...   \n",
       " 26         409438  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 27         316385  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 28         585055  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 29         405955  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 30         585397  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 31         364106  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 32         585034  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 33         595496  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 34         331112  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 35         444450  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 36         331111  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 37         585035  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 38         439855  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 39         585057  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 40         574521  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 41         585056  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 42         199310  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 43         481805  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 44         585054  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " 45            623  Glycerol-3-phosphate acyltransferase (GPAT) (E...  ...   \n",
       " \n",
       "    Glycosylation                               Initiator methionine  \\\n",
       " 0            NaN                        INIT_MET 1; /note=\"Removed\"   \n",
       " 1            NaN                                                NaN   \n",
       " 2            NaN                                                NaN   \n",
       " 3            NaN                                                NaN   \n",
       " 4            NaN                                                NaN   \n",
       " 5            NaN                                                NaN   \n",
       " 6            NaN                                                NaN   \n",
       " 7            NaN                                                NaN   \n",
       " 8            NaN                                                NaN   \n",
       " 9            NaN                                                NaN   \n",
       " 10           NaN                                                NaN   \n",
       " 11           NaN                                                NaN   \n",
       " 12           NaN                                                NaN   \n",
       " 13           NaN                                                NaN   \n",
       " 14           NaN                                                NaN   \n",
       " 15           NaN                                                NaN   \n",
       " 16           NaN                                                NaN   \n",
       " 17           NaN                                                NaN   \n",
       " 18           NaN                                                NaN   \n",
       " 19           NaN                                                NaN   \n",
       " 20           NaN                                                NaN   \n",
       " 21           NaN                                                NaN   \n",
       " 22           NaN                                                NaN   \n",
       " 23           NaN                                                NaN   \n",
       " 24           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 25           NaN                                                NaN   \n",
       " 26           NaN                                                NaN   \n",
       " 27           NaN                                                NaN   \n",
       " 28           NaN                                                NaN   \n",
       " 29           NaN                                                NaN   \n",
       " 30           NaN                                                NaN   \n",
       " 31           NaN                                                NaN   \n",
       " 32           NaN                                                NaN   \n",
       " 33           NaN                                                NaN   \n",
       " 34           NaN                                                NaN   \n",
       " 35           NaN                                                NaN   \n",
       " 36           NaN                                                NaN   \n",
       " 37           NaN                                                NaN   \n",
       " 38           NaN                                                NaN   \n",
       " 39           NaN                                                NaN   \n",
       " 40           NaN                                                NaN   \n",
       " 41           NaN                                                NaN   \n",
       " 42           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 43           NaN                                                NaN   \n",
       " 44           NaN                                                NaN   \n",
       " 45           NaN                                                NaN   \n",
       " \n",
       "     Lipidation                                   Modified residue Peptide  \\\n",
       " 0          NaN                                                NaN     NaN   \n",
       " 1          NaN                                                NaN     NaN   \n",
       " 2          NaN  MOD_RES 380; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 3          NaN                                                NaN     NaN   \n",
       " 4          NaN                                                NaN     NaN   \n",
       " 5          NaN                                                NaN     NaN   \n",
       " 6          NaN                                                NaN     NaN   \n",
       " 7          NaN                                                NaN     NaN   \n",
       " 8          NaN                                                NaN     NaN   \n",
       " 9          NaN                                                NaN     NaN   \n",
       " 10         NaN                                                NaN     NaN   \n",
       " 11         NaN                                                NaN     NaN   \n",
       " 12         NaN                                                NaN     NaN   \n",
       " 13         NaN                                                NaN     NaN   \n",
       " 14         NaN                                                NaN     NaN   \n",
       " 15         NaN                                                NaN     NaN   \n",
       " 16         NaN                                                NaN     NaN   \n",
       " 17         NaN                                                NaN     NaN   \n",
       " 18         NaN                                                NaN     NaN   \n",
       " 19         NaN                                                NaN     NaN   \n",
       " 20         NaN                                                NaN     NaN   \n",
       " 21         NaN                                                NaN     NaN   \n",
       " 22         NaN                                                NaN     NaN   \n",
       " 23         NaN                                                NaN     NaN   \n",
       " 24         NaN                                                NaN     NaN   \n",
       " 25         NaN                                                NaN     NaN   \n",
       " 26         NaN                                                NaN     NaN   \n",
       " 27         NaN                                                NaN     NaN   \n",
       " 28         NaN                                                NaN     NaN   \n",
       " 29         NaN                                                NaN     NaN   \n",
       " 30         NaN                                                NaN     NaN   \n",
       " 31         NaN                                                NaN     NaN   \n",
       " 32         NaN                                                NaN     NaN   \n",
       " 33         NaN                                                NaN     NaN   \n",
       " 34         NaN                                                NaN     NaN   \n",
       " 35         NaN                                                NaN     NaN   \n",
       " 36         NaN                                                NaN     NaN   \n",
       " 37         NaN                                                NaN     NaN   \n",
       " 38         NaN                                                NaN     NaN   \n",
       " 39         NaN                                                NaN     NaN   \n",
       " 40         NaN                                                NaN     NaN   \n",
       " 41         NaN                                                NaN     NaN   \n",
       " 42         NaN                                                NaN     NaN   \n",
       " 43         NaN                                                NaN     NaN   \n",
       " 44         NaN                                                NaN     NaN   \n",
       " 45         NaN                                                NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " 21                              NaN         NaN             NaN   \n",
       " 22                              NaN         NaN             NaN   \n",
       " 23                              NaN         NaN             NaN   \n",
       " 24                              NaN         NaN             NaN   \n",
       " 25                              NaN         NaN             NaN   \n",
       " 26                              NaN         NaN             NaN   \n",
       " 27                              NaN         NaN             NaN   \n",
       " 28                              NaN         NaN             NaN   \n",
       " 29                              NaN         NaN             NaN   \n",
       " 30                              NaN         NaN             NaN   \n",
       " 31                              NaN         NaN             NaN   \n",
       " 32                              NaN         NaN             NaN   \n",
       " 33                              NaN         NaN             NaN   \n",
       " 34                              NaN         NaN             NaN   \n",
       " 35                              NaN         NaN             NaN   \n",
       " 36                              NaN         NaN             NaN   \n",
       " 37                              NaN         NaN             NaN   \n",
       " 38                              NaN         NaN             NaN   \n",
       " 39                              NaN         NaN             NaN   \n",
       " 40                              NaN         NaN             NaN   \n",
       " 41                              NaN         NaN             NaN   \n",
       " 42                              NaN         NaN             NaN   \n",
       " 43                              NaN         NaN             NaN   \n",
       " 44                              NaN         NaN             NaN   \n",
       " 45                              NaN         NaN             NaN   \n",
       " \n",
       "                                       Transit peptide  PDB  \n",
       " 0                                                 NaN  NaN  \n",
       " 1                                                 NaN  NaN  \n",
       " 2   TRANSIT 1..?; /note=\"Mitochondrion\"; /evidence...  NaN  \n",
       " 3                                                 NaN  NaN  \n",
       " 4                                                 NaN  NaN  \n",
       " 5                                                 NaN  NaN  \n",
       " 6                                                 NaN  NaN  \n",
       " 7                                                 NaN  NaN  \n",
       " 8                                                 NaN  NaN  \n",
       " 9                                                 NaN  NaN  \n",
       " 10                                                NaN  NaN  \n",
       " 11                                                NaN  NaN  \n",
       " 12                                                NaN  NaN  \n",
       " 13                                                NaN  NaN  \n",
       " 14                                                NaN  NaN  \n",
       " 15                                                NaN  NaN  \n",
       " 16                                                NaN  NaN  \n",
       " 17                                                NaN  NaN  \n",
       " 18                                                NaN  NaN  \n",
       " 19                                                NaN  NaN  \n",
       " 20                                                NaN  NaN  \n",
       " 21                                                NaN  NaN  \n",
       " 22                                                NaN  NaN  \n",
       " 23                                                NaN  NaN  \n",
       " 24                                                NaN  NaN  \n",
       " 25                                                NaN  NaN  \n",
       " 26                                                NaN  NaN  \n",
       " 27                                                NaN  NaN  \n",
       " 28                                                NaN  NaN  \n",
       " 29                                                NaN  NaN  \n",
       " 30                                                NaN  NaN  \n",
       " 31                                                NaN  NaN  \n",
       " 32                                                NaN  NaN  \n",
       " 33                                                NaN  NaN  \n",
       " 34                                                NaN  NaN  \n",
       " 35                                                NaN  NaN  \n",
       " 36                                                NaN  NaN  \n",
       " 37                                                NaN  NaN  \n",
       " 38                                                NaN  NaN  \n",
       " 39                                                NaN  NaN  \n",
       " 40                                                NaN  NaN  \n",
       " 41                                                NaN  NaN  \n",
       " 42                                                NaN  NaN  \n",
       " 43                                                NaN  NaN  \n",
       " 44                                                NaN  NaN  \n",
       " 45                                                NaN  NaN  \n",
       " \n",
       " [46 rows x 110 columns],\n",
       "     Entry  Entry Name                                      Gene Names  \\\n",
       " 0  Q9SA18  AKH1_ARATH  AKHSDH1 AK-HSDH I At1g31230 F28K20.19 T19E23.1   \n",
       " 1  O81852  AKH2_ARATH           AKHSDH2 AK-HSDH II At4g19710 T16H5.70   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0              AKHSDH1            AK-HSDH I                  At1g31230   \n",
       " 1              AKHSDH2           AK-HSDH II                  At4g19710   \n",
       " \n",
       "      Gene Names (ORF)                                Organism  Organism (ID)  \\\n",
       " 0  F28K20.19 T19E23.1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1            T16H5.70  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Bifunctional aspartokinase/homoserine dehydrog...  ...           NaN   \n",
       " 1  Bifunctional aspartokinase/homoserine dehydrog...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide PDB  \n",
       " 0  TRANSIT 1..82; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " 1  TRANSIT 1..87; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  F1LN46  CPT1C_RAT             Cpt1c                Cpt1c   \n",
       " 1  P18886   CPT2_RAT        Cpt2 Cpt-2                 Cpt2   \n",
       " 2  P32198  CPT1A_RAT  Cpt1a Cpt-1 Cpt1                Cpt1a   \n",
       " 3  Q63704  CPT1B_RAT             Cpt1b                Cpt1b   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1                Cpt-2                         NaN               NaN   \n",
       " 2           Cpt-1 Cpt1                         NaN               NaN   \n",
       " 3                  NaN                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " 1  Rattus norvegicus (Rat)          10116   \n",
       " 2  Rattus norvegicus (Rat)          10116   \n",
       " 3  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Carnitine O-palmitoyltransferase 1, brain isof...  ...           NaN   \n",
       " 1  Carnitine O-palmitoyltransferase 2, mitochondr...  ...           NaN   \n",
       " 2  Carnitine O-palmitoyltransferase 1, liver isof...  ...           NaN   \n",
       " 3  Carnitine O-palmitoyltransferase 1, muscle iso...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES 69; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " 2  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "                         Transit peptide  \\\n",
       " 0                                   NaN   \n",
       " 1  TRANSIT 1..25; /note=\"Mitochondrion\"   \n",
       " 2                                   NaN   \n",
       " 3                                   NaN   \n",
       " \n",
       "                                         PDB  \n",
       " 0                                       NaN  \n",
       " 1  2DEB;2FW3;2FYO;2H4T;2RCU;4EP9;4EPH;4EYW;  \n",
       " 2                                       NaN  \n",
       " 3                                       NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  Q9SUW2  APT3_ARATH       APT3 At4g22570 F7K2.150                 APT3   \n",
       " 1  P31166  APT1_ARATH  APT1 APT At1g27450 F17L21.24                 APT1   \n",
       " 2  Q42563  APT2_ARATH      APT2 At1g80050 F18B13.14                 APT2   \n",
       " 3  Q9SU38  APT4_ARATH       APT4 At4g12440 T1P17.30                 APT4   \n",
       " 4  Q9LFP0  APT5_ARATH       APT5 At5g11160 F2I11.50                 APT5   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                  NaN                  At4g22570         F7K2.150   \n",
       " 1                  APT                  At1g27450        F17L21.24   \n",
       " 2                  NaN                  At1g80050        F18B13.14   \n",
       " 3                  NaN                  At4g12440         T1P17.30   \n",
       " 4                  NaN                  At5g11160         F2I11.50   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 3  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 4  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0   Adenine phosphoribosyltransferase 3 (EC 2.4.2.7)  ...           NaN   \n",
       " 1  Adenine phosphoribosyltransferase 1, chloropla...  ...           NaN   \n",
       " 2  Adenine phosphoribosyltransferase 2 (APRT 2) (...  ...           NaN   \n",
       " 3  Adenine phosphoribosyltransferase 4 (AtAPT4) (...  ...           NaN   \n",
       " 4  Adenine phosphoribosyltransferase 5 (AtAPT5) (...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET P31166-2:1; /note=\"Removed\"; /evidenc...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES P31166-2:2; /note=\"N-acetylalanine\"; /...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " 4                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide PDB  \n",
       " 0                                                NaN NaN  \n",
       " 1  TRANSIT 1..52; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " 2                                                NaN NaN  \n",
       " 3                                                NaN NaN  \n",
       " 4                                                NaN NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P78330  SERB_HUMAN       PSPH                 PSPH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Phosphoserine phosphatase (PSP) (PSPase) (EC 3...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  1L8L;1L8O;1NNL;6HYJ;6HYY;6Q6J;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name                                         Gene Names  \\\n",
       " 0  Q7XSN8  SRR_ORYSJ  SERR Os04g0555900 LOC_Os04g46930 OsJ_15733 OSJ...   \n",
       " 1  A2XWA9  SRR_ORYSI                                          OsI_16936   \n",
       " \n",
       "   Gene Names (primary)  Gene Names (synonym)   Gene Names (ordered locus)  \\\n",
       " 0                 SERR                   NaN  Os04g0555900 LOC_Os04g46930   \n",
       " 1                  NaN                   NaN                          NaN   \n",
       " \n",
       "             Gene Names (ORF)                             Organism  \\\n",
       " 0  OsJ_15733 OSJNBb0012E24.7  Oryza sativa subsp. japonica (Rice)   \n",
       " 1                  OsI_16936    Oryza sativa subsp. indica (Rice)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          39947  Serine racemase (EC 5.1.1.18) (D-serine dehydr...  ...   \n",
       " 1          39946  Serine racemase (EC 4.3.1.17) (EC 4.3.1.18) (E...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 68; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 1  MOD_RES 68; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P47858  PFKAM_RAT  Pfkm Pfk-m                 Pfkm                Pfk-m   \n",
       " 1  P30835  PFKAL_RAT  Pfkl Pfk-l                 Pfkl                Pfk-l   \n",
       " 2  P47860  PFKAP_RAT   Pfkp Pfkc                 Pfkp                 Pfkc   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 2                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  ATP-dependent 6-phosphofructokinase, muscle ty...  ...   \n",
       " 1          10116  ATP-dependent 6-phosphofructokinase, liver typ...  ...   \n",
       " 2          10116  ATP-dependent 6-phosphofructokinase, platelet ...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0  CARBOHYD 530; /note=\"O-linked (GlcNAc) serine\"...   \n",
       " 1  CARBOHYD 529; /note=\"O-linked (GlcNAc) serine\"...   \n",
       " 2  CARBOHYD 540; /note=\"O-linked (GlcNAc) serine\"...   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylthreonine\"; /evidenc...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: GlcNAcylation decreases enzyme activity. ...         NaN   \n",
       " 1  PTM: GlcNAcylation at Ser-529 by OGT decreases...         NaN   \n",
       " 2  PTM: GlcNAcylation decreases enzyme activity. ...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P38160    TTL_PIG        TTL                  TTL                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Sus scrofa (Pig)   \n",
       " \n",
       "    Organism (ID)                                 Protein names  ...  \\\n",
       " 0           9823  Tubulin--tyrosine ligase (TTL) (EC 6.3.2.25)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P07339  CATD_HUMAN  CTSD CPSD                 CTSD                 CPSD   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Cathepsin D (EC 3.4.23.5) [Cleaved into: Cathe...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 63; /note=\"O-linked (GalNAc...) threo...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: N- and O-glycosylated. {ECO:0000269|PubMe...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 21..64; /note=\"Activation peptide\"; /id...   \n",
       " \n",
       "                           Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..20; /evidence=\"ECO:0000255\"             NaN   \n",
       " \n",
       "                               PDB  \n",
       " 0  1LYA;1LYB;1LYW;4OBZ;4OC6;4OD9;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name                        Gene Names Gene Names (primary)  \\\n",
       " 0  Q9SIE1  PAT_ARATH  PAT AAT MEE17 At2g22250 T26C19.9                  PAT   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0            AAT MEE17                  At2g22250         T26C19.9   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Bifunctional aspartate aminotransferase and gl...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 306; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  \\\n",
       " 0  TRANSIT 1..55; /note=\"Chloroplast\"; /evidence=...   \n",
       " \n",
       "                          PDB  \n",
       " 0  5WMH;5WMI;5WMK;5WML;6F5V;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names  Gene Names (primary)  \\\n",
       " 0  P93804  PGMC1_MAIZE         NaN                   NaN   \n",
       " 1  P93805  PGMC2_MAIZE         NaN                   NaN   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN               NaN   \n",
       " 1                   NaN                         NaN               NaN   \n",
       " \n",
       "            Organism  Organism (ID)  \\\n",
       " 0  Zea mays (Maize)           4577   \n",
       " 1  Zea mays (Maize)           4577   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphoglucomutase, cytoplasmic 1 (PGM 1) (EC ...  ...           NaN   \n",
       " 1  Phosphoglucomutase, cytoplasmic 2 (PGM 2) (EC ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Autophosphorylated. {ECO:0000269|PubMed:9...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  P0AED9   DCM_ECOLI  dcm mec b1961 JW1944                  dcm   \n",
       " 1  P05101  MTE2_ECOLX               ecoRIIM              ecoRIIM   \n",
       " 2  P05102  MTH1_HAEPH                 hhaIM                hhaIM   \n",
       " 3  P50196  MTE8_ECOLX              eco47IIM             eco47IIM   \n",
       " 4  P0AEE0   DCM_ECO57     dcm Z3054 ECs2699                  dcm   \n",
       " 5  P15840  MTSI_SPISQ           sssIM mssSI                sssIM   \n",
       " 6  O31073  MTS1_STRAH                 sacIM                sacIM   \n",
       " 7  O52702  MTA1_ACEPA                apaLIM               apaLIM   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  mec               b1961 JW1944               NaN   \n",
       " 1                  NaN                        NaN               NaN   \n",
       " 2                  NaN                        NaN               NaN   \n",
       " 3                  NaN                        NaN               NaN   \n",
       " 4                  NaN              Z3054 ECs2699               NaN   \n",
       " 5                mssSI                        NaN               NaN   \n",
       " 6                  NaN                        NaN               NaN   \n",
       " 7                  NaN                        NaN               NaN   \n",
       " \n",
       "                                            Organism  Organism (ID)  \\\n",
       " 0                     Escherichia coli (strain K12)          83333   \n",
       " 1                                  Escherichia coli            562   \n",
       " 2                      Haemophilus parahaemolyticus            735   \n",
       " 3                                  Escherichia coli            562   \n",
       " 4                          Escherichia coli O157:H7          83334   \n",
       " 5   Spiroplasma monobiae (strain ATCC 33825 / MQ-1)           2136   \n",
       " 6                         Streptomyces achromogenes          67255   \n",
       " 7  Acetobacter pasteurianus (Acetobacter turbidans)            438   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  DNA-cytosine methyltransferase (EC 2.1.1.37) (...  ...           NaN   \n",
       " 1  Type II methyltransferase M.EcoRII (M.EcoRII) ...  ...           NaN   \n",
       " 2  Type II methyltransferase M.HhaI (M.HhaI) (EC ...  ...           NaN   \n",
       " 3  Type II methyltransferase M.Eco47II (M.Eco47II...  ...           NaN   \n",
       " 4  DNA-cytosine methyltransferase (EC 2.1.1.37) (...  ...           NaN   \n",
       " 5  Orphan methyltransferase M.SssI (M.SssI) (EC 2...  ...           NaN   \n",
       " 6  Type II methyltransferase M.SacI (M.SacI) (EC ...  ...           NaN   \n",
       " 7  Type II methyltransferase M.ApaLI (M.ApaLI) (E...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " 6                  NaN         NaN               NaN     NaN   \n",
       " 7                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide Signal peptide Transit peptide  \\\n",
       " 0                              NaN         NaN            NaN             NaN   \n",
       " 1                              NaN         NaN            NaN             NaN   \n",
       " 2                              NaN         NaN            NaN             NaN   \n",
       " 3                              NaN         NaN            NaN             NaN   \n",
       " 4                              NaN         NaN            NaN             NaN   \n",
       " 5                              NaN         NaN            NaN             NaN   \n",
       " 6                              NaN         NaN            NaN             NaN   \n",
       " 7                              NaN         NaN            NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                                NaN  \n",
       " 1                                                NaN  \n",
       " 2  10MH;1FJX;1HMY;1M0E;1MHT;1SKM;1SVU;2C7O;2C7P;2...  \n",
       " 3                                                NaN  \n",
       " 4                                                NaN  \n",
       " 5                                                NaN  \n",
       " 6                                                NaN  \n",
       " 7                                                NaN  \n",
       " \n",
       " [8 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q3B7M2  GLO2_BOVIN       HAGH                 HAGH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Hydroxyacylglutathione hydrolase, mitochondria...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 116; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..?; /note=\"Mitochondrion\"; /evidence...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q43210  PALY_WHEAT        PAL                  PAL                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                   Organism  \\\n",
       " 0                         NaN               NaN  Triticum aestivum (Wheat)   \n",
       " \n",
       "    Organism (ID)                              Protein names  ...  \\\n",
       " 0           4565  Phenylalanine ammonia-lyase (EC 4.3.1.24)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 187; /note=\"2,3-didehydroalanine (Ser)...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Contains an active site 4-methylidene-imi...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P08030  APT_MOUSE       Aprt                 Aprt                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Adenine phosphoribosyltransferase (APRT) (EC 2...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  Q07607  DAPA_RHIML             dapA mosA                 dapA   \n",
       " 1  Q92R55  DAPA_RHIME  dapA R01059 SMc02404                 dapA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 mosA                        NaN              NaN   \n",
       " 1                  NaN                     R01059         SMc02404   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Rhizobium meliloti (Ensifer meliloti) (Sinorhi...            382   \n",
       " 1  Rhizobium meliloti (strain 1021) (Ensifer meli...         266834   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  4-hydroxy-tetrahydrodipicolinate synthase (HTP...  ...           NaN   \n",
       " 1  4-hydroxy-tetrahydrodipicolinate synthase (HTP...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide Signal peptide Transit peptide  \\\n",
       " 0                              NaN         NaN            NaN             NaN   \n",
       " 1                              NaN         NaN            NaN             NaN   \n",
       " \n",
       "      PDB  \n",
       " 0  2VC6;  \n",
       " 1    NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q969G6  RIFK_HUMAN        RFK                  RFK                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Riboflavin kinase (EC 2.7.1.26) (ATP:riboflavi...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                   PDB  \n",
       " 0             NaN  1NB0;1NB9;1P4M;1Q9S;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q2TBT5  RNH2A_BOVIN   RNASEH2A             RNASEH2A                   NaN   \n",
       " 1  P25059    POL_BLVAU        pol                  pol                   NaN   \n",
       " 2  P03361     POL_BLVJ        pol                  pol                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " 2                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                                Bos taurus (Bovine)           9913   \n",
       " 1   Bovine leukemia virus (isolate Australian) (BLV)          11903   \n",
       " 2  Bovine leukemia virus (isolate Japanese BLV-1)...          11907   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Ribonuclease H2 subunit A (RNase H2 subunit A)...  ...           NaN   \n",
       " 1  Gag-Pro-Pol polyprotein [Cleaved into: Matrix ...  ...           NaN   \n",
       " 2  Gag-Pro-Pol polyprotein [Cleaved into: Matrix ...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  \\\n",
       " 0                                                NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed; by host\"; /evidenc...   \n",
       " 2  INIT_MET 1; /note=\"Removed; by host\"; /evidenc...   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0                                                NaN   \n",
       " 1  LIPID 2; /note=\"N-myristoyl glycine; by host\";...   \n",
       " 2  LIPID 2; /note=\"N-myristoyl glycine; by host\";...   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: [Matrix protein p15]: Phosphorylation of ...         NaN   \n",
       " 2  PTM: [Matrix protein p15]: Phosphorylation of ...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide PDB  \n",
       " 0             NaN             NaN NaN  \n",
       " 1             NaN             NaN NaN  \n",
       " 2             NaN             NaN NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  P26283  LDH_STRMU  ldh lct SMU_1115                  ldh   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  lct                   SMU_1115               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Streptococcus mutans serotype c (strain ATCC 7...         210007   \n",
       " \n",
       "                                    Protein names  ... Glycosylation  \\\n",
       " 0  L-lactate dehydrogenase (L-LDH) (EC 1.1.1.27)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 226; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q02759  LOX15_RAT  Alox15 Alox12l               Alox15   \n",
       " 1  F1LQ70  LOX12_RAT          Alox12               Alox12   \n",
       " 2  Q8K4F2  LX15B_RAT         Alox15b              Alox15b   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0              Alox12l                         NaN               NaN   \n",
       " 1                  NaN                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " 1  Rattus norvegicus (Rat)          10116   \n",
       " 2  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Polyunsaturated fatty acid lipoxygenase ALOX15...  ...           NaN   \n",
       " 1  Polyunsaturated fatty acid lipoxygenase ALOX12...  ...           NaN   \n",
       " 2  Polyunsaturated fatty acid lipoxygenase ALOX15...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 149; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 246; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                Gene Names Gene Names (primary)  \\\n",
       " 0  P29760  AMYI_YEASX                 STA2 DEX1                 STA2   \n",
       " 1  P04065  AMYH_YEASX            STA1 DEX2 MAL5                 STA1   \n",
       " 2  P08019  AMYG_YEAST          SGA1 SGA YIL099W                 SGA1   \n",
       " 3  Q03045  AMYG_AMORE                      GAMP                 GAMP   \n",
       " 4  P36914  AMYG_ASPOR  glaA gluB AO090010000746                 glaA   \n",
       " 5  P22861  AMYG_SCHOC                      GAM1                 GAM1   \n",
       " 6  P42042  AMYG_BLAAD                       GAA                  GAA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 DEX1                        NaN              NaN   \n",
       " 1            DEX2 MAL5                        NaN              NaN   \n",
       " 2                  SGA                    YIL099W              NaN   \n",
       " 3                  NaN                        NaN              NaN   \n",
       " 4                 gluB                        NaN   AO090010000746   \n",
       " 5                  NaN                        NaN              NaN   \n",
       " 6                  NaN                        NaN              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0           Saccharomyces cerevisiae (Baker's yeast)           4932   \n",
       " 1           Saccharomyces cerevisiae (Baker's yeast)           4932   \n",
       " 2  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 3  Amorphotheca resinae (Creosote fungus) (Hormoc...           5101   \n",
       " 4  Aspergillus oryzae (strain ATCC 42149 / RIB 40...         510516   \n",
       " 5  Schwanniomyces occidentalis (Yeast) (Debaryomy...          27300   \n",
       " 6  Blastobotrys adeninivorans (Yeast) (Arxula ade...         409370   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Glucoamylase S2 (EC 3.2.1.3) (1,4-alpha-D-gluc...  ...   \n",
       " 1  Glucoamylase S1 (EC 3.2.1.3) (1,4-alpha-D-gluc...  ...   \n",
       " 2  Glucoamylase, intracellular sporulation-specif...  ...   \n",
       " 3  Glucoamylase P (EC 3.2.1.3) (1,4-alpha-D-gluca...  ...   \n",
       " 4  Glucoamylase (EC 3.2.1.3) (1,4-alpha-D-glucan ...  ...   \n",
       " 5  Glucoamylase 1 (EC 3.2.1.3) (1,4-alpha-D-gluca...  ...   \n",
       " 6  Glucoamylase (EC 3.2.1.3) (1,4-alpha-D-glucan ...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 35; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 1  CARBOHYD 35; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 2                                                NaN                  NaN   \n",
       " 3  CARBOHYD 200; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 4  CARBOHYD 39; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 5  CARBOHYD 61; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 6  CARBOHYD 54; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " 2         NaN               NaN     NaN                              NaN   \n",
       " 3         NaN               NaN     NaN                              NaN   \n",
       " 4         NaN               NaN     NaN                              NaN   \n",
       " 5         NaN               NaN     NaN                              NaN   \n",
       " 6         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0                                                NaN   \n",
       " 1                                                NaN   \n",
       " 2                                                NaN   \n",
       " 3                                                NaN   \n",
       " 4  PROPEP 20..25; /evidence=\"ECO:0000250\"; /id=\"P...   \n",
       " 5                                                NaN   \n",
       " 6                                                NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide    PDB  \n",
       " 0                           SIGNAL 1..21             NaN    NaN  \n",
       " 1                           SIGNAL 1..21             NaN    NaN  \n",
       " 2                                    NaN             NaN    NaN  \n",
       " 3                           SIGNAL 1..29             NaN  6FHW;  \n",
       " 4  SIGNAL 1..19; /evidence=\"ECO:0000250\"             NaN    NaN  \n",
       " 5  SIGNAL 1..22; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " 6  SIGNAL 1..18; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P98063  BMP1_MOUSE       Bmp1                 Bmp1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Bone morphogenetic protein 1 (BMP-1) (EC 3.4.2...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 96; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 939; /note=\"Omega-N-methylarginine\"; /...     NaN   \n",
       " \n",
       "    Post-translational modification  \\\n",
       " 0                              NaN   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 26..125; /evidence=\"ECO:0000250|UniProt...   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..25; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  Q51390  GLPK2_PSEAE  glpK2 glpK glpK1 PA3582                glpK2   \n",
       " 1  Q9HY41  GLPK1_PSEAE       glpK1 glpK2 PA3579                glpK1   \n",
       " 2  A4XXN1   GLPK_PSEMY           glpK Pmen_3345                 glpK   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           glpK glpK1                     PA3582               NaN   \n",
       " 1                glpK2                     PA3579               NaN   \n",
       " 2                  NaN                  Pmen_3345               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 1  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 2                 Pseudomonas mendocina (strain ymp)         399739   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glycerol kinase 2 (EC 2.7.1.30) (ATP:glycerol ...  ...           NaN   \n",
       " 1  Glycerol kinase 1 (EC 2.7.1.30) (ATP:glycerol ...  ...           NaN   \n",
       " 2  Glycerol kinase (EC 2.7.1.30) (ATP:glycerol 3-...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q15125  EBP_HUMAN        EBP                  EBP                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  3-beta-hydroxysteroid-Delta(8),Delta(7)-isomer...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylthreonine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  6OHT;6OHU;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q9Y2T3  GUAD_HUMAN  GDA KIAA1258                  GDA             KIAA1258   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Guanine deaminase (Guanase) (Guanine aminase) ...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 453; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  2UZ9;3E0L;4AQL;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P10937   PNMT_RAT       Pnmt                 Pnmt                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Phenylethanolamine N-methyltransferase (PNMTas...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide  PDB  \n",
       " 0         NaN             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                     Gene Names Gene Names (primary)  \\\n",
       " 0  P54716  GLVA_BACSU  glvA glv-1 glvG malA BSU08180                 glvA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0      glv-1 glvG malA                   BSU08180               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Bacillus subtilis (strain 168)         224308   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Maltose-6'-phosphate glucosidase (EC 3.2.1.122...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1U8X;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  P12684  HMDH2_YEAST  HMG2 YLR450W L9324.2                 HMG2   \n",
       " 1  P12683  HMDH1_YEAST          HMG1 YML075C                 HMG1   \n",
       " 2  B2KX91   HMDH_GANLU                   NaN                  NaN   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YLR450W          L9324.2   \n",
       " 1                   NaN                    YML075C              NaN   \n",
       " 2                   NaN                        NaN              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2  Ganoderma lucidum (Ling zhi medicinal fungus) ...           5315   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  3-hydroxy-3-methylglutaryl-coenzyme A reductas...  ...   \n",
       " 1  3-hydroxy-3-methylglutaryl-coenzyme A reductas...  ...   \n",
       " 2  3-hydroxy-3-methylglutaryl-coenzyme A reductas...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 115; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 115; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 2                                                NaN                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 565; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 1         NaN  MOD_RES 552; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "      Entry   Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0   P45395   KDSD_ECOLI  kdsD yrbH b3197 JW3164                 kdsD   \n",
       " 1   P17115   GUTQ_ECOLI  gutQ srlQ b2708 JW5431                 gutQ   \n",
       " 2   Q8FDQ2   KPSF_ECOL6              kpsF c3686                 kpsF   \n",
       " 3   Q47334  KPSF5_ECOLX                    kpsF                 kpsF   \n",
       " 4   P42502  KPSF1_ECOLX                    kpsF                 kpsF   \n",
       " 5   Q8X4S5   GUTQ_ECO57      gutQ Z4015 ECs3564                 gutQ   \n",
       " 6   Q8X9J0   KDSD_ECO57      kdsD Z4560 ECs4076                 kdsD   \n",
       " 7   Q8FEN7   GUTQ_ECOL6              gutQ c3262                 gutQ   \n",
       " 8   Q8FD73   KDSD_ECOL6              kdsD c3957                 kdsD   \n",
       " 9   Q83MK2   GUTQ_SHIFL       gutQ SF2731 S2922                 gutQ   \n",
       " 10  Q83JF4   KDSD_SHIFL       kdsD SF3237 S3455                 kdsD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  yrbH               b3197 JW3164               NaN   \n",
       " 1                  srlQ               b2708 JW5431               NaN   \n",
       " 2                   NaN                      c3686               NaN   \n",
       " 3                   NaN                        NaN               NaN   \n",
       " 4                   NaN                        NaN               NaN   \n",
       " 5                   NaN              Z4015 ECs3564               NaN   \n",
       " 6                   NaN              Z4560 ECs4076               NaN   \n",
       " 7                   NaN                      c3262               NaN   \n",
       " 8                   NaN                      c3957               NaN   \n",
       " 9                   NaN               SF2731 S2922               NaN   \n",
       " 10                  NaN               SF3237 S3455               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                       Escherichia coli (strain K12)          83333   \n",
       " 2   Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 3                                    Escherichia coli            562   \n",
       " 4                                    Escherichia coli            562   \n",
       " 5                            Escherichia coli O157:H7          83334   \n",
       " 6                            Escherichia coli O157:H7          83334   \n",
       " 7   Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 8   Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 9                                   Shigella flexneri            623   \n",
       " 10                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Arabinose 5-phosphate isomerase KdsD (API) (L-...  ...           NaN   \n",
       " 1   Arabinose 5-phosphate isomerase GutQ (API) (G-...  ...           NaN   \n",
       " 2   Arabinose 5-phosphate isomerase KpsF (API) (EC...  ...           NaN   \n",
       " 3   Arabinose 5-phosphate isomerase KpsF (API) (EC...  ...           NaN   \n",
       " 4   Arabinose 5-phosphate isomerase KpsF (API) (EC...  ...           NaN   \n",
       " 5   Arabinose 5-phosphate isomerase GutQ (API) (G-...  ...           NaN   \n",
       " 6   Arabinose 5-phosphate isomerase KdsD (API) (L-...  ...           NaN   \n",
       " 7   Arabinose 5-phosphate isomerase GutQ (API) (G-...  ...           NaN   \n",
       " 8   Arabinose 5-phosphate isomerase KdsD (API) (L-...  ...           NaN   \n",
       " 9   Arabinose 5-phosphate isomerase GutQ (API) (G-...  ...           NaN   \n",
       " 10  Arabinose 5-phosphate isomerase KdsD (API) (L-...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " \n",
       "     Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0                NaN     NaN                              NaN         NaN   \n",
       " 1                NaN     NaN                              NaN         NaN   \n",
       " 2                NaN     NaN                              NaN         NaN   \n",
       " 3                NaN     NaN                              NaN         NaN   \n",
       " 4                NaN     NaN                              NaN         NaN   \n",
       " 5                NaN     NaN                              NaN         NaN   \n",
       " 6                NaN     NaN                              NaN         NaN   \n",
       " 7                NaN     NaN                              NaN         NaN   \n",
       " 8                NaN     NaN                              NaN         NaN   \n",
       " 9                NaN     NaN                              NaN         NaN   \n",
       " 10               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide    PDB  \n",
       " 0              NaN             NaN  2XHZ;  \n",
       " 1              NaN             NaN    NaN  \n",
       " 2              NaN             NaN    NaN  \n",
       " 3              NaN             NaN    NaN  \n",
       " 4              NaN             NaN    NaN  \n",
       " 5              NaN             NaN    NaN  \n",
       " 6              NaN             NaN    NaN  \n",
       " 7              NaN             NaN    NaN  \n",
       " 8              NaN             NaN    NaN  \n",
       " 9              NaN             NaN    NaN  \n",
       " 10             NaN             NaN    NaN  \n",
       " \n",
       " [11 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  Q8Z6F6  DHE4_SALTI  gdhA STY1815 t1178                 gdhA   \n",
       " 1  P15111  DHE4_SALTY        gdhA STM1299                 gdhA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN              STY1815 t1178               NaN   \n",
       " 1                   NaN                    STM1299               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                                   Salmonella typhi          90370   \n",
       " 1  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  NADP-specific glutamate dehydrogenase (NADP-GD...  ...           NaN   \n",
       " 1  NADP-specific glutamate dehydrogenase (NADP-GD...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P27169  PON1_HUMAN   PON1 PON                 PON1                  PON   \n",
       " 1  Q15166  PON3_HUMAN       PON3                 PON3                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Serum paraoxonase/arylesterase 1 (PON 1) (EC 3...  ...   \n",
       " 1           9606  Serum paraoxonase/lactonase 3 (EC 3.1.1.2) (EC...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0  CARBOHYD 253; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 1  CARBOHYD 29; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES 165; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Glycosylated. {ECO:0000269|PubMed:1476071...         NaN   \n",
       " 1  PTM: The signal sequence is not cleaved. {ECO:...         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide    PDB  \n",
       " 0                   SIGNAL 2..?; /note=\"Not cleaved\"             NaN  1V04;  \n",
       " 1  SIGNAL 1..?; /note=\"Not cleaved\"; /evidence=\"E...             NaN    NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  P07801  CHER_SALTY  cheR STM1918                 cheR   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    STM1918               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Chemotaxis protein methyltransferase (EC 2.1.1...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1AF7;1BC5;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0  P0DJA4  INVB_ZYMMO    sacC invB sucE3 ZMO0375                 sacC   \n",
       " 1  P0DJA7   SCR_ZYMMO          sacA invA ZMO0942                 sacA   \n",
       " 2  F8DT27  INVB_ZYMMA  sacC invB sucE3 Zmob_0915                 sacC   \n",
       " 3  F8DVG5   SCR_ZYMMA        sacA invA Zmob_0361                 sacA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           invB sucE3                    ZMO0375               NaN   \n",
       " 1                 invA                    ZMO0942               NaN   \n",
       " 2           invB sucE3                  Zmob_0915               NaN   \n",
       " 3                 invA                  Zmob_0361               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Zymomonas mobilis subsp. mobilis (strain ATCC ...         264203   \n",
       " 1  Zymomonas mobilis subsp. mobilis (strain ATCC ...         264203   \n",
       " 2  Zymomonas mobilis subsp. mobilis (strain ATCC ...         555217   \n",
       " 3  Zymomonas mobilis subsp. mobilis (strain ATCC ...         555217   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Extracellular sucrase (EC 3.2.1.26) (Beta-fruc...  ...           NaN   \n",
       " 1  Sucrose-6-phosphate hydrolase (Sucrase) (EC 3....  ...           NaN   \n",
       " 2  Extracellular sucrase (EC 3.2.1.26) (Beta-fruc...  ...           NaN   \n",
       " 3  Sucrose-6-phosphate hydrolase (Sucrase) (EC 3....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P55159   PON1_RAT   Pon1 Pon                 Pon1                  Pon   \n",
       " 1  Q68FP2   PON3_RAT       Pon3                 Pon3                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Serum paraoxonase/arylesterase 1 (PON 1) (EC 3...  ...   \n",
       " 1          10116  Serum paraoxonase/lactonase 3 (EC 3.1.1.2) (EC...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0  CARBOHYD 253; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 1  CARBOHYD 50; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES 165; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Glycosylated. {ECO:0000250|UniProtKB:P271...         NaN   \n",
       " 1  PTM: Glycosylated. {ECO:0000250}.; PTM: The si...         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  PDB  \n",
       " 0                   SIGNAL 2..?; /note=\"Not cleaved\"             NaN  NaN  \n",
       " 1  SIGNAL 1..?; /note=\"Not cleaved\"; /evidence=\"E...             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q96558  UGDH1_SOYBN       UGD1                 UGD1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                   Organism  Organism (ID)  \\\n",
       " 0  Glycine max (Soybean) (Glycine hispida)           3847   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  UDP-glucose 6-dehydrogenase 1 (UDP-Glc dehydro...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 393; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name                     Gene Names Gene Names (primary)  \\\n",
       " 0  P46367  ALDH4_YEAST  ALD4 ALD7 ALDH2 YOR374W O6730                 ALD4   \n",
       " 1  P54115  ALDH6_YEAST        ALD6 ALDH1 YPL061W LPE9                 ALD6   \n",
       " 2  P40047  ALDH5_YEAST        ALD5 ALD3 ALDH5 YER073W                 ALD5   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0           ALD7 ALDH2                    YOR374W            O6730   \n",
       " 1                ALDH1                    YPL061W             LPE9   \n",
       " 2           ALD3 ALDH5                    YER073W              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Potassium-activated aldehyde dehydrogenase, mi...  ...           NaN   \n",
       " 1  Magnesium-activated aldehyde dehydrogenase, cy...  ...           NaN   \n",
       " 2  Aldehyde dehydrogenase 5, mitochondrial (EC 1....  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 96; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..24; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " 1                                                NaN  NaN  \n",
       " 2  TRANSIT 1..23; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  Q8I1T1  KAD3_PLAF7  GAK PF3D7_0415600                  GAK   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN    PF3D7_0415600   \n",
       " \n",
       "                               Organism  Organism (ID)  \\\n",
       " 0  Plasmodium falciparum (isolate 3D7)          36329   \n",
       " \n",
       "                                       Protein names  ... Glycosylation  \\\n",
       " 0  GTP:AMP phosphotransferase (PfGAK) (EC 2.7.4.10)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..?; /note=\"Mitochondrion\"; /evidence...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O73884  PHOP1_CHICK   PHOSPHO1             PHOSPHO1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Gallus gallus (Chicken)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9031  Phosphoethanolamine/phosphocholine phosphatase...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  O95470  SGPL1_HUMAN  SGPL1 KIAA1252                SGPL1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0             KIAA1252                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Sphingosine-1-phosphate lyase 1 (S1PL) (SP-lya...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 353; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  4Q6R;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q32LG3  MDHM_BOVIN       MDH2                 MDH2                   NaN   \n",
       " 1  Q3T145  MDHC_BOVIN       MDH1                 MDH1                   NaN   \n",
       " 2  Q5EAD2  SERA_BOVIN      PHGDH                PHGDH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 1                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 2                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Malate dehydrogenase, mitochondrial (EC 1.1.1.37)  ...   \n",
       " 1           9913  Malate dehydrogenase, cytoplasmic (EC 1.1.1.37...  ...   \n",
       " 2           9913  D-3-phosphoglycerate dehydrogenase (3-PGDH) (E...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0  CARBOHYD 33; /note=\"O-linked (GlcNAc) serine\";...   \n",
       " 1                                                NaN   \n",
       " 2                                                NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 78; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 2  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Acetylation is enhanced after treatment e...         NaN   \n",
       " 1  PTM: ISGylated. {ECO:0000250|UniProtKB:P40925}...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  PDB  \n",
       " 0             NaN  TRANSIT 1..24; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " 1             NaN                                                NaN  NaN  \n",
       " 2             NaN                                                NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q00667  HGD_EMENI  hmgA AN1897                 hmgA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                         NaN           AN1897   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Emericella nidulans (strain FGSC A4 / ATCC 381...         227321   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Homogentisate 1,2-dioxygenase (EC 1.13.11.5) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  Q9UUB1  RIB4_SCHPO  rib4 SPBC409.13                 rib4   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN       SPBC409.13   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  6,7-dimethyl-8-ribityllumazine synthase (DMRL ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1KYV;1KYX;1KYY;1KZ1;1KZ4;1KZ6;1KZ9;2A57;2A58;2...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P05707  SRLD_ECOLI  srlD gutD b2705 JW2674                 srlD   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 gutD               b2705 JW2674               NaN   \n",
       " \n",
       "                         Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Sorbitol-6-phosphate 2-dehydrogenase (EC 1.1.1...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name                                         Gene Names  \\\n",
       " 0  Q6Z1Y6  SNAT2_ORYSJ  SNAT2 Os08g0102000 LOC_Os08g01170 B1147B12.22 ...   \n",
       " 1  Q5KQI6  SNAT1_ORYSJ  SNAT1 GNAT5 NSI SNAT Os05g0481000 LOC_Os05g402...   \n",
       " 2  A2Y5T7  SNAT1_ORYSI                     SNAT1 NSI OsI_019680 OsI_20361   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym)   Gene Names (ordered locus)  \\\n",
       " 0                SNAT2                  NaN  Os08g0102000 LOC_Os08g01170   \n",
       " 1                SNAT1       GNAT5 NSI SNAT  Os05g0481000 LOC_Os05g40260   \n",
       " 2                SNAT1                  NSI                          NaN   \n",
       " \n",
       "                        Gene Names (ORF)                             Organism  \\\n",
       " 0      B1147B12.22 OsJ_25726 P0015C07.5  Oryza sativa subsp. japonica (Rice)   \n",
       " 1  OsJ_018182 OsJ_18949 OSJNBa0095J22.4  Oryza sativa subsp. japonica (Rice)   \n",
       " 2                  OsI_019680 OsI_20361    Oryza sativa subsp. indica (Rice)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          39947  Serotonin N-acetyltransferase 2, chloroplastic...  ...   \n",
       " 1          39947  Serotonin N-acetyltransferase 1, chloroplastic...  ...   \n",
       " 2          39946  Serotonin N-acetyltransferase 1, chloroplastic...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " 1           NaN                  NaN         NaN               NaN     NaN   \n",
       " 2           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  \\\n",
       " 0  TRANSIT 1..41; /note=\"Chloroplast\"; /evidence=...   \n",
       " 1  TRANSIT 1..74; /note=\"Chloroplast\"; /evidence=...   \n",
       " 2  TRANSIT 1..83; /note=\"Chloroplast\"; /evidence=...   \n",
       " \n",
       "                          PDB  \n",
       " 0                        NaN  \n",
       " 1  6K5M;7DAI;7DAJ;7DAK;7DAL;  \n",
       " 2                        NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O31411  SACC_BACL7         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                  Organism  \\\n",
       " 0                         NaN               NaN  Bacillus sp. (strain L7)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          62626  Levanase (EC 3.2.1.65) (2,6-beta-D-fructan fru...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN              NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P54387  DHE4_AGABI       gdhA                 gdhA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                     Organism  Organism (ID)  \\\n",
       " 0  Agaricus bisporus (White button mushroom)           5341   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  NADP-specific glutamate dehydrogenase (NADP-GD...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0   P51840  GUC2E_RAT     Gucy2e Guc2e Gucy2d               Gucy2e   \n",
       " 1   P20595  GCYB1_RAT  Gucy1b1 Guc1b3 Gucy1b3              Gucy1b1   \n",
       " 2   P23897  GUC2C_RAT            Gucy2c Guc2c               Gucy2c   \n",
       " 3   P55205  GUC2G_RAT             Gucy2g Ksgc               Gucy2g   \n",
       " 4   P18910  ANPRA_RAT                    Npr1                 Npr1   \n",
       " 5   P51839  GUC2D_RAT           Gucy2d Gucy2e               Gucy2d   \n",
       " 6   P51842  GUC2F_RAT            Gucy2f Guc2f               Gucy2f   \n",
       " 7   P16067  ANPRB_RAT                    Npr2                 Npr2   \n",
       " 8   P19686  GCYA1_RAT  Gucy1a1 Guc1a1 Gucy1a3              Gucy1a1   \n",
       " 9   P22717  GCYB2_RAT                 Gucy1b2              Gucy1b2   \n",
       " 10  Q9WVI4  GCYA2_RAT                 Gucy1a2              Gucy1a2   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0          Guc2e Gucy2d                         NaN               NaN   \n",
       " 1        Guc1b3 Gucy1b3                         NaN               NaN   \n",
       " 2                 Guc2c                         NaN               NaN   \n",
       " 3                  Ksgc                         NaN               NaN   \n",
       " 4                   NaN                         NaN               NaN   \n",
       " 5                Gucy2e                         NaN               NaN   \n",
       " 6                 Guc2f                         NaN               NaN   \n",
       " 7                   NaN                         NaN               NaN   \n",
       " 8        Guc1a1 Gucy1a3                         NaN               NaN   \n",
       " 9                   NaN                         NaN               NaN   \n",
       " 10                  NaN                         NaN               NaN   \n",
       " \n",
       "                    Organism  Organism (ID)  \\\n",
       " 0   Rattus norvegicus (Rat)          10116   \n",
       " 1   Rattus norvegicus (Rat)          10116   \n",
       " 2   Rattus norvegicus (Rat)          10116   \n",
       " 3   Rattus norvegicus (Rat)          10116   \n",
       " 4   Rattus norvegicus (Rat)          10116   \n",
       " 5   Rattus norvegicus (Rat)          10116   \n",
       " 6   Rattus norvegicus (Rat)          10116   \n",
       " 7   Rattus norvegicus (Rat)          10116   \n",
       " 8   Rattus norvegicus (Rat)          10116   \n",
       " 9   Rattus norvegicus (Rat)          10116   \n",
       " 10  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                         Protein names  ...  \\\n",
       " 0   Retinal guanylyl cyclase 1 (RETGC-1) (EC 4.6.1...  ...   \n",
       " 1   Guanylate cyclase soluble subunit beta-1 (GCS-...  ...   \n",
       " 2   Guanylyl cyclase C (GC-C) (EC 4.6.1.2) (Heat-s...  ...   \n",
       " 3   Guanylate cyclase 2G (EC 4.6.1.2) (Guanylyl cy...  ...   \n",
       " 4   Atrial natriuretic peptide receptor 1 (EC 4.6....  ...   \n",
       " 5   Guanylate cyclase 2D (EC 4.6.1.2) (Guanylate c...  ...   \n",
       " 6   Retinal guanylyl cyclase 2 (RETGC-2) (EC 4.6.1...  ...   \n",
       " 7   Atrial natriuretic peptide receptor 2 (EC 4.6....  ...   \n",
       " 8   Guanylate cyclase soluble subunit alpha-1 (GCS...  ...   \n",
       " 9   Guanylate cyclase soluble subunit beta-2 (GCS-...  ...   \n",
       " 10  Guanylate cyclase soluble subunit alpha-2 (GCS...  ...   \n",
       " \n",
       "                                         Glycosylation Initiator methionine  \\\n",
       " 0   CARBOHYD 300; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1                                                 NaN                  NaN   \n",
       " 2   CARBOHYD 31; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 3   CARBOHYD 55; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 4   CARBOHYD 41; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 5   CARBOHYD 304; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 6                                                 NaN                  NaN   \n",
       " 7   CARBOHYD 24; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 8                                                 NaN                  NaN   \n",
       " 9                                                 NaN                  NaN   \n",
       " 10                                                NaN                  NaN   \n",
       " \n",
       "     Lipidation                                   Modified residue Peptide  \\\n",
       " 0          NaN                                                NaN     NaN   \n",
       " 1          NaN                                                NaN     NaN   \n",
       " 2          NaN                                                NaN     NaN   \n",
       " 3          NaN                                                NaN     NaN   \n",
       " 4          NaN  MOD_RES 515; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 5          NaN                                                NaN     NaN   \n",
       " 6          NaN                                                NaN     NaN   \n",
       " 7          NaN  MOD_RES 513; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 8          NaN  MOD_RES 266; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 9          NaN                                                NaN     NaN   \n",
       " 10         NaN                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0   PTM: There are 9 conserved cysteine residues i...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2   PTM: Glycosylation at Asn-74 and/or Asn-78 is ...         NaN   \n",
       " 3   PTM: N-glycosylated. {ECO:0000250|UniProtKB:Q6...         NaN   \n",
       " 4   PTM: Phosphorylation of the protein kinase-lik...         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6   PTM: There are 9 conserved cysteine residues i...         NaN   \n",
       " 7   PTM: Phosphorylated. Phosphorylation of the pr...         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " \n",
       "                                        Signal peptide Transit peptide  \\\n",
       " 0   SIGNAL 1..54; /evidence=\"ECO:0000250|UniProtKB...             NaN   \n",
       " 1                                                 NaN             NaN   \n",
       " 2               SIGNAL 1..22; /evidence=\"ECO:0000255\"             NaN   \n",
       " 3               SIGNAL 1..43; /evidence=\"ECO:0000255\"             NaN   \n",
       " 4                                        SIGNAL 1..28             NaN   \n",
       " 5               SIGNAL 1..66; /evidence=\"ECO:0000255\"             NaN   \n",
       " 6               SIGNAL 1..50; /evidence=\"ECO:0000255\"             NaN   \n",
       " 7               SIGNAL 1..16; /evidence=\"ECO:0000255\"             NaN   \n",
       " 8                                                 NaN             NaN   \n",
       " 9                                                 NaN             NaN   \n",
       " 10                                                NaN             NaN   \n",
       " \n",
       "                                               PDB  \n",
       " 0                                             NaN  \n",
       " 1                                           3HLS;  \n",
       " 2                                             NaN  \n",
       " 3                                             NaN  \n",
       " 4   1DP4;1T34;3A3K;7BRG;7BRH;7BRI;7BRJ;7BRK;7BRL;  \n",
       " 5                                             NaN  \n",
       " 6                                             NaN  \n",
       " 7                                             NaN  \n",
       " 8                                             NaN  \n",
       " 9                                             NaN  \n",
       " 10                                            NaN  \n",
       " \n",
       " [11 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P23946  CMA1_HUMAN  CMA1 CYH CYM                 CMA1              CYH CYM   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Chymase (EC 3.4.21.39) (Alpha-chymase) (Mast c...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 80; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "                                           Propeptide  Signal peptide  \\\n",
       " 0  PROPEP 20..21; /note=\"Activation peptide\"; /ev...    SIGNAL 1..19   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1KLT;1NN6;1PJP;1T31;2HVX;3N7O;3S0N;4AFQ;4AFS;4...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P07632   SODC_RAT        Sod1                 Sod1                  NaN   \n",
       " 1  P07895   SODM_RAT        Sod2                 Sod2                  NaN   \n",
       " 2  Q08420   SODE_RAT  Sod3 Sod-3                 Sod3                Sod-3   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 2                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116         Superoxide dismutase [Cu-Zn] (EC 1.15.1.1)  ...   \n",
       " 1          10116  Superoxide dismutase [Mn], mitochondrial (EC 1...  ...   \n",
       " 2          10116  Extracellular superoxide dismutase [Cu-Zn] (EC...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0                                                NaN   \n",
       " 1                                                NaN   \n",
       " 2  CARBOHYD 114; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " \n",
       "                                 Initiator methionine  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1                                                NaN   \n",
       " 2                                                NaN   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0  LIPID 7; /note=\"S-palmitoyl cysteine\"; /eviden...   \n",
       " 1                                                NaN   \n",
       " 2                                                NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 58; /note=\"3'-nitrotyrosine\"; /evidenc...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Palmitoylation helps nuclear targeting an...         NaN   \n",
       " 1  PTM: Nitrated under oxidative stress. Nitratio...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "                           Signal peptide  \\\n",
       " 0                                    NaN   \n",
       " 1                                    NaN   \n",
       " 2  SIGNAL 1..15; /evidence=\"ECO:0000255\"   \n",
       " \n",
       "                         Transit peptide  PDB  \n",
       " 0                                   NaN  NaN  \n",
       " 1  TRANSIT 1..24; /note=\"Mitochondrion\"  NaN  \n",
       " 2                                   NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9JHY8  DNLI1_RAT       Lig1                 Lig1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  DNA ligase 1 (EC 6.5.1.1) (DNA ligase I) (Poly...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 50; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0  P06700  SIR2_YEAST    SIR2 MAR1 YDL042C D2714                 SIR2   \n",
       " 1  P53685  HST1_YEAST               HST1 YOL068C                 HST1   \n",
       " 2  P53686  HST2_YEAST         HST2 YPL015C LPA2C                 HST2   \n",
       " 3  P53687  HST3_YEAST       HST3 YOR025W OR26.15                 HST3   \n",
       " 4  P53688  HST4_YEAST     HST4 YDR191W YD9346.03                 HST4   \n",
       " 5  Q8IXJ6  SIR2_HUMAN         SIRT2 SIR2L SIR2L2                SIRT2   \n",
       " 6  Q96EB6  SIR1_HUMAN               SIRT1 SIR2L1                SIRT1   \n",
       " 7  Q25337  SIR2_LEIMA  SIR2rp1 SIR2 LMJF_26_0210              SIR2rp1   \n",
       " 8  Q757M7  SIR2_ASHGO               SIR2 AEL013C                 SIR2   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 MAR1                    YDL042C            D2714   \n",
       " 1                  NaN                    YOL068C              NaN   \n",
       " 2                  NaN                    YPL015C            LPA2C   \n",
       " 3                  NaN                    YOR025W          OR26.15   \n",
       " 4                  NaN                    YDR191W        YD9346.03   \n",
       " 5         SIR2L SIR2L2                        NaN              NaN   \n",
       " 6               SIR2L1                        NaN              NaN   \n",
       " 7                 SIR2                        NaN     LMJF_26_0210   \n",
       " 8                  NaN                    AEL013C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 3  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 4  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 5                               Homo sapiens (Human)           9606   \n",
       " 6                               Homo sapiens (Human)           9606   \n",
       " 7                                   Leishmania major           5664   \n",
       " 8  Ashbya gossypii (strain ATCC 10895 / CBS 109.5...         284811   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  NAD-dependent histone deacetylase SIR2 (EC 2.3...  ...           NaN   \n",
       " 1  NAD-dependent protein deacetylase HST1 (EC 2.3...  ...           NaN   \n",
       " 2  NAD-dependent protein deacetylase HST2 (EC 2.3...  ...           NaN   \n",
       " 3  NAD-dependent histone deacetylase HST3 (EC 2.3...  ...           NaN   \n",
       " 4  NAD-dependent histone deacetylase HST4 (EC 2.3...  ...           NaN   \n",
       " 5  NAD-dependent protein deacetylase sirtuin-2 (E...  ...           NaN   \n",
       " 6  NAD-dependent protein deacetylase sirtuin-1 (h...  ...           NaN   \n",
       " 7  NAD-dependent protein deacetylase SIR2rp1 (EC ...  ...           NaN   \n",
       " 8  NAD-dependent histone deacetylase SIR2 (EC 2.3...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 6  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 7                                                NaN         NaN   \n",
       " 8                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " 4                                                NaN     NaN   \n",
       " 5  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 6  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 7                                                NaN     NaN   \n",
       " 8                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5  PTM: Phosphorylated at phosphoserine and phosp...         NaN   \n",
       " 6  PTM: Methylated on multiple lysine residues; m...         NaN   \n",
       " 7                                                NaN         NaN   \n",
       " 8                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " 4             NaN             NaN   \n",
       " 5             NaN             NaN   \n",
       " 6             NaN             NaN   \n",
       " 7             NaN             NaN   \n",
       " 8             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                         2HJH;4IAO;  \n",
       " 1                                                NaN  \n",
       " 2  1Q14;1Q17;1Q1A;1SZC;1SZD;2OD2;2OD7;2OD9;2QQF;2...  \n",
       " 3                                                NaN  \n",
       " 4                                                NaN  \n",
       " 5  1J8F;3ZGO;3ZGV;4L3O;4R8M;4RMG;4RMH;4RMI;4RMJ;4...  \n",
       " 6           4I5I;4IF6;4IG9;4KXQ;4ZZH;4ZZI;4ZZJ;5BTR;  \n",
       " 7                                                NaN  \n",
       " 8                                                NaN  \n",
       " \n",
       " [9 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  Q8NS80  HISN_CORGL  hisN Cgl0799 cg0910                 hisN   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN             Cgl0799 cg0910               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Corynebacterium glutamicum (strain ATCC 13032 ...         196627   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Histidinol-phosphatase (HolPase) (EC 3.1.3.15)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P00644  NUC_STAAU            nuc                  nuc   \n",
       " 1  Q5HHM4  NUC_STAAC  nuc SACOL0860                  nuc   \n",
       " 2  Q6GB41  NUC_STAAS    nuc SAS0756                  nuc   \n",
       " 3  Q6GIK1  NUC_STAAR    nuc SAR0847                  nuc   \n",
       " 4  Q7A6P2  NUC_STAAN     nuc SA0746                  nuc   \n",
       " 5  Q8NXI6  NUC_STAAW     nuc MW0769                  nuc   \n",
       " 6  Q99VJ0  NUC_STAAM    nuc SAV0815                  nuc   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                        NaN               NaN   \n",
       " 1                   NaN                  SACOL0860               NaN   \n",
       " 2                   NaN                    SAS0756               NaN   \n",
       " 3                   NaN                    SAR0847               NaN   \n",
       " 4                   NaN                     SA0746               NaN   \n",
       " 5                   NaN                     MW0769               NaN   \n",
       " 6                   NaN                    SAV0815               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                              Staphylococcus aureus           1280   \n",
       " 1                 Staphylococcus aureus (strain COL)          93062   \n",
       " 2             Staphylococcus aureus (strain MSSA476)         282459   \n",
       " 3             Staphylococcus aureus (strain MRSA252)         282458   \n",
       " 4                Staphylococcus aureus (strain N315)         158879   \n",
       " 5                 Staphylococcus aureus (strain MW2)         196620   \n",
       " 6  Staphylococcus aureus (strain Mu50 / ATCC 700699)         158878   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Thermonuclease (TNase) (EC 3.1.31.1) (Micrococ...  ...           NaN   \n",
       " 1  Thermonuclease (TNase) (EC 3.1.31.1) (Micrococ...  ...           NaN   \n",
       " 2  Thermonuclease (TNase) (EC 3.1.31.1) (Micrococ...  ...           NaN   \n",
       " 3  Thermonuclease (TNase) (EC 3.1.31.1) (Micrococ...  ...           NaN   \n",
       " 4  Thermonuclease (TNase) (EC 3.1.31.1) (Micrococ...  ...           NaN   \n",
       " 5  Thermonuclease (TNase) (EC 3.1.31.1) (Micrococ...  ...           NaN   \n",
       " 6  Thermonuclease (TNase) (EC 3.1.31.1) (Micrococ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " 6                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  \\\n",
       " 0                              NaN   \n",
       " 1                              NaN   \n",
       " 2                              NaN   \n",
       " 3                              NaN   \n",
       " 4                              NaN   \n",
       " 5                              NaN   \n",
       " 6                              NaN   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 27..63; /evidence=\"ECO:0000269|PubMed:8...   \n",
       " 1  PROPEP 24..60; /evidence=\"ECO:0000250\"; /id=\"P...   \n",
       " 2  PROPEP 24..60; /evidence=\"ECO:0000250\"; /id=\"P...   \n",
       " 3  PROPEP 24..60; /evidence=\"ECO:0000250\"; /id=\"P...   \n",
       " 4  PROPEP 24..60; /evidence=\"ECO:0000250\"; /id=\"P...   \n",
       " 5  PROPEP 24..60; /evidence=\"ECO:0000250\"; /id=\"P...   \n",
       " 6  PROPEP 24..60; /evidence=\"ECO:0000250\"; /id=\"P...   \n",
       " \n",
       "                           Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..26; /evidence=\"ECO:0000255\"             NaN   \n",
       " 1  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN   \n",
       " 2  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN   \n",
       " 3  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN   \n",
       " 4  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN   \n",
       " 5  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN   \n",
       " 6  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1A2T;1A2U;1A3T;1A3U;1A3V;1AEX;1ENA;1ENC;1EQV;1...  \n",
       " 1                                         4WRD;5DAU;  \n",
       " 2                                                NaN  \n",
       " 3                                                NaN  \n",
       " 4                                                NaN  \n",
       " 5  2LKV;2M00;2OXP;2PW5;2PW7;2PYK;2PZT;2PZU;2PZW;3...  \n",
       " 6                                                NaN  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  P20192   DGKA_PIG  DGKA DAGK DAGK1                 DGKA   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           DAGK DAGK1                         NaN               NaN   \n",
       " \n",
       "            Organism  Organism (ID)  \\\n",
       " 0  Sus scrofa (Pig)           9823   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Diacylglycerol kinase alpha (DAG kinase alpha)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 483; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  O88483   PDP1_RAT  Pdp1 Ppm2c                 Pdp1                Ppm2c   \n",
       " 1  O88484   PDP2_RAT        Pdp2                 Pdp2                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  [Pyruvate dehydrogenase [acetyl-transferring]]...  ...   \n",
       " 1          10116  [Pyruvate dehydrogenase [acetyl-transferring]]...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 202; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide    PDB  \n",
       " 0  TRANSIT 1..71; /note=\"Mitochondrion\"; /evidenc...  2PNQ;  \n",
       " 1  TRANSIT 1..67; /note=\"Mitochondrion\"; /evidenc...    NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P07341  ALDOB_CHICK      ALDOB                ALDOB                   NaN   \n",
       " 1  P53449  ALDOC_CHICK      ALDOC                ALDOC                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Gallus gallus (Chicken)   \n",
       " 1                         NaN               NaN  Gallus gallus (Chicken)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9031  Fructose-bisphosphate aldolase B (EC 4.1.2.13)...  ...   \n",
       " 1           9031  Fructose-bisphosphate aldolase C (EC 4.1.2.13)...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN                                                NaN   \n",
       " \n",
       "    Lipidation Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN              NaN     NaN                              NaN   \n",
       " 1         NaN              NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide  PDB  \n",
       " 0         NaN             NaN             NaN  NaN  \n",
       " 1         NaN             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P45381  ACY2_HUMAN  ASPA ACY2 ASP                 ASPA   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0             ACY2 ASP                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aspartoacylase (EC 3.5.1.15) (Aminoacylase-2) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                       PDB  \n",
       " 0             NaN  2I3C;2O4H;2O53;2Q51;4MRI;4MXU;4NFR;4TNU;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                                Gene Names  \\\n",
       " 0  P00861  DCDA_ECOLI                         lysA b2838 JW2806   \n",
       " 1  E0IWI3  DCDA_ECOLW  lysA ECW_m3083 WFL_15045 EschWDRAFT_0728   \n",
       " \n",
       "   Gene Names (primary)  Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                 lysA                   NaN               b2838 JW2806   \n",
       " 1                 lysA                   NaN        ECW_m3083 WFL_15045   \n",
       " \n",
       "   Gene Names (ORF)                                           Organism  \\\n",
       " 0              NaN                      Escherichia coli (strain K12)   \n",
       " 1  EschWDRAFT_0728  Escherichia coli (strain ATCC 9637 / CCM 2024 ...   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          83333  Diaminopimelate decarboxylase (DAP decarboxyla...  ...   \n",
       " 1         566546  Diaminopimelate decarboxylase (DAP decarboxyla...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 54; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 1  MOD_RES 54; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1KNW;1KO0;  \n",
       " 1             NaN         NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0   P11875  SYR_ECOLI            argS b1876 JW1865                 argS   \n",
       " 1   B6I0S7  SYR_ECOSE               argS ECSE_2111                 argS   \n",
       " 2   Q8XCH2  SYR_ECO57           argS Z2929 ECs2586                 argS   \n",
       " 3   B1XHE4  SYR_ECODH            argS ECDH10B_2017                 argS   \n",
       " 4   B7L7S9  SYR_ECO55            argS EC55989_2055                 argS   \n",
       " 5   A1AC37  SYR_ECOK1  argS Ecok1_17330 APECO1_926                 argS   \n",
       " 6   B7M2G7  SYR_ECO8A             argS ECIAI1_1963                 argS   \n",
       " 7   B7MW70  SYR_ECO81              argS ECED1_2145                 argS   \n",
       " 8   Q1RAQ8  SYR_ECOUT             argS UTI89_C2080                 argS   \n",
       " 9   Q54869  SYR_STRPN                 argS SP_2078                 argS   \n",
       " 10  C4ZQG0  SYR_ECOBW                argS BWG_1690                 argS   \n",
       " 11  A8A178  SYR_ECOHS              argS EcHS_A1971                 argS   \n",
       " 12  A7ZN02  SYR_ECO24          argS EcE24377A_2108                 argS   \n",
       " 13  B5YR22  SYR_ECO5E           argS ECH74115_2613                 argS   \n",
       " 14  B7NS42  SYR_ECO7I            argS ECIAI39_1173                 argS   \n",
       " 15  B7MBT5  SYR_ECO45              argS ECS88_1935                 argS   \n",
       " 16  B1LCZ5  SYR_ECOSM            argS EcSMS35_1310                 argS   \n",
       " 17  B7USQ3  SYR_ECO27             argS E2348C_2001                 argS   \n",
       " 18  B7NBM7  SYR_ECOLU              argS ECUMN_2174                 argS   \n",
       " 19  Q0TGV6  SYR_ECOL5                argS ECP_1822                 argS   \n",
       " 20  Q8FGQ0  SYR_ECOL6                   argS c2291                 argS   \n",
       " 21  B1J0L2  SYR_ECOLC              argS EcolC_1756                 argS   \n",
       " 22  B7LPH0  SYR_ESCF3               argS EFER_1197                 argS   \n",
       " 23  Q83KQ3  SYR_SHIFL            argS SF1917 S2007                 argS   \n",
       " 24  Q8DN69  SYR_STRR6                 argS spr1890                 argS   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN               b1876 JW1865              NaN   \n",
       " 1                    NaN                  ECSE_2111              NaN   \n",
       " 2                    NaN              Z2929 ECs2586              NaN   \n",
       " 3                    NaN               ECDH10B_2017              NaN   \n",
       " 4                    NaN               EC55989_2055              NaN   \n",
       " 5                    NaN                Ecok1_17330       APECO1_926   \n",
       " 6                    NaN                ECIAI1_1963              NaN   \n",
       " 7                    NaN                 ECED1_2145              NaN   \n",
       " 8                    NaN                UTI89_C2080              NaN   \n",
       " 9                    NaN                    SP_2078              NaN   \n",
       " 10                   NaN                   BWG_1690              NaN   \n",
       " 11                   NaN                 EcHS_A1971              NaN   \n",
       " 12                   NaN             EcE24377A_2108              NaN   \n",
       " 13                   NaN              ECH74115_2613              NaN   \n",
       " 14                   NaN               ECIAI39_1173              NaN   \n",
       " 15                   NaN                 ECS88_1935              NaN   \n",
       " 16                   NaN               EcSMS35_1310              NaN   \n",
       " 17                   NaN                E2348C_2001              NaN   \n",
       " 18                   NaN                 ECUMN_2174              NaN   \n",
       " 19                   NaN                   ECP_1822              NaN   \n",
       " 20                   NaN                      c2291              NaN   \n",
       " 21                   NaN                 EcolC_1756              NaN   \n",
       " 22                   NaN                  EFER_1197              NaN   \n",
       " 23                   NaN               SF1917 S2007              NaN   \n",
       " 24                   NaN                    spr1890              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain SE11)         409438   \n",
       " 2                            Escherichia coli O157:H7          83334   \n",
       " 3               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 4              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 5                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 6                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 7                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 8              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 9   Streptococcus pneumoniae serotype 4 (strain AT...         170187   \n",
       " 10    Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 11                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 12  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 13    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 14      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 15       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 16          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 17  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 18  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 19    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 20  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 21  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 22  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 23                                  Shigella flexneri            623   \n",
       " 24  Streptococcus pneumoniae (strain ATCC BAA-255 ...         171101   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 1   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 2   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 3   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 4   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 5   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 6   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 7   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 8   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 9   Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 10  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 11  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 12  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 13  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 14  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 15  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 16  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 17  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 18  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 19  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 20  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 21  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 22  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 23  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " 24  Arginine--tRNA ligase (EC 6.1.1.19) (Arginyl-t...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                   NaN         NaN               NaN     NaN   \n",
       " 1                   NaN         NaN               NaN     NaN   \n",
       " 2                   NaN         NaN               NaN     NaN   \n",
       " 3                   NaN         NaN               NaN     NaN   \n",
       " 4                   NaN         NaN               NaN     NaN   \n",
       " 5                   NaN         NaN               NaN     NaN   \n",
       " 6                   NaN         NaN               NaN     NaN   \n",
       " 7                   NaN         NaN               NaN     NaN   \n",
       " 8                   NaN         NaN               NaN     NaN   \n",
       " 9                   NaN         NaN               NaN     NaN   \n",
       " 10                  NaN         NaN               NaN     NaN   \n",
       " 11                  NaN         NaN               NaN     NaN   \n",
       " 12                  NaN         NaN               NaN     NaN   \n",
       " 13                  NaN         NaN               NaN     NaN   \n",
       " 14                  NaN         NaN               NaN     NaN   \n",
       " 15                  NaN         NaN               NaN     NaN   \n",
       " 16                  NaN         NaN               NaN     NaN   \n",
       " 17                  NaN         NaN               NaN     NaN   \n",
       " 18                  NaN         NaN               NaN     NaN   \n",
       " 19                  NaN         NaN               NaN     NaN   \n",
       " 20                  NaN         NaN               NaN     NaN   \n",
       " 21                  NaN         NaN               NaN     NaN   \n",
       " 22                  NaN         NaN               NaN     NaN   \n",
       " 23                  NaN         NaN               NaN     NaN   \n",
       " 24                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " 21                              NaN         NaN             NaN   \n",
       " 22                              NaN         NaN             NaN   \n",
       " 23                              NaN         NaN             NaN   \n",
       " 24                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                   PDB  \n",
       " 0              NaN  4OBY;5B63;5YYM;5YYN;  \n",
       " 1              NaN                   NaN  \n",
       " 2              NaN                   NaN  \n",
       " 3              NaN                   NaN  \n",
       " 4              NaN                   NaN  \n",
       " 5              NaN                   NaN  \n",
       " 6              NaN                   NaN  \n",
       " 7              NaN                   NaN  \n",
       " 8              NaN                   NaN  \n",
       " 9              NaN                   NaN  \n",
       " 10             NaN                   NaN  \n",
       " 11             NaN                   NaN  \n",
       " 12             NaN                   NaN  \n",
       " 13             NaN                   NaN  \n",
       " 14             NaN                   NaN  \n",
       " 15             NaN                   NaN  \n",
       " 16             NaN                   NaN  \n",
       " 17             NaN                   NaN  \n",
       " 18             NaN                   NaN  \n",
       " 19             NaN                   NaN  \n",
       " 20             NaN                   NaN  \n",
       " 21             NaN                   NaN  \n",
       " 22             NaN                   NaN  \n",
       " 23             NaN                   NaN  \n",
       " 24             NaN                   NaN  \n",
       " \n",
       " [25 rows x 110 columns],\n",
       "      Entry   Entry Name                Gene Names Gene Names (primary)  \\\n",
       " 0   F4JAU3   P4H2_ARATH  P4H2 At3g06300 F24P17.24                 P4H2   \n",
       " 1   Q24JN5   P4H5_ARATH   P4H5 At2g17720 T17A5.10                 P4H5   \n",
       " 2   Q9ZW86   P4H1_ARATH       P4H1 At2g43080 MFL8                 P4H1   \n",
       " 3   Q9LN20   P4H3_ARATH  P4H3 At1g20270 F14O10.12                 P4H3   \n",
       " 4   Q8LAN3   P4H4_ARATH  P4H4 At5g18900 F17K4.150                 P4H4   \n",
       " 5   Q8L970   P4H7_ARATH   P4H7 At3g28480 MFJ20.17                 P4H7   \n",
       " 6   Q8VZJ7   P4H9_ARATH  P4H9 At4g33910 F17I5.100                 P4H9   \n",
       " 7   F4ILF8  P4H13_ARATH    P4H13 At2g23096 F21P24                P4H13   \n",
       " 8   F4JZ24  P4H10_ARATH  P4H10 At5g66060 K2A18.14                P4H10   \n",
       " 9   Q9SZT0  P4H11_ARATH  P4H11 At4g35820 F4B14.90                P4H11   \n",
       " 10  F4JNU8   P4H8_ARATH            P4H8 At4g35810                 P4H8   \n",
       " 11  Q8GXT7  P4H12_ARATH   P4H12 At4g25600 M7J2.30                P4H12   \n",
       " 12  F4J0A8   P4H6_ARATH   P4H6 At3g28490 MFJ20.18                 P4H6   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN                  At3g06300        F24P17.24   \n",
       " 1                    NaN                  At2g17720         T17A5.10   \n",
       " 2                    NaN                  At2g43080             MFL8   \n",
       " 3                    NaN                  At1g20270        F14O10.12   \n",
       " 4                    NaN                  At5g18900        F17K4.150   \n",
       " 5                    NaN                  At3g28480         MFJ20.17   \n",
       " 6                    NaN                  At4g33910        F17I5.100   \n",
       " 7                    NaN                  At2g23096           F21P24   \n",
       " 8                    NaN                  At5g66060         K2A18.14   \n",
       " 9                    NaN                  At4g35820         F4B14.90   \n",
       " 10                   NaN                  At4g35810              NaN   \n",
       " 11                   NaN                  At4g25600          M7J2.30   \n",
       " 12                   NaN                  At3g28490         MFJ20.18   \n",
       " \n",
       "                                   Organism  Organism (ID)  \\\n",
       " 0   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 3   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 4   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 5   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 6   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 7   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 8   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 9   Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 10  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 11  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 12  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                         Protein names  ...  \\\n",
       " 0   Prolyl 4-hydroxylase 2 (AtP4H-2) (AtP4H2) (EC ...  ...   \n",
       " 1      Prolyl 4-hydroxylase 5 (AtP4H5) (EC 1.14.11.2)  ...   \n",
       " 2   Prolyl 4-hydroxylase 1 (AtP4H-1) (AtP4H1) (EC ...  ...   \n",
       " 3   Probable prolyl 4-hydroxylase 3 (AtP4H3) (EC 1...  ...   \n",
       " 4   Probable prolyl 4-hydroxylase 4 (AtP4H4) (EC 1...  ...   \n",
       " 5   Probable prolyl 4-hydroxylase 7 (AtP4H7) (EC 1...  ...   \n",
       " 6   Probable prolyl 4-hydroxylase 9 (AtP4H9) (EC 1...  ...   \n",
       " 7    Prolyl 4-hydroxylase 13 (AtP4H13) (EC 1.14.11.2)  ...   \n",
       " 8   Probable prolyl 4-hydroxylase 10 (AtP4H10) (EC...  ...   \n",
       " 9   Probable prolyl 4-hydroxylase 11 (AtP4H11) (EC...  ...   \n",
       " 10  Probable prolyl 4-hydroxylase 8 (AtP4H8) (EC 1...  ...   \n",
       " 11  Probable prolyl 4-hydroxylase 12 (AtP4H12) (EC...  ...   \n",
       " 12  Probable prolyl 4-hydroxylase 6 (AtP4H6) (EC 1...  ...   \n",
       " \n",
       "                                         Glycosylation Initiator methionine  \\\n",
       " 0   CARBOHYD 165; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1   CARBOHYD 51; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 2                                                 NaN                  NaN   \n",
       " 3   CARBOHYD 218; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 4   CARBOHYD 77; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 5   CARBOHYD 96; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 6   CARBOHYD 221; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 7   CARBOHYD 242; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 8   CARBOHYD 220; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 9                                                 NaN                  NaN   \n",
       " 10  CARBOHYD 46; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 11  CARBOHYD 211; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 12  CARBOHYD 160; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "     Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0          NaN               NaN     NaN                              NaN   \n",
       " 1          NaN               NaN     NaN                              NaN   \n",
       " 2          NaN               NaN     NaN                              NaN   \n",
       " 3          NaN               NaN     NaN                              NaN   \n",
       " 4          NaN               NaN     NaN                              NaN   \n",
       " 5          NaN               NaN     NaN                              NaN   \n",
       " 6          NaN               NaN     NaN                              NaN   \n",
       " 7          NaN               NaN     NaN                              NaN   \n",
       " 8          NaN               NaN     NaN                              NaN   \n",
       " 9          NaN               NaN     NaN                              NaN   \n",
       " 10         NaN               NaN     NaN                              NaN   \n",
       " 11         NaN               NaN     NaN                              NaN   \n",
       " 12         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "     Propeptide  Signal peptide Transit peptide PDB  \n",
       " 0          NaN             NaN             NaN NaN  \n",
       " 1          NaN             NaN             NaN NaN  \n",
       " 2          NaN             NaN             NaN NaN  \n",
       " 3          NaN             NaN             NaN NaN  \n",
       " 4          NaN             NaN             NaN NaN  \n",
       " 5          NaN             NaN             NaN NaN  \n",
       " 6          NaN             NaN             NaN NaN  \n",
       " 7          NaN             NaN             NaN NaN  \n",
       " 8          NaN             NaN             NaN NaN  \n",
       " 9          NaN             NaN             NaN NaN  \n",
       " 10         NaN             NaN             NaN NaN  \n",
       " 11         NaN             NaN             NaN NaN  \n",
       " 12         NaN             NaN             NaN NaN  \n",
       " \n",
       " [13 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P48644  AL1A1_BOVIN                      ALDH1A1              ALDH1A1   \n",
       " 1  Q2KJC9  AL7A1_BOVIN                      ALDH7A1              ALDH7A1   \n",
       " 2  P20000  ALDH2_BOVIN                        ALDH2                ALDH2   \n",
       " 3  Q2KJH9  AL9A1_BOVIN                      ALDH9A1              ALDH9A1   \n",
       " 4  P52476  AL1B1_BOVIN  ALDH1B1 ALDH1B2 ALDH5 ALDHX              ALDH1B1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1                  NaN                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " 3                  NaN                         NaN               NaN   \n",
       " 4  ALDH1B2 ALDH5 ALDHX                         NaN               NaN   \n",
       " \n",
       "               Organism  Organism (ID)  \\\n",
       " 0  Bos taurus (Bovine)           9913   \n",
       " 1  Bos taurus (Bovine)           9913   \n",
       " 2  Bos taurus (Bovine)           9913   \n",
       " 3  Bos taurus (Bovine)           9913   \n",
       " 4  Bos taurus (Bovine)           9913   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aldehyde dehydrogenase 1A1 (EC 1.2.1.19) (EC 1...  ...           NaN   \n",
       " 1  Alpha-aminoadipic semialdehyde dehydrogenase (...  ...           NaN   \n",
       " 2  Aldehyde dehydrogenase, mitochondrial (EC 1.2....  ...           NaN   \n",
       " 3  4-trimethylaminobutyraldehyde dehydrogenase (T...  ...           NaN   \n",
       " 4  Aldehyde dehydrogenase X, mitochondrial (EC 1....  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 1  MOD_RES 94; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " 2  MOD_RES 55; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 3  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 4  MOD_RES 45; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: The N-terminus is blocked most probably b...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  \\\n",
       " 0             NaN                                                NaN   \n",
       " 1             NaN  TRANSIT 1..26; /note=\"Mitochondrion\"; /evidenc...   \n",
       " 2             NaN  TRANSIT 1..21; /note=\"Mitochondrion\"; /evidenc...   \n",
       " 3             NaN                                                NaN   \n",
       " 4             NaN  TRANSIT <1..11; /note=\"Mitochondrion\"; /eviden...   \n",
       " \n",
       "           PDB  \n",
       " 0         NaN  \n",
       " 1         NaN  \n",
       " 2  1A4Z;1AG8;  \n",
       " 3         NaN  \n",
       " 4         NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "      Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0   P0A6V1  GLGC_ECOLI    glgC b3430 JW3393                 glgC   \n",
       " 1   B6I2Z6  GLGC_ECOSE       glgC ECSE_3699                 glgC   \n",
       " 2   P0A6V3  GLGC_ECO57   glgC Z4792 ECs4275                 glgC   \n",
       " 3   B1X775  GLGC_ECODH    glgC ECDH10B_3604                 glgC   \n",
       " 4   B7L4W0  GLGC_ECO55    glgC EC55989_3840                 glgC   \n",
       " 5   B7M2J3  GLGC_ECO8A     glgC ECIAI1_3576                 glgC   \n",
       " 6   B7N1M2  GLGC_ECO81      glgC ECED1_4105                 glgC   \n",
       " 7   Q1R5J6  GLGC_ECOUT     glgC UTI89_C3939                 glgC   \n",
       " 8   C4ZVY0  GLGC_ECOBW        glgC BWG_3122                 glgC   \n",
       " 9   A8A5P0  GLGC_ECOHS      glgC EcHS_A3630                 glgC   \n",
       " 10  B5YUI6  GLGC_ECO5E   glgC ECH74115_4742                 glgC   \n",
       " 11  A7ZSW3  GLGC_ECO24  glgC EcE24377A_3909                 glgC   \n",
       " 12  B1LI91  GLGC_ECOSM    glgC EcSMS35_3712                 glgC   \n",
       " 13  B7MDR5  GLGC_ECO45      glgC ECS88_3828                 glgC   \n",
       " 14  B7NMJ5  GLGC_ECO7I    glgC ECIAI39_3911                 glgC   \n",
       " 15  B7UKY7  GLGC_ECO27     glgC E2348C_3676                 glgC   \n",
       " 16  B7NE40  GLGC_ECOLU      glgC ECUMN_3894                 glgC   \n",
       " 17  Q0TC29  GLGC_ECOL5        glgC ECP_3524                 glgC   \n",
       " 18  P0A6V2  GLGC_ECOL6           glgC c4217                 glgC   \n",
       " 19  B1IP34  GLGC_ECOLC      glgC EcolC_0282                 glgC   \n",
       " 20  B7LSE1  GLGC_ESCF3       glgC EFER_3407                 glgC   \n",
       " 21  P0A6V4  GLGC_SHIFL    glgC SF3453 S4310                 glgC   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                    NaN               b3430 JW3393               NaN   \n",
       " 1                    NaN                  ECSE_3699               NaN   \n",
       " 2                    NaN              Z4792 ECs4275               NaN   \n",
       " 3                    NaN               ECDH10B_3604               NaN   \n",
       " 4                    NaN               EC55989_3840               NaN   \n",
       " 5                    NaN                ECIAI1_3576               NaN   \n",
       " 6                    NaN                 ECED1_4105               NaN   \n",
       " 7                    NaN                UTI89_C3939               NaN   \n",
       " 8                    NaN                   BWG_3122               NaN   \n",
       " 9                    NaN                 EcHS_A3630               NaN   \n",
       " 10                   NaN              ECH74115_4742               NaN   \n",
       " 11                   NaN             EcE24377A_3909               NaN   \n",
       " 12                   NaN               EcSMS35_3712               NaN   \n",
       " 13                   NaN                 ECS88_3828               NaN   \n",
       " 14                   NaN               ECIAI39_3911               NaN   \n",
       " 15                   NaN                E2348C_3676               NaN   \n",
       " 16                   NaN                 ECUMN_3894               NaN   \n",
       " 17                   NaN                   ECP_3524               NaN   \n",
       " 18                   NaN                      c4217               NaN   \n",
       " 19                   NaN                 EcolC_0282               NaN   \n",
       " 20                   NaN                  EFER_3407               NaN   \n",
       " 21                   NaN               SF3453 S4310               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain SE11)         409438   \n",
       " 2                            Escherichia coli O157:H7          83334   \n",
       " 3               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 4              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 5                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 6                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 7              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 8     Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 9                  Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 10    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 11  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 12          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 13       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 14      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 15  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 16  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 17    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 18  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 19  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 20  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 21                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 1   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 2   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 3   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 4   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 5   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 6   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 7   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 8   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 9   Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 10  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 11  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 12  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 13  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 14  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 15  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 16  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 17  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 18  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 19  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 20  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " 21  Glucose-1-phosphate adenylyltransferase (EC 2....  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 19                                                NaN         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "     Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0                NaN     NaN                              NaN         NaN   \n",
       " 1                NaN     NaN                              NaN         NaN   \n",
       " 2                NaN     NaN                              NaN         NaN   \n",
       " 3                NaN     NaN                              NaN         NaN   \n",
       " 4                NaN     NaN                              NaN         NaN   \n",
       " 5                NaN     NaN                              NaN         NaN   \n",
       " 6                NaN     NaN                              NaN         NaN   \n",
       " 7                NaN     NaN                              NaN         NaN   \n",
       " 8                NaN     NaN                              NaN         NaN   \n",
       " 9                NaN     NaN                              NaN         NaN   \n",
       " 10               NaN     NaN                              NaN         NaN   \n",
       " 11               NaN     NaN                              NaN         NaN   \n",
       " 12               NaN     NaN                              NaN         NaN   \n",
       " 13               NaN     NaN                              NaN         NaN   \n",
       " 14               NaN     NaN                              NaN         NaN   \n",
       " 15               NaN     NaN                              NaN         NaN   \n",
       " 16               NaN     NaN                              NaN         NaN   \n",
       " 17               NaN     NaN                              NaN         NaN   \n",
       " 18               NaN     NaN                              NaN         NaN   \n",
       " 19               NaN     NaN                              NaN         NaN   \n",
       " 20               NaN     NaN                              NaN         NaN   \n",
       " 21               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  \\\n",
       " 0              NaN             NaN   \n",
       " 1              NaN             NaN   \n",
       " 2              NaN             NaN   \n",
       " 3              NaN             NaN   \n",
       " 4              NaN             NaN   \n",
       " 5              NaN             NaN   \n",
       " 6              NaN             NaN   \n",
       " 7              NaN             NaN   \n",
       " 8              NaN             NaN   \n",
       " 9              NaN             NaN   \n",
       " 10             NaN             NaN   \n",
       " 11             NaN             NaN   \n",
       " 12             NaN             NaN   \n",
       " 13             NaN             NaN   \n",
       " 14             NaN             NaN   \n",
       " 15             NaN             NaN   \n",
       " 16             NaN             NaN   \n",
       " 17             NaN             NaN   \n",
       " 18             NaN             NaN   \n",
       " 19             NaN             NaN   \n",
       " 20             NaN             NaN   \n",
       " 21             NaN             NaN   \n",
       " \n",
       "                                               PDB  \n",
       " 0   5L6S;5L6V;5MNI;6R8B;6R8U;6SHJ;6SHN;6SHQ;6SI8;  \n",
       " 1                                             NaN  \n",
       " 2                                             NaN  \n",
       " 3                                             NaN  \n",
       " 4                                             NaN  \n",
       " 5                                             NaN  \n",
       " 6                                             NaN  \n",
       " 7                                             NaN  \n",
       " 8                                             NaN  \n",
       " 9                                             NaN  \n",
       " 10                                            NaN  \n",
       " 11                                            NaN  \n",
       " 12                                            NaN  \n",
       " 13                                            NaN  \n",
       " 14                                            NaN  \n",
       " 15                                            NaN  \n",
       " 16                                            NaN  \n",
       " 17                                            NaN  \n",
       " 18                                            NaN  \n",
       " 19                                            NaN  \n",
       " 20                                            NaN  \n",
       " 21                                            NaN  \n",
       " \n",
       " [22 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P00436  PCXA_PSEPU       pcaG                 pcaG                   NaN   \n",
       " 1  P00437  PCXB_PSEPU       pcaH                 pcaH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas putida (Arthrobacter siderocapsula...            303   \n",
       " 1  Pseudomonas putida (Arthrobacter siderocapsula...            303   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Protocatechuate 3,4-dioxygenase alpha chain (E...  ...            NaN   \n",
       " 1  Protocatechuate 3,4-dioxygenase beta chain (EC...  ...            NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1YKK;1YKL;1YKM;1YKN;1YKO;1YKP;2PCD;3LKT;3LMX;3...  \n",
       " 1  1YKK;1YKL;1YKM;1YKN;1YKO;1YKP;2PCD;3LKT;3LMX;3...  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q6AXM5  CEPT1_RAT       Cept1                Cept1                  NaN   \n",
       " 1  Q66H21  CHPT1_RAT  Chpt1 Cpt1                Chpt1                 Cpt1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Choline/ethanolaminephosphotransferase 1 (EC 2...  ...   \n",
       " 1          10116  Cholinephosphotransferase 1 (EC 2.7.8.2) (Diac...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0  CARBOHYD 144; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 1                                                NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 18; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P26302  KPPR_WHEAT         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                   Organism  \\\n",
       " 0                         NaN               NaN  Triticum aestivum (Wheat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           4565  Phosphoribulokinase, chloroplastic (PRK) (PRKa...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..53; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  E7FHX6  NDRZ_PYRFU  rnr PF0440                  rnr                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     PF0440               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pyrococcus furiosus (strain ATCC 43587 / DSM 3...         186497   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Vitamin B12-dependent ribonucleoside-diphospha...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: This protein undergoes a protein self spl...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide    PDB  \n",
       " 0             NaN             NaN  1DQ3;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  P66776  BUTA_STAAW     butA MW0100                 butA   \n",
       " 1  P99120  BUTA_STAAN     butA SA0122                 butA   \n",
       " 2  Q5HJP2  BUTA_STAAC  butA SACOL0111                 butA   \n",
       " 3  Q6GKH9  BUTA_STAAR    butA SAR0129                 butA   \n",
       " 4  Q6GCZ8  BUTA_STAAS    butA SAS0101                 butA   \n",
       " 5  P66775  BUTA_STAAM    butA SAV0126                 butA   \n",
       " 6  Q5HKG6  BUTA_STAEQ   butA SERP2379                 butA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                     MW0100               NaN   \n",
       " 1                   NaN                     SA0122               NaN   \n",
       " 2                   NaN                  SACOL0111               NaN   \n",
       " 3                   NaN                    SAR0129               NaN   \n",
       " 4                   NaN                    SAS0101               NaN   \n",
       " 5                   NaN                    SAV0126               NaN   \n",
       " 6                   NaN                   SERP2379               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                 Staphylococcus aureus (strain MW2)         196620   \n",
       " 1                Staphylococcus aureus (strain N315)         158879   \n",
       " 2                 Staphylococcus aureus (strain COL)          93062   \n",
       " 3             Staphylococcus aureus (strain MRSA252)         282458   \n",
       " 4             Staphylococcus aureus (strain MSSA476)         282459   \n",
       " 5  Staphylococcus aureus (strain Mu50 / ATCC 700699)         158878   \n",
       " 6  Staphylococcus epidermidis (strain ATCC 35984 ...         176279   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Diacetyl reductase [(S)-acetoin forming] (EC 1...  ...           NaN   \n",
       " 1  Diacetyl reductase [(S)-acetoin forming] (EC 1...  ...           NaN   \n",
       " 2  Diacetyl reductase [(S)-acetoin forming] (EC 1...  ...           NaN   \n",
       " 3  Diacetyl reductase [(S)-acetoin forming] (EC 1...  ...           NaN   \n",
       " 4  Diacetyl reductase [(S)-acetoin forming] (EC 1...  ...           NaN   \n",
       " 5  Diacetyl reductase [(S)-acetoin forming] (EC 1...  ...           NaN   \n",
       " 6  Diacetyl reductase [(S)-acetoin forming] (EC 1...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " 6                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " 5             NaN  NaN  \n",
       " 6             NaN  NaN  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P22061  PIMT_HUMAN      PCMT1                PCMT1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Protein-L-isoaspartate(D-aspartate) O-methyltr...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1I1N;1KR5;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q0P5A4  UCK1_BOVIN       UCK1                 UCK1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Uridine-cytidine kinase 1 (UCK 1) (EC 2.7.1.48...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 251; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P06622  XYLE1_PSEPU       xylE                 xylE                   NaN   \n",
       " 1  Q04285  XYLE2_PSEPU       xylE                 xylE                   NaN   \n",
       " 2  P08127   NAHH_PSEPU       nahH                 nahH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " 2                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas putida (Arthrobacter siderocapsula...            303   \n",
       " 1  Pseudomonas putida (Arthrobacter siderocapsula...            303   \n",
       " 2  Pseudomonas putida (Arthrobacter siderocapsula...            303   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Metapyrocatechase (MPC) (EC 1.13.11.2) (CatO2a...  ...            NaN   \n",
       " 1  Metapyrocatechase (MPC) (EC 1.13.11.2) (CatO2a...  ...            NaN   \n",
       " 2  Metapyrocatechase (MPC) (EC 1.13.11.2) (CatO2a...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide Signal peptide Transit peptide  \\\n",
       " 0                              NaN         NaN            NaN             NaN   \n",
       " 1                              NaN         NaN            NaN             NaN   \n",
       " 2                              NaN         NaN            NaN             NaN   \n",
       " \n",
       "      PDB  \n",
       " 0  1MPY;  \n",
       " 1    NaN  \n",
       " 2    NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  P19623  SPEE_HUMAN  SRM SPS1 SRML1                  SRM   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           SPS1 SRML1                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Spermidine synthase (SPDSY) (EC 2.5.1.16) (Put...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                        PDB  \n",
       " 0             NaN  2O05;2O06;2O07;2O0L;3RW9;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  Q04750   TOP1_MOUSE    Top1 Top-1                 Top1   \n",
       " 1  O70157  TOP3A_MOUSE    Top3a Top3                Top3a   \n",
       " 2  Q9Z321  TOP3B_MOUSE  Top3b Top3b1                Top3b   \n",
       " 3  Q8R4U6  TOP1M_MOUSE        Top1mt               Top1mt   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                Top-1                         NaN               NaN   \n",
       " 1                 Top3                         NaN               NaN   \n",
       " 2               Top3b1                         NaN               NaN   \n",
       " 3                  NaN                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " 2  Mus musculus (Mouse)          10090   \n",
       " 3  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  DNA topoisomerase 1 (EC 5.6.2.1) (DNA topoisom...  ...           NaN   \n",
       " 1  DNA topoisomerase 3-alpha (EC 5.6.2.1) (DNA to...  ...           NaN   \n",
       " 2  DNA topoisomerase 3-beta-1 (EC 5.6.2.1) (DNA t...  ...           NaN   \n",
       " 3  DNA topoisomerase I, mitochondrial (TOP1mt) (E...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Sumoylated. Lys-119 is the main site of s...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  PDB  \n",
       " 0             NaN                                                NaN  NaN  \n",
       " 1             NaN                                                NaN  NaN  \n",
       " 2             NaN                                                NaN  NaN  \n",
       " 3             NaN  TRANSIT 1..43; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q96I15  SCLY_HUMAN   SCLY SCL                 SCLY                  SCL   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                              Protein names  ...  \\\n",
       " 0           9606  Selenocysteine lyase (hSCL) (EC 4.4.1.16)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  3GZC;3GZD;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q9I5Y1  ALF_PSEAE  fba fda PA0555                  fba   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  fda                     PA0555               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Fructose-bisphosphate aldolase (FBP aldolase) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q3ZCJ2  AK1A1_BOVIN     AKR1A1               AKR1A1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Aldo-keto reductase family 1 member A1 (EC 1.1...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O88867    KMO_RAT        Kmo                  Kmo                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Kynurenine 3-monooxygenase (EC 1.14.13.9) (Kyn...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  6LKD;6LKE;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   P0A6U8  GLGA_ECOLI             glgA b3429 JW3392                 glgA   \n",
       " 1   Q43847  SSY2_SOLTU                      SS2 SSII                  SS2   \n",
       " 2   B6I2Z5  GLGA_ECOSE                glgA ECSE_3698                 glgA   \n",
       " 3   P0A6V0  GLGA_ECO57            glgA Z4791 ECs4274                 glgA   \n",
       " 4   B1X774  GLGA_ECODH             glgA ECDH10B_3603                 glgA   \n",
       " 5   B7M2J2  GLGA_ECO8A              glgA ECIAI1_3575                 glgA   \n",
       " 6   B7L4W1  GLGA_ECO55             glgA EC55989_3839                 glgA   \n",
       " 7   B7N176  GLGA_ECO81               glgA ECED1_4104                 glgA   \n",
       " 8   Q1R5J7  GLGA_ECOUT              glgA UTI89_C3938                 glgA   \n",
       " 9   A1AGW2  GLGA_ECOK1  glgA Ecok1_34080 APECO1_3028                 glgA   \n",
       " 10  C4ZVX9  GLGA_ECOBW                 glgA BWG_3121                 glgA   \n",
       " 11  A8A5N9  GLGA_ECOHS               glgA EcHS_A3629                 glgA   \n",
       " 12  A7ZSW2  GLGA_ECO24           glgA EcE24377A_3908                 glgA   \n",
       " 13  B5YUI5  GLGA_ECO5E            glgA ECH74115_4741                 glgA   \n",
       " 14  B7NMJ4  GLGA_ECO7I             glgA ECIAI39_3910                 glgA   \n",
       " 15  B1LI90  GLGA_ECOSM             glgA EcSMS35_3711                 glgA   \n",
       " 16  B7MDR4  GLGA_ECO45               glgA ECS88_3827                 glgA   \n",
       " 17  B7UKD4  GLGA_ECO27              glgA E2348C_3675                 glgA   \n",
       " 18  B7NE39  GLGA_ECOLU               glgA ECUMN_3893                 glgA   \n",
       " 19  Q0TC30  GLGA_ECOL5                 glgA ECP_3523                 glgA   \n",
       " 20  P0A6U9  GLGA_ECOL6                    glgA c4216                 glgA   \n",
       " 21  B1IP35  GLGA_ECOLC               glgA EcolC_0283                 glgA   \n",
       " 22  B7LSE0  GLGA_ESCF3                glgA EFER_3406                 glgA   \n",
       " 23  Q83PV4  GLGA_SHIFL             glgA SF3452 S4311                 glgA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN               b3429 JW3392              NaN   \n",
       " 1                  SSII                        NaN              NaN   \n",
       " 2                   NaN                  ECSE_3698              NaN   \n",
       " 3                   NaN              Z4791 ECs4274              NaN   \n",
       " 4                   NaN               ECDH10B_3603              NaN   \n",
       " 5                   NaN                ECIAI1_3575              NaN   \n",
       " 6                   NaN               EC55989_3839              NaN   \n",
       " 7                   NaN                 ECED1_4104              NaN   \n",
       " 8                   NaN                UTI89_C3938              NaN   \n",
       " 9                   NaN                Ecok1_34080      APECO1_3028   \n",
       " 10                  NaN                   BWG_3121              NaN   \n",
       " 11                  NaN                 EcHS_A3629              NaN   \n",
       " 12                  NaN             EcE24377A_3908              NaN   \n",
       " 13                  NaN              ECH74115_4741              NaN   \n",
       " 14                  NaN               ECIAI39_3910              NaN   \n",
       " 15                  NaN               EcSMS35_3711              NaN   \n",
       " 16                  NaN                 ECS88_3827              NaN   \n",
       " 17                  NaN                E2348C_3675              NaN   \n",
       " 18                  NaN                 ECUMN_3893              NaN   \n",
       " 19                  NaN                   ECP_3523              NaN   \n",
       " 20                  NaN                      c4216              NaN   \n",
       " 21                  NaN                 EcolC_0283              NaN   \n",
       " 22                  NaN                  EFER_3406              NaN   \n",
       " 23                  NaN               SF3452 S4311              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                          Solanum tuberosum (Potato)           4113   \n",
       " 2                      Escherichia coli (strain SE11)         409438   \n",
       " 3                            Escherichia coli O157:H7          83334   \n",
       " 4               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 5                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 6              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 7                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 8              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 9                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 10    Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 11                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 12  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 13    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 14      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 15          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 16       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 17  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 18  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 19    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 20  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 21  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 22  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 23                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 1   Granule-bound starch synthase 2, chloroplastic...  ...           NaN   \n",
       " 2   Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 3   Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 4   Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 5   Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 6   Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 7   Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 8   Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 9   Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 10  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 11  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 12  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 13  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 14  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 15  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 16  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 17  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 18  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 19  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 20  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 21  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 22  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " 23  Glycogen synthase (EC 2.4.1.21) (Starch [bacte...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                   NaN         NaN               NaN     NaN   \n",
       " 1                   NaN         NaN               NaN     NaN   \n",
       " 2                   NaN         NaN               NaN     NaN   \n",
       " 3                   NaN         NaN               NaN     NaN   \n",
       " 4                   NaN         NaN               NaN     NaN   \n",
       " 5                   NaN         NaN               NaN     NaN   \n",
       " 6                   NaN         NaN               NaN     NaN   \n",
       " 7                   NaN         NaN               NaN     NaN   \n",
       " 8                   NaN         NaN               NaN     NaN   \n",
       " 9                   NaN         NaN               NaN     NaN   \n",
       " 10                  NaN         NaN               NaN     NaN   \n",
       " 11                  NaN         NaN               NaN     NaN   \n",
       " 12                  NaN         NaN               NaN     NaN   \n",
       " 13                  NaN         NaN               NaN     NaN   \n",
       " 14                  NaN         NaN               NaN     NaN   \n",
       " 15                  NaN         NaN               NaN     NaN   \n",
       " 16                  NaN         NaN               NaN     NaN   \n",
       " 17                  NaN         NaN               NaN     NaN   \n",
       " 18                  NaN         NaN               NaN     NaN   \n",
       " 19                  NaN         NaN               NaN     NaN   \n",
       " 20                  NaN         NaN               NaN     NaN   \n",
       " 21                  NaN         NaN               NaN     NaN   \n",
       " 22                  NaN         NaN               NaN     NaN   \n",
       " 23                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " 21                              NaN         NaN             NaN   \n",
       " 22                              NaN         NaN             NaN   \n",
       " 23                              NaN         NaN             NaN   \n",
       " \n",
       "                                       Transit peptide  \\\n",
       " 0                                                 NaN   \n",
       " 1   TRANSIT 1..45; /note=\"Chloroplast\"; /evidence=...   \n",
       " 2                                                 NaN   \n",
       " 3                                                 NaN   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7                                                 NaN   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " 13                                                NaN   \n",
       " 14                                                NaN   \n",
       " 15                                                NaN   \n",
       " 16                                                NaN   \n",
       " 17                                                NaN   \n",
       " 18                                                NaN   \n",
       " 19                                                NaN   \n",
       " 20                                                NaN   \n",
       " 21                                                NaN   \n",
       " 22                                                NaN   \n",
       " 23                                                NaN   \n",
       " \n",
       "                                     PDB  \n",
       " 0   2QZS;2R4T;2R4U;3COP;3CX4;3D1J;3GUH;  \n",
       " 1                                   NaN  \n",
       " 2                                   NaN  \n",
       " 3                                   NaN  \n",
       " 4                                   NaN  \n",
       " 5                                   NaN  \n",
       " 6                                   NaN  \n",
       " 7                                   NaN  \n",
       " 8                                   NaN  \n",
       " 9                                   NaN  \n",
       " 10                                  NaN  \n",
       " 11                                  NaN  \n",
       " 12                                  NaN  \n",
       " 13                                  NaN  \n",
       " 14                                  NaN  \n",
       " 15                                  NaN  \n",
       " 16                                  NaN  \n",
       " 17                                  NaN  \n",
       " 18                                  NaN  \n",
       " 19                                  NaN  \n",
       " 20                                  NaN  \n",
       " 21                                  NaN  \n",
       " 22                                  NaN  \n",
       " 23                                  NaN  \n",
       " \n",
       " [24 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  Q9LD43  ACCA_ARATH  CAC3 At2g38040 T8P21.5                 CAC3   \n",
       " 1  P56765  ACCD_ARATH          accD AtCg00500                 accD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                  At2g38040          T8P21.5   \n",
       " 1                   NaN                  AtCg00500              NaN   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Acetyl-coenzyme A carboxylase carboxyl transfe...  ...           NaN   \n",
       " 1  Acetyl-coenzyme A carboxylase carboxyl transfe...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 741; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide Signal peptide  \\\n",
       " 0                              NaN         NaN            NaN   \n",
       " 1                              NaN         NaN            NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..54; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 1                                                NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P31570  BTUR_SALTY  btuR cobA STM1718                 btuR   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 cobA                    STM1718               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Corrinoid adenosyltransferase CobA (EC 2.5.1.1...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                   PDB  \n",
       " 0             NaN  1G5R;1G5T;1G64;4HUT;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P06621  CBPG_PSES6       cpg2                 cpg2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas sp. (strain RS-16)            312   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Carboxypeptidase G2 (CPDG2) (EC 3.4.17.11) (Fo...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN    SIGNAL 1..22   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  1CG2;6XJ5;7M6U;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O18964  SYNJ1_BOVIN      SYNJ1                SYNJ1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Synaptojanin-1 (EC 3.1.3.36) (Synaptic inosito...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 820; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                         Gene Names Gene Names (primary)  \\\n",
       " 0  P41911  GPD2_YEAST            GPD2 GPD3 YOL059W O1222                 GPD2   \n",
       " 1  Q00055  GPD1_YEAST  GPD1 DAR1 HOR1 OSG1 YDL022W D2830                 GPD1   \n",
       " 2  Q759G5   GPD_ASHGO                        GPD ADR311C                  GPD   \n",
       " 3  Q4PNS1   GPD_CANTR                                GPD                  GPD   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 GPD3                    YOL059W            O1222   \n",
       " 1       DAR1 HOR1 OSG1                    YDL022W            D2830   \n",
       " 2                  NaN                    ADR311C              NaN   \n",
       " 3                  NaN                        NaN              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2  Ashbya gossypii (strain ATCC 10895 / CBS 109.5...         284811   \n",
       " 3                         Candida tropicalis (Yeast)           5482   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glycerol-3-phosphate dehydrogenase [NAD(+)] 2,...  ...           NaN   \n",
       " 1  Glycerol-3-phosphate dehydrogenase [NAD(+)] 1 ...  ...           NaN   \n",
       " 2  Glycerol-3-phosphate dehydrogenase [NAD(+)] (E...  ...           NaN   \n",
       " 3  Glycerol-3-phosphate dehydrogenase [NAD(+)] (E...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 70; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide    PDB  \n",
       " 0  TRANSIT 1..16; /note=\"Mitochondrion\"; /evidenc...    NaN  \n",
       " 1                                                NaN  4FGW;  \n",
       " 2                                                NaN    NaN  \n",
       " 3                                                NaN    NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name                              Gene Names  \\\n",
       " 0  O88813  ACSL5_RAT                        Acsl5 Acs5 Facl5   \n",
       " 1  P97524  S27A2_RAT  Slc27a2 Acsvl1 Facvl1 Fatp2 Vlacs Vlcs   \n",
       " 2  P97849  S27A1_RAT                      Slc27a1 Fatp Fatp1   \n",
       " 3  Q63151  ACSL3_RAT                        Acsl3 Acs3 Facl3   \n",
       " 4  A1L1K7  ACBG2_RAT                                  Acsbg2   \n",
       " 5  O35547  ACSL4_RAT                        Acsl4 Acs4 Facl4   \n",
       " 6  P18163  ACSL1_RAT                  Acsl1 Acs1 Acsl2 Facl2   \n",
       " 7  P33124  ACSL6_RAT                        Acsl6 Acs2 Facl6   \n",
       " 8  Q924N5  ACBG1_RAT                           Acsbg1 Grlacs   \n",
       " 9  Q9ES38  S27A5_RAT               Slc27a5 Acsb Fatp5 Vlacsr   \n",
       " \n",
       "   Gene Names (primary)            Gene Names (synonym)  \\\n",
       " 0                Acsl5                      Acs5 Facl5   \n",
       " 1              Slc27a2  Acsvl1 Facvl1 Fatp2 Vlacs Vlcs   \n",
       " 2              Slc27a1                      Fatp Fatp1   \n",
       " 3                Acsl3                      Acs3 Facl3   \n",
       " 4               Acsbg2                             NaN   \n",
       " 5                Acsl4                      Acs4 Facl4   \n",
       " 6                Acsl1                Acs1 Acsl2 Facl2   \n",
       " 7                Acsl6                      Acs2 Facl6   \n",
       " 8               Acsbg1                          Grlacs   \n",
       " 9              Slc27a5               Acsb Fatp5 Vlacsr   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 2                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 3                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 4                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 5                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 6                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 7                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 8                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 9                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Long-chain-fatty-acid--CoA ligase 5 (EC 6.2.1....  ...   \n",
       " 1          10116  Long-chain fatty acid transport protein 2 (Ara...  ...   \n",
       " 2          10116  Long-chain fatty acid transport protein 1 (Ara...  ...   \n",
       " 3          10116  Fatty acid CoA ligase Acsl3 (Arachidonate--CoA...  ...   \n",
       " 4          10116  Long-chain-fatty-acid--CoA ligase ACSBG2 (EC 6...  ...   \n",
       " 5          10116  Long-chain-fatty-acid--CoA ligase 4 (EC 6.2.1....  ...   \n",
       " 6          10116  Long-chain-fatty-acid--CoA ligase 1 (EC 6.2.1....  ...   \n",
       " 7          10116  Long-chain-fatty-acid--CoA ligase 6 (EC 6.2.1....  ...   \n",
       " 8          10116  Long-chain-fatty-acid--CoA ligase ACSBG1 (EC 6...  ...   \n",
       " 9          10116  Long-chain fatty acid transport protein 5 (FAT...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0                                                NaN                  NaN   \n",
       " 1                                                NaN                  NaN   \n",
       " 2                                                NaN                  NaN   \n",
       " 3                                                NaN                  NaN   \n",
       " 4                                                NaN                  NaN   \n",
       " 5                                                NaN                  NaN   \n",
       " 6  CARBOHYD 136; /note=\"O-linked (GlcNAc) serine\"...                  NaN   \n",
       " 7                                                NaN                  NaN   \n",
       " 8                                                NaN                  NaN   \n",
       " 9                                                NaN                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 361; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 1         NaN  MOD_RES 291; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " 3         NaN  MOD_RES 683; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 4         NaN                                                NaN     NaN   \n",
       " 5         NaN  MOD_RES 447; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 6         NaN  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 7         NaN                                                NaN     NaN   \n",
       " 8         NaN  MOD_RES 34; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 9         NaN                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " 7                              NaN         NaN             NaN   \n",
       " 8                              NaN         NaN             NaN   \n",
       " 9                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " 5             NaN  NaN  \n",
       " 6             NaN  NaN  \n",
       " 7             NaN  NaN  \n",
       " 8             NaN  NaN  \n",
       " 9             NaN  NaN  \n",
       " \n",
       " [10 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  Q8I4S1  KTHY_PLAF7  TMK TMPK PF3D7_1251300                  TMK   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 TMPK                         NaN    PF3D7_1251300   \n",
       " \n",
       "                               Organism  Organism (ID)  \\\n",
       " 0  Plasmodium falciparum (isolate 3D7)          36329   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Thymidylate kinase (PfTMK) (PfTMPK) (EC 2.7.4....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                  PDB  \n",
       " 0             NaN  2WWF;2WWG;2WWH;2WWI;2YOF;2YOG;2YOH;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   P0A8Y1  YJJG_ECOLI             yjjG b4374 JW4336                 yjjG   \n",
       " 1   P0AF24  NAGD_ECOLI             nagD b0675 JW0661                 nagD   \n",
       " 2   P64636  YRFG_ECOLI             yrfG b3399 JW5865                 yrfG   \n",
       " 3   P0A840  SURE_ECOLI        surE ygbC b2744 JW2714                 surE   \n",
       " 4   P07024  USHA_ECOLI             ushA b0480 JW0469                 ushA   \n",
       " 5   O46411  5NTC_BOVIN                         NT5C2                NT5C2   \n",
       " 6   P49902  5NTC_HUMAN         NT5C2 NT5B NT5CP PNT5                NT5C2   \n",
       " 7   P52307  5NTD_RHIMP                           NaN                  NaN   \n",
       " 8   B6I6D4  SURE_ECOSE                surE ECSE_2996                 surE   \n",
       " 9   P0A842  SURE_ECO57            surE Z4052 ECs3598                 surE   \n",
       " 10  P0A8Y2  YJJG_ECO57            yjjG Z5975 ECs5332                 yjjG   \n",
       " 11  P0AF25  NAGD_ECO57            nagD Z0822 ECs0705                 nagD   \n",
       " 12  B1XCS0  SURE_ECODH             surE ECDH10B_2912                 surE   \n",
       " 13  B7MZ45  SURE_ECO81               surE ECED1_3200                 surE   \n",
       " 14  Q1R7U7  SURE_ECOUT              surE UTI89_C3115                 surE   \n",
       " 15  A1AET8  SURE_ECOK1  surE Ecok1_26840 APECO1_3779                 surE   \n",
       " 16  B7LEG2  SURE_ECO55             surE EC55989_3016                 surE   \n",
       " 17  B7LXF6  SURE_ECO8A              surE ECIAI1_2845                 surE   \n",
       " 18  C4ZZP8  SURE_ECOBW                 surE BWG_2480                 surE   \n",
       " 19  A8A3M3  SURE_ECOHS               surE EcHS_A2882                 surE   \n",
       " 20  A7ZQI8  SURE_ECO24           surE EcE24377A_3045                 surE   \n",
       " 21  B5Z3A6  SURE_ECO5E            surE ECH74115_3995                 surE   \n",
       " 22  B7MKL8  SURE_ECO45               surE ECS88_3014                 surE   \n",
       " 23  B7NT88  SURE_ECO7I             surE ECIAI39_2933                 surE   \n",
       " 24  B7UHG3  SURE_ECO27              surE E2348C_3014                 surE   \n",
       " 25  Q0TEB4  SURE_ECOL5                 surE ECP_2726                 surE   \n",
       " 26  B7N6X4  SURE_ECOLU               surE ECUMN_3068                 surE   \n",
       " 27  P0A841  SURE_ECOL6                    surE c3311                 surE   \n",
       " 28  B1IUT5  SURE_ECOLC               surE EcolC_0968                 surE   \n",
       " 29  B7LWJ1  SURE_ESCF3                surE EFER_0324                 surE   \n",
       " 30  Q83JY2  SURE_SHIFL             surE SF2767 S2960                 surE   \n",
       " 31  P64637  YRFG_SHIFL             yrfG SF3417 S4345                 yrfG   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN               b4374 JW4336              NaN   \n",
       " 1                   NaN               b0675 JW0661              NaN   \n",
       " 2                   NaN               b3399 JW5865              NaN   \n",
       " 3                  ygbC               b2744 JW2714              NaN   \n",
       " 4                   NaN               b0480 JW0469              NaN   \n",
       " 5                   NaN                        NaN              NaN   \n",
       " 6       NT5B NT5CP PNT5                        NaN              NaN   \n",
       " 7                   NaN                        NaN              NaN   \n",
       " 8                   NaN                  ECSE_2996              NaN   \n",
       " 9                   NaN              Z4052 ECs3598              NaN   \n",
       " 10                  NaN              Z5975 ECs5332              NaN   \n",
       " 11                  NaN              Z0822 ECs0705              NaN   \n",
       " 12                  NaN               ECDH10B_2912              NaN   \n",
       " 13                  NaN                 ECED1_3200              NaN   \n",
       " 14                  NaN                UTI89_C3115              NaN   \n",
       " 15                  NaN                Ecok1_26840      APECO1_3779   \n",
       " 16                  NaN               EC55989_3016              NaN   \n",
       " 17                  NaN                ECIAI1_2845              NaN   \n",
       " 18                  NaN                   BWG_2480              NaN   \n",
       " 19                  NaN                 EcHS_A2882              NaN   \n",
       " 20                  NaN             EcE24377A_3045              NaN   \n",
       " 21                  NaN              ECH74115_3995              NaN   \n",
       " 22                  NaN                 ECS88_3014              NaN   \n",
       " 23                  NaN               ECIAI39_2933              NaN   \n",
       " 24                  NaN                E2348C_3014              NaN   \n",
       " 25                  NaN                   ECP_2726              NaN   \n",
       " 26                  NaN                 ECUMN_3068              NaN   \n",
       " 27                  NaN                      c3311              NaN   \n",
       " 28                  NaN                 EcolC_0968              NaN   \n",
       " 29                  NaN                  EFER_0324              NaN   \n",
       " 30                  NaN               SF2767 S2960              NaN   \n",
       " 31                  NaN               SF3417 S4345              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                       Escherichia coli (strain K12)          83333   \n",
       " 2                       Escherichia coli (strain K12)          83333   \n",
       " 3                       Escherichia coli (strain K12)          83333   \n",
       " 4                       Escherichia coli (strain K12)          83333   \n",
       " 5                                 Bos taurus (Bovine)           9913   \n",
       " 6                                Homo sapiens (Human)           9606   \n",
       " 7   Rhipicephalus microplus (Cattle tick) (Boophil...           6941   \n",
       " 8                      Escherichia coli (strain SE11)         409438   \n",
       " 9                            Escherichia coli O157:H7          83334   \n",
       " 10                           Escherichia coli O157:H7          83334   \n",
       " 11                           Escherichia coli O157:H7          83334   \n",
       " 12              Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 13                 Escherichia coli O81 (strain ED1a)         585397   \n",
       " 14             Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 15                      Escherichia coli O1:K1 / APEC         405955   \n",
       " 16             Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 17                  Escherichia coli O8 (strain IAI1)         585034   \n",
       " 18    Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 19                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 20  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 21    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 22       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 23      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 24  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 25    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 26  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 27  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 28  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 29  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 30                                  Shigella flexneri            623   \n",
       " 31                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ...  \\\n",
       " 0   Pyrimidine 5'-nucleotidase YjjG (EC 3.1.3.5) (...  ...   \n",
       " 1    Ribonucleotide monophosphatase NagD (EC 3.1.3.5)  ...   \n",
       " 2              GMP/IMP nucleotidase YrfG (EC 3.1.3.5)  ...   \n",
       " 3   5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 4   Protein UshA [Includes: UDP-sugar hydrolase (E...  ...   \n",
       " 5   Cytosolic purine 5'-nucleotidase (EC 3.1.3.5) ...  ...   \n",
       " 6   Cytosolic purine 5'-nucleotidase (EC 3.1.3.5) ...  ...   \n",
       " 7   Protein 5NUC [Includes: UDP-sugar hydrolase (E...  ...   \n",
       " 8   5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 9   5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 10  Pyrimidine 5'-nucleotidase YjjG (EC 3.1.3.5) (...  ...   \n",
       " 11   Ribonucleotide monophosphatase NagD (EC 3.1.3.5)  ...   \n",
       " 12  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 13  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 14  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 15  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 16  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 17  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 18  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 19  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 20  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 21  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 22  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 23  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 24  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 25  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 26  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 27  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 28  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 29  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 30  5'/3'-nucleotidase SurE (EC 3.1.3.5) (EC 3.1.3...  ...   \n",
       " 31             GMP/IMP nucleotidase YrfG (EC 3.1.3.5)  ...   \n",
       " \n",
       "                                         Glycosylation Initiator methionine  \\\n",
       " 0                                                 NaN                  NaN   \n",
       " 1                                                 NaN                  NaN   \n",
       " 2                                                 NaN                  NaN   \n",
       " 3                                                 NaN                  NaN   \n",
       " 4                                                 NaN                  NaN   \n",
       " 5                                                 NaN                  NaN   \n",
       " 6                                                 NaN                  NaN   \n",
       " 7   CARBOHYD 172; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 8                                                 NaN                  NaN   \n",
       " 9                                                 NaN                  NaN   \n",
       " 10                                                NaN                  NaN   \n",
       " 11                                                NaN                  NaN   \n",
       " 12                                                NaN                  NaN   \n",
       " 13                                                NaN                  NaN   \n",
       " 14                                                NaN                  NaN   \n",
       " 15                                                NaN                  NaN   \n",
       " 16                                                NaN                  NaN   \n",
       " 17                                                NaN                  NaN   \n",
       " 18                                                NaN                  NaN   \n",
       " 19                                                NaN                  NaN   \n",
       " 20                                                NaN                  NaN   \n",
       " 21                                                NaN                  NaN   \n",
       " 22                                                NaN                  NaN   \n",
       " 23                                                NaN                  NaN   \n",
       " 24                                                NaN                  NaN   \n",
       " 25                                                NaN                  NaN   \n",
       " 26                                                NaN                  NaN   \n",
       " 27                                                NaN                  NaN   \n",
       " 28                                                NaN                  NaN   \n",
       " 29                                                NaN                  NaN   \n",
       " 30                                                NaN                  NaN   \n",
       " 31                                                NaN                  NaN   \n",
       " \n",
       "                                            Lipidation  \\\n",
       " 0                                                 NaN   \n",
       " 1                                                 NaN   \n",
       " 2                                                 NaN   \n",
       " 3                                                 NaN   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7   LIPID 555; /note=\"GPI-anchor amidated asparagi...   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " 13                                                NaN   \n",
       " 14                                                NaN   \n",
       " 15                                                NaN   \n",
       " 16                                                NaN   \n",
       " 17                                                NaN   \n",
       " 18                                                NaN   \n",
       " 19                                                NaN   \n",
       " 20                                                NaN   \n",
       " 21                                                NaN   \n",
       " 22                                                NaN   \n",
       " 23                                                NaN   \n",
       " 24                                                NaN   \n",
       " 25                                                NaN   \n",
       " 26                                                NaN   \n",
       " 27                                                NaN   \n",
       " 28                                                NaN   \n",
       " 29                                                NaN   \n",
       " 30                                                NaN   \n",
       " 31                                                NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0                                                 NaN     NaN   \n",
       " 1                                                 NaN     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3                                                 NaN     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " 5   MOD_RES 418; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 6   MOD_RES 418; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 7                                                 NaN     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9                                                 NaN     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11                                                NaN     NaN   \n",
       " 12                                                NaN     NaN   \n",
       " 13                                                NaN     NaN   \n",
       " 14                                                NaN     NaN   \n",
       " 15                                                NaN     NaN   \n",
       " 16                                                NaN     NaN   \n",
       " 17                                                NaN     NaN   \n",
       " 18                                                NaN     NaN   \n",
       " 19                                                NaN     NaN   \n",
       " 20                                                NaN     NaN   \n",
       " 21                                                NaN     NaN   \n",
       " 22                                                NaN     NaN   \n",
       " 23                                                NaN     NaN   \n",
       " 24                                                NaN     NaN   \n",
       " 25                                                NaN     NaN   \n",
       " 26                                                NaN     NaN   \n",
       " 27                                                NaN     NaN   \n",
       " 28                                                NaN     NaN   \n",
       " 29                                                NaN     NaN   \n",
       " 30                                                NaN     NaN   \n",
       " 31                                                NaN     NaN   \n",
       " \n",
       "     Post-translational modification  \\\n",
       " 0                               NaN   \n",
       " 1                               NaN   \n",
       " 2                               NaN   \n",
       " 3                               NaN   \n",
       " 4                               NaN   \n",
       " 5                               NaN   \n",
       " 6                               NaN   \n",
       " 7                PTM: Glycosylated.   \n",
       " 8                               NaN   \n",
       " 9                               NaN   \n",
       " 10                              NaN   \n",
       " 11                              NaN   \n",
       " 12                              NaN   \n",
       " 13                              NaN   \n",
       " 14                              NaN   \n",
       " 15                              NaN   \n",
       " 16                              NaN   \n",
       " 17                              NaN   \n",
       " 18                              NaN   \n",
       " 19                              NaN   \n",
       " 20                              NaN   \n",
       " 21                              NaN   \n",
       " 22                              NaN   \n",
       " 23                              NaN   \n",
       " 24                              NaN   \n",
       " 25                              NaN   \n",
       " 26                              NaN   \n",
       " 27                              NaN   \n",
       " 28                              NaN   \n",
       " 29                              NaN   \n",
       " 30                              NaN   \n",
       " 31                              NaN   \n",
       " \n",
       "                                            Propeptide  \\\n",
       " 0                                                 NaN   \n",
       " 1                                                 NaN   \n",
       " 2                                                 NaN   \n",
       " 3                                                 NaN   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7   PROPEP 556..580; /note=\"Removed in mature form...   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " 13                                                NaN   \n",
       " 14                                                NaN   \n",
       " 15                                                NaN   \n",
       " 16                                                NaN   \n",
       " 17                                                NaN   \n",
       " 18                                                NaN   \n",
       " 19                                                NaN   \n",
       " 20                                                NaN   \n",
       " 21                                                NaN   \n",
       " 22                                                NaN   \n",
       " 23                                                NaN   \n",
       " 24                                                NaN   \n",
       " 25                                                NaN   \n",
       " 26                                                NaN   \n",
       " 27                                                NaN   \n",
       " 28                                                NaN   \n",
       " 29                                                NaN   \n",
       " 30                                                NaN   \n",
       " 31                                                NaN   \n",
       " \n",
       "                                        Signal peptide Transit peptide  \\\n",
       " 0                                                 NaN             NaN   \n",
       " 1                                                 NaN             NaN   \n",
       " 2                                                 NaN             NaN   \n",
       " 3                                                 NaN             NaN   \n",
       " 4   SIGNAL 1..25; /evidence=\"ECO:0000269|PubMed:30...             NaN   \n",
       " 5                                                 NaN             NaN   \n",
       " 6                                                 NaN             NaN   \n",
       " 7   SIGNAL <1..14; /evidence=\"ECO:0000269|PubMed:8...             NaN   \n",
       " 8                                                 NaN             NaN   \n",
       " 9                                                 NaN             NaN   \n",
       " 10                                                NaN             NaN   \n",
       " 11                                                NaN             NaN   \n",
       " 12                                                NaN             NaN   \n",
       " 13                                                NaN             NaN   \n",
       " 14                                                NaN             NaN   \n",
       " 15                                                NaN             NaN   \n",
       " 16                                                NaN             NaN   \n",
       " 17                                                NaN             NaN   \n",
       " 18                                                NaN             NaN   \n",
       " 19                                                NaN             NaN   \n",
       " 20                                                NaN             NaN   \n",
       " 21                                                NaN             NaN   \n",
       " 22                                                NaN             NaN   \n",
       " 23                                                NaN             NaN   \n",
       " 24                                                NaN             NaN   \n",
       " 25                                                NaN             NaN   \n",
       " 26                                                NaN             NaN   \n",
       " 27                                                NaN             NaN   \n",
       " 28                                                NaN             NaN   \n",
       " 29                                                NaN             NaN   \n",
       " 30                                                NaN             NaN   \n",
       " 31                                                NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0                                                 NaN  \n",
       " 1                                               2C4N;  \n",
       " 2                                                 NaN  \n",
       " 3                                                 NaN  \n",
       " 4       1HO5;1HP1;1HPU;1OI8;1OID;1OIE;1USH;2USH;4WWL;  \n",
       " 5                                                 NaN  \n",
       " 6   2J2C;2JC9;2JCM;2XCV;2XCW;2XCX;2XJB;2XJC;2XJD;2...  \n",
       " 7                                                 NaN  \n",
       " 8                                                 NaN  \n",
       " 9                                                 NaN  \n",
       " 10                                                NaN  \n",
       " 11                                                NaN  \n",
       " 12                                                NaN  \n",
       " 13                                                NaN  \n",
       " 14                                                NaN  \n",
       " 15                                                NaN  \n",
       " 16                                                NaN  \n",
       " 17                                                NaN  \n",
       " 18                                                NaN  \n",
       " 19                                                NaN  \n",
       " 20                                                NaN  \n",
       " 21                                                NaN  \n",
       " 22                                                NaN  \n",
       " 23                                                NaN  \n",
       " 24                                                NaN  \n",
       " 25                                                NaN  \n",
       " 26                                                NaN  \n",
       " 27                                                NaN  \n",
       " 28                                                NaN  \n",
       " 29                                                NaN  \n",
       " 30                                                NaN  \n",
       " 31                                                NaN  \n",
       " \n",
       " [32 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  Q9XGZ0  MAOP3_ARATH   NADP-ME3 At5g25880 T1N24.25             NADP-ME3   \n",
       " 1  O82191  MAOP1_ARATH    NADP-ME1 At2g19900 F6F22.7             NADP-ME1   \n",
       " 2  Q9CA83  MAOP4_ARATH  NADP-ME4 At1g79750 F19K16.27             NADP-ME4   \n",
       " 3  Q9LYG3  MAOP2_ARATH  NADP-ME2 At5g11670 T22P22.60             NADP-ME2   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                  At5g25880         T1N24.25   \n",
       " 1                   NaN                  At2g19900          F6F22.7   \n",
       " 2                   NaN                  At1g79750        F19K16.27   \n",
       " 3                   NaN                  At5g11670        T22P22.60   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 3  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  NADP-dependent malic enzyme 3 (AtNADP-ME3) (NA...  ...           NaN   \n",
       " 1  NADP-dependent malic enzyme 1 (AtNADP-ME1) (NA...  ...           NaN   \n",
       " 2  NADP-dependent malic enzyme 4, chloroplastic (...  ...           NaN   \n",
       " 3  NADP-dependent malic enzyme 2 (AtNADP-ME2) (NA...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylglycine\"; /evidence=...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3  MOD_RES 2; /note=\"N-acetylglycine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide PDB  \n",
       " 0                                                NaN NaN  \n",
       " 1                                                NaN NaN  \n",
       " 2  TRANSIT 1..74; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " 3                                                NaN NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry   Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  Q7DBF3  GDPPS_ECO57  perA per rfbE Z3200 ECs2841                 perA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0             per rfbE              Z3200 ECs2841               NaN   \n",
       " \n",
       "                    Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli O157:H7          83334   \n",
       " \n",
       "                             Protein names  ... Glycosylation  \\\n",
       " 0  GDP-perosamine synthase (EC 2.6.1.102)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 183; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry   Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   P33316    DUT_HUMAN                           DUT                  DUT   \n",
       " 1   P03195    DUT_EBVB9                     DUT BLLF3                  DUT   \n",
       " 2   P89079  E4OR1_ADE09                            E4                   E4   \n",
       " 3   Q76RE7    DUT_VACCA  DUT MVA030L ACAM3000_MVA_030                  DUT   \n",
       " 4   P68635    DUT_VACCP                       DUT F16                  DUT   \n",
       " 5   P16824    DUT_HCMVA                      DUT UL72                  DUT   \n",
       " 6   P68634    DUT_VACCC                       DUT F2L                  DUT   \n",
       " 7   P14597    DUT_ORFN2                           DUT                  DUT   \n",
       " 8   P09254     DUT_VZVD                      DUT ORF8                  DUT   \n",
       " 9   P0DSZ7    DUT_VAR67               DUT C6L E2L F2L                  DUT   \n",
       " 10  P52341    DUT_HHV7J                       DUT U45                  DUT   \n",
       " 11  Q06095    DUT_HHV6U                       DUT U45                  DUT   \n",
       " 12  Q2HR78    DUT_HHV8P                     DUT ORF54                  DUT   \n",
       " 13  P0DSZ8     DUT_VARV               DUT C6L E2L F2L                  DUT   \n",
       " 14  P89469    DUT_HHV2H                      DUT UL50                  DUT   \n",
       " 15  P10234    DUT_HHV11                      DUT UL50                  DUT   \n",
       " 16  P52541    DUT_HHV6Z                       DUT U45                  DUT   \n",
       " 17  Q3KST7     DUT_EBVG                     DUT BLLF3                  DUT   \n",
       " 18  Q4JQW7     DUT_VZVO                      DUT ORF8                  DUT   \n",
       " 19  P30007    DUT_HHV6G                       DUT U45                  DUT   \n",
       " 20  Q6SW70    DUT_HCMVM                      DUT UL72                  DUT   \n",
       " 21  Q07275    DUT_EBVA8                     DUT BLLF3                  DUT   \n",
       " 22  Q6TUZ4    DUT_YMTV5                       DUT 17L                  DUT   \n",
       " 23  P87630    DUT_CWPXG                       DUT G2L                  DUT   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN                        NaN              NaN   \n",
       " 1                    NaN                        NaN            BLLF3   \n",
       " 2                    NaN                        NaN              NaN   \n",
       " 3                    NaN   MVA030L ACAM3000_MVA_030              NaN   \n",
       " 4                    NaN                        NaN              F16   \n",
       " 5                    NaN                       UL72              NaN   \n",
       " 6                    NaN                        NaN              F2L   \n",
       " 7                    NaN                        NaN              NaN   \n",
       " 8                    NaN                       ORF8              NaN   \n",
       " 9                    NaN                        NaN      C6L E2L F2L   \n",
       " 10                   NaN                        U45              NaN   \n",
       " 11                   NaN                        U45              NaN   \n",
       " 12                   NaN                      ORF54              NaN   \n",
       " 13                   NaN                        NaN      C6L E2L F2L   \n",
       " 14                   NaN                       UL50              NaN   \n",
       " 15                   NaN                       UL50              NaN   \n",
       " 16                   NaN                        U45              NaN   \n",
       " 17                   NaN                        NaN            BLLF3   \n",
       " 18                   NaN                       ORF8              NaN   \n",
       " 19                   NaN                        U45              NaN   \n",
       " 20                   NaN                       UL72              NaN   \n",
       " 21                   NaN                        NaN            BLLF3   \n",
       " 22                   NaN                        17L              NaN   \n",
       " 23                   NaN                        NaN              G2L   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                Homo sapiens (Human)           9606   \n",
       " 1   Epstein-Barr virus (strain B95-8) (HHV-4) (Hum...          10377   \n",
       " 2   Human adenovirus D serotype 9 (HAdV-9) (Human ...          10527   \n",
       " 3               Vaccinia virus (strain Ankara) (VACV)         126794   \n",
       " 4                Vaccinia virus (strain L-IVP) (VACV)          31531   \n",
       " 5   Human cytomegalovirus (strain AD169) (HHV-5) (...          10360   \n",
       " 6           Vaccinia virus (strain Copenhagen) (VACV)          10249   \n",
       " 7                    Orf virus (strain NZ2) (OV NZ-2)          10259   \n",
       " 8   Varicella-zoster virus (strain Dumas) (HHV-3) ...          10338   \n",
       " 9   Variola virus (isolate Human/India/Ind3/1967) ...         587200   \n",
       " 10  Human herpesvirus 7 (strain JI) (HHV-7) (Human...          57278   \n",
       " 11  Human herpesvirus 6A (strain Uganda-1102) (HHV...          10370   \n",
       " 12  Human herpesvirus 8 type P (isolate GK18) (HHV...         868565   \n",
       " 13                                      Variola virus          10255   \n",
       " 14  Human herpesvirus 2 (strain HG52) (HHV-2) (Hum...          10315   \n",
       " 15  Human herpesvirus 1 (strain 17) (HHV-1) (Human...          10299   \n",
       " 16  Human herpesvirus 6B (strain Z29) (HHV-6 varia...          36351   \n",
       " 17  Epstein-Barr virus (strain GD1) (HHV-4) (Human...          10376   \n",
       " 18  Varicella-zoster virus (strain Oka vaccine) (H...         341980   \n",
       " 19  Human herpesvirus 6A (strain GS) (HHV-6 varian...          10369   \n",
       " 20  Human cytomegalovirus (strain Merlin) (HHV-5) ...         295027   \n",
       " 21  Epstein-Barr virus (strain AG876) (HHV-4) (Hum...          82830   \n",
       " 22      Yaba monkey tumor virus (strain VR587) (YMTV)         928314   \n",
       " 23       Cowpox virus (strain GRI-90 / Grishak) (CPV)         265871   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 1   Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 2   E4-ORF1 (EC 3.6.1.23) (Early E4 14.0 kDa prote...  ...           NaN   \n",
       " 3   Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 4   Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 5   Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 6   Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 7   Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 8   Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 9   Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 10  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 11  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 12  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 13  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 14  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 15  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 16  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 17  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 18  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 19  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 20  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 21  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 22  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " 23  Deoxyuridine 5'-triphosphate nucleotidohydrola...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine Lipidation  \\\n",
       " 0   INIT_MET P33316-2:1; /note=\"Removed\"; /evidenc...        NaN   \n",
       " 1                                                 NaN        NaN   \n",
       " 2                                                 NaN        NaN   \n",
       " 3                                                 NaN        NaN   \n",
       " 4                                                 NaN        NaN   \n",
       " 5                                                 NaN        NaN   \n",
       " 6                                                 NaN        NaN   \n",
       " 7                                                 NaN        NaN   \n",
       " 8                                                 NaN        NaN   \n",
       " 9                                                 NaN        NaN   \n",
       " 10                                                NaN        NaN   \n",
       " 11                                                NaN        NaN   \n",
       " 12                                                NaN        NaN   \n",
       " 13                                                NaN        NaN   \n",
       " 14                                                NaN        NaN   \n",
       " 15                                                NaN        NaN   \n",
       " 16                                                NaN        NaN   \n",
       " 17                                                NaN        NaN   \n",
       " 18                                                NaN        NaN   \n",
       " 19                                                NaN        NaN   \n",
       " 20                                                NaN        NaN   \n",
       " 21                                                NaN        NaN   \n",
       " 22                                                NaN        NaN   \n",
       " 23                                                NaN        NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 88; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1                                                 NaN     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3                                                 NaN     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6                                                 NaN     NaN   \n",
       " 7                                                 NaN     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9                                                 NaN     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11                                                NaN     NaN   \n",
       " 12                                                NaN     NaN   \n",
       " 13                                                NaN     NaN   \n",
       " 14                                                NaN     NaN   \n",
       " 15                                                NaN     NaN   \n",
       " 16                                                NaN     NaN   \n",
       " 17                                                NaN     NaN   \n",
       " 18                                                NaN     NaN   \n",
       " 19                                                NaN     NaN   \n",
       " 20                                                NaN     NaN   \n",
       " 21                                                NaN     NaN   \n",
       " 22                                                NaN     NaN   \n",
       " 23                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0   PTM: Nuclear isoform 2 is phosphorylated in vi...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19                                                NaN         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22                                                NaN         NaN   \n",
       " 23                                                NaN         NaN   \n",
       " \n",
       "     Signal peptide                                    Transit peptide  \\\n",
       " 0              NaN  TRANSIT 1..69; /note=\"Mitochondrion\"; /evidenc...   \n",
       " 1              NaN                                                NaN   \n",
       " 2              NaN                                                NaN   \n",
       " 3              NaN                                                NaN   \n",
       " 4              NaN                                                NaN   \n",
       " 5              NaN                                                NaN   \n",
       " 6              NaN                                                NaN   \n",
       " 7              NaN                                                NaN   \n",
       " 8              NaN                                                NaN   \n",
       " 9              NaN                                                NaN   \n",
       " 10             NaN                                                NaN   \n",
       " 11             NaN                                                NaN   \n",
       " 12             NaN                                                NaN   \n",
       " 13             NaN                                                NaN   \n",
       " 14             NaN                                                NaN   \n",
       " 15             NaN                                                NaN   \n",
       " 16             NaN                                                NaN   \n",
       " 17             NaN                                                NaN   \n",
       " 18             NaN                                                NaN   \n",
       " 19             NaN                                                NaN   \n",
       " 20             NaN                                                NaN   \n",
       " 21             NaN                                                NaN   \n",
       " 22             NaN                                                NaN   \n",
       " 23             NaN                                                NaN   \n",
       " \n",
       "                                               PDB  \n",
       " 0   1Q5H;1Q5U;2HQU;3ARA;3ARN;3EHW;4MZ5;4MZ6;5H4J;  \n",
       " 1                  2BSY;2BT1;2WE0;2WE1;2WE2;2WE3;  \n",
       " 2                                             NaN  \n",
       " 3                                             NaN  \n",
       " 4                                             NaN  \n",
       " 5                                             NaN  \n",
       " 6                                             NaN  \n",
       " 7                                             NaN  \n",
       " 8                                             NaN  \n",
       " 9                                             NaN  \n",
       " 10                                            NaN  \n",
       " 11                                            NaN  \n",
       " 12                                            NaN  \n",
       " 13                                            NaN  \n",
       " 14                                            NaN  \n",
       " 15                                            NaN  \n",
       " 16                                            NaN  \n",
       " 17                                            NaN  \n",
       " 18                                            NaN  \n",
       " 19                                            NaN  \n",
       " 20                                            NaN  \n",
       " 21                                            NaN  \n",
       " 22                                            NaN  \n",
       " 23                                            NaN  \n",
       " \n",
       " [24 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names  Gene Names (primary)  \\\n",
       " 0  Q57U83  PYRD_TRYB2  Tb927.5.3830                   NaN   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN     Tb927.5.3830   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Trypanosoma brucei brucei (strain 927/4 GUTat1...         185431   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Dihydroorotate dehydrogenase (fumarate) (DHOD)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  2B4G;5XFV;5XFW;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9I1L4  PVCB_PSEAE  pvcB PA2255                 pvcB                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     PA2255               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Tyrosine isonitrile desaturase (EC 1.14.20.9) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  3EAT;4YLM;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q92820  GGH_HUMAN        GGH                  GGH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Gamma-glutamyl hydrolase (EC 3.4.19.9) (Conjug...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 116; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide                                     Signal peptide  \\\n",
       " 0         NaN  SIGNAL 1..24; /evidence=\"ECO:0000250|UniProtKB...   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1L9X;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  Q838C9  LDH2_ENTFA  ldh2 ldh-2 EF_0641                 ldh2   \n",
       " 1  Q839C1  LDH1_ENTFA  ldh1 ldh-1 EF_0255                 ldh1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                ldh-2                    EF_0641               NaN   \n",
       " 1                ldh-1                    EF_0255               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Enterococcus faecalis (strain ATCC 700802 / V583)         226185   \n",
       " 1  Enterococcus faecalis (strain ATCC 700802 / V583)         226185   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  L-lactate dehydrogenase 2 (L-LDH 2) (EC 1.1.1.27)  ...           NaN   \n",
       " 1  L-lactate dehydrogenase 1 (L-LDH 1) (EC 1.1.1.27)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 223; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 1  MOD_RES 227; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q8VHF5   CISY_RAT         Cs                   Cs                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Citrate synthase, mitochondrial (EC 2.3.3.1) (...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 57; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Methylated. Trimethylation at Lys-395 by ...         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  PDB  \n",
       " 0             NaN  TRANSIT 1..27; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P11861  NAHC1_PSEPU       nahC                 nahC                   NaN   \n",
       " 1  P0A109  NAHC2_PSEPU       nahC                 nahC                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas putida (Arthrobacter siderocapsula...            303   \n",
       " 1  Pseudomonas putida (Arthrobacter siderocapsula...            303   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  1,2-dihydroxynaphthalene dioxygenase (1,2-DHN ...  ...            NaN   \n",
       " 1  1,2-dihydroxynaphthalene dioxygenase (1,2-DHN ...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide Signal peptide Transit peptide  \\\n",
       " 0                              NaN         NaN            NaN             NaN   \n",
       " 1                              NaN         NaN            NaN             NaN   \n",
       " \n",
       "    PDB  \n",
       " 0  NaN  \n",
       " 1  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                          Gene Names  \\\n",
       " 0  O75164  KDM4A_HUMAN  KDM4A JHDM3A JMJD2 JMJD2A KIAA0677   \n",
       " \n",
       "   Gene Names (primary)          Gene Names (synonym)  \\\n",
       " 0                KDM4A  JHDM3A JMJD2 JMJD2A KIAA0677   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Lysine-specific demethylase 4A (EC 1.14.11.66)...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Ubiquitinated by RNF8 and RNF168 followin...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  2GF7;2GFA;2GP3;2GP5;2OQ6;2OQ7;2OS2;2OT7;2OX0;2...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P0A9N4  PFLA_ECOLI   pflA act b0902 JW0885                 pflA   \n",
       " 1  P75794  PFLE_ECOLI       ybiY b0824 JW0808                 ybiY   \n",
       " 2  P32675  PFLC_ECOLI  pflC yijM b3952 JW3924                 pflC   \n",
       " 3  P0A9N6  PFLA_ECO57      pflA Z1246 ECs0985                 pflA   \n",
       " 4  P0A9N5  PFLA_ECOL6              pflA c1038                 pflA   \n",
       " 5  P0A9N7  PFLA_SHIFL       pflA SF0897 S0961                 pflA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  act               b0902 JW0885               NaN   \n",
       " 1                  NaN               b0824 JW0808               NaN   \n",
       " 2                 yijM               b3952 JW3924               NaN   \n",
       " 3                  NaN              Z1246 ECs0985               NaN   \n",
       " 4                  NaN                      c1038               NaN   \n",
       " 5                  NaN               SF0897 S0961               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain K12)          83333   \n",
       " 2                      Escherichia coli (strain K12)          83333   \n",
       " 3                           Escherichia coli O157:H7          83334   \n",
       " 4  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 5                                  Shigella flexneri            623   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Pyruvate formate-lyase 1-activating enzyme (EC...  ...           NaN   \n",
       " 1  Putative pyruvate formate-lyase 3-activating e...  ...           NaN   \n",
       " 2  Pyruvate formate-lyase 2-activating enzyme (EC...  ...           NaN   \n",
       " 3  Pyruvate formate-lyase 1-activating enzyme (EC...  ...           NaN   \n",
       " 4  Pyruvate formate-lyase 1-activating enzyme (EC...  ...           NaN   \n",
       " 5  Pyruvate formate-lyase 1-activating enzyme (EC...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                        INIT_MET 1; /note=\"Removed\"         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 4  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 5  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " 2               NaN     NaN                              NaN         NaN   \n",
       " 3               NaN     NaN                              NaN         NaN   \n",
       " 4               NaN     NaN                              NaN         NaN   \n",
       " 5               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide         PDB  \n",
       " 0             NaN             NaN  3C8F;3CB8;  \n",
       " 1             NaN             NaN         NaN  \n",
       " 2             NaN             NaN         NaN  \n",
       " 3             NaN             NaN         NaN  \n",
       " 4             NaN             NaN         NaN  \n",
       " 5             NaN             NaN         NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9H2M3  BHMT2_HUMAN      BHMT2                BHMT2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  S-methylmethionine--homocysteine S-methyltrans...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 321; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P30566  PUR8_HUMAN  ADSL AMPS                 ADSL                 AMPS   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Adenylosuccinate lyase (ADSL) (ASL) (EC 4.3.2....  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                   PDB  \n",
       " 0             NaN  2J91;2VD6;4FFX;4FLC;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P0A1P6  GLN1B_SALTY        glnA STM4007                 glnA   \n",
       " 1  P0A1P7  GLN1B_SALTI  glnA STY3874 t3614                 glnA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    STM4007               NaN   \n",
       " 1                   NaN              STY3874 t3614               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 1                                   Salmonella typhi          90370   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutamine synthetase (GS) (EC 6.3.1.2) (Glutam...  ...           NaN   \n",
       " 1  Glutamine synthetase (GS) (EC 6.3.1.2) (Glutam...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 398; /note=\"O-AMP-tyrosine\"; /evidence...     NaN   \n",
       " 1  MOD_RES 398; /note=\"O-AMP-tyrosine\"; /evidence...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  1F1H;1F52;1FPY;1LGR;2GLS;2LGS;  \n",
       " 1             NaN                           5LDF;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  Q5EA87  B4GT3_BOVIN             B4GALT3              B4GALT3   \n",
       " 1  P08037  B4GT1_BOVIN  B4GALT1 GALT GGTB2              B4GALT1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1           GALT GGTB2                         NaN               NaN   \n",
       " \n",
       "               Organism  Organism (ID)  \\\n",
       " 0  Bos taurus (Bovine)           9913   \n",
       " 1  Bos taurus (Bovine)           9913   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Beta-1,4-galactosyltransferase 3 (Beta-1,4-Gal...  ...   \n",
       " 1  Beta-1,4-galactosyltransferase 1 (Beta-1,4-Gal...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 57; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 1  CARBOHYD 90; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " 1         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: The soluble form derives from the membran...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                                NaN  \n",
       " 1  1FGX;1FR8;1NF5;1NHE;1NKH;1NMM;1NQI;1NWG;1O0R;1...  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P13491  LDHA_RABIT       LDHA                 LDHA                   NaN   \n",
       " 1  P13490  LDHB_RABIT       LDHB                 LDHB                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Oryctolagus cuniculus (Rabbit)           9986   \n",
       " 1  Oryctolagus cuniculus (Rabbit)           9986   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  L-lactate dehydrogenase A chain (LDH-A) (EC 1....  ...           NaN   \n",
       " 1  L-lactate dehydrogenase B chain (LDH-B) (EC 1....  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 123; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " \n",
       "                    Post-translational modification  Propeptide  \\\n",
       " 0  PTM: ISGylated. {ECO:0000250|UniProtKB:P00338}.         NaN   \n",
       " 1                                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  3H3F;4I8X;4I9H;4I9N;4I9U;5KKC;5NQB;5NQQ;6P6U;7...  \n",
       " 1                                                NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P0ABH7  CISY_ECOLI  gltA gluT icdB b0720 JW0710                 gltA   \n",
       " 1  P31660  PRPC_ECOLI  prpC yahS yzzD b0333 JW0324                 prpC   \n",
       " 2  P26491  CISY_MYCSM                         gltA                 gltA   \n",
       " 3  D4GS06  CISY_HALVD     citZ HVO_0466 C498_16349                 citZ   \n",
       " 4  P0ABH8  CISY_ECOL6                   gltA c0796                 gltA   \n",
       " 5  P51045  CISY_ACIFI                         gltA                 gltA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0            gluT icdB               b0720 JW0710              NaN   \n",
       " 1            yahS yzzD               b0333 JW0324              NaN   \n",
       " 2                  NaN                        NaN              NaN   \n",
       " 3                  NaN                   HVO_0466       C498_16349   \n",
       " 4                  NaN                      c0796              NaN   \n",
       " 5                  NaN                        NaN              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain K12)          83333   \n",
       " 2  Mycolicibacterium smegmatis (Mycobacterium sme...           1772   \n",
       " 3  Haloferax volcanii (strain ATCC 29605 / DSM 37...         309800   \n",
       " 4  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 5                      Acidithiobacillus ferridurans        1232575   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0                     Citrate synthase (EC 2.3.3.16)  ...           NaN   \n",
       " 1  2-methylcitrate synthase (2-MCS) (MCS) (EC 2.3...  ...           NaN   \n",
       " 2                     Citrate synthase (EC 2.3.3.16)  ...           NaN   \n",
       " 3                     Citrate synthase (EC 2.3.3.16)  ...           NaN   \n",
       " 4                     Citrate synthase (EC 2.3.3.16)  ...           NaN   \n",
       " 5                     Citrate synthase (EC 2.3.3.16)  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 283; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " 4  MOD_RES 283; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 5                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                            PDB  \n",
       " 0             NaN  1NXE;1NXG;1OWB;1OWC;4G6B;4JAD;4JAE;4JAF;4JAG;  \n",
       " 1             NaN                                            NaN  \n",
       " 2             NaN                                            NaN  \n",
       " 3             NaN                                            NaN  \n",
       " 4             NaN                                            NaN  \n",
       " 5             NaN                                            NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  A4FV08  GNPI1_BOVIN     GNPDA1               GNPDA1                   NaN   \n",
       " 1  Q17QL1  GNPI2_BOVIN     GNPDA2               GNPDA2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 1                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Glucosamine-6-phosphate isomerase 1 (EC 3.5.99...  ...   \n",
       " 1           9913  Glucosamine-6-phosphate isomerase 2 (EC 3.5.99...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 64; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 1  MOD_RES 161; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "         Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  A0A3L6E0R4  GOX1_MAIZE  GO1 Zm00014a_001774                  GO1   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN  Zm00014a_001774   \n",
       " \n",
       "            Organism  Organism (ID)                            Protein names  \\\n",
       " 0  Zea mays (Maize)           4577  Glycolate oxidase 1 (GOX) (EC 1.1.3.15)   \n",
       " \n",
       "    ... Glycosylation Initiator methionine  Lipidation  Modified residue  \\\n",
       " 0  ...           NaN                  NaN         NaN               NaN   \n",
       " \n",
       "   Peptide  Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0     NaN                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P50336  PPOX_HUMAN       PPOX                 PPOX                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                  Protein names  ...  \\\n",
       " 0           9606  Protoporphyrinogen oxidase (PPO) (EC 1.3.3.4)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  3NKS;4IVM;4IVO;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P40011  RRAAH_YEAST    YER010C                   NaN                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                    YER010C               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  4-hydroxy-4-methyl-2-oxoglutarate aldolase (HM...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2C5Q;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q59034  TRMY_METJA  trmY MJ1640                 trmY                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     MJ1640               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  tRNA (pseudouridine(54)-N(1))-methyltransferas...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  3AI9;3AIA;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P27001  SYFA_THETH           pheS                 pheS   \n",
       " 1  P27002  SYFB_THETH           pheT                 pheT   \n",
       " 2  Q5SGX1  SYFB_THET8  pheT TTHA1959                 pheT   \n",
       " 3  Q5SGX2  SYFA_THET8  pheS TTHA1958                 pheS   \n",
       " 4  Q72HA1  SYFA_THET2  pheS TT_C1594                 pheS   \n",
       " 5  Q72HA0  SYFB_THET2  pheT TT_C1595                 pheT   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                        NaN               NaN   \n",
       " 1                   NaN                        NaN               NaN   \n",
       " 2                   NaN                   TTHA1959               NaN   \n",
       " 3                   NaN                   TTHA1958               NaN   \n",
       " 4                   NaN                   TT_C1594               NaN   \n",
       " 5                   NaN                   TT_C1595               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                               Thermus thermophilus            274   \n",
       " 1                               Thermus thermophilus            274   \n",
       " 2  Thermus thermophilus (strain ATCC 27634 / DSM ...         300852   \n",
       " 3  Thermus thermophilus (strain ATCC 27634 / DSM ...         300852   \n",
       " 4  Thermus thermophilus (strain ATCC BAA-163 / DS...         262724   \n",
       " 5  Thermus thermophilus (strain ATCC BAA-163 / DS...         262724   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phenylalanine--tRNA ligase alpha subunit (EC 6...  ...           NaN   \n",
       " 1  Phenylalanine--tRNA ligase beta subunit (EC 6....  ...           NaN   \n",
       " 2  Phenylalanine--tRNA ligase beta subunit (EC 6....  ...           NaN   \n",
       " 3  Phenylalanine--tRNA ligase alpha subunit (EC 6...  ...           NaN   \n",
       " 4  Phenylalanine--tRNA ligase alpha subunit (EC 6...  ...           NaN   \n",
       " 5  Phenylalanine--tRNA ligase beta subunit (EC 6....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                       PDB  \n",
       " 0             NaN  1B70;1B7Y;2AKW;2ALY;2AMC;2IY5;3TEH;4TVA;  \n",
       " 1             NaN  1B70;1B7Y;2AKW;2ALY;2AMC;2IY5;3TEH;4TVA;  \n",
       " 2             NaN                      1EIY;1JJC;1PYS;3HFZ;  \n",
       " 3             NaN                      1EIY;1JJC;1PYS;3HFZ;  \n",
       " 4             NaN                                       NaN  \n",
       " 5             NaN                                       NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry   Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  Q9UBQ7  GRHPR_HUMAN  GRHPR GLXR MSTP035                GRHPR   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 GLXR                         NaN          MSTP035   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glyoxylate reductase/hydroxypyruvate reductase...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 36; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                   PDB  \n",
       " 0             NaN  2GCG;2H1S;2Q50;2WWR;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0   P0A8K1   PSD_ECOLI    psd b4160 JW4121                  psd   \n",
       " 1   P39006  PSD1_YEAST  PSD1 YNL169C N1692                 PSD1   \n",
       " 2   P0A8K3   PSD_ECO57   psd Z5766 ECs5139                  psd   \n",
       " 3   B6I267   PSD_ECOSE       psd ECSE_4460                  psd   \n",
       " 4   B1XDR4   PSD_ECODH    psd ECDH10B_4355                  psd   \n",
       " 5   B7M8S3   PSD_ECO8A     psd ECIAI1_4395                  psd   \n",
       " 6   Q1R397   PSD_ECOUT     psd UTI89_C4760                  psd   \n",
       " 7   B7LC21   PSD_ECO55    psd EC55989_4717                  psd   \n",
       " 8   B7MSX5   PSD_ECO81      psd ECED1_4947                  psd   \n",
       " 9   C5A1F4   PSD_ECOBW        psd BWG_3875                  psd   \n",
       " 10  A8A7Q7   PSD_ECOHS      psd EcHS_A4402                  psd   \n",
       " 11  B5Z2G9   PSD_ECO5E   psd ECH74115_5676                  psd   \n",
       " 12  A7ZV31   PSD_ECO24  psd EcE24377A_4717                  psd   \n",
       " 13  B1LQI3   PSD_ECOSM    psd EcSMS35_4631                  psd   \n",
       " 14  B7MKW7   PSD_ECO45      psd ECS88_4748                  psd   \n",
       " 15  B7NTL9   PSD_ECO7I    psd ECIAI39_4625                  psd   \n",
       " 16  B7UPY0   PSD_ECO27     psd E2348C_4486                  psd   \n",
       " 17  Q0T9N1   PSD_ECOL5        psd ECP_4404                  psd   \n",
       " 18  B7NG97   PSD_ECOLU      psd ECUMN_4694                  psd   \n",
       " 19  P0A8K2   PSD_ECOL6           psd c5247                  psd   \n",
       " 20  B1IT43   PSD_ECOLC      psd EcolC_3850                  psd   \n",
       " 21  B7LLU4   PSD_ESCF3       psd EFER_4214                  psd   \n",
       " 22  P0A8K4   PSD_SHIFL    psd SF4318 S4582                  psd   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN               b4160 JW4121              NaN   \n",
       " 1                    NaN                    YNL169C            N1692   \n",
       " 2                    NaN              Z5766 ECs5139              NaN   \n",
       " 3                    NaN                  ECSE_4460              NaN   \n",
       " 4                    NaN               ECDH10B_4355              NaN   \n",
       " 5                    NaN                ECIAI1_4395              NaN   \n",
       " 6                    NaN                UTI89_C4760              NaN   \n",
       " 7                    NaN               EC55989_4717              NaN   \n",
       " 8                    NaN                 ECED1_4947              NaN   \n",
       " 9                    NaN                   BWG_3875              NaN   \n",
       " 10                   NaN                 EcHS_A4402              NaN   \n",
       " 11                   NaN              ECH74115_5676              NaN   \n",
       " 12                   NaN             EcE24377A_4717              NaN   \n",
       " 13                   NaN               EcSMS35_4631              NaN   \n",
       " 14                   NaN                 ECS88_4748              NaN   \n",
       " 15                   NaN               ECIAI39_4625              NaN   \n",
       " 16                   NaN                E2348C_4486              NaN   \n",
       " 17                   NaN                   ECP_4404              NaN   \n",
       " 18                   NaN                 ECUMN_4694              NaN   \n",
       " 19                   NaN                      c5247              NaN   \n",
       " 20                   NaN                 EcolC_3850              NaN   \n",
       " 21                   NaN                  EFER_4214              NaN   \n",
       " 22                   NaN               SF4318 S4582              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1   Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2                            Escherichia coli O157:H7          83334   \n",
       " 3                      Escherichia coli (strain SE11)         409438   \n",
       " 4               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 5                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 6              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 7              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 8                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 9     Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 10                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 11    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 12  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 13          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 14       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 15      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 16  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 17    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 18  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 19  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 20  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 21  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 22                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ...  \\\n",
       " 0   Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 1   Phosphatidylserine decarboxylase proenzyme 1, ...  ...   \n",
       " 2   Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 3   Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 4   Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 5   Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 6   Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 7   Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 8   Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 9   Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 10  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 11  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 12  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 13  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 14  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 15  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 16  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 17  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 18  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 19  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 20  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 21  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " 22  Phosphatidylserine decarboxylase proenzyme (EC...  ...   \n",
       " \n",
       "                                         Glycosylation Initiator methionine  \\\n",
       " 0                                                 NaN                  NaN   \n",
       " 1   CARBOHYD 34; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 2                                                 NaN                  NaN   \n",
       " 3                                                 NaN                  NaN   \n",
       " 4                                                 NaN                  NaN   \n",
       " 5                                                 NaN                  NaN   \n",
       " 6                                                 NaN                  NaN   \n",
       " 7                                                 NaN                  NaN   \n",
       " 8                                                 NaN                  NaN   \n",
       " 9                                                 NaN                  NaN   \n",
       " 10                                                NaN                  NaN   \n",
       " 11                                                NaN                  NaN   \n",
       " 12                                                NaN                  NaN   \n",
       " 13                                                NaN                  NaN   \n",
       " 14                                                NaN                  NaN   \n",
       " 15                                                NaN                  NaN   \n",
       " 16                                                NaN                  NaN   \n",
       " 17                                                NaN                  NaN   \n",
       " 18                                                NaN                  NaN   \n",
       " 19                                                NaN                  NaN   \n",
       " 20                                                NaN                  NaN   \n",
       " 21                                                NaN                  NaN   \n",
       " 22                                                NaN                  NaN   \n",
       " \n",
       "     Lipidation                                   Modified residue Peptide  \\\n",
       " 0          NaN  MOD_RES 1; /note=\"N-formylmethionine\"; /eviden...     NaN   \n",
       " 1          NaN  MOD_RES 463; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 2          NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 3          NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 4          NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 5          NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 6          NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 7          NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 8          NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 9          NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 10         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 11         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 12         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 13         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 14         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 15         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 16         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 17         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 18         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 19         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 20         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 21         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " 22         NaN  MOD_RES 254; /note=\"Pyruvic acid (Ser); by aut...     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0   PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 1   PTM: Glycosylated at Asn-34 in the endoplasmic...         NaN   \n",
       " 2   PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 3   PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 4   PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 5   PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 6   PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 7   PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 8   PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 9   PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 10  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 11  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 12  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 13  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 14  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 15  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 16  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 17  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 18  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 19  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 20  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 21  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 22  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " \n",
       "     Signal peptide                                    Transit peptide  PDB  \n",
       " 0              NaN                                                NaN  NaN  \n",
       " 1              NaN  TRANSIT 1..48; /note=\"Mitochondrion; not cleav...  NaN  \n",
       " 2              NaN                                                NaN  NaN  \n",
       " 3              NaN                                                NaN  NaN  \n",
       " 4              NaN                                                NaN  NaN  \n",
       " 5              NaN                                                NaN  NaN  \n",
       " 6              NaN                                                NaN  NaN  \n",
       " 7              NaN                                                NaN  NaN  \n",
       " 8              NaN                                                NaN  NaN  \n",
       " 9              NaN                                                NaN  NaN  \n",
       " 10             NaN                                                NaN  NaN  \n",
       " 11             NaN                                                NaN  NaN  \n",
       " 12             NaN                                                NaN  NaN  \n",
       " 13             NaN                                                NaN  NaN  \n",
       " 14             NaN                                                NaN  NaN  \n",
       " 15             NaN                                                NaN  NaN  \n",
       " 16             NaN                                                NaN  NaN  \n",
       " 17             NaN                                                NaN  NaN  \n",
       " 18             NaN                                                NaN  NaN  \n",
       " 19             NaN                                                NaN  NaN  \n",
       " 20             NaN                                                NaN  NaN  \n",
       " 21             NaN                                                NaN  NaN  \n",
       " 22             NaN                                                NaN  NaN  \n",
       " \n",
       " [23 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q41062   SECA_PEA       SecA                 SecA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                    Organism  \\\n",
       " 0                         NaN               NaN  Pisum sativum (Garden pea)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           3888  Protein translocase subunit SecA, chloroplasti...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..59; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q60715  P4HA1_MOUSE      P4ha1                P4ha1                   NaN   \n",
       " 1  Q60716  P4HA2_MOUSE      P4ha2                P4ha2                   NaN   \n",
       " 2  Q6W3F0  P4HA3_MOUSE      P4ha3                P4ha3                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 1                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 2                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Prolyl 4-hydroxylase subunit alpha-1 (4-PH alp...  ...   \n",
       " 1          10090  Prolyl 4-hydroxylase subunit alpha-2 (4-PH alp...  ...   \n",
       " 2          10090  Prolyl 4-hydroxylase subunit alpha-3 (4-PH alp...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 113; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 117; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 2  CARBOHYD 240; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN                                                NaN     NaN   \n",
       " 1         NaN  MOD_RES 482; /note=\"N6-succinyllysine\"; /evide...     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: N-glycosylation plays no role in the cata...         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..17; /evidence=\"ECO:0000250\"             NaN  NaN  \n",
       " 1  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " 2  SIGNAL 1..24; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "         Entry Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0      Q9Z1L0  PK3CB_RAT     Pik3cb               Pik3cb                  NaN   \n",
       " 1  A0A0G2K344  PK3CA_RAT     Pik3ca               Pik3ca                  NaN   \n",
       " 2      Q99NI4   IPMK_RAT  Ipmk Impk                 Ipmk                 Impk   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 2                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...   \n",
       " 1          10116  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...   \n",
       " 2          10116  Inositol polyphosphate multikinase (EC 2.7.1.1...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN                                                NaN   \n",
       " 1           NaN                                                NaN   \n",
       " 2           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 324; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1         NaN                                                NaN     NaN   \n",
       " 2         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Autophosphorylation at Ser-1070 negativel...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry   Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  P14550  AK1A1_HUMAN  AKR1A1 ALDR1 ALR               AKR1A1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0            ALDR1 ALR                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aldo-keto reductase family 1 member A1 (EC 1.1...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2ALR;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9HXE3  DAUA_PSEAE  dauA PA3863                 dauA                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     PA3863               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  FAD-dependent catabolic D-arginine dehydrogena...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                  PDB  \n",
       " 0             NaN  3NYC;3NYE;3NYF;3SM8;6P9D;6PLD;7RDF;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name    Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q9ES71  GNPAT_RAT  Gnpat Dhapat                Gnpat               Dhapat   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Dihydroxyacetone phosphate acyltransferase (DA...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 12; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q3MHR2  DCK_BOVIN        DCK                  DCK                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Deoxycytidine kinase (dCK) (EC 2.7.1.74) (Deox...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 11; /note=\"Phosphoserine; by CK1\"; /ev...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylated and activated in vitro upo...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P29466  CASP1_HUMAN  CASP1 IL1BC IL1BCE                CASP1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0         IL1BC IL1BCE                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Caspase-1 (CASP-1) (EC 3.4.22.36) (Interleukin...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 302; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: The two subunits are derived from the pre...   \n",
       " \n",
       "                                           Propeptide  Signal peptide  \\\n",
       " 0  PROPEP 1..119; /id=\"PRO_0000004521\"; PROPEP 29...             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1BMQ;1IBC;1ICE;1RWK;1RWM;1RWN;1RWO;1RWP;1RWV;1...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P30011  NADC_ECOLI  nadC b0109 JW0105                 nadC   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN               b0109 JW0105               NaN   \n",
       " \n",
       "                         Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Nicotinate-nucleotide pyrophosphorylase [carbo...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q5BQE6  DHC24_RAT     Dhcr24               Dhcr24                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Delta(24)-sterol reductase (EC 1.3.1.72) (24-d...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..22; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name       Gene Names  Gene Names (primary)  \\\n",
       " 0  Q9UWU1  ARGDC_SACS2  SSO0536 C22_015                   NaN   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    SSO0536          C22_015   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharolobus solfataricus (strain ATCC 35092 ...         273057   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Arginine decarboxylase proenzyme (ADC) (ArgDC)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 82; /note=\"Pyruvic acid (Ser); by auto...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  O64390  HXK1_SOLTU   HXK1 HXK                 HXK1                  HXK   \n",
       " 1  Q9SQ76  HXK2_SOLTU       HXK2                 HXK2                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                    Organism  \\\n",
       " 0                         NaN               NaN  Solanum tuberosum (Potato)   \n",
       " 1                         NaN               NaN  Solanum tuberosum (Potato)   \n",
       " \n",
       "    Organism (ID)                      Protein names  ... Glycosylation  \\\n",
       " 0           4113  Hexokinase-1 (EC 2.7.1.1) (StHK1)  ...           NaN   \n",
       " 1           4113  Hexokinase-2 (EC 2.7.1.1) (StHK2)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                      Gene Names Gene Names (primary)  \\\n",
       " 0  P25373  GLRX1_YEAST             GRX1 YCL035C YCL35C                 GRX1   \n",
       " 1  P17695  GLRX2_YEAST  GRX2 TTR TTR1 YDR513W D9719.17                 GRX2   \n",
       " 2  P23202   URE2_YEAST              URE2 YNL229C N1165                 URE2   \n",
       " 3  P36014   GPX1_YEAST                    GPX1 YKL026C                 GPX1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                  NaN                    YCL035C           YCL35C   \n",
       " 1             TTR TTR1                    YDR513W         D9719.17   \n",
       " 2                  NaN                    YNL229C            N1165   \n",
       " 3                  NaN                    YKL026C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 3  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutaredoxin-1 (EC 1.11.1.9) (EC 2.5.1.18) (Gl...  ...           NaN   \n",
       " 1  Glutaredoxin-2 (EC 1.11.1.9) (EC 2.5.1.18) (Gl...  ...           NaN   \n",
       " 2  Transcriptional regulator URE2 (Disulfide redu...  ...           NaN   \n",
       " 3  Glutathione peroxidase-like peroxiredoxin 1 (E...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET P17695-2:1; /note=\"Removed\"; /evidenc...         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 27; /note=\"S-glutathionyl cysteine; al...     NaN   \n",
       " 1  MOD_RES 37; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 2  MOD_RES 2; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  \\\n",
       " 0                                                NaN   \n",
       " 1  TRANSIT 1..30; /note=\"Mitochondrion\"; /evidenc...   \n",
       " 2                                                NaN   \n",
       " 3                                                NaN   \n",
       " \n",
       "                                         PDB  \n",
       " 0                 2JAC;2JAD;3C1R;3C1S;6MWS;  \n",
       " 1                      3CTF;3CTG;3D4M;3D5J;  \n",
       " 2  1G6W;1G6Y;1HQO;1JZR;1K0A;1K0B;1K0C;1K0D;  \n",
       " 3                                       NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q58579  RIBL_METJA  ribL MJ1179                 ribL                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     MJ1179               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  FAD synthase (EC 2.7.7.2) (FMN adenylyltransfe...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0   P0A2K1  TRPB_SALTY        trpB STM1726                 trpB   \n",
       " 1   P00929  TRPA_SALTY        trpA STM1727                 trpA   \n",
       " 2   P0A2K2  TRPB_SALTI  trpB STY1325 t1638                 trpB   \n",
       " 3   Q8Z7E0  TRPA_SALTI  trpA STY1324 t1639                 trpA   \n",
       " 4   B4TJK8  TRPB_SALHS     trpB SeHA_C1916                 trpB   \n",
       " 5   B4TJK9  TRPA_SALHS     trpA SeHA_C1917                 trpA   \n",
       " 6   B4TX38  TRPB_SALSV     trpB SeSA_A1858                 trpB   \n",
       " 7   B5F4M3  TRPA_SALA4     trpA SeAg_B1419                 trpA   \n",
       " 8   B4TX39  TRPA_SALSV     trpA SeSA_A1859                 trpA   \n",
       " 9   B5F4M4  TRPB_SALA4     trpB SeAg_B1420                 trpB   \n",
       " 10  C0Q3N3  TRPA_SALPC       trpA SPC_2002                 trpA   \n",
       " 11  C0Q3N4  TRPB_SALPC       trpB SPC_2003                 trpB   \n",
       " 12  A9MWR3  TRPA_SALPB     trpA SPAB_01518                 trpA   \n",
       " 13  B4T6X2  TRPA_SALNS  trpA SNSL254_A1854                 trpA   \n",
       " 14  Q57NT3  TRPB_SALCH       trpB SCH_1722                 trpB   \n",
       " 15  B4T6X1  TRPB_SALNS  trpB SNSL254_A1853                 trpB   \n",
       " 16  Q57NT2  TRPA_SALCH       trpA SCH_1723                 trpA   \n",
       " 17  A9MWR4  TRPB_SALPB     trpB SPAB_01519                 trpB   \n",
       " 18  A9MPY6  TRPA_SALAR     trpA SARI_01228                 trpA   \n",
       " 19  A9MPY7  TRPB_SALAR     trpB SARI_01229                 trpB   \n",
       " 20  B5FU65  TRPA_SALDC      trpA SeD_A1601                 trpA   \n",
       " 21  B5FU66  TRPB_SALDC      trpB SeD_A1602                 trpB   \n",
       " 22  B5R3P4  TRPB_SALEP        trpB SEN1308                 trpB   \n",
       " 23  B5R3P3  TRPA_SALEP        trpA SEN1307                 trpA   \n",
       " 24  B5BIC1  TRPB_SALPK       trpB SSPA1070                 trpB   \n",
       " 25  B5BIC0  TRPA_SALPK       trpA SSPA1069                 trpA   \n",
       " 26  B5R6M4  TRPA_SALG2         trpA SG1388                 trpA   \n",
       " 27  B5R6M5  TRPB_SALG2         trpB SG1389                 trpB   \n",
       " 28  Q5PNM0  TRPA_SALPA        trpA SPA1150                 trpA   \n",
       " 29  Q5PD17  TRPB_SALPA        trpB SPA1151                 trpB   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                    NaN                    STM1726               NaN   \n",
       " 1                    NaN                    STM1727               NaN   \n",
       " 2                    NaN              STY1325 t1638               NaN   \n",
       " 3                    NaN              STY1324 t1639               NaN   \n",
       " 4                    NaN                 SeHA_C1916               NaN   \n",
       " 5                    NaN                 SeHA_C1917               NaN   \n",
       " 6                    NaN                 SeSA_A1858               NaN   \n",
       " 7                    NaN                 SeAg_B1419               NaN   \n",
       " 8                    NaN                 SeSA_A1859               NaN   \n",
       " 9                    NaN                 SeAg_B1420               NaN   \n",
       " 10                   NaN                   SPC_2002               NaN   \n",
       " 11                   NaN                   SPC_2003               NaN   \n",
       " 12                   NaN                 SPAB_01518               NaN   \n",
       " 13                   NaN              SNSL254_A1854               NaN   \n",
       " 14                   NaN                   SCH_1722               NaN   \n",
       " 15                   NaN              SNSL254_A1853               NaN   \n",
       " 16                   NaN                   SCH_1723               NaN   \n",
       " 17                   NaN                 SPAB_01519               NaN   \n",
       " 18                   NaN                 SARI_01228               NaN   \n",
       " 19                   NaN                 SARI_01229               NaN   \n",
       " 20                   NaN                  SeD_A1601               NaN   \n",
       " 21                   NaN                  SeD_A1602               NaN   \n",
       " 22                   NaN                    SEN1308               NaN   \n",
       " 23                   NaN                    SEN1307               NaN   \n",
       " 24                   NaN                   SSPA1070               NaN   \n",
       " 25                   NaN                   SSPA1069               NaN   \n",
       " 26                   NaN                     SG1388               NaN   \n",
       " 27                   NaN                     SG1389               NaN   \n",
       " 28                   NaN                    SPA1150               NaN   \n",
       " 29                   NaN                    SPA1151               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0   Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 1   Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 2                                    Salmonella typhi          90370   \n",
       " 3                                    Salmonella typhi          90370   \n",
       " 4                Salmonella heidelberg (strain SL476)         454169   \n",
       " 5                Salmonella heidelberg (strain SL476)         454169   \n",
       " 6         Salmonella schwarzengrund (strain CVM19633)         439843   \n",
       " 7                     Salmonella agona (strain SL483)         454166   \n",
       " 8         Salmonella schwarzengrund (strain CVM19633)         439843   \n",
       " 9                     Salmonella agona (strain SL483)         454166   \n",
       " 10            Salmonella paratyphi C (strain RKS4594)         476213   \n",
       " 11            Salmonella paratyphi C (strain RKS4594)         476213   \n",
       " 12  Salmonella paratyphi B (strain ATCC BAA-1250 /...        1016998   \n",
       " 13                  Salmonella newport (strain SL254)         423368   \n",
       " 14            Salmonella choleraesuis (strain SC-B67)         321314   \n",
       " 15                  Salmonella newport (strain SL254)         423368   \n",
       " 16            Salmonella choleraesuis (strain SC-B67)         321314   \n",
       " 17  Salmonella paratyphi B (strain ATCC BAA-1250 /...        1016998   \n",
       " 18  Salmonella arizonae (strain ATCC BAA-731 / CDC...          41514   \n",
       " 19  Salmonella arizonae (strain ATCC BAA-731 / CDC...          41514   \n",
       " 20             Salmonella dublin (strain CT_02021853)         439851   \n",
       " 21             Salmonella dublin (strain CT_02021853)         439851   \n",
       " 22        Salmonella enteritidis PT4 (strain P125109)         550537   \n",
       " 23        Salmonella enteritidis PT4 (strain P125109)         550537   \n",
       " 24          Salmonella paratyphi A (strain AKU_12601)         554290   \n",
       " 25          Salmonella paratyphi A (strain AKU_12601)         554290   \n",
       " 26  Salmonella gallinarum (strain 287/91 / NCTC 13...         550538   \n",
       " 27  Salmonella gallinarum (strain 287/91 / NCTC 13...         550538   \n",
       " 28  Salmonella paratyphi A (strain ATCC 9150 / SAR...         295319   \n",
       " 29  Salmonella paratyphi A (strain ATCC 9150 / SAR...         295319   \n",
       " \n",
       "                                     Protein names  ... Glycosylation  \\\n",
       " 0    Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 1   Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 2    Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 3   Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 4    Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 5   Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 6    Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 7   Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 8   Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 9    Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 10  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 11   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 12  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 13  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 14   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 15   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 16  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 17   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 18  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 19   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 20  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 21   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 22   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 23  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 24   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 25  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 26  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 27   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " 28  Tryptophan synthase alpha chain (EC 4.2.1.20)  ...           NaN   \n",
       " 29   Tryptophan synthase beta chain (EC 4.2.1.20)  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0                         INIT_MET 1; /note=\"Removed\"         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19                                                NaN         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22                                                NaN         NaN   \n",
       " 23                                                NaN         NaN   \n",
       " 24                                                NaN         NaN   \n",
       " 25                                                NaN         NaN   \n",
       " 26                                                NaN         NaN   \n",
       " 27                                                NaN         NaN   \n",
       " 28                                                NaN         NaN   \n",
       " 29                                                NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 1                                                 NaN     NaN   \n",
       " 2   MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 3                                                 NaN     NaN   \n",
       " 4   MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6   MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 7                                                 NaN     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9   MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 12                                                NaN     NaN   \n",
       " 13                                                NaN     NaN   \n",
       " 14  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 15  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 16                                                NaN     NaN   \n",
       " 17  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 18                                                NaN     NaN   \n",
       " 19  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 20                                                NaN     NaN   \n",
       " 21  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 22  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 23                                                NaN     NaN   \n",
       " 24  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 25                                                NaN     NaN   \n",
       " 26                                                NaN     NaN   \n",
       " 27  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 28                                                NaN     NaN   \n",
       " 29  MOD_RES 87; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " 21                              NaN         NaN             NaN   \n",
       " 22                              NaN         NaN             NaN   \n",
       " 23                              NaN         NaN             NaN   \n",
       " 24                              NaN         NaN             NaN   \n",
       " 25                              NaN         NaN             NaN   \n",
       " 26                              NaN         NaN             NaN   \n",
       " 27                              NaN         NaN             NaN   \n",
       " 28                              NaN         NaN             NaN   \n",
       " 29                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                                                PDB  \n",
       " 0              NaN  1A50;1A5A;1A5B;1A5S;1BEU;1BKS;1C29;1C8V;1C9D;1...  \n",
       " 1              NaN  1A50;1A5A;1A5B;1A5S;1BEU;1BKS;1C29;1C8V;1C9D;1...  \n",
       " 2              NaN                                                NaN  \n",
       " 3              NaN                                                NaN  \n",
       " 4              NaN                                                NaN  \n",
       " 5              NaN                                                NaN  \n",
       " 6              NaN                                                NaN  \n",
       " 7              NaN                                                NaN  \n",
       " 8              NaN                                                NaN  \n",
       " 9              NaN                                                NaN  \n",
       " 10             NaN                                                NaN  \n",
       " 11             NaN                                                NaN  \n",
       " 12             NaN                                                NaN  \n",
       " 13             NaN                                                NaN  \n",
       " 14             NaN                                                NaN  \n",
       " 15             NaN                                                NaN  \n",
       " 16             NaN                                                NaN  \n",
       " 17             NaN                                                NaN  \n",
       " 18             NaN                                                NaN  \n",
       " 19             NaN                                                NaN  \n",
       " 20             NaN                                                NaN  \n",
       " 21             NaN                                                NaN  \n",
       " 22             NaN                                                NaN  \n",
       " 23             NaN                                                NaN  \n",
       " 24             NaN                                                NaN  \n",
       " 25             NaN                                                NaN  \n",
       " 26             NaN                                                NaN  \n",
       " 27             NaN                                                NaN  \n",
       " 28             NaN                                                NaN  \n",
       " 29             NaN                                                NaN  \n",
       " \n",
       " [30 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  P40029  MSRA_YEAST  MXR1 YER042W                 MXR1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    YER042W               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Peptide methionine sulfoxide reductase (EC 1.8...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 58; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  3PIL;3PIM;3PIN;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P26443  DHE3_MOUSE  Glud1 Glud                Glud1                 Glud   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Glutamate dehydrogenase 1, mitochondrial (GDH ...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 68; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Acetylation of Lys-84 is observed in live...         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  PDB  \n",
       " 0             NaN  TRANSIT 1..53; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  Q55367  CYNS_SYNY3          cynS slr0899                 cynS   \n",
       " 1  Q59948  CYNS_SYNE7  cynS Synpcc7942_2104                 cynS   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    slr0899               NaN   \n",
       " 1                   NaN            Synpcc7942_2104               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0       Synechocystis sp. (strain PCC 6803 / Kazusa)        1111708   \n",
       " 1  Synechococcus elongatus (strain PCC 7942 / FAC...           1140   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Cyanate hydratase (Cyanase) (EC 4.2.1.104) (Cy...  ...           NaN   \n",
       " 1  Cyanate hydratase (Cyanase) (EC 4.2.1.104) (Cy...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q76IQ3  TY3H_CANLF         TH                   TH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                           Organism  Organism (ID)  \\\n",
       " 0  Canis lupus familiaris (Dog) (Canis familiaris)           9615   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Tyrosine 3-monooxygenase (EC 1.14.16.2) (Tyros...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 19; /note=\"Phosphoserine; by CaMK2\"; /...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylated on Ser-19, Ser-31 and Ser-...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  Q48230   BCS1_HAEIF                 bcs1                 bcs1   \n",
       " 1  Q2G1C4  TARJ2_STAA8  tarJ' SAOUHSC_00221                tarJ'   \n",
       " 2  Q2G1B9  TARJ1_STAA8   tarJ SAOUHSC_00226                 tarJ   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                        NaN               NaN   \n",
       " 1                   NaN              SAOUHSC_00221               NaN   \n",
       " 2                   NaN              SAOUHSC_00226               NaN   \n",
       " \n",
       "                                            Organism  Organism (ID)  \\\n",
       " 0                            Haemophilus influenzae            727   \n",
       " 1  Staphylococcus aureus (strain NCTC 8325 / PS 47)          93061   \n",
       " 2  Staphylococcus aureus (strain NCTC 8325 / PS 47)          93061   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Bifunctional ribulose 5-phosphate reductase/CD...  ...           NaN   \n",
       " 1  Ribulose-5-phosphate reductase 2 (Ribulose-5-P...  ...           NaN   \n",
       " 2  Ribulose-5-phosphate reductase 1 (Ribulose-5-P...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN         NaN  \n",
       " 1             NaN         NaN  \n",
       " 2             NaN  6XH9;6XHK;  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                  Gene Names Gene Names (primary)  \\\n",
       " 0  Q04119  PPN1_YEAST  PPN1 PHM5 YDR452W D9461.37                 PPN1   \n",
       " 1  P38698  PPX1_YEAST                PPX1 YHR201C                 PPX1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 PHM5                    YDR452W         D9461.37   \n",
       " 1                  NaN                    YHR201C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Endopolyphosphatase (EC 3.6.1.10) (Deoxyadenos...  ...   \n",
       " 1  Polyphosphatase (PolyPase) (EC 3.6.1.11) (Exop...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 58; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 1                                                NaN                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " 1         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: Processing by proteases in the vacuole is...   \n",
       " 1                                                NaN   \n",
       " \n",
       "                                           Propeptide  Signal peptide  \\\n",
       " 0  PROPEP 1..83; /note=\"Removed in mature form\"; ...             NaN   \n",
       " 1                                                NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN              NaN  \n",
       " 1             NaN  2QB6;2QB7;2QB8;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O60507  TPST1_HUMAN      TPST1                TPST1                   NaN   \n",
       " 1  O60704  TPST2_HUMAN      TPST2                TPST2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Protein-tyrosine sulfotransferase 1 (EC 2.8.2....  ...   \n",
       " 1           9606  Protein-tyrosine sulfotransferase 2 (EC 2.8.2....  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 60; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 1  CARBOHYD 343; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " 1         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated. {ECO:0000269|PubMed:16859...         NaN   \n",
       " 1  PTM: N-glycosylated. {ECO:0000269|PubMed:97337...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide              PDB  \n",
       " 0             NaN             NaN       5WRI;5WRJ;  \n",
       " 1             NaN             NaN  3AP1;3AP2;3AP3;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  B0EXJ8  HTOMT_CATRO      16OMT                16OMT                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Catharanthus roseus (Madagascar periwinkle) (V...           4058   \n",
       " \n",
       "                                       Protein names  ...  Glycosylation  \\\n",
       " 0  Tabersonine 16-O-methyltransferase (EC 2.1.1.94)  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  Q5T6J7  GNTK_HUMAN  IDNK C9orf103                 IDNK   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0             C9orf103                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Probable gluconokinase (EC 2.7.1.12) (Gluconat...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                         Gene Names Gene Names (primary)  \\\n",
       " 0  Q9LIS3  GAE6_ARATH    GAE6 UGlcAE2 At3g23820 F14O13.1                 GAE6   \n",
       " 1  Q9M0B6  GAE1_ARATH  GAE1 UGlcAE3 At4g30440 F17I23.220                 GAE1   \n",
       " 2  Q9STI6  GAE5_ARATH             GAE5 At4g12250 T4C9.90                 GAE5   \n",
       " 3  O22141  GAE4_ARATH       GAE4 UGlcAE1 At2g45310 F4L23                 GAE4   \n",
       " 4  O81312  GAE3_ARATH            GAE3 At4g00110 F6N15.16                 GAE3   \n",
       " 5  Q9LPC1  GAE2_ARATH            GAE2 At1g02000 F22M8.13                 GAE2   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0              UGlcAE2                  At3g23820         F14O13.1   \n",
       " 1              UGlcAE3                  At4g30440       F17I23.220   \n",
       " 2                  NaN                  At4g12250          T4C9.90   \n",
       " 3              UGlcAE1                  At2g45310            F4L23   \n",
       " 4                  NaN                  At4g00110         F6N15.16   \n",
       " 5                  NaN                  At1g02000         F22M8.13   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 3  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 4  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 5  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  UDP-glucuronate 4-epimerase 6 (EC 5.1.3.6) (UD...  ...           NaN   \n",
       " 1  UDP-glucuronate 4-epimerase 1 (EC 5.1.3.6) (UD...  ...           NaN   \n",
       " 2  UDP-glucuronate 4-epimerase 5 (EC 5.1.3.6) (UD...  ...           NaN   \n",
       " 3  UDP-glucuronate 4-epimerase 4 (EC 5.1.3.6) (UD...  ...           NaN   \n",
       " 4  UDP-glucuronate 4-epimerase 3 (EC 5.1.3.6) (UD...  ...           NaN   \n",
       " 5  UDP-glucuronate 4-epimerase 2 (EC 5.1.3.6) (UD...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " 5             NaN  NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q93YB2  AADH2_PEA     AMADH2               AMADH2                   NaN   \n",
       " 1  Q8VWZ1  AADH1_PEA     AMADH1               AMADH1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                    Organism  \\\n",
       " 0                         NaN               NaN  Pisum sativum (Garden pea)   \n",
       " 1                         NaN               NaN  Pisum sativum (Garden pea)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           3888  Aminoaldehyde dehydrogenase 2, peroxisomal (Ps...  ...   \n",
       " 1           3888  Aminoaldehyde dehydrogenase 1, peroxisomal (Ps...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " 1            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  3IWJ;  \n",
       " 1             NaN  3IWK;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name                       Gene Names Gene Names (primary)  \\\n",
       " 0  A0R2E2  GLGM_MYCS2  glgM glgA MSMEG_5080 MSMEI_4954                 glgM   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 glgA      MSMEG_5080 MSMEI_4954               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycolicibacterium smegmatis (strain ATCC 70008...         246196   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Alpha-maltose-1-phosphate synthase (M1P syntha...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  6TVP;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                        Gene Names Gene Names (primary)  \\\n",
       " 0  P27828  WECB_ECOLI  wecB nfrC rffE yifF b3786 JW5600                 wecB   \n",
       " 1  Q8XAR8  WECB_ECO57      wecB nfrC rffE Z5297 ECs4719                 wecB   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0       nfrC rffE yifF               b3786 JW5600               NaN   \n",
       " 1            nfrC rffE              Z5297 ECs4719               NaN   \n",
       " \n",
       "                         Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli (strain K12)          83333   \n",
       " 1       Escherichia coli O157:H7          83334   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3....  ...           NaN   \n",
       " 1  UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1F6D;1VGV;  \n",
       " 1             NaN         NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P0AGG0  THIL_ECOLI   thiL b0417 JW0407                 thiL   \n",
       " 1  Q8XE87  THIL_ECO57  thiL Z0519 ECs0470                 thiL   \n",
       " 2  P0AGG1  THIL_ECOL6          thiL c0528                 thiL   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN               b0417 JW0407               NaN   \n",
       " 1                   NaN              Z0519 ECs0470               NaN   \n",
       " 2                   NaN                      c0528               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                           Escherichia coli O157:H7          83334   \n",
       " 2  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Thiamine-monophosphate kinase (TMP kinase) (Th...  ...           NaN   \n",
       " 1  Thiamine-monophosphate kinase (TMP kinase) (Th...  ...           NaN   \n",
       " 2  Thiamine-monophosphate kinase (TMP kinase) (Th...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P00549  KPYK1_YEAST  CDC19 PYK1 YAL038W                CDC19   \n",
       " 1  P52489  KPYK2_YEAST  PYK2 YOR347C O6342                 PYK2   \n",
       " 2  Q759A9   KPYK_ASHGO        PYK1 ADR368W                 PYK1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 PYK1                    YAL038W              NaN   \n",
       " 1                  NaN                    YOR347C            O6342   \n",
       " 2                  NaN                    ADR368W              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2  Ashbya gossypii (strain ATCC 10895 / CBS 109.5...         284811   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Pyruvate kinase 1 (PK 1) (EC 2.7.1.40) (cell d...  ...           NaN   \n",
       " 1             Pyruvate kinase 2 (PK 2) (EC 2.7.1.40)  ...           NaN   \n",
       " 2                 Pyruvate kinase (PK) (EC 2.7.1.40)  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 1  MOD_RES 24; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1A3W;1A3X;  \n",
       " 1             NaN         NaN  \n",
       " 2             NaN         NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q58626  PYCA_METJA  pycA MJ1229                 pycA                   NaN   \n",
       " 1  Q58628  PYCB_METJA  pycB MJ1231                 pycB                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     MJ1229               NaN   \n",
       " 1                     MJ1231               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " 1  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Pyruvate carboxylase subunit A (EC 6.4.1.1) (P...  ...           NaN   \n",
       " 1  Pyruvate carboxylase subunit B (EC 6.4.1.1) (P...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES 172; /note=\"N6-carboxylysine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O68282  G6PD_PSEAE  zwf PA3183                  zwf                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     PA3183               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glucose-6-phosphate 1-dehydrogenase (G6PD) (EC...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                                         Gene Names  \\\n",
       " 0  Q6ZDJ7  UGE2_ORYSJ      UGE-2 Os08g0374800 LOC_Os08g28730 P0436B06.11   \n",
       " 1  Q6K2E1  UGE4_ORYSJ  UGE-4 Os09g0323000 LOC_Os09g15420 OsJ_28859 OS...   \n",
       " 2  Q8LNZ3  UGE1_ORYSJ  UGE-1 Os05g0595100 LOC_Os05g51670 OsJ_19776 OS...   \n",
       " 3  Q652A8  UGE3_ORYSJ  UGE-3 Os09g0526700 LOC_Os09g35800 OJ1439_F07.1...   \n",
       " \n",
       "   Gene Names (primary)  Gene Names (synonym)   Gene Names (ordered locus)  \\\n",
       " 0                UGE-2                   NaN  Os08g0374800 LOC_Os08g28730   \n",
       " 1                UGE-4                   NaN  Os09g0323000 LOC_Os09g15420   \n",
       " 2                UGE-1                   NaN  Os05g0595100 LOC_Os05g51670   \n",
       " 3                UGE-3                   NaN  Os09g0526700 LOC_Os09g35800   \n",
       " \n",
       "                Gene Names (ORF)                             Organism  \\\n",
       " 0                   P0436B06.11  Oryza sativa subsp. japonica (Rice)   \n",
       " 1  OsJ_28859 OSJNBb0024J13.13-1  Oryza sativa subsp. japonica (Rice)   \n",
       " 2     OsJ_19776 OSJNBa0030I14.9  Oryza sativa subsp. japonica (Rice)   \n",
       " 3       OJ1439_F07.18 OsJ_30076  Oryza sativa subsp. japonica (Rice)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          39947  UDP-glucose 4-epimerase 2 (OsUGE-2) (EC 5.1.3....  ...   \n",
       " 1          39947  UDP-glucose 4-epimerase 4 (OsUGE-4) (EC 5.1.3....  ...   \n",
       " 2          39947  UDP-glucose 4-epimerase 1 (OsUGE-1) (EC 5.1.3....  ...   \n",
       " 3          39947  UDP-glucose 4-epimerase 3 (OsUGE-3) (EC 5.1.3....  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " 1           NaN                  NaN         NaN               NaN     NaN   \n",
       " 2           NaN                  NaN         NaN               NaN     NaN   \n",
       " 3           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  P00968  CARB_ECOLI   carB pyrA b0033 JW0031                 carB   \n",
       " 1  P0A6F1  CARA_ECOLI   carA pyrA b0032 JW0030                 carA   \n",
       " 2  P0A6F2  CARA_ECO57  carA pyrA Z0037 ECs0035                 carA   \n",
       " 3  P03965  CARB_YEAST       CPA2 YJR109C J2002                 CPA2   \n",
       " 4  P63737  CARB_ECO57       carB Z0038 ECs0036                 carB   \n",
       " 5  Q8FLB1  CARA_ECOL6               carA c0040                 carA   \n",
       " 6  Q8FLB0  CARB_ECOL6               carB c0041                 carB   \n",
       " 7  P63738  CARB_SHIFL        carB SF0030 S0032                 carB   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 pyrA               b0033 JW0031              NaN   \n",
       " 1                 pyrA               b0032 JW0030              NaN   \n",
       " 2                 pyrA              Z0037 ECs0035              NaN   \n",
       " 3                  NaN                    YJR109C            J2002   \n",
       " 4                  NaN              Z0038 ECs0036              NaN   \n",
       " 5                  NaN                      c0040              NaN   \n",
       " 6                  NaN                      c0041              NaN   \n",
       " 7                  NaN               SF0030 S0032              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain K12)          83333   \n",
       " 2                           Escherichia coli O157:H7          83334   \n",
       " 3  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 4                           Escherichia coli O157:H7          83334   \n",
       " 5  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 6  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 7                                  Shigella flexneri            623   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Carbamoyl-phosphate synthase large chain (EC 6...  ...           NaN   \n",
       " 1  Carbamoyl-phosphate synthase small chain (EC 6...  ...           NaN   \n",
       " 2  Carbamoyl-phosphate synthase small chain (EC 6...  ...           NaN   \n",
       " 3  Carbamoyl-phosphate synthase arginine-specific...  ...           NaN   \n",
       " 4  Carbamoyl-phosphate synthase large chain (EC 6...  ...           NaN   \n",
       " 5  Carbamoyl-phosphate synthase small chain (EC 6...  ...           NaN   \n",
       " 6  Carbamoyl-phosphate synthase large chain (EC 6...  ...           NaN   \n",
       " 7  Carbamoyl-phosphate synthase large chain (EC 6...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 7  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " 2               NaN     NaN                              NaN         NaN   \n",
       " 3               NaN     NaN                              NaN         NaN   \n",
       " 4               NaN     NaN                              NaN         NaN   \n",
       " 5               NaN     NaN                              NaN         NaN   \n",
       " 6               NaN     NaN                              NaN         NaN   \n",
       " 7               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " 4             NaN             NaN   \n",
       " 5             NaN             NaN   \n",
       " 6             NaN             NaN   \n",
       " 7             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1A9X;1BXR;1C30;1C3O;1CE8;1CS0;1JDB;1KEE;1M6V;1...  \n",
       " 1  1A9X;1BXR;1C30;1C3O;1CE8;1CS0;1JDB;1KEE;1M6V;1...  \n",
       " 2                                                NaN  \n",
       " 3                                                NaN  \n",
       " 4                                                NaN  \n",
       " 5                                                NaN  \n",
       " 6                                                NaN  \n",
       " 7                                                NaN  \n",
       " \n",
       " [8 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P51569  AGAL_MOUSE    Gla Ags                  Gla                  Ags   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Alpha-galactosidase A (EC 3.2.1.22) (Alpha-D-g...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 139; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 186; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..31; /evidence=\"ECO:0000250\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9ZS21  LGUL_SOYBN       GLXI                 GLXI                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                   Organism  Organism (ID)  \\\n",
       " 0  Glycine max (Soybean) (Glycine hispida)           3847   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Lactoylglutathione lyase (EC 4.4.1.5) (Aldoket...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P38913  FAD1_YEAST  FAD1 YDL045C D2702                 FAD1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YDL045C            D2702   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Flavin adenine dinucleotide synthase (FAD synt...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2WSI;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  P68530  COX2_BOVIN  MT-CO2 COII COX2 COXII MTCO2               MT-CO2   \n",
       " 1  P00396  COX1_BOVIN         MT-CO1 COI COXI MTCO1               MT-CO1   \n",
       " 2  P00415  COX3_BOVIN     MT-CO3 COIII COXIII MTCO3               MT-CO3   \n",
       " 3  Q576B8  COX3_BOSIN     MT-CO3 COIII COXIII MTCO3               MT-CO3   \n",
       " 4  P68553  COX2_BOSIN  MT-CO2 COII COX2 COXII MTCO2               MT-CO2   \n",
       " \n",
       "     Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0  COII COX2 COXII MTCO2                         NaN               NaN   \n",
       " 1         COI COXI MTCO1                         NaN               NaN   \n",
       " 2     COIII COXIII MTCO3                         NaN               NaN   \n",
       " 3     COIII COXIII MTCO3                         NaN               NaN   \n",
       " 4  COII COX2 COXII MTCO2                         NaN               NaN   \n",
       " \n",
       "               Organism  Organism (ID)  \\\n",
       " 0  Bos taurus (Bovine)           9913   \n",
       " 1  Bos taurus (Bovine)           9913   \n",
       " 2  Bos taurus (Bovine)           9913   \n",
       " 3   Bos indicus (Zebu)           9915   \n",
       " 4   Bos indicus (Zebu)           9915   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Cytochrome c oxidase subunit 2 (EC 7.1.1.9) (C...  ...           NaN   \n",
       " 1  Cytochrome c oxidase subunit 1 (EC 7.1.1.9) (C...  ...           NaN   \n",
       " 2  Cytochrome c oxidase subunit 3 (EC 7.1.1.9) (C...  ...           NaN   \n",
       " 3  Cytochrome c oxidase subunit 3 (EC 7.1.1.9) (C...  ...           NaN   \n",
       " 4  Cytochrome c oxidase subunit 2 (EC 7.1.1.9) (C...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " 3                  NaN         NaN   \n",
       " 4                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-formylmethionine\"; /eviden...     NaN   \n",
       " 1  MOD_RES 1; /note=\"N-formylmethionine\"; /eviden...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " 4  MOD_RES 1; /note=\"N-formylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: His-240 and Tyr-244 are involved in the f...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " \n",
       "   Signal peptide Transit peptide  \\\n",
       " 0            NaN             NaN   \n",
       " 1            NaN             NaN   \n",
       " 2            NaN             NaN   \n",
       " 3            NaN             NaN   \n",
       " 4            NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1OCC;1OCO;1OCR;1OCZ;1V54;1V55;2DYR;2DYS;2EIJ;2...  \n",
       " 1  1OCC;1OCO;1OCR;1OCZ;1V54;1V55;2DYR;2DYS;2EIJ;2...  \n",
       " 2  1OCC;1OCO;1OCR;1OCZ;1V54;1V55;2DYR;2DYS;2EIJ;2...  \n",
       " 3                                                NaN  \n",
       " 4                                                NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry   Entry Name                  Gene Names Gene Names (primary)  \\\n",
       " 0  Q8GSJ1  HIS1B_ARATH  HISN1B At1g09795 F21M12.39               HISN1B   \n",
       " 1  Q9S762  HIS1A_ARATH    HISN1A At1g58080 T15M6.9               HISN1A   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                  At1g09795        F21M12.39   \n",
       " 1                   NaN                  At1g58080          T15M6.9   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  ATP phosphoribosyltransferase 2, chloroplastic...  ...           NaN   \n",
       " 1  ATP phosphoribosyltransferase 1, chloroplastic...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES 50; /note=\"N-acetylvaline\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..57; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 1  TRANSIT 1..49; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  Q9BTZ2  DHRS4_HUMAN  DHRS4 SDR25C2 UNQ851/PRO1800                DHRS4   \n",
       " 1  P16152   CBR1_HUMAN          CBR1 CBR CRN SDR21C1                 CBR1   \n",
       " 2  O75828   CBR3_HUMAN                  CBR3 SDR21C2                 CBR3   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0              SDR25C2                         NaN   UNQ851/PRO1800   \n",
       " 1      CBR CRN SDR21C1                         NaN              NaN   \n",
       " 2              SDR21C2                         NaN              NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Dehydrogenase/reductase SDR family member 4 (E...  ...           NaN   \n",
       " 1  Carbonyl reductase [NADPH] 1 (EC 1.1.1.184) (1...  ...           NaN   \n",
       " 2  Carbonyl reductase [NADPH] 3 (EC 1.1.1.184) (N...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 92; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 2  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN                           3O4R;  \n",
       " 1             NaN  1WMA;2PFG;3BHI;3BHJ;3BHM;4Z3D;  \n",
       " 2             NaN                           2HRB;  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P19923  KPPR2_CUPNH     cfxP H16_B1389                 cfxP   \n",
       " 1  P19924  KPPR1_CUPNH  cfxP cbbPP PHG421                 cfxP   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                  H16_B1389               NaN   \n",
       " 1                cbbPP                     PHG421               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Cupriavidus necator (strain ATCC 17699 / DSM 4...         381666   \n",
       " 1  Cupriavidus necator (strain ATCC 17699 / DSM 4...         381666   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphoribulokinase, chromosomal (PRK) (PRKase...  ...           NaN   \n",
       " 1  Phosphoribulokinase, plasmid (PRK) (PRKase) (E...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "   Signal peptide Transit peptide  PDB  \n",
       " 0            NaN             NaN  NaN  \n",
       " 1            NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  Q9AEM1  PCKG_CORGL  pckG pck Cgl2863 cg3169                 pckG   \n",
       " 1  A4QHQ4  PCKG_CORGB            pckG cgR_2751                 pckG   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  pck             Cgl2863 cg3169               NaN   \n",
       " 1                  NaN                   cgR_2751               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Corynebacterium glutamicum (strain ATCC 13032 ...         196627   \n",
       " 1              Corynebacterium glutamicum (strain R)         340322   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphoenolpyruvate carboxykinase [GTP] (PEP c...  ...           NaN   \n",
       " 1  Phosphoenolpyruvate carboxykinase [GTP] (PEP c...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2ZCI;  \n",
       " 1             NaN    NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P35574  GDE_RABIT        AGL                  AGL                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Oryctolagus cuniculus (Rabbit)           9986   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glycogen debranching enzyme (Glycogen debranch...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 87; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: The N-terminus is blocked.; PTM: Ubiquiti...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q15126  PMVK_HUMAN  PMVK PMKI                 PMVK                 PMKI   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Phosphomevalonate kinase (PMKase) (hPMK) (EC 2...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  3CH4;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  Q96KP4  CNDP2_HUMAN  CNDP2 CN2 CPGL HEL-S-13 PEPA                CNDP2   \n",
       " \n",
       "      Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0  CN2 CPGL HEL-S-13 PEPA                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Cytosolic non-specific dipeptidase (EC 3.4.13....  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  4RUH;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q57764  PDAD_METJA  pdaD MJ0316                 pdaD                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     MJ0316               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Pyruvoyl-dependent arginine decarboxylase (Pvl...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 53; /note=\"Pyruvic acid (Ser)\"; /evide...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                        PDB  \n",
       " 0             NaN  1MT1;1N13;1N2M;2QQC;2QQD;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                          Gene Names  \\\n",
       " 0  Q12884  SEPR_HUMAN                                 FAP   \n",
       " 1  Q86TI2  DPP9_HUMAN                          DPP9 DPRP2   \n",
       " 2  P27487  DPP4_HUMAN                     DPP4 ADCP2 CD26   \n",
       " 3  Q6V1X1  DPP8_HUMAN  DPP8 DPRP1 MSTP097 MSTP135 MSTP141   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym)  Gene Names (ordered locus)  \\\n",
       " 0                  FAP                  NaN                         NaN   \n",
       " 1                 DPP9                DPRP2                         NaN   \n",
       " 2                 DPP4           ADCP2 CD26                         NaN   \n",
       " 3                 DPP8                DPRP1                         NaN   \n",
       " \n",
       "           Gene Names (ORF)              Organism  Organism (ID)  \\\n",
       " 0                      NaN  Homo sapiens (Human)           9606   \n",
       " 1                      NaN  Homo sapiens (Human)           9606   \n",
       " 2                      NaN  Homo sapiens (Human)           9606   \n",
       " 3  MSTP097 MSTP135 MSTP141  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Prolyl endopeptidase FAP (EC 3.4.21.26) (170 k...  ...   \n",
       " 1  Dipeptidyl peptidase 9 (DP9) (EC 3.4.14.5) (Di...  ...   \n",
       " 2  Dipeptidyl peptidase 4 (EC 3.4.14.5) (ADABP) (...  ...   \n",
       " 3  Dipeptidyl peptidase 8 (DP8) (EC 3.4.14.5) (Di...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0  CARBOHYD 49; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 1                                                NaN   \n",
       " 2  CARBOHYD 85; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 3                                                NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated. {ECO:0000269|PubMed:15809...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: The soluble form (Dipeptidyl peptidase 4 ...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                         1Z68;6Y0F;  \n",
       " 1           6EOQ;6EOR;6QZV;6X6A;6X6C;7A3F;7JKQ;7JN7;  \n",
       " 2  1J2E;1N1M;1NU6;1NU8;1PFQ;1R9M;1R9N;1RWQ;1TK3;1...  \n",
       " 3  6EOO;6EOP;6EOS;6EOT;6HP8;6QZW;6TRW;6TRX;7A3G;7...  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9NVS9  PNPO_HUMAN       PNPO                 PNPO                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Pyridoxine-5'-phosphate oxidase (EC 1.4.3.5) (...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 238; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  1NRG;3HY8;6H00;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  Q8BWM0  PGES2_MOUSE    Ptges2 Gbf1 Pges2               Ptges2   \n",
       " 1  Q9JM51  PTGES_MOUSE           Ptges Pges                Ptges   \n",
       " 2  Q9R0Q7   TEBP_MOUSE  Ptges3 Sid3177 Tebp               Ptges3   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           Gbf1 Pges2                         NaN               NaN   \n",
       " 1                 Pges                         NaN               NaN   \n",
       " 2         Sid3177 Tebp                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " 2  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Prostaglandin E synthase 2 (EC 5.3.99.3) (GATE...  ...           NaN   \n",
       " 1  Prostaglandin E synthase (mPGES-1) (EC 5.3.99....  ...           NaN   \n",
       " 2  Prostaglandin E synthase 3 (EC 5.3.99.3) (Cyto...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2  MOD_RES 33; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Synthesized as a Golgi membrane-associate...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: Proteolytically cleaved by caspase-7 (CAS...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry   Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0   P0AEE8    DMA_ECOLI       dam b3387 JW3350                  dam   \n",
       " 1   P08957   T1MK_ECOLI  hsdM hsm b4349 JW4312                 hsdM   \n",
       " 2   P10484   T1M1_ECOLX               hsdM hsm                 hsdM   \n",
       " 3   P0DUF9   PGLX_ECOHS   pglX brxX EcHS_A0339                 pglX   \n",
       " 4   P28638   YHDJ_ECOLI      yhdJ b3262 JW5543                 yhdJ   \n",
       " 5   P25240   MT57_ECOLX       eco57IBM eco57IM             eco57IBM   \n",
       " 6   P12364   T3MO_ECOLX         mod ecoP15Imod                  mod   \n",
       " 7   P00472   MTE1_ECOLX                 ecoRIM               ecoRIM   \n",
       " 8   P25239   T257_ECOLX                eco57IR              eco57IR   \n",
       " 9   P04392     DMA_BPT4                    DAM                  DAM   \n",
       " 10  Q38156     DAM_BPT1                   M.T1                 M.T1   \n",
       " 11  P04043  MTD21_STREE                   dpnM                 dpnM   \n",
       " 12  P12427     DMA_BPT2                    DAM                  DAM   \n",
       " 13  P04393   MTE5_ECOLX                 ecoRVM               ecoRVM   \n",
       " 14  Q47282   T1ME_ECOLX                   hsdM                 hsdM   \n",
       " 15  P21311   DMA7_ECOLX                    dam                  dam   \n",
       " 16  Q47163   T1MP_ECOLX                   hsdM                 hsdM   \n",
       " 17  P0AEE9    DMA_ECO57      dam Z4740 ECs4229                  dam   \n",
       " 18  P40814   T3MO_SALTY            mod STM0357                  mod   \n",
       " 19  P50190   MTM1_MICAM                  mamIM                mamIM   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN               b3387 JW3350               NaN   \n",
       " 1                   hsm               b4349 JW4312               NaN   \n",
       " 2                   hsm                        NaN               NaN   \n",
       " 3                  brxX                 EcHS_A0339               NaN   \n",
       " 4                   NaN               b3262 JW5543               NaN   \n",
       " 5               eco57IM                        NaN               NaN   \n",
       " 6            ecoP15Imod                        NaN               NaN   \n",
       " 7                   NaN                        NaN               NaN   \n",
       " 8                   NaN                        NaN               NaN   \n",
       " 9                   NaN                        NaN               NaN   \n",
       " 10                  NaN                        NaN               NaN   \n",
       " 11                  NaN                        NaN               NaN   \n",
       " 12                  NaN                        NaN               NaN   \n",
       " 13                  NaN                        NaN               NaN   \n",
       " 14                  NaN                        NaN               NaN   \n",
       " 15                  NaN                        NaN               NaN   \n",
       " 16                  NaN                        NaN               NaN   \n",
       " 17                  NaN              Z4740 ECs4229               NaN   \n",
       " 18                  NaN                    STM0357               NaN   \n",
       " 19                  NaN                        NaN               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                       Escherichia coli (strain K12)          83333   \n",
       " 2                                    Escherichia coli            562   \n",
       " 3                  Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 4                       Escherichia coli (strain K12)          83333   \n",
       " 5                                    Escherichia coli            562   \n",
       " 6                                    Escherichia coli            562   \n",
       " 7                                    Escherichia coli            562   \n",
       " 8                                    Escherichia coli            562   \n",
       " 9          Enterobacteria phage T4 (Bacteriophage T4)          10665   \n",
       " 10            Escherichia phage T1 (Bacteriophage T1)        1921008   \n",
       " 11                           Streptococcus pneumoniae           1313   \n",
       " 12         Enterobacteria phage T2 (Bacteriophage T2)          10664   \n",
       " 13                                   Escherichia coli            562   \n",
       " 14                                   Escherichia coli            562   \n",
       " 15                                   Escherichia coli            562   \n",
       " 16                                   Escherichia coli            562   \n",
       " 17                           Escherichia coli O157:H7          83334   \n",
       " 18  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 19                       Microbacterium ammoniaphilum          42460   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   DNA adenine methylase (EC 2.1.1.72) (DNA adeni...  ...           NaN   \n",
       " 1   Type I restriction enzyme EcoKI methylase subu...  ...           NaN   \n",
       " 2   Type I restriction enzyme EcoR124II methylase ...  ...           NaN   \n",
       " 3   Adenine-specific methyltransferase BrxX (EC 2....  ...           NaN   \n",
       " 4   DNA adenine methyltransferase YhdJ (EC 2.1.1.7...  ...           NaN   \n",
       " 5   Type II methyltransferase M.Eco57I (M.Eco57I) ...  ...           NaN   \n",
       " 6   Type III restriction-modification enzyme EcoP1...  ...           NaN   \n",
       " 7   Type II methyltransferase M.EcoRI (M.EcoRI) (E...  ...           NaN   \n",
       " 8   Type II restriction enzyme and methyltransfera...  ...           NaN   \n",
       " 9   DNA adenine methylase (EC 2.1.1.72) (DNA-(N(6)...  ...           NaN   \n",
       " 10  DNA N-6-adenine-methyltransferase (DAM) (EC 2....  ...           NaN   \n",
       " 11  Type II methyltransferase M1.DpnII (M1.DpnII) ...  ...           NaN   \n",
       " 12  DNA adenine methylase (EC 2.1.1.72) (DNA-(N(6)...  ...           NaN   \n",
       " 13  Type II methyltransferase M.EcoRV (M.EcoRV) (E...  ...           NaN   \n",
       " 14  Type I restriction enzyme EcoEI methylase subu...  ...           NaN   \n",
       " 15  Retron Ec67 DNA adenine methylase (EC 2.1.1.72...  ...           NaN   \n",
       " 16  Type I restriction enzyme EcoprrI methylase su...  ...           NaN   \n",
       " 17  DNA adenine methylase (EC 2.1.1.72) (DNA adeni...  ...           NaN   \n",
       " 18  Type III restriction-modification enzyme StyLT...  ...           NaN   \n",
       " 19  Type II methyltransferase M.MamI (M.MamI) (EC ...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine Lipidation  \\\n",
       " 0                                                 NaN        NaN   \n",
       " 1                                                 NaN        NaN   \n",
       " 2                                                 NaN        NaN   \n",
       " 3                                                 NaN        NaN   \n",
       " 4                                                 NaN        NaN   \n",
       " 5                                                 NaN        NaN   \n",
       " 6                                                 NaN        NaN   \n",
       " 7   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...        NaN   \n",
       " 8                                                 NaN        NaN   \n",
       " 9                                                 NaN        NaN   \n",
       " 10                                                NaN        NaN   \n",
       " 11                                                NaN        NaN   \n",
       " 12                                                NaN        NaN   \n",
       " 13                                                NaN        NaN   \n",
       " 14                                                NaN        NaN   \n",
       " 15                                                NaN        NaN   \n",
       " 16                                                NaN        NaN   \n",
       " 17                                                NaN        NaN   \n",
       " 18                                                NaN        NaN   \n",
       " 19                                                NaN        NaN   \n",
       " \n",
       "     Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0                NaN     NaN                              NaN         NaN   \n",
       " 1                NaN     NaN                              NaN         NaN   \n",
       " 2                NaN     NaN                              NaN         NaN   \n",
       " 3                NaN     NaN                              NaN         NaN   \n",
       " 4                NaN     NaN                              NaN         NaN   \n",
       " 5                NaN     NaN                              NaN         NaN   \n",
       " 6                NaN     NaN                              NaN         NaN   \n",
       " 7                NaN     NaN                              NaN         NaN   \n",
       " 8                NaN     NaN                              NaN         NaN   \n",
       " 9                NaN     NaN                              NaN         NaN   \n",
       " 10               NaN     NaN                              NaN         NaN   \n",
       " 11               NaN     NaN                              NaN         NaN   \n",
       " 12               NaN     NaN                              NaN         NaN   \n",
       " 13               NaN     NaN                              NaN         NaN   \n",
       " 14               NaN     NaN                              NaN         NaN   \n",
       " 15               NaN     NaN                              NaN         NaN   \n",
       " 16               NaN     NaN                              NaN         NaN   \n",
       " 17               NaN     NaN                              NaN         NaN   \n",
       " 18               NaN     NaN                              NaN         NaN   \n",
       " 19               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " 4             NaN             NaN   \n",
       " 5             NaN             NaN   \n",
       " 6             NaN             NaN   \n",
       " 7             NaN             NaN   \n",
       " 8             NaN             NaN   \n",
       " 9             NaN             NaN   \n",
       " 10            NaN             NaN   \n",
       " 11            NaN             NaN   \n",
       " 12            NaN             NaN   \n",
       " 13            NaN             NaN   \n",
       " 14            NaN             NaN   \n",
       " 15            NaN             NaN   \n",
       " 16            NaN             NaN   \n",
       " 17            NaN             NaN   \n",
       " 18            NaN             NaN   \n",
       " 19            NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0   2G1P;2ORE;4GBE;4GOL;4GOM;4GON;4GOO;4RTJ;4RTK;4...  \n",
       " 1                                     2AR0;2Y7C;2Y7H;  \n",
       " 2                           7BST;7BTO;7BTP;7BTQ;7BTR;  \n",
       " 3                                                 NaN  \n",
       " 4                                                 NaN  \n",
       " 5                                                 NaN  \n",
       " 6                                               4ZCF;  \n",
       " 7                                                 NaN  \n",
       " 8                                                 NaN  \n",
       " 9                           1Q0S;1Q0T;1YF3;1YFJ;1YFL;  \n",
       " 10                                                NaN  \n",
       " 11                                              2DPM;  \n",
       " 12                                                NaN  \n",
       " 13                                                NaN  \n",
       " 14                                                NaN  \n",
       " 15                                                NaN  \n",
       " 16                                                NaN  \n",
       " 17                                                NaN  \n",
       " 18                                                NaN  \n",
       " 19                                                NaN  \n",
       " \n",
       " [20 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P9WKK6  ACEA1_MYCTO                  icl1 MT0483                 icl1   \n",
       " 1  H8F3R6  ACEA2_MYCTE            aceAb ERDMAN_2108                aceAb   \n",
       " 2  H8EVV4  ACEA1_MYCTE  icl1 ERDMAN_0512 Q643_00485                 icl1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                     MT0483              NaN   \n",
       " 1                   NaN                ERDMAN_2108              NaN   \n",
       " 2                   NaN                ERDMAN_0512       Q643_00485   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 1  Mycobacterium tuberculosis (strain ATCC 35801 ...         652616   \n",
       " 2  Mycobacterium tuberculosis (strain ATCC 35801 ...         652616   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Isocitrate lyase 1 (ICL1) (EC 4.1.3.1) (Isocit...  ...           NaN   \n",
       " 1  Isocitrate lyase 2 (ICL2) (EC 4.1.3.1) (Isocit...  ...           NaN   \n",
       " 2  Isocitrate lyase 1 (ICL1) (EC 4.1.3.1) (Isocit...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P14568  ASSY_BOVIN       ASS1                 ASS1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Argininosuccinate synthase (EC 6.3.4.5) (Citru...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 87; /note=\"Phosphotyrosine\"; /evidence...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Acetylated by CLOCK in a circadian manner...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P09810   TPH1_RAT   Tph1 Tph                 Tph1                  Tph   \n",
       " 1  Q8CGU9   TPH2_RAT  Tph2 Ntph                 Tph2                 Ntph   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Tryptophan 5-hydroxylase 1 (EC 1.14.16.4) (Try...  ...   \n",
       " 1          10116  Tryptophan 5-hydroxylase 2 (EC 1.14.16.4) (Neu...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 58; /note=\"Phosphoserine; by PKA\"; /ev...     NaN   \n",
       " 1  MOD_RES 19; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Ubiquitinated, leading to its degradation...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  P26876  LIP_PSEAE  lip lipA PA2862                  lip   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 lipA                     PA2862               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Triacylglycerol lipase (EC 3.1.1.3) (Extracell...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide    PDB  \n",
       " 0  SIGNAL 1..26; /evidence=\"ECO:0000269|PubMed:15...             NaN  1EX9;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  Q9ZLT0  MURI_HELPJ  murI glr jhp_0496                 murI   \n",
       " 1  P56068  MURI_HELPY   murI glr HP_0549                 murI   \n",
       " 2  B5Z6S2  MURI_HELPG     murI HPG27_509                 murI   \n",
       " 3  B2UTT1  MURI_HELPS    murI HPSH_04140                 murI   \n",
       " 4  Q1CTX8  MURI_HELPH    murI HPAG1_0527                 murI   \n",
       " 5  B6JLD2  MURI_HELP2    murI HPP12_0556                 murI   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  glr                   jhp_0496               NaN   \n",
       " 1                  glr                    HP_0549               NaN   \n",
       " 2                  NaN                  HPG27_509               NaN   \n",
       " 3                  NaN                 HPSH_04140               NaN   \n",
       " 4                  NaN                 HPAG1_0527               NaN   \n",
       " 5                  NaN                 HPP12_0556               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Helicobacter pylori (strain J99 / ATCC 700824)...          85963   \n",
       " 1  Helicobacter pylori (strain ATCC 700392 / 2669...          85962   \n",
       " 2                   Helicobacter pylori (strain G27)         563041   \n",
       " 3                Helicobacter pylori (strain Shi470)         512562   \n",
       " 4                 Helicobacter pylori (strain HPAG1)         357544   \n",
       " 5                   Helicobacter pylori (strain P12)         570508   \n",
       " \n",
       "                      Protein names  ... Glycosylation Initiator methionine  \\\n",
       " 0  Glutamate racemase (EC 5.1.1.3)  ...           NaN                  NaN   \n",
       " 1  Glutamate racemase (EC 5.1.1.3)  ...           NaN                  NaN   \n",
       " 2  Glutamate racemase (EC 5.1.1.3)  ...           NaN                  NaN   \n",
       " 3  Glutamate racemase (EC 5.1.1.3)  ...           NaN                  NaN   \n",
       " 4  Glutamate racemase (EC 5.1.1.3)  ...           NaN                  NaN   \n",
       " 5  Glutamate racemase (EC 5.1.1.3)  ...           NaN                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " 2         NaN               NaN     NaN                              NaN   \n",
       " 3         NaN               NaN     NaN                              NaN   \n",
       " 4         NaN               NaN     NaN                              NaN   \n",
       " 5         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide                        PDB  \n",
       " 0         NaN             NaN             NaN  2JFX;2JFY;2JFZ;2W4I;4B1F;  \n",
       " 1         NaN             NaN             NaN                        NaN  \n",
       " 2         NaN             NaN             NaN                        NaN  \n",
       " 3         NaN             NaN             NaN                        NaN  \n",
       " 4         NaN             NaN             NaN                        NaN  \n",
       " 5         NaN             NaN             NaN                        NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P81072  AMPD1_RABIT      AMPD1                AMPD1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Oryctolagus cuniculus (Rabbit)           9986   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  AMP deaminase 1 (EC 3.5.4.6) (AMP deaminase is...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation Modified residue Peptide  \\\n",
       " 0                  NaN         NaN              NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O88496   VKGC_RAT       Ggcx                 Ggcx                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Vitamin K-dependent gamma-carboxylase (EC 4.1....  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0  O59812  G6PD2_SCHPO      gcd1 zwf2 SPCC794.01c                 gcd1   \n",
       " 1  O00091   G6PD_SCHPO  zwf1 SPAC3A12.18 SPAC9.01                 zwf1   \n",
       " 2  O14137  G6PD3_SCHPO                SPAC3C7.13c                  NaN   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)      Gene Names (ORF)  \\\n",
       " 0                 zwf2                         NaN           SPCC794.01c   \n",
       " 1                  NaN                         NaN  SPAC3A12.18 SPAC9.01   \n",
       " 2                  NaN                         NaN           SPAC3C7.13c   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " 1  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " 2  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glucose-6-phosphate 1-dehydrogenase gcd1 (G6PD...  ...           NaN   \n",
       " 1  Glucose-6-phosphate 1-dehydrogenase (G6PD) (EC...  ...           NaN   \n",
       " 2  Probable glucose-6-phosphate 1-dehydrogenase C...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q2KJG4  MGST2_BOVIN       MGST2                MGST2                  NaN   \n",
       " 1  Q28035  GSTA1_BOVIN       GSTA1                GSTA1                  NaN   \n",
       " 2  Q95L14  PTGES_BOVIN       PTGES                PTGES                  NaN   \n",
       " 3  F1MVX2  LANC1_BOVIN      LANCL1               LANCL1                  NaN   \n",
       " 4  O18879  GSTA2_BOVIN       GSTA2                GSTA2                  NaN   \n",
       " 5  P28801  GSTP1_BOVIN       GSTP1                GSTP1                  NaN   \n",
       " 6  Q9N0V4  GSTM1_BOVIN  GSTM1 GSTM                GSTM1                 GSTM   \n",
       " 7  Q2NL00  GSTT1_BOVIN       GSTT1                GSTT1                  NaN   \n",
       " 8  Q64L89  MGST1_BOVIN       MGST1                MGST1                  NaN   \n",
       " 9  Q5E9G0  GSTA4_BOVIN       GSTA4                GSTA4                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 1                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 2                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 3                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 4                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 5                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 6                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 7                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 8                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 9                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Microsomal glutathione S-transferase 2 (Micros...  ...   \n",
       " 1           9913  Glutathione S-transferase A1 (EC 2.5.1.18) (13...  ...   \n",
       " 2           9913  Prostaglandin E synthase (EC 5.3.99.3) (Glutat...  ...   \n",
       " 3           9913  Glutathione S-transferase LANCL1 (EC 2.5.1.18)...  ...   \n",
       " 4           9913  Glutathione S-transferase A2 (EC 2.5.1.18) (GS...  ...   \n",
       " 5           9913  Glutathione S-transferase P (EC 2.5.1.18) (GST...  ...   \n",
       " 6           9913  Glutathione S-transferase Mu 1 (EC 2.5.1.18) (...  ...   \n",
       " 7           9913  Glutathione S-transferase theta-1 (EC 2.5.1.18...  ...   \n",
       " 8           9913  Microsomal glutathione S-transferase 1 (Micros...  ...   \n",
       " 9           9913  Glutathione S-transferase A4 (EC 2.5.1.18) (GS...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN                                                NaN   \n",
       " 1           NaN  INIT_MET 1; /note=\"Removed; alternate\"; /evide...   \n",
       " 2           NaN                                                NaN   \n",
       " 3           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 4           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 5           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 6           NaN                                                NaN   \n",
       " 7           NaN                                                NaN   \n",
       " 8           NaN                                                NaN   \n",
       " 9           NaN                                                NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN                                                NaN     NaN   \n",
       " 1         NaN  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " 3         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 4         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 5         NaN  MOD_RES 4; /note=\"Phosphotyrosine; by EGFR\"; /...     NaN   \n",
       " 6         NaN                                                NaN     NaN   \n",
       " 7         NaN                                                NaN     NaN   \n",
       " 8         NaN  MOD_RES 42; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 9         NaN  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " 7                              NaN         NaN             NaN   \n",
       " 8                              NaN         NaN             NaN   \n",
       " 9                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " 5             NaN  NaN  \n",
       " 6             NaN  NaN  \n",
       " 7             NaN  NaN  \n",
       " 8             NaN  NaN  \n",
       " 9             NaN  NaN  \n",
       " \n",
       " [10 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  A1B9D5  SYC_PARDP  cysS Pden_4063                 cysS   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                  Pden_4063               NaN   \n",
       " \n",
       "                                     Organism  Organism (ID)  \\\n",
       " 0  Paracoccus denitrificans (strain Pd 1222)         318586   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P50216  IDH_CORGL  icd Cgl0664 cg0766                  icd   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN             Cgl0664 cg0766               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Corynebacterium glutamicum (strain ATCC 13032 ...         196627   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Isocitrate dehydrogenase [NADP] (IDH) (EC 1.1....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  2B0T;3MBC;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P00592  PA21B_PIG    PLA2G1B              PLA2G1B                   NaN   \n",
       " 1  Q9TSX9  PRDX6_PIG      PRDX6                PRDX6                   NaN   \n",
       " 2  P04416   PA22_PIG        NaN                  NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Sus scrofa (Pig)   \n",
       " 1                         NaN               NaN  Sus scrofa (Pig)   \n",
       " 2                         NaN               NaN  Sus scrofa (Pig)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9823  Phospholipase A2, major isoenzyme (EC 3.1.1.4)...  ...   \n",
       " 1           9823  Peroxiredoxin-6 (EC 1.11.1.27) (1-Cys peroxire...  ...   \n",
       " 2           9823  Phospholipase A2, minor isoenzyme (EC 3.1.1.4)...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  \\\n",
       " 0           NaN                  NaN   \n",
       " 1           NaN                  NaN   \n",
       " 2           NaN                  NaN   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0  LIPID 78; /note=\"N6-palmitoyl lysine\"; /eviden...   \n",
       " 1                                                NaN   \n",
       " 2                                                NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 16; /note=\"Pyrrolidone carboxylic acid...     NaN   \n",
       " 1  MOD_RES 44; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: Acylation causes dimerization. {ECO:00002...   \n",
       " 1  PTM: Irreversibly inactivated by overoxidation...   \n",
       " 2                                                NaN   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 16..22; /note=\"Activation peptide\"; /id...   \n",
       " 1                                                NaN   \n",
       " 2                                                NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..15; /evidence=\"ECO:0000269|PubMed:55...             NaN   \n",
       " 1                                                NaN             NaN   \n",
       " 2                                                NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1FX9;1FXF;1HN4;1L8S;1P2P;1PIR;1PIS;1SFV;1SFW;1...  \n",
       " 1                                                NaN  \n",
       " 2                                                NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  P50998  AMPD_SCHPO  ada1 SPBC106.04                 ada1   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN       SPBC106.04   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  AMP deaminase (EC 3.5.4.6) (Myoadenylate deami...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 79; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0   P07913  TDH_ECOLI             tdh b3616 JW3591                  tdh   \n",
       " 1   B6I3J6  TDH_ECOSE                tdh ECSE_3899                  tdh   \n",
       " 2   Q8XEJ1  TDH_ECO57            tdh Z5043 ECs4494                  tdh   \n",
       " 3   B1X950  TDH_ECODH             tdh ECDH10B_3798                  tdh   \n",
       " 4   B7M4A3  TDH_ECO8A              tdh ECIAI1_3789                  tdh   \n",
       " 5   B7N1S0  TDH_ECO81               tdh ECED1_4302                  tdh   \n",
       " 6   Q1R4X4  TDH_ECOUT              tdh UTI89_C4162                  tdh   \n",
       " 7   A1AHF3  TDH_ECOK1  tdh Ecok1_35990 APECO1_2839                  tdh   \n",
       " 8   B7L742  TDH_ECO55             tdh EC55989_4083                  tdh   \n",
       " 9   C4ZXK8  TDH_ECOBW                 tdh BWG_3307                  tdh   \n",
       " 10  A8A681  TDH_ECOHS               tdh EcHS_A3828                  tdh   \n",
       " 11  A7ZTH0  TDH_ECO24           tdh EcE24377A_4120                  tdh   \n",
       " 12  B5YWB7  TDH_ECO5E            tdh ECH74115_4989                  tdh   \n",
       " 13  B1LK55  TDH_ECOSM             tdh EcSMS35_3953                  tdh   \n",
       " 14  B7NPC5  TDH_ECO7I             tdh ECIAI39_4137                  tdh   \n",
       " 15  B7MFI0  TDH_ECO45               tdh ECS88_4033                  tdh   \n",
       " 16  B7ULH2  TDH_ECO27              tdh E2348C_3865                  tdh   \n",
       " 17  B7NES3  TDH_ECOLU               tdh ECUMN_4133                  tdh   \n",
       " 18  Q0TBJ0  TDH_ECOL5                 tdh ECP_3717                  tdh   \n",
       " 19  Q8FCA2  TDH_ECOL6                    tdh c4443                  tdh   \n",
       " 20  B1IZH4  TDH_ECOLC               tdh EcolC_0092                  tdh   \n",
       " 21  B7LVH6  TDH_ESCF3                tdh EFER_3906                  tdh   \n",
       " 22  P59409  TDH_SHIFL             tdh SF3656 S4111                  tdh   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN               b3616 JW3591              NaN   \n",
       " 1                    NaN                  ECSE_3899              NaN   \n",
       " 2                    NaN              Z5043 ECs4494              NaN   \n",
       " 3                    NaN               ECDH10B_3798              NaN   \n",
       " 4                    NaN                ECIAI1_3789              NaN   \n",
       " 5                    NaN                 ECED1_4302              NaN   \n",
       " 6                    NaN                UTI89_C4162              NaN   \n",
       " 7                    NaN                Ecok1_35990      APECO1_2839   \n",
       " 8                    NaN               EC55989_4083              NaN   \n",
       " 9                    NaN                   BWG_3307              NaN   \n",
       " 10                   NaN                 EcHS_A3828              NaN   \n",
       " 11                   NaN             EcE24377A_4120              NaN   \n",
       " 12                   NaN              ECH74115_4989              NaN   \n",
       " 13                   NaN               EcSMS35_3953              NaN   \n",
       " 14                   NaN               ECIAI39_4137              NaN   \n",
       " 15                   NaN                 ECS88_4033              NaN   \n",
       " 16                   NaN                E2348C_3865              NaN   \n",
       " 17                   NaN                 ECUMN_4133              NaN   \n",
       " 18                   NaN                   ECP_3717              NaN   \n",
       " 19                   NaN                      c4443              NaN   \n",
       " 20                   NaN                 EcolC_0092              NaN   \n",
       " 21                   NaN                  EFER_3906              NaN   \n",
       " 22                   NaN               SF3656 S4111              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain SE11)         409438   \n",
       " 2                            Escherichia coli O157:H7          83334   \n",
       " 3               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 4                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 5                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 6              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 7                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 8              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 9     Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 10                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 11  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 12    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 13          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 14      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 15       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 16  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 17  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 18    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 19  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 20  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 21  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 22                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.10...  ...           NaN   \n",
       " 1    L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 2    L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 3    L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 4    L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 5    L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 6    L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 7    L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 8    L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 9    L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 10   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 11   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 12   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 13   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 14   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 15   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 16   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 17   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 18   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 19   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 20   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 21   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " 22   L-threonine 3-dehydrogenase (TDH) (EC 1.1.1.103)  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                   NaN         NaN               NaN     NaN   \n",
       " 1                   NaN         NaN               NaN     NaN   \n",
       " 2                   NaN         NaN               NaN     NaN   \n",
       " 3                   NaN         NaN               NaN     NaN   \n",
       " 4                   NaN         NaN               NaN     NaN   \n",
       " 5                   NaN         NaN               NaN     NaN   \n",
       " 6                   NaN         NaN               NaN     NaN   \n",
       " 7                   NaN         NaN               NaN     NaN   \n",
       " 8                   NaN         NaN               NaN     NaN   \n",
       " 9                   NaN         NaN               NaN     NaN   \n",
       " 10                  NaN         NaN               NaN     NaN   \n",
       " 11                  NaN         NaN               NaN     NaN   \n",
       " 12                  NaN         NaN               NaN     NaN   \n",
       " 13                  NaN         NaN               NaN     NaN   \n",
       " 14                  NaN         NaN               NaN     NaN   \n",
       " 15                  NaN         NaN               NaN     NaN   \n",
       " 16                  NaN         NaN               NaN     NaN   \n",
       " 17                  NaN         NaN               NaN     NaN   \n",
       " 18                  NaN         NaN               NaN     NaN   \n",
       " 19                  NaN         NaN               NaN     NaN   \n",
       " 20                  NaN         NaN               NaN     NaN   \n",
       " 21                  NaN         NaN               NaN     NaN   \n",
       " 22                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " 21                              NaN         NaN             NaN   \n",
       " 22                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide  PDB  \n",
       " 0              NaN  NaN  \n",
       " 1              NaN  NaN  \n",
       " 2              NaN  NaN  \n",
       " 3              NaN  NaN  \n",
       " 4              NaN  NaN  \n",
       " 5              NaN  NaN  \n",
       " 6              NaN  NaN  \n",
       " 7              NaN  NaN  \n",
       " 8              NaN  NaN  \n",
       " 9              NaN  NaN  \n",
       " 10             NaN  NaN  \n",
       " 11             NaN  NaN  \n",
       " 12             NaN  NaN  \n",
       " 13             NaN  NaN  \n",
       " 14             NaN  NaN  \n",
       " 15             NaN  NaN  \n",
       " 16             NaN  NaN  \n",
       " 17             NaN  NaN  \n",
       " 18             NaN  NaN  \n",
       " 19             NaN  NaN  \n",
       " 20             NaN  NaN  \n",
       " 21             NaN  NaN  \n",
       " 22             NaN  NaN  \n",
       " \n",
       " [23 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O33799  HPRT_SALTY  hpt STM0170                  hpt                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                    STM0170               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Hypoxanthine phosphoribosyltransferase (HPRT) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1J7J;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q48677  PEPA_LACLM  pepA llmg_0403                 pepA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                  llmg_0403               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Lactococcus lactis subsp. cremoris (strain MG1...         416870   \n",
       " \n",
       "                            Protein names  ... Glycosylation  \\\n",
       " 0  Glutamyl aminopeptidase (EC 3.4.11.7)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P53557  BIOB_BACSU          bioB BSU30200                 bioB   \n",
       " 1  Q8KZM7  BIOB_BACNA                   bioB                 bioB   \n",
       " 2  Q70JZ1  BIOB_BACAM                   bioB                 bioB   \n",
       " 3  A7Z5B2  BIOB_BACVZ       bioB RBAM_018250                 bioB   \n",
       " 4  Q9KC26  BIOB_HALH5            bioB BH1748                 bioB   \n",
       " 5  Q65MK9  BIOB_BACLD  bioB BLi00770 BL00956                 bioB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   BSU30200               NaN   \n",
       " 1                   NaN                        NaN               NaN   \n",
       " 2                   NaN                        NaN               NaN   \n",
       " 3                   NaN                RBAM_018250               NaN   \n",
       " 4                   NaN                     BH1748               NaN   \n",
       " 5                   NaN           BLi00770 BL00956               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                     Bacillus subtilis (strain 168)         224308   \n",
       " 1                     Bacillus subtilis subsp. natto          86029   \n",
       " 2   Bacillus amyloliquefaciens (Bacillus velezensis)           1390   \n",
       " 3  Bacillus velezensis (strain DSM 23117 / BGSC 1...         326423   \n",
       " 4  Halalkalibacterium halodurans (strain ATCC BAA...         272558   \n",
       " 5  Bacillus licheniformis (strain ATCC 14580 / DS...         279010   \n",
       " \n",
       "                   Protein names  ... Glycosylation Initiator methionine  \\\n",
       " 0  Biotin synthase (EC 2.8.1.6)  ...           NaN                  NaN   \n",
       " 1  Biotin synthase (EC 2.8.1.6)  ...           NaN                  NaN   \n",
       " 2  Biotin synthase (EC 2.8.1.6)  ...           NaN                  NaN   \n",
       " 3  Biotin synthase (EC 2.8.1.6)  ...           NaN                  NaN   \n",
       " 4  Biotin synthase (EC 2.8.1.6)  ...           NaN                  NaN   \n",
       " 5  Biotin synthase (EC 2.8.1.6)  ...           NaN                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " 2         NaN               NaN     NaN                              NaN   \n",
       " 3         NaN               NaN     NaN                              NaN   \n",
       " 4         NaN               NaN     NaN                              NaN   \n",
       " 5         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide  PDB  \n",
       " 0         NaN             NaN             NaN  NaN  \n",
       " 1         NaN             NaN             NaN  NaN  \n",
       " 2         NaN             NaN             NaN  NaN  \n",
       " 3         NaN             NaN             NaN  NaN  \n",
       " 4         NaN             NaN             NaN  NaN  \n",
       " 5         NaN             NaN             NaN  NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  C7ASJ5  BGAL2_ARTSP        NaN                  NaN                   NaN   \n",
       " 1  Q44233  BGAL1_ARTSP        NaN                  NaN                   NaN   \n",
       " 2  Q59140   BGAL_ARTSB       lacZ                 lacZ                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                      Organism  \\\n",
       " 0                         NaN               NaN               Arthrobacter sp   \n",
       " 1                         NaN               NaN               Arthrobacter sp   \n",
       " 2                         NaN               NaN  Arthrobacter sp. (strain B7)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           1667        Beta-galactosidase (Beta-gal) (EC 3.2.1.23)  ...   \n",
       " 1           1667        Beta-galactosidase (Beta-gal) (EC 3.2.1.23)  ...   \n",
       " 2          86041  Beta-galactosidase (Beta-gal) (EC 3.2.1.23) (L...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " 1            NaN                  NaN         NaN               NaN     NaN   \n",
       " 2            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  Q60034  DYR_THEMA  folA dyrA TM_1641                 folA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 dyrA                    TM_1641               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Thermotoga maritima (strain ATCC 43589 / DSM 3...         243274   \n",
       " \n",
       "                                  Protein names  ... Glycosylation  \\\n",
       " 0  Dihydrofolate reductase (DHFR) (EC 1.5.1.3)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1CZ3;1D1G;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  A4QEY0  DAPB_CORGB        dapB cgR_1802                 dapB   \n",
       " 1  P40110  DAPB_CORGL  dapB Cgl1973 cg2163                 dapB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   cgR_1802               NaN   \n",
       " 1                   NaN             Cgl1973 cg2163               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0              Corynebacterium glutamicum (strain R)         340322   \n",
       " 1  Corynebacterium glutamicum (strain ATCC 13032 ...         196627   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  4-hydroxy-tetrahydrodipicolinate reductase (HT...  ...           NaN   \n",
       " 1  4-hydroxy-tetrahydrodipicolinate reductase (HT...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN         NaN  \n",
       " 1             NaN  5EER;5EES;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q3MHG7  URIC_BOVIN        UOX                  UOX                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                         Protein names  ... Glycosylation  \\\n",
       " 0           9913  Uricase (EC 1.7.3.3) (Urate oxidase)  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0  P38067  UGA2_YEAST  UGA2 UGA5 YBR006W YBR0112                 UGA2   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 UGA5                    YBR006W          YBR0112   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Succinate-semialdehyde dehydrogenase [NADP(+)]...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P06208  LEU1_YEAST     LEU4 YNL104C N2173                 LEU4   \n",
       " 1  Q12166  LEU9_YEAST  LEU9 YOR108W YOR3227w                 LEU9   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YNL104C            N2173   \n",
       " 1                   NaN                    YOR108W         YOR3227w   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  2-isopropylmalate synthase (EC 2.3.3.13) (Alph...  ...           NaN   \n",
       " 1  2-isopropylmalate synthase 2, mitochondrial (E...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0                                                NaN  NaN  \n",
       " 1  TRANSIT 1..50; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P22768  ASSY_YEAST  ARG1 YOL058W O1228                 ARG1   \n",
       " 1  P0A6E4  ASSY_ECOLI   argG b3172 JW3140                 argG   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YOL058W            O1228   \n",
       " 1                   NaN               b3172 JW3140              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1                      Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Argininosuccinate synthase (EC 6.3.4.5) (Citru...  ...           NaN   \n",
       " 1  Argininosuccinate synthase (EC 6.3.4.5) (Citru...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                   PDB  \n",
       " 0             NaN             NaN                   NaN  \n",
       " 1             NaN             NaN  1K92;1K97;1KP2;1KP3;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name   Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q9NPI6  DCP1A_HUMAN   DCP1A SMIF                DCP1A                 SMIF   \n",
       " 1  Q8IZD4  DCP1B_HUMAN        DCP1B                DCP1B                  NaN   \n",
       " 2  Q8IU60   DCP2_HUMAN  DCP2 NUDT20                 DCP2               NUDT20   \n",
       " 3  Q96DE0  NUD16_HUMAN       NUDT16               NUDT16                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " 2                         NaN               NaN  Homo sapiens (Human)   \n",
       " 3                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  mRNA-decapping enzyme 1A (EC 3.6.1.62) (Smad4-...  ...   \n",
       " 1           9606             mRNA-decapping enzyme 1B (EC 3.6.1.62)  ...   \n",
       " 2           9606  m7GpppN-mRNA hydrolase (EC 3.6.1.62) (Nucleosi...  ...   \n",
       " 3           9606  U8 snoRNA-decapping enzyme (EC 3.6.1.62) (IDP ...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN                                                NaN   \n",
       " 1           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 2           NaN                                                NaN   \n",
       " 3           NaN                                                NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 62; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2         NaN  MOD_RES 246; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 3         NaN                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: Phosphorylated at ser-249 in a MTOR-depen...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                         2WX3;4B6H;  \n",
       " 1                                                NaN  \n",
       " 2  5MP0;5QOH;5QOI;5QOJ;5QOK;5QOL;5QOM;5QON;5QOO;5...  \n",
       " 3  2XSQ;3COU;3MGM;5VY2;5W6X;5W6Z;5WJI;6B09;6CO2;6...  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P80457  XDH_BOVIN        XDH                  XDH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Xanthine dehydrogenase/oxidase [Includes: Xant...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Subject to partial proteolysis; this alte...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1FIQ;1FO4;1N5X;1V97;1VDV;3AM9;3AMZ;3AX7;3AX9;3...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P09342  ILVB1_TOBAC   ALS SURA             ALS SURA                   NaN   \n",
       " 1  P09114  ILVB2_TOBAC   ALS SURB             ALS SURB                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " \n",
       "                              Organism  Organism (ID)  \\\n",
       " 0  Nicotiana tabacum (Common tobacco)           4097   \n",
       " 1  Nicotiana tabacum (Common tobacco)           4097   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Acetolactate synthase 1, chloroplastic (EC 2.2...  ...           NaN   \n",
       " 1  Acetolactate synthase 2, chloroplastic (EC 2.2...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                       Transit peptide  PDB  \n",
       " 0  TRANSIT 1..94; /note=\"Chloroplast\"  NaN  \n",
       " 1  TRANSIT 1..91; /note=\"Chloroplast\"  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  P00830  ATPB_YEAST            ATP2 YJR121W J2041                 ATP2   \n",
       " 1  P17255  VATA_YEAST  VMA1 CLS8 TFP1 YDL185W D1286                 VMA1   \n",
       " 2  Q9UVJ8  VATA_ASHGO                  VMA1 ADR102W                 VMA1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                  NaN                    YJR121W            J2041   \n",
       " 1            CLS8 TFP1                    YDL185W            D1286   \n",
       " 2                  NaN                    ADR102W              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2  Ashbya gossypii (strain ATCC 10895 / CBS 109.5...         284811   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  ATP synthase subunit beta, mitochondrial (EC 7...  ...           NaN   \n",
       " 1  V-type proton ATPase catalytic subunit A (V-AT...  ...           NaN   \n",
       " 2  V-type proton ATPase catalytic subunit A (V-AT...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 112; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: This protein undergoes a protein self spl...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide                       Transit peptide  \\\n",
       " 0             NaN  TRANSIT 1..33; /note=\"Mitochondrion\"   \n",
       " 1             NaN                                   NaN   \n",
       " 2             NaN                                   NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  2HLD;2WPD;2XOK;3FKS;3OE7;3OEE;3OEH;3OFN;3ZIA;3...  \n",
       " 1  1DFA;1EF0;1GPP;1JVA;1LWS;1LWT;1UM2;1VDE;3J9T;3...  \n",
       " 2                                                NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q75UV1  NDX1_THETH       ndx1                 ndx1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Thermus thermophilus   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0            274  Diadenosine hexaphosphate hydrolase (Ap6A hydr...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  1VC8;1VC9;1VCD;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P80405  MTHFS_RABIT      MTHFS                MTHFS                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Oryctolagus cuniculus (Rabbit)           9986   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  5-formyltetrahydrofolate cyclo-ligase (EC 6.3....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P81140   GCDH_PIG       GCDH                 GCDH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Sus scrofa (Pig)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9823  Glutaryl-CoA dehydrogenase, mitochondrial (GCD...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 210; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT <1..13; /note=\"Mitochondrion\"; /eviden...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P0C559   GYRB_MYCSM                         gyrB                 gyrB   \n",
       " 1  A0QNE0   GYRB_MYCS2   gyrB MSMEG_0005 MSMEI_0007                 gyrB   \n",
       " 2  Q59556   GYRA_MYCSM                         gyrA                 gyrA   \n",
       " 3  A0QPN2  TOPON_MYCS2  topoN MSMEG_0457 MSMEI_0444                topoN   \n",
       " 4  P48354   GYRA_MYCS2   gyrA MSMEG_0006 MSMEI_0008                 gyrA   \n",
       " 5  A0QPN1  TOPOM_MYCS2  topoM MSMEG_0456 MSMEI_0443                topoM   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                        NaN               NaN   \n",
       " 1                   NaN      MSMEG_0005 MSMEI_0007               NaN   \n",
       " 2                   NaN                        NaN               NaN   \n",
       " 3                   NaN      MSMEG_0457 MSMEI_0444               NaN   \n",
       " 4                   NaN      MSMEG_0006 MSMEI_0008               NaN   \n",
       " 5                   NaN      MSMEG_0456 MSMEI_0443               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycolicibacterium smegmatis (Mycobacterium sme...           1772   \n",
       " 1  Mycolicibacterium smegmatis (strain ATCC 70008...         246196   \n",
       " 2  Mycolicibacterium smegmatis (Mycobacterium sme...           1772   \n",
       " 3  Mycolicibacterium smegmatis (strain ATCC 70008...         246196   \n",
       " 4  Mycolicibacterium smegmatis (strain ATCC 70008...         246196   \n",
       " 5  Mycolicibacterium smegmatis (strain ATCC 70008...         246196   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0                  DNA gyrase subunit B (EC 5.6.2.2)  ...           NaN   \n",
       " 1                  DNA gyrase subunit B (EC 5.6.2.2)  ...           NaN   \n",
       " 2                  DNA gyrase subunit A (EC 5.6.2.2)  ...           NaN   \n",
       " 3  Topoisomerase subunit TopoN (EC 5.6.2.2) (DNA ...  ...           NaN   \n",
       " 4                  DNA gyrase subunit A (EC 5.6.2.2)  ...           NaN   \n",
       " 5           Topoisomerase subunit TopoM (EC 5.6.2.2)  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " 2               NaN     NaN                              NaN         NaN   \n",
       " 3               NaN     NaN                              NaN         NaN   \n",
       " 4               NaN     NaN                              NaN         NaN   \n",
       " 5               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                   PDB  \n",
       " 0             NaN             NaN            4BAE;6Y8O;  \n",
       " 1             NaN             NaN  4B6C;4BAE;6ZT3;6ZT5;  \n",
       " 2             NaN             NaN                   NaN  \n",
       " 3             NaN             NaN                   NaN  \n",
       " 4             NaN             NaN                   NaN  \n",
       " 5             NaN             NaN                   NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9P8Q7  ACEA_CANAX       ICL1                 ICL1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                  Organism  \\\n",
       " 0                         NaN               NaN  Candida albicans (Yeast)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           5476  Isocitrate lyase (ICL) (Isocitrase) (Isocitrat...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  7EBE;7EBF;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  P11154  PYC1_YEAST      PYC1 PYV YGL062W                 PYC1   \n",
       " 1  P32327  PYC2_YEAST  PYC2 YBR218C YBR1507                 PYC2   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                  PYV                    YGL062W              NaN   \n",
       " 1                  NaN                    YBR218C          YBR1507   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Pyruvate carboxylase 1 (EC 6.4.1.1) (Pyruvic c...  ...           NaN   \n",
       " 1  Pyruvate carboxylase 2 (EC 6.4.1.1) (Pyruvic c...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 734; /note=\"N6-carboxylysine\"; /eviden...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  P14605  CYAA_SCHPO  cyr1 SPBC19C7.03                 cyr1   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN      SPBC19C7.03   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Adenylate cyclase (EC 4.6.1.1) (ATP pyrophosph...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  O73944  PCP_PYRFU              pcp PF1299                  pcp   \n",
       " 1  Q9UYQ9  PCP_PYRAB  pcp PYRAB14480 PAB1419                  pcp   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                     PF1299              NaN   \n",
       " 1                   NaN                 PYRAB14480          PAB1419   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pyrococcus furiosus (strain ATCC 43587 / DSM 3...         186497   \n",
       " 1             Pyrococcus abyssi (strain GE5 / Orsay)         272844   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Pyrrolidone-carboxylate peptidase (EC 3.4.19.3...  ...           NaN   \n",
       " 1  Pyrrolidone-carboxylate peptidase (EC 3.4.19.3...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                            PDB  \n",
       " 0             NaN  1IOF;1IOI;1X10;1X12;1Z8T;1Z8W;1Z8X;2DF5;2EO8;  \n",
       " 1             NaN                                            NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  Q07251  LDH_CUPNH  ldh H16_A0666                  ldh   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                  H16_A0666               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Cupriavidus necator (strain ATCC 17699 / DSM 4...         381666   \n",
       " \n",
       "                            Protein names  ... Glycosylation  \\\n",
       " 0  L-lactate dehydrogenase (EC 1.1.1.27)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "      Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0   A6QDA0  SBNA_STAAE       sbnA NWMN_0060                 sbnA   \n",
       " 1   Q2G1N3  SBNA_STAA8   sbnA SAOUHSC_00075                 sbnA   \n",
       " 2   Q5HJQ3  SBNA_STAAC       sbnA SACOL0100                 sbnA   \n",
       " 3   Q2FKE3  SBNA_STAA3   sbnA SAUSA300_0118                 sbnA   \n",
       " 4   D0K799  SBNA_STAAD       sbnA SAAV_0084                 sbnA   \n",
       " 5   Q7A1Z6  SBNA_STAAW          sbnA MW0089                 sbnA   \n",
       " 6   Q6GKI9  SBNA_STAAR         sbnA SAR0119                 sbnA   \n",
       " 7   Q99X99  SBNA_STAAN          sbnA SA0112                 sbnA   \n",
       " 8   Q6GD09  SBNA_STAAS         sbnA SAS0090                 sbnA   \n",
       " 9   A5INY7  SBNA_STAA9    sbnA SaurJH9_0103                 sbnA   \n",
       " 10  A6TXQ1  SBNA_STAA2    sbnA SaurJH1_0107                 sbnA   \n",
       " 11  A8YZ41  SBNA_STAAT  sbnA USA300HOU_0127                 sbnA   \n",
       " 12  D1GUM7  SBNA_STAA0    sbnA SATW20_01280                 sbnA   \n",
       " 13  D3ERF5  SBNA_STAA4     sbnA SA2981_0117                 sbnA   \n",
       " 14  A7WX68  SBNA_STAA1       sbnA SAHV_0115                 sbnA   \n",
       " 15  Q932K9  SBNA_STAAM         sbnA SAV0116                 sbnA   \n",
       " 16  Q2YUU0  SBNA_STAAB         sbnA SAB0055                 sbnA   \n",
       " 17  D2N3G7  SBNA_STAA5       sbnA SAPIG0127                 sbnA   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                    NaN                  NWMN_0060               NaN   \n",
       " 1                    NaN              SAOUHSC_00075               NaN   \n",
       " 2                    NaN                  SACOL0100               NaN   \n",
       " 3                    NaN              SAUSA300_0118               NaN   \n",
       " 4                    NaN                  SAAV_0084               NaN   \n",
       " 5                    NaN                     MW0089               NaN   \n",
       " 6                    NaN                    SAR0119               NaN   \n",
       " 7                    NaN                     SA0112               NaN   \n",
       " 8                    NaN                    SAS0090               NaN   \n",
       " 9                    NaN               SaurJH9_0103               NaN   \n",
       " 10                   NaN               SaurJH1_0107               NaN   \n",
       " 11                   NaN             USA300HOU_0127               NaN   \n",
       " 12                   NaN               SATW20_01280               NaN   \n",
       " 13                   NaN                SA2981_0117               NaN   \n",
       " 14                   NaN                  SAHV_0115               NaN   \n",
       " 15                   NaN                    SAV0116               NaN   \n",
       " 16                   NaN                    SAB0055               NaN   \n",
       " 17                   NaN                  SAPIG0127               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0               Staphylococcus aureus (strain Newman)         426430   \n",
       " 1    Staphylococcus aureus (strain NCTC 8325 / PS 47)          93061   \n",
       " 2                  Staphylococcus aureus (strain COL)          93062   \n",
       " 3               Staphylococcus aureus (strain USA300)         367830   \n",
       " 4                 Staphylococcus aureus (strain ED98)         681288   \n",
       " 5                  Staphylococcus aureus (strain MW2)         196620   \n",
       " 6              Staphylococcus aureus (strain MRSA252)         282458   \n",
       " 7                 Staphylococcus aureus (strain N315)         158879   \n",
       " 8              Staphylococcus aureus (strain MSSA476)         282459   \n",
       " 9                  Staphylococcus aureus (strain JH9)         359786   \n",
       " 10                 Staphylococcus aureus (strain JH1)         359787   \n",
       " 11    Staphylococcus aureus (strain USA300 / TCH1516)         451516   \n",
       " 12         Staphylococcus aureus (strain TW20 / 0582)         663951   \n",
       " 13            Staphylococcus aureus (strain 04-02981)         703339   \n",
       " 14   Staphylococcus aureus (strain Mu3 / ATCC 700698)         418127   \n",
       " 15  Staphylococcus aureus (strain Mu50 / ATCC 700699)         158878   \n",
       " 16  Staphylococcus aureus (strain bovine RF122 / E...         273036   \n",
       " 17  Staphylococcus aureus (strain MRSA ST398 / iso...         523796   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 1   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 2   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 3   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 4   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 5   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 6   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 7   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 8   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 9   N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 10  N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 11  N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 12  N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 13  N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 14  N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 15  N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 16  N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " 17  N-(2-amino-2-carboxyethyl)-L-glutamate synthas...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  \\\n",
       " 0                   NaN         NaN   \n",
       " 1                   NaN         NaN   \n",
       " 2                   NaN         NaN   \n",
       " 3                   NaN         NaN   \n",
       " 4                   NaN         NaN   \n",
       " 5                   NaN         NaN   \n",
       " 6                   NaN         NaN   \n",
       " 7                   NaN         NaN   \n",
       " 8                   NaN         NaN   \n",
       " 9                   NaN         NaN   \n",
       " 10                  NaN         NaN   \n",
       " 11                  NaN         NaN   \n",
       " 12                  NaN         NaN   \n",
       " 13                  NaN         NaN   \n",
       " 14                  NaN         NaN   \n",
       " 15                  NaN         NaN   \n",
       " 16                  NaN         NaN   \n",
       " 17                  NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 1   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 2   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 3   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 4   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 5   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 6   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 7   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 8   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 9   MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 10  MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 11  MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 12  MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 13  MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 14  MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 15  MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 16  MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 17  MOD_RES 47; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                   PDB  \n",
       " 0              NaN  5D84;5D85;5D86;5D87;  \n",
       " 1              NaN                   NaN  \n",
       " 2              NaN                   NaN  \n",
       " 3              NaN                   NaN  \n",
       " 4              NaN                   NaN  \n",
       " 5              NaN                   NaN  \n",
       " 6              NaN                   NaN  \n",
       " 7              NaN                   NaN  \n",
       " 8              NaN                   NaN  \n",
       " 9              NaN                   NaN  \n",
       " 10             NaN                   NaN  \n",
       " 11             NaN                   NaN  \n",
       " 12             NaN                   NaN  \n",
       " 13             NaN                   NaN  \n",
       " 14             NaN                   NaN  \n",
       " 15             NaN                   NaN  \n",
       " 16             NaN                   NaN  \n",
       " 17             NaN                   NaN  \n",
       " \n",
       " [18 rows x 110 columns],\n",
       "     Entry   Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  O02776   PARG_BOVIN                PARG                 PARG   \n",
       " 1  Q3SYV9  ADPRS_BOVIN  ADPRS ADPRHL2 ARH3                ADPRS   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1         ADPRHL2 ARH3                         NaN               NaN   \n",
       " \n",
       "               Organism  Organism (ID)  \\\n",
       " 0  Bos taurus (Bovine)           9913   \n",
       " 1  Bos taurus (Bovine)           9913   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0     Poly(ADP-ribose) glycohydrolase (EC 3.2.1.143)  ...           NaN   \n",
       " 1  ADP-ribosylhydrolase ARH3 (ADP-ribose glycohyd...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 69; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P54422   GGT_BACSU   ggt BSU18410                  ggt   \n",
       " 1  P63186   GGT_BACNA            ggt                  ggt   \n",
       " 2  O05218  YWRD_BACSU  ywrD BSU36100                 ywrD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   BSU18410               NaN   \n",
       " 1                   NaN                        NaN               NaN   \n",
       " 2                   NaN                   BSU36100               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Bacillus subtilis (strain 168)         224308   \n",
       " 1  Bacillus subtilis subsp. natto          86029   \n",
       " 2  Bacillus subtilis (strain 168)         224308   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutathione hydrolase proenzyme (EC 3.4.19.13)...  ...           NaN   \n",
       " 1  Glutathione hydrolase proenzyme (EC 3.4.19.13)...  ...           NaN   \n",
       " 2  Glutathione hydrolase-like YwrD proenzyme (EC ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: Cleaved by autocatalysis into a large and...   \n",
       " 1  PTM: Cleaved by autocatalysis into a large and...   \n",
       " 2  PTM: Cleaved by autocatalysis into a large and...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 29..35; /evidence=\"ECO:0000255\"; /id=\"P...   \n",
       " 1  PROPEP 29..35; /evidence=\"ECO:0000255\"; /id=\"P...   \n",
       " 2                                                NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..28; /evidence=\"ECO:0000255\"             NaN   \n",
       " 1  SIGNAL 1..28; /evidence=\"ECO:0000255\"             NaN   \n",
       " 2                                    NaN             NaN   \n",
       " \n",
       "                          PDB  \n",
       " 0  2V36;3A75;3WHQ;3WHR;3WHS;  \n",
       " 1                        NaN  \n",
       " 2                        NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P37610  TAUD_ECOLI  tauD ssiD yaiG b0368 JW0360                 tauD   \n",
       " 1  Q88RA3  TAUD_PSEPK            tauD atsK PP_0230                 tauD   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0            ssiD yaiG               b0368 JW0360               NaN   \n",
       " 1                 atsK                    PP_0230               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1  Pseudomonas putida (strain ATCC 47054 / DSM 61...         160488   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Alpha-ketoglutarate-dependent taurine dioxygen...  ...           NaN   \n",
       " 1  Alpha-ketoglutarate-dependent taurine dioxygen...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 128; /note=\"3-hydroxytryptophan; by au...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                        PDB  \n",
       " 0             NaN  1GQW;1GY9;1OS7;1OTJ;6EDH;  \n",
       " 1             NaN            3PVJ;3V15;3V17;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "      Entry   Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0   P19367   HXK1_HUMAN        HK1                  HK1                  NaN   \n",
       " 1   P35557   HXK4_HUMAN        GCK                  GCK                  NaN   \n",
       " 2   P52789   HXK2_HUMAN        HK2                  HK2                  NaN   \n",
       " 3   P52790   HXK3_HUMAN        HK3                  HK3                  NaN   \n",
       " 4   Q2TB90  HKDC1_HUMAN      HKDC1                HKDC1                  NaN   \n",
       " 5   P05708     HXK1_RAT        Hk1                  Hk1                  NaN   \n",
       " 6   P17710   HXK1_MOUSE        Hk1                  Hk1                  NaN   \n",
       " 7   P27595   HXK1_BOVIN        HK1                  HK1                  NaN   \n",
       " 8   P17712     HXK4_RAT        Gck                  Gck                  NaN   \n",
       " 9   P52792   HXK4_MOUSE     Gck Gk                  Gck                   Gk   \n",
       " 10  P27881     HXK2_RAT        Hk2                  Hk2                  NaN   \n",
       " 11  O08528   HXK2_MOUSE        Hk2                  Hk2                  NaN   \n",
       " 12  P27926     HXK3_RAT        Hk3                  Hk3                  NaN   \n",
       " 13  Q3TRM8   HXK3_MOUSE        Hk3                  Hk3                  NaN   \n",
       " 14  Q91W97  HKDC1_MOUSE      Hkdc1                Hkdc1                  NaN   \n",
       " 15  Q5RC71   HXK1_PONAB        HK1                  HK1                  NaN   \n",
       " 16  A2PYL6   HXK2_HORSE        HK2                  HK2                  NaN   \n",
       " 17  Q1W674     HXK2_PIG        HK2                  HK2                  NaN   \n",
       " 18  A2PYL8   HXK2_EQUGR        HK2                  HK2                  NaN   \n",
       " 19  A2PYL7   HXK2_EQUZE        HK2                  HK2                  NaN   \n",
       " \n",
       "     Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                          NaN               NaN   \n",
       " 1                          NaN               NaN   \n",
       " 2                          NaN               NaN   \n",
       " 3                          NaN               NaN   \n",
       " 4                          NaN               NaN   \n",
       " 5                          NaN               NaN   \n",
       " 6                          NaN               NaN   \n",
       " 7                          NaN               NaN   \n",
       " 8                          NaN               NaN   \n",
       " 9                          NaN               NaN   \n",
       " 10                         NaN               NaN   \n",
       " 11                         NaN               NaN   \n",
       " 12                         NaN               NaN   \n",
       " 13                         NaN               NaN   \n",
       " 14                         NaN               NaN   \n",
       " 15                         NaN               NaN   \n",
       " 16                         NaN               NaN   \n",
       " 17                         NaN               NaN   \n",
       " 18                         NaN               NaN   \n",
       " 19                         NaN               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                Homo sapiens (Human)           9606   \n",
       " 1                                Homo sapiens (Human)           9606   \n",
       " 2                                Homo sapiens (Human)           9606   \n",
       " 3                                Homo sapiens (Human)           9606   \n",
       " 4                                Homo sapiens (Human)           9606   \n",
       " 5                             Rattus norvegicus (Rat)          10116   \n",
       " 6                                Mus musculus (Mouse)          10090   \n",
       " 7                                 Bos taurus (Bovine)           9913   \n",
       " 8                             Rattus norvegicus (Rat)          10116   \n",
       " 9                                Mus musculus (Mouse)          10090   \n",
       " 10                            Rattus norvegicus (Rat)          10116   \n",
       " 11                               Mus musculus (Mouse)          10090   \n",
       " 12                            Rattus norvegicus (Rat)          10116   \n",
       " 13                               Mus musculus (Mouse)          10090   \n",
       " 14                               Mus musculus (Mouse)          10090   \n",
       " 15  Pongo abelii (Sumatran orangutan) (Pongo pygma...           9601   \n",
       " 16                             Equus caballus (Horse)           9796   \n",
       " 17                                   Sus scrofa (Pig)           9823   \n",
       " 18                       Equus grevyi (Grevy's zebra)           9792   \n",
       " 19                       Equus zebra (Mountain zebra)           9791   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Hexokinase-1 (EC 2.7.1.1) (Brain form hexokina...  ...           NaN   \n",
       " 1   Hexokinase-4 (HK4) (EC 2.7.1.1) (Glucokinase) ...  ...           NaN   \n",
       " 2   Hexokinase-2 (EC 2.7.1.1) (Hexokinase type II)...  ...           NaN   \n",
       " 3   Hexokinase-3 (EC 2.7.1.1) (Hexokinase type III...  ...           NaN   \n",
       " 4   Hexokinase HKDC1 (EC 2.7.1.1) (Hexokinase doma...  ...           NaN   \n",
       " 5   Hexokinase-1 (EC 2.7.1.1) (Brain form hexokina...  ...           NaN   \n",
       " 6   Hexokinase-1 (EC 2.7.1.1) (Hexokinase type I) ...  ...           NaN   \n",
       " 7   Hexokinase-1 (EC 2.7.1.1) (Brain form hexokina...  ...           NaN   \n",
       " 8   Hexokinase-4 (HK4) (EC 2.7.1.1) (Glucokinase) ...  ...           NaN   \n",
       " 9   Hexokinase-4 (HK4) (EC 2.7.1.1) (Glucokinase) ...  ...           NaN   \n",
       " 10  Hexokinase-2 (EC 2.7.1.1) (Hexokinase type II)...  ...           NaN   \n",
       " 11  Hexokinase-2 (EC 2.7.1.1) (Hexokinase type II)...  ...           NaN   \n",
       " 12  Hexokinase-3 (EC 2.7.1.1) (Hexokinase type III...  ...           NaN   \n",
       " 13  Hexokinase-3 (EC 2.7.1.1) (Hexokinase type III...  ...           NaN   \n",
       " 14  Hexokinase HKDC1 (EC 2.7.1.1) (Hexokinase doma...  ...           NaN   \n",
       " 15  Hexokinase-1 (EC 2.7.1.1) (Hexokinase type I) ...  ...           NaN   \n",
       " 16  Hexokinase-2 (EC 2.7.1.1) (Hexokinase type II)...  ...           NaN   \n",
       " 17  Hexokinase-2 (EC 2.7.1.1) (Hexokinase type II)...  ...           NaN   \n",
       " 18  Hexokinase-2 (EC 2.7.1.1) (Hexokinase type II)...  ...           NaN   \n",
       " 19  Hexokinase-2 (EC 2.7.1.1) (Hexokinase type II)...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  \\\n",
       " 0                   NaN         NaN   \n",
       " 1                   NaN         NaN   \n",
       " 2                   NaN         NaN   \n",
       " 3                   NaN         NaN   \n",
       " 4                   NaN         NaN   \n",
       " 5                   NaN         NaN   \n",
       " 6                   NaN         NaN   \n",
       " 7                   NaN         NaN   \n",
       " 8                   NaN         NaN   \n",
       " 9                   NaN         NaN   \n",
       " 10                  NaN         NaN   \n",
       " 11                  NaN         NaN   \n",
       " 12                  NaN         NaN   \n",
       " 13                  NaN         NaN   \n",
       " 14                  NaN         NaN   \n",
       " 15                  NaN         NaN   \n",
       " 16                  NaN         NaN   \n",
       " 17                  NaN         NaN   \n",
       " 18                  NaN         NaN   \n",
       " 19                  NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 1                                                 NaN     NaN   \n",
       " 2   MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 3                                                 NaN     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " 5   MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 6   MOD_RES 393; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 7   MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9                                                 NaN     NaN   \n",
       " 10  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 11  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 12                                                NaN     NaN   \n",
       " 13                                                NaN     NaN   \n",
       " 14                                                NaN     NaN   \n",
       " 15  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 16  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 17  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 18  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 19  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6   PTM: [Isoform HK1-SC]: Tyrosine-phosphorylated...         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19                                                NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  \\\n",
       " 0              NaN             NaN   \n",
       " 1              NaN             NaN   \n",
       " 2              NaN             NaN   \n",
       " 3              NaN             NaN   \n",
       " 4              NaN             NaN   \n",
       " 5              NaN             NaN   \n",
       " 6              NaN             NaN   \n",
       " 7              NaN             NaN   \n",
       " 8              NaN             NaN   \n",
       " 9              NaN             NaN   \n",
       " 10             NaN             NaN   \n",
       " 11             NaN             NaN   \n",
       " 12             NaN             NaN   \n",
       " 13             NaN             NaN   \n",
       " 14             NaN             NaN   \n",
       " 15             NaN             NaN   \n",
       " 16             NaN             NaN   \n",
       " 17             NaN             NaN   \n",
       " 18             NaN             NaN   \n",
       " 19             NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0   1CZA;1DGK;1HKB;1HKC;1QHA;4F9O;4FOE;4FOI;4FPA;4...  \n",
       " 1   1V4S;1V4T;3A0I;3F9M;3FGU;3FR0;3GOI;3H1V;3ID8;3...  \n",
       " 2                                2NZT;5HEX;5HFU;5HG1;  \n",
       " 3                                               3HM8;  \n",
       " 4                                                 NaN  \n",
       " 5                                               1BG3;  \n",
       " 6                                                 NaN  \n",
       " 7                                                 NaN  \n",
       " 8                                                 NaN  \n",
       " 9                                                 NaN  \n",
       " 10                                                NaN  \n",
       " 11                                                NaN  \n",
       " 12                                                NaN  \n",
       " 13                                                NaN  \n",
       " 14                                                NaN  \n",
       " 15                                                NaN  \n",
       " 16                                                NaN  \n",
       " 17                                                NaN  \n",
       " 18                                                NaN  \n",
       " 19                                                NaN  \n",
       " \n",
       " [20 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  Q9HU85  HUTH_PSEAE      hutH PA5098                 hutH   \n",
       " 1  B7V3J1  HUTH_PSEA8  hutH PLES_54881                 hutH   \n",
       " 2  Q02ER8  HUTH_PSEAB  hutH PA14_67320                 hutH   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                     PA5098               NaN   \n",
       " 1                   NaN                 PLES_54881               NaN   \n",
       " 2                   NaN                 PA14_67320               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 1             Pseudomonas aeruginosa (strain LESB58)         557722   \n",
       " 2         Pseudomonas aeruginosa (strain UCBPP-PA14)         208963   \n",
       " \n",
       "                                       Protein names  ... Glycosylation  \\\n",
       " 0  Histidine ammonia-lyase (Histidase) (EC 4.3.1.3)  ...           NaN   \n",
       " 1  Histidine ammonia-lyase (Histidase) (EC 4.3.1.3)  ...           NaN   \n",
       " 2  Histidine ammonia-lyase (Histidase) (EC 4.3.1.3)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 143; /note=\"2,3-didehydroalanine (Ser)...     NaN   \n",
       " 1  MOD_RES 143; /note=\"2,3-didehydroalanine (Ser)...     NaN   \n",
       " 2  MOD_RES 143; /note=\"2,3-didehydroalanine (Ser)...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Contains an active site 4-methylidene-imi...         NaN   \n",
       " 1  PTM: Contains an active site 4-methylidene-imi...         NaN   \n",
       " 2  PTM: Contains an active site 4-methylidene-imi...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "      Entry   Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0   P15717  LEUC1_SALTY  leuC1 leuC STM0111                leuC1   \n",
       " 1   Q8Z9I2   LEUC_SALTI  leuC STY0130 t0115                 leuC   \n",
       " 2   B5F7U7   LEUC_SALA4     leuC SeAg_B0127                 leuC   \n",
       " 3   B4TWV9   LEUC_SALSV     leuC SeSA_A0127                 leuC   \n",
       " 4   Q57TE8  LEUC1_SALCH      leuC1 SCH_0107                leuC1   \n",
       " 5   Q57SN1  LEUC2_SALCH      leuC2 SCH_0374                leuC2   \n",
       " 6   A9MQE0   LEUC_SALAR     leuC SARI_02891                 leuC   \n",
       " 7   Q8ZRJ0  LEUC2_SALTY       leuC2 STM0329                leuC2   \n",
       " 8   B5BLB3   LEUC_SALPK       leuC SSPA0109                 leuC   \n",
       " 9   P04787  LEUD1_SALTY  leuD1 leuD STM0110                leuD1   \n",
       " 10  Q5PDG3   LEUC_SALPA        leuC SPA0113                 leuC   \n",
       " 11  Q8Z9I3   LEUD_SALTI  leuD STY0129 t0114                 leuD   \n",
       " 12  B4TWV8   LEUD_SALSV     leuD SeSA_A0126                 leuD   \n",
       " 13  Q57SN0  LEUD2_SALCH      leuD2 SCH_0375                leuD2   \n",
       " 14  Q57TE9  LEUD1_SALCH      leuD1 SCH_0106                leuD1   \n",
       " 15  A9MQE1   LEUD_SALAR     leuD SARI_02892                 leuD   \n",
       " 16  Q8ZRI9  LEUD2_SALTY       leuD2 STM0330                leuD2   \n",
       " 17  B5BLB2   LEUD_SALPK       leuD SSPA0108                 leuD   \n",
       " 18  Q5PDG4   LEUD_SALPA        leuD SPA0112                 leuD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  leuC                    STM0111               NaN   \n",
       " 1                   NaN              STY0130 t0115               NaN   \n",
       " 2                   NaN                 SeAg_B0127               NaN   \n",
       " 3                   NaN                 SeSA_A0127               NaN   \n",
       " 4                   NaN                   SCH_0107               NaN   \n",
       " 5                   NaN                   SCH_0374               NaN   \n",
       " 6                   NaN                 SARI_02891               NaN   \n",
       " 7                   NaN                    STM0329               NaN   \n",
       " 8                   NaN                   SSPA0109               NaN   \n",
       " 9                  leuD                    STM0110               NaN   \n",
       " 10                  NaN                    SPA0113               NaN   \n",
       " 11                  NaN              STY0129 t0114               NaN   \n",
       " 12                  NaN                 SeSA_A0126               NaN   \n",
       " 13                  NaN                   SCH_0375               NaN   \n",
       " 14                  NaN                   SCH_0106               NaN   \n",
       " 15                  NaN                 SARI_02892               NaN   \n",
       " 16                  NaN                    STM0330               NaN   \n",
       " 17                  NaN                   SSPA0108               NaN   \n",
       " 18                  NaN                    SPA0112               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0   Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 1                                    Salmonella typhi          90370   \n",
       " 2                     Salmonella agona (strain SL483)         454166   \n",
       " 3         Salmonella schwarzengrund (strain CVM19633)         439843   \n",
       " 4             Salmonella choleraesuis (strain SC-B67)         321314   \n",
       " 5             Salmonella choleraesuis (strain SC-B67)         321314   \n",
       " 6   Salmonella arizonae (strain ATCC BAA-731 / CDC...          41514   \n",
       " 7   Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 8           Salmonella paratyphi A (strain AKU_12601)         554290   \n",
       " 9   Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 10  Salmonella paratyphi A (strain ATCC 9150 / SAR...         295319   \n",
       " 11                                   Salmonella typhi          90370   \n",
       " 12        Salmonella schwarzengrund (strain CVM19633)         439843   \n",
       " 13            Salmonella choleraesuis (strain SC-B67)         321314   \n",
       " 14            Salmonella choleraesuis (strain SC-B67)         321314   \n",
       " 15  Salmonella arizonae (strain ATCC BAA-731 / CDC...          41514   \n",
       " 16  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 17          Salmonella paratyphi A (strain AKU_12601)         554290   \n",
       " 18  Salmonella paratyphi A (strain ATCC 9150 / SAR...         295319   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   3-isopropylmalate dehydratase large subunit 1 ...  ...           NaN   \n",
       " 1   3-isopropylmalate dehydratase large subunit (E...  ...           NaN   \n",
       " 2   3-isopropylmalate dehydratase large subunit (E...  ...           NaN   \n",
       " 3   3-isopropylmalate dehydratase large subunit (E...  ...           NaN   \n",
       " 4   3-isopropylmalate dehydratase large subunit 1 ...  ...           NaN   \n",
       " 5   3-isopropylmalate dehydratase large subunit 2 ...  ...           NaN   \n",
       " 6   3-isopropylmalate dehydratase large subunit (E...  ...           NaN   \n",
       " 7   3-isopropylmalate dehydratase large subunit 2 ...  ...           NaN   \n",
       " 8   3-isopropylmalate dehydratase large subunit (E...  ...           NaN   \n",
       " 9   3-isopropylmalate dehydratase small subunit 1 ...  ...           NaN   \n",
       " 10  3-isopropylmalate dehydratase large subunit (E...  ...           NaN   \n",
       " 11  3-isopropylmalate dehydratase small subunit (E...  ...           NaN   \n",
       " 12  3-isopropylmalate dehydratase small subunit (E...  ...           NaN   \n",
       " 13  3-isopropylmalate dehydratase small subunit 2 ...  ...           NaN   \n",
       " 14  3-isopropylmalate dehydratase small subunit 1 ...  ...           NaN   \n",
       " 15  3-isopropylmalate dehydratase small subunit (E...  ...           NaN   \n",
       " 16  3-isopropylmalate dehydratase small subunit 2 ...  ...           NaN   \n",
       " 17  3-isopropylmalate dehydratase small subunit (E...  ...           NaN   \n",
       " 18  3-isopropylmalate dehydratase small subunit (E...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " \n",
       "     Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0                NaN     NaN                              NaN         NaN   \n",
       " 1                NaN     NaN                              NaN         NaN   \n",
       " 2                NaN     NaN                              NaN         NaN   \n",
       " 3                NaN     NaN                              NaN         NaN   \n",
       " 4                NaN     NaN                              NaN         NaN   \n",
       " 5                NaN     NaN                              NaN         NaN   \n",
       " 6                NaN     NaN                              NaN         NaN   \n",
       " 7                NaN     NaN                              NaN         NaN   \n",
       " 8                NaN     NaN                              NaN         NaN   \n",
       " 9                NaN     NaN                              NaN         NaN   \n",
       " 10               NaN     NaN                              NaN         NaN   \n",
       " 11               NaN     NaN                              NaN         NaN   \n",
       " 12               NaN     NaN                              NaN         NaN   \n",
       " 13               NaN     NaN                              NaN         NaN   \n",
       " 14               NaN     NaN                              NaN         NaN   \n",
       " 15               NaN     NaN                              NaN         NaN   \n",
       " 16               NaN     NaN                              NaN         NaN   \n",
       " 17               NaN     NaN                              NaN         NaN   \n",
       " 18               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  PDB  \n",
       " 0              NaN             NaN  NaN  \n",
       " 1              NaN             NaN  NaN  \n",
       " 2              NaN             NaN  NaN  \n",
       " 3              NaN             NaN  NaN  \n",
       " 4              NaN             NaN  NaN  \n",
       " 5              NaN             NaN  NaN  \n",
       " 6              NaN             NaN  NaN  \n",
       " 7              NaN             NaN  NaN  \n",
       " 8              NaN             NaN  NaN  \n",
       " 9              NaN             NaN  NaN  \n",
       " 10             NaN             NaN  NaN  \n",
       " 11             NaN             NaN  NaN  \n",
       " 12             NaN             NaN  NaN  \n",
       " 13             NaN             NaN  NaN  \n",
       " 14             NaN             NaN  NaN  \n",
       " 15             NaN             NaN  NaN  \n",
       " 16             NaN             NaN  NaN  \n",
       " 17             NaN             NaN  NaN  \n",
       " 18             NaN             NaN  NaN  \n",
       " \n",
       " [19 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P35136  SERA_BACSU  serA BSU23070                 serA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   BSU23070               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Bacillus subtilis (strain 168)         224308   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  D-3-phosphoglycerate dehydrogenase (PGDH) (EC ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P38100  CARB_PSEAE  carB PA4756                 carB                   NaN   \n",
       " 1  P38098  CARA_PSEAE  carA PA4758                 carA                   NaN   \n",
       " 2  P38099  CARA_STUST         carA                 carA                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     PA4756               NaN   \n",
       " 1                     PA4758               NaN   \n",
       " 2                        NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 1  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 2      Stutzerimonas stutzeri (Pseudomonas stutzeri)            316   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Carbamoyl-phosphate synthase large chain (EC 6...  ...           NaN   \n",
       " 1  Carbamoyl-phosphate synthase small chain (EC 6...  ...           NaN   \n",
       " 2  Carbamoyl-phosphate synthase small chain (EC 6...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " 2               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  O08557  DDAH1_RAT  Ddah1 Ddah                Ddah1                 Ddah   \n",
       " 1  Q6MG60  DDAH2_RAT       Ddah2                Ddah2                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  N(G),N(G)-dimethylarginine dimethylaminohydrol...  ...   \n",
       " 1          10116  N(G),N(G)-dimethylarginine dimethylaminohydrol...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN                                                NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 1         NaN                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P35669  GSHB_SCHPO  gsa1 gsh2 SPAC3F10.04                 gsa1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 gsh2                         NaN      SPAC3F10.04   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutathione synthetase large chain (GSH synthe...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 356; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name     Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P28037  AL1L1_RAT  Aldh1l1 Fthfd              Aldh1l1                Fthfd   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Cytosolic 10-formyltetrahydrofolate dehydrogen...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 9; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphopantetheinylation at Ser-354 by AA...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1S3I;2O2P;2O2Q;2O2R;3RHJ;3RHL;3RHM;3RHO;3RHP;3...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P70695  F16P2_MOUSE           Fbp2                 Fbp2   \n",
       " 1  Q9QXD6  F16P1_MOUSE  Fbp1 Fbp Fbp3                 Fbp1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1             Fbp Fbp3                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Fructose-1,6-bisphosphatase isozyme 2 (FBPase ...  ...           NaN   \n",
       " 1  Fructose-1,6-bisphosphatase 1 (FBPase 1) (EC 3...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 216; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P38158  MAL32_YEAST  MAL32 MAL3S YBR299W YBR2117                MAL32   \n",
       " 1  P53341  MAL12_YEAST          MAL12 MAL1S YGR292W                MAL12   \n",
       " 2  P07265  MAL62_YEASX                  MAL62 MAL6S                MAL62   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                MAL3S                    YBR299W          YBR2117   \n",
       " 1                MAL1S                    YGR292W              NaN   \n",
       " 2                MAL6S                        NaN              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2           Saccharomyces cerevisiae (Baker's yeast)           4932   \n",
       " \n",
       "                                      Protein names  ... Glycosylation  \\\n",
       " 0  Alpha-glucosidase MAL32 (EC 3.2.1.20) (Maltase)  ...           NaN   \n",
       " 1  Alpha-glucosidase MAL12 (EC 3.2.1.20) (Maltase)  ...           NaN   \n",
       " 2  Alpha-glucosidase MAL62 (EC 3.2.1.20) (Maltase)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1VAD;  \n",
       " 1             NaN    NaN  \n",
       " 2             NaN  1G7P;  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  P9WFX7  TRPC_MYCTU  trpC Rv1611 MTCY01B2.03                 trpC   \n",
       " 1  A5U2W7  TRPC_MYCTA            trpC MRA_1621                 trpC   \n",
       " 2  P9WFX6  TRPC_MYCTO              trpC MT1646                 trpC   \n",
       " 3  A1KJ27  TRPC_MYCBP            trpC BCG_1649                 trpC   \n",
       " 4  C1ANN3  TRPC_MYCBT            trpC JTY_1624                 trpC   \n",
       " 5  P0A633  TRPC_MYCBO       trpC BQ2027_MB1637                 trpC   \n",
       " 6  B2HQX7  TRPC_MYCMM           trpC MMAR_2413                 trpC   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                     Rv1611      MTCY01B2.03   \n",
       " 1                   NaN                   MRA_1621              NaN   \n",
       " 2                   NaN                     MT1646              NaN   \n",
       " 3                   NaN                   BCG_1649              NaN   \n",
       " 4                   NaN                   JTY_1624              NaN   \n",
       " 5                   NaN              BQ2027_MB1637              NaN   \n",
       " 6                   NaN                  MMAR_2413              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain ATCC 25177 ...         419947   \n",
       " 2  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 3  Mycobacterium bovis (strain BCG / Pasteur 1173P2)         410289   \n",
       " 4  Mycobacterium bovis (strain BCG / Tokyo 172 / ...         561275   \n",
       " 5  Mycobacterium bovis (strain ATCC BAA-935 / AF2...         233413   \n",
       " 6    Mycobacterium marinum (strain ATCC BAA-535 / M)         216594   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Indole-3-glycerol phosphate synthase (IGPS) (E...  ...           NaN   \n",
       " 1  Indole-3-glycerol phosphate synthase (IGPS) (E...  ...           NaN   \n",
       " 2  Indole-3-glycerol phosphate synthase (IGPS) (E...  ...           NaN   \n",
       " 3  Indole-3-glycerol phosphate synthase (IGPS) (E...  ...           NaN   \n",
       " 4  Indole-3-glycerol phosphate synthase (IGPS) (E...  ...           NaN   \n",
       " 5  Indole-3-glycerol phosphate synthase (IGPS) (E...  ...           NaN   \n",
       " 6  Indole-3-glycerol phosphate synthase (IGPS) (E...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " 6                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  3QJA;3T40;3T44;3T55;3T78;4FB7;  \n",
       " 1             NaN                             NaN  \n",
       " 2             NaN                             NaN  \n",
       " 3             NaN                             NaN  \n",
       " 4             NaN                             NaN  \n",
       " 5             NaN                             NaN  \n",
       " 6             NaN                             NaN  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  O70631  EST2C_RAT  Ces2c Ces2 Ces2l                Ces2c   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           Ces2 Ces2l                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Acylcarnitine hydrolase (ACH M1) (EC 3.1.1.28)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..26; /evidence=\"ECO:0000255|RuleBase:...             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "      Entry  Entry Name                      Gene Names Gene Names (primary)  \\\n",
       " 0   O31156  PHNX_BACCE                            phnX                 phnX   \n",
       " 1   Q9I433  PHNX_PSEAE                     phnX PA1311                 phnX   \n",
       " 2   Q81TE1  PHNX_BACAN  phnX BA_1340 GBAA_1340 BAS1239                 phnX   \n",
       " 3   Q81G82  PHNX_BACCR                    phnX BC_1326                 phnX   \n",
       " 4   A0RBE8  PHNX_BACAH                  phnX BALH_1189                 phnX   \n",
       " 5   A7GML9  PHNX_BACCN                phnX Bcer98_1051                 phnX   \n",
       " 6   B7JFQ8  PHNX_BACC0               phnX BCAH820_1414                 phnX   \n",
       " 7   B9IUR5  PHNX_BACCQ                   phnX BCQ_1398                 phnX   \n",
       " 8   B7HH80  PHNX_BACC4              phnX BCB4264_A1377                 phnX   \n",
       " 9   B7HK47  PHNX_BACC7              phnX BCAH187_A1480                 phnX   \n",
       " 10  B7IN18  PHNX_BACC2              phnX BCG9842_B3967                 phnX   \n",
       " 11  Q63E46  PHNX_BACCZ                 phnX BCE33L1217                 phnX   \n",
       " 12  Q73BH9  PHNX_BACC1                   phnX BCE_1439                 phnX   \n",
       " 13  A9VKQ2  PHNX_BACMK             phnX BcerKBAB4_1240                 phnX   \n",
       " 14  Q6HLM1  PHNX_BACHK                phnX BT9727_1215                 phnX   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                    NaN                        NaN               NaN   \n",
       " 1                    NaN                     PA1311               NaN   \n",
       " 2                    NaN  BA_1340 GBAA_1340 BAS1239               NaN   \n",
       " 3                    NaN                    BC_1326               NaN   \n",
       " 4                    NaN                  BALH_1189               NaN   \n",
       " 5                    NaN                Bcer98_1051               NaN   \n",
       " 6                    NaN               BCAH820_1414               NaN   \n",
       " 7                    NaN                   BCQ_1398               NaN   \n",
       " 8                    NaN              BCB4264_A1377               NaN   \n",
       " 9                    NaN              BCAH187_A1480               NaN   \n",
       " 10                   NaN              BCG9842_B3967               NaN   \n",
       " 11                   NaN                 BCE33L1217               NaN   \n",
       " 12                   NaN                   BCE_1439               NaN   \n",
       " 13                   NaN             BcerKBAB4_1240               NaN   \n",
       " 14                   NaN                BT9727_1215               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                     Bacillus cereus           1396   \n",
       " 1   Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 2                                  Bacillus anthracis           1392   \n",
       " 3   Bacillus cereus (strain ATCC 14579 / DSM 31 / ...         226900   \n",
       " 4            Bacillus thuringiensis (strain Al Hakam)         412694   \n",
       " 5   Bacillus cytotoxicus (strain DSM 22905 / CIP 1...         315749   \n",
       " 6                      Bacillus cereus (strain AH820)         405535   \n",
       " 7                         Bacillus cereus (strain Q1)         361100   \n",
       " 8                      Bacillus cereus (strain B4264)         405532   \n",
       " 9                      Bacillus cereus (strain AH187)         405534   \n",
       " 10                     Bacillus cereus (strain G9842)         405531   \n",
       " 11                 Bacillus cereus (strain ZK / E33L)         288681   \n",
       " 12      Bacillus cereus (strain ATCC 10987 / NRS 248)         222523   \n",
       " 13  Bacillus mycoides (strain KBAB4) (Bacillus wei...         315730   \n",
       " 14  Bacillus thuringiensis subsp. konkukian (strai...         281309   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 1   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 2   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 3   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 4   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 5   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 6   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 7   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 8   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 9   Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 10  Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 11  Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 12  Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 13  Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " 14  Phosphonoacetaldehyde hydrolase (Phosphonatase...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                   NaN         NaN               NaN     NaN   \n",
       " 1                   NaN         NaN               NaN     NaN   \n",
       " 2                   NaN         NaN               NaN     NaN   \n",
       " 3                   NaN         NaN               NaN     NaN   \n",
       " 4                   NaN         NaN               NaN     NaN   \n",
       " 5                   NaN         NaN               NaN     NaN   \n",
       " 6                   NaN         NaN               NaN     NaN   \n",
       " 7                   NaN         NaN               NaN     NaN   \n",
       " 8                   NaN         NaN               NaN     NaN   \n",
       " 9                   NaN         NaN               NaN     NaN   \n",
       " 10                  NaN         NaN               NaN     NaN   \n",
       " 11                  NaN         NaN               NaN     NaN   \n",
       " 12                  NaN         NaN               NaN     NaN   \n",
       " 13                  NaN         NaN               NaN     NaN   \n",
       " 14                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                                       PDB  \n",
       " 0              NaN  1FEZ;1RDF;1RQL;1RQN;1SWV;1SWW;2IOF;2IOH;  \n",
       " 1              NaN                                       NaN  \n",
       " 2              NaN                                       NaN  \n",
       " 3              NaN                                       NaN  \n",
       " 4              NaN                                       NaN  \n",
       " 5              NaN                                       NaN  \n",
       " 6              NaN                                       NaN  \n",
       " 7              NaN                                       NaN  \n",
       " 8              NaN                                       NaN  \n",
       " 9              NaN                                       NaN  \n",
       " 10             NaN                                       NaN  \n",
       " 11             NaN                                       NaN  \n",
       " 12             NaN                                       NaN  \n",
       " 13             NaN                                       NaN  \n",
       " 14             NaN                                       NaN  \n",
       " \n",
       " [15 rows x 110 columns],\n",
       "     Entry   Entry Name   Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P49888  ST1E1_HUMAN  SULT1E1 STE              SULT1E1                  STE   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Sulfotransferase 1E1 (ST1E1) (EC 2.8.2.4) (EST...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                        PDB  \n",
       " 0             NaN  1G3M;1HY3;4JVL;4JVM;4JVN;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P85973   PNPH_RAT     Pnp Np                  Pnp                   Np   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Purine nucleoside phosphorylase (PNP) (EC 2.4....  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  Q2UCU3  BGALA_ASPOR  lacA AO090012000445                 lacA   \n",
       " 1  Q2UMD5  BGALC_ASPOR  lacC AO090003000042                 lacC   \n",
       " 2  Q2U6P1  BGALB_ASPOR  lacB AO090120000158                 lacB   \n",
       " 3  Q5BFC4  BGALA_EMENI          lacA AN0756                 lacA   \n",
       " 4  Q5BEQ0  BGALB_EMENI          lacB AN0980                 lacB   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN   AO090012000445   \n",
       " 1                   NaN                         NaN   AO090003000042   \n",
       " 2                   NaN                         NaN   AO090120000158   \n",
       " 3                   NaN                         NaN           AN0756   \n",
       " 4                   NaN                         NaN           AN0980   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Aspergillus oryzae (strain ATCC 42149 / RIB 40...         510516   \n",
       " 1  Aspergillus oryzae (strain ATCC 42149 / RIB 40...         510516   \n",
       " 2  Aspergillus oryzae (strain ATCC 42149 / RIB 40...         510516   \n",
       " 3  Emericella nidulans (strain FGSC A4 / ATCC 381...         227321   \n",
       " 4  Emericella nidulans (strain FGSC A4 / ATCC 381...         227321   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Beta-galactosidase A (EC 3.2.1.23) (Acidic bet...  ...   \n",
       " 1  Probable beta-galactosidase C (EC 3.2.1.23) (L...  ...   \n",
       " 2  Probable beta-galactosidase B (EC 3.2.1.23) (L...  ...   \n",
       " 3  Probable beta-galactosidase A (EC 3.2.1.23) (L...  ...   \n",
       " 4  Probable beta-galactosidase B (EC 3.2.1.23) (L...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 156; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 197; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 2  CARBOHYD 25; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 3  CARBOHYD 156; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 4  CARBOHYD 23; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " 2         NaN               NaN     NaN                              NaN   \n",
       " 3         NaN               NaN     NaN                              NaN   \n",
       " 4         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide                         Signal peptide Transit peptide    PDB  \n",
       " 0         NaN  SIGNAL 1..18; /evidence=\"ECO:0000255\"             NaN  4IUG;  \n",
       " 1         NaN  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " 2         NaN  SIGNAL 1..22; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " 3         NaN  SIGNAL 1..18; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " 4         NaN  SIGNAL 1..21; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry   Entry Name                       Gene Names Gene Names (primary)  \\\n",
       " 0  P55229  GLGL1_ARATH    ADG2 APL1 At5g19220 T24G5.120                 ADG2   \n",
       " 1  P55228   GLGS_ARATH  APS1 ADG1 APS At5g48300 K23F3.2                 APS1   \n",
       " 2  P55230  GLGL2_ARATH          APL2 At1g27680 T22C5.13                 APL2   \n",
       " 3  Q9SIK1  GLGL4_ARATH                At2g21590 F2G1.14                  NaN   \n",
       " 4  P55231  GLGL3_ARATH         APL3 At4g39210 T22F8.110                 APL3   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 APL1                  At5g19220        T24G5.120   \n",
       " 1             ADG1 APS                  At5g48300          K23F3.2   \n",
       " 2                  NaN                  At1g27680         T22C5.13   \n",
       " 3                  NaN                  At2g21590          F2G1.14   \n",
       " 4                  NaN                  At4g39210        T22F8.110   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 3  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 4  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glucose-1-phosphate adenylyltransferase large ...  ...           NaN   \n",
       " 1  Glucose-1-phosphate adenylyltransferase small ...  ...           NaN   \n",
       " 2  Glucose-1-phosphate adenylyltransferase large ...  ...           NaN   \n",
       " 3  Probable glucose-1-phosphate adenylyltransfera...  ...           NaN   \n",
       " 4  Glucose-1-phosphate adenylyltransferase large ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " 3                  NaN         NaN   \n",
       " 4                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 428; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2  MOD_RES 424; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " 4                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..54; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 1  TRANSIT 1..71; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 2  TRANSIT 1..?; /note=\"Chloroplast\"; /evidence=\"...  NaN  \n",
       " 3  TRANSIT 1..?; /note=\"Chloroplast\"; /evidence=\"...  NaN  \n",
       " 4  TRANSIT 1..61; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P96111  PYRB_THEMA  pyrBI TM_1642                pyrBI   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    TM_1642               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Thermotoga maritima (strain ATCC 43589 / DSM 3...         243274   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Protein PyrBI [Includes: Aspartate carbamoyltr...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                  Gene Names Gene Names (primary)  \\\n",
       " 0  A0QWS1  PYRB_MYCS2  pyrB MSMEG_3043 MSMEI_2969                 pyrB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN      MSMEG_3043 MSMEI_2969               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycolicibacterium smegmatis (strain ATCC 70008...         246196   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aspartate carbamoyltransferase (EC 2.1.3.2) (A...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0  P9WQA3  ALF_MYCTU  fba Rv0363c MTCY13E10.25c                  fba   \n",
       " 1  P9WQA2  ALF_MYCTO                 fba MT0379                  fba   \n",
       " 2  P67476  ALF_MYCBO         fba BQ2027_MB0370C                  fba   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    Rv0363c    MTCY13E10.25c   \n",
       " 1                   NaN                     MT0379              NaN   \n",
       " 2                   NaN             BQ2027_MB0370C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 2  Mycobacterium bovis (strain ATCC BAA-935 / AF2...         233413   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Fructose-bisphosphate aldolase (FBP aldolase) ...  ...           NaN   \n",
       " 1  Fructose-bisphosphate aldolase (FBP aldolase) ...  ...           NaN   \n",
       " 2  Fructose-bisphosphate aldolase (FBP aldolase) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                       PDB  \n",
       " 0             NaN  3EKL;3EKZ;3ELF;4A21;4A22;4DEF;4DEL;4LV4;  \n",
       " 1             NaN                                       NaN  \n",
       " 2             NaN                                       NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P97584  PTGR1_RAT  Ptgr1 Dig1 Ltb4dh                Ptgr1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0          Dig1 Ltb4dh                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Prostaglandin reductase 1 (PRG-1) (15-oxoprost...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 18; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  P39765  PYRR_BACSU         pyrR BSU15470                 pyrR   \n",
       " 1  P39149   UPP_BACSU  upp BSU36890 ipc-35d                  upp   \n",
       " 2  A7Z9Q8   UPP_BACVZ       upp RBAM_034050                  upp   \n",
       " 3  Q9K6G5   UPP_HALH5            upp BH3764                  upp   \n",
       " 4  Q65DW6   UPP_BACLD  upp BLi03934 BL03992                  upp   \n",
       " 5  A7Z4G7  PYRR_BACVZ      pyrR RBAM_015300                 pyrR   \n",
       " 6  Q9K9V4  PYRR_HALH5           pyrR BH2541                 pyrR   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                   BSU15470              NaN   \n",
       " 1                   NaN                   BSU36890          ipc-35d   \n",
       " 2                   NaN                RBAM_034050              NaN   \n",
       " 3                   NaN                     BH3764              NaN   \n",
       " 4                   NaN           BLi03934 BL03992              NaN   \n",
       " 5                   NaN                RBAM_015300              NaN   \n",
       " 6                   NaN                     BH2541              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                     Bacillus subtilis (strain 168)         224308   \n",
       " 1                     Bacillus subtilis (strain 168)         224308   \n",
       " 2  Bacillus velezensis (strain DSM 23117 / BGSC 1...         326423   \n",
       " 3  Halalkalibacterium halodurans (strain ATCC BAA...         272558   \n",
       " 4  Bacillus licheniformis (strain ATCC 14580 / DS...         279010   \n",
       " 5  Bacillus velezensis (strain DSM 23117 / BGSC 1...         326423   \n",
       " 6  Halalkalibacterium halodurans (strain ATCC BAA...         272558   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Bifunctional protein PyrR [Includes: Pyrimidin...  ...           NaN   \n",
       " 1  Uracil phosphoribosyltransferase (EC 2.4.2.9) ...  ...           NaN   \n",
       " 2  Uracil phosphoribosyltransferase (EC 2.4.2.9) ...  ...           NaN   \n",
       " 3  Uracil phosphoribosyltransferase (EC 2.4.2.9) ...  ...           NaN   \n",
       " 4  Uracil phosphoribosyltransferase (EC 2.4.2.9) ...  ...           NaN   \n",
       " 5  Bifunctional protein PyrR [Includes: Pyrimidin...  ...           NaN   \n",
       " 6  Bifunctional protein PyrR [Includes: Pyrimidin...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " 6                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  1A3C;1A4X;4P82;  \n",
       " 1             NaN              NaN  \n",
       " 2             NaN              NaN  \n",
       " 3             NaN              NaN  \n",
       " 4             NaN              NaN  \n",
       " 5             NaN              NaN  \n",
       " 6             NaN              NaN  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  G3XD67  HCNA_PSEAE  hcnA PA2193                 hcnA                   NaN   \n",
       " 1  G3XD12  HCNC_PSEAE  hcnC PA2195                 hcnC                   NaN   \n",
       " 2  Q9I1S2  HCNB_PSEAE  hcnB PA2194                 hcnB                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     PA2193               NaN   \n",
       " 1                     PA2195               NaN   \n",
       " 2                     PA2194               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 1  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 2  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Hydrogen cyanide synthase subunit HcnA (HcnA) ...  ...           NaN   \n",
       " 1  Hydrogen cyanide synthase subunit HcnC (HcnC) ...  ...           NaN   \n",
       " 2  Hydrogen cyanide synthase subunit HcnB (HcnB) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine                                         Lipidation  \\\n",
       " 0                  NaN                                                NaN   \n",
       " 1                  NaN  LIPID 19; /note=\"N-palmitoyl cysteine\"; /evide...   \n",
       " 2                  NaN                                                NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " 2               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  PDB  \n",
       " 0                                                NaN             NaN  NaN  \n",
       " 1  SIGNAL 1..18; /evidence=\"ECO:0000255|PROSITE-P...             NaN  NaN  \n",
       " 2                                                NaN             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name     Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q6P6S9  ENTP5_RAT         Entpd5               Entpd5                  NaN   \n",
       " 1  Q9ER31  ENTP6_RAT  Entpd6 Cd39l2               Entpd6               Cd39l2   \n",
       " 2  Q8K4Y7  CANT1_RAT    Cant1 Srapy                Cant1                Srapy   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 2                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Ectonucleoside triphosphate diphosphohydrolase...  ...   \n",
       " 1          10116  Ectonucleoside triphosphate diphosphohydrolase...  ...   \n",
       " 2          10116  Soluble calcium-activated nucleotidase 1 (SCAN...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 231; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 192; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 2  CARBOHYD 90; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " 1         NaN               NaN     NaN   \n",
       " 2         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated; high-mannose type. {ECO:0...         NaN   \n",
       " 1  PTM: Might be cleaved at the N-terminus, retai...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..24; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " 1                                    NaN             NaN  NaN  \n",
       " 2                                    NaN             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name                                         Gene Names  \\\n",
       " 0  Q10MW3  PDC2_ORYSJ                   PDC2 Os03g0293500 LOC_Os03g18220   \n",
       " 1  Q0D3D2  PDC3_ORYSJ        PDC3 Os07g0693100 LOC_Os07g49250 OsJ_024667   \n",
       " 2  Q0DHF6  PDC1_ORYSJ  PDC1 Os05g0469600 LOC_Os05g39310 OsJ_018109 OS...   \n",
       " 3  A2XFI3  PDC2_ORYSI                                    PDC2 OsI_010826   \n",
       " 4  A2Y5L9  PDC1_ORYSI                                    PDC1 OsI_019612   \n",
       " 5  A2YQ76  PDC3_ORYSI                                    PDC3 OsI_026469   \n",
       " \n",
       "   Gene Names (primary)  Gene Names (synonym)   Gene Names (ordered locus)  \\\n",
       " 0                 PDC2                   NaN  Os03g0293500 LOC_Os03g18220   \n",
       " 1                 PDC3                   NaN  Os07g0693100 LOC_Os07g49250   \n",
       " 2                 PDC1                   NaN  Os05g0469600 LOC_Os05g39310   \n",
       " 3                 PDC2                   NaN                          NaN   \n",
       " 4                 PDC1                   NaN                          NaN   \n",
       " 5                 PDC3                   NaN                          NaN   \n",
       " \n",
       "              Gene Names (ORF)                             Organism  \\\n",
       " 0                         NaN  Oryza sativa subsp. japonica (Rice)   \n",
       " 1                  OsJ_024667  Oryza sativa subsp. japonica (Rice)   \n",
       " 2  OsJ_018109 OSJNBa0052E20.2  Oryza sativa subsp. japonica (Rice)   \n",
       " 3                  OsI_010826    Oryza sativa subsp. indica (Rice)   \n",
       " 4                  OsI_019612    Oryza sativa subsp. indica (Rice)   \n",
       " 5                  OsI_026469    Oryza sativa subsp. indica (Rice)   \n",
       " \n",
       "    Organism (ID)                                Protein names  ...  \\\n",
       " 0          39947  Pyruvate decarboxylase 2 (PDC) (EC 4.1.1.1)  ...   \n",
       " 1          39947  Pyruvate decarboxylase 3 (PDC) (EC 4.1.1.1)  ...   \n",
       " 2          39947  Pyruvate decarboxylase 1 (PDC) (EC 4.1.1.1)  ...   \n",
       " 3          39946  Pyruvate decarboxylase 2 (PDC) (EC 4.1.1.1)  ...   \n",
       " 4          39946  Pyruvate decarboxylase 1 (PDC) (EC 4.1.1.1)  ...   \n",
       " 5          39946  Pyruvate decarboxylase 3 (PDC) (EC 4.1.1.1)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " 1           NaN                  NaN         NaN               NaN     NaN   \n",
       " 2           NaN                  NaN         NaN               NaN     NaN   \n",
       " 3           NaN                  NaN         NaN               NaN     NaN   \n",
       " 4           NaN                  NaN         NaN               NaN     NaN   \n",
       " 5           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " 5             NaN  NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  Q01134   CHKA_RAT     Chka Chk Ckr                 Chka   \n",
       " 1  O54783   CHKB_RAT  Chkb Chetk Chkl                 Chkb   \n",
       " 2  D3ZRW8   EKI2_RAT            Etnk2                Etnk2   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0              Chk Ckr                         NaN               NaN   \n",
       " 1           Chetk Chkl                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " 1  Rattus norvegicus (Rat)          10116   \n",
       " 2  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Choline kinase alpha (CK) (EC 2.7.1.32) (CHETK...  ...           NaN   \n",
       " 1  Choline/ethanolamine kinase (Choline kinase be...  ...           NaN   \n",
       " 2  Ethanolamine kinase 2 (EKI 2) (EC 2.7.1.82) (E...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 71; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: [Isoform 1]: Phosphorylated at Ser-275 by...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide PDB  \n",
       " 0             NaN             NaN NaN  \n",
       " 1             NaN             NaN NaN  \n",
       " 2             NaN             NaN NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P07003  POXB_ECOLI  poxB b0871 JW0855                 poxB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN               b0871 JW0855               NaN   \n",
       " \n",
       "                         Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Pyruvate dehydrogenase [ubiquinone] (EC 1.2.5....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Activated by limited proteolytic digestio...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide         PDB  \n",
       " 0             NaN             NaN  3EY9;3EYA;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P30750  METN_ECOLI  metN abc b0199 JW0195                 metN   \n",
       " 1  P63356  METN_ECO57     metN Z0211 ECs0201                 metN   \n",
       " 2  Q1RFY9  METN_ECOUT       metN UTI89_C0215                 metN   \n",
       " 3  Q0TLD2  METN_ECOL5          metN ECP_0209                 metN   \n",
       " 4  P63355  METN_ECOL6             metN c0240                 metN   \n",
       " 5  Q83MC5  METN_SHIFL      metN SF0190 S0192                 metN   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  abc               b0199 JW0195               NaN   \n",
       " 1                  NaN              Z0211 ECs0201               NaN   \n",
       " 2                  NaN                UTI89_C0215               NaN   \n",
       " 3                  NaN                   ECP_0209               NaN   \n",
       " 4                  NaN                      c0240               NaN   \n",
       " 5                  NaN               SF0190 S0192               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                           Escherichia coli O157:H7          83334   \n",
       " 2             Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 3    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 4  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 5                                  Shigella flexneri            623   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Methionine import ATP-binding protein MetN (EC...  ...           NaN   \n",
       " 1  Methionine import ATP-binding protein MetN (EC...  ...           NaN   \n",
       " 2  Methionine import ATP-binding protein MetN (EC...  ...           NaN   \n",
       " 3  Methionine import ATP-binding protein MetN (EC...  ...           NaN   \n",
       " 4  Methionine import ATP-binding protein MetN (EC...  ...           NaN   \n",
       " 5  Methionine import ATP-binding protein MetN (EC...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  3DHW;3DHX;3TUI;3TUJ;3TUZ;6CVL;  \n",
       " 1             NaN                             NaN  \n",
       " 2             NaN                             NaN  \n",
       " 3             NaN                             NaN  \n",
       " 4             NaN                             NaN  \n",
       " 5             NaN                             NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P54001  P4HA1_RAT  P4ha1 P4ha                P4ha1                 P4ha   \n",
       " 1  Q6W3E9  P4HA3_RAT       P4ha3                P4ha3                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Prolyl 4-hydroxylase subunit alpha-1 (4-PH alp...  ...   \n",
       " 1          10116  Prolyl 4-hydroxylase subunit alpha-3 (4-PH alp...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 113; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 242; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " 1         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: N-glycosylation plays no role in the cata...         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..17; /evidence=\"ECO:0000250\"             NaN  NaN  \n",
       " 1  SIGNAL 1..24; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  Q90YB1  DNLI4_CHICK  LIG4 RCJMB04_10b2                 LIG4   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN     RCJMB04_10b2   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Gallus gallus (Chicken)           9031   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  DNA ligase 4 (EC 6.5.1.1) (DNA ligase IV) (Pol...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q0KBP1  BKTB_CUPNH  bktB H16_A1445                 bktB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                  H16_A1445               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Cupriavidus necator (strain ATCC 17699 / DSM 4...         381666   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Beta-ketothiolase BktB (EC 2.3.1.16) (EC 2.3.1...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  4NZS;4W61;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P23733  PP11_TRYBB         NaN                   NaN                   NaN   \n",
       " 1  P23734  PP12_TRYBB         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                   Organism  \\\n",
       " 0                         NaN               NaN  Trypanosoma brucei brucei   \n",
       " 1                         NaN               NaN  Trypanosoma brucei brucei   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           5702  Serine/threonine-protein phosphatase PP1(4.8) ...  ...   \n",
       " 1           5702  Serine/threonine-protein phosphatase PP1(5.9) ...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " 1            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P56818  BACE1_MOUSE  Bace1 Bace                Bace1                 Bace   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Beta-secretase 1 (EC 3.4.23.46) (Aspartyl prot...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 153; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0  LIPID 474; /note=\"S-palmitoyl cysteine\"; /evid...   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 126; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: N-Glycosylated (By similarity). Addition ...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 22..45; /evidence=\"ECO:0000255\"; /id=\"P...   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..21; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P97494  GSH1_MOUSE  Gclc Glclc                 Gclc                Glclc   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Glutamate--cysteine ligase catalytic subunit (...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry   Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0   A2RST1  MGST2_MOUSE              Mgst2 Gst2                Mgst2   \n",
       " 1   O89112  LANC1_MOUSE           Lancl1 Gpr69a               Lancl1   \n",
       " 2   Q8R5I6  GSTM4_MOUSE     Gstm4 Gstm7 Gstm7-7                Gstm4   \n",
       " 3   P13745  GSTA1_MOUSE        Gsta1 Gsta Gstya                Gsta1   \n",
       " 4   P19639  GSTM3_MOUSE                   Gstm3                Gstm3   \n",
       " 5   P46425  GSTP2_MOUSE            Gstp2 Gstpia                Gstp2   \n",
       " 6   O09131  GSTO1_MOUSE       Gsto1 Gstx Gtsttl                Gsto1   \n",
       " 7   P10648  GSTA2_MOUSE                   Gsta2                Gsta2   \n",
       " 8   P15626  GSTM2_MOUSE                   Gstm2                Gstm2   \n",
       " 9   P19157  GSTP1_MOUSE            Gstp1 Gstpib                Gstp1   \n",
       " 10  P24472  GSTA4_MOUSE              Gsta4 Gsta                Gsta4   \n",
       " 11  P30115  GSTA3_MOUSE             Gsta3 Gstyc                Gsta3   \n",
       " 12  P48774  GSTM5_MOUSE        Gstm5 Fsc2 Gstm3                Gstm5   \n",
       " 13  Q91VS7  MGST1_MOUSE                   Mgst1                Mgst1   \n",
       " 14  P10649  GSTM1_MOUSE                   Gstm1                Gstm1   \n",
       " 15  Q64471  GSTT1_MOUSE                   Gstt1                Gstt1   \n",
       " 16  Q9DCM2  GSTK1_MOUSE                   Gstk1                Gstk1   \n",
       " 17  Q9WVL0   MAAI_MOUSE              Gstz1 Maai                Gstz1   \n",
       " 18  Q80W21  GSTM7_MOUSE                   Gstm7                Gstm7   \n",
       " 19  Q9JM51  PTGES_MOUSE              Ptges Pges                Ptges   \n",
       " 20  Q9JHF7  HPGDS_MOUSE  Hpgds Gsts Pgds Ptgds2                Hpgds   \n",
       " 21  Q8K2Q2  GSTO2_MOUSE                   Gsto2                Gsto2   \n",
       " 22  O35660  GSTM6_MOUSE             Gstm6 Gstm5                Gstm6   \n",
       " 23  Q61133  GSTT2_MOUSE                   Gstt2                Gstt2   \n",
       " 24  Q9D4P7  GSTT4_MOUSE                   Gstt4                Gstt4   \n",
       " 25  Q99L20  GSTT3_MOUSE                   Gstt3                Gstt3   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  Gst2                         NaN               NaN   \n",
       " 1                Gpr69a                         NaN               NaN   \n",
       " 2         Gstm7 Gstm7-7                         NaN               NaN   \n",
       " 3            Gsta Gstya                         NaN               NaN   \n",
       " 4                   NaN                         NaN               NaN   \n",
       " 5                Gstpia                         NaN               NaN   \n",
       " 6           Gstx Gtsttl                         NaN               NaN   \n",
       " 7                   NaN                         NaN               NaN   \n",
       " 8                   NaN                         NaN               NaN   \n",
       " 9                Gstpib                         NaN               NaN   \n",
       " 10                 Gsta                         NaN               NaN   \n",
       " 11                Gstyc                         NaN               NaN   \n",
       " 12           Fsc2 Gstm3                         NaN               NaN   \n",
       " 13                  NaN                         NaN               NaN   \n",
       " 14                  NaN                         NaN               NaN   \n",
       " 15                  NaN                         NaN               NaN   \n",
       " 16                  NaN                         NaN               NaN   \n",
       " 17                 Maai                         NaN               NaN   \n",
       " 18                  NaN                         NaN               NaN   \n",
       " 19                 Pges                         NaN               NaN   \n",
       " 20     Gsts Pgds Ptgds2                         NaN               NaN   \n",
       " 21                  NaN                         NaN               NaN   \n",
       " 22                Gstm5                         NaN               NaN   \n",
       " 23                  NaN                         NaN               NaN   \n",
       " 24                  NaN                         NaN               NaN   \n",
       " 25                  NaN                         NaN               NaN   \n",
       " \n",
       "                 Organism  Organism (ID)  \\\n",
       " 0   Mus musculus (Mouse)          10090   \n",
       " 1   Mus musculus (Mouse)          10090   \n",
       " 2   Mus musculus (Mouse)          10090   \n",
       " 3   Mus musculus (Mouse)          10090   \n",
       " 4   Mus musculus (Mouse)          10090   \n",
       " 5   Mus musculus (Mouse)          10090   \n",
       " 6   Mus musculus (Mouse)          10090   \n",
       " 7   Mus musculus (Mouse)          10090   \n",
       " 8   Mus musculus (Mouse)          10090   \n",
       " 9   Mus musculus (Mouse)          10090   \n",
       " 10  Mus musculus (Mouse)          10090   \n",
       " 11  Mus musculus (Mouse)          10090   \n",
       " 12  Mus musculus (Mouse)          10090   \n",
       " 13  Mus musculus (Mouse)          10090   \n",
       " 14  Mus musculus (Mouse)          10090   \n",
       " 15  Mus musculus (Mouse)          10090   \n",
       " 16  Mus musculus (Mouse)          10090   \n",
       " 17  Mus musculus (Mouse)          10090   \n",
       " 18  Mus musculus (Mouse)          10090   \n",
       " 19  Mus musculus (Mouse)          10090   \n",
       " 20  Mus musculus (Mouse)          10090   \n",
       " 21  Mus musculus (Mouse)          10090   \n",
       " 22  Mus musculus (Mouse)          10090   \n",
       " 23  Mus musculus (Mouse)          10090   \n",
       " 24  Mus musculus (Mouse)          10090   \n",
       " 25  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Microsomal glutathione S-transferase 2 (Micros...  ...           NaN   \n",
       " 1   Glutathione S-transferase LANCL1 (EC 2.5.1.18)...  ...           NaN   \n",
       " 2   Glutathione S-transferase Mu 4 (EC 2.5.1.18) (...  ...           NaN   \n",
       " 3   Glutathione S-transferase A1 (EC 2.5.1.18) (13...  ...           NaN   \n",
       " 4   Glutathione S-transferase Mu 3 (EC 2.5.1.18) (...  ...           NaN   \n",
       " 5   Glutathione S-transferase P 2 (Gst P2) (EC 2.5...  ...           NaN   \n",
       " 6   Glutathione S-transferase omega-1 (GSTO-1) (EC...  ...           NaN   \n",
       " 7   Glutathione S-transferase A2 (EC 2.5.1.18) (GS...  ...           NaN   \n",
       " 8   Glutathione S-transferase Mu 2 (EC 2.5.1.18) (...  ...           NaN   \n",
       " 9   Glutathione S-transferase P 1 (Gst P1) (EC 2.5...  ...           NaN   \n",
       " 10  Glutathione S-transferase A4 (EC 2.5.1.18) (GS...  ...           NaN   \n",
       " 11  Glutathione S-transferase A3 (EC 2.5.1.18) (GS...  ...           NaN   \n",
       " 12  Glutathione S-transferase Mu 5 (EC 2.5.1.18) (...  ...           NaN   \n",
       " 13  Microsomal glutathione S-transferase 1 (Micros...  ...           NaN   \n",
       " 14  Glutathione S-transferase Mu 1 (EC 2.5.1.18) (...  ...           NaN   \n",
       " 15  Glutathione S-transferase theta-1 (EC 2.5.1.18...  ...           NaN   \n",
       " 16  Glutathione S-transferase kappa 1 (EC 2.5.1.18...  ...           NaN   \n",
       " 17  Maleylacetoacetate isomerase (MAAI) (EC 5.2.1....  ...           NaN   \n",
       " 18  Glutathione S-transferase Mu 7 (EC 2.5.1.18) (...  ...           NaN   \n",
       " 19  Prostaglandin E synthase (mPGES-1) (EC 5.3.99....  ...           NaN   \n",
       " 20  Hematopoietic prostaglandin D synthase (H-PGDS...  ...           NaN   \n",
       " 21  Glutathione S-transferase omega-2 (GSTO-2) (EC...  ...           NaN   \n",
       " 22  Glutathione S-transferase Mu 6 (EC 2.5.1.18) (...  ...           NaN   \n",
       " 23  Glutathione S-transferase theta-2 (EC 2.5.1.18...  ...           NaN   \n",
       " 24  Glutathione S-transferase theta-4 (EC 2.5.1.18...  ...           NaN   \n",
       " 25    Glutathione S-transferase theta-3 (EC 2.5.1.18)  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3   INIT_MET 1; /note=\"Removed; alternate\"; /evide...         NaN   \n",
       " 4   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 7   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 8   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 9   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19                                                NaN         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22                                                NaN         NaN   \n",
       " 23                                                NaN         NaN   \n",
       " 24                                                NaN         NaN   \n",
       " 25                                                NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0                                                 NaN     NaN   \n",
       " 1   MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3   MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6   MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 7   MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 8   MOD_RES 27; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 9   MOD_RES 4; /note=\"Phosphotyrosine; by EGFR\"; /...     NaN   \n",
       " 10  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 11  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 12  MOD_RES 5; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " 13  MOD_RES 42; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 14  MOD_RES 34; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 15                                                NaN     NaN   \n",
       " 16  MOD_RES 36; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " 17  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 18                                                NaN     NaN   \n",
       " 19                                                NaN     NaN   \n",
       " 20                                                NaN     NaN   \n",
       " 21                                                NaN     NaN   \n",
       " 22                                                NaN     NaN   \n",
       " 23                                                NaN     NaN   \n",
       " 24                                                NaN     NaN   \n",
       " 25                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                    PTM: The N-terminus is blocked.         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13  PTM: Acetylation of Lys-42 and Lys-55 is obser...         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16  PTM: Acetylation of Lys-93 is observed in live...         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19                                                NaN         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22                                                NaN         NaN   \n",
       " 23                                                NaN         NaN   \n",
       " 24                                                NaN         NaN   \n",
       " 25                                                NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  \\\n",
       " 0              NaN             NaN   \n",
       " 1              NaN             NaN   \n",
       " 2              NaN             NaN   \n",
       " 3              NaN             NaN   \n",
       " 4              NaN             NaN   \n",
       " 5              NaN             NaN   \n",
       " 6              NaN             NaN   \n",
       " 7              NaN             NaN   \n",
       " 8              NaN             NaN   \n",
       " 9              NaN             NaN   \n",
       " 10             NaN             NaN   \n",
       " 11             NaN             NaN   \n",
       " 12             NaN             NaN   \n",
       " 13             NaN             NaN   \n",
       " 14             NaN             NaN   \n",
       " 15             NaN             NaN   \n",
       " 16             NaN             NaN   \n",
       " 17             NaN             NaN   \n",
       " 18             NaN             NaN   \n",
       " 19             NaN             NaN   \n",
       " 20             NaN             NaN   \n",
       " 21             NaN             NaN   \n",
       " 22             NaN             NaN   \n",
       " 23             NaN             NaN   \n",
       " 24             NaN             NaN   \n",
       " 25             NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0                                                 NaN  \n",
       " 1                                                 NaN  \n",
       " 2                                                 NaN  \n",
       " 3                                          1F3A;1F3B;  \n",
       " 4                                                 NaN  \n",
       " 5                                                 NaN  \n",
       " 6                                                 NaN  \n",
       " 7                                               1ML6;  \n",
       " 8                                                 NaN  \n",
       " 9   1BAY;1GLP;1GLQ;1GSY;1GTI;2GLR;2OA7;2OAC;2OAD;3...  \n",
       " 10                                         1B48;1GUK;  \n",
       " 11                                                NaN  \n",
       " 12                                                NaN  \n",
       " 13                                                NaN  \n",
       " 14                                                NaN  \n",
       " 15                                                NaN  \n",
       " 16                                                NaN  \n",
       " 17                                         2CZ2;2CZ3;  \n",
       " 18                                              2DC5;  \n",
       " 19                                                NaN  \n",
       " 20                                                NaN  \n",
       " 21                                                NaN  \n",
       " 22                                                NaN  \n",
       " 23                                                NaN  \n",
       " 24                                                NaN  \n",
       " 25                                                NaN  \n",
       " \n",
       " [26 rows x 110 columns],\n",
       "     Entry   Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  Q9HAY6  BCDO1_HUMAN  BCO1 BCDO BCDO1 BCMO1                 BCO1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0     BCDO BCDO1 BCMO1                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Beta,beta-carotene 15,15'-dioxygenase (EC 1.13...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   P22259  PCKA_ECOLI         pckA pck b3403 JW3366                 pckA   \n",
       " 1   B6I2W3  PCKA_ECOSE                pckA ECSE_3666                 pckA   \n",
       " 2   Q8X733  PCKA_ECO57            pckA Z4758 ECs4245                 pckA   \n",
       " 3   B1X750  PCKA_ECODH             pckA ECDH10B_3578                 pckA   \n",
       " 4   A1AGS9  PCKA_ECOK1  pckA Ecok1_33750 APECO1_3061                 pckA   \n",
       " 5   B7M1V7  PCKA_ECO8A              pckA ECIAI1_3544                 pckA   \n",
       " 6   B7L4T1  PCKA_ECO55             pckA EC55989_3809                 pckA   \n",
       " 7   B7N136  PCKA_ECO81               pckA ECED1_4063                 pckA   \n",
       " 8   C4ZUQ8  PCKA_ECOBW                 pckA BWG_3094                 pckA   \n",
       " 9   A8A5L1  PCKA_ECOHS               pckA EcHS_A3600                 pckA   \n",
       " 10  A7ZST1  PCKA_ECO24           pckA EcE24377A_3877                 pckA   \n",
       " 11  B5YTV3  PCKA_ECO5E            pckA ECH74115_4709                 pckA   \n",
       " 12  B7MDM9  PCKA_ECO45               pckA ECS88_3790                 pckA   \n",
       " 13  B7NMG6  PCKA_ECO7I             pckA ECIAI39_3881                 pckA   \n",
       " 14  B1LHK1  PCKA_ECOSM             pckA EcSMS35_3682                 pckA   \n",
       " 15  B7UKA7  PCKA_ECO27              pckA E2348C_3647                 pckA   \n",
       " 16  B7NE08  PCKA_ECOLU               pckA ECUMN_3862                 pckA   \n",
       " 17  Q0TC64  PCKA_ECOL5                 pckA ECP_3489                 pckA   \n",
       " 18  Q8FCU4  PCKA_ECOL6                    pckA c4176                 pckA   \n",
       " 19  B1IP62  PCKA_ECOLC               pckA EcolC_0310                 pckA   \n",
       " 20  B7LSA4  PCKA_ESCF3                pckA EFER_3369                 pckA   \n",
       " 21  Q83J96  PCKA_SHIFL             pckA SF3422 S4341                 pckA   \n",
       " 22  P41033  PCKA_SALTY              pckA pck STM3500                 pckA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   pck               b3403 JW3366              NaN   \n",
       " 1                   NaN                  ECSE_3666              NaN   \n",
       " 2                   NaN              Z4758 ECs4245              NaN   \n",
       " 3                   NaN               ECDH10B_3578              NaN   \n",
       " 4                   NaN                Ecok1_33750      APECO1_3061   \n",
       " 5                   NaN                ECIAI1_3544              NaN   \n",
       " 6                   NaN               EC55989_3809              NaN   \n",
       " 7                   NaN                 ECED1_4063              NaN   \n",
       " 8                   NaN                   BWG_3094              NaN   \n",
       " 9                   NaN                 EcHS_A3600              NaN   \n",
       " 10                  NaN             EcE24377A_3877              NaN   \n",
       " 11                  NaN              ECH74115_4709              NaN   \n",
       " 12                  NaN                 ECS88_3790              NaN   \n",
       " 13                  NaN               ECIAI39_3881              NaN   \n",
       " 14                  NaN               EcSMS35_3682              NaN   \n",
       " 15                  NaN                E2348C_3647              NaN   \n",
       " 16                  NaN                 ECUMN_3862              NaN   \n",
       " 17                  NaN                   ECP_3489              NaN   \n",
       " 18                  NaN                      c4176              NaN   \n",
       " 19                  NaN                 EcolC_0310              NaN   \n",
       " 20                  NaN                  EFER_3369              NaN   \n",
       " 21                  NaN               SF3422 S4341              NaN   \n",
       " 22                  pck                    STM3500              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain SE11)         409438   \n",
       " 2                            Escherichia coli O157:H7          83334   \n",
       " 3               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 4                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 5                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 6              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 7                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 8     Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 9                  Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 10  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 11    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 12       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 13      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 14          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 15  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 16  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 17    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 18  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 19  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 20  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 21                                  Shigella flexneri            623   \n",
       " 22  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 1   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 2   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 3   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 4   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 5   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 6   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 7   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 8   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 9   Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 10  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 11  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 12  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 13  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 14  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 15  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 16  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 17  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 18  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 19  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 20  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 21  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " 22  Phosphoenolpyruvate carboxykinase (ATP) (PCK) ...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  \\\n",
       " 0                   NaN         NaN   \n",
       " 1                   NaN         NaN   \n",
       " 2                   NaN         NaN   \n",
       " 3                   NaN         NaN   \n",
       " 4                   NaN         NaN   \n",
       " 5                   NaN         NaN   \n",
       " 6                   NaN         NaN   \n",
       " 7                   NaN         NaN   \n",
       " 8                   NaN         NaN   \n",
       " 9                   NaN         NaN   \n",
       " 10                  NaN         NaN   \n",
       " 11                  NaN         NaN   \n",
       " 12                  NaN         NaN   \n",
       " 13                  NaN         NaN   \n",
       " 14                  NaN         NaN   \n",
       " 15                  NaN         NaN   \n",
       " 16                  NaN         NaN   \n",
       " 17                  NaN         NaN   \n",
       " 18                  NaN         NaN   \n",
       " 19                  NaN         NaN   \n",
       " 20                  NaN         NaN   \n",
       " 21                  NaN         NaN   \n",
       " 22                  NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 1   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 2   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 3   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 4   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 5   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 6   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 7   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 8   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 9   MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 10  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 11  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 12  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 13  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 14  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 15  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 16  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 17  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 18  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 19  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 20  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 21  MOD_RES 87; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 22                                                NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " 21                              NaN         NaN             NaN   \n",
       " 22                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                                                PDB  \n",
       " 0              NaN  1AQ2;1AYL;1K3C;1K3D;1OEN;1OS1;2OLQ;2OLR;2PXZ;2...  \n",
       " 1              NaN                                                NaN  \n",
       " 2              NaN                                                NaN  \n",
       " 3              NaN                                                NaN  \n",
       " 4              NaN                                                NaN  \n",
       " 5              NaN                                                NaN  \n",
       " 6              NaN                                                NaN  \n",
       " 7              NaN                                                NaN  \n",
       " 8              NaN                                                NaN  \n",
       " 9              NaN                                                NaN  \n",
       " 10             NaN                                                NaN  \n",
       " 11             NaN                                                NaN  \n",
       " 12             NaN                                                NaN  \n",
       " 13             NaN                                                NaN  \n",
       " 14             NaN                                                NaN  \n",
       " 15             NaN                                                NaN  \n",
       " 16             NaN                                                NaN  \n",
       " 17             NaN                                                NaN  \n",
       " 18             NaN                                                NaN  \n",
       " 19             NaN                                                NaN  \n",
       " 20             NaN                                                NaN  \n",
       " 21             NaN                                                NaN  \n",
       " 22             NaN                                                NaN  \n",
       " \n",
       " [23 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name     Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P11715  CP17A_RAT  Cyp17a1 Cyp17              Cyp17a1                Cyp17   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Steroid 17-alpha-hydroxylase/17,20 lyase (EC 1...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P17169  GLMS_ECOLI   glmS b3729 JW3707                 glmS   \n",
       " 1  Q8XEG2  GLMS_ECO57  glmS Z5227 ECs4671                 glmS   \n",
       " 2  Q8FBT4  GLMS_ECOL6          glmS c4654                 glmS   \n",
       " 3  Q83IY4  GLMS_SHIFL   glmS SF3809 S3959                 glmS   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN               b3729 JW3707               NaN   \n",
       " 1                   NaN              Z5227 ECs4671               NaN   \n",
       " 2                   NaN                      c4654               NaN   \n",
       " 3                   NaN               SF3809 S3959               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                           Escherichia coli O157:H7          83334   \n",
       " 2  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 3                                  Shigella flexneri            623   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutamine--fructose-6-phosphate aminotransfera...  ...           NaN   \n",
       " 1  Glutamine--fructose-6-phosphate aminotransfera...  ...           NaN   \n",
       " 2  Glutamine--fructose-6-phosphate aminotransfera...  ...           NaN   \n",
       " 3  Glutamine--fructose-6-phosphate aminotransfera...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                        INIT_MET 1; /note=\"Removed\"         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " 2               NaN     NaN                              NaN         NaN   \n",
       " 3               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1JXA;1MOQ;1MOR;1MOS;1XFF;1XFG;2J6H;2VF4;2VF5;3...  \n",
       " 1                                                NaN  \n",
       " 2                                                NaN  \n",
       " 3                                                NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry   Entry Name                         Gene Names  \\\n",
       " 0  P42330  AK1C3_HUMAN  AKR1C3 DDH1 HSD17B5 KIAA0119 PGFS   \n",
       " \n",
       "   Gene Names (primary)        Gene Names (synonym)  \\\n",
       " 0               AKR1C3  DDH1 HSD17B5 KIAA0119 PGFS   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Aldo-keto reductase family 1 member C3 (EC 1.1...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1RY0;1RY8;1S1P;1S1R;1S2A;1S2C;1XF0;1ZQ5;2F38;2...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "      Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   P0A6I3  COAA_ECOLI    coaA panK rts b3974 JW3942                 coaA   \n",
       " 1   P0A6I5  COAA_ECO57   coaA panK rts Z5545 ECs4901                 coaA   \n",
       " 2   A1AIF2  COAA_ECOK1  coaA Ecok1_39480 APECO1_2493                 coaA   \n",
       " 3   Q1R5X9  COAA_ECOUT              coaA UTI89_C3802                 coaA   \n",
       " 4   Q0TA86  COAA_ECOL5                 coaA ECP_4188                 coaA   \n",
       " 5   P0A6I4  COAA_ECOL6           coaA panK rts c4933                 coaA   \n",
       " 6   Q45338  COAX_BORPE               coaX baf BP0097                 coaX   \n",
       " 7   B1XBY1  COAA_ECODH             coaA ECDH10B_4163                 coaA   \n",
       " 8   Q83PC5  COAA_SHIFL         coaA rts SF4052 S3688                 coaA   \n",
       " 9   B7LA72  COAA_ECO55             coaA EC55989_4460                 coaA   \n",
       " 10  B7M726  COAA_ECO8A              coaA ECIAI1_4191                 coaA   \n",
       " 11  B7MR65  COAA_ECO81               coaA ECED1_4685                 coaA   \n",
       " 12  C5A0R9  COAA_ECOBW                 coaA BWG_3638                 coaA   \n",
       " 13  A8A778  COAA_ECOHS               coaA EcHS_A4208                 coaA   \n",
       " 14  A7ZUJ0  COAA_ECO24           coaA EcE24377A_4513                 coaA   \n",
       " 15  B7NR61  COAA_ECO7I             coaA ECIAI39_4363                 coaA   \n",
       " 16  B7MIW5  COAA_ECO45               coaA ECS88_4438                 coaA   \n",
       " 17  B1LNT0  COAA_ECOSM             coaA EcSMS35_4422                 coaA   \n",
       " 18  B7UNV0  COAA_ECO27              coaA E2348C_4286                 coaA   \n",
       " 19  B7NFR9  COAA_ECOLU               coaA ECUMN_4500                 coaA   \n",
       " 20  B7LUM3  COAA_ESCF3                coaA EFER_3777                 coaA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0              panK rts               b3974 JW3942              NaN   \n",
       " 1              panK rts              Z5545 ECs4901              NaN   \n",
       " 2                   NaN                Ecok1_39480      APECO1_2493   \n",
       " 3                   NaN                UTI89_C3802              NaN   \n",
       " 4                   NaN                   ECP_4188              NaN   \n",
       " 5              panK rts                      c4933              NaN   \n",
       " 6                   baf                     BP0097              NaN   \n",
       " 7                   NaN               ECDH10B_4163              NaN   \n",
       " 8                   rts               SF4052 S3688              NaN   \n",
       " 9                   NaN               EC55989_4460              NaN   \n",
       " 10                  NaN                ECIAI1_4191              NaN   \n",
       " 11                  NaN                 ECED1_4685              NaN   \n",
       " 12                  NaN                   BWG_3638              NaN   \n",
       " 13                  NaN                 EcHS_A4208              NaN   \n",
       " 14                  NaN             EcE24377A_4513              NaN   \n",
       " 15                  NaN               ECIAI39_4363              NaN   \n",
       " 16                  NaN                 ECS88_4438              NaN   \n",
       " 17                  NaN               EcSMS35_4422              NaN   \n",
       " 18                  NaN                E2348C_4286              NaN   \n",
       " 19                  NaN                 ECUMN_4500              NaN   \n",
       " 20                  NaN                  EFER_3777              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                            Escherichia coli O157:H7          83334   \n",
       " 2                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 3              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 4     Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 5   Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 6   Bordetella pertussis (strain Tohama I / ATCC B...         257313   \n",
       " 7               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 8                                   Shigella flexneri            623   \n",
       " 9              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 10                  Escherichia coli O8 (strain IAI1)         585034   \n",
       " 11                 Escherichia coli O81 (strain ED1a)         585397   \n",
       " 12    Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 13                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 14  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 15      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 16       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 17          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 18  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 19  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 20  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 1   Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 2   Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 3   Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 4   Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 5   Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 6   Type III pantothenate kinase (EC 2.7.1.33) (Bv...  ...           NaN   \n",
       " 7   Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 8   Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 9   Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 10  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 11  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 12  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 13  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 14  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 15  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 16  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 17  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 18  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 19  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " 20  Pantothenate kinase (EC 2.7.1.33) (Pantothenic...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                   NaN         NaN               NaN     NaN   \n",
       " 1                   NaN         NaN               NaN     NaN   \n",
       " 2                   NaN         NaN               NaN     NaN   \n",
       " 3                   NaN         NaN               NaN     NaN   \n",
       " 4                   NaN         NaN               NaN     NaN   \n",
       " 5                   NaN         NaN               NaN     NaN   \n",
       " 6                   NaN         NaN               NaN     NaN   \n",
       " 7                   NaN         NaN               NaN     NaN   \n",
       " 8                   NaN         NaN               NaN     NaN   \n",
       " 9                   NaN         NaN               NaN     NaN   \n",
       " 10                  NaN         NaN               NaN     NaN   \n",
       " 11                  NaN         NaN               NaN     NaN   \n",
       " 12                  NaN         NaN               NaN     NaN   \n",
       " 13                  NaN         NaN               NaN     NaN   \n",
       " 14                  NaN         NaN               NaN     NaN   \n",
       " 15                  NaN         NaN               NaN     NaN   \n",
       " 16                  NaN         NaN               NaN     NaN   \n",
       " 17                  NaN         NaN               NaN     NaN   \n",
       " 18                  NaN         NaN               NaN     NaN   \n",
       " 19                  NaN         NaN               NaN     NaN   \n",
       " 20                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide              PDB  \n",
       " 0              NaN  1ESM;1ESN;1SQ5;  \n",
       " 1              NaN              NaN  \n",
       " 2              NaN              NaN  \n",
       " 3              NaN              NaN  \n",
       " 4              NaN              NaN  \n",
       " 5              NaN              NaN  \n",
       " 6              NaN              NaN  \n",
       " 7              NaN              NaN  \n",
       " 8              NaN              NaN  \n",
       " 9              NaN              NaN  \n",
       " 10             NaN              NaN  \n",
       " 11             NaN              NaN  \n",
       " 12             NaN              NaN  \n",
       " 13             NaN              NaN  \n",
       " 14             NaN              NaN  \n",
       " 15             NaN              NaN  \n",
       " 16             NaN              NaN  \n",
       " 17             NaN              NaN  \n",
       " 18             NaN              NaN  \n",
       " 19             NaN              NaN  \n",
       " 20             NaN              NaN  \n",
       " \n",
       " [21 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P31327  CPSM_HUMAN       CPS1                 CPS1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Carbamoyl-phosphate synthase [ammonia], mitoch...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 537; /note=\"O-linked (GlcNAc) serine;...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 55; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Undergoes proteolytic cleavage in the C-t...         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  \\\n",
       " 0             NaN  TRANSIT 1..38; /note=\"Mitochondrion\"; /evidenc...   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  2YVQ;4UTR;4UTV;4UTX;4UTZ;4UU7;4UU8;4UUA;4UUB;5...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "      Entry  Entry Name                     Gene Names Gene Names (primary)  \\\n",
       " 0   P0A6J1  CYSC_ECOLI              cysC b2750 JW2720                 cysC   \n",
       " 1   B6I6E0  CYSC_ECOSE                 cysC ECSE_3002                 cysC   \n",
       " 2   P0A6J2  CYSC_ECO57             cysC Z4058 ECs3604                 cysC   \n",
       " 3   B1XCS6  CYSC_ECODH              cysC ECDH10B_2918                 cysC   \n",
       " 4   B7LEG8  CYSC_ECO55              cysC EC55989_3023                 cysC   \n",
       " 5   B7MYQ5  CYSC_ECO81                cysC ECED1_3206                 cysC   \n",
       " 6   A1AEU4  CYSC_ECOK1  cysC Ecok1_26900 APECO1_37732                 cysC   \n",
       " 7   B7LXG2  CYSC_ECO8A               cysC ECIAI1_2852                 cysC   \n",
       " 8   Q1R7U1  CYSC_ECOUT               cysC UTI89_C3121                 cysC   \n",
       " 9   C4ZZQ4  CYSC_ECOBW                  cysC BWG_2486                 cysC   \n",
       " 10  A8A3M9  CYSC_ECOHS                cysC EcHS_A2888                 cysC   \n",
       " 11  A7ZQJ4  CYSC_ECO24            cysC EcE24377A_3051                 cysC   \n",
       " 12  B5Z3B2  CYSC_ECO5E             cysC ECH74115_4002                 cysC   \n",
       " 13  B7MKM4  CYSC_ECO45                cysC ECS88_3021                 cysC   \n",
       " 14  B1LQ71  CYSC_ECOSM              cysC EcSMS35_2876                 cysC   \n",
       " 15  B7NT94  CYSC_ECO7I              cysC ECIAI39_2939                 cysC   \n",
       " 16  B7UHG9  CYSC_ECO27               cysC E2348C_3020                 cysC   \n",
       " 17  B7N6Y0  CYSC_ECOLU                cysC ECUMN_3074                 cysC   \n",
       " 18  Q0TEA8  CYSC_ECOL5                  cysC ECP_2732                 cysC   \n",
       " 19  Q8FEJ2  CYSC_ECOL6                     cysC c3317                 cysC   \n",
       " 20  B1IUS9  CYSC_ECOLC                cysC EcolC_0962                 cysC   \n",
       " 21  B7LWK5  CYSC_ESCF3                 cysC EFER_0317                 cysC   \n",
       " 22  P29811  CYSC_PSEAI                            NaN                  NaN   \n",
       " 23  Q83JX9  CYSC_SHIFL              cysC SF2773 S2966                 cysC   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN               b2750 JW2720              NaN   \n",
       " 1                    NaN                  ECSE_3002              NaN   \n",
       " 2                    NaN              Z4058 ECs3604              NaN   \n",
       " 3                    NaN               ECDH10B_2918              NaN   \n",
       " 4                    NaN               EC55989_3023              NaN   \n",
       " 5                    NaN                 ECED1_3206              NaN   \n",
       " 6                    NaN                Ecok1_26900     APECO1_37732   \n",
       " 7                    NaN                ECIAI1_2852              NaN   \n",
       " 8                    NaN                UTI89_C3121              NaN   \n",
       " 9                    NaN                   BWG_2486              NaN   \n",
       " 10                   NaN                 EcHS_A2888              NaN   \n",
       " 11                   NaN             EcE24377A_3051              NaN   \n",
       " 12                   NaN              ECH74115_4002              NaN   \n",
       " 13                   NaN                 ECS88_3021              NaN   \n",
       " 14                   NaN               EcSMS35_2876              NaN   \n",
       " 15                   NaN               ECIAI39_2939              NaN   \n",
       " 16                   NaN                E2348C_3020              NaN   \n",
       " 17                   NaN                 ECUMN_3074              NaN   \n",
       " 18                   NaN                   ECP_2732              NaN   \n",
       " 19                   NaN                      c3317              NaN   \n",
       " 20                   NaN                 EcolC_0962              NaN   \n",
       " 21                   NaN                  EFER_0317              NaN   \n",
       " 22                   NaN                        NaN              NaN   \n",
       " 23                   NaN               SF2773 S2966              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain SE11)         409438   \n",
       " 2                            Escherichia coli O157:H7          83334   \n",
       " 3               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 4              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 5                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 6                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 7                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 8              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 9     Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 10                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 11  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 12    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 13       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 14          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 15      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 16  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 17  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 18    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 19  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 20  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 21  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 22                             Pseudomonas aeruginosa            287   \n",
       " 23                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 1   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 2   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 3   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 4   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 5   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 6   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 7   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 8   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 9   Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 10  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 11  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 12  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 13  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 14  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 15  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 16  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 17  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 18  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 19  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 20  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 21  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " 22  Probable adenylyl-sulfate kinase (EC 2.7.1.25)...  ...           NaN   \n",
       " 23  Adenylyl-sulfate kinase (EC 2.7.1.25) (APS kin...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19                        INIT_MET 1; /note=\"Removed\"         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22                                                NaN         NaN   \n",
       " 23                                                NaN         NaN   \n",
       " \n",
       "     Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0                NaN     NaN                              NaN         NaN   \n",
       " 1                NaN     NaN                              NaN         NaN   \n",
       " 2                NaN     NaN                              NaN         NaN   \n",
       " 3                NaN     NaN                              NaN         NaN   \n",
       " 4                NaN     NaN                              NaN         NaN   \n",
       " 5                NaN     NaN                              NaN         NaN   \n",
       " 6                NaN     NaN                              NaN         NaN   \n",
       " 7                NaN     NaN                              NaN         NaN   \n",
       " 8                NaN     NaN                              NaN         NaN   \n",
       " 9                NaN     NaN                              NaN         NaN   \n",
       " 10               NaN     NaN                              NaN         NaN   \n",
       " 11               NaN     NaN                              NaN         NaN   \n",
       " 12               NaN     NaN                              NaN         NaN   \n",
       " 13               NaN     NaN                              NaN         NaN   \n",
       " 14               NaN     NaN                              NaN         NaN   \n",
       " 15               NaN     NaN                              NaN         NaN   \n",
       " 16               NaN     NaN                              NaN         NaN   \n",
       " 17               NaN     NaN                              NaN         NaN   \n",
       " 18               NaN     NaN                              NaN         NaN   \n",
       " 19               NaN     NaN                              NaN         NaN   \n",
       " 20               NaN     NaN                              NaN         NaN   \n",
       " 21               NaN     NaN                              NaN         NaN   \n",
       " 22               NaN     NaN                              NaN         NaN   \n",
       " 23               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  PDB  \n",
       " 0              NaN             NaN  NaN  \n",
       " 1              NaN             NaN  NaN  \n",
       " 2              NaN             NaN  NaN  \n",
       " 3              NaN             NaN  NaN  \n",
       " 4              NaN             NaN  NaN  \n",
       " 5              NaN             NaN  NaN  \n",
       " 6              NaN             NaN  NaN  \n",
       " 7              NaN             NaN  NaN  \n",
       " 8              NaN             NaN  NaN  \n",
       " 9              NaN             NaN  NaN  \n",
       " 10             NaN             NaN  NaN  \n",
       " 11             NaN             NaN  NaN  \n",
       " 12             NaN             NaN  NaN  \n",
       " 13             NaN             NaN  NaN  \n",
       " 14             NaN             NaN  NaN  \n",
       " 15             NaN             NaN  NaN  \n",
       " 16             NaN             NaN  NaN  \n",
       " 17             NaN             NaN  NaN  \n",
       " 18             NaN             NaN  NaN  \n",
       " 19             NaN             NaN  NaN  \n",
       " 20             NaN             NaN  NaN  \n",
       " 21             NaN             NaN  NaN  \n",
       " 22             NaN             NaN  NaN  \n",
       " 23             NaN             NaN  NaN  \n",
       " \n",
       " [24 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q3ZCJ8  CATC_BOVIN       CTSC                 CTSC                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Dipeptidyl peptidase 1 (EC 3.4.14.1) (Cathepsi...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 29; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 135..230; /evidence=\"ECO:0000250\"; /id=...   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..24; /evidence=\"ECO:0000250\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P07024  USHA_ECOLI  ushA b0480 JW0469                 ushA   \n",
       " 1  P52307  5NTD_RHIMP                NaN                  NaN   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN               b0480 JW0469               NaN   \n",
       " 1                   NaN                        NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1  Rhipicephalus microplus (Cattle tick) (Boophil...           6941   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Protein UshA [Includes: UDP-sugar hydrolase (E...  ...   \n",
       " 1  Protein 5NUC [Includes: UDP-sugar hydrolase (E...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0                                                NaN                  NaN   \n",
       " 1  CARBOHYD 172; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "                                           Lipidation Modified residue Peptide  \\\n",
       " 0                                                NaN              NaN     NaN   \n",
       " 1  LIPID 555; /note=\"GPI-anchor amidated asparagi...              NaN     NaN   \n",
       " \n",
       "    Post-translational modification  \\\n",
       " 0                              NaN   \n",
       " 1               PTM: Glycosylated.   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0                                                NaN   \n",
       " 1  PROPEP 556..580; /note=\"Removed in mature form...   \n",
       " \n",
       "                                       Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..25; /evidence=\"ECO:0000269|PubMed:30...             NaN   \n",
       " 1  SIGNAL <1..14; /evidence=\"ECO:0000269|PubMed:8...             NaN   \n",
       " \n",
       "                                              PDB  \n",
       " 0  1HO5;1HP1;1HPU;1OI8;1OID;1OIE;1USH;2USH;4WWL;  \n",
       " 1                                            NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P26997  XYLA_THET8       xylA                 xylA                   NaN   \n",
       " 1  P56681  XYLA_THECA       xylA                 xylA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Thermus thermophilus (strain ATCC 27634 / DSM ...         300852   \n",
       " 1                                Thermus caldophilus            272   \n",
       " \n",
       "                    Protein names  ...  Glycosylation Initiator methionine  \\\n",
       " 0  Xylose isomerase (EC 5.3.1.5)  ...            NaN                  NaN   \n",
       " 1  Xylose isomerase (EC 5.3.1.5)  ...            NaN                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide    PDB  \n",
       " 0         NaN             NaN             NaN  1BXB;  \n",
       " 1         NaN             NaN             NaN  1BXC;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  Q1CT39  GATA_HELPH  gatA HPAG1_0816                 gatA   \n",
       " 1  B5Z7J5  GATA_HELPG   gatA HPG27_789                 gatA   \n",
       " 2  B2USZ8  GATA_HELPS  gatA HPSH_02650                 gatA   \n",
       " 3  B6JM61  GATA_HELP2  gatA HPP12_0837                 gatA   \n",
       " 4  P56114  GATA_HELPY     gatA HP_0830                 gatA   \n",
       " 5  Q9ZL13  GATA_HELPJ    gatA jhp_0769                 gatA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                 HPAG1_0816               NaN   \n",
       " 1                   NaN                  HPG27_789               NaN   \n",
       " 2                   NaN                 HPSH_02650               NaN   \n",
       " 3                   NaN                 HPP12_0837               NaN   \n",
       " 4                   NaN                    HP_0830               NaN   \n",
       " 5                   NaN                   jhp_0769               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                 Helicobacter pylori (strain HPAG1)         357544   \n",
       " 1                   Helicobacter pylori (strain G27)         563041   \n",
       " 2                Helicobacter pylori (strain Shi470)         512562   \n",
       " 3                   Helicobacter pylori (strain P12)         570508   \n",
       " 4  Helicobacter pylori (strain ATCC 700392 / 2669...          85962   \n",
       " 5  Helicobacter pylori (strain J99 / ATCC 700824)...          85963   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutamyl-tRNA(Gln) amidotransferase subunit A ...  ...           NaN   \n",
       " 1  Glutamyl-tRNA(Gln) amidotransferase subunit A ...  ...           NaN   \n",
       " 2  Glutamyl-tRNA(Gln) amidotransferase subunit A ...  ...           NaN   \n",
       " 3  Glutamyl-tRNA(Gln) amidotransferase subunit A ...  ...           NaN   \n",
       " 4  Glutamyl-tRNA(Gln) amidotransferase subunit A ...  ...           NaN   \n",
       " 5  Glutamyl-tRNA(Gln) amidotransferase subunit A ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " 5             NaN  NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P9WMS7  GUAA_MYCTU  guaA Rv3396c MTCY78.32                 guaA   \n",
       " 1  P9WMS6  GUAA_MYCTO             guaA MT3504                 guaA   \n",
       " 2  A5U871  GUAA_MYCTA           guaA MRA_3436                 guaA   \n",
       " 3  A1KP87  GUAA_MYCBP          guaA BCG_3466c                 guaA   \n",
       " 4  C1AHL0  GUAA_MYCBT           guaA JTY_3466                 guaA   \n",
       " 5  P0A5A2  GUAA_MYCBO     guaA BQ2027_MB3429C                 guaA   \n",
       " 6  B2HDP0  GUAA_MYCMM          guaA MMAR_1148                 guaA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    Rv3396c        MTCY78.32   \n",
       " 1                   NaN                     MT3504              NaN   \n",
       " 2                   NaN                   MRA_3436              NaN   \n",
       " 3                   NaN                  BCG_3466c              NaN   \n",
       " 4                   NaN                   JTY_3466              NaN   \n",
       " 5                   NaN             BQ2027_MB3429C              NaN   \n",
       " 6                   NaN                  MMAR_1148              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 2  Mycobacterium tuberculosis (strain ATCC 25177 ...         419947   \n",
       " 3  Mycobacterium bovis (strain BCG / Pasteur 1173P2)         410289   \n",
       " 4  Mycobacterium bovis (strain BCG / Tokyo 172 / ...         561275   \n",
       " 5  Mycobacterium bovis (strain ATCC BAA-935 / AF2...         233413   \n",
       " 6    Mycobacterium marinum (strain ATCC BAA-535 / M)         216594   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  GMP synthase [glutamine-hydrolyzing] (EC 6.3.5...  ...           NaN   \n",
       " 1  GMP synthase [glutamine-hydrolyzing] (EC 6.3.5...  ...           NaN   \n",
       " 2  GMP synthase [glutamine-hydrolyzing] (EC 6.3.5...  ...           NaN   \n",
       " 3  GMP synthase [glutamine-hydrolyzing] (EC 6.3.5...  ...           NaN   \n",
       " 4  GMP synthase [glutamine-hydrolyzing] (EC 6.3.5...  ...           NaN   \n",
       " 5  GMP synthase [glutamine-hydrolyzing] (EC 6.3.5...  ...           NaN   \n",
       " 6  GMP synthase [glutamine-hydrolyzing] (EC 6.3.5...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylvaline\"; /evidence=\"...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " 4                                                NaN     NaN   \n",
       " 5                                                NaN     NaN   \n",
       " 6                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " 5             NaN  NaN  \n",
       " 6             NaN  NaN  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name   Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P21146  ARBK1_BOVIN  GRK2 ADRBK1                 GRK2               ADRBK1   \n",
       " 1  P26818  ARBK2_BOVIN  GRK3 ADRBK2                 GRK3               ADRBK2   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 1                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Beta-adrenergic receptor kinase 1 (Beta-ARK-1)...  ...   \n",
       " 1           9913  Beta-adrenergic receptor kinase 2 (Beta-ARK-2)...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 670; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "       Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                                 NaN         NaN             NaN   \n",
       " 1  PTM: Ubiquitinated. {ECO:0000250}.         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1OMW;1YM7;2BCJ;3PSC;3PVU;3PVW;3UZS;3UZT;5HE0;5...  \n",
       " 1             NaN                                                NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q4JCJ1  CAPPA_SULAC  ppcA Saci_0059                 ppcA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                  Saci_0059               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Sulfolobus acidocaldarius (strain ATCC 33909 /...         330779   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphoenolpyruvate carboxylase (PEPC) (PEPCas...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P50583  AP4A_HUMAN  NUDT2 APAH1                NUDT2                APAH1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Bis(5'-nucleosyl)-tetraphosphatase [asymmetric...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  1XSA;1XSB;1XSC;3U53;4ICK;4IJX;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  P80576  AROF_NEUCR      aro-8 aroh NCU02785                aro-8   \n",
       " 1  P80575  AROF_STRRM                     aroH                 aroH   \n",
       " 2  P80574  AROF_STRCO  aroH SCO2115 SC6E10.09c                 aroH   \n",
       " 3  P55911  AROF_STRLI                      NaN                  NaN   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 aroh                        NaN         NCU02785   \n",
       " 1                  NaN                        NaN              NaN   \n",
       " 2                  NaN                    SCO2115       SC6E10.09c   \n",
       " 3                  NaN                        NaN              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Neurospora crassa (strain ATCC 24698 / 74-OR23...         367110   \n",
       " 1                               Streptomyces rimosus           1927   \n",
       " 2  Streptomyces coelicolor (strain ATCC BAA-471 /...         100226   \n",
       " 3                              Streptomyces lividans           1916   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phospho-2-dehydro-3-deoxyheptonate aldolase (E...  ...           NaN   \n",
       " 1  Phospho-2-dehydro-3-deoxyheptonate aldolase (E...  ...           NaN   \n",
       " 2  Phospho-2-dehydro-3-deoxyheptonate aldolase (E...  ...           NaN   \n",
       " 3  Phospho-2-dehydro-3-deoxyheptonate aldolase (E...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "   Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0              NaN     NaN  PTM: The N-terminus is blocked.         NaN   \n",
       " 1              NaN     NaN  PTM: The N-terminus is blocked.         NaN   \n",
       " 2              NaN     NaN                              NaN         NaN   \n",
       " 3              NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " 3             NaN             NaN  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q1W2L8  GSH1_TOBAC       GSH1                 GSH1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                              Organism  Organism (ID)  \\\n",
       " 0  Nicotiana tabacum (Common tobacco)           4097   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutamate--cysteine ligase, chloroplastic (EC ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: The Cys-186-Cys-406 disulfide bridge is k...         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  PDB  \n",
       " 0             NaN  TRANSIT 1..45; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q8BNJ2  ATS4_MOUSE    Adamts4              Adamts4                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  A disintegrin and metalloproteinase with throm...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 63; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: The precursor is cleaved by a furin endop...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 50..208; /evidence=\"ECO:0000250\"; /id=\"...   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..49; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P17182  ENOA_MOUSE  Eno1 Eno-1                 Eno1                Eno-1   \n",
       " 1  P21550  ENOB_MOUSE  Eno3 Eno-3                 Eno3                Eno-3   \n",
       " 2  Q8C042  ENO4_MOUSE        Eno4                 Eno4                  NaN   \n",
       " 3  P17183  ENOG_MOUSE  Eno2 Eno-2                 Eno2                Eno-2   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 1                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 2                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 3                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Alpha-enolase (EC 4.2.1.11) (2-phospho-D-glyce...  ...   \n",
       " 1          10090  Beta-enolase (EC 4.2.1.11) (2-phospho-D-glycer...  ...   \n",
       " 2          10090  Enolase 4 (EC 4.2.1.11) (2-phospho-D-glycerate...  ...   \n",
       " 3          10090  Gamma-enolase (EC 4.2.1.11) (2-phospho-D-glyce...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 2           NaN                                                NaN   \n",
       " 3           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 1         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " 3         NaN  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: ISGylated. {ECO:0000269|PubMed:16139798}....         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: Synthesized as an approximately 70-kDa pr...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " 3             NaN             NaN  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P20646   PPAP_RAT  Acp3 Acpp                 Acp3                 Acpp   \n",
       " 1  P29288   PPA5_RAT       Acp5                 Acp5                  NaN   \n",
       " 2  P20611   PPAL_RAT       Acp2                 Acp2                  NaN   \n",
       " 3  P41498   PPAC_RAT       Acp1                 Acp1                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 2                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 3                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Prostatic acid phosphatase (EC 3.1.3.2) (5'-nu...  ...   \n",
       " 1          10116  Tartrate-resistant acid phosphatase type 5 (TR...  ...   \n",
       " 2          10116      Lysosomal acid phosphatase (LAP) (EC 3.1.3.2)  ...   \n",
       " 3          10116  Low molecular weight phosphotyrosine protein p...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0  CARBOHYD 93; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 1  CARBOHYD 118; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 2  CARBOHYD 92; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 3                                                NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated. {ECO:0000269|PubMed:83349...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: The membrane-bound form is converted to t...         NaN   \n",
       " 3  PTM: [Isoform 2]: Phosphorylated by LCK. Phosp...         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..31; /evidence=\"ECO:0000250|UniProtKB...             NaN   \n",
       " 1  SIGNAL 1..22; /evidence=\"ECO:0000269|PubMed:18...             NaN   \n",
       " 2                                       SIGNAL 1..30             NaN   \n",
       " 3                                                NaN             NaN   \n",
       " \n",
       "           PDB  \n",
       " 0  1RPA;1RPT;  \n",
       " 1  1QFC;1QHW;  \n",
       " 2         NaN  \n",
       " 3         NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q64560   TPP2_RAT       Tpp2                 Tpp2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Tripeptidyl-peptidase 2 (TPP-2) (EC 3.4.14.10)...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                              Gene Names  \\\n",
       " 0  Q96PD5  PGRP2_HUMAN  PGLYRP2 PGLYRPL PGRPL UNQ3103/PRO10102   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym)  Gene Names (ordered locus)  \\\n",
       " 0              PGLYRP2        PGLYRPL PGRPL                         NaN   \n",
       " \n",
       "    Gene Names (ORF)              Organism  Organism (ID)  \\\n",
       " 0  UNQ3103/PRO10102  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.2...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 77; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 239; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..21; /evidence=\"ECO:0000269|PubMed:15...             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  O06961  TDCD_SALTY  tdcD oxd-2 STM3242                 tdcD   \n",
       " 1  P74879  PDUW_SALTY        pduW STM2057                 pduW   \n",
       " 2  Q8Z3K5  TDCD_SALTI  tdcD STY3424 t3162                 tdcD   \n",
       " 3  B5F6P9  TDCD_SALA4     tdcD SeAg_B3430                 tdcD   \n",
       " 4  C0Q1L8  PDUW_SALPC       pduW SPC_1657                 pduW   \n",
       " 5  C9XHG0  TDCD_SALTD    tdcD STMMW_32421                 tdcD   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                oxd-2                    STM3242               NaN   \n",
       " 1                  NaN                    STM2057               NaN   \n",
       " 2                  NaN              STY3424 t3162               NaN   \n",
       " 3                  NaN                 SeAg_B3430               NaN   \n",
       " 4                  NaN                   SPC_1657               NaN   \n",
       " 5                  NaN                STMMW_32421               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 1  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 2                                   Salmonella typhi          90370   \n",
       " 3                    Salmonella agona (strain SL483)         454166   \n",
       " 4            Salmonella paratyphi C (strain RKS4594)         476213   \n",
       " 5             Salmonella typhimurium (strain D23580)         568708   \n",
       " \n",
       "                           Protein names  ... Glycosylation  \\\n",
       " 0       Propionate kinase (EC 2.7.2.15)  ...           NaN   \n",
       " 1  Propionate kinase PduW (EC 2.7.2.15)  ...           NaN   \n",
       " 2       Propionate kinase (EC 2.7.2.15)  ...           NaN   \n",
       " 3       Propionate kinase (EC 2.7.2.15)  ...           NaN   \n",
       " 4       Propionate kinase (EC 2.7.2.15)  ...           NaN   \n",
       " 5       Propionate kinase (EC 2.7.2.15)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " 5                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1X3M;1X3N;2E1Y;2E1Z;2E20;4FWK;4FWL;4FWM;4FWN;4...  \n",
       " 1             NaN                                                NaN  \n",
       " 2             NaN                                                NaN  \n",
       " 3             NaN                                                NaN  \n",
       " 4             NaN                                                NaN  \n",
       " 5             NaN                                                NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P47990  XDH_CHICK        XDH                  XDH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Gallus gallus (Chicken)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9031  Xanthine dehydrogenase/oxidase [Includes: Xant...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0  O14495  PLPP3_HUMAN             PLPP3 LPP3 PPAP2B                PLPP3   \n",
       " 1  Q8NEB5  PLPP5_HUMAN    PLPP5 DPPL1 HTPAP PPAPDC1B                PLPP5   \n",
       " 2  O43688  PLPP2_HUMAN             PLPP2 LPP2 PPAP2C                PLPP2   \n",
       " 3  Q9BQK8  LPIN3_HUMAN                  LPIN3 LIPN3L                LPIN3   \n",
       " 4  Q5VZY2  PLPP4_HUMAN  PLPP4 DPPL2 PPAPDC1 PPAPDC1A                PLPP4   \n",
       " 5  Q92539  LPIN2_HUMAN                LPIN2 KIAA0249                LPIN2   \n",
       " 6  O14494  PLPP1_HUMAN             PLPP1 LPP1 PPAP2A                PLPP1   \n",
       " 7  Q14693  LPIN1_HUMAN                LPIN1 KIAA0188                LPIN1   \n",
       " \n",
       "      Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0             LPP3 PPAP2B                         NaN               NaN   \n",
       " 1    DPPL1 HTPAP PPAPDC1B                         NaN               NaN   \n",
       " 2             LPP2 PPAP2C                         NaN               NaN   \n",
       " 3                  LIPN3L                         NaN               NaN   \n",
       " 4  DPPL2 PPAPDC1 PPAPDC1A                         NaN               NaN   \n",
       " 5                KIAA0249                         NaN               NaN   \n",
       " 6             LPP1 PPAP2A                         NaN               NaN   \n",
       " 7                KIAA0188                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " 3  Homo sapiens (Human)           9606   \n",
       " 4  Homo sapiens (Human)           9606   \n",
       " 5  Homo sapiens (Human)           9606   \n",
       " 6  Homo sapiens (Human)           9606   \n",
       " 7  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Phospholipid phosphatase 3 (EC 3.1.3.-) (EC 3....  ...   \n",
       " 1  Phospholipid phosphatase 5 (EC 3.1.3.4) (EC 3....  ...   \n",
       " 2  Phospholipid phosphatase 2 (EC 3.1.3.-) (EC 3....  ...   \n",
       " 3  Phosphatidate phosphatase LPIN3 (EC 3.1.3.4) (...  ...   \n",
       " 4  Phospholipid phosphatase 4 (EC 3.1.3.4) (EC 3....  ...   \n",
       " 5  Phosphatidate phosphatase LPIN2 (EC 3.1.3.4) (...  ...   \n",
       " 6  Phospholipid phosphatase 1 (EC 3.1.3.-) (EC 3....  ...   \n",
       " 7  Phosphatidate phosphatase LPIN1 (EC 3.1.3.4) (...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 170; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1                                                NaN                  NaN   \n",
       " 2  CARBOHYD 139; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 3                                                NaN                  NaN   \n",
       " 4                                                NaN                  NaN   \n",
       " 5                                                NaN                  NaN   \n",
       " 6  CARBOHYD 142; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 7                                                NaN                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 19; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1         NaN                                                NaN     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " 3         NaN  MOD_RES 159; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 4         NaN                                                NaN     NaN   \n",
       " 5         NaN  MOD_RES 106; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 6         NaN                                                NaN     NaN   \n",
       " 7         NaN  MOD_RES 106; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated (PubMed:9705349). Contains...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: N-glycosylated. {ECO:0000269|PubMed:97053...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6  PTM: N-glycosylated (PubMed:9305923, PubMed:10...         NaN   \n",
       " 7  PTM: Phosphorylated at multiple sites in respo...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " 3             NaN             NaN  NaN  \n",
       " 4             NaN             NaN  NaN  \n",
       " 5             NaN             NaN  NaN  \n",
       " 6             NaN             NaN  NaN  \n",
       " 7             NaN             NaN  NaN  \n",
       " \n",
       " [8 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q0VD19  ASM_BOVIN      SMPD1                SMPD1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Sphingomyelin phosphodiesterase (EC 3.1.4.12) ...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 82; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 504; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Proteolytically processed. Mature lysosom...         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..40; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry   Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   Q15311   RBP1_HUMAN      RALBP1 RLIP RLIP1 RLIP76               RALBP1   \n",
       " 1   Q2M3G0  ABCB5_HUMAN                         ABCB5                ABCB5   \n",
       " 2   Q96J66   MRP8_HUMAN                   ABCC11 MRP8               ABCC11   \n",
       " 3   Q5T3U5   MRP7_HUMAN            ABCC10 MRP7 SIMRP7               ABCC10   \n",
       " 4   O15438   MRP3_HUMAN        ABCC3 CMOAT2 MLP2 MRP3                ABCC3   \n",
       " 5   O15439   MRP4_HUMAN              ABCC4 MOATB MRP4                ABCC4   \n",
       " 6   Q9UNQ0  ABCG2_HUMAN     ABCG2 ABCP BCRP BCRP1 MXR                ABCG2   \n",
       " 7   O15440   MRP5_HUMAN                    ABCC5 MRP5                ABCC5   \n",
       " 8   P08183   MDR1_HUMAN               ABCB1 MDR1 PGY1                ABCB1   \n",
       " 9   P33527   MRP1_HUMAN                ABCC1 MRP MRP1                ABCC1   \n",
       " 10  Q92887   MRP2_HUMAN  ABCC2 CMOAT CMOAT1 CMRP MRP2                ABCC2   \n",
       " 11  Q99758  ABCA3_HUMAN                    ABCA3 ABC3                ABCA3   \n",
       " \n",
       "       Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0        RLIP RLIP1 RLIP76                         NaN               NaN   \n",
       " 1                      NaN                         NaN               NaN   \n",
       " 2                     MRP8                         NaN               NaN   \n",
       " 3              MRP7 SIMRP7                         NaN               NaN   \n",
       " 4         CMOAT2 MLP2 MRP3                         NaN               NaN   \n",
       " 5               MOATB MRP4                         NaN               NaN   \n",
       " 6      ABCP BCRP BCRP1 MXR                         NaN               NaN   \n",
       " 7                     MRP5                         NaN               NaN   \n",
       " 8                MDR1 PGY1                         NaN               NaN   \n",
       " 9                 MRP MRP1                         NaN               NaN   \n",
       " 10  CMOAT CMOAT1 CMRP MRP2                         NaN               NaN   \n",
       " 11                    ABC3                         NaN               NaN   \n",
       " \n",
       "                 Organism  Organism (ID)  \\\n",
       " 0   Homo sapiens (Human)           9606   \n",
       " 1   Homo sapiens (Human)           9606   \n",
       " 2   Homo sapiens (Human)           9606   \n",
       " 3   Homo sapiens (Human)           9606   \n",
       " 4   Homo sapiens (Human)           9606   \n",
       " 5   Homo sapiens (Human)           9606   \n",
       " 6   Homo sapiens (Human)           9606   \n",
       " 7   Homo sapiens (Human)           9606   \n",
       " 8   Homo sapiens (Human)           9606   \n",
       " 9   Homo sapiens (Human)           9606   \n",
       " 10  Homo sapiens (Human)           9606   \n",
       " 11  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                         Protein names  ...  \\\n",
       " 0   RalA-binding protein 1 (RalBP1) (76 kDa Ral-in...  ...   \n",
       " 1   ATP-binding cassette sub-family B member 5 (AB...  ...   \n",
       " 2   ATP-binding cassette sub-family C member 11 (E...  ...   \n",
       " 3   ATP-binding cassette sub-family C member 10 (E...  ...   \n",
       " 4   ATP-binding cassette sub-family C member 3 (EC...  ...   \n",
       " 5   ATP-binding cassette sub-family C member 4 (EC...  ...   \n",
       " 6   Broad substrate specificity ATP-binding casset...  ...   \n",
       " 7   ATP-binding cassette sub-family C member 5 (EC...  ...   \n",
       " 8   ATP-dependent translocase ABCB1 (ATP-binding c...  ...   \n",
       " 9   Multidrug resistance-associated protein 1 (EC ...  ...   \n",
       " 10  ATP-binding cassette sub-family C member 2 (EC...  ...   \n",
       " 11  Phospholipid-transporting ATPase ABCA3 (EC 7.6...  ...   \n",
       " \n",
       "                                         Glycosylation  \\\n",
       " 0                                                 NaN   \n",
       " 1   CARBOHYD 17; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 2   CARBOHYD 838; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 3                                                 NaN   \n",
       " 4   CARBOHYD 18; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 5   CARBOHYD 746; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 6   CARBOHYD 596; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 7   CARBOHYD 494; /note=\"N-linked (GlcNAc...) aspa...   \n",
       " 8   CARBOHYD 91; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 9   CARBOHYD 19; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " 10  CARBOHYD 7; /note=\"N-linked (GlcNAc...) aspara...   \n",
       " 11  CARBOHYD 14; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 2; /note=\"N-acetylthreonine\"; /evidenc...     NaN   \n",
       " 1                                                 NaN     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3   MOD_RES 463; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 4   MOD_RES 908; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 5   MOD_RES 646; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 6   MOD_RES 362; /note=\"Phosphothreonine; by PIM1\"...     NaN   \n",
       " 7   MOD_RES 14; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 8   MOD_RES 660; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 9   MOD_RES 277; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 10  MOD_RES 281; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 11                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0   PTM: Tyrosine-phosphorylated upon stimulation ...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5   PTM: N-glycosylated; leading to substrate-sele...         NaN   \n",
       " 6   PTM: N-glycosylated (PubMed:15807535, PubMed:2...         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11  PTM: N-glycosylated (PubMed:16959783, PubMed:2...         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  \\\n",
       " 0              NaN             NaN   \n",
       " 1              NaN             NaN   \n",
       " 2              NaN             NaN   \n",
       " 3              NaN             NaN   \n",
       " 4              NaN             NaN   \n",
       " 5              NaN             NaN   \n",
       " 6              NaN             NaN   \n",
       " 7              NaN             NaN   \n",
       " 8              NaN             NaN   \n",
       " 9              NaN             NaN   \n",
       " 10             NaN             NaN   \n",
       " 11             NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0                           2KWH;2KWI;2MBG;6ZQT;6ZRN;  \n",
       " 1                                                 NaN  \n",
       " 2                                                 NaN  \n",
       " 3                                                 NaN  \n",
       " 4                                                 NaN  \n",
       " 5                                                 NaN  \n",
       " 6   5NJ3;5NJG;6ETI;6FEQ;6FFC;6HBU;6HCO;6HIJ;6HZM;6...  \n",
       " 7                                                 NaN  \n",
       " 8   6C0V;6FN1;6FN4;6QEX;7A65;7A69;7A6C;7A6E;7A6F;7...  \n",
       " 9                                          2CBZ;4C3Z;  \n",
       " 10                                                NaN  \n",
       " 11                                         7W01;7W02;  \n",
       " \n",
       " [12 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q9Y223  GLCNE_HUMAN  GNE GLCNE                  GNE                GLCNE   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Bifunctional UDP-N-acetylglucosamine 2-epimera...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "               Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0  PTM: Phosphorylated by PKC. {ECO:0000250}.         NaN             NaN   \n",
       " \n",
       "   Transit peptide                        PDB  \n",
       " 0             NaN  2YHW;2YHY;2YI1;3EO3;4ZHT;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names  Gene Names (primary)  \\\n",
       " 0  Q46ZL2  3HAPM_CUPPJ  Reut_A2057                   NaN   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                 Reut_A2057               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Cupriavidus pinatubonensis (strain JMP 134 / L...         264198   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  3-hydroxylaminophenol mutase (3HAP mutase) (EC...  ...            NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q9U8D3  PURA_PLAFA            Adss                 Adss   \n",
       " 1  Q8IDF6  PURA_PLAF7  Adss PF13_0287                 Adss   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN              NaN   \n",
       " 1                   NaN                         NaN        PF13_0287   \n",
       " \n",
       "                               Organism  Organism (ID)  \\\n",
       " 0                Plasmodium falciparum           5833   \n",
       " 1  Plasmodium falciparum (isolate 3D7)          36329   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Adenylosuccinate synthetase (AMPSase) (AdSS) (...  ...           NaN   \n",
       " 1  Adenylosuccinate synthetase (AMPSase) (AdSS) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1P9B;  \n",
       " 1             NaN    NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  P33883  LASI_PSEAE       lasI PA1432                 lasI   \n",
       " 1  P54291  RHLI_PSEAE  rhlI vsmI PA3476                 rhlI   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                     PA1432               NaN   \n",
       " 1                 vsmI                     PA3476               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 1  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Acyl-homoserine-lactone synthase (EC 2.3.1.184...  ...           NaN   \n",
       " 1  Acyl-homoserine-lactone synthase (EC 2.3.1.184...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1RO5;  \n",
       " 1             NaN    NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O75390  CISY_HUMAN         CS                   CS                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Citrate synthase, mitochondrial (EC 2.3.3.1) (...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 57; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Methylated (PubMed:28391595, PubMed:28887...         NaN   \n",
       " \n",
       "    Signal peptide                       Transit peptide              PDB  \n",
       " 0             NaN  TRANSIT 1..27; /note=\"Mitochondrion\"  5UZP;5UZQ;5UZR;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P68698  TOP1_VACCW  TOP1 VACWR104 H6R                 TOP1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                   VACWR104              H6R   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Vaccinia virus (strain Western Reserve) (VACV)...          10254   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  DNA topoisomerase 1B (TopIB) (EC 5.6.2.1) (DNA...  ...           NaN   \n",
       " \n",
       "   Initiator methionine Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN        NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1A41;1VCC;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                  Gene Names Gene Names (primary)  \\\n",
       " 0  P18335  ARGD_ECOLI  argD dapC dtu b3359 JW3322                 argD   \n",
       " 1  Q8X4S6  ARGD_ECO57     argD dapC Z4720 ECs4210                 argD   \n",
       " 2  P59317  ARGD_ECOL6             argD dapC c4134                 argD   \n",
       " 3  P40732  ARGD_SALTY       argD dapC dtu STM3468                 argD   \n",
       " 4  P59321  ARGD_SHIFL      argD dapC SF3378 S4385                 argD   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0             dapC dtu               b3359 JW3322               NaN   \n",
       " 1                 dapC              Z4720 ECs4210               NaN   \n",
       " 2                 dapC                      c4134               NaN   \n",
       " 3             dapC dtu                    STM3468               NaN   \n",
       " 4                 dapC               SF3378 S4385               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                           Escherichia coli O157:H7          83334   \n",
       " 2  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 3  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 4                                  Shigella flexneri            623   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Acetylornithine/succinyldiaminopimelate aminot...  ...           NaN   \n",
       " 1  Acetylornithine/succinyldiaminopimelate aminot...  ...           NaN   \n",
       " 2  Acetylornithine/succinyldiaminopimelate aminot...  ...           NaN   \n",
       " 3  Acetylornithine/succinyldiaminopimelate aminot...  ...           NaN   \n",
       " 4  Acetylornithine/succinyldiaminopimelate aminot...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 255; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 1  MOD_RES 255; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 2  MOD_RES 255; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 3  MOD_RES 255; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 4  MOD_RES 255; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                            PDB  \n",
       " 0             NaN                                            NaN  \n",
       " 1             NaN                                            NaN  \n",
       " 2             NaN                                            NaN  \n",
       " 3             NaN  2PB0;2PB2;4JEV;4JEW;4JEX;4JEY;4JEZ;4JF0;4JF1;  \n",
       " 4             NaN                                            NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P76558  MAO2_ECOLI  maeB ypfF b2463 JW2447                 maeB   \n",
       " 1  P16243  MAOC_MAIZE                MOD1 ME1                 MOD1   \n",
       " 2  P28227  MAOX_ANAPL                     ME1                  ME1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 ypfF               b2463 JW2447               NaN   \n",
       " 1                  ME1                        NaN               NaN   \n",
       " 2                  NaN                        NaN               NaN   \n",
       " \n",
       "                                       Organism  Organism (ID)  \\\n",
       " 0                Escherichia coli (strain K12)          83333   \n",
       " 1                             Zea mays (Maize)           4577   \n",
       " 2  Anas platyrhynchos (Mallard) (Anas boschas)           8839   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  NADP-dependent malic enzyme (NADP-ME) (EC 1.1....  ...           NaN   \n",
       " 1  NADP-dependent malic enzyme, chloroplastic (NA...  ...           NaN   \n",
       " 2  NADP-dependent malic enzyme (NADP-ME) (EC 1.1....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 56; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide    PDB  \n",
       " 0                                                NaN    NaN  \n",
       " 1  TRANSIT 1..62; /note=\"Chloroplast\"; /evidence=...  5OU5;  \n",
       " 2                                                NaN    NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  P00560  PGK_YEAST  PGK1 YCR012W YCR12W                 PGK1   \n",
       " 1  Q757Q0  PGK_ASHGO         PGK1 AEL038C                 PGK1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YCR012W           YCR12W   \n",
       " 1                   NaN                    AEL038C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Ashbya gossypii (strain ATCC 10895 / CBS 109.5...         284811   \n",
       " \n",
       "                           Protein names  ... Glycosylation  \\\n",
       " 0  Phosphoglycerate kinase (EC 2.7.2.3)  ...           NaN   \n",
       " 1  Phosphoglycerate kinase (EC 2.7.2.3)  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  1FW8;1QPG;3PGK;  \n",
       " 1             NaN              NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry   Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0   P0A870   TALB_ECOLI   talB yaaK b0008 JW0007                 talB   \n",
       " 1   P37837  TALDO_HUMAN  TALDO1 TAL TALDO TALDOR               TALDO1   \n",
       " 2   P0A867   TALA_ECOLI        talA b2464 JW2448                 talA   \n",
       " 3   P0A869   TALA_ECO57       talA Z3720 ECs3326                 talA   \n",
       " 4   P0A871   TALB_ECO57       talB Z0008 ECs0008                 talB   \n",
       " 5   K0BE10   TALB_ECO1E           talB O3O_03880                 talB   \n",
       " 6   Q8FLD1   TALB_ECOL6               talB c0012                 talB   \n",
       " 7   P0A868   TALA_ECOL6               talA c2989                 talA   \n",
       " 8   P19669    TAL_BACSU        tal ywjH BSU37110                  tal   \n",
       " 9   Q83QM8   TALA_SHIFL        talA SF2506 S2657                 talA   \n",
       " 10  P0A872   TALB_SHIFL        talB SF0009 S0008                 talB   \n",
       " 11  Q9WYD1    TAL_THEMA              tal TM_0295                  tal   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  yaaK               b0008 JW0007               NaN   \n",
       " 1      TAL TALDO TALDOR                        NaN               NaN   \n",
       " 2                   NaN               b2464 JW2448               NaN   \n",
       " 3                   NaN              Z3720 ECs3326               NaN   \n",
       " 4                   NaN              Z0008 ECs0008               NaN   \n",
       " 5                   NaN                  O3O_03880               NaN   \n",
       " 6                   NaN                      c0012               NaN   \n",
       " 7                   NaN                      c2989               NaN   \n",
       " 8                  ywjH                   BSU37110               NaN   \n",
       " 9                   NaN               SF2506 S2657               NaN   \n",
       " 10                  NaN               SF0009 S0008               NaN   \n",
       " 11                  NaN                    TM_0295               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                                Homo sapiens (Human)           9606   \n",
       " 2                       Escherichia coli (strain K12)          83333   \n",
       " 3                            Escherichia coli O157:H7          83334   \n",
       " 4                            Escherichia coli O157:H7          83334   \n",
       " 5       Escherichia coli O104:H4 (strain 2009EL-2071)        1133853   \n",
       " 6   Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 7   Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 8                      Bacillus subtilis (strain 168)         224308   \n",
       " 9                                   Shigella flexneri            623   \n",
       " 10                                  Shigella flexneri            623   \n",
       " 11  Thermotoga maritima (strain ATCC 43589 / DSM 3...         243274   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0                        Transaldolase B (EC 2.2.1.2)  ...           NaN   \n",
       " 1                          Transaldolase (EC 2.2.1.2)  ...           NaN   \n",
       " 2                        Transaldolase A (EC 2.2.1.2)  ...           NaN   \n",
       " 3                        Transaldolase A (EC 2.2.1.2)  ...           NaN   \n",
       " 4                        Transaldolase B (EC 2.2.1.2)  ...           NaN   \n",
       " 5                          Transaldolase (EC 2.2.1.2)  ...           NaN   \n",
       " 6                        Transaldolase B (EC 2.2.1.2)  ...           NaN   \n",
       " 7                        Transaldolase A (EC 2.2.1.2)  ...           NaN   \n",
       " 8   Transaldolase (EC 2.2.1.2) (20 kDa phosphoprot...  ...           NaN   \n",
       " 9                        Transaldolase A (EC 2.2.1.2)  ...           NaN   \n",
       " 10                       Transaldolase B (EC 2.2.1.2)  ...           NaN   \n",
       " 11                         Transaldolase (EC 2.2.1.2)  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 5   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 6   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0                                                 NaN     NaN   \n",
       " 1   MOD_RES 115; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3                                                 NaN     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6                                                 NaN     NaN   \n",
       " 7                                                 NaN     NaN   \n",
       " 8   MOD_RES 39; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 9                                                 NaN     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11                                                NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                                                PDB  \n",
       " 0              NaN  1I2N;1I2O;1I2P;1I2Q;1I2R;1ONR;1UCW;3CWN;3KOF;4...  \n",
       " 1              NaN                                              1F05;  \n",
       " 2              NaN                                                NaN  \n",
       " 3              NaN                                                NaN  \n",
       " 4              NaN                                                NaN  \n",
       " 5              NaN                                                NaN  \n",
       " 6              NaN                                                NaN  \n",
       " 7              NaN                                                NaN  \n",
       " 8              NaN                                              3R8R;  \n",
       " 9              NaN                                                NaN  \n",
       " 10             NaN                                                NaN  \n",
       " 11             NaN                                              1VPX;  \n",
       " \n",
       " [12 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P13006  GOX_ASPNG        gox                  gox                   NaN   \n",
       " 1  P81156  GOX_PENAG        NaN                  NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                   Organism  \\\n",
       " 0                         NaN               NaN          Aspergillus niger   \n",
       " 1                         NaN               NaN  Penicillium amagasakiense   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           5061  Glucose oxidase (EC 1.1.3.4) (Beta-D-glucose:o...  ...   \n",
       " 1          63559  Glucose oxidase (EC 1.1.3.4) (Beta-D-glucose:o...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 111; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 93; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide               Signal peptide Transit peptide  \\\n",
       " 0         NaN  SIGNAL 1..22; /note=\"Or 48\"             NaN   \n",
       " 1         NaN                          NaN             NaN   \n",
       " \n",
       "                               PDB  \n",
       " 0  1CF3;1GAL;3QVP;3QVR;5NIT;5NIW;  \n",
       " 1                           1GPE;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q63707   PYRD_RAT      Dhodh                Dhodh                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Dihydroorotate dehydrogenase (quinone), mitoch...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: The uncleaved transit peptide is required...         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  \\\n",
       " 0             NaN  TRANSIT 1..10; /note=\"Mitochondrion; not cleaved\"   \n",
       " \n",
       "                PDB  \n",
       " 0  1UUM;1UUO;4ORI;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  Q13131  AAPK1_HUMAN       PRKAA1 AMPK1               PRKAA1   \n",
       " 1  P54646  AAPK2_HUMAN  PRKAA2 AMPK AMPK2               PRKAA2   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                AMPK1                         NaN               NaN   \n",
       " 1           AMPK AMPK2                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  5'-AMP-activated protein kinase catalytic subu...  ...           NaN   \n",
       " 1  5'-AMP-activated protein kinase catalytic subu...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 32; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 1  MOD_RES 172; /note=\"Phosphothreonine; by LKB1 ...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Ubiquitinated. {ECO:0000250|UniProtKB:Q5E...         NaN   \n",
       " 1  PTM: Ubiquitinated. {ECO:0000250|UniProtKB:Q8B...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  4RED;4RER;4REW;5EZV;6C9F;6C9G;6C9H;6C9J;7JHG;7...  \n",
       " 1  2H6D;2LTU;2YZA;3AQV;4CFE;4CFF;4ZHX;5EZV;5ISO;6...  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q9HPP1  SUCC_HALSA  sucC VNG_1541G                 sucC   \n",
       " 1  B0R5U5  SUCC_HALS3   sucC OE_3195F                 sucC   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                  VNG_1541G               NaN   \n",
       " 1                   NaN                   OE_3195F               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Halobacterium salinarum (strain ATCC 700922 / ...          64091   \n",
       " 1  Halobacterium salinarum (strain ATCC 29341 / D...         478009   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Succinate--CoA ligase [ADP-forming] subunit be...  ...           NaN   \n",
       " 1  Succinate--CoA ligase [ADP-forming] subunit be...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  Q88S51  RHAA_LACPL  rhaA lp_3593                 rhaA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    lp_3593               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Lactiplantibacillus plantarum (strain ATCC BAA...         220668   \n",
       " \n",
       "                         Protein names  ... Glycosylation Initiator methionine  \\\n",
       " 0  L-rhamnose isomerase (EC 5.3.1.14)  ...           NaN                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide  PDB  \n",
       " 0         NaN             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  Q16773   KAT1_HUMAN       KYAT1 CCBL1                KYAT1   \n",
       " 1  Q6YP21   KAT3_HUMAN  KYAT3 CCBL2 KAT3                KYAT3   \n",
       " 2  P00505   AATM_HUMAN        GOT2 KYAT4                 GOT2   \n",
       " 3  Q8N5Z0  AADAT_HUMAN  AADAT KAT2 KYAT2                AADAT   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                CCBL1                         NaN               NaN   \n",
       " 1           CCBL2 KAT3                         NaN               NaN   \n",
       " 2                KYAT4                         NaN               NaN   \n",
       " 3           KAT2 KYAT2                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " 3  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Kynurenine--oxoglutarate transaminase 1 (EC 2....  ...           NaN   \n",
       " 1  Kynurenine--oxoglutarate transaminase 3 (EC 2....  ...           NaN   \n",
       " 2  Aspartate aminotransferase, mitochondrial (mAs...  ...           NaN   \n",
       " 3  Kynurenine/alpha-aminoadipate aminotransferase...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET Q6YP21-3:1; /note=\"Removed\"; /evidenc...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 247; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 1  MOD_RES 116; /note=\"N6-acetyllysine; alternate...     NaN   \n",
       " 2  MOD_RES 48; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 3  MOD_RES 69; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  \\\n",
       " 0                                                NaN   \n",
       " 1                                                NaN   \n",
       " 2  TRANSIT 1..29; /note=\"Mitochondrion\"; /evidenc...   \n",
       " 3  TRANSIT 1..29; /note=\"Mitochondrion\"; /evidenc...   \n",
       " \n",
       "                                                  PDB  \n",
       " 0      1W7L;1W7M;1W7N;3FVS;3FVU;3FVX;4WLH;4WLJ;4WP0;  \n",
       " 1                                                NaN  \n",
       " 2                                              5AX8;  \n",
       " 3  2QLR;2R2N;2VGZ;2XH1;3DC1;3UE8;4GDY;4GE4;4GE7;4...  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P28298  ACEA_EMENI            acuD AN5634                 acuD   \n",
       " 1  P28299  ACEA_NEUCR  acu-3 2E4.10 NCU04230                acu-3   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN           AN5634   \n",
       " 1                   NaN                         NaN  2E4.10 NCU04230   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Emericella nidulans (strain FGSC A4 / ATCC 381...         227321   \n",
       " 1  Neurospora crassa (strain ATCC 24698 / 74-OR23...         367110   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Isocitrate lyase (ICL) (Isocitrase) (Isocitrat...  ...           NaN   \n",
       " 1  Isocitrate lyase (ICL) (Isocitrase) (Isocitrat...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1DQU;  \n",
       " 1             NaN    NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P31102  AROB_BACSU          aroB BSU22700                 aroB   \n",
       " 1  A7Z622  AROB_BACVZ       aroB RBAM_020860                 aroB   \n",
       " 2  Q9KCB6  AROB_HALH5            aroB BH1657                 aroB   \n",
       " 3  Q65I29  AROB_BACLD  aroB BLi02405 BL02777                 aroB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   BSU22700               NaN   \n",
       " 1                   NaN                RBAM_020860               NaN   \n",
       " 2                   NaN                     BH1657               NaN   \n",
       " 3                   NaN           BLi02405 BL02777               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                     Bacillus subtilis (strain 168)         224308   \n",
       " 1  Bacillus velezensis (strain DSM 23117 / BGSC 1...         326423   \n",
       " 2  Halalkalibacterium halodurans (strain ATCC BAA...         272558   \n",
       " 3  Bacillus licheniformis (strain ATCC 14580 / DS...         279010   \n",
       " \n",
       "                                    Protein names  ... Glycosylation  \\\n",
       " 0  3-dehydroquinate synthase (DHQS) (EC 4.2.3.4)  ...           NaN   \n",
       " 1  3-dehydroquinate synthase (DHQS) (EC 4.2.3.4)  ...           NaN   \n",
       " 2  3-dehydroquinate synthase (DHQS) (EC 4.2.3.4)  ...           NaN   \n",
       " 3  3-dehydroquinate synthase (DHQS) (EC 4.2.3.4)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9XSJ4  ENOA_BOVIN       ENO1                 ENO1                   NaN   \n",
       " 1  Q3ZC09  ENOB_BOVIN       ENO3                 ENO3                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 1                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Alpha-enolase (EC 4.2.1.11) (2-phospho-D-glyce...  ...   \n",
       " 1           9913  Beta-enolase (EC 4.2.1.11) (2-phospho-D-glycer...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 1         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: ISGylated. {ECO:0000250|UniProtKB:P06733}...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name                     Gene Names Gene Names (primary)  \\\n",
       " 0  P00561  AK1H_ECOLI  thrA thrA1 thrA2 b0002 JW0001                 thrA   \n",
       " 1  P00562  AK2H_ECOLI         metL metM b3940 JW3911                 metL   \n",
       " 2  P19582  DHOM_BACSU               hom tdm BSU32260                  hom   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0          thrA1 thrA2               b0002 JW0001               NaN   \n",
       " 1                 metM               b3940 JW3911               NaN   \n",
       " 2                  tdm                   BSU32260               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0   Escherichia coli (strain K12)          83333   \n",
       " 1   Escherichia coli (strain K12)          83333   \n",
       " 2  Bacillus subtilis (strain 168)         224308   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Bifunctional aspartokinase/homoserine dehydrog...  ...           NaN   \n",
       " 1  Bifunctional aspartokinase/homoserine dehydrog...  ...           NaN   \n",
       " 2        Homoserine dehydrogenase (HDH) (EC 1.1.1.3)  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " 2               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide    PDB  \n",
       " 0             NaN             NaN  6MX1;  \n",
       " 1             NaN             NaN    NaN  \n",
       " 2             NaN             NaN    NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q59093  ELH1_ACIAD  pcaD ACIAD1708                 pcaD   \n",
       " 1  P00632  ELH2_ACIAD  catD ACIAD1451                 catD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                  ACIAD1708               NaN   \n",
       " 1                   NaN                  ACIAD1451               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Acinetobacter baylyi (strain ATCC 33305 / BD41...          62977   \n",
       " 1  Acinetobacter baylyi (strain ATCC 33305 / BD41...          62977   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  3-oxoadipate enol-lactonase 1 (EC 3.1.1.24) (3...  ...           NaN   \n",
       " 1  3-oxoadipate enol-lactonase 2 (EC 3.1.1.24) (3...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "         Entry Entry Name Gene Names Gene Names (primary)  \\\n",
       " 0  A0A0A1H8I4  AIS_PSESP        ais                  ais   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN               NaN   \n",
       " \n",
       "          Organism  Organism (ID)                          Protein names  ...  \\\n",
       " 0  Pseudomonas sp            306  Aconitate isomerase (AI) (EC 5.3.3.7)  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..22; /evidence=\"ECO:0000269|PubMed:26...             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P51570  GALK1_HUMAN  GALK1 GALK                GALK1                 GALK   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                  Protein names  ...  \\\n",
       " 0           9606  Galactokinase (EC 2.7.1.6) (Galactose kinase)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 230; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1WUU;6GR2;6Q3W;6Q3X;6Q8Z;6Q90;6Q91;6QJE;6ZFH;6...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P32528  DUR1_YEAST  DUR1,2 YBR208C YBR1448               DUR1,2   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YBR208C          YBR1448   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Urea amidolyase [Includes: Urea carboxylase (E...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 803; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  P69908  DCEA_ECOLI   gadA gadS b3517 JW3485                 gadA   \n",
       " 1  P69910  DCEB_ECOLI        gadB b1493 JW1488                 gadB   \n",
       " 2  P58228  DCEA_ECO57  gadA gadS Z4930 ECs4397                 gadA   \n",
       " 3  P69909  DCEA_ECOL6          gadA gadS c4328                 gadA   \n",
       " 4  Q8FHG5  DCEB_ECOL6               gadB c1922                 gadB   \n",
       " 5  P69911  DCEB_ECO57       gadB Z2215 ECs2098                 gadB   \n",
       " 6  P69912  DCEB_SHIFL        gadB SF1734 S1867                 gadB   \n",
       " 7  Q83PR1  DCEA_SHIFL        gadA SF3594 S4173                 gadA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 gadS               b3517 JW3485               NaN   \n",
       " 1                  NaN               b1493 JW1488               NaN   \n",
       " 2                 gadS              Z4930 ECs4397               NaN   \n",
       " 3                 gadS                      c4328               NaN   \n",
       " 4                  NaN                      c1922               NaN   \n",
       " 5                  NaN              Z2215 ECs2098               NaN   \n",
       " 6                  NaN               SF1734 S1867               NaN   \n",
       " 7                  NaN               SF3594 S4173               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain K12)          83333   \n",
       " 2                           Escherichia coli O157:H7          83334   \n",
       " 3  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 4  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 5                           Escherichia coli O157:H7          83334   \n",
       " 6                                  Shigella flexneri            623   \n",
       " 7                                  Shigella flexneri            623   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutamate decarboxylase alpha (GAD-alpha) (EC ...  ...           NaN   \n",
       " 1  Glutamate decarboxylase beta (GAD-beta) (EC 4....  ...           NaN   \n",
       " 2  Glutamate decarboxylase alpha (GAD-alpha) (EC ...  ...           NaN   \n",
       " 3  Glutamate decarboxylase alpha (GAD-alpha) (EC ...  ...           NaN   \n",
       " 4  Glutamate decarboxylase beta (GAD-beta) (EC 4....  ...           NaN   \n",
       " 5  Glutamate decarboxylase beta (GAD-beta) (EC 4....  ...           NaN   \n",
       " 6  Glutamate decarboxylase beta (GAD-beta) (EC 4....  ...           NaN   \n",
       " 7  Glutamate decarboxylase alpha (GAD-alpha) (EC ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " 3                  NaN         NaN   \n",
       " 4                  NaN         NaN   \n",
       " 5                  NaN         NaN   \n",
       " 6                  NaN         NaN   \n",
       " 7                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 276; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 1  MOD_RES 276; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 2  MOD_RES 276; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 3  MOD_RES 276; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 4  MOD_RES 276; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 5  MOD_RES 276; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 6  MOD_RES 276; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 7  MOD_RES 276; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " 7                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                       PDB  \n",
       " 0             NaN                                     1XEY;  \n",
       " 1             NaN  1PMM;1PMO;2DGK;2DGL;2DGM;3FZ6;3FZ7;3FZ8;  \n",
       " 2             NaN                                       NaN  \n",
       " 3             NaN                                       NaN  \n",
       " 4             NaN                                       NaN  \n",
       " 5             NaN                                       NaN  \n",
       " 6             NaN                                       NaN  \n",
       " 7             NaN                                       NaN  \n",
       " \n",
       " [8 rows x 110 columns],\n",
       "     Entry Entry Name    Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q01062  PDE2A_RAT         Pde2a                Pde2a                  NaN   \n",
       " 1  Q62865  PDE3A_RAT         Pde3a                Pde3a                  NaN   \n",
       " 2  Q01066  PDE1B_RAT  Pde1b Pde1b1                Pde1b               Pde1b1   \n",
       " 3  Q63085  PDE3B_RAT         Pde3b                Pde3b                  NaN   \n",
       " 4  Q63421  PDE1C_RAT  Pde1c Pde1c2                Pde1c               Pde1c2   \n",
       " 5  Q9QYJ6  PDE10_RAT        Pde10a               Pde10a                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 2                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 3                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 4                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 5                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  cGMP-dependent 3',5'-cyclic phosphodiesterase ...  ...   \n",
       " 1          10116  cGMP-inhibited 3',5'-cyclic phosphodiesterase ...  ...   \n",
       " 2          10116  Dual specificity calcium/calmodulin-dependent ...  ...   \n",
       " 3          10116  cGMP-inhibited 3',5'-cyclic phosphodiesterase ...  ...   \n",
       " 4          10116  Dual specificity calcium/calmodulin-dependent ...  ...   \n",
       " 5          10116  cAMP and cAMP-inhibited cGMP 3',5'-cyclic phos...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " 2           NaN                  NaN         NaN   \n",
       " 3           NaN                  NaN         NaN   \n",
       " 4           NaN                  NaN         NaN   \n",
       " 5           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 109; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 310; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 2  MOD_RES 7; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " 3  MOD_RES 15; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 4  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 5                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  PTM: Phosphorylation at Ser-279 mediates insul...         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " 4             NaN             NaN   \n",
       " 5             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                                NaN  \n",
       " 1                                                NaN  \n",
       " 2                                                NaN  \n",
       " 3                                                NaN  \n",
       " 4                                                NaN  \n",
       " 5  2O8H;2OVV;2OVY;3HQW;3HQY;3HQZ;3HR1;3LXG;3QPN;3...  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  O00219  HYAS3_HUMAN       HAS3                 HAS3                  NaN   \n",
       " 1  Q92819  HYAS2_HUMAN       HAS2                 HAS2                  NaN   \n",
       " 2  Q92839  HYAS1_HUMAN   HAS1 HAS                 HAS1                  HAS   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " 2                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Hyaluronan synthase 3 (EC 2.4.1.212) (Hyaluron...  ...   \n",
       " 1           9606  Hyaluronan synthase 2 (EC 2.4.1.212) (Hyaluron...  ...   \n",
       " 2           9606  Hyaluronan synthase 1 (EC 2.4.1.212) (Hyaluron...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 462; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 221; /note=\"O-linked (GlcNAc) serine\"...                  NaN   \n",
       " 2                                                NaN                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN                                                NaN     NaN   \n",
       " 1         NaN  MOD_RES 110; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: O-GlcNAcylation increases the hyaluronan ...         NaN   \n",
       " 1  PTM: Phosphorylation at Thr-328 is essential f...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P32264  PROB_YEAST  PRO1 YDR300C D9740.12                 PRO1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YDR300C         D9740.12   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutamate 5-kinase (GK) (EC 2.7.2.11) (Gamma-g...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q8KZT5  GBH_ARTS8        gbh                  gbh                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                              Organism  Organism (ID)  \\\n",
       " 0  Arthrobacter sp. (strain KUJ 8602)         196628   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Guanidinobutyrase (GBase) (EC 3.5.3.7) (D-argi...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P09034   ASSY_RAT   Ass1 Ass                 Ass1                  Ass   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Argininosuccinate synthase (EC 6.3.4.5) (Citru...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 87; /note=\"Phosphotyrosine\"; /evidence...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Acetylated by CLOCK in a circadian manner...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0   Q60006  FOLD_SALTI  folD STY0588 t2321                 folD   \n",
       " 1   B4TN64  FOLD_SALSV     folD SeSA_A0587                 folD   \n",
       " 2   B4TA94  FOLD_SALHS     folD SeHA_C0650                 folD   \n",
       " 3   B5EYE2  FOLD_SALA4     folD SeAg_B0588                 folD   \n",
       " 4   C0PVJ3  FOLD_SALPC       folD SPC_0556                 folD   \n",
       " 5   B4SXP7  FOLD_SALNS  folD SNSL254_A0595                 folD   \n",
       " 6   A9MW26  FOLD_SALPB     folD SPAB_03022                 folD   \n",
       " 7   Q57S24  FOLD_SALCH       folD SCH_0581                 folD   \n",
       " 8   A9MLA4  FOLD_SALAR     folD SARI_02412                 folD   \n",
       " 9   B5QUV5  FOLD_SALEP        folD SEN0523                 folD   \n",
       " 10  B5FLQ2  FOLD_SALDC      folD SeD_A0590                 folD   \n",
       " 11  P58688  FOLD_SALTY        folD STM0542                 folD   \n",
       " 12  B5BCZ5  FOLD_SALPK       folD SSPA2029                 folD   \n",
       " 13  B5R6X5  FOLD_SALG2         folD SG0554                 folD   \n",
       " 14  Q5PCE5  FOLD_SALPA        folD SPA2183                 folD   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                    NaN              STY0588 t2321               NaN   \n",
       " 1                    NaN                 SeSA_A0587               NaN   \n",
       " 2                    NaN                 SeHA_C0650               NaN   \n",
       " 3                    NaN                 SeAg_B0588               NaN   \n",
       " 4                    NaN                   SPC_0556               NaN   \n",
       " 5                    NaN              SNSL254_A0595               NaN   \n",
       " 6                    NaN                 SPAB_03022               NaN   \n",
       " 7                    NaN                   SCH_0581               NaN   \n",
       " 8                    NaN                 SARI_02412               NaN   \n",
       " 9                    NaN                    SEN0523               NaN   \n",
       " 10                   NaN                  SeD_A0590               NaN   \n",
       " 11                   NaN                    STM0542               NaN   \n",
       " 12                   NaN                   SSPA2029               NaN   \n",
       " 13                   NaN                     SG0554               NaN   \n",
       " 14                   NaN                    SPA2183               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                    Salmonella typhi          90370   \n",
       " 1         Salmonella schwarzengrund (strain CVM19633)         439843   \n",
       " 2                Salmonella heidelberg (strain SL476)         454169   \n",
       " 3                     Salmonella agona (strain SL483)         454166   \n",
       " 4             Salmonella paratyphi C (strain RKS4594)         476213   \n",
       " 5                   Salmonella newport (strain SL254)         423368   \n",
       " 6   Salmonella paratyphi B (strain ATCC BAA-1250 /...        1016998   \n",
       " 7             Salmonella choleraesuis (strain SC-B67)         321314   \n",
       " 8   Salmonella arizonae (strain ATCC BAA-731 / CDC...          41514   \n",
       " 9         Salmonella enteritidis PT4 (strain P125109)         550537   \n",
       " 10             Salmonella dublin (strain CT_02021853)         439851   \n",
       " 11  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 12          Salmonella paratyphi A (strain AKU_12601)         554290   \n",
       " 13  Salmonella gallinarum (strain 287/91 / NCTC 13...         550538   \n",
       " 14  Salmonella paratyphi A (strain ATCC 9150 / SAR...         295319   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 1   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 2   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 3   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 4   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 5   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 6   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 7   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 8   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 9   Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 10  Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 11  Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 12  Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 13  Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " 14  Bifunctional protein FolD [Includes: Methylene...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "     Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0                NaN     NaN                              NaN         NaN   \n",
       " 1                NaN     NaN                              NaN         NaN   \n",
       " 2                NaN     NaN                              NaN         NaN   \n",
       " 3                NaN     NaN                              NaN         NaN   \n",
       " 4                NaN     NaN                              NaN         NaN   \n",
       " 5                NaN     NaN                              NaN         NaN   \n",
       " 6                NaN     NaN                              NaN         NaN   \n",
       " 7                NaN     NaN                              NaN         NaN   \n",
       " 8                NaN     NaN                              NaN         NaN   \n",
       " 9                NaN     NaN                              NaN         NaN   \n",
       " 10               NaN     NaN                              NaN         NaN   \n",
       " 11               NaN     NaN                              NaN         NaN   \n",
       " 12               NaN     NaN                              NaN         NaN   \n",
       " 13               NaN     NaN                              NaN         NaN   \n",
       " 14               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  PDB  \n",
       " 0              NaN             NaN  NaN  \n",
       " 1              NaN             NaN  NaN  \n",
       " 2              NaN             NaN  NaN  \n",
       " 3              NaN             NaN  NaN  \n",
       " 4              NaN             NaN  NaN  \n",
       " 5              NaN             NaN  NaN  \n",
       " 6              NaN             NaN  NaN  \n",
       " 7              NaN             NaN  NaN  \n",
       " 8              NaN             NaN  NaN  \n",
       " 9              NaN             NaN  NaN  \n",
       " 10             NaN             NaN  NaN  \n",
       " 11             NaN             NaN  NaN  \n",
       " 12             NaN             NaN  NaN  \n",
       " 13             NaN             NaN  NaN  \n",
       " 14             NaN             NaN  NaN  \n",
       " \n",
       " [15 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P12676  CHOD_STRS0       choA                 choA                   NaN   \n",
       " 1  P22637  CHOD_BREST       choB                 choB                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " \n",
       "                            Organism  Organism (ID)  \\\n",
       " 0  Streptomyces sp. (strain SA-COO)          74576   \n",
       " 1         Brevibacterium sterolicum           1702   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Cholesterol oxidase (CHOD) (EC 1.1.3.6) (Chole...  ...            NaN   \n",
       " 1  Cholesterol oxidase (CHOD) (EC 1.1.3.6) (Chole...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Predicted to be exported by the Tat syste...         NaN   \n",
       " 1  PTM: Predicted to be exported by the Tat syste...         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  \\\n",
       " 0              SIGNAL 1..42; /note=\"Tat-type signal\"             NaN   \n",
       " 1  SIGNAL 1..45; /note=\"Tat-type signal\"; /eviden...             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1B4V;1B8S;1CBO;1CC2;1IJH;1MXT;1N1P;1N4U;1N4V;1...  \n",
       " 1                                         1COY;3COX;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                        Gene Names Gene Names (primary)  \\\n",
       " 0  P06785  TYSY_YEAST  CDC21 CRT9 TMP1 YOR074C YOR29-25                CDC21   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0            CRT9 TMP1                    YOR074C         YOR29-25   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Thymidylate synthase (TS) (TSase) (EC 2.1.1.45...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P09467  F16P1_HUMAN   FBP1 FBP                 FBP1                  FBP   \n",
       " 1  O00757  F16P2_HUMAN       FBP2                 FBP2                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Fructose-1,6-bisphosphatase 1 (FBPase 1) (EC 3...  ...   \n",
       " 1           9606  Fructose-1,6-bisphosphatase isozyme 2 (FBPase ...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN                                                NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 1         NaN  MOD_RES 216; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1FTA;2FHY;2FIE;2FIX;2JJK;2VT5;2WBB;2WBD;2Y5K;2...  \n",
       " 1             NaN  3IFA;3IFC;4HE0;4HE1;4HE2;5ET5;5ET6;5ET7;5ET8;5...  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  P07934  PHKG1_MOUSE           Phkg1 Phkg                Phkg1   \n",
       " 1  Q8VHJ5  MARK1_MOUSE  Mark1 Emk3 Kiaa1477                Mark1   \n",
       " 2  Q2NL51  GSK3A_MOUSE                Gsk3a                Gsk3a   \n",
       " 3  Q9DC28   KC1D_MOUSE         Csnk1d Hckid               Csnk1d   \n",
       " 4  Q9WV60  GSK3B_MOUSE                Gsk3b                Gsk3b   \n",
       " 5  Q05512  MARK2_MOUSE            Mark2 Emk                Mark2   \n",
       " 6  Q5EG47  AAPK1_MOUSE               Prkaa1               Prkaa1   \n",
       " 7  Q69Z98  BRSK2_MOUSE  Brsk2 Kiaa4256 Sada                Brsk2   \n",
       " 8  Q5RJI5  BRSK1_MOUSE    Brsk1 Gm1100 Sadb                Brsk1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 Phkg                         NaN               NaN   \n",
       " 1        Emk3 Kiaa1477                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " 3                Hckid                         NaN               NaN   \n",
       " 4                  NaN                         NaN               NaN   \n",
       " 5                  Emk                         NaN               NaN   \n",
       " 6                  NaN                         NaN               NaN   \n",
       " 7        Kiaa4256 Sada                         NaN               NaN   \n",
       " 8          Gm1100 Sadb                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " 2  Mus musculus (Mouse)          10090   \n",
       " 3  Mus musculus (Mouse)          10090   \n",
       " 4  Mus musculus (Mouse)          10090   \n",
       " 5  Mus musculus (Mouse)          10090   \n",
       " 6  Mus musculus (Mouse)          10090   \n",
       " 7  Mus musculus (Mouse)          10090   \n",
       " 8  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphorylase b kinase gamma catalytic chain, ...  ...           NaN   \n",
       " 1  Serine/threonine-protein kinase MARK1 (EC 2.7....  ...           NaN   \n",
       " 2  Glycogen synthase kinase-3 alpha (GSK-3 alpha)...  ...           NaN   \n",
       " 3  Casein kinase I isoform delta (CKI-delta) (CKI...  ...           NaN   \n",
       " 4  Glycogen synthase kinase-3 beta (GSK-3 beta) (...  ...           NaN   \n",
       " 5  Serine/threonine-protein kinase MARK2 (EC 2.7....  ...           NaN   \n",
       " 6  5'-AMP-activated protein kinase catalytic subu...  ...           NaN   \n",
       " 7  Serine/threonine-protein kinase BRSK2 (EC 2.7....  ...           NaN   \n",
       " 8  Serine/threonine-protein kinase BRSK1 (EC 2.7....  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6                                                NaN         NaN   \n",
       " 7                                                NaN         NaN   \n",
       " 8                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES 5; /note=\"Phosphothreonine\"; /evidence...     NaN   \n",
       " 2  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 3  MOD_RES 328; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 4  MOD_RES 9; /note=\"Phosphoserine; by PKB/AKT1, ...     NaN   \n",
       " 5  MOD_RES 40; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 6  MOD_RES 32; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 7  MOD_RES 175; /note=\"Phosphothreonine; by LKB1\"...     NaN   \n",
       " 8  MOD_RES 189; /note=\"Phosphothreonine; by LKB1\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: Phosphorylated at Thr-215 by STK11/LKB1 i...         NaN   \n",
       " 2  PTM: Phosphorylated by AKT1 at Ser-21: upon in...         NaN   \n",
       " 3  PTM: Autophosphorylated on serine and threonin...         NaN   \n",
       " 4  PTM: Phosphorylated by AKT1 and ILK1. Upon ins...         NaN   \n",
       " 5  PTM: Autophosphorylated. Phosphorylated at Thr...         NaN   \n",
       " 6  PTM: Phosphorylated at Thr-183 by STK11/LKB1 i...         NaN   \n",
       " 7  PTM: May be phosphorylated at Thr-261 by PKA (...         NaN   \n",
       " 8  PTM: Phosphorylated at Thr-189 by STK11/LKB1 i...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide              PDB  \n",
       " 0             NaN             NaN              NaN  \n",
       " 1             NaN             NaN              NaN  \n",
       " 2             NaN             NaN              NaN  \n",
       " 3             NaN             NaN       4JJR;5X17;  \n",
       " 4             NaN             NaN  4NU1;5AIR;6AE3;  \n",
       " 5             NaN             NaN              NaN  \n",
       " 6             NaN             NaN            5UFU;  \n",
       " 7             NaN             NaN       4YNZ;4YOM;  \n",
       " 8             NaN             NaN            5IRI;  \n",
       " \n",
       " [9 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P25553  ALDA_ECOLI  aldA ald b1415 JW1412                 aldA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  ald               b1415 JW1412               NaN   \n",
       " \n",
       "                         Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Lactaldehyde dehydrogenase (EC 1.2.1.22) (Alde...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                   PDB  \n",
       " 0             NaN             NaN  2HG2;2ILU;2IMP;2OPX;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  O00329  PK3CD_HUMAN         PIK3CD               PIK3CD   \n",
       " 1  Q8NFU5   IPMK_HUMAN      IPMK IMPK                 IPMK   \n",
       " 2  P42338  PK3CB_HUMAN  PIK3CB PIK3C1               PIK3CB   \n",
       " 3  O00443  P3C2A_HUMAN        PIK3C2A              PIK3C2A   \n",
       " 4  P48736  PK3CG_HUMAN         PIK3CG               PIK3CG   \n",
       " 5  P42336  PK3CA_HUMAN         PIK3CA               PIK3CA   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1                 IMPK                         NaN               NaN   \n",
       " 2               PIK3C1                         NaN               NaN   \n",
       " 3                  NaN                         NaN               NaN   \n",
       " 4                  NaN                         NaN               NaN   \n",
       " 5                  NaN                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " 3  Homo sapiens (Human)           9606   \n",
       " 4  Homo sapiens (Human)           9606   \n",
       " 5  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...           NaN   \n",
       " 1  Inositol polyphosphate multikinase (EC 2.7.1.1...  ...           NaN   \n",
       " 2  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...           NaN   \n",
       " 3  Phosphatidylinositol 4-phosphate 3-kinase C2 d...  ...           NaN   \n",
       " 4  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...           NaN   \n",
       " 5  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 524; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2  MOD_RES 324; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 3  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 4  MOD_RES 1024; /note=\"Phosphothreonine; by PKA\"...     NaN   \n",
       " 5                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Autophosphorylation on Ser-1039 results i...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: Autophosphorylation at Ser-1070 negativel...         NaN   \n",
       " 3  PTM: Phosphorylated upon insulin stimulation; ...         NaN   \n",
       " 4  PTM: Autophosphorylation at Ser-1101 has no ef...         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " 4             NaN             NaN   \n",
       " 5             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  5DXU;5M6U;5T8F;5UBT;5VLR;6G6W;6OCO;6OCU;6PYR;6...  \n",
       " 1  5W2G;5W2H;5W2I;6E7F;6M88;6M89;6M8A;6M8B;6M8C;6...  \n",
       " 2                                                NaN  \n",
       " 3           2AR5;2IWL;2REA;2RED;6BTY;6BTZ;6BU0;6BUB;  \n",
       " 4  1E8Y;1E8Z;1HE8;2A4Z;2A5U;2CHW;2CHX;2CHZ;2V4L;3...  \n",
       " 5  2ENQ;2RD0;3HHM;3HIZ;3ZIM;4JPS;4L1B;4L23;4L2Y;4...  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P51163  HEM4_MOUSE  Uros Uros3                 Uros                Uros3   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Uroporphyrinogen-III synthase (UROIIIS) (UROS)...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P10768  ESTD_HUMAN        ESD                  ESD                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  S-formylglutathione hydrolase (FGH) (EC 3.1.2....  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  3FCX;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   P21888  SYC_ECOLI             cysS b0526 JW0515                 cysS   \n",
       " 1   B6I0H1  SYC_ECOSE                cysS ECSE_0551                 cysS   \n",
       " 2   Q8XCT9  SYC_ECO57            cysS Z0681 ECs0588                 cysS   \n",
       " 3   A1A8J2  SYC_ECOK1  cysS Ecok1_04880 APECO1_1488                 cysS   \n",
       " 4   Q1RF08  SYC_ECOUT              cysS UTI89_C0555                 cysS   \n",
       " 5   B7L7F3  SYC_ECO55             cysS EC55989_0540                 cysS   \n",
       " 6   B7MRI9  SYC_ECO81               cysS ECED1_0546                 cysS   \n",
       " 7   A7ZIT5  SYC_ECO24           cysS EcE24377A_0566                 cysS   \n",
       " 8   B1LKE5  SYC_ECOSM             cysS EcSMS35_0571                 cysS   \n",
       " 9   B7ME49  SYC_ECO45               cysS ECS88_0526                 cysS   \n",
       " 10  B7NL20  SYC_ECO7I             cysS ECIAI39_0491                 cysS   \n",
       " 11  B7N980  SYC_ECOLU               cysS ECUMN_0566                 cysS   \n",
       " 12  Q0TKB5  SYC_ECOL5                 cysS ECP_0587                 cysS   \n",
       " 13  Q8FK44  SYC_ECOL6                    cysS c0642                 cysS   \n",
       " 14  B7LJI4  SYC_ESCF3                cysS EFER_0568                 cysS   \n",
       " 15  B1XGC4  SYC_ECODH             cysS ECDH10B_0482                 cysS   \n",
       " 16  Q83M27  SYC_SHIFL             cysS SF0457 S0465                 cysS   \n",
       " 17  B7M4M9  SYC_ECO8A              cysS ECIAI1_0528                 cysS   \n",
       " 18  C4ZUX4  SYC_ECOBW                 cysS BWG_0401                 cysS   \n",
       " 19  A7ZXI1  SYC_ECOHS               cysS EcHS_A0601                 cysS   \n",
       " 20  B5YPP1  SYC_ECO5E            cysS ECH74115_0627                 cysS   \n",
       " 21  B7UKK3  SYC_ECO27              cysS E2348C_0459                 cysS   \n",
       " 22  B1IZ75  SYC_ECOLC               cysS EcolC_3096                 cysS   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN               b0526 JW0515              NaN   \n",
       " 1                    NaN                  ECSE_0551              NaN   \n",
       " 2                    NaN              Z0681 ECs0588              NaN   \n",
       " 3                    NaN                Ecok1_04880      APECO1_1488   \n",
       " 4                    NaN                UTI89_C0555              NaN   \n",
       " 5                    NaN               EC55989_0540              NaN   \n",
       " 6                    NaN                 ECED1_0546              NaN   \n",
       " 7                    NaN             EcE24377A_0566              NaN   \n",
       " 8                    NaN               EcSMS35_0571              NaN   \n",
       " 9                    NaN                 ECS88_0526              NaN   \n",
       " 10                   NaN               ECIAI39_0491              NaN   \n",
       " 11                   NaN                 ECUMN_0566              NaN   \n",
       " 12                   NaN                   ECP_0587              NaN   \n",
       " 13                   NaN                      c0642              NaN   \n",
       " 14                   NaN                  EFER_0568              NaN   \n",
       " 15                   NaN               ECDH10B_0482              NaN   \n",
       " 16                   NaN               SF0457 S0465              NaN   \n",
       " 17                   NaN                ECIAI1_0528              NaN   \n",
       " 18                   NaN                   BWG_0401              NaN   \n",
       " 19                   NaN                 EcHS_A0601              NaN   \n",
       " 20                   NaN              ECH74115_0627              NaN   \n",
       " 21                   NaN                E2348C_0459              NaN   \n",
       " 22                   NaN                 EcolC_3096              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain SE11)         409438   \n",
       " 2                            Escherichia coli O157:H7          83334   \n",
       " 3                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 4              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 5              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 6                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 7   Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 8           Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 9        Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 10      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 11  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 12    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 13  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 14  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 15              Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 16                                  Shigella flexneri            623   \n",
       " 17                  Escherichia coli O8 (strain IAI1)         585034   \n",
       " 18    Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 19                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 20    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 21  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 22  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 1   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 2   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 3   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 4   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 5   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 6   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 7   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 8   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 9   Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 10  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 11  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 12  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 13  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 14  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 15  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 16  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 17  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 18  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 19  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 20  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 21  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " 22  Cysteine--tRNA ligase (EC 6.1.1.16) (Cysteinyl...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                   NaN         NaN               NaN     NaN   \n",
       " 1                   NaN         NaN               NaN     NaN   \n",
       " 2                   NaN         NaN               NaN     NaN   \n",
       " 3                   NaN         NaN               NaN     NaN   \n",
       " 4                   NaN         NaN               NaN     NaN   \n",
       " 5                   NaN         NaN               NaN     NaN   \n",
       " 6                   NaN         NaN               NaN     NaN   \n",
       " 7                   NaN         NaN               NaN     NaN   \n",
       " 8                   NaN         NaN               NaN     NaN   \n",
       " 9                   NaN         NaN               NaN     NaN   \n",
       " 10                  NaN         NaN               NaN     NaN   \n",
       " 11                  NaN         NaN               NaN     NaN   \n",
       " 12                  NaN         NaN               NaN     NaN   \n",
       " 13                  NaN         NaN               NaN     NaN   \n",
       " 14                  NaN         NaN               NaN     NaN   \n",
       " 15                  NaN         NaN               NaN     NaN   \n",
       " 16                  NaN         NaN               NaN     NaN   \n",
       " 17                  NaN         NaN               NaN     NaN   \n",
       " 18                  NaN         NaN               NaN     NaN   \n",
       " 19                  NaN         NaN               NaN     NaN   \n",
       " 20                  NaN         NaN               NaN     NaN   \n",
       " 21                  NaN         NaN               NaN     NaN   \n",
       " 22                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " 21                              NaN         NaN             NaN   \n",
       " 22                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide              PDB  \n",
       " 0              NaN  1LI5;1LI7;1U0B;  \n",
       " 1              NaN              NaN  \n",
       " 2              NaN              NaN  \n",
       " 3              NaN              NaN  \n",
       " 4              NaN              NaN  \n",
       " 5              NaN              NaN  \n",
       " 6              NaN              NaN  \n",
       " 7              NaN              NaN  \n",
       " 8              NaN              NaN  \n",
       " 9              NaN              NaN  \n",
       " 10             NaN              NaN  \n",
       " 11             NaN              NaN  \n",
       " 12             NaN              NaN  \n",
       " 13             NaN              NaN  \n",
       " 14             NaN              NaN  \n",
       " 15             NaN              NaN  \n",
       " 16             NaN              NaN  \n",
       " 17             NaN              NaN  \n",
       " 18             NaN              NaN  \n",
       " 19             NaN              NaN  \n",
       " 20             NaN              NaN  \n",
       " 21             NaN              NaN  \n",
       " 22             NaN              NaN  \n",
       " \n",
       " [23 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name                          Gene Names  \\\n",
       " 0   P05042  FUMC_ECOLI                   fumC b1611 JW1603   \n",
       " 1   P0AC33  FUMA_ECOLI                   fumA b1612 JW1604   \n",
       " 2   P14407  FUMB_ECOLI                   fumB b4122 JW4083   \n",
       " 3   P0ACX5  FUMD_ECOLI              fumD ydhZ b1675 JW1665   \n",
       " 4   Q8X4P8  FUMA_ECO57                  fumA Z2615 ECs2318   \n",
       " 5   Q8X769  FUMC_ECO57                  fumC Z2614 ECs2317   \n",
       " 6   P0AC34  FUMA_ECOL6                          fumA c2004   \n",
       " 7   Q8FHA7  FUMC_ECOL6                          fumC c2003   \n",
       " 8   P11663  FUME_ECOLI              fumE yggD b2929 JW2896   \n",
       " 9   P0ACX7  FUMD_ECO57             fumD ydhZ Z2703 ECs2382   \n",
       " 10  Q83ML8  FUMC_SHIFL                   fumC SF1634 S1765   \n",
       " 11  P0ACX6  FUMD_ECOL6                     fumD ydhZ c2069   \n",
       " 12  P0ACX8  FUMD_SHIFL              fumD ydhZ SF1703 S1835   \n",
       " 13  Q83Q96  FUME_SHIFL  fumE yggD S3120 SF2919 AJR24_04990   \n",
       " \n",
       "    Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                  fumC                  NaN               b1611 JW1603   \n",
       " 1                  fumA                  NaN               b1612 JW1604   \n",
       " 2                  fumB                  NaN               b4122 JW4083   \n",
       " 3                  fumD                 ydhZ               b1675 JW1665   \n",
       " 4                  fumA                  NaN              Z2615 ECs2318   \n",
       " 5                  fumC                  NaN              Z2614 ECs2317   \n",
       " 6                  fumA                  NaN                      c2004   \n",
       " 7                  fumC                  NaN                      c2003   \n",
       " 8                  fumE                 yggD               b2929 JW2896   \n",
       " 9                  fumD                 ydhZ              Z2703 ECs2382   \n",
       " 10                 fumC                  NaN               SF1634 S1765   \n",
       " 11                 fumD                 ydhZ                      c2069   \n",
       " 12                 fumD                 ydhZ               SF1703 S1835   \n",
       " 13                 fumE                 yggD               S3120 SF2919   \n",
       " \n",
       "    Gene Names (ORF)                                           Organism  \\\n",
       " 0               NaN                      Escherichia coli (strain K12)   \n",
       " 1               NaN                      Escherichia coli (strain K12)   \n",
       " 2               NaN                      Escherichia coli (strain K12)   \n",
       " 3               NaN                      Escherichia coli (strain K12)   \n",
       " 4               NaN                           Escherichia coli O157:H7   \n",
       " 5               NaN                           Escherichia coli O157:H7   \n",
       " 6               NaN  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...   \n",
       " 7               NaN  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...   \n",
       " 8               NaN                      Escherichia coli (strain K12)   \n",
       " 9               NaN                           Escherichia coli O157:H7   \n",
       " 10              NaN                                  Shigella flexneri   \n",
       " 11              NaN  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...   \n",
       " 12              NaN                                  Shigella flexneri   \n",
       " 13      AJR24_04990                                  Shigella flexneri   \n",
       " \n",
       "     Organism (ID)                                      Protein names  ...  \\\n",
       " 0           83333  Fumarate hydratase class II (Fumarase C) (EC 4...  ...   \n",
       " 1           83333  Fumarate hydratase class I, aerobic (EC 4.2.1....  ...   \n",
       " 2           83333  Fumarate hydratase class I, anaerobic (EC 4.2....  ...   \n",
       " 3           83333                            Fumarase D (EC 4.2.1.2)  ...   \n",
       " 4           83334  Fumarate hydratase class I, aerobic (EC 4.2.1....  ...   \n",
       " 5           83334  Fumarate hydratase class II (Fumarase C) (EC 4...  ...   \n",
       " 6          199310  Fumarate hydratase class I, aerobic (EC 4.2.1....  ...   \n",
       " 7          199310  Fumarate hydratase class II (Fumarase C) (EC 4...  ...   \n",
       " 8           83333                            Fumarase E (EC 4.2.1.2)  ...   \n",
       " 9           83334                            Fumarase D (EC 4.2.1.2)  ...   \n",
       " 10            623  Fumarate hydratase class II (Fumarase C) (EC 4...  ...   \n",
       " 11         199310                            Fumarase D (EC 4.2.1.2)  ...   \n",
       " 12            623                            Fumarase D (EC 4.2.1.2)  ...   \n",
       " 13            623                            Fumarase E (EC 4.2.1.2)  ...   \n",
       " \n",
       "    Glycosylation                               Initiator methionine  \\\n",
       " 0            NaN                                                NaN   \n",
       " 1            NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 2            NaN                                                NaN   \n",
       " 3            NaN                                                NaN   \n",
       " 4            NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 5            NaN                                                NaN   \n",
       " 6            NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 7            NaN                                                NaN   \n",
       " 8            NaN                                                NaN   \n",
       " 9            NaN                                                NaN   \n",
       " 10           NaN                                                NaN   \n",
       " 11           NaN                                                NaN   \n",
       " 12           NaN                                                NaN   \n",
       " 13           NaN                                                NaN   \n",
       " \n",
       "     Lipidation                                   Modified residue Peptide  \\\n",
       " 0          NaN                                                NaN     NaN   \n",
       " 1          NaN                                                NaN     NaN   \n",
       " 2          NaN  MOD_RES 192; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 3          NaN                                                NaN     NaN   \n",
       " 4          NaN                                                NaN     NaN   \n",
       " 5          NaN                                                NaN     NaN   \n",
       " 6          NaN                                                NaN     NaN   \n",
       " 7          NaN                                                NaN     NaN   \n",
       " 8          NaN                                                NaN     NaN   \n",
       " 9          NaN                                                NaN     NaN   \n",
       " 10         NaN                                                NaN     NaN   \n",
       " 11         NaN                                                NaN     NaN   \n",
       " 12         NaN                                                NaN     NaN   \n",
       " 13         NaN                                                NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                                                PDB  \n",
       " 0              NaN  1FUO;1FUP;1FUQ;1FUR;1KQ7;1YFE;2FUS;6NZ9;6NZA;6...  \n",
       " 1              NaN                                                NaN  \n",
       " 2              NaN                                                NaN  \n",
       " 3              NaN                                                NaN  \n",
       " 4              NaN                                                NaN  \n",
       " 5              NaN                                                NaN  \n",
       " 6              NaN                                                NaN  \n",
       " 7              NaN                                                NaN  \n",
       " 8              NaN                                                NaN  \n",
       " 9              NaN                                                NaN  \n",
       " 10             NaN                                                NaN  \n",
       " 11             NaN                                                NaN  \n",
       " 12             NaN                                                NaN  \n",
       " 13             NaN                                              3C8G;  \n",
       " \n",
       " [14 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q93088  BHMT1_HUMAN       BHMT                 BHMT                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Betaine--homocysteine S-methyltransferase 1 (E...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 40; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  1LT7;1LT8;4M3P;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q8VCN5  CGL_MOUSE        Cth                  Cth                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Cystathionine gamma-lyase (CGL) (CSE) (EC 4.4....  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 211; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P54646  AAPK2_HUMAN  PRKAA2 AMPK AMPK2               PRKAA2   \n",
       " 1  Q13131  AAPK1_HUMAN       PRKAA1 AMPK1               PRKAA1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           AMPK AMPK2                         NaN               NaN   \n",
       " 1                AMPK1                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  5'-AMP-activated protein kinase catalytic subu...  ...           NaN   \n",
       " 1  5'-AMP-activated protein kinase catalytic subu...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 172; /note=\"Phosphothreonine; by LKB1 ...     NaN   \n",
       " 1  MOD_RES 32; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Ubiquitinated. {ECO:0000250|UniProtKB:Q8B...         NaN   \n",
       " 1  PTM: Ubiquitinated. {ECO:0000250|UniProtKB:Q5E...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  2H6D;2LTU;2YZA;3AQV;4CFE;4CFF;4ZHX;5EZV;5ISO;6...  \n",
       " 1  4RED;4RER;4REW;5EZV;6C9F;6C9G;6C9H;6C9J;7JHG;7...  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q5SIP7  RRAAH_THET8   TTHA1322                   NaN                   NaN   \n",
       " 1  Q72J23  RRAAH_THET2   TT_C0958                   NaN                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   TTHA1322               NaN   \n",
       " 1                   TT_C0958               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Thermus thermophilus (strain ATCC 27634 / DSM ...         300852   \n",
       " 1  Thermus thermophilus (strain ATCC BAA-163 / DS...         262724   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  4-hydroxy-4-methyl-2-oxoglutarate aldolase (HM...  ...           NaN   \n",
       " 1  Putative 4-hydroxy-4-methyl-2-oxoglutarate ald...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1J3L;  \n",
       " 1             NaN    NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P35571   GPDM_RAT       Gpd2                 Gpd2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Glycerol-3-phosphate dehydrogenase, mitochondr...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 601; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..42; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9N1E2  G6PI_RABIT        GPI                  GPI                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Oryctolagus cuniculus (Rabbit)           9986   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glucose-6-phosphate isomerase (GPI) (EC 5.3.1....  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylation at Ser-185 by CK2 has bee...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                                  PDB  \n",
       " 0             NaN             NaN  1DQR;1G98;1HM5;1HOX;1KOJ;1N8T;1XTB;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q5FVR2   TYPH_RAT  Tymp Ecgf1                 Tymp                Ecgf1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Thymidine phosphorylase (TP) (EC 2.4.2.4) (TdR...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 6; /note=\"Phosphothreonine\"; /evidence...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q28198  TPA_BOVIN       PLAT                 PLAT                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Tissue-type plasminogen activator (t-PA) (t-pl...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 153; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: The single chain, almost fully active enz...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 22..33; /evidence=\"ECO:0000250\"; /id=\"P...   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..21; /evidence=\"ECO:0000250\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names  Gene Names (primary)  \\\n",
       " 0  P05980  PGFS1_BOVIN         NaN                   NaN   \n",
       " 1  P52897  PGFS2_BOVIN         NaN                   NaN   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN               NaN   \n",
       " 1                   NaN                         NaN               NaN   \n",
       " \n",
       "               Organism  Organism (ID)  \\\n",
       " 0  Bos taurus (Bovine)           9913   \n",
       " 1  Bos taurus (Bovine)           9913   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Prostaglandin F synthase 1 (PGF 1) (PGF syntha...  ...           NaN   \n",
       " 1  Prostaglandin F synthase 2 (PGF 2) (PGF syntha...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0  PTM: The N-terminus is blocked.         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  E9PUQ8  DGKD_MOUSE           Dgkd                 Dgkd   \n",
       " 1  D3YWQ0  DGKI_MOUSE           Dgki                 Dgki   \n",
       " 2  Q6NS52  DGKB_MOUSE  Dgkb Kiaa0718                 Dgkb   \n",
       " 3  Q9ESW4   AGK_MOUSE       Agk Mulk                  Agk   \n",
       " 4  D3YXJ0  DGKH_MOUSE           Dgkh                 Dgkh   \n",
       " 5  O88673  DGKA_MOUSE     Dgka Dagk1                 Dgka   \n",
       " 6  Q6P5E8  DGKQ_MOUSE           Dgkq                 Dgkq   \n",
       " 7  Q9R1C6  DGKE_MOUSE           Dgke                 Dgke   \n",
       " 8  Q91WG7  DGKG_MOUSE     Dgkg Dagk3                 Dgkg   \n",
       " 9  Q80UP3  DGKZ_MOUSE           Dgkz                 Dgkz   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1                  NaN                         NaN               NaN   \n",
       " 2             Kiaa0718                         NaN               NaN   \n",
       " 3                 Mulk                         NaN               NaN   \n",
       " 4                  NaN                         NaN               NaN   \n",
       " 5                Dagk1                         NaN               NaN   \n",
       " 6                  NaN                         NaN               NaN   \n",
       " 7                  NaN                         NaN               NaN   \n",
       " 8                Dagk3                         NaN               NaN   \n",
       " 9                  NaN                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " 2  Mus musculus (Mouse)          10090   \n",
       " 3  Mus musculus (Mouse)          10090   \n",
       " 4  Mus musculus (Mouse)          10090   \n",
       " 5  Mus musculus (Mouse)          10090   \n",
       " 6  Mus musculus (Mouse)          10090   \n",
       " 7  Mus musculus (Mouse)          10090   \n",
       " 8  Mus musculus (Mouse)          10090   \n",
       " 9  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Diacylglycerol kinase delta (DAG kinase delta)...  ...           NaN   \n",
       " 1  Diacylglycerol kinase iota (DAG kinase iota) (...  ...           NaN   \n",
       " 2  Diacylglycerol kinase beta (DAG kinase beta) (...  ...           NaN   \n",
       " 3  Acylglycerol kinase, mitochondrial (EC 2.7.1.1...  ...           NaN   \n",
       " 4  Diacylglycerol kinase eta (DAG kinase eta) (EC...  ...           NaN   \n",
       " 5  Diacylglycerol kinase alpha (DAG kinase alpha)...  ...           NaN   \n",
       " 6  Diacylglycerol kinase theta (DAG kinase theta)...  ...           NaN   \n",
       " 7  Diacylglycerol kinase epsilon (DAG kinase epsi...  ...           NaN   \n",
       " 8  Diacylglycerol kinase gamma (DAG kinase gamma)...  ...           NaN   \n",
       " 9  Diacylglycerol kinase zeta (DAG kinase zeta) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " 3                  NaN         NaN   \n",
       " 4                  NaN         NaN   \n",
       " 5                  NaN         NaN   \n",
       " 6                  NaN         NaN   \n",
       " 7                  NaN         NaN   \n",
       " 8                  NaN         NaN   \n",
       " 9                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2  MOD_RES 117; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 3  MOD_RES 6; /note=\"N6-acetyllysine\"; /evidence=...     NaN   \n",
       " 4                                                NaN     NaN   \n",
       " 5  MOD_RES 479; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 6  MOD_RES 22; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 7                                                NaN     NaN   \n",
       " 8                                                NaN     NaN   \n",
       " 9  MOD_RES 706; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4  PTM: Phosphorylated. Phosphorylation does not ...         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6  PTM: Phosphorylated by PRKCE and PRKCH in vitr...         NaN   \n",
       " 7                                                NaN         NaN   \n",
       " 8                                                NaN         NaN   \n",
       " 9                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " 3             NaN             NaN  NaN  \n",
       " 4             NaN             NaN  NaN  \n",
       " 5             NaN             NaN  NaN  \n",
       " 6             NaN             NaN  NaN  \n",
       " 7             NaN             NaN  NaN  \n",
       " 8             NaN             NaN  NaN  \n",
       " 9             NaN             NaN  NaN  \n",
       " \n",
       " [10 rows x 110 columns],\n",
       "     Entry   Entry Name                                         Gene Names  \\\n",
       " 0  Q6AVA8  PPDK1_ORYSJ  PPDK1 CPDK1 PPDKB Os05g0405000 LOC_Os05g33570 ...   \n",
       " 1  Q75KR1  PPDK2_ORYSJ  PPDK2 CPDK2 Os03g0432100 LOC_Os03g31750 OsJ_01...   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym)   Gene Names (ordered locus)  \\\n",
       " 0                PPDK1          CPDK1 PPDKB  Os05g0405000 LOC_Os05g33570   \n",
       " 1                PPDK2                CPDK2  Os03g0432100 LOC_Os03g31750   \n",
       " \n",
       "               Gene Names (ORF)                             Organism  \\\n",
       " 0      OJ1174_H11.5 OsJ_017742  Oryza sativa subsp. japonica (Rice)   \n",
       " 1  OsJ_010907 OSJNBa0036E17.10  Oryza sativa subsp. japonica (Rice)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          39947  Pyruvate, phosphate dikinase 1, chloroplastic ...  ...   \n",
       " 1          39947  Pyruvate, phosphate dikinase 2 (EC 2.7.9.1) (P...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 527; /note=\"Phosphothreonine; by PDRP1...     NaN   \n",
       " 1  MOD_RES 467; /note=\"Phosphothreonine; by PDRP1...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylation of Thr-527 in the dark in...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide PDB  \n",
       " 0             NaN  TRANSIT 1..71; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " 1             NaN                                                NaN NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  P14755  CRYL1_RABIT  CRYL1 GUL3DH                CRYL1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0               GUL3DH                         NaN               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Oryctolagus cuniculus (Rabbit)           9986   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Lambda-crystallin (EC 1.1.1.45) (L-gulonate 3-...  ...           NaN   \n",
       " \n",
       "           Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  3ADO;3ADP;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name    Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P10584  MDH_THETH           mdh                  mdh                   NaN   \n",
       " 1  Q5SKV7  MDH_THET8  mdh TTHA0536                  mdh                   NaN   \n",
       " 2  P61977  MDH_THET2  mdh TT_C0168                  mdh                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                        NaN               NaN   \n",
       " 1                   TTHA0536               NaN   \n",
       " 2                   TT_C0168               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                               Thermus thermophilus            274   \n",
       " 1  Thermus thermophilus (strain ATCC 27634 / DSM ...         300852   \n",
       " 2  Thermus thermophilus (strain ATCC BAA-163 / DS...         262724   \n",
       " \n",
       "                         Protein names  ... Glycosylation Initiator methionine  \\\n",
       " 0  Malate dehydrogenase (EC 1.1.1.37)  ...           NaN                  NaN   \n",
       " 1  Malate dehydrogenase (EC 1.1.1.37)  ...           NaN                  NaN   \n",
       " 2  Malate dehydrogenase (EC 1.1.1.37)  ...           NaN                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " 2         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide  \\\n",
       " 0         NaN             NaN             NaN   \n",
       " 1         NaN             NaN             NaN   \n",
       " 2         NaN             NaN             NaN   \n",
       " \n",
       "                                              PDB  \n",
       " 0  1BDM;1BMD;1IZ9;1WZE;1WZI;1Y7T;2CVQ;4KDE;4KDF;  \n",
       " 1                                            NaN  \n",
       " 2                                            NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  P34884    MIF_MOUSE               Mif                  Mif   \n",
       " 1  P29812  TYRP2_MOUSE  Dct Tyrp-2 Tyrp2                  Dct   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1         Tyrp-2 Tyrp2                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Macrophage migration inhibitory factor (MIF) (...  ...   \n",
       " 1  L-dopachrome tautomerase (DCT) (DT) (EC 5.3.3....  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0                                                NaN   \n",
       " 1  CARBOHYD 92; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 78; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "                     Post-translational modification  Propeptide  \\\n",
       " 0                                               NaN         NaN   \n",
       " 1  PTM: Glycosylated. {ECO:0000269|PubMed:1537333}.         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide              PDB  \n",
       " 0                                    NaN             NaN  1MFF;1MFI;2GDG;  \n",
       " 1  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN              NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q8BRK8  AAPK2_MOUSE     Prkaa2               Prkaa2                   NaN   \n",
       " 1  Q5EG47  AAPK1_MOUSE     Prkaa1               Prkaa1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 1                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  5'-AMP-activated protein kinase catalytic subu...  ...   \n",
       " 1          10090  5'-AMP-activated protein kinase catalytic subu...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 172; /note=\"Phosphothreonine; by LKB1 ...     NaN   \n",
       " 1  MOD_RES 32; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Ubiquitinated. {ECO:0000269|PubMed:182547...         NaN   \n",
       " 1  PTM: Phosphorylated at Thr-183 by STK11/LKB1 i...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide    PDB  \n",
       " 0             NaN             NaN    NaN  \n",
       " 1             NaN             NaN  5UFU;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  Q60597  ODO1_MOUSE  Ogdh Kiaa4192                 Ogdh   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0             Kiaa4192                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  2-oxoglutarate dehydrogenase complex component...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 74; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide         PDB  \n",
       " 0  TRANSIT 1..40; /note=\"Mitochondrion\"; /evidenc...  3E2H;3TF7;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P35596  GLPO_STRPN  glpO exp6 SP_2185                 glpO   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 exp6                    SP_2185               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Streptococcus pneumoniae serotype 4 (strain AT...         170187   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Alpha-glycerophosphate oxidase (EC 1.1.3.21) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P00484  CAT3_ECOLX       cat3                 cat3                  NaN   \n",
       " 1  P22615  CAT2_ECOLX       cmlA                 cmlA                  NaN   \n",
       " 2  P62577   CAT_ECOLX        cat                  cat                  NaN   \n",
       " 3  P26838  CAT4_ECOLX  catB2 cat                catB2                  cat   \n",
       " 4  P22616  CAT2_HAEIF    cat-IIH              cat-IIH                  NaN   \n",
       " 5  P26826  CAT2_CLOPF       catP                 catP                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN         Escherichia coli   \n",
       " 1                         NaN               NaN         Escherichia coli   \n",
       " 2                         NaN               NaN         Escherichia coli   \n",
       " 3                         NaN               NaN         Escherichia coli   \n",
       " 4                         NaN               NaN   Haemophilus influenzae   \n",
       " 5                         NaN               NaN  Clostridium perfringens   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0            562  Chloramphenicol acetyltransferase 3 (EC 2.3.1....  ...   \n",
       " 1            562  Chloramphenicol acetyltransferase 2 (EC 2.3.1....  ...   \n",
       " 2            562  Chloramphenicol acetyltransferase (CAT) (EC 2....  ...   \n",
       " 3            562    Chloramphenicol acetyltransferase (EC 2.3.1.28)  ...   \n",
       " 4            727  Chloramphenicol acetyltransferase 2 (EC 2.3.1....  ...   \n",
       " 5           1502  Chloramphenicol acetyltransferase (CAT) (EC 2....  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " 1            NaN                  NaN         NaN               NaN     NaN   \n",
       " 2            NaN                  NaN         NaN               NaN     NaN   \n",
       " 3            NaN                  NaN         NaN               NaN     NaN   \n",
       " 4            NaN                  NaN         NaN               NaN     NaN   \n",
       " 5            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide Signal peptide Transit peptide  \\\n",
       " 0                              NaN         NaN            NaN             NaN   \n",
       " 1                              NaN         NaN            NaN             NaN   \n",
       " 2                              NaN         NaN            NaN             NaN   \n",
       " 3                              NaN         NaN            NaN             NaN   \n",
       " 4                              NaN         NaN            NaN             NaN   \n",
       " 5                              NaN         NaN            NaN             NaN   \n",
       " \n",
       "                                    PDB  \n",
       " 0  1CIA;1CLA;1QCA;2CLA;3CLA;4CLA;6X7Q;  \n",
       " 1                                  NaN  \n",
       " 2            1NOC;1PD5;1Q23;3U9B;3U9F;  \n",
       " 3                                  NaN  \n",
       " 4                                  NaN  \n",
       " 5                                  NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry   Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  Q6P587  FAHD1_HUMAN  FAHD1 C16orf36 YISKL                FAHD1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0       C16orf36 YISKL                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Acylpyruvase FAHD1, mitochondrial (EC 3.7.1.5)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 40; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide              PDB  \n",
       " 0  TRANSIT 1..27; /note=\"Mitochondrion\"; /evidenc...  1SAW;6FOG;6FOH;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q831A3  ATPA_ENTFA    atpA EF_2610                 atpA   \n",
       " 1  Q831A5  ATPB_ENTFA    atpD EF_2608                 atpD   \n",
       " 2  P26679  ATPA_ENTHA  atpA EHR_08490                 atpA   \n",
       " 3  P43451  ATPB_ENTHA  atpD EHR_08500                 atpD   \n",
       " 4  Q834X9  VATA_ENTFA    atpA EF_1498                 atpA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    EF_2610               NaN   \n",
       " 1                   NaN                    EF_2608               NaN   \n",
       " 2                   NaN                  EHR_08490               NaN   \n",
       " 3                   NaN                  EHR_08500               NaN   \n",
       " 4                   NaN                    EF_1498               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Enterococcus faecalis (strain ATCC 700802 / V583)         226185   \n",
       " 1  Enterococcus faecalis (strain ATCC 700802 / V583)         226185   \n",
       " 2  Enterococcus hirae (strain ATCC 9790 / DSM 201...         768486   \n",
       " 3  Enterococcus hirae (strain ATCC 9790 / DSM 201...         768486   \n",
       " 4  Enterococcus faecalis (strain ATCC 700802 / V583)         226185   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  ATP synthase subunit alpha (EC 7.1.2.2) (ATP s...  ...           NaN   \n",
       " 1  ATP synthase subunit beta (EC 7.1.2.2) (ATP sy...  ...           NaN   \n",
       " 2  ATP synthase subunit alpha (EC 7.1.2.2) (ATP s...  ...           NaN   \n",
       " 3  ATP synthase subunit beta (EC 7.1.2.2) (ATP sy...  ...           NaN   \n",
       " 4  V-type ATP synthase alpha chain (EC 7.1.2.2) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  D4QF25  LIPL_STRSQ       lipL                 lipL                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)         Organism  \\\n",
       " 0                         NaN               NaN  Streptomyces sp   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           1931  Uridine-5'-phosphate dioxygenase (EC 1.14.11.4...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P34947  GRK5_HUMAN         GRK5 GPRK5                 GRK5   \n",
       " 1  P43250  GRK6_HUMAN         GRK6 GPRK6                 GRK6   \n",
       " 2  P32298  GRK4_HUMAN  GRK4 GPRK2L GPRK4                 GRK4   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                GPRK5                         NaN               NaN   \n",
       " 1                GPRK6                         NaN               NaN   \n",
       " 2         GPRK2L GPRK4                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  G protein-coupled receptor kinase 5 (EC 2.7.11...  ...           NaN   \n",
       " 1  G protein-coupled receptor kinase 6 (EC 2.7.11...  ...           NaN   \n",
       " 2  G protein-coupled receptor kinase 4 (EC 2.7.11...  ...           NaN   \n",
       " \n",
       "   Initiator methionine                                         Lipidation  \\\n",
       " 0                  NaN                                                NaN   \n",
       " 1                  NaN  LIPID 561; /note=\"S-palmitoyl cysteine\"; /evid...   \n",
       " 2                  NaN                                                NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 484; /note=\"Phosphoserine; by autocata...     NaN   \n",
       " 1  MOD_RES 484; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 2  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Autophosphorylated. Autophosphorylation m...         NaN   \n",
       " 1  PTM: It is uncertain whether palmitoylation is...         NaN   \n",
       " 2  PTM: Palmitoylated. {ECO:0000269|PubMed:8626439}.         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide              PDB  \n",
       " 0             NaN             NaN  4TNB;4TND;6PJX;  \n",
       " 1             NaN             NaN  2ACX;3NYN;3NYO;  \n",
       " 2             NaN             NaN            4YHJ;  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  A6ZR27  ALDH5_YEAS7  ALD5 SCY_1574                 ALD5   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN         SCY_1574   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain YJM789) (Bake...         307796   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aldehyde dehydrogenase 5, mitochondrial (EC 1....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..23; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  Q07637  KPYK_LACLA  pyk LL1332 L0003                  pyk   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                     LL1332            L0003   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Lactococcus lactis subsp. lactis (strain IL140...         272623   \n",
       " \n",
       "                         Protein names  ... Glycosylation Initiator methionine  \\\n",
       " 0  Pyruvate kinase (PK) (EC 2.7.1.40)  ...           NaN                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide  PDB  \n",
       " 0         NaN             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  P21695   GPDA_HUMAN            GPD1                 GPD1   \n",
       " 1  Q8N335  GPD1L_HUMAN  GPD1L KIAA0089                GPD1L   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1             KIAA0089                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glycerol-3-phosphate dehydrogenase [NAD(+)], c...  ...           NaN   \n",
       " 1  Glycerol-3-phosphate dehydrogenase 1-like prot...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 154; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                  PDB  \n",
       " 0             NaN  1WPQ;1X0V;1X0X;6E8Y;6E8Z;6E90;6PYP;  \n",
       " 1             NaN                                2PLA;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q06518   NOS2_RAT       Nos2                 Nos2                  NaN   \n",
       " 1  P29476   NOS1_RAT  Nos1 Bnos                 Nos1                 Bnos   \n",
       " 2  Q62600   NOS3_RAT       Nos3                 Nos3                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 2                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Nitric oxide synthase, inducible (EC 1.14.13.3...  ...   \n",
       " 1          10116  Nitric oxide synthase, brain (EC 1.14.13.39) (...  ...   \n",
       " 2          10116  Nitric oxide synthase, endothelial (EC 1.14.13...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN                                                NaN   \n",
       " 1           NaN                                                NaN   \n",
       " 2           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0                                                NaN   \n",
       " 1                                                NaN   \n",
       " 2  LIPID 2; /note=\"N-myristoyl glycine\"; /evidenc...   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 564; /note=\"Phosphothreonine\"; /eviden...     NaN   \n",
       " 1  MOD_RES 280; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 2  MOD_RES 494; /note=\"Phosphothreonine; by AMPK\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Polyubiquitinated; mediated by SPSB1, SPS...         NaN   \n",
       " 1  PTM: Ubiquitinated; mediated by STUB1/CHIP in ...         NaN   \n",
       " 2  PTM: Phosphorylation by AMPK at Ser-1176 in th...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                                NaN  \n",
       " 1  1B8Q;1CMI;1F20;1K2R;1K2S;1K2T;1K2U;1LZX;1LZZ;1...  \n",
       " 2                                                NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q8BH73  QPCTL_MOUSE      Qpctl                Qpctl                   NaN   \n",
       " 1  Q9CYK2   QPCT_MOUSE       Qpct                 Qpct                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 1                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Glutaminyl-peptide cyclotransferase-like prote...  ...   \n",
       " 1          10090  Glutaminyl-peptide cyclotransferase (EC 2.3.2....  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0                                                NaN                  NaN   \n",
       " 1  CARBOHYD 29; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide                         Signal peptide Transit peptide  \\\n",
       " 0         NaN                                    NaN             NaN   \n",
       " 1         NaN  SIGNAL 1..35; /evidence=\"ECO:0000255\"             NaN   \n",
       " \n",
       "           PDB  \n",
       " 0         NaN  \n",
       " 1  3SI1;3SI2;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name   Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P53690  MMP14_MOUSE  Mmp14 Mtmmp                Mmp14                Mtmmp   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Matrix metalloproteinase-14 (MMP-14) (EC 3.4.2...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 399; /note=\"Phosphotyrosine; by PKDCC\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: The precursor is cleaved by a furin endop...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 21..111; /evidence=\"ECO:0000250\"; /id=\"...   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..20; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  Q43153  CYSC_SPIOL  CYSC CAS CSase C                 CYSC   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0          CAS CSase C                         NaN               NaN   \n",
       " \n",
       "                       Organism  Organism (ID)  \\\n",
       " 0  Spinacia oleracea (Spinach)           3562   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Bifunctional L-3-cyanoalanine synthase/cystein...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 90; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..27; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  Q0P5I5  HOGA1_BOVIN  HOGA1 DHDPSL                HOGA1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0               DHDPSL                         NaN               NaN   \n",
       " \n",
       "               Organism  Organism (ID)  \\\n",
       " 0  Bos taurus (Bovine)           9913   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  4-hydroxy-2-oxoglutarate aldolase, mitochondri...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..25; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  O34598  GUAD_BACSU  guaD gde BSU13170                 guaD   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  gde                   BSU13170               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Bacillus subtilis (strain 168)         224308   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Guanine deaminase (GDEase) (Guanase) (Guanine ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1TIY;1WKQ;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  O88509  DNM3B_MOUSE               Dnmt3b               Dnmt3b   \n",
       " 1  P0DOY1  DNM3C_MOUSE       Dnmt3c Gm14490               Dnmt3c   \n",
       " 2  O88508  DNM3A_MOUSE               Dnmt3a               Dnmt3a   \n",
       " 3  P13864  DNMT1_MOUSE  Dnmt1 Dnmt Met1 Uim                Dnmt1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1              Gm14490                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " 3        Dnmt Met1 Uim                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " 2  Mus musculus (Mouse)          10090   \n",
       " 3  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  DNA (cytosine-5)-methyltransferase 3B (Dnmt3b)...  ...           NaN   \n",
       " 1  DNA (cytosine-5)-methyltransferase 3C (Dnmt3c)...  ...           NaN   \n",
       " 2  DNA (cytosine-5)-methyltransferase 3A (Dnmt3a)...  ...           NaN   \n",
       " 3  DNA (cytosine-5)-methyltransferase 1 (Dnmt1) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " 3                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 96; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2  MOD_RES 102; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 3  MOD_RES 15; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Sumoylated. {ECO:0000250}.; PTM: Citrulli...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: Auto-methylated at Cys-706: auto-methylat...         NaN   \n",
       " 3  PTM: Sumoylated; sumoylation increases activit...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                              1KHC;  \n",
       " 1                                                NaN  \n",
       " 2                                         3SW9;3SWC;  \n",
       " 3  3AV4;3AV5;3AV6;3PT6;3PT9;4DA4;5GUT;5GUV;5WY1;6...  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  P39105  PLB1_YEAST  PLB1 YMR008C YM8270.10C                 PLB1   \n",
       " 1  Q03674  PLB2_YEAST  PLB2 YMR006C YM8270.08C                 PLB2   \n",
       " 2  Q04958  NTE1_YEAST  NTE1 YML059C YM9958.03C                 NTE1   \n",
       " 3  Q08108  PLB3_YEAST             PLB3 YOL011W                 PLB3   \n",
       " 4  Q756Z0  NTE1_ASHGO             NTE1 AER124W                 NTE1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YMR008C       YM8270.10C   \n",
       " 1                   NaN                    YMR006C       YM8270.08C   \n",
       " 2                   NaN                    YML059C       YM9958.03C   \n",
       " 3                   NaN                    YOL011W              NaN   \n",
       " 4                   NaN                    AER124W              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 2  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 3  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 4  Ashbya gossypii (strain ATCC 10895 / CBS 109.5...         284811   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Lysophospholipase 1 (EC 3.1.1.5) (Phospholipas...  ...   \n",
       " 1  Lysophospholipase 2 (EC 3.1.1.5) (Phospholipas...  ...   \n",
       " 2  Lysophospholipase NTE1 (EC 3.1.1.5) (Intracell...  ...   \n",
       " 3  Lysophospholipase 3 (EC 3.1.1.5) (Phospholipas...  ...   \n",
       " 4  Lysophospholipase NTE1 (EC 3.1.1.5) (Intracell...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 26; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 1  CARBOHYD 47; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 2                                                NaN                  NaN   \n",
       " 3  CARBOHYD 56; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 4                                                NaN                  NaN   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0  LIPID 634; /note=\"GPI-anchor amidated asparagi...   \n",
       " 1  LIPID 680; /note=\"GPI-anchor amidated asparagi...   \n",
       " 2                                                NaN   \n",
       " 3  LIPID 659; /note=\"GPI-anchor amidated asparagi...   \n",
       " 4                                                NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2  MOD_RES 300; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 3                                                NaN     NaN   \n",
       " 4                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0                                                NaN   \n",
       " 1  PTM: The GPI-anchor is attached to the protein...   \n",
       " 2                                                NaN   \n",
       " 3                                                NaN   \n",
       " 4                                                NaN   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 635..664; /note=\"Removed in mature form...   \n",
       " 1  PROPEP 681..706; /note=\"Removed in mature form...   \n",
       " 2                                                NaN   \n",
       " 3  PROPEP 660..686; /note=\"Removed in mature form...   \n",
       " 4                                                NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..22; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " 1  SIGNAL 1..19; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " 2                                    NaN             NaN  NaN  \n",
       " 3  SIGNAL 1..26; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " 4                                    NaN             NaN  NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0  A5U8S6  PAND_MYCTA              panD MRA_3640                 panD   \n",
       " 1  P9WIL3  PAND_MYCTU  panD Rv3601c MTCY07H7B.21                 panD   \n",
       " 2  P9WIL2  PAND_MYCTO              panD MT3706.1                 panD   \n",
       " 3  A1KPT6  PAND_MYCBP             panD BCG_3665c                 panD   \n",
       " 4  C1AI59  PAND_MYCBT              panD JTY_3666                 panD   \n",
       " 5  P65661  PAND_MYCBO        panD BQ2027_MB3631C                 panD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                   MRA_3640              NaN   \n",
       " 1                   NaN                    Rv3601c     MTCY07H7B.21   \n",
       " 2                   NaN                   MT3706.1              NaN   \n",
       " 3                   NaN                  BCG_3665c              NaN   \n",
       " 4                   NaN                   JTY_3666              NaN   \n",
       " 5                   NaN             BQ2027_MB3631C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25177 ...         419947   \n",
       " 1  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 2  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 3  Mycobacterium bovis (strain BCG / Pasteur 1173P2)         410289   \n",
       " 4  Mycobacterium bovis (strain BCG / Tokyo 172 / ...         561275   \n",
       " 5  Mycobacterium bovis (strain ATCC BAA-935 / AF2...         233413   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aspartate 1-decarboxylase (EC 4.1.1.11) (Aspar...  ...           NaN   \n",
       " 1  Aspartate 1-decarboxylase (EC 4.1.1.11) (Aspar...  ...           NaN   \n",
       " 2  Aspartate 1-decarboxylase (EC 4.1.1.11) (Aspar...  ...           NaN   \n",
       " 3  Aspartate 1-decarboxylase (EC 4.1.1.11) (Aspar...  ...           NaN   \n",
       " 4  Aspartate 1-decarboxylase (EC 4.1.1.11) (Aspar...  ...           NaN   \n",
       " 5  Aspartate 1-decarboxylase (EC 4.1.1.11) (Aspar...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " 3                  NaN         NaN   \n",
       " 4                  NaN         NaN   \n",
       " 5                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 25; /note=\"Pyruvic acid (Ser)\"; /evide...     NaN   \n",
       " 1  MOD_RES 25; /note=\"Pyruvic acid (Ser)\"; /evide...     NaN   \n",
       " 2  MOD_RES 25; /note=\"Pyruvic acid (Ser)\"; /evide...     NaN   \n",
       " 3  MOD_RES 25; /note=\"Pyruvic acid (Ser)\"; /evide...     NaN   \n",
       " 4  MOD_RES 25; /note=\"Pyruvic acid (Ser)\"; /evide...     NaN   \n",
       " 5  MOD_RES 25; /note=\"Pyruvic acid (Ser)\"; /evide...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 1  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 2  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 3  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 4  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " 5  PTM: Is synthesized initially as an inactive p...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide    PDB  \n",
       " 0             NaN             NaN    NaN  \n",
       " 1             NaN             NaN  2C45;  \n",
       " 2             NaN             NaN    NaN  \n",
       " 3             NaN             NaN    NaN  \n",
       " 4             NaN             NaN    NaN  \n",
       " 5             NaN             NaN    NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P00791   PEPA_PIG        PGA                  PGA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Sus scrofa (Pig)   \n",
       " \n",
       "    Organism (ID)           Protein names  ... Glycosylation  \\\n",
       " 0           9823  Pepsin A (EC 3.4.23.1)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 127; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: Minor amounts of the active enzyme occur ...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 16..59; /note=\"Activation peptide\"; /ev...   \n",
       " \n",
       "                                       Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..15; /evidence=\"ECO:0000269|PubMed:24...             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1F34;1PSA;1YX9;2PSG;3PEP;3PSG;4PEP;5PEP;6XCT;6...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  C7FF04  PPO3_AGABI       PPO3                 PPO3                   NaN   \n",
       " 1  Q00024  PPO1_AGABI       PPO1                 PPO1                   NaN   \n",
       " 2  C7FF05  PPO4_AGABI       PPO4                 PPO4                   NaN   \n",
       " 3  O42713  PPO2_AGABI       PPO2                 PPO2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " 2                         NaN               NaN   \n",
       " 3                         NaN               NaN   \n",
       " \n",
       "                                     Organism  Organism (ID)  \\\n",
       " 0  Agaricus bisporus (White button mushroom)           5341   \n",
       " 1  Agaricus bisporus (White button mushroom)           5341   \n",
       " 2  Agaricus bisporus (White button mushroom)           5341   \n",
       " 3  Agaricus bisporus (White button mushroom)           5341   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Polyphenol oxidase 3 (PPO3) (Phenolase 3) (EC ...  ...            NaN   \n",
       " 1  Polyphenol oxidase 1 (PPO1) (Phenolase 1) (EC ...  ...            NaN   \n",
       " 2  Polyphenol oxidase 4 (PPO4) (Phenolase 4) (EC ...  ...            NaN   \n",
       " 3  Polyphenol oxidase 2 (PPO2) (Phenolase 2) (EC ...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: The C-ter is probably cleaved after Gly-3...   \n",
       " 1  PTM: The C-ter is probably cleaved after Gly-3...   \n",
       " 2  PTM: The C-ter is probably cleaved after Gly-3...   \n",
       " 3  PTM: The C-ter is probably cleaved after Gly-3...   \n",
       " \n",
       "                                           Propeptide  Signal peptide  \\\n",
       " 0  PROPEP 393..576; /note=\"Removed in mature form...             NaN   \n",
       " 1  PROPEP 383..565; /note=\"Removed in mature form...             NaN   \n",
       " 2  PROPEP 380..611; /note=\"Removed in mature form...             NaN   \n",
       " 3  PROPEP 379..556; /note=\"Removed in mature form...             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  2Y9W;2Y9X;  \n",
       " 1             NaN         NaN  \n",
       " 2             NaN       5M6B;  \n",
       " 3             NaN         NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P0ABF8  PGSA_ECOLI            pgsA b1912 JW1897                 pgsA   \n",
       " 1  P0ABG0  PGSA_ECO57           pgsA Z3000 ECs2650                 pgsA   \n",
       " 2  P0ABF9  PGSA_ECOL6                   pgsA c2325                 pgsA   \n",
       " 3  P46322  PGSA_BACSU           pgsA ymfN BSU16920                 pgsA   \n",
       " 4  Q83R47  PGSA_SHIFL            pgsA SF1955 S2051                 pgsA   \n",
       " 5  Q1RAM8  PGSA_ECOUT             pgsA UTI89_C2113                 pgsA   \n",
       " 6  A1AC64  PGSA_ECOK1  pgsA Ecok1_17600 APECO1_954                 pgsA   \n",
       " 7  Q0TGS6  PGSA_ECOL5                pgsA ECP_1852                 pgsA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                  NaN               b1912 JW1897              NaN   \n",
       " 1                  NaN              Z3000 ECs2650              NaN   \n",
       " 2                  NaN                      c2325              NaN   \n",
       " 3                 ymfN                   BSU16920              NaN   \n",
       " 4                  NaN               SF1955 S2051              NaN   \n",
       " 5                  NaN                UTI89_C2113              NaN   \n",
       " 6                  NaN                Ecok1_17600       APECO1_954   \n",
       " 7                  NaN                   ECP_1852              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                           Escherichia coli O157:H7          83334   \n",
       " 2  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 3                     Bacillus subtilis (strain 168)         224308   \n",
       " 4                                  Shigella flexneri            623   \n",
       " 5             Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 6                      Escherichia coli O1:K1 / APEC         405955   \n",
       " 7    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  CDP-diacylglycerol--glycerol-3-phosphate 3-pho...  ...           NaN   \n",
       " 1  CDP-diacylglycerol--glycerol-3-phosphate 3-pho...  ...           NaN   \n",
       " 2  CDP-diacylglycerol--glycerol-3-phosphate 3-pho...  ...           NaN   \n",
       " 3  CDP-diacylglycerol--glycerol-3-phosphate 3-pho...  ...           NaN   \n",
       " 4  CDP-diacylglycerol--glycerol-3-phosphate 3-pho...  ...           NaN   \n",
       " 5  CDP-diacylglycerol--glycerol-3-phosphate 3-pho...  ...           NaN   \n",
       " 6  CDP-diacylglycerol--glycerol-3-phosphate 3-pho...  ...           NaN   \n",
       " 7  CDP-diacylglycerol--glycerol-3-phosphate 3-pho...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6                                                NaN         NaN   \n",
       " 7                                                NaN         NaN   \n",
       " \n",
       "    Modified residue Peptide  \\\n",
       " 0               NaN     NaN   \n",
       " 1               NaN     NaN   \n",
       " 2               NaN     NaN   \n",
       " 3               NaN     NaN   \n",
       " 4               NaN     NaN   \n",
       " 5               NaN     NaN   \n",
       " 6               NaN     NaN   \n",
       " 7               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: The N-terminus is probably blocked. {ECO:...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6                                                NaN         NaN   \n",
       " 7                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " 3             NaN             NaN  NaN  \n",
       " 4             NaN             NaN  NaN  \n",
       " 5             NaN             NaN  NaN  \n",
       " 6             NaN             NaN  NaN  \n",
       " 7             NaN             NaN  NaN  \n",
       " \n",
       " [8 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P43188  KADC_MAIZE       ADK1                 ADK1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Zea mays (Maize)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           4577  Adenylate kinase, chloroplastic (AK) (EC 2.7.4...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1ZAK;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q58667  HACB_METJA  hacB MJ1271                 hacB                   NaN   \n",
       " 1  Q58409  HACA_METJA  hacA MJ1003                 hacA                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     MJ1271               NaN   \n",
       " 1                     MJ1003               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " 1  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Methanogen homoaconitase small subunit (HACN) ...  ...           NaN   \n",
       " 1  Methanogen homoaconitase large subunit (HACN) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2PKP;  \n",
       " 1             NaN  4KP2;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name                     Gene Names Gene Names (primary)  \\\n",
       " 0  P41893  PPAL_SCHPO  stp1 SPAC1071.12c SPAC926.01c                 stp1   \n",
       " 1  P08091  PPA1_SCHPO                pho1 SPBP4G3.02                 pho1   \n",
       " 2  Q01682  PPA2_SCHPO               pho4 SPBC428.03c                 pho4   \n",
       " 3  O60172  PPA3_SCHPO                   SPBC21H7.03c                  NaN   \n",
       " 4  Q9USS6  PPA4_SCHPO                       SPBC4.06                  NaN   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus)          Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN  SPAC1071.12c SPAC926.01c   \n",
       " 1                   NaN                         NaN                SPBP4G3.02   \n",
       " 2                   NaN                         NaN               SPBC428.03c   \n",
       " 3                   NaN                         NaN              SPBC21H7.03c   \n",
       " 4                   NaN                         NaN                  SPBC4.06   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " 1  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " 2  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " 3  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " 4  Schizosaccharomyces pombe (strain 972 / ATCC 2...         284812   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Low molecular weight phosphotyrosine protein p...  ...   \n",
       " 1                      Acid phosphatase (EC 3.1.3.2)  ...   \n",
       " 2  Thiamine-repressible acid phosphatase pho4 (EC...  ...   \n",
       " 3  Thiamine-repressible acid phosphatase SPBC21H7...  ...   \n",
       " 4    Probable acid phosphatase SPBC4.06 (EC 3.1.3.2)  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0                                                NaN                  NaN   \n",
       " 1  CARBOHYD 95; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 2  CARBOHYD 98; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 3  CARBOHYD 98; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 4                                                NaN                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " 2         NaN               NaN     NaN                              NaN   \n",
       " 3         NaN               NaN     NaN                              NaN   \n",
       " 4         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide                                     Signal peptide  \\\n",
       " 0         NaN                                                NaN   \n",
       " 1         NaN                                       SIGNAL 1..18   \n",
       " 2         NaN  SIGNAL 1..18; /evidence=\"ECO:0000269|PubMed:22...   \n",
       " 3         NaN              SIGNAL 1..18; /evidence=\"ECO:0000255\"   \n",
       " 4         NaN                                                NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q42777  MCCA_SOYBN       MCCA                 MCCA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                   Organism  Organism (ID)  \\\n",
       " 0  Glycine max (Soybean) (Glycine hispida)           3847   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Methylcrotonoyl-CoA carboxylase subunit alpha,...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 695; /note=\"N6-biotinyllysine\"; /evide...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..22; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  O26061  THYX_HELPY   thyX HP_1533                 thyX   \n",
       " 1  Q9ZJ93  THYX_HELPJ  thyX jhp_1421                 thyX   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    HP_1533               NaN   \n",
       " 1                   NaN                   jhp_1421               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Helicobacter pylori (strain ATCC 700392 / 2669...          85962   \n",
       " 1  Helicobacter pylori (strain J99 / ATCC 700824)...          85963   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Flavin-dependent thymidylate synthase (FDTS) (...  ...           NaN   \n",
       " 1  Flavin-dependent thymidylate synthase (FDTS) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  3AH5;  \n",
       " 1             NaN    NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name    Gene Names Gene Names (primary)  \\\n",
       " 0  P06773  DCTD_YEAST  DCD1 YHR144C                 DCD1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    YHR144C               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Deoxycytidylate deaminase (EC 3.5.4.12) (dCMP ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "      Entry   Entry Name                                      Gene Names  \\\n",
       " 0   P0A6F5   CH60_ECOLI                    groEL groL mopA b4143 JW4103   \n",
       " 1   P10809   CH60_HUMAN                                     HSPD1 HSP60   \n",
       " 2   P9WPE9  CH601_MYCTU          groEL1 cpn60.1 groL1 Rv3417c MTCY78.12   \n",
       " 3   P0A6F7   CH60_ECO57                   groEL groL mopA Z5748 ECs5124   \n",
       " 4   B6I615   CH60_ECOSE                            groEL groL ECSE_4442   \n",
       " 5   B1XDP7   CH60_ECODH                         groEL groL ECDH10B_4336   \n",
       " 6   A1AJ51  CH601_ECOK1            groEL1 groL1 Ecok1_41970 APECO1_2246   \n",
       " 7   B7MSV9   CH60_ECO81                           groEL groL ECED1_4931   \n",
       " 8   Q1R3B6   CH60_ECOUT                          groEL groL UTI89_C4741   \n",
       " 9   B7LC02   CH60_ECO55                         groEL groL EC55989_4698   \n",
       " 10  B7M8Q4   CH60_ECO8A                          groEL groL ECIAI1_4376   \n",
       " 11  Q19NJ4  CH602_ECOK1  groEL2 groL2 Ecok1_R_48900 APECO1_O1R91 O1R_91   \n",
       " 12  C5A1D5   CH60_ECOBW                             groEL groL BWG_3856   \n",
       " 13  A8A7N9   CH60_ECOHS                           groEL groL EcHS_A4384   \n",
       " 14  B5Z2F2   CH60_ECO5E                        groEL groL ECH74115_5659   \n",
       " 15  A7ZV12   CH60_ECO24                       groEL groL EcE24377A_4698   \n",
       " 16  B1LQG4   CH60_ECOSM                         groEL groL EcSMS35_4612   \n",
       " 17  B7MKU8   CH60_ECO45                           groEL groL ECS88_4729   \n",
       " 18  B7NTK2   CH60_ECO7I                         groEL groL ECIAI39_4608   \n",
       " 19  B7UPW3   CH60_ECO27                          groEL groL E2348C_4469   \n",
       " 20  B7NG81   CH60_ECOLU                           groEL groL ECUMN_4678   \n",
       " 21  Q0T9P8   CH60_ECOL5                             groEL groL ECP_4387   \n",
       " 22  P0A6F6   CH60_ECOL6                           groEL groL mopA c5227   \n",
       " 23  P23033   CH60_TREPA                     groEL groL mopA Tp4 TP_0030   \n",
       " 24  P19421   CH60_COXBU                   groEL groL htpB mopA CBU_1718   \n",
       " 25  B1ITQ5   CH60_ECOLC                           groEL groL EcolC_3869   \n",
       " 26  P0A521  CH602_MYCBO                groEL2 groL2 hsp65 BQ2027_MB0448   \n",
       " 27  B7LLS5   CH60_ESCF3                            groEL groL EFER_4195   \n",
       " 28  P42382   CH60_EHRCH                                 groEL groL mopA   \n",
       " 29  P31293   CH60_ALLVI                                 groEL groL mopA   \n",
       " 30  P0C193   CH60_SODGL                                 groEL groL mopA   \n",
       " 31  Q8VV84   CH60_PARTM                                      groEL groL   \n",
       " 32  Q3KMQ9   CH60_CHLTA                        groEL groL hypB CTA_0117   \n",
       " 33  P25750   CH60_BUCAI                      groEL groL mopA symL BU019   \n",
       " 34  P0A6F8   CH60_SHIFL                    groEL groL mopA SF4297 S4564   \n",
       " 35  Q9ANR9   CH60_WIGBR                       groEL groL mopA WIGBR2580   \n",
       " 36  Q2NW94   CH60_SODGM                          groEL groL mopA SG0306   \n",
       " \n",
       "    Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                 groEL            groL mopA               b4143 JW4103   \n",
       " 1                 HSPD1                HSP60                        NaN   \n",
       " 2                groEL1        cpn60.1 groL1                    Rv3417c   \n",
       " 3                 groEL            groL mopA              Z5748 ECs5124   \n",
       " 4                 groEL                 groL                  ECSE_4442   \n",
       " 5                 groEL                 groL               ECDH10B_4336   \n",
       " 6                groEL1                groL1                Ecok1_41970   \n",
       " 7                 groEL                 groL                 ECED1_4931   \n",
       " 8                 groEL                 groL                UTI89_C4741   \n",
       " 9                 groEL                 groL               EC55989_4698   \n",
       " 10                groEL                 groL                ECIAI1_4376   \n",
       " 11               groEL2                groL2              Ecok1_R_48900   \n",
       " 12                groEL                 groL                   BWG_3856   \n",
       " 13                groEL                 groL                 EcHS_A4384   \n",
       " 14                groEL                 groL              ECH74115_5659   \n",
       " 15                groEL                 groL             EcE24377A_4698   \n",
       " 16                groEL                 groL               EcSMS35_4612   \n",
       " 17                groEL                 groL                 ECS88_4729   \n",
       " 18                groEL                 groL               ECIAI39_4608   \n",
       " 19                groEL                 groL                E2348C_4469   \n",
       " 20                groEL                 groL                 ECUMN_4678   \n",
       " 21                groEL                 groL                   ECP_4387   \n",
       " 22                groEL            groL mopA                      c5227   \n",
       " 23                groEL        groL mopA Tp4                    TP_0030   \n",
       " 24                groEL       groL htpB mopA                   CBU_1718   \n",
       " 25                groEL                 groL                 EcolC_3869   \n",
       " 26               groEL2          groL2 hsp65              BQ2027_MB0448   \n",
       " 27                groEL                 groL                  EFER_4195   \n",
       " 28                groEL            groL mopA                        NaN   \n",
       " 29                groEL            groL mopA                        NaN   \n",
       " 30                groEL            groL mopA                        NaN   \n",
       " 31                groEL                 groL                        NaN   \n",
       " 32                groEL            groL hypB                   CTA_0117   \n",
       " 33                groEL       groL mopA symL                      BU019   \n",
       " 34                groEL            groL mopA               SF4297 S4564   \n",
       " 35                groEL            groL mopA                  WIGBR2580   \n",
       " 36                groEL            groL mopA                     SG0306   \n",
       " \n",
       "        Gene Names (ORF)                                           Organism  \\\n",
       " 0                   NaN                      Escherichia coli (strain K12)   \n",
       " 1                   NaN                               Homo sapiens (Human)   \n",
       " 2             MTCY78.12  Mycobacterium tuberculosis (strain ATCC 25618 ...   \n",
       " 3                   NaN                           Escherichia coli O157:H7   \n",
       " 4                   NaN                     Escherichia coli (strain SE11)   \n",
       " 5                   NaN              Escherichia coli (strain K12 / DH10B)   \n",
       " 6           APECO1_2246                      Escherichia coli O1:K1 / APEC   \n",
       " 7                   NaN                 Escherichia coli O81 (strain ED1a)   \n",
       " 8                   NaN             Escherichia coli (strain UTI89 / UPEC)   \n",
       " 9                   NaN             Escherichia coli (strain 55989 / EAEC)   \n",
       " 10                  NaN                  Escherichia coli O8 (strain IAI1)   \n",
       " 11  APECO1_O1R91 O1R_91                      Escherichia coli O1:K1 / APEC   \n",
       " 12                  NaN    Escherichia coli (strain K12 / MC4100 / BW2952)   \n",
       " 13                  NaN                 Escherichia coli O9:H4 (strain HS)   \n",
       " 14                  NaN    Escherichia coli O157:H7 (strain EC4115 / EHEC)   \n",
       " 15                  NaN  Escherichia coli O139:H28 (strain E24377A / ETEC)   \n",
       " 16                  NaN          Escherichia coli (strain SMS-3-5 / SECEC)   \n",
       " 17                  NaN       Escherichia coli O45:K1 (strain S88 / ExPEC)   \n",
       " 18                  NaN      Escherichia coli O7:K1 (strain IAI39 / ExPEC)   \n",
       " 19                  NaN  Escherichia coli O127:H6 (strain E2348/69 / EPEC)   \n",
       " 20                  NaN  Escherichia coli O17:K52:H18 (strain UMN026 / ...   \n",
       " 21                  NaN    Escherichia coli O6:K15:H31 (strain 536 / UPEC)   \n",
       " 22                  NaN  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...   \n",
       " 23                  NaN                Treponema pallidum (strain Nichols)   \n",
       " 24                  NaN  Coxiella burnetii (strain RSA 493 / Nine Mile ...   \n",
       " 25                  NaN  Escherichia coli (strain ATCC 8739 / DSM 1576 ...   \n",
       " 26                  NaN  Mycobacterium bovis (strain ATCC BAA-935 / AF2...   \n",
       " 27                  NaN  Escherichia fergusonii (strain ATCC 35469 / DS...   \n",
       " 28                  NaN                              Ehrlichia chaffeensis   \n",
       " 29                  NaN        Allochromatium vinosum (Chromatium vinosum)   \n",
       " 30                  NaN                               Sodalis glossinidius   \n",
       " 31                  NaN  Parageobacillus thermoglucosidasius (Geobacill...   \n",
       " 32                  NaN  Chlamydia trachomatis serovar A (strain ATCC V...   \n",
       " 33                  NaN  Buchnera aphidicola subsp. Acyrthosiphon pisum...   \n",
       " 34                  NaN                                  Shigella flexneri   \n",
       " 35                  NaN             Wigglesworthia glossinidia brevipalpis   \n",
       " 36                  NaN            Sodalis glossinidius (strain morsitans)   \n",
       " \n",
       "     Organism (ID)                                      Protein names  ...  \\\n",
       " 0           83333  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 1            9606  60 kDa heat shock protein, mitochondrial (EC 5...  ...   \n",
       " 2           83332  Chaperonin GroEL 1 (EC 5.6.1.7) (60 kDa chaper...  ...   \n",
       " 3           83334  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 4          409438  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 5          316385  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 6          405955  Chaperonin GroEL 1 (EC 5.6.1.7) (60 kDa chaper...  ...   \n",
       " 7          585397  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 8          364106  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 9          585055  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 10         585034  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 11         405955  Chaperonin GroEL 2 (EC 5.6.1.7) (60 kDa chaper...  ...   \n",
       " 12         595496  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 13         331112  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 14         444450  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 15         331111  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 16         439855  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 17         585035  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 18         585057  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 19         574521  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 20         585056  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 21         362663  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 22         199310  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 23         243276  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 24         227377  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 25         481805  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 26         233413  Chaperonin GroEL 2 (EC 5.6.1.7) (60 kDa chaper...  ...   \n",
       " 27         585054  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 28            945  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 29           1049  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 30          63612  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 31           1426  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 32         315277  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 33         107806  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 34            623  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 35          36870  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " 36         343509  Chaperonin GroEL (EC 5.6.1.7) (60 kDa chaperon...  ...   \n",
       " \n",
       "    Glycosylation                               Initiator methionine  \\\n",
       " 0            NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1            NaN                                                NaN   \n",
       " 2            NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 3            NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 4            NaN                                                NaN   \n",
       " 5            NaN                                                NaN   \n",
       " 6            NaN                                                NaN   \n",
       " 7            NaN                                                NaN   \n",
       " 8            NaN                                                NaN   \n",
       " 9            NaN                                                NaN   \n",
       " 10           NaN                                                NaN   \n",
       " 11           NaN                                                NaN   \n",
       " 12           NaN                                                NaN   \n",
       " 13           NaN                                                NaN   \n",
       " 14           NaN                                                NaN   \n",
       " 15           NaN                                                NaN   \n",
       " 16           NaN                                                NaN   \n",
       " 17           NaN                                                NaN   \n",
       " 18           NaN                                                NaN   \n",
       " 19           NaN                                                NaN   \n",
       " 20           NaN                                                NaN   \n",
       " 21           NaN                                                NaN   \n",
       " 22           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 23           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 24           NaN                                                NaN   \n",
       " 25           NaN                                                NaN   \n",
       " 26           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 27           NaN                                                NaN   \n",
       " 28           NaN                                                NaN   \n",
       " 29           NaN                                                NaN   \n",
       " 30           NaN                                                NaN   \n",
       " 31           NaN                                                NaN   \n",
       " 32           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 33           NaN                                                NaN   \n",
       " 34           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 35           NaN                                                NaN   \n",
       " 36           NaN                                                NaN   \n",
       " \n",
       "     Lipidation                                   Modified residue Peptide  \\\n",
       " 0          NaN  MOD_RES 34; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " 1          NaN  MOD_RES 31; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " 2          NaN  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 3          NaN                                                NaN     NaN   \n",
       " 4          NaN                                                NaN     NaN   \n",
       " 5          NaN                                                NaN     NaN   \n",
       " 6          NaN                                                NaN     NaN   \n",
       " 7          NaN                                                NaN     NaN   \n",
       " 8          NaN                                                NaN     NaN   \n",
       " 9          NaN                                                NaN     NaN   \n",
       " 10         NaN                                                NaN     NaN   \n",
       " 11         NaN                                                NaN     NaN   \n",
       " 12         NaN                                                NaN     NaN   \n",
       " 13         NaN                                                NaN     NaN   \n",
       " 14         NaN                                                NaN     NaN   \n",
       " 15         NaN                                                NaN     NaN   \n",
       " 16         NaN                                                NaN     NaN   \n",
       " 17         NaN                                                NaN     NaN   \n",
       " 18         NaN                                                NaN     NaN   \n",
       " 19         NaN                                                NaN     NaN   \n",
       " 20         NaN                                                NaN     NaN   \n",
       " 21         NaN                                                NaN     NaN   \n",
       " 22         NaN                                                NaN     NaN   \n",
       " 23         NaN                                                NaN     NaN   \n",
       " 24         NaN                                                NaN     NaN   \n",
       " 25         NaN                                                NaN     NaN   \n",
       " 26         NaN                                                NaN     NaN   \n",
       " 27         NaN                                                NaN     NaN   \n",
       " 28         NaN                                                NaN     NaN   \n",
       " 29         NaN                                                NaN     NaN   \n",
       " 30         NaN                                                NaN     NaN   \n",
       " 31         NaN                                                NaN     NaN   \n",
       " 32         NaN                                                NaN     NaN   \n",
       " 33         NaN                                                NaN     NaN   \n",
       " 34         NaN                                                NaN     NaN   \n",
       " 35         NaN                                                NaN     NaN   \n",
       " 36         NaN                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0   PTM: Phosphorylated reversibly during heat sho...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2   PTM: Phosphorylated on Thr-25 and Thr-54 by Pk...         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19                                                NaN         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22                                                NaN         NaN   \n",
       " 23                                                NaN         NaN   \n",
       " 24                                                NaN         NaN   \n",
       " 25                                                NaN         NaN   \n",
       " 26                                                NaN         NaN   \n",
       " 27                                                NaN         NaN   \n",
       " 28                                                NaN         NaN   \n",
       " 29                                                NaN         NaN   \n",
       " 30                                                NaN         NaN   \n",
       " 31                                                NaN         NaN   \n",
       " 32                                                NaN         NaN   \n",
       " 33                                                NaN         NaN   \n",
       " 34                                                NaN         NaN   \n",
       " 35                                                NaN         NaN   \n",
       " 36                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  \\\n",
       " 0             NaN                                                NaN   \n",
       " 1             NaN  TRANSIT 1..26; /note=\"Mitochondrion\"; /evidenc...   \n",
       " 2             NaN                                                NaN   \n",
       " 3             NaN                                                NaN   \n",
       " 4             NaN                                                NaN   \n",
       " 5             NaN                                                NaN   \n",
       " 6             NaN                                                NaN   \n",
       " 7             NaN                                                NaN   \n",
       " 8             NaN                                                NaN   \n",
       " 9             NaN                                                NaN   \n",
       " 10            NaN                                                NaN   \n",
       " 11            NaN                                                NaN   \n",
       " 12            NaN                                                NaN   \n",
       " 13            NaN                                                NaN   \n",
       " 14            NaN                                                NaN   \n",
       " 15            NaN                                                NaN   \n",
       " 16            NaN                                                NaN   \n",
       " 17            NaN                                                NaN   \n",
       " 18            NaN                                                NaN   \n",
       " 19            NaN                                                NaN   \n",
       " 20            NaN                                                NaN   \n",
       " 21            NaN                                                NaN   \n",
       " 22            NaN                                                NaN   \n",
       " 23            NaN                                                NaN   \n",
       " 24            NaN                                                NaN   \n",
       " 25            NaN                                                NaN   \n",
       " 26            NaN                                                NaN   \n",
       " 27            NaN                                                NaN   \n",
       " 28            NaN                                                NaN   \n",
       " 29            NaN                                                NaN   \n",
       " 30            NaN                                                NaN   \n",
       " 31            NaN                                                NaN   \n",
       " 32            NaN                                                NaN   \n",
       " 33            NaN                                                NaN   \n",
       " 34            NaN                                                NaN   \n",
       " 35            NaN                                                NaN   \n",
       " 36            NaN                                                NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0   1AON;1DK7;1DKD;1FY9;1FYA;1GR5;1GRL;1GRU;1JON;1...  \n",
       " 1                      4PJ1;6HT7;6MRC;6MRD;7AZP;7L7S;  \n",
       " 2                                               3M6C;  \n",
       " 3                                                 NaN  \n",
       " 4                                                 NaN  \n",
       " 5                                                 NaN  \n",
       " 6                                                 NaN  \n",
       " 7                                                 NaN  \n",
       " 8                                          2YNJ;3E76;  \n",
       " 9                                                 NaN  \n",
       " 10                                                NaN  \n",
       " 11                                                NaN  \n",
       " 12                                                NaN  \n",
       " 13                                                NaN  \n",
       " 14                                                NaN  \n",
       " 15                                                NaN  \n",
       " 16                                                NaN  \n",
       " 17                                                NaN  \n",
       " 18                                                NaN  \n",
       " 19                                                NaN  \n",
       " 20                                                NaN  \n",
       " 21                                                NaN  \n",
       " 22                                                NaN  \n",
       " 23                                                NaN  \n",
       " 24                                                NaN  \n",
       " 25                                                NaN  \n",
       " 26                                                NaN  \n",
       " 27                                                NaN  \n",
       " 28                                                NaN  \n",
       " 29                                                NaN  \n",
       " 30                                                NaN  \n",
       " 31                                                NaN  \n",
       " 32                                                NaN  \n",
       " 33                                                NaN  \n",
       " 34                                                NaN  \n",
       " 35                                                NaN  \n",
       " 36                                                NaN  \n",
       " \n",
       " [37 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q42652  SUS_BETVU        SS1                  SS1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                    Organism  \\\n",
       " 0                         NaN               NaN  Beta vulgaris (Sugar beet)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0         161934  Sucrose synthase (EC 2.4.1.13) (Sucrose-UDP gl...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN              NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q99N42  TYPH_MOUSE  Tymp Ecgf1                 Tymp                Ecgf1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Thymidine phosphorylase (TP) (EC 2.4.2.4) (TdR...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 6; /note=\"Phosphothreonine\"; /evidence...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q9JL56  GDE1_MOUSE  Gde1 Mir16                 Gde1                Mir16   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Glycerophosphodiester phosphodiesterase 1 (Gly...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 168; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated. {ECO:0000269|PubMed:18227...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name   Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P47923   NDK2_PEA        NDPK2                NDPK2                  NaN   \n",
       " 1  P47922   NDK1_PEA  NDPK1 NDKN1                NDPK1                NDKN1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                    Organism  \\\n",
       " 0                         NaN               NaN  Pisum sativum (Garden pea)   \n",
       " 1                         NaN               NaN  Pisum sativum (Garden pea)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           3888  Nucleoside diphosphate kinase 2, chloroplastic...  ...   \n",
       " 1           3888  Nucleoside diphosphate kinase 1 (EC 2.7.4.6) (...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " 1            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..64; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 1                                                NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P08254  MMP3_HUMAN  MMP3 STMY1                 MMP3                STMY1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Stromelysin-1 (SL-1) (EC 3.4.24.17) (Matrix me...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: Directly cleaved by HTRA2 to produce acti...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 18..99; /note=\"Activation peptide\"; /id...   \n",
       " \n",
       "                                       Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..17; /evidence=\"ECO:0000305|PubMed:33...             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1B3D;1B8Y;1BIW;1BM6;1BQO;1C3I;1C8T;1CAQ;1CIZ;1...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name                Gene Names Gene Names (primary)  \\\n",
       " 0  P11103  PARP1_MOUSE  Parp1 Adprp Adprt Adprt1                Parp1   \n",
       " 1  Q54YW4  Y8045_DICDI              DDB_G0278045                  NaN   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0   Adprp Adprt Adprt1                         NaN              NaN   \n",
       " 1                  NaN                         NaN     DDB_G0278045   \n",
       " \n",
       "                                 Organism  Organism (ID)  \\\n",
       " 0                   Mus musculus (Mouse)          10090   \n",
       " 1  Dictyostelium discoideum (Slime mold)          44689   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Poly [ADP-ribose] polymerase 1 (PARP-1) (EC 2....  ...           NaN   \n",
       " 1  Probable poly [ADP-ribose] polymerase DDB_G027...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Poly-ADP-ribosylated on serine, glutamate...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P06450   ATPA_SPIOL       atpA                 atpA                  NaN   \n",
       " 1  P00825   ATPB_SPIOL       atpB                 atpB                  NaN   \n",
       " 2  P80083  ATPBM_SPIOL  ATPB ATP2                 ATPB                 ATP2   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                     Organism  \\\n",
       " 0                         NaN               NaN  Spinacia oleracea (Spinach)   \n",
       " 1                         NaN               NaN  Spinacia oleracea (Spinach)   \n",
       " 2                         NaN               NaN  Spinacia oleracea (Spinach)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           3562  ATP synthase subunit alpha, chloroplastic (EC ...  ...   \n",
       " 1           3562  ATP synthase subunit beta, chloroplastic (EC 7...  ...   \n",
       " 2           3562  ATP synthase subunit beta, mitochondrial (EC 7...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN              NaN     NaN   \n",
       " 1           NaN                  NaN         NaN              NaN     NaN   \n",
       " 2           NaN                  NaN         NaN              NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide Signal peptide Transit peptide  \\\n",
       " 0                              NaN         NaN            NaN             NaN   \n",
       " 1                              NaN         NaN            NaN             NaN   \n",
       " 2                              NaN         NaN            NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1FX0;1KMH;6FKF;6FKH;6FKI;6VM1;6VM4;6VMB;6VMD;6...  \n",
       " 1  1FX0;1KMH;6FKF;6FKH;6FKI;6VM1;6VM4;6VMB;6VMD;6...  \n",
       " 2                                                NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P30363  ASPG_BACLI       ansA                 ansA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                Organism  \\\n",
       " 0                         NaN               NaN  Bacillus licheniformis   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           1402  L-asparaginase (L-ASNase) (EC 3.5.1.1) (L-aspa...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  D3KCC4  CRNS1_CHICK  CARNS1 ATPGD1               CARNS1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0               ATPGD1                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Gallus gallus (Chicken)           9031   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Carnosine synthase 1 (EC 6.3.2.11) (ATP-grasp ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  O35049  NSMA2_RAT  Smpd3 Cca1                Smpd3                 Cca1   \n",
       " 1  Q5EZ72  ENPP7_RAT       Enpp7                Enpp7                  NaN   \n",
       " 2  Q9ET64   NSMA_RAT       Smpd2                Smpd2                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 2                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Sphingomyelin phosphodiesterase 3 (EC 3.1.4.12...  ...   \n",
       " 1          10116  Ectonucleotide pyrophosphatase/phosphodiestera...  ...   \n",
       " 2          10116  Sphingomyelin phosphodiesterase 2 (EC 3.1.4.12...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0                                                NaN                  NaN   \n",
       " 1  CARBOHYD 99; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 2                                                NaN                  NaN   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0  LIPID 53; /note=\"S-palmitoyl cysteine\"; /evide...   \n",
       " 1                                                NaN   \n",
       " 2                                                NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 178; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Palmitoylated, palmitoylation-deficient p...         NaN   \n",
       " 1  PTM: N-glycosylated; required for activity and...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0                                    NaN             NaN  NaN  \n",
       " 1  SIGNAL 1..21; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " 2                                    NaN             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P0C5I2  GMPPB_PIG      GMPPB                GMPPB                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Sus scrofa (Pig)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9823  Mannose-1-phosphate guanyltransferase beta (EC...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P39126  IDH_BACSU  icd citC BSU29130                  icd   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 citC                   BSU29130               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Bacillus subtilis (strain 168)         224308   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Isocitrate dehydrogenase [NADP] (IDH) (EC 1.1....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1HQS;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  O74641  INUA_ASPNG             inuA                 inuA   \n",
       " 1  A5ABL2  INUA_ASPNC  inuA An11g03200                 inuA   \n",
       " 2  O74642  INUB_ASPNG             inuB                 inuB   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN              NaN   \n",
       " 1                   NaN                         NaN       An11g03200   \n",
       " 2                   NaN                         NaN              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                                  Aspergillus niger           5061   \n",
       " 1  Aspergillus niger (strain CBS 513.88 / FGSC A1...         425011   \n",
       " 2                                  Aspergillus niger           5061   \n",
       " \n",
       "                                     Protein names  ...  \\\n",
       " 0  Extracellular endo-inulinase inuA (EC 3.2.1.7)  ...   \n",
       " 1  Extracellular endo-inulinase inuA (EC 3.2.1.7)  ...   \n",
       " 2  Extracellular endo-inulinase inuB (EC 3.2.1.7)  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 109; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 109; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 2  CARBOHYD 109; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " 2         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide                         Signal peptide Transit peptide  PDB  \n",
       " 0         NaN  SIGNAL 1..25; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " 1         NaN  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " 2         NaN  SIGNAL 1..25; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P10173   FUMH_PIG         FH                   FH                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Sus scrofa (Pig)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9823  Fumarate hydratase, mitochondrial (Fumarase) (...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET P10173-2:1; /note=\"Removed\"; /evidenc...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 63; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: [Isoform Cytoplasmic]: Phosphorylation at...         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  PDB  \n",
       " 0             NaN  TRANSIT 1..46; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  A6NGU5  GGT3_HUMAN        GGT3P GGT3                GGT3P   \n",
       " 1  P36269  GGT5_HUMAN       GGT5 GGTLA1                 GGT5   \n",
       " 2  Q6P531  GGT6_HUMAN              GGT6                 GGT6   \n",
       " 3  Q9UJ14  GGT7_HUMAN  GGT7 GGTL3 GGTL5                 GGT7   \n",
       " 4  P19440  GGT1_HUMAN          GGT1 GGT                 GGT1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 GGT3                         NaN               NaN   \n",
       " 1               GGTLA1                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " 3          GGTL3 GGTL5                         NaN               NaN   \n",
       " 4                  GGT                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " 3  Homo sapiens (Human)           9606   \n",
       " 4  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Putative glutathione hydrolase 3 proenzyme (EC...  ...   \n",
       " 1  Glutathione hydrolase 5 proenzyme (EC 3.4.19.1...  ...   \n",
       " 2  Glutathione hydrolase 6 (EC 3.4.19.13) (Gamma-...  ...   \n",
       " 3  Glutathione hydrolase 7 (EC 3.4.19.13) (Gamma-...  ...   \n",
       " 4  Glutathione hydrolase 1 proenzyme (EC 3.4.19.1...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 95; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 1  CARBOHYD 98; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 2  CARBOHYD 161; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 3  CARBOHYD 198; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 4  CARBOHYD 95; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN                                                NaN     NaN   \n",
       " 1         NaN                                                NaN     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " 3         NaN  MOD_RES 17; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 4         NaN                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Cleaved by autocatalysis into a large and...         NaN   \n",
       " 1  PTM: Cleaved by autocatalysis into a large and...         NaN   \n",
       " 2  PTM: Cleaved by autocatalysis into a large and...         NaN   \n",
       " 3  PTM: Cleaved by autocatalysis into a large and...         NaN   \n",
       " 4  PTM: N-glycosylated on both chains. Contains h...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                                  PDB  \n",
       " 0             NaN             NaN                                  NaN  \n",
       " 1             NaN             NaN                                  NaN  \n",
       " 2             NaN             NaN                                  NaN  \n",
       " 3             NaN             NaN                                  NaN  \n",
       " 4             NaN             NaN  4GDX;4GG2;4Z9O;4ZBK;4ZC6;4ZCG;5V4Q;  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q27793  DRTS_TRYCR         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)           Organism  \\\n",
       " 0                         NaN               NaN  Trypanosoma cruzi   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           5693  Bifunctional dihydrofolate reductase-thymidyla...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                       PDB  \n",
       " 0             NaN  2H2Q;3CL9;3CLB;3HBB;3INV;3IRM;3IRN;3IRO;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P9WK19  MAP12_MYCTU  map mapB Rv2861c MTV003.07c                  map   \n",
       " 1  P9WK21  MAP11_MYCTU            map-1 mapA Rv0734                map-1   \n",
       " 2  P9WK20  MAP11_MYCTO            map-1 mapA MT0758                map-1   \n",
       " 3  P9WK18  MAP12_MYCTO              map mapB MT2929                  map   \n",
       " 4  P0A5J3   MAP1_MYCBO      map mapB BQ2027_MB2886C                  map   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 mapB                    Rv2861c       MTV003.07c   \n",
       " 1                 mapA                     Rv0734              NaN   \n",
       " 2                 mapA                     MT0758              NaN   \n",
       " 3                 mapB                     MT2929              NaN   \n",
       " 4                 mapB             BQ2027_MB2886C              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 2  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 3  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 4  Mycobacterium bovis (strain ATCC BAA-935 / AF2...         233413   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Methionine aminopeptidase 2 (MAP 2) (MetAP 2) ...  ...           NaN   \n",
       " 1  Methionine aminopeptidase 1 (MAP 1) (MetAP 1) ...  ...           NaN   \n",
       " 2  Methionine aminopeptidase 1 (MAP 1) (MetAP 1) ...  ...           NaN   \n",
       " 3  Methionine aminopeptidase 2 (MAP 2) (MetAP 2) ...  ...           NaN   \n",
       " 4  Methionine aminopeptidase (MAP) (MetAP) (EC 3....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1Y1N;1YJ3;3IU7;3IU8;3IU9;3PKA;3PKB;3PKC;3PKD;3...  \n",
       " 1             NaN                                                NaN  \n",
       " 2             NaN                                                NaN  \n",
       " 3             NaN                                                NaN  \n",
       " 4             NaN                                                NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P20035  HGXR_PLAFG       LACZ                 LACZ                   NaN   \n",
       " 1  P07833  HGXR_PLAFK       LACZ                 LACZ                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " \n",
       "                                          Organism  Organism (ID)  \\\n",
       " 0  Plasmodium falciparum (isolate FCR-3 / Gambia)           5838   \n",
       " 1   Plasmodium falciparum (isolate K1 / Thailand)           5839   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Hypoxanthine-guanine-xanthine phosphoribosyltr...  ...            NaN   \n",
       " 1  Hypoxanthine-guanine-xanthine phosphoribosyltr...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                   PDB  \n",
       " 0             NaN  1CJB;2VFA;3OZF;3OZG;  \n",
       " 1             NaN                   NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name     Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q10984   FUT2_RAT  Fut2 Ftb Sec1                 Fut2             Ftb Sec1   \n",
       " 1  Q10980   FUT1_RAT       Fut1 Fta                 Fut1                  Fta   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Galactoside alpha-(1,2)-fucosyltransferase 2 (...  ...   \n",
       " 1          10116  Galactoside alpha-(1,2)-fucosyltransferase 1 (...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 199; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 64; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide  PDB  \n",
       " 0         NaN             NaN             NaN  NaN  \n",
       " 1         NaN             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "      Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   Q46893  ISPD_ECOLI        ispD ygbP b2747 JW2717                 ispD   \n",
       " 1   Q8X7Y4  ISPD_ECO57            ispD Z4055 ECs3601                 ispD   \n",
       " 2   B6I6D7  ISPD_ECOSE                ispD ECSE_2999                 ispD   \n",
       " 3   B1XCS3  ISPD_ECODH             ispD ECDH10B_2915                 ispD   \n",
       " 4   Q1R7U4  ISPD_ECOUT              ispD UTI89_C3118                 ispD   \n",
       " 5   B7MYQ2  ISPD_ECO81               ispD ECED1_3203                 ispD   \n",
       " 6   A1AEU1  ISPD_ECOK1  ispD Ecok1_26870 APECO1_3776                 ispD   \n",
       " 7   B7LEG5  ISPD_ECO55             ispD EC55989_3019                 ispD   \n",
       " 8   B7LXF9  ISPD_ECO8A              ispD ECIAI1_2848                 ispD   \n",
       " 9   C4ZZQ1  ISPD_ECOBW                 ispD BWG_2483                 ispD   \n",
       " 10  A8A3M6  ISPD_ECOHS               ispD EcHS_A2885                 ispD   \n",
       " 11  A7ZQJ1  ISPD_ECO24           ispD EcE24377A_3048                 ispD   \n",
       " 12  B5Z3A9  ISPD_ECO5E            ispD ECH74115_3998                 ispD   \n",
       " 13  B7NT91  ISPD_ECO7I             ispD ECIAI39_2936                 ispD   \n",
       " 14  B1LQ67  ISPD_ECOSM             ispD EcSMS35_2872                 ispD   \n",
       " 15  B7MKM1  ISPD_ECO45               ispD ECS88_3017                 ispD   \n",
       " 16  B7UHG6  ISPD_ECO27              ispD E2348C_3017                 ispD   \n",
       " 17  B7N6X7  ISPD_ECOLU               ispD ECUMN_3071                 ispD   \n",
       " 18  Q0TEB1  ISPD_ECOL5                 ispD ECP_2729                 ispD   \n",
       " 19  Q8FEJ5  ISPD_ECOL6                    ispD c3314                 ispD   \n",
       " 20  B1IUT2  ISPD_ECOLC               ispD EcolC_0965                 ispD   \n",
       " 21  B7LWK8  ISPD_ESCF3                ispD EFER_0321                 ispD   \n",
       " 22  Q7C093  ISPD_SHIFL             ispD SF2770 S2963                 ispD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                  ygbP               b2747 JW2717              NaN   \n",
       " 1                   NaN              Z4055 ECs3601              NaN   \n",
       " 2                   NaN                  ECSE_2999              NaN   \n",
       " 3                   NaN               ECDH10B_2915              NaN   \n",
       " 4                   NaN                UTI89_C3118              NaN   \n",
       " 5                   NaN                 ECED1_3203              NaN   \n",
       " 6                   NaN                Ecok1_26870      APECO1_3776   \n",
       " 7                   NaN               EC55989_3019              NaN   \n",
       " 8                   NaN                ECIAI1_2848              NaN   \n",
       " 9                   NaN                   BWG_2483              NaN   \n",
       " 10                  NaN                 EcHS_A2885              NaN   \n",
       " 11                  NaN             EcE24377A_3048              NaN   \n",
       " 12                  NaN              ECH74115_3998              NaN   \n",
       " 13                  NaN               ECIAI39_2936              NaN   \n",
       " 14                  NaN               EcSMS35_2872              NaN   \n",
       " 15                  NaN                 ECS88_3017              NaN   \n",
       " 16                  NaN                E2348C_3017              NaN   \n",
       " 17                  NaN                 ECUMN_3071              NaN   \n",
       " 18                  NaN                   ECP_2729              NaN   \n",
       " 19                  NaN                      c3314              NaN   \n",
       " 20                  NaN                 EcolC_0965              NaN   \n",
       " 21                  NaN                  EFER_0321              NaN   \n",
       " 22                  NaN               SF2770 S2963              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                            Escherichia coli O157:H7          83334   \n",
       " 2                      Escherichia coli (strain SE11)         409438   \n",
       " 3               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 4              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 5                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 6                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 7              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 8                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 9     Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 10                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 11  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 12    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 13      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 14          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 15       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 16  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 17  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 18    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 19  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 20  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 21  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 22                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 1   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 2   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 3   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 4   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 5   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 6   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 7   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 8   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 9   2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 10  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 11  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 12  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 13  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 14  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 15  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 16  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 17  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 18  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 19  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 20  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 21  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " 22  2-C-methyl-D-erythritol 4-phosphate cytidylylt...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0                         INIT_MET 1; /note=\"Removed\"         NaN   \n",
       " 1   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "     Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0                NaN     NaN                              NaN         NaN   \n",
       " 1                NaN     NaN                              NaN         NaN   \n",
       " 2                NaN     NaN                              NaN         NaN   \n",
       " 3                NaN     NaN                              NaN         NaN   \n",
       " 4                NaN     NaN                              NaN         NaN   \n",
       " 5                NaN     NaN                              NaN         NaN   \n",
       " 6                NaN     NaN                              NaN         NaN   \n",
       " 7                NaN     NaN                              NaN         NaN   \n",
       " 8                NaN     NaN                              NaN         NaN   \n",
       " 9                NaN     NaN                              NaN         NaN   \n",
       " 10               NaN     NaN                              NaN         NaN   \n",
       " 11               NaN     NaN                              NaN         NaN   \n",
       " 12               NaN     NaN                              NaN         NaN   \n",
       " 13               NaN     NaN                              NaN         NaN   \n",
       " 14               NaN     NaN                              NaN         NaN   \n",
       " 15               NaN     NaN                              NaN         NaN   \n",
       " 16               NaN     NaN                              NaN         NaN   \n",
       " 17               NaN     NaN                              NaN         NaN   \n",
       " 18               NaN     NaN                              NaN         NaN   \n",
       " 19               NaN     NaN                              NaN         NaN   \n",
       " 20               NaN     NaN                              NaN         NaN   \n",
       " 21               NaN     NaN                              NaN         NaN   \n",
       " 22               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide                                  PDB  \n",
       " 0              NaN             NaN  1H3M;1I52;1INI;1INJ;1VGT;1VGU;3N9W;  \n",
       " 1              NaN             NaN                                  NaN  \n",
       " 2              NaN             NaN                                  NaN  \n",
       " 3              NaN             NaN                                  NaN  \n",
       " 4              NaN             NaN                                  NaN  \n",
       " 5              NaN             NaN                                  NaN  \n",
       " 6              NaN             NaN                                  NaN  \n",
       " 7              NaN             NaN                                  NaN  \n",
       " 8              NaN             NaN                                  NaN  \n",
       " 9              NaN             NaN                                  NaN  \n",
       " 10             NaN             NaN                                  NaN  \n",
       " 11             NaN             NaN                                  NaN  \n",
       " 12             NaN             NaN                                  NaN  \n",
       " 13             NaN             NaN                                  NaN  \n",
       " 14             NaN             NaN                                  NaN  \n",
       " 15             NaN             NaN                                  NaN  \n",
       " 16             NaN             NaN                                  NaN  \n",
       " 17             NaN             NaN                                  NaN  \n",
       " 18             NaN             NaN                                  NaN  \n",
       " 19             NaN             NaN                                  NaN  \n",
       " 20             NaN             NaN                                  NaN  \n",
       " 21             NaN             NaN                                  NaN  \n",
       " 22             NaN             NaN                                  NaN  \n",
       " \n",
       " [23 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q2PDF6  RTPR_EUGGR        rnr                  rnr                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Euglena gracilis   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           3039  Adenosylcobalamin-dependent ribonucleoside-tri...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P48596  GCH1_DROME  Pu CG9441                   Pu                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                         NaN           CG9441   \n",
       " \n",
       "                               Organism  Organism (ID)  \\\n",
       " 0  Drosophila melanogaster (Fruit fly)           7227   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  GTP cyclohydrolase 1 (EC 3.5.4.16) (GTP cycloh...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  P48570  HOSC_YEAST  LYS20 YDL182W D1298                LYS20   \n",
       " 1  Q12122  HOSM_YEAST  LYS21 YDL131W D2195                LYS21   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YDL182W            D1298   \n",
       " 1                   NaN                    YDL131W            D2195   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Homocitrate synthase, cytosolic isozyme (HCS) ...  ...           NaN   \n",
       " 1  Homocitrate synthase, mitochondrial (HCS) (EC ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 385; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 399; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0                                                NaN  NaN  \n",
       " 1  TRANSIT 1..?; /note=\"Mitochondrion\"; /evidence...  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  E9PU28  IMDH2_RAT     Impdh2               Impdh2                   NaN   \n",
       " 1  D3ZLZ7  IMDH1_RAT     Impdh1               Impdh1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " 1                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Inosine-5'-monophosphate dehydrogenase 2 (IMP ...  ...   \n",
       " 1          10116  Inosine-5'-monophosphate dehydrogenase 1 (IMP ...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 122; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 160; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Acetylated by CLOCK in a circadian manner...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  O57579  AMPN_CHICK  ANPEP APDE                ANPEP                 APDE   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Gallus gallus (Chicken)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9031  Aminopeptidase Ey (EC 3.4.11.20) (Aminopeptida...  ...   \n",
       " \n",
       "                                        Glycosylation  \\\n",
       " 0  CARBOHYD 38; /note=\"N-linked (GlcNAc...) aspar...   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q95JH2  PLCA_BOVIN     AGPAT1               AGPAT1                   NaN   \n",
       " 1  Q5E9R2  PLCD_BOVIN     AGPAT4               AGPAT4                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " 1                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  1-acyl-sn-glycerol-3-phosphate acyltransferase...  ...   \n",
       " 1           9913  1-acyl-sn-glycerol-3-phosphate acyltransferase...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " 1           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " 1                              NaN         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..26; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " 1                                    NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P0AE22  APHA_ECOLI  aphA napA yjbP b4055 JW4015                 aphA   \n",
       " 1  D2TRT2  APHA_CITRI               aphA ROD_36361                 aphA   \n",
       " 2  P0AE23  APHA_SHIFL            aphA SF4149 S3580                 aphA   \n",
       " 3  O08430  APHA_SALTI      aphA napA STY4445 t4155                 aphA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0            napA yjbP               b4055 JW4015               NaN   \n",
       " 1                  NaN                  ROD_36361               NaN   \n",
       " 2                  NaN               SF4149 S3580               NaN   \n",
       " 3                 napA              STY4445 t4155               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1  Citrobacter rodentium (strain ICC168) (Citroba...         637910   \n",
       " 2                                  Shigella flexneri            623   \n",
       " 3                                   Salmonella typhi          90370   \n",
       " \n",
       "                                   Protein names  ... Glycosylation  \\\n",
       " 0  Class B acid phosphatase (CBAP) (EC 3.1.3.2)  ...           NaN   \n",
       " 1  Class B acid phosphatase (CBAP) (EC 3.1.3.2)  ...           NaN   \n",
       " 2  Class B acid phosphatase (CBAP) (EC 3.1.3.2)  ...           NaN   \n",
       " 3  Class B acid phosphatase (CBAP) (EC 3.1.3.2)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " 1                              NaN         NaN   \n",
       " 2                              NaN         NaN   \n",
       " 3                              NaN         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  \\\n",
       " 0  SIGNAL 1..25; /note=\"Or 23\"; /evidence=\"ECO:00...             NaN   \n",
       " 1  SIGNAL 1..25; /evidence=\"ECO:0000255, ECO:0000...             NaN   \n",
       " 2  SIGNAL 1..25; /note=\"Or 23\"; /evidence=\"ECO:00...             NaN   \n",
       " 3              SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1N8N;1N9K;1RMQ;1RMT;1RMY;2B82;2B8J;2G1A;2HEG;2...  \n",
       " 1                                                NaN  \n",
       " 2                                                NaN  \n",
       " 3                                                NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  Q72L32  HEMH_THET2  hemH TT_C0231                 hemH   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   TT_C0231               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Thermus thermophilus (strain ATCC BAA-163 / DS...         262724   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Ferrochelatase (EC 4.98.1.1) (Heme synthase) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  P39812  GLTA_BACSU  gltA BSU18450                 gltA   \n",
       " 1  O34399  GLTB_BACSU  gltB BSU18440                 gltB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   BSU18450               NaN   \n",
       " 1                   NaN                   BSU18440               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Bacillus subtilis (strain 168)         224308   \n",
       " 1  Bacillus subtilis (strain 168)         224308   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glutamate synthase [NADPH] large chain (EC 1.4...  ...           NaN   \n",
       " 1  Glutamate synthase [NADPH] small chain (EC 1.4...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0   P65774  NADK_SALTY        nadK STM2683                 nadK   \n",
       " 1   P65775  NADK_SALTI  nadK STY2869 t2637                 nadK   \n",
       " 2   B5F295  NADK_SALA4     nadK SeAg_B2827                 nadK   \n",
       " 3   B4TS62  NADK_SALSV     nadK SeSA_A2878                 nadK   \n",
       " 4   B4TE58  NADK_SALHS     nadK SeHA_C2898                 nadK   \n",
       " 5   Q57L24  NADK_SALCH       nadK SCH_2682                 nadK   \n",
       " 6   B4T2C0  NADK_SALNS  nadK SNSL254_A2896                 nadK   \n",
       " 7   B5QUH0  NADK_SALEP        nadK SEN2603                 nadK   \n",
       " 8   B5FS14  NADK_SALDC      nadK SeD_A3009                 nadK   \n",
       " 9   B5BEA1  NADK_SALPK       nadK SSPA2367                 nadK   \n",
       " 10  B5RD91  NADK_SALG2         nadK SG2660                 nadK   \n",
       " 11  Q5PFG7  NADK_SALPA        nadK SPA2542                 nadK   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                    NaN                    STM2683               NaN   \n",
       " 1                    NaN              STY2869 t2637               NaN   \n",
       " 2                    NaN                 SeAg_B2827               NaN   \n",
       " 3                    NaN                 SeSA_A2878               NaN   \n",
       " 4                    NaN                 SeHA_C2898               NaN   \n",
       " 5                    NaN                   SCH_2682               NaN   \n",
       " 6                    NaN              SNSL254_A2896               NaN   \n",
       " 7                    NaN                    SEN2603               NaN   \n",
       " 8                    NaN                  SeD_A3009               NaN   \n",
       " 9                    NaN                   SSPA2367               NaN   \n",
       " 10                   NaN                     SG2660               NaN   \n",
       " 11                   NaN                    SPA2542               NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0   Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 1                                    Salmonella typhi          90370   \n",
       " 2                     Salmonella agona (strain SL483)         454166   \n",
       " 3         Salmonella schwarzengrund (strain CVM19633)         439843   \n",
       " 4                Salmonella heidelberg (strain SL476)         454169   \n",
       " 5             Salmonella choleraesuis (strain SC-B67)         321314   \n",
       " 6                   Salmonella newport (strain SL254)         423368   \n",
       " 7         Salmonella enteritidis PT4 (strain P125109)         550537   \n",
       " 8              Salmonella dublin (strain CT_02021853)         439851   \n",
       " 9           Salmonella paratyphi A (strain AKU_12601)         554290   \n",
       " 10  Salmonella gallinarum (strain 287/91 / NCTC 13...         550538   \n",
       " 11  Salmonella paratyphi A (strain ATCC 9150 / SAR...         295319   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 1   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 2   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 3   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 4   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 5   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 6   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 7   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 8   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 9   NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 10  NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " 11  NAD kinase (EC 2.7.1.23) (ATP-dependent NAD ki...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                   NaN         NaN               NaN     NaN   \n",
       " 1                   NaN         NaN               NaN     NaN   \n",
       " 2                   NaN         NaN               NaN     NaN   \n",
       " 3                   NaN         NaN               NaN     NaN   \n",
       " 4                   NaN         NaN               NaN     NaN   \n",
       " 5                   NaN         NaN               NaN     NaN   \n",
       " 6                   NaN         NaN               NaN     NaN   \n",
       " 7                   NaN         NaN               NaN     NaN   \n",
       " 8                   NaN         NaN               NaN     NaN   \n",
       " 9                   NaN         NaN               NaN     NaN   \n",
       " 10                  NaN         NaN               NaN     NaN   \n",
       " 11                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide    PDB  \n",
       " 0              NaN  2AN1;  \n",
       " 1              NaN    NaN  \n",
       " 2              NaN    NaN  \n",
       " 3              NaN    NaN  \n",
       " 4              NaN    NaN  \n",
       " 5              NaN    NaN  \n",
       " 6              NaN    NaN  \n",
       " 7              NaN    NaN  \n",
       " 8              NaN    NaN  \n",
       " 9              NaN    NaN  \n",
       " 10             NaN    NaN  \n",
       " 11             NaN    NaN  \n",
       " \n",
       " [12 rows x 110 columns],\n",
       "     Entry Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  P83868   TEBP_RAT       Ptges3 Tebp               Ptges3   \n",
       " 1  Q9JHF3  PTGES_RAT  Ptges Pges Pig12                Ptges   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 Tebp                         NaN               NaN   \n",
       " 1           Pges Pig12                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " 1  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Prostaglandin E synthase 3 (EC 5.3.99.3) (Cyto...  ...           NaN   \n",
       " 1  Prostaglandin E synthase (mPGES-1) (EC 5.3.99....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 33; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Proteolytically cleaved by caspase-7 (CAS...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q09324  GCNT1_MOUSE      Gcnt1                Gcnt1                   NaN   \n",
       " 1  Q5JCT0  GCNT3_MOUSE      Gcnt3                Gcnt3                   NaN   \n",
       " 2  E9Q649  GCNT4_MOUSE      Gcnt4                Gcnt4                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 1                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 2                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Beta-1,3-galactosyl-O-glycosyl-glycoprotein be...  ...   \n",
       " 1          10090  Beta-1,3-galactosyl-O-glycosyl-glycoprotein be...  ...   \n",
       " 2          10090  Beta-1,3-galactosyl-O-glycosyl-glycoprotein be...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 58; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 1  CARBOHYD 288; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 2  CARBOHYD 73; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " 1         NaN               NaN     NaN   \n",
       " 2         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated. {ECO:0000269|PubMed:12954...         NaN   \n",
       " 1                PTM: N-glycosylated. {ECO:0000250}.         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide              PDB  \n",
       " 0             NaN             NaN  2GAK;2GAM;3OTK;  \n",
       " 1             NaN             NaN              NaN  \n",
       " 2             NaN             NaN              NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                  Gene Names Gene Names (primary)  \\\n",
       " 0  A0QX86  IMPA_MYCS2  impA MSMEG_3210 MSMEI_3128                 impA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN      MSMEG_3210 MSMEI_3128               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycolicibacterium smegmatis (strain ATCC 70008...         246196   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Inositol-1-monophosphatase ImpA (I-1-Pase) (IM...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9ALJ4  AGAA_GEOSE       agaA                 agaA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Geobacillus stearothermophilus (Bacillus stear...           1422   \n",
       " \n",
       "                             Protein names  ...  Glycosylation  \\\n",
       " 0  Alpha-galactosidase AgaA (EC 3.2.1.22)  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                        PDB  \n",
       " 0             NaN  4FNP;4FNR;4FNS;4FNT;4FNU;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P26361  CFTR_MOUSE  Cftr Abcc7                 Cftr                Abcc7   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Cystic fibrosis transmembrane conductance regu...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 889; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0  LIPID 524; /note=\"S-palmitoyl cysteine\"; /evid...   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 549; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated. {ECO:0000250|UniProtKB:P1...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " \n",
       "                                              PDB  \n",
       " 0  1Q3H;1R0W;1R0X;1R0Y;1R0Z;1R10;1XF9;1XFA;3SI7;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q9I676  HYDA_PSEAE  dht PA0441                  dht                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     PA0441               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  D-hydantoinase/dihydropyrimidinase (DHPase) (E...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 150; /note=\"N6-carboxylysine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Carboxylation allows a single lysine to c...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                        PDB  \n",
       " 0             NaN             NaN  5E5C;5YKD;6AJD;6KLK;7E3U;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P00860  DCOR_MOUSE   Odc1 Odc                 Odc1                  Odc   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                Protein names  ...  \\\n",
       " 0          10090  Ornithine decarboxylase (ODC) (EC 4.1.1.17)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 69; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  7ODC;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  P0AEP3  GALU_ECOLI   galU ychD b1236 JW1224                 galU   \n",
       " 1  Q16851  UGPA_HUMAN                UGP2 UGP1                 UGP2   \n",
       " 2  P0AAB6  GALF_ECOLI   galF wcaN b2042 JW2027                 galF   \n",
       " 3  P19595  UGPA_SOLTU                      NaN                  NaN   \n",
       " 4  P0AAB7  GALF_ECO57  galF wcaN Z3205 ECs2846                 galF   \n",
       " 5  P0AEP5  GALU_ECO57       galU Z2012 ECs1738                 galU   \n",
       " 6  P0AEP4  GALU_ECOL6               galU c1700                 galU   \n",
       " 7  P0AEP6  GALU_SHIFL        galU SF1236 S1322                 galU   \n",
       " 8  P37776  GALF_SHIFL        galF SF2105 S2228                 galF   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 ychD               b1236 JW1224               NaN   \n",
       " 1                 UGP1                        NaN               NaN   \n",
       " 2                 wcaN               b2042 JW2027               NaN   \n",
       " 3                  NaN                        NaN               NaN   \n",
       " 4                 wcaN              Z3205 ECs2846               NaN   \n",
       " 5                  NaN              Z2012 ECs1738               NaN   \n",
       " 6                  NaN                      c1700               NaN   \n",
       " 7                  NaN               SF1236 S1322               NaN   \n",
       " 8                  NaN               SF2105 S2228               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                               Homo sapiens (Human)           9606   \n",
       " 2                      Escherichia coli (strain K12)          83333   \n",
       " 3                         Solanum tuberosum (Potato)           4113   \n",
       " 4                           Escherichia coli O157:H7          83334   \n",
       " 5                           Escherichia coli O157:H7          83334   \n",
       " 6  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 7                                  Shigella flexneri            623   \n",
       " 8                                  Shigella flexneri            623   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  UTP--glucose-1-phosphate uridylyltransferase (...  ...           NaN   \n",
       " 1  UTP--glucose-1-phosphate uridylyltransferase (...  ...           NaN   \n",
       " 2  UTP--glucose-1-phosphate uridylyltransferase (...  ...           NaN   \n",
       " 3  UTP--glucose-1-phosphate uridylyltransferase (...  ...           NaN   \n",
       " 4  UTP--glucose-1-phosphate uridylyltransferase (...  ...           NaN   \n",
       " 5  UTP--glucose-1-phosphate uridylyltransferase (...  ...           NaN   \n",
       " 6  UTP--glucose-1-phosphate uridylyltransferase (...  ...           NaN   \n",
       " 7  UTP--glucose-1-phosphate uridylyltransferase (...  ...           NaN   \n",
       " 8  UTP--glucose-1-phosphate uridylyltransferase (...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET Q16851-2:1; /note=\"Removed\"; /evidenc...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 6  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 7  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 8                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES 13; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " 3  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 4                                                NaN     NaN   \n",
       " 5                                                NaN     NaN   \n",
       " 6                                                NaN     NaN   \n",
       " 7                                                NaN     NaN   \n",
       " 8                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " 7                              NaN         NaN             NaN   \n",
       " 8                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN            2E3D;  \n",
       " 1             NaN  3R2W;3R3I;4R7P;  \n",
       " 2             NaN              NaN  \n",
       " 3             NaN              NaN  \n",
       " 4             NaN              NaN  \n",
       " 5             NaN              NaN  \n",
       " 6             NaN              NaN  \n",
       " 7             NaN              NaN  \n",
       " 8             NaN              NaN  \n",
       " \n",
       " [9 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P47820    ACE_RAT   Ace Dcp1                  Ace                 Dcp1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Angiotensin-converting enzyme (ACE) (EC 3.4.15...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 44; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 1306; /note=\"Phosphoserine\"; /evidence...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: [Angiotensin-converting enzyme, soluble f...         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..35; /evidence=\"ECO:0000250|UniProtKB...             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P35028  KADA_SULAC  adkA adk Saci_0573                 adkA   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  adk                  Saci_0573               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Sulfolobus acidocaldarius (strain ATCC 33909 /...         330779   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Adenylate kinase (AK) (EC 2.7.4.3) (ATP-AMP tr...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1NKS;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  O70536  SOAT1_RAT  Soat1 Acact Acat                Soat1   \n",
       " 1  Q7TQM4  SOAT2_RAT       Soat2 Acat2                Soat2   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           Acact Acat                         NaN               NaN   \n",
       " 1                Acat2                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " 1  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Sterol O-acyltransferase 1 (EC 2.3.1.26) (Acyl...  ...           NaN   \n",
       " 1  Sterol O-acyltransferase 2 (EC 2.3.1.26) (Acyl...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 1  MOD_RES 279; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: Polyubiquitinated by AMFR/gp78 at Cys-279...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "      Entry   Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0   O14578   CTRO_HUMAN  CIT CRIK KIAA0949 STK21                  CIT   \n",
       " 1   O14920   IKKB_HUMAN               IKBKB IKKB                IKBKB   \n",
       " 2   O14936   CSKP_HUMAN                CASK LIN2                 CASK   \n",
       " 3   O15021  MAST4_HUMAN           MAST4 KIAA0303                MAST4   \n",
       " 4   O15530  PDPK1_HUMAN               PDPK1 PDK1                PDPK1   \n",
       " ..     ...          ...                      ...                  ...   \n",
       " 95  P78368  KC1G2_HUMAN            CSNK1G2 CK1G2              CSNK1G2   \n",
       " 96  P78527  PRKDC_HUMAN         PRKDC HYRC HYRC1                PRKDC   \n",
       " 97  Q09013   DMPK_HUMAN          DMPK DM1PK MDPK                 DMPK   \n",
       " 98  Q13131  AAPK1_HUMAN             PRKAA1 AMPK1               PRKAA1   \n",
       " 99  Q15349  KS6A2_HUMAN   RPS6KA2 MAPKAPK1C RSK3              RPS6KA2   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0   CRIK KIAA0949 STK21                         NaN              NaN   \n",
       " 1                  IKKB                         NaN              NaN   \n",
       " 2                  LIN2                         NaN              NaN   \n",
       " 3              KIAA0303                         NaN              NaN   \n",
       " 4                  PDK1                         NaN              NaN   \n",
       " ..                  ...                         ...              ...   \n",
       " 95                CK1G2                         NaN              NaN   \n",
       " 96           HYRC HYRC1                         NaN              NaN   \n",
       " 97           DM1PK MDPK                         NaN              NaN   \n",
       " 98                AMPK1                         NaN              NaN   \n",
       " 99       MAPKAPK1C RSK3                         NaN              NaN   \n",
       " \n",
       "                 Organism  Organism (ID)  \\\n",
       " 0   Homo sapiens (Human)           9606   \n",
       " 1   Homo sapiens (Human)           9606   \n",
       " 2   Homo sapiens (Human)           9606   \n",
       " 3   Homo sapiens (Human)           9606   \n",
       " 4   Homo sapiens (Human)           9606   \n",
       " ..                   ...            ...   \n",
       " 95  Homo sapiens (Human)           9606   \n",
       " 96  Homo sapiens (Human)           9606   \n",
       " 97  Homo sapiens (Human)           9606   \n",
       " 98  Homo sapiens (Human)           9606   \n",
       " 99  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Citron Rho-interacting kinase (CRIK) (EC 2.7.1...  ...           NaN   \n",
       " 1   Inhibitor of nuclear factor kappa-B kinase sub...  ...           NaN   \n",
       " 2   Peripheral plasma membrane protein CASK (hCASK...  ...           NaN   \n",
       " 3   Microtubule-associated serine/threonine-protei...  ...           NaN   \n",
       " 4   3-phosphoinositide-dependent protein kinase 1 ...  ...           NaN   \n",
       " ..                                                ...  ...           ...   \n",
       " 95  Casein kinase I isoform gamma-2 (CKI-gamma 2) ...  ...           NaN   \n",
       " 96  DNA-dependent protein kinase catalytic subunit...  ...           NaN   \n",
       " 97  Myotonin-protein kinase (MT-PK) (EC 2.7.11.1) ...  ...           NaN   \n",
       " 98  5'-AMP-activated protein kinase catalytic subu...  ...           NaN   \n",
       " 99  Ribosomal protein S6 kinase alpha-2 (S6K-alpha...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  \\\n",
       " 0                   NaN         NaN   \n",
       " 1                   NaN         NaN   \n",
       " 2                   NaN         NaN   \n",
       " 3                   NaN         NaN   \n",
       " 4                   NaN         NaN   \n",
       " ..                  ...         ...   \n",
       " 95                  NaN         NaN   \n",
       " 96                  NaN         NaN   \n",
       " 97                  NaN         NaN   \n",
       " 98                  NaN         NaN   \n",
       " 99                  NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 1   MOD_RES 177; /note=\"Phosphoserine; by TBK1 and...     NaN   \n",
       " 2   MOD_RES 51; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 3   MOD_RES 206; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 4   MOD_RES 9; /note=\"Phosphotyrosine; by SRC and ...     NaN   \n",
       " ..                                                ...     ...   \n",
       " 95                                                NaN     NaN   \n",
       " 96  MOD_RES 117; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " 97  MOD_RES 216; /note=\"Phosphoserine; by autocata...     NaN   \n",
       " 98  MOD_RES 32; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 99  MOD_RES 218; /note=\"Phosphoserine; by PDPK1\"; ...     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1   PTM: Upon cytokine stimulation, phosphorylated...         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4   PTM: Phosphorylation on Ser-241 in the activat...         NaN   \n",
       " ..                                                ...         ...   \n",
       " 95  PTM: Autophosphorylated (PubMed:18794808). Pho...         NaN   \n",
       " 96  PTM: Autophosphorylated at two clusters, the T...         NaN   \n",
       " 97  PTM: Phosphorylated. Autophosphorylates. Phosp...         NaN   \n",
       " 98  PTM: Ubiquitinated. {ECO:0000250|UniProtKB:Q5E...         NaN   \n",
       " 99  PTM: Activated by phosphorylation at Ser-218 b...         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  \\\n",
       " 0              NaN             NaN   \n",
       " 1              NaN             NaN   \n",
       " 2              NaN             NaN   \n",
       " 3              NaN             NaN   \n",
       " 4              NaN             NaN   \n",
       " ..             ...             ...   \n",
       " 95             NaN             NaN   \n",
       " 96             NaN             NaN   \n",
       " 97             NaN             NaN   \n",
       " 98             NaN             NaN   \n",
       " 99             NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0                                                 NaN  \n",
       " 1                                3BRT;3BRV;4E3C;4KIK;  \n",
       " 2   1KGD;1KWA;1ZL8;3C0G;3C0H;3C0I;3MFR;3MFS;3MFT;3...  \n",
       " 3                                               2W7R;  \n",
       " 4   1H1W;1OKY;1OKZ;1UU3;1UU7;1UU8;1UU9;1UVR;1W1D;1...  \n",
       " ..                                                ...  \n",
       " 95                                              2C47;  \n",
       " 96  5LUQ;5W1R;5Y3R;6ZFP;6ZH2;6ZH4;6ZH6;6ZH8;6ZHA;6...  \n",
       " 97                                         1WT6;2VD5;  \n",
       " 98  4RED;4RER;4REW;5EZV;6C9F;6C9G;6C9H;6C9J;7JHG;7...  \n",
       " 99                                                NaN  \n",
       " \n",
       " [100 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  B7V5R3  BETA_PSEA8  betA PLES_57671                 betA   \n",
       " 1  A6VEI3  BETA_PSEA7  betA PSPA7_6157                 betA   \n",
       " 2  Q02DZ0  BETA_PSEAB  betA PA14_70940                 betA   \n",
       " 3  Q9HTJ2  BETA_PSEAE      betA PA5372                 betA   \n",
       " 4  A4XPI5  BETA_PSEMY   betA Pmen_0481                 betA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                 PLES_57671               NaN   \n",
       " 1                   NaN                 PSPA7_6157               NaN   \n",
       " 2                   NaN                 PA14_70940               NaN   \n",
       " 3                   NaN                     PA5372               NaN   \n",
       " 4                   NaN                  Pmen_0481               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0             Pseudomonas aeruginosa (strain LESB58)         557722   \n",
       " 1                Pseudomonas aeruginosa (strain PA7)         381754   \n",
       " 2         Pseudomonas aeruginosa (strain UCBPP-PA14)         208963   \n",
       " 3  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 4                 Pseudomonas mendocina (strain ymp)         399739   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " 1  Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " 2  Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " 3  Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " 4  Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " 3             NaN  NaN  \n",
       " 4             NaN  NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  C9WPN6   IF2H_RAT               Eif2s3y              Eif2s3y   \n",
       " 1  B2GUV7   IF2P_RAT             Eif5b If2                Eif5b   \n",
       " 2  P81795   IF2G_RAT  Eif2s3 Eif2g Eif2s3x               Eif2s3   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1                  If2                         NaN               NaN   \n",
       " 2        Eif2g Eif2s3x                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " 1  Rattus norvegicus (Rat)          10116   \n",
       " 2  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Eukaryotic translation initiation factor 2 sub...  ...           NaN   \n",
       " 1  Eukaryotic translation initiation factor 5B (e...  ...           NaN   \n",
       " 2  Eukaryotic translation initiation factor 2 sub...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 66; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 2  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P05153  PCKGC_CHICK       PCK1                 PCK1                   NaN   \n",
       " 1  P21642  PCKGM_CHICK       PCK2                 PCK2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Gallus gallus (Chicken)   \n",
       " 1                         NaN               NaN  Gallus gallus (Chicken)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9031  Phosphoenolpyruvate carboxykinase, cytosolic [...  ...   \n",
       " 1           9031  Phosphoenolpyruvate carboxykinase [GTP], mitoc...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 90; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylation at Ser-90 reduces the bin...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide                       Transit peptide              PDB  \n",
       " 0             NaN                                   NaN              NaN  \n",
       " 1             NaN  TRANSIT 1..33; /note=\"Mitochondrion\"  2FAF;2FAH;2QZY;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name   Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q80Z29  NAMPT_RAT  Nampt Pbef1                Nampt                Pbef1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Nicotinamide phosphoribosyltransferase (NAmPRT...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                   PDB  \n",
       " 0             NaN  2G95;2G96;2G97;3G8E;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  A4IFH5  ALAT1_BOVIN   GPT GPT1                  GPT                 GPT1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Alanine aminotransferase 1 (ALT1) (EC 2.6.1.2)...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name                                        Gene Names  \\\n",
       " 0  P04807   HXKB_YEAST                      HXK2 HEX1 HKB YGL253W NRB486   \n",
       " 1  P04806   HXKA_YEAST                                  HXK1 HKA YFR053C   \n",
       " 2  Q06204  YL446_YEAST                                           YLR446W   \n",
       " 3  Q59RW5   HXK1_CANAL  HXK1 NAG5 CAALFM_C604580WA CaO19.2154 CaO19.9701   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                 HXK2             HEX1 HKB                    YGL253W   \n",
       " 1                 HXK1                  HKA                    YFR053C   \n",
       " 2                  NaN                  NaN                    YLR446W   \n",
       " 3                 HXK1                 NAG5           CAALFM_C604580WA   \n",
       " \n",
       "         Gene Names (ORF)                                           Organism  \\\n",
       " 0                 NRB486  Saccharomyces cerevisiae (strain ATCC 204508 /...   \n",
       " 1                    NaN  Saccharomyces cerevisiae (strain ATCC 204508 /...   \n",
       " 2                    NaN  Saccharomyces cerevisiae (strain ATCC 204508 /...   \n",
       " 3  CaO19.2154 CaO19.9701  Candida albicans (strain SC5314 / ATCC MYA-287...   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0         559292  Hexokinase-2 (EC 2.7.1.1) (Hexokinase PII) (He...  ...   \n",
       " 1         559292  Hexokinase-1 (EC 2.7.1.1) (Hexokinase PI) (Hex...  ...   \n",
       " 2         559292           Putative hexokinase YLR446W (EC 2.7.1.1)  ...   \n",
       " 3         237561  N-acetylglucosamine kinase 1 (GlcNAc kinase 1)...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN                                                NaN   \n",
       " 2           NaN                                                NaN   \n",
       " 3           NaN                                                NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 15; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1         NaN  MOD_RES 245; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 2         NaN                                                NaN     NaN   \n",
       " 3         NaN                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  1IG8;2YHX;5UWT;  \n",
       " 1             NaN       1HKG;3B8A;  \n",
       " 2             NaN              NaN  \n",
       " 3             NaN              NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P40817  DPAL_SALTY       dpaL STM1002                 dpaL   \n",
       " 1  P66899  DPAL_ECOLI  ygeX b2871 JW2839                 ygeX   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                    STM1002               NaN   \n",
       " 1                   NaN               b2871 JW2839               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 1                      Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Diaminopropionate ammonia-lyase (DAPAL) (EC 4....  ...           NaN   \n",
       " 1  Diaminopropionate ammonia-lyase (DAPAL) (EC 4....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 78; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " 1  MOD_RES 77; /note=\"N6-(pyridoxal phosphate)lys...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                        PDB  \n",
       " 0             NaN                      5YGR;  \n",
       " 1             NaN  4D9G;4D9I;4D9K;4D9M;4D9N;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  O31616  GLYOX_BACSU  thiO goxB yjbR BSU11670                 thiO   \n",
       " 1  S5FMM4  GLYOX_BACLI                     thiO                 thiO   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0            goxB yjbR                   BSU11670               NaN   \n",
       " 1                  NaN                        NaN               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Bacillus subtilis (strain 168)         224308   \n",
       " 1          Bacillus licheniformis           1402   \n",
       " \n",
       "                                 Protein names  ... Glycosylation  \\\n",
       " 0          Glycine oxidase (GO) (EC 1.4.3.19)  ...           NaN   \n",
       " 1  Glycine oxidase (GO) (EC 1.4.3.19) (BliGO)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                   PDB  \n",
       " 0             NaN  1NG3;1NG4;1RYI;3IF9;  \n",
       " 1             NaN                   NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  P42337  PK3CA_MOUSE          Pik3ca               Pik3ca   \n",
       " 1  Q7TT16   IPMK_MOUSE  Ipmk Impk Ipk2                 Ipmk   \n",
       " 2  O35904  PK3CD_MOUSE          Pik3cd               Pik3cd   \n",
       " 3  Q61194  P3C2A_MOUSE     Pik3c2a Cpk              Pik3c2a   \n",
       " 4  Q8BTI9  PK3CB_MOUSE          Pik3cb               Pik3cb   \n",
       " 5  Q9JHG7  PK3CG_MOUSE   Pik3cg Pi3kg1               Pik3cg   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1            Impk Ipk2                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " 3                  Cpk                         NaN               NaN   \n",
       " 4                  NaN                         NaN               NaN   \n",
       " 5               Pi3kg1                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " 2  Mus musculus (Mouse)          10090   \n",
       " 3  Mus musculus (Mouse)          10090   \n",
       " 4  Mus musculus (Mouse)          10090   \n",
       " 5  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...           NaN   \n",
       " 1  Inositol polyphosphate multikinase (EC 2.7.1.1...  ...           NaN   \n",
       " 2  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...           NaN   \n",
       " 3  Phosphatidylinositol 4-phosphate 3-kinase C2 d...  ...           NaN   \n",
       " 4  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...           NaN   \n",
       " 5  Phosphatidylinositol 4,5-bisphosphate 3-kinase...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 2  MOD_RES 523; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 3  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 4  MOD_RES 318; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 5  MOD_RES 1024; /note=\"Phosphothreonine; by PKA\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2  PTM: Autophosphorylation on Ser-1038 results i...         NaN   \n",
       " 3  PTM: Phosphorylated on Ser-261 during mitosis ...         NaN   \n",
       " 4  PTM: Phosphorylation at Ser-1064 down-regulate...         NaN   \n",
       " 5  PTM: Phosphorylated at Thr-1024 by PKA. Phosph...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " 2             NaN             NaN   \n",
       " 3             NaN             NaN   \n",
       " 4             NaN             NaN   \n",
       " 5             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0                                              4A55;  \n",
       " 1                                                NaN  \n",
       " 2  2WXF;2WXG;2WXH;2WXI;2WXJ;2WXK;2WXL;2WXM;2WXN;2...  \n",
       " 3           2B3R;7BI2;7BI4;7BI6;7BI9;7Z74;7Z75;8A9I;  \n",
       " 4                                         2Y3A;4BFR;  \n",
       " 5                                                NaN  \n",
       " \n",
       " [6 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O88618   FTCD_RAT       Ftcd                 Ftcd                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Formimidoyltransferase-cyclodeaminase (58 kDa ...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 520; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1TT9;2PFD;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  P29401    TKT_HUMAN             TKT                  TKT   \n",
       " 1  P51854  TKTL1_HUMAN  TKTL1 TKR TKT2                TKTL1   \n",
       " 2  Q9H0I9  TKTL2_HUMAN           TKTL2                TKTL2   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  NaN                         NaN               NaN   \n",
       " 1             TKR TKT2                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0                    Transketolase (TK) (EC 2.2.1.1)  ...           NaN   \n",
       " 1  Transketolase-like protein 1 (EC 2.2.1.1) (Tra...  ...           NaN   \n",
       " 2          Transketolase-like protein 2 (EC 2.2.1.1)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  3MOS;3OOY;4KXU;4KXV;4KXW;4KXX;4KXY;6HA3;6HAD;6...  \n",
       " 1             NaN                                                NaN  \n",
       " 2             NaN                                                NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  P56891  CYSH_RHIME  cysH R00944 SMc00092                 cysH   \n",
       " 1  I3X057  CYSH_SINF2   cysH USDA257_c06700                 cysH   \n",
       " 2  C3MIE1  CYSH_SINFN       cysH NGR_c06960                 cysH   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                     R00944         SMc00092   \n",
       " 1                   NaN                        NaN   USDA257_c06700   \n",
       " 2                   NaN                 NGR_c06960              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Rhizobium meliloti (strain 1021) (Ensifer meli...         266834   \n",
       " 1             Sinorhizobium fredii (strain USDA 257)        1185652   \n",
       " 2  Sinorhizobium fredii (strain NBRC 101917 / NGR...            394   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Adenosine 5'-phosphosulfate reductase (APS red...  ...           NaN   \n",
       " 1  Adenosine 5'-phosphosulfate reductase (APS red...  ...           NaN   \n",
       " 2  Adenosine 5'-phosphosulfate reductase (APS red...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                         Gene Names  \\\n",
       " 0  Q8VYG2  GALAK_ARATH  GALAK At3g10700 F13M14.1 T7M13.22   \n",
       " \n",
       "   Gene Names (primary)  Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                GALAK                   NaN                  At3g10700   \n",
       " \n",
       "     Gene Names (ORF)                                Organism  Organism (ID)  \\\n",
       " 0  F13M14.1 T7M13.22  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Galacturonokinase (EC 2.7.1.44) (D-galacturoni...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide PDB  \n",
       " 0             NaN NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P18485  1A12_SOLLC        ACS2 ACC2 PCVV4A                 ACS2   \n",
       " 1  P29535  1A14_SOLLC  ACS4 ACC4 BTAS4 PCVV4B                 ACS4   \n",
       " 2  Q42881  1A13_SOLLC                    ACS3                 ACS3   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0          ACC2 PCVV4A                         NaN               NaN   \n",
       " 1    ACC4 BTAS4 PCVV4B                         NaN               NaN   \n",
       " 2                  NaN                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Solanum lycopersicum (Tomato) (Lycopersicon es...           4081   \n",
       " 1  Solanum lycopersicum (Tomato) (Lycopersicon es...           4081   \n",
       " 2  Solanum lycopersicum (Tomato) (Lycopersicon es...           4081   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  1-aminocyclopropane-1-carboxylate synthase 2 (...  ...           NaN   \n",
       " 1  1-aminocyclopropane-1-carboxylate synthase 4 (...  ...           NaN   \n",
       " 2  1-aminocyclopropane-1-carboxylate synthase 3 (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 278; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 1  MOD_RES 282; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 2  MOD_RES 272; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylated on Ser 460; phosphorylatio...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide         PDB  \n",
       " 0             NaN             NaN  1IAX;1IAY;  \n",
       " 1             NaN             NaN         NaN  \n",
       " 2             NaN             NaN         NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q59000  G6PI_METJA  pgi MJ1605                  pgi                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     MJ1605               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glucose-6-phosphate isomerase (GPI) (EC 5.3.1....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  Q06703  CDA2_YEAST  CDA2 YLR308W L2142.1                 CDA2   \n",
       " 1  Q06702  CDA1_YEAST  CDA1 YLR307W L2142.2                 CDA1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YLR308W          L2142.1   \n",
       " 1                   NaN                    YLR307W          L2142.2   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                         Protein names  ...  \\\n",
       " 0  Chitin deacetylase 2 (EC 3.5.1.41)  ...   \n",
       " 1  Chitin deacetylase 1 (EC 3.5.1.41)  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 142; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 1  CARBOHYD 26; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " 1         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated. {ECO:0000269|PubMed:11812...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                       Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..44; /evidence=\"ECO:0000269|PubMed:11...             NaN  NaN  \n",
       " 1              SIGNAL 1..24; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  P38145  RISA_YEAST  RIB5 YBR256C YBR1724                 RIB5   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YBR256C          YBR1724   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                            Protein names  ... Glycosylation  \\\n",
       " 0  Riboflavin synthase (RS) (EC 2.5.1.9)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 95; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                             Gene Names  \\\n",
       " 0  Q93ZC5  AOC4_ARATH          AOC4 At1g13280 T6J4.4 T6J4_23   \n",
       " 1  Q9LS02  AOC2_ARATH        AOC2 At3g25770 K13N2.11 K13N2_9   \n",
       " 2  Q9LS03  AOC1_ARATH  AOC1 ERD12 At3g25760 K13N2.10 K13N2_8   \n",
       " 3  Q9LS01  AOC3_ARATH       AOC3 At3g25780 K13N2.12 K13N2_10   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                 AOC4                  NaN                  At1g13280   \n",
       " 1                 AOC2                  NaN                  At3g25770   \n",
       " 2                 AOC1                ERD12                  At3g25760   \n",
       " 3                 AOC3                  NaN                  At3g25780   \n",
       " \n",
       "     Gene Names (ORF)                                Organism  Organism (ID)  \\\n",
       " 0     T6J4.4 T6J4_23  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1   K13N2.11 K13N2_9  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2   K13N2.10 K13N2_8  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 3  K13N2.12 K13N2_10  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Allene oxide cyclase 4, chloroplastic (EC 5.3....  ...           NaN   \n",
       " 1  Allene oxide cyclase 2, chloroplastic (EC 5.3....  ...           NaN   \n",
       " 2  Allene oxide cyclase 1, chloroplastic (EC 5.3....  ...           NaN   \n",
       " 3  Allene oxide cyclase 3, chloroplastic (EC 5.3....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  \\\n",
       " 0  TRANSIT 1..52; /note=\"Chloroplast\"; /evidence=...   \n",
       " 1  TRANSIT 1..77; /note=\"Chloroplast\"; /evidence=...   \n",
       " 2  TRANSIT 1..78; /note=\"Chloroplast\"; /evidence=...   \n",
       " 3  TRANSIT 1..56; /note=\"Chloroplast\"; /evidence=...   \n",
       " \n",
       "                                    PDB  \n",
       " 0                                  NaN  \n",
       " 1  1Z8K;2BRJ;2DIO;2GIN;2Q4I;4CQ6;4CQ7;  \n",
       " 2                                1ZVC;  \n",
       " 3                                  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "      Entry   Entry Name                                   Gene Names  \\\n",
       " 0   Q9SFU6  CALS9_ARATH                CALS9 GSL10 At3g07160 T1B9.18   \n",
       " 1   Q9ZT82  CALSC_ARATH   CALS12 GSL5 PMR4 At4g03550 F9H3.18 T5L23.4   \n",
       " 2   Q3B724  CALS5_ARATH  CALS5 GSL2 LAP1 At2g13680 F13J11.3 T10F5.22   \n",
       " 3   Q9S9U0  CALSB_ARATH                CALS11 GSL1 At4g04970 T32N4.8   \n",
       " 4   Q9AUE0  CALS1_ARATH       CALS1 GSL6 At1g05570 F3F20.1 T25N20.22   \n",
       " 5   Q9SJM0  CALSA_ARATH                 CALS10 GSL8 At2g36850 T1J8.3   \n",
       " 6   Q9LYS6  CALS6_ARATH             CALS6 GSL11 At3g59100 F17J16.150   \n",
       " 7   Q9SHJ3  CALS7_ARATH               CALS7 GSL7 At1g06490 F12K11.17   \n",
       " 8   Q9LXT9  CALS3_ARATH             CALS3 GSL12 At5g13000 T24H18.170   \n",
       " 9   Q9SL03  CALS2_ARATH               CALS2 GSL3 At2g31960 F22D22.29   \n",
       " 10  Q9LTG5  CALS4_ARATH                 CALS4 GSL9 At5g36870 F5H8.14   \n",
       " 11  Q9LUD7  CALS8_ARATH                  CALS8 GSL4 At3g14570 MIE1.7   \n",
       " \n",
       "    Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                 CALS9                GSL10                  At3g07160   \n",
       " 1                CALS12            GSL5 PMR4                  At4g03550   \n",
       " 2                 CALS5            GSL2 LAP1                  At2g13680   \n",
       " 3                CALS11                 GSL1                  At4g04970   \n",
       " 4                 CALS1                 GSL6                  At1g05570   \n",
       " 5                CALS10                 GSL8                  At2g36850   \n",
       " 6                 CALS6                GSL11                  At3g59100   \n",
       " 7                 CALS7                 GSL7                  At1g06490   \n",
       " 8                 CALS3                GSL12                  At5g13000   \n",
       " 9                 CALS2                 GSL3                  At2g31960   \n",
       " 10                CALS4                 GSL9                  At5g36870   \n",
       " 11                CALS8                 GSL4                  At3g14570   \n",
       " \n",
       "      Gene Names (ORF)                                Organism  Organism (ID)  \\\n",
       " 0             T1B9.18  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1     F9H3.18 T5L23.4  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2   F13J11.3 T10F5.22  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 3             T32N4.8  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 4   F3F20.1 T25N20.22  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 5              T1J8.3  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 6          F17J16.150  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 7           F12K11.17  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 8          T24H18.170  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 9           F22D22.29  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 10            F5H8.14  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 11             MIE1.7  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                         Protein names  ...  \\\n",
       " 0   Callose synthase 9 (EC 2.4.1.34) (1,3-beta-glu...  ...   \n",
       " 1   Callose synthase 12 (EC 2.4.1.34) (1,3-beta-gl...  ...   \n",
       " 2   Callose synthase 5 (EC 2.4.1.34) (1,3-beta-glu...  ...   \n",
       " 3   Callose synthase 11 (EC 2.4.1.34) (1,3-beta-gl...  ...   \n",
       " 4   Callose synthase 1 (EC 2.4.1.34) (1,3-beta-glu...  ...   \n",
       " 5   Callose synthase 10 (EC 2.4.1.34) (1,3-beta-gl...  ...   \n",
       " 6   Putative callose synthase 6 (EC 2.4.1.34) (1,3...  ...   \n",
       " 7   Callose synthase 7 (EC 2.4.1.34) (1,3-beta-glu...  ...   \n",
       " 8   Callose synthase 3 (EC 2.4.1.34) (1,3-beta-glu...  ...   \n",
       " 9   Callose synthase 2 (EC 2.4.1.34) (1,3-beta-glu...  ...   \n",
       " 10  Callose synthase 4 (EC 2.4.1.34) (1,3-beta-glu...  ...   \n",
       " 11  Putative callose synthase 8 (EC 2.4.1.34) (1,3...  ...   \n",
       " \n",
       "                                         Glycosylation Initiator methionine  \\\n",
       " 0                                                 NaN                  NaN   \n",
       " 1   CARBOHYD 1712; /note=\"N-linked (GlcNAc...) asp...                  NaN   \n",
       " 2   CARBOHYD 1710; /note=\"N-linked (GlcNAc...) asp...                  NaN   \n",
       " 3                                                 NaN                  NaN   \n",
       " 4   CARBOHYD 577; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " 5                                                 NaN                  NaN   \n",
       " 6                                                 NaN                  NaN   \n",
       " 7                                                 NaN                  NaN   \n",
       " 8                                                 NaN                  NaN   \n",
       " 9                                                 NaN                  NaN   \n",
       " 10                                                NaN                  NaN   \n",
       " 11  CARBOHYD 1676; /note=\"N-linked (GlcNAc...) asp...                  NaN   \n",
       " \n",
       "     Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0          NaN               NaN     NaN                              NaN   \n",
       " 1          NaN               NaN     NaN                              NaN   \n",
       " 2          NaN               NaN     NaN                              NaN   \n",
       " 3          NaN               NaN     NaN                              NaN   \n",
       " 4          NaN               NaN     NaN                              NaN   \n",
       " 5          NaN               NaN     NaN                              NaN   \n",
       " 6          NaN               NaN     NaN                              NaN   \n",
       " 7          NaN               NaN     NaN                              NaN   \n",
       " 8          NaN               NaN     NaN                              NaN   \n",
       " 9          NaN               NaN     NaN                              NaN   \n",
       " 10         NaN               NaN     NaN                              NaN   \n",
       " 11         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "     Propeptide  Signal peptide Transit peptide PDB  \n",
       " 0          NaN             NaN             NaN NaN  \n",
       " 1          NaN             NaN             NaN NaN  \n",
       " 2          NaN             NaN             NaN NaN  \n",
       " 3          NaN             NaN             NaN NaN  \n",
       " 4          NaN             NaN             NaN NaN  \n",
       " 5          NaN             NaN             NaN NaN  \n",
       " 6          NaN             NaN             NaN NaN  \n",
       " 7          NaN             NaN             NaN NaN  \n",
       " 8          NaN             NaN             NaN NaN  \n",
       " 9          NaN             NaN             NaN NaN  \n",
       " 10         NaN             NaN             NaN NaN  \n",
       " 11         NaN             NaN             NaN NaN  \n",
       " \n",
       " [12 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P08440  ALF_MAIZE         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Zea mays (Maize)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           4577  Fructose-bisphosphate aldolase, cytoplasmic is...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name                  Gene Names Gene Names (primary)  \\\n",
       " 0  O53289  SERB2_MYCTU               serB2 Rv3042c                serB2   \n",
       " 1  P9WGJ3  SERB1_MYCTU  serB1 Rv0505c MTCY20G9.32c                serB1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    Rv3042c              NaN   \n",
       " 1                   NaN                    Rv0505c     MTCY20G9.32c   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphoserine phosphatase SerB2 (PSP) (PSPase)...  ...           NaN   \n",
       " 1  Phosphoserine phosphatase SerB1 (PSP) (PSPase)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q8BH82  NAPEP_MOUSE  Napepld Mbldc1              Napepld   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0               Mbldc1                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  N-acyl-phosphatidylethanolamine-hydrolyzing ph...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  D3ZVU9  NAT8L_RAT      Nat8l                Nat8l                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  N-acetylaspartate synthetase (NAA synthetase) ...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  O07564  NTDC_BACSU  ntdC yhjJ BSU10530                 ntdC   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 yhjJ                   BSU10530               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Bacillus subtilis (strain 168)         224308   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glucose-6-phosphate 3-dehydrogenase (EC 1.1.1....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P39849  XYLB_PSEPU       xylB                 xylB                   NaN   \n",
       " 1  P46364  XYLB_ACIGI        NaN                  NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas putida (Arthrobacter siderocapsula...            303   \n",
       " 1  Acinetobacter guillouiae (Acinetobacter genomo...         106649   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Aryl-alcohol dehydrogenase (EC 1.1.1.90) (Benz...  ...            NaN   \n",
       " 1  Aryl-alcohol dehydrogenase (EC 1.1.1.90) (Benz...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation Modified residue Peptide  \\\n",
       " 0                  NaN         NaN              NaN     NaN   \n",
       " 1                  NaN         NaN              NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide Signal peptide Transit peptide  \\\n",
       " 0                              NaN         NaN            NaN             NaN   \n",
       " 1                              NaN         NaN            NaN             NaN   \n",
       " \n",
       "    PDB  \n",
       " 0  NaN  \n",
       " 1  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P80209  CATD_BOVIN       CTSD                 CTSD                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)              Protein names  ...  \\\n",
       " 0           9913  Cathepsin D (EC 3.4.23.5)  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 114; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: N- and O-glycosylated. {ECO:0000250|UniPr...   \n",
       " \n",
       "                                           Propeptide  Signal peptide  \\\n",
       " 0  PROPEP 1..44; /note=\"Activation peptide\"; /evi...             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P28467  ACEA_ACICA       aceA                 aceA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                     Organism  \\\n",
       " 0                         NaN               NaN  Acinetobacter calcoaceticus   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0            471  Isocitrate lyase (ICL) (EC 4.1.3.1) (Isocitras...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN              NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P00727  AMPL_BOVIN       LAP3                 LAP3                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)             Organism  \\\n",
       " 0                         NaN               NaN  Bos taurus (Bovine)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9913  Cytosol aminopeptidase (EC 3.4.11.1) (Cysteiny...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET P00727-3:1; /note=\"Removed\"; /evidenc...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 42; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                            PDB  \n",
       " 0             NaN  1BLL;1BPM;1BPN;1LAM;1LAN;1LAP;1LCP;2EWB;2J9A;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O06543  AMACR_MYCTU  mcr Rv1143                  mcr                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     Rv1143               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Alpha-methylacyl-CoA racemase (AMACR) (EC 5.1....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                  PDB  \n",
       " 0             NaN  1X74;2GCE;2GCI;2GD0;2GD2;2GD6;2YIM;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "      Entry   Entry Name                Gene Names Gene Names (primary)  \\\n",
       " 0   P23921   RIR1_HUMAN                  RRM1 RR1                 RRM1   \n",
       " 1   Q7LG56  RIR2B_HUMAN               RRM2B P53R2                RRM2B   \n",
       " 2   P31350   RIR2_HUMAN                  RRM2 RR2                 RRM2   \n",
       " 3   P89462   RIR1_HHV2H                 RIR1 UL39                 RIR1   \n",
       " 4   P08543   RIR1_HHV11            RIR1 ICP6 UL39                 RIR1   \n",
       " 5   P0DSV1   RIR1_VAR67             I4L K4L ORF3L                  NaN   \n",
       " 6   Q4JQV7    RIR2_VZVO                RIR2 ORF18                 RIR2   \n",
       " 7   P0DSV2    RIR1_VARV             I4L K4L ORF3L                  NaN   \n",
       " 8   Q76RD8   RIR1_VACCA  MVA065L ACAM3000_MVA_065                  NaN   \n",
       " 9   P20503   RIR1_VACCC                       I4L                  NaN   \n",
       " 10  Q9DHU2    RIR2_YLDV                       20L                  NaN   \n",
       " 11  P29883   RIR2_VACCP                       F14                  NaN   \n",
       " 12  Q4JQV6    RIR1_VZVO                RIR1 ORF19                 RIR1   \n",
       " 13  P20493   RIR2_VACCC                       F4L                  NaN   \n",
       " 14  P0C702   RIR1_EBVA8                RIR1 BORF2                 RIR1   \n",
       " 15  Q3KSU5    RIR1_EBVG                RIR1 BORF2                 RIR1   \n",
       " 16  O57175   RIR2_VACCA  MVA032L ACAM3000_MVA_032                  NaN   \n",
       " 17  P0DSS7   RIR2_VAR67                   C8L F4L                  NaN   \n",
       " 18  P09248    RIR1_VZVD                RIR1 ORF19                 RIR1   \n",
       " 19  P0DSS8    RIR2_VARV                   C8L F4L                  NaN   \n",
       " 20  F5HAW0   RIR2_HHV8P                RIR2 ORF60                 RIR2   \n",
       " 21  P10224   RIR2_HHV11                 RIR2 UL40                 RIR2   \n",
       " 22  P0CAP6   RIR2_EBVB9                RIR2 BaRF1                 RIR2   \n",
       " 23  P69521   RIR2_HHV2H                 RIR2 UL40                 RIR2   \n",
       " 24  Q2HR67   RIR1_HHV8P                RIR1 ORF61                 RIR1   \n",
       " 25  P03190   RIR1_EBVB9                RIR1 BORF2                 RIR1   \n",
       " 26  P09247    RIR2_VZVD                RIR2 ORF18                 RIR2   \n",
       " 27  P0C701    RIR2_EBVG                RIR2 BaRF1                 RIR2   \n",
       " 28  P0CAP7   RIR2_EBVA8                RIR2 BaRF1                 RIR2   \n",
       " 29  P06474   RIR2_HHV1K                 RIR2 UL40                 RIR2   \n",
       " 30  P69520   RIR2_HHV23                      RIR2                 RIR2   \n",
       " 31  P09853   RIR1_HHV23                      RIR1                 RIR1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   RR1                        NaN              NaN   \n",
       " 1                 P53R2                        NaN              NaN   \n",
       " 2                   RR2                        NaN              NaN   \n",
       " 3                   NaN                       UL39              NaN   \n",
       " 4                  ICP6                       UL39              NaN   \n",
       " 5                   NaN                        NaN    I4L K4L ORF3L   \n",
       " 6                   NaN                        NaN            ORF18   \n",
       " 7                   NaN                        NaN    I4L K4L ORF3L   \n",
       " 8                   NaN   MVA065L ACAM3000_MVA_065              NaN   \n",
       " 9                   NaN                        NaN              I4L   \n",
       " 10                  NaN                        20L              NaN   \n",
       " 11                  NaN                        NaN              F14   \n",
       " 12                  NaN                        NaN            ORF19   \n",
       " 13                  NaN                        NaN              F4L   \n",
       " 14                  NaN                        NaN            BORF2   \n",
       " 15                  NaN                        NaN            BORF2   \n",
       " 16                  NaN   MVA032L ACAM3000_MVA_032              NaN   \n",
       " 17                  NaN                        NaN          C8L F4L   \n",
       " 18                  NaN                      ORF19              NaN   \n",
       " 19                  NaN                        NaN          C8L F4L   \n",
       " 20                ORF60                        NaN              NaN   \n",
       " 21                  NaN                        NaN             UL40   \n",
       " 22                  NaN                        NaN            BaRF1   \n",
       " 23                  NaN                        NaN             UL40   \n",
       " 24                  NaN                      ORF61              NaN   \n",
       " 25                  NaN                        NaN            BORF2   \n",
       " 26                  NaN                        NaN            ORF18   \n",
       " 27                  NaN                        NaN            BaRF1   \n",
       " 28                  NaN                        NaN            BaRF1   \n",
       " 29                 UL40                        NaN              NaN   \n",
       " 30                  NaN                        NaN              NaN   \n",
       " 31                  NaN                        NaN              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                Homo sapiens (Human)           9606   \n",
       " 1                                Homo sapiens (Human)           9606   \n",
       " 2                                Homo sapiens (Human)           9606   \n",
       " 3   Human herpesvirus 2 (strain HG52) (HHV-2) (Hum...          10315   \n",
       " 4   Human herpesvirus 1 (strain 17) (HHV-1) (Human...          10299   \n",
       " 5   Variola virus (isolate Human/India/Ind3/1967) ...         587200   \n",
       " 6   Varicella-zoster virus (strain Oka vaccine) (H...         341980   \n",
       " 7                                       Variola virus          10255   \n",
       " 8               Vaccinia virus (strain Ankara) (VACV)         126794   \n",
       " 9           Vaccinia virus (strain Copenhagen) (VACV)          10249   \n",
       " 10                     Yaba-like disease virus (YLDV)         132475   \n",
       " 11               Vaccinia virus (strain L-IVP) (VACV)          31531   \n",
       " 12  Varicella-zoster virus (strain Oka vaccine) (H...         341980   \n",
       " 13          Vaccinia virus (strain Copenhagen) (VACV)          10249   \n",
       " 14  Epstein-Barr virus (strain AG876) (HHV-4) (Hum...          82830   \n",
       " 15  Epstein-Barr virus (strain GD1) (HHV-4) (Human...          10376   \n",
       " 16              Vaccinia virus (strain Ankara) (VACV)         126794   \n",
       " 17  Variola virus (isolate Human/India/Ind3/1967) ...         587200   \n",
       " 18  Varicella-zoster virus (strain Dumas) (HHV-3) ...          10338   \n",
       " 19                                      Variola virus          10255   \n",
       " 20  Human herpesvirus 8 type P (isolate GK18) (HHV...         868565   \n",
       " 21  Human herpesvirus 1 (strain 17) (HHV-1) (Human...          10299   \n",
       " 22  Epstein-Barr virus (strain B95-8) (HHV-4) (Hum...          10377   \n",
       " 23  Human herpesvirus 2 (strain HG52) (HHV-2) (Hum...          10315   \n",
       " 24  Human herpesvirus 8 type P (isolate GK18) (HHV...         868565   \n",
       " 25  Epstein-Barr virus (strain B95-8) (HHV-4) (Hum...          10377   \n",
       " 26  Varicella-zoster virus (strain Dumas) (HHV-3) ...          10338   \n",
       " 27  Epstein-Barr virus (strain GD1) (HHV-4) (Human...          10376   \n",
       " 28  Epstein-Barr virus (strain AG876) (HHV-4) (Hum...          82830   \n",
       " 29  Human herpesvirus 1 (strain KOS) (HHV-1) (Huma...          10306   \n",
       " 30  Human herpesvirus 2 (strain 333) (HHV-2) (Huma...          10313   \n",
       " 31  Human herpesvirus 2 (strain 333) (HHV-2) (Huma...          10313   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 1   Ribonucleoside-diphosphate reductase subunit M...  ...           NaN   \n",
       " 2   Ribonucleoside-diphosphate reductase subunit M...  ...           NaN   \n",
       " 3   Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 4   Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 5   Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 6   Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 7   Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 8   Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 9   Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 10  Ribonucleoside-diphosphate reductase small cha...  ...           NaN   \n",
       " 11  Ribonucleoside-diphosphate reductase small cha...  ...           NaN   \n",
       " 12  Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 13  Ribonucleoside-diphosphate reductase small cha...  ...           NaN   \n",
       " 14  Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 15  Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 16  Ribonucleoside-diphosphate reductase small cha...  ...           NaN   \n",
       " 17  Ribonucleoside-diphosphate reductase small cha...  ...           NaN   \n",
       " 18  Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 19  Ribonucleoside-diphosphate reductase small cha...  ...           NaN   \n",
       " 20  Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 21  Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 22  Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 23  Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 24  Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 25  Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " 26  Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 27  Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 28  Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 29  Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 30  Ribonucleoside-diphosphate reductase small sub...  ...           NaN   \n",
       " 31  Ribonucleoside-diphosphate reductase large sub...  ...           NaN   \n",
       " \n",
       "    Initiator methionine Lipidation  \\\n",
       " 0                   NaN        NaN   \n",
       " 1                   NaN        NaN   \n",
       " 2                   NaN        NaN   \n",
       " 3                   NaN        NaN   \n",
       " 4                   NaN        NaN   \n",
       " 5                   NaN        NaN   \n",
       " 6                   NaN        NaN   \n",
       " 7                   NaN        NaN   \n",
       " 8                   NaN        NaN   \n",
       " 9                   NaN        NaN   \n",
       " 10                  NaN        NaN   \n",
       " 11                  NaN        NaN   \n",
       " 12                  NaN        NaN   \n",
       " 13                  NaN        NaN   \n",
       " 14                  NaN        NaN   \n",
       " 15                  NaN        NaN   \n",
       " 16                  NaN        NaN   \n",
       " 17                  NaN        NaN   \n",
       " 18                  NaN        NaN   \n",
       " 19                  NaN        NaN   \n",
       " 20                  NaN        NaN   \n",
       " 21                  NaN        NaN   \n",
       " 22                  NaN        NaN   \n",
       " 23                  NaN        NaN   \n",
       " 24                  NaN        NaN   \n",
       " 25                  NaN        NaN   \n",
       " 26                  NaN        NaN   \n",
       " 27                  NaN        NaN   \n",
       " 28                  NaN        NaN   \n",
       " 29                  NaN        NaN   \n",
       " 30                  NaN        NaN   \n",
       " 31                  NaN        NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 17; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 1                                                 NaN     NaN   \n",
       " 2   MOD_RES 20; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 3                                                 NaN     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6                                                 NaN     NaN   \n",
       " 7                                                 NaN     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9                                                 NaN     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11                                                NaN     NaN   \n",
       " 12                                                NaN     NaN   \n",
       " 13                                                NaN     NaN   \n",
       " 14                                                NaN     NaN   \n",
       " 15                                                NaN     NaN   \n",
       " 16                                                NaN     NaN   \n",
       " 17                                                NaN     NaN   \n",
       " 18                                                NaN     NaN   \n",
       " 19                                                NaN     NaN   \n",
       " 20                                                NaN     NaN   \n",
       " 21                                                NaN     NaN   \n",
       " 22                                                NaN     NaN   \n",
       " 23                                                NaN     NaN   \n",
       " 24                                                NaN     NaN   \n",
       " 25                                                NaN     NaN   \n",
       " 26                                                NaN     NaN   \n",
       " 27                                                NaN     NaN   \n",
       " 28                                                NaN     NaN   \n",
       " 29                                                NaN     NaN   \n",
       " 30                                                NaN     NaN   \n",
       " 31                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0                                                 NaN         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2   PTM: Phosphorylation on Ser-20 relieves the in...         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19                                                NaN         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22                                                NaN         NaN   \n",
       " 23                                                NaN         NaN   \n",
       " 24                                                NaN         NaN   \n",
       " 25                                                NaN         NaN   \n",
       " 26                                                NaN         NaN   \n",
       " 27                                                NaN         NaN   \n",
       " 28                                                NaN         NaN   \n",
       " 29                                                NaN         NaN   \n",
       " 30                                                NaN         NaN   \n",
       " 31                                                NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide  \\\n",
       " 0              NaN             NaN   \n",
       " 1              NaN             NaN   \n",
       " 2              NaN             NaN   \n",
       " 3              NaN             NaN   \n",
       " 4              NaN             NaN   \n",
       " 5              NaN             NaN   \n",
       " 6              NaN             NaN   \n",
       " 7              NaN             NaN   \n",
       " 8              NaN             NaN   \n",
       " 9              NaN             NaN   \n",
       " 10             NaN             NaN   \n",
       " 11             NaN             NaN   \n",
       " 12             NaN             NaN   \n",
       " 13             NaN             NaN   \n",
       " 14             NaN             NaN   \n",
       " 15             NaN             NaN   \n",
       " 16             NaN             NaN   \n",
       " 17             NaN             NaN   \n",
       " 18             NaN             NaN   \n",
       " 19             NaN             NaN   \n",
       " 20             NaN             NaN   \n",
       " 21             NaN             NaN   \n",
       " 22             NaN             NaN   \n",
       " 23             NaN             NaN   \n",
       " 24             NaN             NaN   \n",
       " 25             NaN             NaN   \n",
       " 26             NaN             NaN   \n",
       " 27             NaN             NaN   \n",
       " 28             NaN             NaN   \n",
       " 29             NaN             NaN   \n",
       " 30             NaN             NaN   \n",
       " 31             NaN             NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0   2WGH;3HNC;3HND;3HNE;3HNF;4X3V;5D1Y;5TUS;6AUI;6...  \n",
       " 1                                     2VUX;3HF1;4DJN;  \n",
       " 2                           2UW2;3OLJ;3VPM;3VPN;3VPO;  \n",
       " 3                                                 NaN  \n",
       " 4                                                 NaN  \n",
       " 5                                                 NaN  \n",
       " 6                                                 NaN  \n",
       " 7                                                 NaN  \n",
       " 8                                                 NaN  \n",
       " 9                                                 NaN  \n",
       " 10                                                NaN  \n",
       " 11                                                NaN  \n",
       " 12                                                NaN  \n",
       " 13                                                NaN  \n",
       " 14                                                NaN  \n",
       " 15                                                NaN  \n",
       " 16                                                NaN  \n",
       " 17                                                NaN  \n",
       " 18                                                NaN  \n",
       " 19                                                NaN  \n",
       " 20                                                NaN  \n",
       " 21                                                NaN  \n",
       " 22                                                NaN  \n",
       " 23                                                NaN  \n",
       " 24                                                NaN  \n",
       " 25                                              7RW6;  \n",
       " 26                                                NaN  \n",
       " 27                                                NaN  \n",
       " 28                                                NaN  \n",
       " 29                                                NaN  \n",
       " 30                                                NaN  \n",
       " 31                                                NaN  \n",
       " \n",
       " [32 rows x 110 columns],\n",
       "     Entry Entry Name                       Gene Names Gene Names (primary)  \\\n",
       " 0  Q9S7H4  CM2_ARATH         CM2 At5g10870 T30N20.140                  CM2   \n",
       " 1  Q9C544  CM3_ARATH  CM3 At1g69370 F10D13.6 F23O10.5                  CM3   \n",
       " 2  P42738  CM1_ARATH            CM1 At3g29200 MXO21.4                  CM1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)   Gene Names (ORF)  \\\n",
       " 0                   NaN                  At5g10870         T30N20.140   \n",
       " 1                   NaN                  At1g69370  F10D13.6 F23O10.5   \n",
       " 2                   NaN                  At3g29200            MXO21.4   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0   Chorismate mutase 2 (AtCM2) (EC 5.4.99.5) (CM-2)  ...           NaN   \n",
       " 1  Chorismate mutase 3, chloroplastic (AtCM3) (EC...  ...           NaN   \n",
       " 2  Chorismate mutase 1, chloroplastic (AtCM1) (EC...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0                                                NaN     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2  MOD_RES 66; /note=\"N-acetylalanine\"; /evidence...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide         PDB  \n",
       " 0                                                NaN         NaN  \n",
       " 1  TRANSIT 1..47; /note=\"Chloroplast\"; /evidence=...         NaN  \n",
       " 2  TRANSIT 1..65; /note=\"Chloroplast\"; /evidence=...  4PPU;4PPV;  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q43092   SSG1_PEA         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                    Organism  \\\n",
       " 0                         NaN               NaN  Pisum sativum (Garden pea)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           3888  Granule-bound starch synthase 1, chloroplastic...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..75; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P15169  CBPN_HUMAN  CPN1 ACBP                 CPN1                 ACBP   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Carboxypeptidase N catalytic chain (CPN) (EC 3...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 400; /note=\"O-linked (GalNAc...) thre...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide                                     Signal peptide  \\\n",
       " 0         NaN  SIGNAL 1..20; /evidence=\"ECO:0000269|PubMed:34...   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2NSM;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P30190  TOP2_TRYCR       TOP2                 TOP2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)           Organism  \\\n",
       " 0                         NaN               NaN  Trypanosoma cruzi   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           5693  DNA topoisomerase 2 (EC 5.6.2.2) (DNA topoisom...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P24521  ERG8_YEAST  ERG8 YMR220W YM9959.02                 ERG8   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    YMR220W        YM9959.02   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                            Protein names  ... Glycosylation  \\\n",
       " 0  Phosphomevalonate kinase (EC 2.7.4.2)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  Q55GH4  GYS_DICDI  glcS DDB_G0267674                 glcS   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN     DDB_G0267674   \n",
       " \n",
       "                                 Organism  Organism (ID)  \\\n",
       " 0  Dictyostelium discoideum (Slime mold)          44689   \n",
       " \n",
       "                               Protein names  ... Glycosylation  \\\n",
       " 0  Glycogen [starch] synthase (EC 2.4.1.11)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q9W197  5NT3B_DROME  cN-IIIB CG3362              cN-IIIB   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN           CG3362   \n",
       " \n",
       "                               Organism  Organism (ID)  \\\n",
       " 0  Drosophila melanogaster (Fruit fly)           7227   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  7-methylguanosine phosphate-specific 5'-nucleo...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  4NV0;4NWI;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q91VF2  HNMT_MOUSE   Hnmt Hmt                 Hnmt                  Hmt   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                     Protein names  ...  \\\n",
       " 0          10090  Histamine N-methyltransferase (HMT) (EC 2.1.1.8)  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  P09147  GALE_ECOLI   galE galD b0759 JW0742                 galE   \n",
       " 1  Q42605  UGE1_ARATH  UGE1 At1g12780 F13K23.3                 UGE1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 galD               b0759 JW0742              NaN   \n",
       " 1                  NaN                  At1g12780         F13K23.3   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0           Escherichia coli (strain K12)          83333   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  UDP-glucose 4-epimerase (EC 5.1.3.2) (Galactow...  ...           NaN   \n",
       " 1  Bifunctional UDP-glucose 4-epimerase and UDP-x...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1A9Y;1A9Z;1KVQ;1KVR;1KVS;1KVT;1KVU;1LRJ;1LRK;1...  \n",
       " 1             NaN                                                NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P0AEW6  INGK_ECOLI   gsk b0477 JW0466                  gsk   \n",
       " 1  P0AEW8  INGK_ECO57  gsk Z0596 ECs0530                  gsk   \n",
       " 2  P0AEW7  INGK_ECOL6          gsk c0597                  gsk   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN               b0477 JW0466               NaN   \n",
       " 1                   NaN              Z0596 ECs0530               NaN   \n",
       " 2                   NaN                      c0597               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                           Escherichia coli O157:H7          83334   \n",
       " 2  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " \n",
       "                             Protein names  ... Glycosylation  \\\n",
       " 0  Guanosine-inosine kinase (EC 2.7.1.73)  ...           NaN   \n",
       " 1  Guanosine-inosine kinase (EC 2.7.1.73)  ...           NaN   \n",
       " 2  Guanosine-inosine kinase (EC 2.7.1.73)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  6VWO;6VWP;  \n",
       " 1             NaN         NaN  \n",
       " 2             NaN         NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  Q0P5G4   KAT3_BOVIN  KYAT3 CCBL2 KAT3                KYAT3   \n",
       " 1  Q5E9N4  AADAT_BOVIN             AADAT                AADAT   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           CCBL2 KAT3                         NaN               NaN   \n",
       " 1                  NaN                         NaN               NaN   \n",
       " \n",
       "               Organism  Organism (ID)  \\\n",
       " 0  Bos taurus (Bovine)           9913   \n",
       " 1  Bos taurus (Bovine)           9913   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Kynurenine--oxoglutarate transaminase 3 (EC 2....  ...           NaN   \n",
       " 1  Kynurenine/alpha-aminoadipate aminotransferase...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 116; /note=\"N6-acetyllysine; alternate...     NaN   \n",
       " 1  MOD_RES 69; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0                                                NaN  NaN  \n",
       " 1  TRANSIT 1..29; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                Gene Names Gene Names (primary)  \\\n",
       " 0  P9WK17  MASZ_MYCTU  glcB Rv1837c MTCY1A11.06                 glcB   \n",
       " 1  P9WK16  MASZ_MYCTO               glcB MT1885                 glcB   \n",
       " 2  A5U3K4  MASZ_MYCTA             glcB MRA_1848                 glcB   \n",
       " 3  A1KJP9  MASZ_MYCBP            glcB BCG_1872c                 glcB   \n",
       " 4  C1APB4  MASZ_MYCBT             glcB JTY_1856                 glcB   \n",
       " 5  P0A5J5  MASZ_MYCBO       glcB BQ2027_MB1868C                 glcB   \n",
       " 6  B2HSY2  MASZ_MYCMM            glcB MMAR_2713                 glcB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    Rv1837c      MTCY1A11.06   \n",
       " 1                   NaN                     MT1885              NaN   \n",
       " 2                   NaN                   MRA_1848              NaN   \n",
       " 3                   NaN                  BCG_1872c              NaN   \n",
       " 4                   NaN                   JTY_1856              NaN   \n",
       " 5                   NaN             BQ2027_MB1868C              NaN   \n",
       " 6                   NaN                  MMAR_2713              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 2  Mycobacterium tuberculosis (strain ATCC 25177 ...         419947   \n",
       " 3  Mycobacterium bovis (strain BCG / Pasteur 1173P2)         410289   \n",
       " 4  Mycobacterium bovis (strain BCG / Tokyo 172 / ...         561275   \n",
       " 5  Mycobacterium bovis (strain ATCC BAA-935 / AF2...         233413   \n",
       " 6    Mycobacterium marinum (strain ATCC BAA-535 / M)         216594   \n",
       " \n",
       "                     Protein names  ... Glycosylation  \\\n",
       " 0  Malate synthase G (EC 2.3.3.9)  ...           NaN   \n",
       " 1  Malate synthase G (EC 2.3.3.9)  ...           NaN   \n",
       " 2  Malate synthase G (EC 2.3.3.9)  ...           NaN   \n",
       " 3  Malate synthase G (EC 2.3.3.9)  ...           NaN   \n",
       " 4  Malate synthase G (EC 2.3.3.9)  ...           NaN   \n",
       " 5  Malate synthase G (EC 2.3.3.9)  ...           NaN   \n",
       " 6  Malate synthase G (EC 2.3.3.9)  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 6                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 619; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 1  MOD_RES 619; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 2  MOD_RES 619; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 3  MOD_RES 619; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 4  MOD_RES 619; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 5  MOD_RES 619; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 6  MOD_RES 623; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  2GQ3;3S9I;3S9Z;3SAD;3SAZ;3SB0;5C7V;5C9R;5C9U;5...  \n",
       " 1             NaN                                         1N8I;1N8W;  \n",
       " 2             NaN                                                NaN  \n",
       " 3             NaN                                                NaN  \n",
       " 4             NaN                                                NaN  \n",
       " 5             NaN                                                NaN  \n",
       " 6             NaN                                              6AXE;  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                Gene Names Gene Names (primary)  \\\n",
       " 0  Q9JHW9  AL1A3_MOUSE      Aldh1a3 Aldh6 Raldh3              Aldh1a3   \n",
       " 1  P24549  AL1A1_MOUSE  Aldh1a1 Ahd-2 Ahd2 Aldh1              Aldh1a1   \n",
       " 2  Q62148  AL1A2_MOUSE    Aldh1a2 Aldh1a7 Raldh2              Aldh1a2   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0         Aldh6 Raldh3                         NaN               NaN   \n",
       " 1     Ahd-2 Ahd2 Aldh1                         NaN               NaN   \n",
       " 2       Aldh1a7 Raldh2                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " 2  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aldehyde dehydrogenase family 1 member A3 (EC ...  ...           NaN   \n",
       " 1  Aldehyde dehydrogenase 1A1 (EC 1.2.1.19) (EC 1...  ...           NaN   \n",
       " 2  Retinal dehydrogenase 2 (RALDH 2) (RalDH2) (EC...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 2  MOD_RES 168; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: The N-terminus is blocked most probably b...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide    PDB  \n",
       " 0             NaN             NaN    NaN  \n",
       " 1             NaN             NaN  7YOB;  \n",
       " 2             NaN             NaN    NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  P14697  PHAB_CUPNH  phaB phbB H16_A1439                 phaB   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 phbB                  H16_A1439               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Cupriavidus necator (strain ATCC 17699 / DSM 4...         381666   \n",
       " \n",
       "                              Protein names  ... Glycosylation  \\\n",
       " 0  Acetoacetyl-CoA reductase (EC 1.1.1.36)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                  PDB  \n",
       " 0             NaN  3VZP;3VZQ;3VZR;3VZS;4N5L;4N5M;4N5N;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  P05041  PABB_ECOLI  pabB b1812 JW1801                 pabB   \n",
       " 1  P00903  PABA_ECOLI  pabA b3360 JW3323                 pabA   \n",
       " 2  P06193  PABA_SALTY       pabA STM3469                 pabA   \n",
       " 3  P27629  PABB_LACLL               pabB                 pabB   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN               b1812 JW1801               NaN   \n",
       " 1                   NaN               b3360 JW3323               NaN   \n",
       " 2                   NaN                    STM3469               NaN   \n",
       " 3                   NaN                        NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain K12)          83333   \n",
       " 2  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 3  Lactococcus lactis subsp. lactis (Streptococcu...           1360   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aminodeoxychorismate synthase component 1 (ADC...  ...           NaN   \n",
       " 1  Aminodeoxychorismate synthase component 2 (ADC...  ...           NaN   \n",
       " 2  Aminodeoxychorismate synthase component 2 (ADC...  ...           NaN   \n",
       " 3  Aminodeoxychorismate synthase component 1 (ADC...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1K0E;1K0G;  \n",
       " 1             NaN         NaN  \n",
       " 2             NaN         NaN  \n",
       " 3             NaN         NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P19836  PCY1A_RAT  Pcyt1a Ctpct Pcyt1               Pcyt1a   \n",
       " 1  Q9QZC4  PCY1B_RAT         Pcyt1b Cctb               Pcyt1b   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0          Ctpct Pcyt1                         NaN               NaN   \n",
       " 1                 Cctb                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " 1  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Choline-phosphate cytidylyltransferase A (EC 2...  ...           NaN   \n",
       " 1  Choline-phosphate cytidylyltransferase B (EC 2...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 1  MOD_RES 315; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: The serine residues of the C-terminus are...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                        PDB  \n",
       " 0             NaN             NaN  1PEH;1PEI;3HL4;4MVC;4MVD;  \n",
       " 1             NaN             NaN                        NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name      Gene Names  Gene Names (primary)  \\\n",
       " 0  P80402  DBD23_ASPOR  AO090005000447                   NaN   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN   AO090005000447   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Aspergillus oryzae (strain ATCC 42149 / RIB 40...         510516   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  2,3-dihydroxybenzoate decarboxylase (2,3-DHBA ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  7A19;7A1A;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name                      Gene Names Gene Names (primary)  \\\n",
       " 0   P10408  SECA_ECOLI  secA azi pea prlD b0098 JW0096                 secA   \n",
       " 1   P28366  SECA_BACSU              secA div+ BSU35300                 secA   \n",
       " 2   P45759  GSPE_ECOLI          gspE yheG b3326 JW3288                 gspE   \n",
       " 3   B6HZ76  SECA_ECOSE                  secA ECSE_0100                 secA   \n",
       " 4   Q8X996  SECA_ECO57              secA Z0108 ECs0102                 secA   \n",
       " 5   B1XC75  SECA_ECODH               secA ECDH10B_0080                 secA   \n",
       " 6   B7M141  SECA_ECO8A                secA ECIAI1_0098                 secA   \n",
       " 7   B7MNV7  SECA_ECO81                 secA ECED1_0099                 secA   \n",
       " 8   A1A7E3  SECA_ECOK1    secA Ecok1_00890 APECO1_1888                 secA   \n",
       " 9   B7LFW8  SECA_ECO55               secA EC55989_0094                 secA   \n",
       " 10  Q1RG97  SECA_ECOUT                secA UTI89_C0107                 secA   \n",
       " 11  P0A1C1  SCTN_SHIFL     sctN mxiB spa47 spaL CP0149                 sctN   \n",
       " 12  C4ZRJ3  SECA_ECOBW                   secA BWG_0093                 secA   \n",
       " 13  A7ZW50  SECA_ECOHS                 secA EcHS_A0104                 secA   \n",
       " 14  B5YZD4  SECA_ECO5E              secA ECH74115_0106                 secA   \n",
       " 15  A7ZHI9  SECA_ECO24             secA EcE24377A_0100                 secA   \n",
       " 16  B1LG35  SECA_ECOSM               secA EcSMS35_0103                 secA   \n",
       " 17  B7NHK4  SECA_ECO7I               secA ECIAI39_0101                 secA   \n",
       " 18  B7MAM1  SECA_ECO45                 secA ECS88_0102                 secA   \n",
       " 19  B7UIE8  SECA_ECO27                secA E2348C_0103                 secA   \n",
       " 20  B7N7X1  SECA_ECOLU                 secA ECUMN_0098                 secA   \n",
       " 21  Q0TLP1  SECA_ECOL5                   secA ECP_0100                 secA   \n",
       " 22  Q8FL61  SECA_ECOL6                      secA c0116                 secA   \n",
       " 23  B1IR80  SECA_ECOLC                 secA EcolC_3559                 secA   \n",
       " 24  B7LWG4  SECA_ESCF3                  secA EFER_0120                 secA   \n",
       " 25  P15645  GSPE_KLEPN                            pulE                 pulE   \n",
       " 26  P0A4G7  SECA_STRLI                            secA                 secA   \n",
       " 27  P52966  SECA_RHOCA                            secA                 secA   \n",
       " 28  P47994  SECA_STACT                   secA Sca_0401                 secA   \n",
       " 29  Q83SN2  SECA_SHIFL               secA SF0095 S0097                 secA   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0          azi pea prlD               b0098 JW0096              NaN   \n",
       " 1                  div+                   BSU35300              NaN   \n",
       " 2                  yheG               b3326 JW3288              NaN   \n",
       " 3                   NaN                  ECSE_0100              NaN   \n",
       " 4                   NaN              Z0108 ECs0102              NaN   \n",
       " 5                   NaN               ECDH10B_0080              NaN   \n",
       " 6                   NaN                ECIAI1_0098              NaN   \n",
       " 7                   NaN                 ECED1_0099              NaN   \n",
       " 8                   NaN                Ecok1_00890      APECO1_1888   \n",
       " 9                   NaN               EC55989_0094              NaN   \n",
       " 10                  NaN                UTI89_C0107              NaN   \n",
       " 11      mxiB spa47 spaL                     CP0149              NaN   \n",
       " 12                  NaN                   BWG_0093              NaN   \n",
       " 13                  NaN                 EcHS_A0104              NaN   \n",
       " 14                  NaN              ECH74115_0106              NaN   \n",
       " 15                  NaN             EcE24377A_0100              NaN   \n",
       " 16                  NaN               EcSMS35_0103              NaN   \n",
       " 17                  NaN               ECIAI39_0101              NaN   \n",
       " 18                  NaN                 ECS88_0102              NaN   \n",
       " 19                  NaN                E2348C_0103              NaN   \n",
       " 20                  NaN                 ECUMN_0098              NaN   \n",
       " 21                  NaN                   ECP_0100              NaN   \n",
       " 22                  NaN                      c0116              NaN   \n",
       " 23                  NaN                 EcolC_3559              NaN   \n",
       " 24                  NaN                  EFER_0120              NaN   \n",
       " 25                  NaN                        NaN              NaN   \n",
       " 26                  NaN                        NaN              NaN   \n",
       " 27                  NaN                        NaN              NaN   \n",
       " 28                  NaN                   Sca_0401              NaN   \n",
       " 29                  NaN               SF0095 S0097              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                      Bacillus subtilis (strain 168)         224308   \n",
       " 2                       Escherichia coli (strain K12)          83333   \n",
       " 3                      Escherichia coli (strain SE11)         409438   \n",
       " 4                            Escherichia coli O157:H7          83334   \n",
       " 5               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 6                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 7                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 8                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 9              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 10             Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 11                                  Shigella flexneri            623   \n",
       " 12    Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 13                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 14    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 15  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 16          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 17      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 18       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 19  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 20  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 21    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 22  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 23  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 24  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 25                              Klebsiella pneumoniae            573   \n",
       " 26                              Streptomyces lividans           1916   \n",
       " 27  Rhodobacter capsulatus (Rhodopseudomonas capsu...           1061   \n",
       " 28             Staphylococcus carnosus (strain TM300)         396513   \n",
       " 29                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0       Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 1       Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 2   Putative type II secretion system protein E (T...  ...           NaN   \n",
       " 3       Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 4       Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 5       Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 6       Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 7       Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 8       Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 9       Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 10      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 11  Type 3 secretion system ATPase (T3SS ATPase) (...  ...           NaN   \n",
       " 12      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 13      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 14      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 15      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 16      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 17      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 18      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 19      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 20      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 21      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 22      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 23      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 24      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 25  Type II secretion system protein E (T2SS prote...  ...           NaN   \n",
       " 26      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 27      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 28      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " 29      Protein translocase subunit SecA (EC 7.4.2.8)  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                   NaN         NaN               NaN     NaN   \n",
       " 1                   NaN         NaN               NaN     NaN   \n",
       " 2                   NaN         NaN               NaN     NaN   \n",
       " 3                   NaN         NaN               NaN     NaN   \n",
       " 4                   NaN         NaN               NaN     NaN   \n",
       " 5                   NaN         NaN               NaN     NaN   \n",
       " 6                   NaN         NaN               NaN     NaN   \n",
       " 7                   NaN         NaN               NaN     NaN   \n",
       " 8                   NaN         NaN               NaN     NaN   \n",
       " 9                   NaN         NaN               NaN     NaN   \n",
       " 10                  NaN         NaN               NaN     NaN   \n",
       " 11                  NaN         NaN               NaN     NaN   \n",
       " 12                  NaN         NaN               NaN     NaN   \n",
       " 13                  NaN         NaN               NaN     NaN   \n",
       " 14                  NaN         NaN               NaN     NaN   \n",
       " 15                  NaN         NaN               NaN     NaN   \n",
       " 16                  NaN         NaN               NaN     NaN   \n",
       " 17                  NaN         NaN               NaN     NaN   \n",
       " 18                  NaN         NaN               NaN     NaN   \n",
       " 19                  NaN         NaN               NaN     NaN   \n",
       " 20                  NaN         NaN               NaN     NaN   \n",
       " 21                  NaN         NaN               NaN     NaN   \n",
       " 22                  NaN         NaN               NaN     NaN   \n",
       " 23                  NaN         NaN               NaN     NaN   \n",
       " 24                  NaN         NaN               NaN     NaN   \n",
       " 25                  NaN         NaN               NaN     NaN   \n",
       " 26                  NaN         NaN               NaN     NaN   \n",
       " 27                  NaN         NaN               NaN     NaN   \n",
       " 28                  NaN         NaN               NaN     NaN   \n",
       " 29                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide Signal peptide  \\\n",
       " 0                               NaN         NaN            NaN   \n",
       " 1                               NaN         NaN            NaN   \n",
       " 2                               NaN         NaN            NaN   \n",
       " 3                               NaN         NaN            NaN   \n",
       " 4                               NaN         NaN            NaN   \n",
       " 5                               NaN         NaN            NaN   \n",
       " 6                               NaN         NaN            NaN   \n",
       " 7                               NaN         NaN            NaN   \n",
       " 8                               NaN         NaN            NaN   \n",
       " 9                               NaN         NaN            NaN   \n",
       " 10                              NaN         NaN            NaN   \n",
       " 11                              NaN         NaN            NaN   \n",
       " 12                              NaN         NaN            NaN   \n",
       " 13                              NaN         NaN            NaN   \n",
       " 14                              NaN         NaN            NaN   \n",
       " 15                              NaN         NaN            NaN   \n",
       " 16                              NaN         NaN            NaN   \n",
       " 17                              NaN         NaN            NaN   \n",
       " 18                              NaN         NaN            NaN   \n",
       " 19                              NaN         NaN            NaN   \n",
       " 20                              NaN         NaN            NaN   \n",
       " 21                              NaN         NaN            NaN   \n",
       " 22                              NaN         NaN            NaN   \n",
       " 23                              NaN         NaN            NaN   \n",
       " 24                              NaN         NaN            NaN   \n",
       " 25                              NaN         NaN            NaN   \n",
       " 26                              NaN         NaN            NaN   \n",
       " 27                              NaN         NaN            NaN   \n",
       " 28                              NaN         NaN            NaN   \n",
       " 29                              NaN         NaN            NaN   \n",
       " \n",
       "    Transit peptide                                                PDB  \n",
       " 0              NaN           1TM6;2FSF;2FSG;2FSH;2FSI;2VDA;3BXZ;6GOX;  \n",
       " 1              NaN  1M6N;1M74;1TF2;1TF5;2IBM;3DL8;3IQM;3IQY;3JV2;5...  \n",
       " 2              NaN                                                NaN  \n",
       " 3              NaN                                                NaN  \n",
       " 4              NaN                                                NaN  \n",
       " 5              NaN                                                NaN  \n",
       " 6              NaN                                                NaN  \n",
       " 7              NaN                                                NaN  \n",
       " 8              NaN                                                NaN  \n",
       " 9              NaN                                                NaN  \n",
       " 10             NaN                                                NaN  \n",
       " 11             NaN  5SWJ;5SWL;5SYP;5SYR;5YBH;5YBI;5ZT1;6N6L;6N6M;6...  \n",
       " 12             NaN                                                NaN  \n",
       " 13             NaN                                                NaN  \n",
       " 14             NaN                                                NaN  \n",
       " 15             NaN                                                NaN  \n",
       " 16             NaN                                                NaN  \n",
       " 17             NaN                                                NaN  \n",
       " 18             NaN                                                NaN  \n",
       " 19             NaN                                                NaN  \n",
       " 20             NaN                                                NaN  \n",
       " 21             NaN                                                NaN  \n",
       " 22             NaN                                                NaN  \n",
       " 23             NaN                                                NaN  \n",
       " 24             NaN                                                NaN  \n",
       " 25             NaN                                                NaN  \n",
       " 26             NaN                                                NaN  \n",
       " 27             NaN                                                NaN  \n",
       " 28             NaN                                                NaN  \n",
       " 29             NaN                                                NaN  \n",
       " \n",
       " [30 rows x 110 columns],\n",
       "     Entry   Entry Name                                Gene Names  \\\n",
       " 0  P11155  PPDK1_MAIZE           PPDK1 C4PPDKZM1 CYPPDKZM1 PPDK2   \n",
       " 1  Q42368  PPDK2_MAIZE  PPDK2 CYPPDKZM2 ZEAMMB73_872355 Zm.19608   \n",
       " \n",
       "   Gene Names (primary)       Gene Names (synonym)  Gene Names (ordered locus)  \\\n",
       " 0                PPDK1  C4PPDKZM1 CYPPDKZM1 PPDK2                         NaN   \n",
       " 1                PPDK2                  CYPPDKZM2                         NaN   \n",
       " \n",
       "            Gene Names (ORF)          Organism  Organism (ID)  \\\n",
       " 0                       NaN  Zea mays (Maize)           4577   \n",
       " 1  ZEAMMB73_872355 Zm.19608  Zea mays (Maize)           4577   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Pyruvate, phosphate dikinase 1, chloroplastic ...  ...           NaN   \n",
       " 1  Pyruvate, phosphate dikinase 2 (EC 2.7.9.1) (P...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 63; /note=\"N-acetylalanine; partial\"; ...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylation of Thr-527 in the dark in...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  \\\n",
       " 0             NaN  TRANSIT 1..62; /note=\"Chloroplast\"; /evidence=...   \n",
       " 1             NaN                                                NaN   \n",
       " \n",
       "           PDB  \n",
       " 0  1VBG;1VBH;  \n",
       " 1         NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  Q02418  MTLD_STRMU  mtlD SMU_1182                 mtlD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   SMU_1182               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Streptococcus mutans serotype c (strain ATCC 7...         210007   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Mannitol-1-phosphate 5-dehydrogenase (EC 1.1.1...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P90521  PFKA_DICDI  pfkA pfk DDB_G0274111                 pfkA   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                  pfk                         NaN     DDB_G0274111   \n",
       " \n",
       "                                 Organism  Organism (ID)  \\\n",
       " 0  Dictyostelium discoideum (Slime mold)          44689   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  ATP-dependent 6-phosphofructokinase (ATP-PFK) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0  PTM: The N-terminus is blocked.         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P17067   CAHC_PEA         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                    Organism  \\\n",
       " 0                         NaN               NaN  Pisum sativum (Garden pea)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           3888  Carbonic anhydrase, chloroplastic (EC 4.2.1.1)...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide    PDB  \n",
       " 0  TRANSIT 1..70; /note=\"Chloroplast\"; /evidence=...  1EKJ;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  Q9WVM8  AADAT_MOUSE        Aadat Kat2                Aadat   \n",
       " 1  P05202   AATM_MOUSE        Got2 Got-2                 Got2   \n",
       " 2  Q71RI9   KAT3_MOUSE  Kyat3 Ccbl2 Kat3                Kyat3   \n",
       " 3  Q8BTY1   KAT1_MOUSE   Kyat1 Ccbl1 Kat                Kyat1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 Kat2                         NaN               NaN   \n",
       " 1                Got-2                         NaN               NaN   \n",
       " 2           Ccbl2 Kat3                         NaN               NaN   \n",
       " 3            Ccbl1 Kat                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " 2  Mus musculus (Mouse)          10090   \n",
       " 3  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Kynurenine/alpha-aminoadipate aminotransferase...  ...           NaN   \n",
       " 1  Aspartate aminotransferase, mitochondrial (mAs...  ...           NaN   \n",
       " 2  Kynurenine--oxoglutarate transaminase 3 (EC 2....  ...           NaN   \n",
       " 3  Kynurenine--oxoglutarate transaminase 1 (EC 2....  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " 3                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 40; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1  MOD_RES 48; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 2  MOD_RES 117; /note=\"N6-acetyllysine; alternate...     NaN   \n",
       " 3  MOD_RES 82; /note=\"N6-succinyllysine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: Acetylation of Lys-296, Lys-345 and Lys-3...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  \\\n",
       " 0             NaN  TRANSIT 1..29; /note=\"Mitochondrion\"; /evidenc...   \n",
       " 1             NaN               TRANSIT 1..29; /note=\"Mitochondrion\"   \n",
       " 2             NaN                                                NaN   \n",
       " 3             NaN                                                NaN   \n",
       " \n",
       "                                    PDB  \n",
       " 0                                  NaN  \n",
       " 1                      3HLM;3PD6;3PDB;  \n",
       " 2  2ZJG;3E2F;3E2Y;3E2Z;5VEP;5VEQ;5VER;  \n",
       " 3                                  NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P0CX08  DSF1_YEAST   DSF1 MAN1 YEL070W                 DSF1   \n",
       " 1  P0CX09  MAN2_YEAST  MAN2 YNR073C N3810                 MAN2   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                 MAN1                    YEL070W              NaN   \n",
       " 1                  NaN                    YNR073C            N3810   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " 1  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Mannitol dehydrogenase DSF1 (EC 1.1.1.67) (Del...  ...           NaN   \n",
       " 1             Mannitol dehydrogenase 2 (EC 1.1.1.67)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  P15559  NQO1_HUMAN  NQO1 DIA4 NMOR1                 NQO1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           DIA4 NMOR1                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  NAD(P)H dehydrogenase [quinone] 1 (EC 1.6.5.2)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 82; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1D4A;1DXO;1GG5;1H66;1H69;1KBO;1KBQ;1QBG;2F1O;3...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q58595  LEU1_METJA  leuA MJ1195                 leuA                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     MJ1195               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  2-isopropylmalate synthase (EC 2.3.3.13) (Alph...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                              Gene Names  \\\n",
       " 0  P25087  ERG6_YEAST  ERG6 ISE1 LIS1 SED6 YML008C YM9571.10C   \n",
       " 1  Q759S7  ERG6_ASHGO                            ERG6 ADR196W   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                 ERG6       ISE1 LIS1 SED6                    YML008C   \n",
       " 1                 ERG6                  NaN                    ADR196W   \n",
       " \n",
       "   Gene Names (ORF)                                           Organism  \\\n",
       " 0       YM9571.10C  Saccharomyces cerevisiae (strain ATCC 204508 /...   \n",
       " 1              NaN  Ashbya gossypii (strain ATCC 10895 / CBS 109.5...   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0         559292  Sterol 24-C-methyltransferase ERG6 (SCMT) (EC ...  ...   \n",
       " 1         284811  Sterol 24-C-methyltransferase (EC 2.1.1.41) (D...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN                                                NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylserine\"; /evidence=\"...     NaN   \n",
       " 1         NaN                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name            Gene Names Gene Names (primary)  \\\n",
       " 0  P13714  LDH_BACSU     ldh lctE BSU03050                  ldh   \n",
       " 1  A7Z152  LDH_BACVZ       ldh RBAM_003290                  ldh   \n",
       " 2  Q9K5Z8  LDH_HALH5       ldh lctE BH3937                  ldh   \n",
       " 3  Q65NP0  LDH_BACLD  ldh BLi00366 BL01703                  ldh   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 lctE                   BSU03050               NaN   \n",
       " 1                  NaN                RBAM_003290               NaN   \n",
       " 2                 lctE                     BH3937               NaN   \n",
       " 3                  NaN           BLi00366 BL01703               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                     Bacillus subtilis (strain 168)         224308   \n",
       " 1  Bacillus velezensis (strain DSM 23117 / BGSC 1...         326423   \n",
       " 2  Halalkalibacterium halodurans (strain ATCC BAA...         272558   \n",
       " 3  Bacillus licheniformis (strain ATCC 14580 / DS...         279010   \n",
       " \n",
       "                                    Protein names  ... Glycosylation  \\\n",
       " 0  L-lactate dehydrogenase (L-LDH) (EC 1.1.1.27)  ...           NaN   \n",
       " 1  L-lactate dehydrogenase (L-LDH) (EC 1.1.1.27)  ...           NaN   \n",
       " 2  L-lactate dehydrogenase (L-LDH) (EC 1.1.1.27)  ...           NaN   \n",
       " 3  L-lactate dehydrogenase (L-LDH) (EC 1.1.1.27)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " 3                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 223; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 1  MOD_RES 224; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 2  MOD_RES 218; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " 3  MOD_RES 224; /note=\"Phosphotyrosine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  3PQD;3PQE;3PQF;  \n",
       " 1             NaN              NaN  \n",
       " 2             NaN              NaN  \n",
       " 3             NaN              NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q64565   AGT2_RAT  Agxt2 Agt2                Agxt2                 Agt2   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Alanine--glyoxylate aminotransferase 2, mitoch...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 55; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..39; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O08651   SERA_RAT      Phgdh                Phgdh                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  D-3-phosphoglycerate dehydrogenase (3-PGDH) (E...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name                              Gene Names  \\\n",
       " 0  P9WJH7  NDK_MYCTU  ndkA ndk Rv2445c MTCY428.01 MTV008.01c   \n",
       " 1  P9WJH6  NDK_MYCTO                         ndkA ndk MT2521   \n",
       " 2  A5U5E1  NDK_MYCTA                            ndk MRA_2471   \n",
       " 3  A1KLE1  NDK_MYCBP                           ndk BCG_2465c   \n",
       " 4  C1AER2  NDK_MYCBT                            ndk JTY_2459   \n",
       " 5  P84283  NDK_MYCBO                 ndk ndkA BQ2027_MB2472C   \n",
       " 6  B2HMG5  NDK_MYCMM                           ndk MMAR_3769   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym) Gene Names (ordered locus)  \\\n",
       " 0                 ndkA                  ndk                    Rv2445c   \n",
       " 1                 ndkA                  ndk                     MT2521   \n",
       " 2                  ndk                  NaN                   MRA_2471   \n",
       " 3                  ndk                  NaN                  BCG_2465c   \n",
       " 4                  ndk                  NaN                   JTY_2459   \n",
       " 5                  ndk                 ndkA             BQ2027_MB2472C   \n",
       " 6                  ndk                  NaN                  MMAR_3769   \n",
       " \n",
       "         Gene Names (ORF)                                           Organism  \\\n",
       " 0  MTCY428.01 MTV008.01c  Mycobacterium tuberculosis (strain ATCC 25618 ...   \n",
       " 1                    NaN  Mycobacterium tuberculosis (strain CDC 1551 / ...   \n",
       " 2                    NaN  Mycobacterium tuberculosis (strain ATCC 25177 ...   \n",
       " 3                    NaN  Mycobacterium bovis (strain BCG / Pasteur 1173P2)   \n",
       " 4                    NaN  Mycobacterium bovis (strain BCG / Tokyo 172 / ...   \n",
       " 5                    NaN  Mycobacterium bovis (strain ATCC BAA-935 / AF2...   \n",
       " 6                    NaN    Mycobacterium marinum (strain ATCC BAA-535 / M)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          83332  Nucleoside diphosphate kinase (NDK) (NDKA) (ND...  ...   \n",
       " 1          83331  Nucleoside diphosphate kinase (NDK) (NDP kinas...  ...   \n",
       " 2         419947  Nucleoside diphosphate kinase (NDK) (NDP kinas...  ...   \n",
       " 3         410289  Nucleoside diphosphate kinase (NDK) (NDP kinas...  ...   \n",
       " 4         561275  Nucleoside diphosphate kinase (NDK) (NDP kinas...  ...   \n",
       " 5         233413  Nucleoside diphosphate kinase (NDK) (NDP kinas...  ...   \n",
       " 6         216594  Nucleoside diphosphate kinase (NDK) (NDP kinas...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0           NaN                  NaN         NaN               NaN     NaN   \n",
       " 1           NaN                  NaN         NaN               NaN     NaN   \n",
       " 2           NaN                  NaN         NaN               NaN     NaN   \n",
       " 3           NaN                  NaN         NaN               NaN     NaN   \n",
       " 4           NaN                  NaN         NaN               NaN     NaN   \n",
       " 5           NaN                  NaN         NaN               NaN     NaN   \n",
       " 6           NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " 5                              NaN         NaN             NaN   \n",
       " 6                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                   PDB  \n",
       " 0             NaN  1K44;4ANC;4AND;4ANE;  \n",
       " 1             NaN                   NaN  \n",
       " 2             NaN                   NaN  \n",
       " 3             NaN                   NaN  \n",
       " 4             NaN                   NaN  \n",
       " 5             NaN                   NaN  \n",
       " 6             NaN                   NaN  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       "     Entry Entry Name             Gene Names Gene Names (primary)  \\\n",
       " 0  P37747  GLF_ECOLI  glf yefE b2036 JW2021                  glf   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 yefE               b2036 JW2021               NaN   \n",
       " \n",
       "                         Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  UDP-galactopyranose mutase (UGM) (EC 5.4.99.9)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1I8T;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P9WKY5   HAT_MYCTU   Rv3423.1                   NaN                   NaN   \n",
       " 1  P9WKY4   HAT_MYCTO   MT3532.1                   NaN                   NaN   \n",
       " 2  P96274  HAT2_MYCTU    Rv0428c                   NaN                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   Rv3423.1               NaN   \n",
       " 1                   MT3532.1               NaN   \n",
       " 2                    Rv0428c               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " 2  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0      Histone acetyltransferase (HAT) (EC 2.3.1.48)  ...           NaN   \n",
       " 1      Histone acetyltransferase (HAT) (EC 2.3.1.48)  ...           NaN   \n",
       " 2  Probable histone acetyltransferase Rv0428c (EC...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " 2             NaN  NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q27783  DRTS_TRYBB         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                   Organism  \\\n",
       " 0                         NaN               NaN  Trypanosoma brucei brucei   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           5702  Bifunctional dihydrofolate reductase-thymidyla...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  3QFX;3RG9;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  O77334  YVH1_PLAF7  YVH1 PF3D7_0309000                 YVH1   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN    PF3D7_0309000   \n",
       " \n",
       "                               Organism  Organism (ID)  \\\n",
       " 0  Plasmodium falciparum (isolate 3D7)          36329   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Dual specificity protein phosphatase YVH1 (PfY...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide PDB  \n",
       " 0             NaN NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "      Entry  Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0   Q9HTJ1  BETB_PSEAE       betB PA5373                 betB   \n",
       " 1   P17202  BADH_SPIOL  BADH SOVF_114720                 BADH   \n",
       " 2   B7V5R4  BETB_PSEA8   betB PLES_57681                 betB   \n",
       " 3   A6VEI4  BETB_PSEA7   betB PSPA7_6158                 betB   \n",
       " 4   B7V5R3  BETA_PSEA8   betA PLES_57671                 betA   \n",
       " 5   A6VEI3  BETA_PSEA7   betA PSPA7_6157                 betA   \n",
       " 6   Q02DY9  BETB_PSEAB   betB PA14_70950                 betB   \n",
       " 7   Q02DZ0  BETA_PSEAB   betA PA14_70940                 betA   \n",
       " 8   Q9HTJ2  BETA_PSEAE       betA PA5372                 betA   \n",
       " 9   A4XPI6  BETB_PSEMY    betB Pmen_0482                 betB   \n",
       " 10  A4XPI5  BETA_PSEMY    betA Pmen_0481                 betA   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN                     PA5373              NaN   \n",
       " 1                    NaN                        NaN      SOVF_114720   \n",
       " 2                    NaN                 PLES_57681              NaN   \n",
       " 3                    NaN                 PSPA7_6158              NaN   \n",
       " 4                    NaN                 PLES_57671              NaN   \n",
       " 5                    NaN                 PSPA7_6157              NaN   \n",
       " 6                    NaN                 PA14_70950              NaN   \n",
       " 7                    NaN                 PA14_70940              NaN   \n",
       " 8                    NaN                     PA5372              NaN   \n",
       " 9                    NaN                  Pmen_0482              NaN   \n",
       " 10                   NaN                  Pmen_0481              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0   Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 1                         Spinacia oleracea (Spinach)           3562   \n",
       " 2              Pseudomonas aeruginosa (strain LESB58)         557722   \n",
       " 3                 Pseudomonas aeruginosa (strain PA7)         381754   \n",
       " 4              Pseudomonas aeruginosa (strain LESB58)         557722   \n",
       " 5                 Pseudomonas aeruginosa (strain PA7)         381754   \n",
       " 6          Pseudomonas aeruginosa (strain UCBPP-PA14)         208963   \n",
       " 7          Pseudomonas aeruginosa (strain UCBPP-PA14)         208963   \n",
       " 8   Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 9                  Pseudomonas mendocina (strain ymp)         399739   \n",
       " 10                 Pseudomonas mendocina (strain ymp)         399739   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   NAD/NADP-dependent betaine aldehyde dehydrogen...  ...           NaN   \n",
       " 1   Aminoaldehyde dehydrogenase BADH (EC 1.2.1.-) ...  ...           NaN   \n",
       " 2   Betaine aldehyde dehydrogenase (BADH) (EC 1.2....  ...           NaN   \n",
       " 3   Betaine aldehyde dehydrogenase (BADH) (EC 1.2....  ...           NaN   \n",
       " 4   Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " 5   Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " 6   Betaine aldehyde dehydrogenase (BADH) (EC 1.2....  ...           NaN   \n",
       " 7   Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " 8   Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " 9   Betaine aldehyde dehydrogenase (BADH) (EC 1.2....  ...           NaN   \n",
       " 10  Oxygen-dependent choline dehydrogenase (CDH) (...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  \\\n",
       " 0                   NaN         NaN   \n",
       " 1                   NaN         NaN   \n",
       " 2                   NaN         NaN   \n",
       " 3                   NaN         NaN   \n",
       " 4                   NaN         NaN   \n",
       " 5                   NaN         NaN   \n",
       " 6                   NaN         NaN   \n",
       " 7                   NaN         NaN   \n",
       " 8                   NaN         NaN   \n",
       " 9                   NaN         NaN   \n",
       " 10                  NaN         NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 286; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 1          MOD_RES 8; /note=\"Blocked amino end (Arg)\"     NaN   \n",
       " 2   MOD_RES 286; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 3   MOD_RES 286; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 4                                                 NaN     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6   MOD_RES 286; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 7                                                 NaN     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9   MOD_RES 286; /note=\"Cysteine sulfenic acid (-S...     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide                                       PDB  \n",
       " 0              NaN  2WME;2WOX;2XDR;3ZQA;4CAZ;4CBB;6BJP;6BPG;  \n",
       " 1              NaN                      4A0M;4V37;4V3F;5A2D;  \n",
       " 2              NaN                                       NaN  \n",
       " 3              NaN                                       NaN  \n",
       " 4              NaN                                       NaN  \n",
       " 5              NaN                                       NaN  \n",
       " 6              NaN                                       NaN  \n",
       " 7              NaN                                       NaN  \n",
       " 8              NaN                                       NaN  \n",
       " 9              NaN                                       NaN  \n",
       " 10             NaN                                       NaN  \n",
       " \n",
       " [11 rows x 110 columns],\n",
       "     Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  Q5F3N1  PIMT_CHICK  PCMT1 RCJMB04_11o11                PCMT1   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN    RCJMB04_11o11   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Gallus gallus (Chicken)           9031   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Protein-L-isoaspartate(D-aspartate) O-methyltr...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry   Entry Name                     Gene Names Gene Names (primary)  \\\n",
       " 0   P51003  PAPOA_HUMAN                     PAPOLA PAP               PAPOLA   \n",
       " 1   Q9BWT3  PAPOG_HUMAN               PAPOLG PAP2 PAPG               PAPOLG   \n",
       " 2   Q8NEK8  TET5D_HUMAN                  TENT5D FAM46D               TENT5D   \n",
       " 3   Q9H6E5  STPAP_HUMAN                     TUT1 RBM21                 TUT1   \n",
       " 4   Q9NVV4  PAPD1_HUMAN                    MTPAP PAPD1                MTPAP   \n",
       " 5   Q5VWP2  TET5C_HUMAN                  TENT5C FAM46C               TENT5C   \n",
       " 6   Q6PIY7   GLD2_HUMAN               TENT2 GLD2 PAPD4                TENT2   \n",
       " 7   Q8NDF8  PAPD5_HUMAN  TENT4B GLD4 PAPD5 TRF4-2 TUT3               TENT4B   \n",
       " 8   Q96IP4  TET5A_HUMAN    TENT5A C6orf37 FAM46A XTP11               TENT5A   \n",
       " 9   Q5XG87  PAPD7_HUMAN         TENT4A PAPD7 POLS TRF4               TENT4A   \n",
       " 10  Q96A09  TET5B_HUMAN                  TENT5B FAM46B               TENT5B   \n",
       " 11  Q9NRJ5  PAPOB_HUMAN                    PAPOLB PAPT               PAPOLB   \n",
       " 12  P13888    POLN_RRVT                            NaN                  NaN   \n",
       " 13  P13886   POLN_ONNVG                            NaN                  NaN   \n",
       " 14  O90370   POLN_ONNVI                            NaN                  NaN   \n",
       " 15  P87515     POLN_BFV                            NaN                  NaN   \n",
       " 16  Q306W6   POLN_EEEV1                            NaN                  NaN   \n",
       " 17  Q306W8   POLN_EEEV8                            NaN                  NaN   \n",
       " 18  Q4QXJ8   POLN_EEEVF                            NaN                  NaN   \n",
       " 19  Q5Y389    POLN_GETV                            NaN                  NaN   \n",
       " 20  P27283   POLN_SINDO                            NaN                  NaN   \n",
       " 21  Q8QZ73   POLN_MAYAB                            NaN                  NaN   \n",
       " 22  Q8V294   POLN_EEVVC                            NaN                  NaN   \n",
       " 23  Q9WJC7   POLN_EEVVM                            NaN                  NaN   \n",
       " 24  P36327   POLN_EEVV3                            NaN                  NaN   \n",
       " 25  Q9JGL0    POLN_SAGV                            NaN                  NaN   \n",
       " 26  Q8JUX6   POLN_CHIKS                            NaN                  NaN   \n",
       " 27  O90368   POLN_ONNVS                            NaN                  NaN   \n",
       " 28  P13887    POLN_RRVN                            NaN                  NaN   \n",
       " 29  P13896    POLN_WEEV                            NaN                  NaN   \n",
       " 30  Q5XXP4   POLN_CHIK3                            NaN                  NaN   \n",
       " 31  P27282   POLN_EEVVT                            NaN                  NaN   \n",
       " 32  P36328   POLN_EEVVP                            NaN                  NaN   \n",
       " 33  P08411     POLN_SFV                            NaN                  NaN   \n",
       " 34  P03317   POLN_SINDV                            NaN                  NaN   \n",
       " 35  O57184   PAP1_VACCA  PAPL MVA048L ACAM3000_MVA_048                 PAPL   \n",
       " 36  P21079   PAP1_VACCC                       PAPL E1L                 PAPL   \n",
       " 37  Q9JFD9   PAP1_VACCT                      PAPL TE1L                 PAPL   \n",
       " 38  P0DOM1   PAP1_VAR67                   PAPL C1L E1L                 PAPL   \n",
       " 39  Q98199    PAP1_MCV1                    PAPL MC031L                 PAPL   \n",
       " 40  Q9DHT0    PAP1_YLDV                       PAPL 32L                 PAPL   \n",
       " 41  P0DOM2    PAP1_VARV                   PAPL C1L E1L                 PAPL   \n",
       " 42  Q2F8F7   PAP1_ORFN2                           PAPL                 PAPL   \n",
       " 43  Q0GP30    PAP1_HSPV                   PAPL HSPV058                 PAPL   \n",
       " 44  Q6TUY0   PAP1_YMTV5                       PAPL 32L                 PAPL   \n",
       " 45  Q8QMZ9   PAP1_CWPXB                   PAPL CPXV067                 PAPL   \n",
       " 46  Q80E12   PAP1_CWPXG                       PAPL F1L                 PAPL   \n",
       " \n",
       "       Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                      PAP                        NaN              NaN   \n",
       " 1                PAP2 PAPG                        NaN              NaN   \n",
       " 2                   FAM46D                        NaN              NaN   \n",
       " 3                    RBM21                        NaN              NaN   \n",
       " 4                    PAPD1                        NaN              NaN   \n",
       " 5                   FAM46C                        NaN              NaN   \n",
       " 6               GLD2 PAPD4                        NaN              NaN   \n",
       " 7   GLD4 PAPD5 TRF4-2 TUT3                        NaN              NaN   \n",
       " 8     C6orf37 FAM46A XTP11                        NaN              NaN   \n",
       " 9          PAPD7 POLS TRF4                        NaN              NaN   \n",
       " 10                  FAM46B                        NaN              NaN   \n",
       " 11                    PAPT                        NaN              NaN   \n",
       " 12                     NaN                        NaN              NaN   \n",
       " 13                     NaN                        NaN              NaN   \n",
       " 14                     NaN                        NaN              NaN   \n",
       " 15                     NaN                        NaN              NaN   \n",
       " 16                     NaN                        NaN              NaN   \n",
       " 17                     NaN                        NaN              NaN   \n",
       " 18                     NaN                        NaN              NaN   \n",
       " 19                     NaN                        NaN              NaN   \n",
       " 20                     NaN                        NaN              NaN   \n",
       " 21                     NaN                        NaN              NaN   \n",
       " 22                     NaN                        NaN              NaN   \n",
       " 23                     NaN                        NaN              NaN   \n",
       " 24                     NaN                        NaN              NaN   \n",
       " 25                     NaN                        NaN              NaN   \n",
       " 26                     NaN                        NaN              NaN   \n",
       " 27                     NaN                        NaN              NaN   \n",
       " 28                     NaN                        NaN              NaN   \n",
       " 29                     NaN                        NaN              NaN   \n",
       " 30                     NaN                        NaN              NaN   \n",
       " 31                     NaN                        NaN              NaN   \n",
       " 32                     NaN                        NaN              NaN   \n",
       " 33                     NaN                        NaN              NaN   \n",
       " 34                     NaN                        NaN              NaN   \n",
       " 35                     NaN   MVA048L ACAM3000_MVA_048              NaN   \n",
       " 36                     NaN                        NaN              E1L   \n",
       " 37                     NaN                        NaN             TE1L   \n",
       " 38                     NaN                        NaN          C1L E1L   \n",
       " 39                     NaN                     MC031L              NaN   \n",
       " 40                     NaN                        32L              NaN   \n",
       " 41                     NaN                        NaN          C1L E1L   \n",
       " 42                     NaN                        NaN              NaN   \n",
       " 43                     NaN                    HSPV058              NaN   \n",
       " 44                     NaN                        32L              NaN   \n",
       " 45                     NaN                    CPXV067              NaN   \n",
       " 46                     NaN                        NaN              F1L   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                                Homo sapiens (Human)           9606   \n",
       " 1                                Homo sapiens (Human)           9606   \n",
       " 2                                Homo sapiens (Human)           9606   \n",
       " 3                                Homo sapiens (Human)           9606   \n",
       " 4                                Homo sapiens (Human)           9606   \n",
       " 5                                Homo sapiens (Human)           9606   \n",
       " 6                                Homo sapiens (Human)           9606   \n",
       " 7                                Homo sapiens (Human)           9606   \n",
       " 8                                Homo sapiens (Human)           9606   \n",
       " 9                                Homo sapiens (Human)           9606   \n",
       " 10                               Homo sapiens (Human)           9606   \n",
       " 11                               Homo sapiens (Human)           9606   \n",
       " 12                Ross river virus (strain T48) (RRV)          11032   \n",
       " 13           O'nyong-nyong virus (strain Gulu) (ONNV)          11028   \n",
       " 14  O'nyong-nyong virus (strain Igbo Ora) (ONNV) (...          79899   \n",
       " 15                          Barmah forest virus (BFV)          11020   \n",
       " 16  Eastern equine encephalitis virus (strain PE-0...         374596   \n",
       " 17  Eastern equine encephalitis virus (strain PE-3...         374597   \n",
       " 18  Eastern equine encephalitis virus (strain Flor...         374598   \n",
       " 19                                 Getah virus (GETV)          59300   \n",
       " 20  Sindbis virus subtype Ockelbo (strain Edsbyn 8...          31699   \n",
       " 21                Mayaro virus (strain Brazil) (MAYV)         374990   \n",
       " 22  Venezuelan equine encephalitis virus (strain C...         376610   \n",
       " 23  Venezuelan equine encephalitis virus (strain M...          36384   \n",
       " 24  Venezuelan equine encephalitis virus (strain 3...          36382   \n",
       " 25                              Sagiyama virus (SAGV)          59303   \n",
       " 26  Chikungunya virus (strain S27-African prototyp...         371094   \n",
       " 27          O'nyong-nyong virus (strain SG650) (ONNV)         374989   \n",
       " 28             Ross river virus (strain NB5092) (RRV)          11031   \n",
       " 29           Western equine encephalitis virus (WEEV)          11039   \n",
       " 30           Chikungunya virus (strain 37997) (CHIKV)         371095   \n",
       " 31  Venezuelan equine encephalitis virus (strain T...          11038   \n",
       " 32  Venezuelan equine encephalitis virus (strain P...          36385   \n",
       " 33                         Semliki forest virus (SFV)          11033   \n",
       " 34                               Sindbis virus (SINV)          11034   \n",
       " 35              Vaccinia virus (strain Ankara) (VACV)         126794   \n",
       " 36          Vaccinia virus (strain Copenhagen) (VACV)          10249   \n",
       " 37            Vaccinia virus (strain Tian Tan) (VACV)          10253   \n",
       " 38  Variola virus (isolate Human/India/Ind3/1967) ...         587200   \n",
       " 39  Molluscum contagiosum virus subtype 1 (MOCV) (...          10280   \n",
       " 40                     Yaba-like disease virus (YLDV)         132475   \n",
       " 41                                      Variola virus          10255   \n",
       " 42                   Orf virus (strain NZ2) (OV NZ-2)          10259   \n",
       " 43                              Horsepox virus (HSPV)         397342   \n",
       " 44      Yaba monkey tumor virus (strain VR587) (YMTV)         928314   \n",
       " 45           Cowpox virus (strain Brighton Red) (CPV)         265872   \n",
       " 46       Cowpox virus (strain GRI-90 / Grishak) (CPV)         265871   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Poly(A) polymerase alpha (PAP-alpha) (EC 2.7.7...  ...           NaN   \n",
       " 1   Poly(A) polymerase gamma (PAP-gamma) (EC 2.7.7...  ...           NaN   \n",
       " 2   Terminal nucleotidyltransferase 5D (EC 2.7.7.1...  ...           NaN   \n",
       " 3   Speckle targeted PIP5K1A-regulated poly(A) pol...  ...           NaN   \n",
       " 4   Poly(A) RNA polymerase, mitochondrial (PAP) (E...  ...           NaN   \n",
       " 5   Terminal nucleotidyltransferase 5C (EC 2.7.7.1...  ...           NaN   \n",
       " 6   Poly(A) RNA polymerase GLD2 (hGLD-2) (EC 2.7.7...  ...           NaN   \n",
       " 7   Terminal nucleotidyltransferase 4B (Non-canoni...  ...           NaN   \n",
       " 8   Terminal nucleotidyltransferase 5A (EC 2.7.7.1...  ...           NaN   \n",
       " 9   Terminal nucleotidyltransferase 4A (DNA polyme...  ...           NaN   \n",
       " 10  Terminal nucleotidyltransferase 5B (EC 2.7.7.1...  ...           NaN   \n",
       " 11  Poly(A) polymerase beta (PAP-beta) (EC 2.7.7.1...  ...           NaN   \n",
       " 12  Polyprotein nsP1234 (P1234) (Non-structural po...  ...           NaN   \n",
       " 13  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 14  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 15  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 16  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 17  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 18  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 19  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 20  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 21  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 22  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 23  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 24  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 25  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 26  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 27  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 28  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 29  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 30  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 31  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 32  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 33  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 34  Polyprotein P1234 (P1234) (Non-structural poly...  ...           NaN   \n",
       " 35  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 36  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 37  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 38  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 39  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 40  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 41  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 42  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 43  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 44  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 45  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " 46  Poly(A) polymerase catalytic subunit (EC 2.7.7...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1                                                 NaN   \n",
       " 2                                                 NaN   \n",
       " 3                                                 NaN   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7                                                 NaN   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " 13                                                NaN   \n",
       " 14                                                NaN   \n",
       " 15                                                NaN   \n",
       " 16                                                NaN   \n",
       " 17                                                NaN   \n",
       " 18                                                NaN   \n",
       " 19                                                NaN   \n",
       " 20                                                NaN   \n",
       " 21                                                NaN   \n",
       " 22                                                NaN   \n",
       " 23                                                NaN   \n",
       " 24                                                NaN   \n",
       " 25                                                NaN   \n",
       " 26                                                NaN   \n",
       " 27                                                NaN   \n",
       " 28                                                NaN   \n",
       " 29                                                NaN   \n",
       " 30                                                NaN   \n",
       " 31                                                NaN   \n",
       " 32                                                NaN   \n",
       " 33                                                NaN   \n",
       " 34                                                NaN   \n",
       " 35                                                NaN   \n",
       " 36                                                NaN   \n",
       " 37                                                NaN   \n",
       " 38                                                NaN   \n",
       " 39                                                NaN   \n",
       " 40                                                NaN   \n",
       " 41                                                NaN   \n",
       " 42                                                NaN   \n",
       " 43                                                NaN   \n",
       " 44                                                NaN   \n",
       " 45                                                NaN   \n",
       " 46                                                NaN   \n",
       " \n",
       "                                            Lipidation  \\\n",
       " 0                                                 NaN   \n",
       " 1                                                 NaN   \n",
       " 2                                                 NaN   \n",
       " 3                                                 NaN   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7                                                 NaN   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 12                                                NaN   \n",
       " 13  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 14  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 15  LIPID 415; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 16  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 17  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 18  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 19  LIPID 416; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 20  LIPID 420; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 21  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 22  LIPID 419; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 23  LIPID 419; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 24  LIPID 419; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 25  LIPID 416; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 26  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 27  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 28  LIPID 415; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 29  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 30  LIPID 417; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 31  LIPID 419; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 32  LIPID 419; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 33  LIPID 418; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 34  LIPID 420; /note=\"S-palmitoyl cysteine; by hos...   \n",
       " 35                                                NaN   \n",
       " 36                                                NaN   \n",
       " 37                                                NaN   \n",
       " 38                                                NaN   \n",
       " 39                                                NaN   \n",
       " 40                                                NaN   \n",
       " 41                                                NaN   \n",
       " 42                                                NaN   \n",
       " 43                                                NaN   \n",
       " 44                                                NaN   \n",
       " 45                                                NaN   \n",
       " 46                                                NaN   \n",
       " \n",
       "                                      Modified residue Peptide  \\\n",
       " 0   MOD_RES 10; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1   MOD_RES 2; /note=\"N6-acetyllysine\"; /evidence=...     NaN   \n",
       " 2                                                 NaN     NaN   \n",
       " 3   MOD_RES 750; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 4   MOD_RES 90; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       " 5                                                 NaN     NaN   \n",
       " 6   MOD_RES 62; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 7   MOD_RES 484; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 8                                                 NaN     NaN   \n",
       " 9                                                 NaN     NaN   \n",
       " 10                                                NaN     NaN   \n",
       " 11                                                NaN     NaN   \n",
       " 12  MOD_RES 344; /note=\"Phosphothreonine; by host\"...     NaN   \n",
       " 13                                                NaN     NaN   \n",
       " 14                                                NaN     NaN   \n",
       " 15                                                NaN     NaN   \n",
       " 16                                                NaN     NaN   \n",
       " 17                                                NaN     NaN   \n",
       " 18                                                NaN     NaN   \n",
       " 19                                                NaN     NaN   \n",
       " 20                                                NaN     NaN   \n",
       " 21                                                NaN     NaN   \n",
       " 22                                                NaN     NaN   \n",
       " 23                                                NaN     NaN   \n",
       " 24                                                NaN     NaN   \n",
       " 25                                                NaN     NaN   \n",
       " 26                                                NaN     NaN   \n",
       " 27                                                NaN     NaN   \n",
       " 28  MOD_RES 1675; /note=\"Phosphothreonine; by host...     NaN   \n",
       " 29                                                NaN     NaN   \n",
       " 30                                                NaN     NaN   \n",
       " 31                                                NaN     NaN   \n",
       " 32                                                NaN     NaN   \n",
       " 33  MOD_RES 1680; /note=\"Phosphothreonine; by host...     NaN   \n",
       " 34                                                NaN     NaN   \n",
       " 35                                                NaN     NaN   \n",
       " 36                                                NaN     NaN   \n",
       " 37                                                NaN     NaN   \n",
       " 38                                                NaN     NaN   \n",
       " 39                                                NaN     NaN   \n",
       " 40                                                NaN     NaN   \n",
       " 41                                                NaN     NaN   \n",
       " 42                                                NaN     NaN   \n",
       " 43                                                NaN     NaN   \n",
       " 44                                                NaN     NaN   \n",
       " 45                                                NaN     NaN   \n",
       " 46                                                NaN     NaN   \n",
       " \n",
       "                       Post-translational modification  Propeptide  \\\n",
       " 0   PTM: Polysumoylated. Varying sumolyation depen...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2                                                 NaN         NaN   \n",
       " 3   PTM: Phosphorylated by CK1 in the proline-rich...         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12  PTM: Polyprotein P1234: Specific enzymatic cle...         NaN   \n",
       " 13  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 14  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 15  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 16  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 17  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 18  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 19  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 20  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 21  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 22  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 23  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 24  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 25  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 26  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 27  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 28  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 29  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 30  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 31  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 32  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 33  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 34  PTM: [Polyprotein P1234]: Specific enzymatic c...         NaN   \n",
       " 35                                                NaN         NaN   \n",
       " 36                                                NaN         NaN   \n",
       " 37                                                NaN         NaN   \n",
       " 38                                                NaN         NaN   \n",
       " 39                                                NaN         NaN   \n",
       " 40                                                NaN         NaN   \n",
       " 41                                                NaN         NaN   \n",
       " 42                                                NaN         NaN   \n",
       " 43                                                NaN         NaN   \n",
       " 44                                                NaN         NaN   \n",
       " 45                                                NaN         NaN   \n",
       " 46                                                NaN         NaN   \n",
       " \n",
       "     Signal peptide                                    Transit peptide  \\\n",
       " 0              NaN                                                NaN   \n",
       " 1              NaN                                                NaN   \n",
       " 2              NaN                                                NaN   \n",
       " 3              NaN                                                NaN   \n",
       " 4              NaN  TRANSIT 1..37; /note=\"Mitochondrion\"; /evidenc...   \n",
       " 5              NaN                                                NaN   \n",
       " 6              NaN                                                NaN   \n",
       " 7              NaN                                                NaN   \n",
       " 8              NaN                                                NaN   \n",
       " 9              NaN                                                NaN   \n",
       " 10             NaN                                                NaN   \n",
       " 11             NaN                                                NaN   \n",
       " 12             NaN                                                NaN   \n",
       " 13             NaN                                                NaN   \n",
       " 14             NaN                                                NaN   \n",
       " 15             NaN                                                NaN   \n",
       " 16             NaN                                                NaN   \n",
       " 17             NaN                                                NaN   \n",
       " 18             NaN                                                NaN   \n",
       " 19             NaN                                                NaN   \n",
       " 20             NaN                                                NaN   \n",
       " 21             NaN                                                NaN   \n",
       " 22             NaN                                                NaN   \n",
       " 23             NaN                                                NaN   \n",
       " 24             NaN                                                NaN   \n",
       " 25             NaN                                                NaN   \n",
       " 26             NaN                                                NaN   \n",
       " 27             NaN                                                NaN   \n",
       " 28             NaN                                                NaN   \n",
       " 29             NaN                                                NaN   \n",
       " 30             NaN                                                NaN   \n",
       " 31             NaN                                                NaN   \n",
       " 32             NaN                                                NaN   \n",
       " 33             NaN                                                NaN   \n",
       " 34             NaN                                                NaN   \n",
       " 35             NaN                                                NaN   \n",
       " 36             NaN                                                NaN   \n",
       " 37             NaN                                                NaN   \n",
       " 38             NaN                                                NaN   \n",
       " 39             NaN                                                NaN   \n",
       " 40             NaN                                                NaN   \n",
       " 41             NaN                                                NaN   \n",
       " 42             NaN                                                NaN   \n",
       " 43             NaN                                                NaN   \n",
       " 44             NaN                                                NaN   \n",
       " 45             NaN                                                NaN   \n",
       " 46             NaN                                                NaN   \n",
       " \n",
       "                                                   PDB  \n",
       " 0                                                 NaN  \n",
       " 1                                               4LT6;  \n",
       " 2                                                 NaN  \n",
       " 3                 2E5G;5WU1;5WU2;5WU3;5WU4;5WU5;5WU6;  \n",
       " 4                                               3PQ1;  \n",
       " 5                                6W36;6W38;6W3I;6W3J;  \n",
       " 6                                                 NaN  \n",
       " 7                                                 NaN  \n",
       " 8                                                 NaN  \n",
       " 9                                                 NaN  \n",
       " 10                                                NaN  \n",
       " 11                                                NaN  \n",
       " 12                                              7F0S;  \n",
       " 13                                                NaN  \n",
       " 14                                                NaN  \n",
       " 15                                                NaN  \n",
       " 16                                                NaN  \n",
       " 17                                                NaN  \n",
       " 18                                                NaN  \n",
       " 19                     6QZU;6R0F;6R0G;6R0P;6R0R;6R0T;  \n",
       " 20                                                NaN  \n",
       " 21                                              5IQ5;  \n",
       " 22                                                NaN  \n",
       " 23                                                NaN  \n",
       " 24                                                NaN  \n",
       " 25                                                NaN  \n",
       " 26  3GPG;3GPO;3GPQ;3TRK;4TU0;5I22;6VUQ;6W7H;6W8K;6...  \n",
       " 27                                                NaN  \n",
       " 28                                                NaN  \n",
       " 29                                                NaN  \n",
       " 30                                              6W0T;  \n",
       " 31                               2HWK;5EZQ;5EZS;6BCM;  \n",
       " 32                               3GQE;3GQO;5ISN;5MQX;  \n",
       " 33                                    1FW5;5DRV;5FW5;  \n",
       " 34                                    4GUA;7VB4;7VW5;  \n",
       " 35                                                NaN  \n",
       " 36                                                NaN  \n",
       " 37                                                NaN  \n",
       " 38                                                NaN  \n",
       " 39                                                NaN  \n",
       " 40                                                NaN  \n",
       " 41                                                NaN  \n",
       " 42                                                NaN  \n",
       " 43                                                NaN  \n",
       " 44                                                NaN  \n",
       " 45                                                NaN  \n",
       " 46                                                NaN  \n",
       " \n",
       " [47 rows x 110 columns],\n",
       "     Entry   Entry Name        Gene Names Gene Names (primary)  \\\n",
       " 0  Q5SWU9  ACACA_MOUSE  Acaca Acac Gm738                Acaca   \n",
       " 1  E9Q4Z2  ACACB_MOUSE   Acacb Acc2 Accb                Acacb   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           Acac Gm738                         NaN               NaN   \n",
       " 1            Acc2 Accb                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Mus musculus (Mouse)          10090   \n",
       " 1  Mus musculus (Mouse)          10090   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Acetyl-CoA carboxylase 1 (ACC1) (EC 6.4.1.2) (...  ...           NaN   \n",
       " 1   Acetyl-CoA carboxylase 2 (EC 6.4.1.2) (ACC-beta)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " 1  MOD_RES 35; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylation on Ser-1262 is required f...         NaN   \n",
       " 1  PTM: The biotin cofactor is covalently attache...         NaN   \n",
       " \n",
       "    Signal peptide                                    Transit peptide  PDB  \n",
       " 0             NaN                                                NaN  NaN  \n",
       " 1             NaN  TRANSIT 1..?; /note=\"Mitochondrion\"; /evidence...  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  P9WNE1  FGD_MYCTU  fgd1 fgd Rv0407                 fgd1   \n",
       " 1  P9WNE0  FGD_MYCTO  fgd1 fgd MT0420                 fgd1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                  fgd                     Rv0407               NaN   \n",
       " 1                  fgd                     MT0420               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  F420-dependent glucose-6-phosphate dehydrogena...  ...           NaN   \n",
       " 1  F420-dependent glucose-6-phosphate dehydrogena...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  3B4Y;3C8N;  \n",
       " 1             NaN         NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry   Entry Name                     Gene Names Gene Names (primary)  \\\n",
       " 0  Q8NB78  KDM1B_HUMAN       KDM1B AOF1 C6orf193 LSD2                KDM1B   \n",
       " 1  O60341  KDM1A_HUMAN  KDM1A AOF2 KDM1 KIAA0601 LSD1                KDM1A   \n",
       " \n",
       "       Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0       AOF1 C6orf193 LSD2                         NaN               NaN   \n",
       " 1  AOF2 KDM1 KIAA0601 LSD1                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Lysine-specific histone demethylase 2 (EC 1.14...  ...           NaN   \n",
       " 1  Lysine-specific histone demethylase 1A (EC 1.1...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 13; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1  MOD_RES 59; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  PTM: Polyubiquitinated by JADE2; which leads t...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " 1             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  4FWE;4FWF;4FWJ;4GU0;4GU1;4GUR;4GUS;4GUT;4GUU;4...  \n",
       " 1  2COM;2DW4;2EJR;2H94;2HKO;2IW5;2L3D;2UXN;2UXX;2...  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P11497  ACACA_RAT  Acaca Acac                Acaca                 Acac   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Acetyl-CoA carboxylase 1 (ACC1) (EC 6.4.1.2) (...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: The N-terminus is blocked.; PTM: Phosphor...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                 Gene Names Gene Names (primary)  \\\n",
       " 0  O82567  AAPT1_ARATH    AAPT1 At1g13560 F13B4.5                AAPT1   \n",
       " 1  O82568  AAPT2_ARATH  AAPT2 At3g25585 MWL2 T5M7                AAPT2   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                  At1g13560          F13B4.5   \n",
       " 1                   NaN                  At3g25585        MWL2 T5M7   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Choline/ethanolaminephosphotransferase 1 (EC 2...  ...           NaN   \n",
       " 1  Choline/ethanolaminephosphotransferase 2 (EC 2...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name      Gene Names Gene Names (primary)  \\\n",
       " 0  Q64057  AL7A1_RAT  Aldh7a1 Ald7a1              Aldh7a1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0               Ald7a1                         NaN               NaN   \n",
       " \n",
       "                   Organism  Organism (ID)  \\\n",
       " 0  Rattus norvegicus (Rat)          10116   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Alpha-aminoadipic semialdehyde dehydrogenase (...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 86; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..26; /note=\"Mitochondrion\"; /evidenc...  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  P26276  ALGC_PSEAE      algC PA5322                 algC   \n",
       " 1  Q02E40  ALGC_PSEAB  algC PA14_70270                 algC   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                     PA5322               NaN   \n",
       " 1                   NaN                 PA14_70270               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Pseudomonas aeruginosa (strain ATCC 15692 / DS...         208964   \n",
       " 1         Pseudomonas aeruginosa (strain UCBPP-PA14)         208963   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphomannomutase/phosphoglucomutase (PMM / P...  ...           NaN   \n",
       " 1  Phosphomannomutase/phosphoglucomutase (PMM / P...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 108; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 108; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  1K2Y;1K35;1P5D;1P5G;1PCJ;1PCM;2FKF;2FKM;2H4L;2...  \n",
       " 1             NaN                                                NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name                  Gene Names Gene Names (primary)  \\\n",
       " 0  O95544   NADK_HUMAN                        NADK                 NADK   \n",
       " 1  Q4G0N4  NAKD2_HUMAN  NADK2 C5orf33 MNADK NADKD1                NADK2   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN               NaN   \n",
       " 1  C5orf33 MNADK NADKD1                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  NAD kinase (EC 2.7.1.23) (Poly(P)/ATP NAD kinase)  ...           NaN   \n",
       " 1  NAD kinase 2, mitochondrial (EC 2.7.1.23) (Mit...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 46; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 1  MOD_RES 76; /note=\"N6-acetyllysine; alternate\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  \\\n",
       " 0                                                NaN   \n",
       " 1  TRANSIT 1..62; /note=\"Mitochondrion\"; /evidenc...   \n",
       " \n",
       "                          PDB  \n",
       " 0                      3PFN;  \n",
       " 1  7N29;7R4J;7R4K;7R4L;7R4M;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q29550   EST1_PIG         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)          Organism  \\\n",
       " 0                         NaN               NaN  Sus scrofa (Pig)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9823  Liver carboxylesterase (EC 3.1.1.1) (Proline-b...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 80; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 379; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide    PDB  \n",
       " 0  SIGNAL 1..18; /evidence=\"ECO:0000250\"             NaN  5FV4;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P49088   ASNS_RAT       Asns                 Asns                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Asparagine synthetase [glutamine-hydrolyzing] ...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 385; /note=\"N6-acetyllysine\"; /evidenc...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P12527   LOX5_RAT      Alox5                Alox5                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Polyunsaturated fatty acid 5-lipoxygenase (EC ...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 271; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Serine phosphorylation by MAPKAPK2 is sti...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   P36553  HEM6_ECOLI             hemF b2436 JW2429                 hemF   \n",
       " 1   Q8XBI4  HEM6_ECO57            hemF Z3701 ECs3307                 hemF   \n",
       " 2   B6I512  HEM6_ECOSE                hemF ECSE_2726                 hemF   \n",
       " 3   B1XAA7  HEM6_ECODH             hemF ECDH10B_2601                 hemF   \n",
       " 4   B7LCI0  HEM6_ECO55             hemF EC55989_2725                 hemF   \n",
       " 5   B7MY86  HEM6_ECO81               hemF ECED1_2878                 hemF   \n",
       " 6   B7M6U5  HEM6_ECO8A              hemF ECIAI1_2493                 hemF   \n",
       " 7   A1ADV0  HEM6_ECOK1  hemF Ecok1_23460 APECO1_4112                 hemF   \n",
       " 8   C4ZX09  HEM6_ECOBW                 hemF BWG_2198                 hemF   \n",
       " 9   A8A2T4  HEM6_ECOHS               hemF EcHS_A2573                 hemF   \n",
       " 10  B5YZY1  HEM6_ECO5E            hemF ECH74115_3666                 hemF   \n",
       " 11  A7ZPN6  HEM6_ECO24           hemF EcE24377A_2722                 hemF   \n",
       " 12  B7MHT7  HEM6_ECO45               hemF ECS88_2624                 hemF   \n",
       " 13  B1LMN0  HEM6_ECOSM             hemF EcSMS35_2591                 hemF   \n",
       " 14  B7NPX2  HEM6_ECO7I             hemF ECIAI39_2581                 hemF   \n",
       " 15  B7UGD5  HEM6_ECO27              hemF E2348C_2620                 hemF   \n",
       " 16  B7N626  HEM6_ECOLU               hemF ECUMN_2757                 hemF   \n",
       " 17  Q0TF33  HEM6_ECOL5                 hemF ECP_2457                 hemF   \n",
       " 18  Q8FFA3  HEM6_ECOL6                    hemF c2970                 hemF   \n",
       " 19  B1IWM6  HEM6_ECOLC               hemF EcolC_1243                 hemF   \n",
       " 20  B7LKJ9  HEM6_ESCF3                hemF EFER_0736                 hemF   \n",
       " 21  Q83QN1  HEM6_SHIFL             hemF SF2489 S2637                 hemF   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN               b2436 JW2429              NaN   \n",
       " 1                    NaN              Z3701 ECs3307              NaN   \n",
       " 2                    NaN                  ECSE_2726              NaN   \n",
       " 3                    NaN               ECDH10B_2601              NaN   \n",
       " 4                    NaN               EC55989_2725              NaN   \n",
       " 5                    NaN                 ECED1_2878              NaN   \n",
       " 6                    NaN                ECIAI1_2493              NaN   \n",
       " 7                    NaN                Ecok1_23460      APECO1_4112   \n",
       " 8                    NaN                   BWG_2198              NaN   \n",
       " 9                    NaN                 EcHS_A2573              NaN   \n",
       " 10                   NaN              ECH74115_3666              NaN   \n",
       " 11                   NaN             EcE24377A_2722              NaN   \n",
       " 12                   NaN                 ECS88_2624              NaN   \n",
       " 13                   NaN               EcSMS35_2591              NaN   \n",
       " 14                   NaN               ECIAI39_2581              NaN   \n",
       " 15                   NaN                E2348C_2620              NaN   \n",
       " 16                   NaN                 ECUMN_2757              NaN   \n",
       " 17                   NaN                   ECP_2457              NaN   \n",
       " 18                   NaN                      c2970              NaN   \n",
       " 19                   NaN                 EcolC_1243              NaN   \n",
       " 20                   NaN                  EFER_0736              NaN   \n",
       " 21                   NaN               SF2489 S2637              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                            Escherichia coli O157:H7          83334   \n",
       " 2                      Escherichia coli (strain SE11)         409438   \n",
       " 3               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 4              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 5                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 6                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 7                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 8     Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 9                  Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 10    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 11  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 12       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 13          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 14      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 15  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 16  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 17    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 18  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 19  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 20  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 21                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 1   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 2   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 3   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 4   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 5   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 6   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 7   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 8   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 9   Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 10  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 11  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 12  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 13  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 14  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 15  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 16  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 17  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 18  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 19  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 20  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " 21  Oxygen-dependent coproporphyrinogen-III oxidas...  ...           NaN   \n",
       " \n",
       "    Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                   NaN         NaN               NaN     NaN   \n",
       " 1                   NaN         NaN               NaN     NaN   \n",
       " 2                   NaN         NaN               NaN     NaN   \n",
       " 3                   NaN         NaN               NaN     NaN   \n",
       " 4                   NaN         NaN               NaN     NaN   \n",
       " 5                   NaN         NaN               NaN     NaN   \n",
       " 6                   NaN         NaN               NaN     NaN   \n",
       " 7                   NaN         NaN               NaN     NaN   \n",
       " 8                   NaN         NaN               NaN     NaN   \n",
       " 9                   NaN         NaN               NaN     NaN   \n",
       " 10                  NaN         NaN               NaN     NaN   \n",
       " 11                  NaN         NaN               NaN     NaN   \n",
       " 12                  NaN         NaN               NaN     NaN   \n",
       " 13                  NaN         NaN               NaN     NaN   \n",
       " 14                  NaN         NaN               NaN     NaN   \n",
       " 15                  NaN         NaN               NaN     NaN   \n",
       " 16                  NaN         NaN               NaN     NaN   \n",
       " 17                  NaN         NaN               NaN     NaN   \n",
       " 18                  NaN         NaN               NaN     NaN   \n",
       " 19                  NaN         NaN               NaN     NaN   \n",
       " 20                  NaN         NaN               NaN     NaN   \n",
       " 21                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "     Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                               NaN         NaN             NaN   \n",
       " 1                               NaN         NaN             NaN   \n",
       " 2                               NaN         NaN             NaN   \n",
       " 3                               NaN         NaN             NaN   \n",
       " 4                               NaN         NaN             NaN   \n",
       " 5                               NaN         NaN             NaN   \n",
       " 6                               NaN         NaN             NaN   \n",
       " 7                               NaN         NaN             NaN   \n",
       " 8                               NaN         NaN             NaN   \n",
       " 9                               NaN         NaN             NaN   \n",
       " 10                              NaN         NaN             NaN   \n",
       " 11                              NaN         NaN             NaN   \n",
       " 12                              NaN         NaN             NaN   \n",
       " 13                              NaN         NaN             NaN   \n",
       " 14                              NaN         NaN             NaN   \n",
       " 15                              NaN         NaN             NaN   \n",
       " 16                              NaN         NaN             NaN   \n",
       " 17                              NaN         NaN             NaN   \n",
       " 18                              NaN         NaN             NaN   \n",
       " 19                              NaN         NaN             NaN   \n",
       " 20                              NaN         NaN             NaN   \n",
       " 21                              NaN         NaN             NaN   \n",
       " \n",
       "    Transit peptide  PDB  \n",
       " 0              NaN  NaN  \n",
       " 1              NaN  NaN  \n",
       " 2              NaN  NaN  \n",
       " 3              NaN  NaN  \n",
       " 4              NaN  NaN  \n",
       " 5              NaN  NaN  \n",
       " 6              NaN  NaN  \n",
       " 7              NaN  NaN  \n",
       " 8              NaN  NaN  \n",
       " 9              NaN  NaN  \n",
       " 10             NaN  NaN  \n",
       " 11             NaN  NaN  \n",
       " 12             NaN  NaN  \n",
       " 13             NaN  NaN  \n",
       " 14             NaN  NaN  \n",
       " 15             NaN  NaN  \n",
       " 16             NaN  NaN  \n",
       " 17             NaN  NaN  \n",
       " 18             NaN  NaN  \n",
       " 19             NaN  NaN  \n",
       " 20             NaN  NaN  \n",
       " 21             NaN  NaN  \n",
       " \n",
       " [22 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P11214  TPA_MOUSE       Plat                 Plat                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  Tissue-type plasminogen activator (t-PA) (t-pl...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 149; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  \\\n",
       " 0  PTM: The single chain, almost fully active enz...   \n",
       " \n",
       "                                           Propeptide  \\\n",
       " 0  PROPEP 18..29; /evidence=\"ECO:0000250\"; /id=\"P...   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..17; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P25238  MTK1_KLEPN      kpnIM                kpnIM                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)               Organism  \\\n",
       " 0                         NaN               NaN  Klebsiella pneumoniae   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0            573  Type II methyltransferase M.KpnI (M.KpnI) (EC ...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name                Gene Names Gene Names (primary)  \\\n",
       " 0  O15111  IKKA_HUMAN           CHUK IKKA TCF16                 CHUK   \n",
       " 1  Q14164  IKKE_HUMAN  IKBKE IKKE IKKI KIAA0151                IKBKE   \n",
       " 2  O14920  IKKB_HUMAN                IKBKB IKKB                IKBKB   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0           IKKA TCF16                         NaN               NaN   \n",
       " 1   IKKE IKKI KIAA0151                         NaN               NaN   \n",
       " 2                 IKKB                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " 2  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Inhibitor of nuclear factor kappa-B kinase sub...  ...           NaN   \n",
       " 1  Inhibitor of nuclear factor kappa-B kinase sub...  ...           NaN   \n",
       " 2  Inhibitor of nuclear factor kappa-B kinase sub...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 23; /note=\"Phosphothreonine; by PKB/AK...     NaN   \n",
       " 1  MOD_RES 172; /note=\"Phosphoserine; by autocata...     NaN   \n",
       " 2  MOD_RES 177; /note=\"Phosphoserine; by TBK1 and...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Phosphorylated by MAP3K14/NIK, AKT and to...         NaN   \n",
       " 1  PTM: Autophosphorylated and phosphorylated by ...         NaN   \n",
       " 2  PTM: Upon cytokine stimulation, phosphorylated...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide                        PDB  \n",
       " 0             NaN             NaN  3BRT;5EBZ;5TQW;5TQX;5TQY;  \n",
       " 1             NaN             NaN                        NaN  \n",
       " 2             NaN             NaN       3BRT;3BRV;4E3C;4KIK;  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name    Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  Q6XQN1   PNCB_RAT  Naprt Naprt1                Naprt               Naprt1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Nicotinate phosphoribosyltransferase (NAPRTase...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 213; /note=\"Phosphohistidine\"; /eviden...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Transiently phosphorylated on a His resid...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name    Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  O35217  MINP1_RAT  Minpp1 Mipp1               Minpp1                Mipp1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  Multiple inositol polyphosphate phosphatase 1 ...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 236; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide                         Signal peptide Transit peptide  PDB  \n",
       " 0         NaN  SIGNAL 1..30; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q96480  P5CS_SOLLC       PRO2                 PRO2                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Solanum lycopersicum (Tomato) (Lycopersicon es...           4081   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Delta-1-pyrroline-5-carboxylate synthase (P5CS...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P11413  G6PD_HUMAN       G6PD                 G6PD                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Glucose-6-phosphate 1-dehydrogenase (G6PD) (EC...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: Acetylated by ELP3 at Lys-403; acetylatio...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  \\\n",
       " 0             NaN             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  1QKI;2BH9;2BHL;5UKW;6E07;6E08;6JYU;6VA0;6VA7;6...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P08507  GPDA_RABIT       GPD1                 GPD1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Oryctolagus cuniculus (Rabbit)           9986   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Glycerol-3-phosphate dehydrogenase [NAD(+)], c...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 154; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P15057  LYS_BPCP1    CPL1 22                 CPL1                  22.0   \n",
       " 1  P19386  LYS_BPCP9       CPL9                 CPL9                   NaN   \n",
       " 2  P19385  LYS_BPCP7       CPL7                 CPL7                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " 1                         NaN               NaN   \n",
       " 2                         NaN               NaN   \n",
       " \n",
       "                                         Organism  Organism (ID)  \\\n",
       " 0  Streptococcus phage Cp-1 (Bacteriophage Cp-1)          10747   \n",
       " 1  Streptococcus phage Cp-9 (Bacteriophage Cp-9)          10749   \n",
       " 2  Streptococcus phage Cp-7 (Bacteriophage Cp-7)          10748   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Lysozyme (EC 3.2.1.17) (CP-1 lysin) (Endolysin...  ...           NaN   \n",
       " 1  Lysozyme (EC 3.2.1.17) (CP-9 lysin) (Endolysin...  ...           NaN   \n",
       " 2  Lysozyme (EC 3.2.1.17) (CP-7 lysin) (Endolysin...  ...           NaN   \n",
       " \n",
       "   Initiator methionine Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN        NaN               NaN     NaN   \n",
       " 1                  NaN        NaN               NaN     NaN   \n",
       " 2                  NaN        NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  1H09;1OBA;2IXU;2IXV;2J8F;2J8G;  \n",
       " 1             NaN                             NaN  \n",
       " 2             NaN                      4CVD;5I8L;  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "      Entry  Entry Name                    Gene Names Gene Names (primary)  \\\n",
       " 0   P17443  MURG_ECOLI             murG b0090 JW0088                 murG   \n",
       " 1   B6HZ67  MURG_ECOSE                murG ECSE_0092                 murG   \n",
       " 2   Q8X9Y8  MURG_ECO57            murG Z0100 ECs0094                 murG   \n",
       " 3   B1XC67  MURG_ECODH             murG ECDH10B_0072                 murG   \n",
       " 4   B7MNU9  MURG_ECO81               murG ECED1_0091                 murG   \n",
       " 5   Q1RGA5  MURG_ECOUT              murG UTI89_C0099                 murG   \n",
       " 6   A1A7D5  MURG_ECOK1  murG Ecok1_00810 APECO1_1896                 murG   \n",
       " 7   B7LFW0  MURG_ECO55             murG EC55989_0086                 murG   \n",
       " 8   B7M133  MURG_ECO8A              murG ECIAI1_0089                 murG   \n",
       " 9   C4ZRI5  MURG_ECOBW                 murG BWG_0085                 murG   \n",
       " 10  A7ZW42  MURG_ECOHS               murG EcHS_A0096                 murG   \n",
       " 11  A7ZHI1  MURG_ECO24           murG EcE24377A_0092                 murG   \n",
       " 12  B5YZC6  MURG_ECO5E            murG ECH74115_0098                 murG   \n",
       " 13  B1LG27  MURG_ECOSM             murG EcSMS35_0095                 murG   \n",
       " 14  B7MAL3  MURG_ECO45               murG ECS88_0093                 murG   \n",
       " 15  B7NHJ6  MURG_ECO7I             murG ECIAI39_0093                 murG   \n",
       " 16  B7UIE0  MURG_ECO27              murG E2348C_0095                 murG   \n",
       " 17  B7N7W3  MURG_ECOLU               murG ECUMN_0090                 murG   \n",
       " 18  Q0TLP9  MURG_ECOL5                 murG ECP_0092                 murG   \n",
       " 19  Q8FL64  MURG_ECOL6                    murG c0108                 murG   \n",
       " 20  B1IR88  MURG_ECOLC               murG EcolC_3567                 murG   \n",
       " 21  B7LWF6  MURG_ESCF3                murG EFER_0112                 murG   \n",
       " 22  P37585  MURG_BACSU                 murG BSU15220                 murG   \n",
       " 23  Q83MN4  MURG_SHIFL             murG SF0087 S0089                 murG   \n",
       " \n",
       "     Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                    NaN               b0090 JW0088              NaN   \n",
       " 1                    NaN                  ECSE_0092              NaN   \n",
       " 2                    NaN              Z0100 ECs0094              NaN   \n",
       " 3                    NaN               ECDH10B_0072              NaN   \n",
       " 4                    NaN                 ECED1_0091              NaN   \n",
       " 5                    NaN                UTI89_C0099              NaN   \n",
       " 6                    NaN                Ecok1_00810      APECO1_1896   \n",
       " 7                    NaN               EC55989_0086              NaN   \n",
       " 8                    NaN                ECIAI1_0089              NaN   \n",
       " 9                    NaN                   BWG_0085              NaN   \n",
       " 10                   NaN                 EcHS_A0096              NaN   \n",
       " 11                   NaN             EcE24377A_0092              NaN   \n",
       " 12                   NaN              ECH74115_0098              NaN   \n",
       " 13                   NaN               EcSMS35_0095              NaN   \n",
       " 14                   NaN                 ECS88_0093              NaN   \n",
       " 15                   NaN               ECIAI39_0093              NaN   \n",
       " 16                   NaN                E2348C_0095              NaN   \n",
       " 17                   NaN                 ECUMN_0090              NaN   \n",
       " 18                   NaN                   ECP_0092              NaN   \n",
       " 19                   NaN                      c0108              NaN   \n",
       " 20                   NaN                 EcolC_3567              NaN   \n",
       " 21                   NaN                  EFER_0112              NaN   \n",
       " 22                   NaN                   BSU15220              NaN   \n",
       " 23                   NaN               SF0087 S0089              NaN   \n",
       " \n",
       "                                              Organism  Organism (ID)  \\\n",
       " 0                       Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain SE11)         409438   \n",
       " 2                            Escherichia coli O157:H7          83334   \n",
       " 3               Escherichia coli (strain K12 / DH10B)         316385   \n",
       " 4                  Escherichia coli O81 (strain ED1a)         585397   \n",
       " 5              Escherichia coli (strain UTI89 / UPEC)         364106   \n",
       " 6                       Escherichia coli O1:K1 / APEC         405955   \n",
       " 7              Escherichia coli (strain 55989 / EAEC)         585055   \n",
       " 8                   Escherichia coli O8 (strain IAI1)         585034   \n",
       " 9     Escherichia coli (strain K12 / MC4100 / BW2952)         595496   \n",
       " 10                 Escherichia coli O9:H4 (strain HS)         331112   \n",
       " 11  Escherichia coli O139:H28 (strain E24377A / ETEC)         331111   \n",
       " 12    Escherichia coli O157:H7 (strain EC4115 / EHEC)         444450   \n",
       " 13          Escherichia coli (strain SMS-3-5 / SECEC)         439855   \n",
       " 14       Escherichia coli O45:K1 (strain S88 / ExPEC)         585035   \n",
       " 15      Escherichia coli O7:K1 (strain IAI39 / ExPEC)         585057   \n",
       " 16  Escherichia coli O127:H6 (strain E2348/69 / EPEC)         574521   \n",
       " 17  Escherichia coli O17:K52:H18 (strain UMN026 / ...         585056   \n",
       " 18    Escherichia coli O6:K15:H31 (strain 536 / UPEC)         362663   \n",
       " 19  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 20  Escherichia coli (strain ATCC 8739 / DSM 1576 ...         481805   \n",
       " 21  Escherichia fergusonii (strain ATCC 35469 / DS...         585054   \n",
       " 22                     Bacillus subtilis (strain 168)         224308   \n",
       " 23                                  Shigella flexneri            623   \n",
       " \n",
       "                                         Protein names  ... Glycosylation  \\\n",
       " 0   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 1   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 2   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 3   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 4   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 5   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 6   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 7   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 8   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 9   UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 10  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 11  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 12  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 13  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 14  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 15  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 16  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 17  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 18  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 19  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 20  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 21  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 22  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " 23  UDP-N-acetylglucosamine--N-acetylmuramyl-(pent...  ...           NaN   \n",
       " \n",
       "                                  Initiator methionine  Lipidation  \\\n",
       " 0   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1                                                 NaN         NaN   \n",
       " 2   INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 3                                                 NaN         NaN   \n",
       " 4                                                 NaN         NaN   \n",
       " 5                                                 NaN         NaN   \n",
       " 6                                                 NaN         NaN   \n",
       " 7                                                 NaN         NaN   \n",
       " 8                                                 NaN         NaN   \n",
       " 9                                                 NaN         NaN   \n",
       " 10                                                NaN         NaN   \n",
       " 11                                                NaN         NaN   \n",
       " 12                                                NaN         NaN   \n",
       " 13                                                NaN         NaN   \n",
       " 14                                                NaN         NaN   \n",
       " 15                                                NaN         NaN   \n",
       " 16                                                NaN         NaN   \n",
       " 17                                                NaN         NaN   \n",
       " 18                                                NaN         NaN   \n",
       " 19  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 20                                                NaN         NaN   \n",
       " 21                                                NaN         NaN   \n",
       " 22                                                NaN         NaN   \n",
       " 23                                                NaN         NaN   \n",
       " \n",
       "     Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0                NaN     NaN                              NaN         NaN   \n",
       " 1                NaN     NaN                              NaN         NaN   \n",
       " 2                NaN     NaN                              NaN         NaN   \n",
       " 3                NaN     NaN                              NaN         NaN   \n",
       " 4                NaN     NaN                              NaN         NaN   \n",
       " 5                NaN     NaN                              NaN         NaN   \n",
       " 6                NaN     NaN                              NaN         NaN   \n",
       " 7                NaN     NaN                              NaN         NaN   \n",
       " 8                NaN     NaN                              NaN         NaN   \n",
       " 9                NaN     NaN                              NaN         NaN   \n",
       " 10               NaN     NaN                              NaN         NaN   \n",
       " 11               NaN     NaN                              NaN         NaN   \n",
       " 12               NaN     NaN                              NaN         NaN   \n",
       " 13               NaN     NaN                              NaN         NaN   \n",
       " 14               NaN     NaN                              NaN         NaN   \n",
       " 15               NaN     NaN                              NaN         NaN   \n",
       " 16               NaN     NaN                              NaN         NaN   \n",
       " 17               NaN     NaN                              NaN         NaN   \n",
       " 18               NaN     NaN                              NaN         NaN   \n",
       " 19               NaN     NaN                              NaN         NaN   \n",
       " 20               NaN     NaN                              NaN         NaN   \n",
       " 21               NaN     NaN                              NaN         NaN   \n",
       " 22               NaN     NaN                              NaN         NaN   \n",
       " 23               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "     Signal peptide Transit peptide         PDB  \n",
       " 0              NaN             NaN  1F0K;1NLM;  \n",
       " 1              NaN             NaN         NaN  \n",
       " 2              NaN             NaN         NaN  \n",
       " 3              NaN             NaN         NaN  \n",
       " 4              NaN             NaN         NaN  \n",
       " 5              NaN             NaN         NaN  \n",
       " 6              NaN             NaN         NaN  \n",
       " 7              NaN             NaN         NaN  \n",
       " 8              NaN             NaN         NaN  \n",
       " 9              NaN             NaN         NaN  \n",
       " 10             NaN             NaN         NaN  \n",
       " 11             NaN             NaN         NaN  \n",
       " 12             NaN             NaN         NaN  \n",
       " 13             NaN             NaN         NaN  \n",
       " 14             NaN             NaN         NaN  \n",
       " 15             NaN             NaN         NaN  \n",
       " 16             NaN             NaN         NaN  \n",
       " 17             NaN             NaN         NaN  \n",
       " 18             NaN             NaN         NaN  \n",
       " 19             NaN             NaN         NaN  \n",
       " 20             NaN             NaN         NaN  \n",
       " 21             NaN             NaN         NaN  \n",
       " 22             NaN             NaN         NaN  \n",
       " 23             NaN             NaN         NaN  \n",
       " \n",
       " [24 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name              Gene Names Gene Names (primary)  \\\n",
       " 0  P39288  QUEG_ECOLI  queG yjeS b4166 JW4124                 queG   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 yjeS               b4166 JW4124               NaN   \n",
       " \n",
       "                         Organism  Organism (ID)  \\\n",
       " 0  Escherichia coli (strain K12)          83333   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Epoxyqueuosine reductase (EC 1.17.99.6) (Queuo...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry  Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P08201  NIRB_ECOLI   nirB b3365 JW3328                 nirB   \n",
       " 1  P0A9I8  NIRD_ECOLI   nirD b3366 JW3329                 nirD   \n",
       " 2  P0A229  NIRD_SALTY        nirD STM3475                 nirD   \n",
       " 3  P0A9I9  NIRD_ECO57  nirD Z4727 ECs4217                 nirD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN               b3365 JW3328               NaN   \n",
       " 1                   NaN               b3366 JW3329               NaN   \n",
       " 2                   NaN                    STM3475               NaN   \n",
       " 3                   NaN              Z4727 ECs4217               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                      Escherichia coli (strain K12)          83333   \n",
       " 2  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " 3                           Escherichia coli O157:H7          83334   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Nitrite reductase (NADH) large subunit (EC 1.7...  ...           NaN   \n",
       " 1  Nitrite reductase (NADH) small subunit (EC 1.7...  ...           NaN   \n",
       " 2  Nitrite reductase (NADH) small subunit (EC 1.7...  ...           NaN   \n",
       " 3  Nitrite reductase (NADH) small subunit (EC 1.7...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN    NaN  \n",
       " 1             NaN  2JO6;  \n",
       " 2             NaN    NaN  \n",
       " 3             NaN    NaN  \n",
       " \n",
       " [4 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name   Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  Q57703  PSLS_METJA  comA MJ0255                 comA                   NaN   \n",
       " \n",
       "   Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                     MJ0255               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Methanocaldococcus jannaschii (strain ATCC 430...         243232   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Phosphosulfolactate synthase (EC 4.4.1.19) ((2...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  1QWG;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name      Gene Names  Gene Names (primary)  \\\n",
       " 0  Q8NRW6  RRAAH_CORGL  Cgl0925 cg1055                   NaN   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN             Cgl0925 cg1055               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Corynebacterium glutamicum (strain ATCC 13032 ...         196627   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Putative 4-hydroxy-4-methyl-2-oxoglutarate ald...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry Entry Name  Gene Names  Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P22503  GUN_PHAVU         NaN                   NaN                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                         NaN               NaN   \n",
       " \n",
       "                                          Organism  Organism (ID)  \\\n",
       " 0  Phaseolus vulgaris (Kidney bean) (French bean)           3885   \n",
       " \n",
       "                                        Protein names  ...  Glycosylation  \\\n",
       " 0  Endoglucanase (EC 3.2.1.4) (Abscission cellula...  ...            NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  \\\n",
       " 0                              NaN         NaN   \n",
       " \n",
       "                           Signal peptide Transit peptide  PDB  \n",
       " 0  SIGNAL 1..23; /evidence=\"ECO:0000255\"             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name                      Gene Names Gene Names (primary)  \\\n",
       " 0  P20485  KICH_YEAST  CKI1 CKI YLR133W L3130 L9606.8                 CKI1   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                  CKI                    YLR133W    L3130 L9606.8   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Saccharomyces cerevisiae (strain ATCC 204508 /...         559292   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Choline kinase (EC 2.7.1.32) (ATP:choline phos...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 30; /note=\"Phosphoserine; by PKA\"; /ev...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  P95480  PRNA_PSEFL       prnA                 prnA                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Pseudomonas fluorescens   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0            294  Flavin-dependent tryptophan halogenase PrnA (E...  ...   \n",
       " \n",
       "    Glycosylation Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0            NaN                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                  PDB  \n",
       " 0             NaN  2APG;2AQJ;2AR8;2ARD;2JKC;4Z43;4Z44;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name Gene Names Gene Names (primary)  Gene Names (synonym)  \\\n",
       " 0  O95749  GGPPS_HUMAN      GGPS1                GGPS1                   NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Geranylgeranyl pyrophosphate synthase (GGPP sy...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  2Q80;6C56;6C57;6G31;6G32;6R4V;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name                   Gene Names Gene Names (primary)  \\\n",
       " 0  P0ABS5   DNAG_ECOLI  dnaG dnaP parB b3066 JW3038                 dnaG   \n",
       " 1  P07112  MOBA2_ECOLX                    mobA repB                 mobA   \n",
       " 2  P0ABS6   DNAG_ECO57           dnaG Z4419 ECs3949                 dnaG   \n",
       " 3  Q8FDG5   DNAG_ECOL6                   dnaG c3817                 dnaG   \n",
       " 4  P05096   DNAG_BACSU           dnaG dnaE BSU25210                 dnaG   \n",
       " 5  P0ABS7   DNAG_SHIFL            dnaG SF3107 S3312                 dnaG   \n",
       " 6  P07362   DNAG_SALTY                 dnaG STM3210                 dnaG   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0            dnaP parB               b3066 JW3038               NaN   \n",
       " 1                 repB                        NaN               NaN   \n",
       " 2                  NaN              Z4419 ECs3949               NaN   \n",
       " 3                  NaN                      c3817               NaN   \n",
       " 4                 dnaE                   BSU25210               NaN   \n",
       " 5                  NaN               SF3107 S3312               NaN   \n",
       " 6                  NaN                    STM3210               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                      Escherichia coli (strain K12)          83333   \n",
       " 1                                   Escherichia coli            562   \n",
       " 2                           Escherichia coli O157:H7          83334   \n",
       " 3  Escherichia coli O6:H1 (strain CFT073 / ATCC 7...         199310   \n",
       " 4                     Bacillus subtilis (strain 168)         224308   \n",
       " 5                                  Shigella flexneri            623   \n",
       " 6  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0                         DNA primase (EC 2.7.7.101)  ...           NaN   \n",
       " 1  Mobilization protein A (DNA strand transferase...  ...           NaN   \n",
       " 2                         DNA primase (EC 2.7.7.101)  ...           NaN   \n",
       " 3                         DNA primase (EC 2.7.7.101)  ...           NaN   \n",
       " 4                         DNA primase (EC 2.7.7.101)  ...           NaN   \n",
       " 5                         DNA primase (EC 2.7.7.101)  ...           NaN   \n",
       " 6                         DNA primase (EC 2.7.7.101)  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3                                                NaN         NaN   \n",
       " 4                                                NaN         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6                                                NaN         NaN   \n",
       " \n",
       "    Modified residue Peptide  Post-translational modification  Propeptide  \\\n",
       " 0               NaN     NaN                              NaN         NaN   \n",
       " 1               NaN     NaN                              NaN         NaN   \n",
       " 2               NaN     NaN                              NaN         NaN   \n",
       " 3               NaN     NaN                              NaN         NaN   \n",
       " 4               NaN     NaN                              NaN         NaN   \n",
       " 5               NaN     NaN                              NaN         NaN   \n",
       " 6               NaN     NaN                              NaN         NaN   \n",
       " \n",
       "   Signal peptide Transit peptide  \\\n",
       " 0            NaN             NaN   \n",
       " 1            NaN             NaN   \n",
       " 2            NaN             NaN   \n",
       " 3            NaN             NaN   \n",
       " 4            NaN             NaN   \n",
       " 5            NaN             NaN   \n",
       " 6            NaN             NaN   \n",
       " \n",
       "                                              PDB  \n",
       " 0  1DD9;1DDE;1EQN;1T3W;2HAJ;3B39;6CBR;6CBS;6CBT;  \n",
       " 1                                          2NS6;  \n",
       " 2                                            NaN  \n",
       " 3                                            NaN  \n",
       " 4                                          5GUJ;  \n",
       " 5                                            NaN  \n",
       " 6                                            NaN  \n",
       " \n",
       " [7 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name       Gene Names Gene Names (primary)  \\\n",
       " 0  E3PY95  OAME_ACESD  oraE CLOST_1290                 oraE   \n",
       " 1  E3PY96  OAMS_ACESD  oraS CLOST_1291                 oraS   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                 CLOST_1290               NaN   \n",
       " 1                   NaN                 CLOST_1291               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Acetoanaerobium sticklandii (strain ATCC 12662...         499177   \n",
       " 1  Acetoanaerobium sticklandii (strain ATCC 12662...         499177   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  D-ornithine 4,5-aminomutase subunit beta (EC 5...  ...           NaN   \n",
       " 1  D-ornithine 4,5-aminomutase subunit alpha (EC ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 626; /note=\"N6-(pyridoxal phosphate)ly...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                             PDB  \n",
       " 0             NaN  3KOW;3KOX;3KOY;3KOZ;3KP0;3KP1;  \n",
       " 1             NaN  3KOW;3KOX;3KOY;3KOZ;3KP0;3KP1;  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  P42404  PHI_BACSU  hxlB yckF BSU03450                 hxlB   \n",
       " \n",
       "   Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                 yckF                   BSU03450               NaN   \n",
       " \n",
       "                          Organism  Organism (ID)  \\\n",
       " 0  Bacillus subtilis (strain 168)         224308   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  3-hexulose-6-phosphate isomerase (EC 5.3.1.27)...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide         PDB  \n",
       " 0             NaN  1M3S;1VIV;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  P9WPX3   AK_MYCTU  ask Rv3709c MTV025.057c                  ask   \n",
       " 1  P0A4Z9   AK_MYCBO       ask BQ2027_MB3736C                  ask   \n",
       " 2  P9WPX2   AK_MYCTO               ask MT3812                  ask   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                    Rv3709c      MTV025.057c   \n",
       " 1                   NaN             BQ2027_MB3736C              NaN   \n",
       " 2                   NaN                     MT3812              NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Mycobacterium tuberculosis (strain ATCC 25618 ...          83332   \n",
       " 1  Mycobacterium bovis (strain ATCC BAA-935 / AF2...         233413   \n",
       " 2  Mycobacterium tuberculosis (strain CDC 1551 / ...          83331   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Aspartokinase (EC 2.7.2.4) (Aspartate kinase) ...  ...           NaN   \n",
       " 1  Aspartokinase (EC 2.7.2.4) (Aspartate kinase) ...  ...           NaN   \n",
       " 2  Aspartokinase (EC 2.7.2.4) (Aspartate kinase) ...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide              PDB  \n",
       " 0             NaN  3S1T;4GO5;4GO7;  \n",
       " 1             NaN              NaN  \n",
       " 2             NaN              NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry   Entry Name     Gene Names Gene Names (primary)  \\\n",
       " 0  O74288   ABFB_EMENI    abfB AN1571                 abfB   \n",
       " 1  Q5BDV3   ABFC_EMENI    abfC AN1277                 abfC   \n",
       " 2  Q5AUX2  AXHA2_EMENI  axhA-2 AN7908               axhA-2   \n",
       " 3  Q5BA89   ABFA_EMENI    abfA AN2541                 abfA   \n",
       " 4  Q5B9Z8  AXHA1_EMENI  axhA-1 AN2632               axhA-1   \n",
       " \n",
       "    Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                         NaN           AN1571   \n",
       " 1                   NaN                         NaN           AN1277   \n",
       " 2                   NaN                         NaN           AN7908   \n",
       " 3                   NaN                         NaN           AN2541   \n",
       " 4                   NaN                         NaN           AN2632   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0  Emericella nidulans (strain FGSC A4 / ATCC 381...         227321   \n",
       " 1  Emericella nidulans (strain FGSC A4 / ATCC 381...         227321   \n",
       " 2  Emericella nidulans (strain FGSC A4 / ATCC 381...         227321   \n",
       " 3  Emericella nidulans (strain FGSC A4 / ATCC 381...         227321   \n",
       " 4  Emericella nidulans (strain FGSC A4 / ATCC 381...         227321   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  Alpha-L-arabinofuranosidase B (ABF B) (Arabino...  ...   \n",
       " 1  Alpha-L-arabinofuranosidase C (ABF C) (Arabino...  ...   \n",
       " 2  Alpha-L-arabinofuranosidase axhA-2 (EC 3.2.1.5...  ...   \n",
       " 3  Probable alpha-L-arabinofuranosidase A (ABF A)...  ...   \n",
       " 4  Probable alpha-L-arabinofuranosidase axhA-1 (E...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 89; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 1  CARBOHYD 81; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 2  CARBOHYD 83; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 3  CARBOHYD 71; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " 4  CARBOHYD 309; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " 1         NaN               NaN     NaN                              NaN   \n",
       " 2         NaN               NaN     NaN                              NaN   \n",
       " 3         NaN               NaN     NaN                              NaN   \n",
       " 4         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide                         Signal peptide Transit peptide    PDB  \n",
       " 0         NaN  SIGNAL 1..24; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " 1         NaN   SIGNAL 1..?; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " 2         NaN  SIGNAL 1..22; /evidence=\"ECO:0000255\"             NaN  5UBJ;  \n",
       " 3         NaN  SIGNAL 1..19; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " 4         NaN  SIGNAL 1..26; /evidence=\"ECO:0000255\"             NaN    NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry   Entry Name                  Gene Names Gene Names (primary)  \\\n",
       " 0  Q9ZUR7   ARGJ_ARATH           At2g37500 F3G5.29                  NaN   \n",
       " 1  B5X4Z4  NAGS2_ARATH  NAGS2 At4g37670 F19F18.160                NAGS2   \n",
       " 2  Q84JF4  NAGS1_ARATH    NAGS1 At2g22910 T20K9.12                NAGS1   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0                   NaN                  At2g37500          F3G5.29   \n",
       " 1                   NaN                  At4g37670       F19F18.160   \n",
       " 2                   NaN                  At2g22910         T20K9.12   \n",
       " \n",
       "                                  Organism  Organism (ID)  \\\n",
       " 0  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 1  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " 2  Arabidopsis thaliana (Mouse-ear cress)           3702   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Arginine biosynthesis bifunctional protein Arg...  ...           NaN   \n",
       " 1  Probable amino-acid acetyltransferase NAGS2, c...  ...           NaN   \n",
       " 2  Probable amino-acid acetyltransferase NAGS1, c...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " 2                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 36; /note=\"N-acetylvaline\"; /evidence=...     NaN   \n",
       " 1                                                NaN     NaN   \n",
       " 2                                                NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide PDB  \n",
       " 0  TRANSIT 1..35; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " 1  TRANSIT 1..74; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " 2  TRANSIT 1..67; /note=\"Chloroplast\"; /evidence=... NaN  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P04075  ALDOA_HUMAN  ALDOA ALDA                ALDOA                 ALDA   \n",
       " 1  P09972  ALDOC_HUMAN  ALDOC ALDC                ALDOC                 ALDC   \n",
       " 2  P05062  ALDOB_HUMAN  ALDOB ALDB                ALDOB                 ALDB   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " 1                         NaN               NaN  Homo sapiens (Human)   \n",
       " 2                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Fructose-bisphosphate aldolase A (EC 4.1.2.13)...  ...   \n",
       " 1           9606  Fructose-bisphosphate aldolase C (EC 4.1.2.13)...  ...   \n",
       " 2           9606  Fructose-bisphosphate aldolase B (EC 4.1.2.13)...  ...   \n",
       " \n",
       "   Glycosylation                               Initiator methionine  \\\n",
       " 0           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " 1           NaN                                                NaN   \n",
       " 2           NaN  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...   \n",
       " \n",
       "    Lipidation                                   Modified residue Peptide  \\\n",
       " 0         NaN  MOD_RES 5; /note=\"Phosphotyrosine\"; /evidence=...     NaN   \n",
       " 1         NaN  MOD_RES 5; /note=\"Phosphotyrosine\"; /evidence=...     NaN   \n",
       " 2         NaN  MOD_RES 2; /note=\"N-acetylalanine\"; /evidence=...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                       PDB  \n",
       " 0             NaN  1ALD;2ALD;4ALD;5KY6;6XMH;6XML;6XMM;6XMO;  \n",
       " 1             NaN                                     1XFB;  \n",
       " 2             NaN                           1QO5;1XDL;1XDM;  \n",
       " \n",
       " [3 rows x 110 columns],\n",
       "     Entry  Entry Name         Gene Names Gene Names (primary)  \\\n",
       " 0  Q8Z4R0  PPK1_SALTI  ppk STY2742 t0356                  ppk   \n",
       " 1  O86090  PPK1_SALTY        ppk STM2501                  ppk   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN              STY2742 t0356               NaN   \n",
       " 1                   NaN                    STM2501               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0                                   Salmonella typhi          90370   \n",
       " 1  Salmonella typhimurium (strain LT2 / SGSC1412 ...          99287   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Polyphosphate kinase (EC 2.7.4.1) (ATP-polypho...  ...           NaN   \n",
       " 1  Polyphosphate kinase (EC 2.7.4.1) (ATP-polypho...  ...           NaN   \n",
       " \n",
       "                                 Initiator methionine  Lipidation  \\\n",
       " 0  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " 1  INIT_MET 1; /note=\"Removed\"; /evidence=\"ECO:00...         NaN   \n",
       " \n",
       "    Modified residue Peptide  \\\n",
       " 0               NaN     NaN   \n",
       " 1               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: An intermediate of this reaction is the a...         NaN   \n",
       " 1  PTM: An intermediate of this reaction is the a...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry  Entry Name                                         Gene Names  \\\n",
       " 0  Q94GF1  ASA1_ORYSJ  ASA1 OASA1 Os03g0826500 LOC_Os03g61120 OJ1111_...   \n",
       " 1  Q7XUS2  ASB1_ORYSJ  ASB1 OASB1 Os04g0463500 LOC_Os04g38950 B1358B1...   \n",
       " 2  Q764B9  ASB2_ORYSJ   ASB2 OASB2 Os03g0718000 LOC_Os03g50880 OsJ_12361   \n",
       " 3  Q9XJ29  ASA2_ORYSJ             ASA2 OASA2 Os03g0264400 LOC_Os03g15780   \n",
       " 4  A2XNK3  ASA1_ORYSI                                     ASA1 OsI_14147   \n",
       " \n",
       "   Gene Names (primary) Gene Names (synonym)   Gene Names (ordered locus)  \\\n",
       " 0                 ASA1                OASA1  Os03g0826500 LOC_Os03g61120   \n",
       " 1                 ASB1                OASB1  Os04g0463500 LOC_Os04g38950   \n",
       " 2                 ASB2                OASB2  Os03g0718000 LOC_Os03g50880   \n",
       " 3                 ASA2                OASA2  Os03g0264400 LOC_Os03g15780   \n",
       " 4                 ASA1                  NaN                          NaN   \n",
       " \n",
       "                           Gene Names (ORF)  \\\n",
       " 0  OJ1111_B11.9 OsJ_13197 OSJNBa0010E04.11   \n",
       " 1              B1358B12.20 OSJNBa0060P14.1   \n",
       " 2                                OsJ_12361   \n",
       " 3                                      NaN   \n",
       " 4                                OsI_14147   \n",
       " \n",
       "                               Organism  Organism (ID)  \\\n",
       " 0  Oryza sativa subsp. japonica (Rice)          39947   \n",
       " 1  Oryza sativa subsp. japonica (Rice)          39947   \n",
       " 2  Oryza sativa subsp. japonica (Rice)          39947   \n",
       " 3  Oryza sativa subsp. japonica (Rice)          39947   \n",
       " 4    Oryza sativa subsp. indica (Rice)          39946   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Anthranilate synthase alpha subunit 1, chlorop...  ...           NaN   \n",
       " 1  Anthranilate synthase beta subunit 1, chloropl...  ...           NaN   \n",
       " 2  Anthranilate synthase beta subunit 2, chloropl...  ...           NaN   \n",
       " 3  Anthranilate synthase alpha subunit 2, chlorop...  ...           NaN   \n",
       " 4  Anthranilate synthase alpha subunit 1, chlorop...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " 2                  NaN         NaN               NaN     NaN   \n",
       " 3                  NaN         NaN               NaN     NaN   \n",
       " 4                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " 2                              NaN         NaN             NaN   \n",
       " 3                              NaN         NaN             NaN   \n",
       " 4                              NaN         NaN             NaN   \n",
       " \n",
       "                                      Transit peptide  PDB  \n",
       " 0  TRANSIT 1..34; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 1  TRANSIT 1..58; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 2  TRANSIT 1..47; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 3  TRANSIT 1..49; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " 4  TRANSIT 1..34; /note=\"Chloroplast\"; /evidence=...  NaN  \n",
       " \n",
       " [5 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  E9Q4S1  PDE8B_MOUSE       Pde8b                Pde8b                  NaN   \n",
       " 1  P0C1Q2  PDE11_MOUSE      Pde11a               Pde11a                  NaN   \n",
       " 2  Q3UEI1  PDE4C_MOUSE       Pde4c                Pde4c                  NaN   \n",
       " 3  Q01063  PDE4D_MOUSE       Pde4d                Pde4d                  NaN   \n",
       " 4  O88502  PDE8A_MOUSE  Pde8a Pde8                Pde8a                 Pde8   \n",
       " 5  P70453  PDE7A_MOUSE       Pde7a                Pde7a                  NaN   \n",
       " 6  O89084  PDE4A_MOUSE       Pde4a                Pde4a                  NaN   \n",
       " 7  Q9QXQ1  PDE7B_MOUSE       Pde7b                Pde7b                  NaN   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 1                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 2                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 3                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 4                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 5                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 6                         NaN               NaN  Mus musculus (Mouse)   \n",
       " 7                         NaN               NaN  Mus musculus (Mouse)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10090  High affinity cAMP-specific and IBMX-insensiti...  ...   \n",
       " 1          10090  Dual 3',5'-cyclic-AMP and -GMP phosphodiestera...  ...   \n",
       " 2          10090  cAMP-specific 3',5'-cyclic phosphodiesterase 4...  ...   \n",
       " 3          10090  cAMP-specific 3',5'-cyclic phosphodiesterase 4...  ...   \n",
       " 4          10090  High affinity cAMP-specific and IBMX-insensiti...  ...   \n",
       " 5          10090  High affinity cAMP-specific 3',5'-cyclic phosp...  ...   \n",
       " 6          10090  cAMP-specific 3',5'-cyclic phosphodiesterase 4...  ...   \n",
       " 7          10090  cAMP-specific 3',5'-cyclic phosphodiesterase 7...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  Lipidation  \\\n",
       " 0           NaN                  NaN         NaN   \n",
       " 1           NaN                  NaN         NaN   \n",
       " 2           NaN                  NaN         NaN   \n",
       " 3           NaN                  NaN         NaN   \n",
       " 4           NaN                  NaN         NaN   \n",
       " 5           NaN                  NaN         NaN   \n",
       " 6           NaN                  NaN         NaN   \n",
       " 7           NaN                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 497; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 1  MOD_RES 162; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 2  MOD_RES 9; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " 3  MOD_RES 238; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " 4  MOD_RES 355; /note=\"Phosphoserine; by PKA\"; /e...     NaN   \n",
       " 5                                                NaN     NaN   \n",
       " 6  MOD_RES 13; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " 7  MOD_RES 426; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0                                                NaN         NaN   \n",
       " 1                                                NaN         NaN   \n",
       " 2                                                NaN         NaN   \n",
       " 3  PTM: Sumoylation of long isoforms by PIAS4 aug...         NaN   \n",
       " 4  PTM: Phosphorylated at Ser-355 by PKA under el...         NaN   \n",
       " 5                                                NaN         NaN   \n",
       " 6  PTM: Proteolytically cleaved by CASP3. {ECO:00...         NaN   \n",
       " 7                                                NaN         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide  PDB  \n",
       " 0             NaN             NaN  NaN  \n",
       " 1             NaN             NaN  NaN  \n",
       " 2             NaN             NaN  NaN  \n",
       " 3             NaN             NaN  NaN  \n",
       " 4             NaN             NaN  NaN  \n",
       " 5             NaN             NaN  NaN  \n",
       " 6             NaN             NaN  NaN  \n",
       " 7             NaN             NaN  NaN  \n",
       " \n",
       " [8 rows x 110 columns],\n",
       "     Entry   Entry Name               Gene Names Gene Names (primary)  \\\n",
       " 0  Q96PB1  CASD1_HUMAN  CASD1 C7orf12 Nbla04196                CASD1   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       " 0              C7orf12                         NaN        Nbla04196   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ...  \\\n",
       " 0  N-acetylneuraminate 9-O-acetyltransferase (EC ...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 46; /note=\"N-linked (GlcNAc...) aspar...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  \\\n",
       " 0         NaN               NaN     NaN   \n",
       " \n",
       "                      Post-translational modification  Propeptide  \\\n",
       " 0  PTM: N-glycosylated. {ECO:0000269|PubMed:26169...         NaN   \n",
       " \n",
       "    Signal peptide Transit peptide PDB  \n",
       " 0             NaN             NaN NaN  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry   Entry Name          Gene Names Gene Names (primary)  \\\n",
       " 0  Q9NX46  ADPRS_HUMAN  ADPRS ADPRHL2 ARH3                ADPRS   \n",
       " 1  Q86W56   PARG_HUMAN                PARG                 PARG   \n",
       " \n",
       "   Gene Names (synonym)  Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0         ADPRHL2 ARH3                         NaN               NaN   \n",
       " 1                  NaN                         NaN               NaN   \n",
       " \n",
       "                Organism  Organism (ID)  \\\n",
       " 0  Homo sapiens (Human)           9606   \n",
       " 1  Homo sapiens (Human)           9606   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  ADP-ribosylhydrolase ARH3 (ADP-ribose glycohyd...  ...           NaN   \n",
       " 1     Poly(ADP-ribose) glycohydrolase (EC 3.2.1.143)  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  \\\n",
       " 0                  NaN         NaN   \n",
       " 1                  NaN         NaN   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 64; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       " 1  MOD_RES 22; /note=\"Phosphoserine\"; /evidence=\"...     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide                                                PDB  \n",
       " 0             NaN  2FOZ;2FP0;2G4K;5ZQY;6D36;6D3A;7AKR;7AKS;7ARW;7...  \n",
       " 1             NaN  4A0D;4B1G;4B1H;4B1I;4B1J;5A7R;5LHB;6HH6;6HMK;6...  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       " Empty DataFrame\n",
       " Columns: [Entry, Entry Name, Gene Names, Gene Names (primary), Gene Names (synonym), Gene Names (ordered locus), Gene Names (ORF), Organism, Organism (ID), Protein names, Proteomes, Taxonomic lineage, Virus hosts, Fragment, Sequence, Length, Mass, Gene encoded by, Alternative products (isoforms), Erroneous gene model prediction, Mass spectrometry, Polymorphism, RNA Editing, Sequence caution, Alternative sequence, Natural variant, Non-adjacent residues, Non-standard residue, Non-terminal residue, Sequence conflict, Sequence uncertainty, Sequence version, Coiled coil, Compositional bias, Domain [CC], Domain [FT], Motif, Protein families, Region, Repeat, Zinc finger, Absorption, Active site, Activity regulation, Binding site, Catalytic activity, Cofactor, DNA binding, EC number, Function [CC], Kinetics, Pathway, pH dependence, Redox potential, Site, Temperature dependence, Gene Ontology (GO), Gene Ontology (biological process), Gene Ontology (molecular function), Gene Ontology (cellular component), Gene Ontology IDs, Interacts with, Subunit structure, Developmental stage, Induction, Tissue specificity, PubMed ID, Date of creation, Date of last modification, Date of last sequence modification, Entry version, 3D, Beta strand, Helix, Turn, Subcellular location [CC], Intramembrane, Topological domain, Transmembrane, Annotation, Caution, Comments, Features, Keywords, Keyword ID, Miscellaneous [CC], Protein existence, Tools, Reviewed, UniParc, Allergenic Properties, Biotechnological use, Disruption phenotype, Involvement in disease, Mutagenesis, Pharmaceutical use, Toxic dose, Chain, Cross-link, Disulfide bond, ...]\n",
       " Index: []\n",
       " \n",
       " [0 rows x 110 columns],\n",
       "     Entry  Entry Name           Gene Names Gene Names (primary)  \\\n",
       " 0  A4QI74  TRPD_CORGB        trpD cgR_2918                 trpD   \n",
       " 1  P06559  TRPD_CORGL  trpD Cgl3032 cg3361                 trpD   \n",
       " \n",
       "    Gene Names (synonym) Gene Names (ordered locus)  Gene Names (ORF)  \\\n",
       " 0                   NaN                   cgR_2918               NaN   \n",
       " 1                   NaN             Cgl3032 cg3361               NaN   \n",
       " \n",
       "                                             Organism  Organism (ID)  \\\n",
       " 0              Corynebacterium glutamicum (strain R)         340322   \n",
       " 1  Corynebacterium glutamicum (strain ATCC 13032 ...         196627   \n",
       " \n",
       "                                        Protein names  ... Glycosylation  \\\n",
       " 0  Anthranilate phosphoribosyltransferase (EC 2.4...  ...           NaN   \n",
       " 1  Anthranilate phosphoribosyltransferase (EC 2.4...  ...           NaN   \n",
       " \n",
       "   Initiator methionine  Lipidation  Modified residue Peptide  \\\n",
       " 0                  NaN         NaN               NaN     NaN   \n",
       " 1                  NaN         NaN               NaN     NaN   \n",
       " \n",
       "    Post-translational modification  Propeptide  Signal peptide  \\\n",
       " 0                              NaN         NaN             NaN   \n",
       " 1                              NaN         NaN             NaN   \n",
       " \n",
       "   Transit peptide  PDB  \n",
       " 0             NaN  NaN  \n",
       " 1             NaN  NaN  \n",
       " \n",
       " [2 rows x 110 columns],\n",
       "     Entry Entry Name Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P13233   CN37_RAT   Cnp Cnp1                  Cnp                 Cnp1   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)                 Organism  \\\n",
       " 0                         NaN               NaN  Rattus norvegicus (Rat)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0          10116  2',3'-cyclic-nucleotide 3'-phosphodiesterase (...  ...   \n",
       " \n",
       "   Glycosylation Initiator methionine  \\\n",
       " 0           NaN                  NaN   \n",
       " \n",
       "                                           Lipidation  \\\n",
       " 0  LIPID 417; /note=\"S-farnesyl cysteine\"; /evide...   \n",
       " \n",
       "                                     Modified residue Peptide  \\\n",
       " 0  MOD_RES 9; /note=\"Phosphoserine\"; /evidence=\"E...     NaN   \n",
       " \n",
       "    Post-translational modification  \\\n",
       " 0                              NaN   \n",
       " \n",
       "                                           Propeptide  Signal peptide  \\\n",
       " 0  PROPEP 418..420; /note=\"Removed in mature form...             NaN   \n",
       " \n",
       "   Transit peptide    PDB  \n",
       " 0             NaN  2ILX;  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       "     Entry   Entry Name  Gene Names Gene Names (primary) Gene Names (synonym)  \\\n",
       " 0  P03952  KLKB1_HUMAN  KLKB1 KLK3                KLKB1                 KLK3   \n",
       " \n",
       "    Gene Names (ordered locus)  Gene Names (ORF)              Organism  \\\n",
       " 0                         NaN               NaN  Homo sapiens (Human)   \n",
       " \n",
       "    Organism (ID)                                      Protein names  ...  \\\n",
       " 0           9606  Plasma kallikrein (EC 3.4.21.34) (Fletcher fac...  ...   \n",
       " \n",
       "                                        Glycosylation Initiator methionine  \\\n",
       " 0  CARBOHYD 127; /note=\"N-linked (GlcNAc...) aspa...                  NaN   \n",
       " \n",
       "    Lipidation  Modified residue Peptide  Post-translational modification  \\\n",
       " 0         NaN               NaN     NaN                              NaN   \n",
       " \n",
       "    Propeptide  Signal peptide Transit peptide  \\\n",
       " 0         NaN    SIGNAL 1..19             NaN   \n",
       " \n",
       "                                                  PDB  \n",
       " 0  2ANW;2ANY;4OGX;4OGY;5F8T;5F8X;5F8Z;5TJX;6I44;6...  \n",
       " \n",
       " [1 rows x 110 columns],\n",
       " ...]"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "feature_dfs = []\n",
    "\n",
    "entries = set(df_ec_enz_org['Entry_string'])\n",
    "\n",
    "for entry in entries:\n",
    "    feature_dfs.append(u.get_df(entry, limit=100))\n",
    "\n",
    "feature_dfs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entry</th>\n",
       "      <th>Entry Name</th>\n",
       "      <th>Gene Names</th>\n",
       "      <th>Gene Names (primary)</th>\n",
       "      <th>Gene Names (synonym)</th>\n",
       "      <th>Gene Names (ordered locus)</th>\n",
       "      <th>Gene Names (ORF)</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Organism (ID)</th>\n",
       "      <th>Protein names</th>\n",
       "      <th>...</th>\n",
       "      <th>Glycosylation</th>\n",
       "      <th>Initiator methionine</th>\n",
       "      <th>Lipidation</th>\n",
       "      <th>Modified residue</th>\n",
       "      <th>Peptide</th>\n",
       "      <th>Post-translational modification</th>\n",
       "      <th>Propeptide</th>\n",
       "      <th>Signal peptide</th>\n",
       "      <th>Transit peptide</th>\n",
       "      <th>PDB</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Q7YRC6</td>\n",
       "      <td>AURKB_BOVIN</td>\n",
       "      <td>AURKB AIK2 AIM1 AIRK2 ARK2 STK1 STK12 STK5</td>\n",
       "      <td>AURKB</td>\n",
       "      <td>AIK2 AIM1 AIRK2 ARK2 STK1 STK12 STK5</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>9913</td>\n",
       "      <td>Aurora kinase B (EC 2.7.11.1) (Aurora 1) (Auro...</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MOD_RES 35; /note=\"Phosphothreonine\"; /evidenc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PTM: The phosphorylation of Thr-232 requires t...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Q32PI1</td>\n",
       "      <td>VRK1_BOVIN</td>\n",
       "      <td>VRK1</td>\n",
       "      <td>VRK1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>9913</td>\n",
       "      <td>Serine/threonine-protein kinase VRK1 (EC 2.7.1...</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MOD_RES 376; /note=\"Phosphoserine\"; /evidence=...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PTM: Autophosphorylated at various serine and ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>F1MH24</td>\n",
       "      <td>AAK1_BOVIN</td>\n",
       "      <td>AAK1</td>\n",
       "      <td>AAK1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>9913</td>\n",
       "      <td>AP2-associated protein kinase 1 (EC 2.7.11.1) ...</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PTM: Autophosphorylated. {ECO:0000250|UniProtK...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>P35508</td>\n",
       "      <td>KC1D_BOVIN</td>\n",
       "      <td>CSNK1D HCKID</td>\n",
       "      <td>CSNK1D</td>\n",
       "      <td>HCKID</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>9913</td>\n",
       "      <td>Casein kinase I isoform delta (CKI-delta) (CKI...</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MOD_RES 328; /note=\"Phosphoserine\"; /evidence=...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PTM: Autophosphorylated on serine and threonin...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Q01314</td>\n",
       "      <td>AKT1_BOVIN</td>\n",
       "      <td>AKT1 PKB</td>\n",
       "      <td>AKT1</td>\n",
       "      <td>PKB</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>9913</td>\n",
       "      <td>RAC-alpha serine/threonine-protein kinase (EC ...</td>\n",
       "      <td>...</td>\n",
       "      <td>CARBOHYD 129; /note=\"O-linked (GlcNAc) serine;...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MOD_RES 14; /note=\"N6-acetyllysine\"; /evidence...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PTM: O-GlcNAcylation at Thr-305 and Thr-312 in...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>P00731</td>\n",
       "      <td>CBPA1_RAT</td>\n",
       "      <td>Cpa1 Cpa</td>\n",
       "      <td>Cpa1</td>\n",
       "      <td>Cpa</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rattus norvegicus (Rat)</td>\n",
       "      <td>10116</td>\n",
       "      <td>Carboxypeptidase A1 (EC 3.4.17.1)</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PROPEP 17..110; /note=\"Activation peptide\"; /i...</td>\n",
       "      <td>SIGNAL 1..16</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>P21961</td>\n",
       "      <td>CBPA3_RAT</td>\n",
       "      <td>Cpa3</td>\n",
       "      <td>Cpa3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rattus norvegicus (Rat)</td>\n",
       "      <td>10116</td>\n",
       "      <td>Mast cell carboxypeptidase A (EC 3.4.17.1) (Ca...</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PROPEP 11..104; /note=\"Activation peptide\"; /i...</td>\n",
       "      <td>SIGNAL &lt;1..10; /evidence=\"ECO:0000250\"</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>P39669</td>\n",
       "      <td>GLGC_RHIRD</td>\n",
       "      <td>glgC</td>\n",
       "      <td>glgC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rhizobium radiobacter (Agrobacterium tumefacie...</td>\n",
       "      <td>358</td>\n",
       "      <td>Glucose-1-phosphate adenylyltransferase (EC 2....</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3BRK;5W5R;5W5T;5W6J;6VR0;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Q8U8L5</td>\n",
       "      <td>GLGC_AGRFC</td>\n",
       "      <td>glgC Atu4076 AGR_L_1560</td>\n",
       "      <td>glgC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Atu4076</td>\n",
       "      <td>AGR_L_1560</td>\n",
       "      <td>Agrobacterium fabrum (strain C58 / ATCC 33970)...</td>\n",
       "      <td>176299</td>\n",
       "      <td>Glucose-1-phosphate adenylyltransferase (EC 2....</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>6V96;6V99;6V9A;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>B9JAA3</td>\n",
       "      <td>GLGC_AGRRK</td>\n",
       "      <td>glgC Arad_3871</td>\n",
       "      <td>glgC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Arad_3871</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Agrobacterium radiobacter (strain K84 / ATCC B...</td>\n",
       "      <td>311403</td>\n",
       "      <td>Glucose-1-phosphate adenylyltransferase (EC 2....</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>17298 rows × 110 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entry   Entry Name                                  Gene Names  \\\n",
       "0   Q7YRC6  AURKB_BOVIN  AURKB AIK2 AIM1 AIRK2 ARK2 STK1 STK12 STK5   \n",
       "1   Q32PI1   VRK1_BOVIN                                        VRK1   \n",
       "2   F1MH24   AAK1_BOVIN                                        AAK1   \n",
       "3   P35508   KC1D_BOVIN                                CSNK1D HCKID   \n",
       "4   Q01314   AKT1_BOVIN                                    AKT1 PKB   \n",
       "..     ...          ...                                         ...   \n",
       "0   P00731    CBPA1_RAT                                    Cpa1 Cpa   \n",
       "1   P21961    CBPA3_RAT                                        Cpa3   \n",
       "0   P39669   GLGC_RHIRD                                        glgC   \n",
       "1   Q8U8L5   GLGC_AGRFC                     glgC Atu4076 AGR_L_1560   \n",
       "2   B9JAA3   GLGC_AGRRK                              glgC Arad_3871   \n",
       "\n",
       "   Gene Names (primary)                  Gene Names (synonym)  \\\n",
       "0                 AURKB  AIK2 AIM1 AIRK2 ARK2 STK1 STK12 STK5   \n",
       "1                  VRK1                                   NaN   \n",
       "2                  AAK1                                   NaN   \n",
       "3                CSNK1D                                 HCKID   \n",
       "4                  AKT1                                   PKB   \n",
       "..                  ...                                   ...   \n",
       "0                  Cpa1                                   Cpa   \n",
       "1                  Cpa3                                   NaN   \n",
       "0                  glgC                                   NaN   \n",
       "1                  glgC                                   NaN   \n",
       "2                  glgC                                   NaN   \n",
       "\n",
       "   Gene Names (ordered locus) Gene Names (ORF)  \\\n",
       "0                         NaN              NaN   \n",
       "1                         NaN              NaN   \n",
       "2                         NaN              NaN   \n",
       "3                         NaN              NaN   \n",
       "4                         NaN              NaN   \n",
       "..                        ...              ...   \n",
       "0                         NaN              NaN   \n",
       "1                         NaN              NaN   \n",
       "0                         NaN              NaN   \n",
       "1                     Atu4076       AGR_L_1560   \n",
       "2                   Arad_3871              NaN   \n",
       "\n",
       "                                             Organism Organism (ID)  \\\n",
       "0                                 Bos taurus (Bovine)          9913   \n",
       "1                                 Bos taurus (Bovine)          9913   \n",
       "2                                 Bos taurus (Bovine)          9913   \n",
       "3                                 Bos taurus (Bovine)          9913   \n",
       "4                                 Bos taurus (Bovine)          9913   \n",
       "..                                                ...           ...   \n",
       "0                             Rattus norvegicus (Rat)         10116   \n",
       "1                             Rattus norvegicus (Rat)         10116   \n",
       "0   Rhizobium radiobacter (Agrobacterium tumefacie...           358   \n",
       "1   Agrobacterium fabrum (strain C58 / ATCC 33970)...        176299   \n",
       "2   Agrobacterium radiobacter (strain K84 / ATCC B...        311403   \n",
       "\n",
       "                                        Protein names  ...  \\\n",
       "0   Aurora kinase B (EC 2.7.11.1) (Aurora 1) (Auro...  ...   \n",
       "1   Serine/threonine-protein kinase VRK1 (EC 2.7.1...  ...   \n",
       "2   AP2-associated protein kinase 1 (EC 2.7.11.1) ...  ...   \n",
       "3   Casein kinase I isoform delta (CKI-delta) (CKI...  ...   \n",
       "4   RAC-alpha serine/threonine-protein kinase (EC ...  ...   \n",
       "..                                                ...  ...   \n",
       "0                   Carboxypeptidase A1 (EC 3.4.17.1)  ...   \n",
       "1   Mast cell carboxypeptidase A (EC 3.4.17.1) (Ca...  ...   \n",
       "0   Glucose-1-phosphate adenylyltransferase (EC 2....  ...   \n",
       "1   Glucose-1-phosphate adenylyltransferase (EC 2....  ...   \n",
       "2   Glucose-1-phosphate adenylyltransferase (EC 2....  ...   \n",
       "\n",
       "                                        Glycosylation Initiator methionine  \\\n",
       "0                                                 NaN                  NaN   \n",
       "1                                                 NaN                  NaN   \n",
       "2                                                 NaN                  NaN   \n",
       "3                                                 NaN                  NaN   \n",
       "4   CARBOHYD 129; /note=\"O-linked (GlcNAc) serine;...                  NaN   \n",
       "..                                                ...                  ...   \n",
       "0                                                 NaN                  NaN   \n",
       "1                                                 NaN                  NaN   \n",
       "0                                                 NaN                  NaN   \n",
       "1                                                 NaN                  NaN   \n",
       "2                                                 NaN                  NaN   \n",
       "\n",
       "   Lipidation                                   Modified residue Peptide  \\\n",
       "0         NaN  MOD_RES 35; /note=\"Phosphothreonine\"; /evidenc...     NaN   \n",
       "1         NaN  MOD_RES 376; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       "2         NaN  MOD_RES 1; /note=\"N-acetylmethionine\"; /eviden...     NaN   \n",
       "3         NaN  MOD_RES 328; /note=\"Phosphoserine\"; /evidence=...     NaN   \n",
       "4         NaN  MOD_RES 14; /note=\"N6-acetyllysine\"; /evidence...     NaN   \n",
       "..        ...                                                ...     ...   \n",
       "0         NaN                                                NaN     NaN   \n",
       "1         NaN                                                NaN     NaN   \n",
       "0         NaN                                                NaN     NaN   \n",
       "1         NaN                                                NaN     NaN   \n",
       "2         NaN                                                NaN     NaN   \n",
       "\n",
       "                      Post-translational modification  \\\n",
       "0   PTM: The phosphorylation of Thr-232 requires t...   \n",
       "1   PTM: Autophosphorylated at various serine and ...   \n",
       "2   PTM: Autophosphorylated. {ECO:0000250|UniProtK...   \n",
       "3   PTM: Autophosphorylated on serine and threonin...   \n",
       "4   PTM: O-GlcNAcylation at Thr-305 and Thr-312 in...   \n",
       "..                                                ...   \n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "\n",
       "                                           Propeptide  \\\n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "3                                                 NaN   \n",
       "4                                                 NaN   \n",
       "..                                                ...   \n",
       "0   PROPEP 17..110; /note=\"Activation peptide\"; /i...   \n",
       "1   PROPEP 11..104; /note=\"Activation peptide\"; /i...   \n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "\n",
       "                            Signal peptide Transit peptide  \\\n",
       "0                                      NaN             NaN   \n",
       "1                                      NaN             NaN   \n",
       "2                                      NaN             NaN   \n",
       "3                                      NaN             NaN   \n",
       "4                                      NaN             NaN   \n",
       "..                                     ...             ...   \n",
       "0                             SIGNAL 1..16             NaN   \n",
       "1   SIGNAL <1..10; /evidence=\"ECO:0000250\"             NaN   \n",
       "0                                      NaN             NaN   \n",
       "1                                      NaN             NaN   \n",
       "2                                      NaN             NaN   \n",
       "\n",
       "                          PDB  \n",
       "0                         NaN  \n",
       "1                         NaN  \n",
       "2                         NaN  \n",
       "3                         NaN  \n",
       "4                         NaN  \n",
       "..                        ...  \n",
       "0                         NaN  \n",
       "1                         NaN  \n",
       "0   3BRK;5W5R;5W5T;5W6J;6VR0;  \n",
       "1             6V96;6V99;6V9A;  \n",
       "2                         NaN  \n",
       "\n",
       "[17298 rows x 110 columns]"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.concat(feature_dfs, axis=0)\n",
    "\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "16088"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(set(df['Entry']))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Reduce dataframe to fewer columns\n",
    "columns_of_interest = ['Entry', 'Gene Names (primary)', 'Organism', 'Protein names', 'Domain [CC]', 'Domain [FT]', 'Motif', 'Protein families', 'Region', 'Active site', 'Activity regulation', 'Binding site']\n",
    "df_features = df[columns_of_interest]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_features = df_features.drop_duplicates()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entry</th>\n",
       "      <th>Gene Names (primary)</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Protein names</th>\n",
       "      <th>Domain [CC]</th>\n",
       "      <th>Domain [FT]</th>\n",
       "      <th>Motif</th>\n",
       "      <th>Protein families</th>\n",
       "      <th>Region</th>\n",
       "      <th>Active site</th>\n",
       "      <th>Activity regulation</th>\n",
       "      <th>Binding site</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Q7YRC6</td>\n",
       "      <td>AURKB</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>Aurora kinase B (EC 2.7.11.1) (Aurora 1) (Auro...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DOMAIN 77..327; /note=\"Protein kinase\"; /evide...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Protein kinase superfamily, Ser/Thr protein ki...</td>\n",
       "      <td>REGION 46..65; /note=\"Disordered\"; /evidence=\"...</td>\n",
       "      <td>ACT_SITE 200; /note=\"Proton acceptor\"; /eviden...</td>\n",
       "      <td>ACTIVITY REGULATION: Activity is greatly incre...</td>\n",
       "      <td>BINDING 83..91; /ligand=\"ATP\"; /ligand_id=\"ChE...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Q32PI1</td>\n",
       "      <td>VRK1</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>Serine/threonine-protein kinase VRK1 (EC 2.7.1...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DOMAIN 37..317; /note=\"Protein kinase\"; /evide...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Protein kinase superfamily, CK1 Ser/Thr protei...</td>\n",
       "      <td>REGION 352..396; /note=\"Disordered\"; /evidence...</td>\n",
       "      <td>ACT_SITE 177; /note=\"Proton acceptor\"; /eviden...</td>\n",
       "      <td>ACTIVITY REGULATION: Active in presence of Mn(...</td>\n",
       "      <td>BINDING 43..51; /ligand=\"ATP\"; /ligand_id=\"ChE...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>F1MH24</td>\n",
       "      <td>AAK1</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>AP2-associated protein kinase 1 (EC 2.7.11.1) ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DOMAIN 46..315; /note=\"Protein kinase\"; /evide...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Protein kinase superfamily, Ser/Thr protein ki...</td>\n",
       "      <td>REGION 1..25; /note=\"Disordered\"; /evidence=\"E...</td>\n",
       "      <td>ACT_SITE 176; /note=\"Proton acceptor\"; /eviden...</td>\n",
       "      <td>ACTIVITY REGULATION: Stimulated by clathrin. {...</td>\n",
       "      <td>BINDING 52..60; /ligand=\"ATP\"; /ligand_id=\"ChE...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>P35508</td>\n",
       "      <td>CSNK1D</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>Casein kinase I isoform delta (CKI-delta) (CKI...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DOMAIN 9..277; /note=\"Protein kinase\"; /eviden...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Protein kinase superfamily, CK1 Ser/Thr protei...</td>\n",
       "      <td>REGION 278..364; /note=\"Centrosomal localizati...</td>\n",
       "      <td>ACT_SITE 128; /note=\"Proton acceptor\"; /eviden...</td>\n",
       "      <td>ACTIVITY REGULATION: Drug-mediated inhibition ...</td>\n",
       "      <td>BINDING 15..23; /ligand=\"ATP\"; /ligand_id=\"ChE...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Q01314</td>\n",
       "      <td>AKT1</td>\n",
       "      <td>Bos taurus (Bovine)</td>\n",
       "      <td>RAC-alpha serine/threonine-protein kinase (EC ...</td>\n",
       "      <td>DOMAIN: Binding of the PH domain to phosphatid...</td>\n",
       "      <td>DOMAIN 5..108; /note=\"PH\"; /evidence=\"ECO:0000...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Protein kinase superfamily, AGC Ser/Thr protei...</td>\n",
       "      <td>REGION 450..480; /note=\"Disordered\"; /evidence...</td>\n",
       "      <td>ACT_SITE 274; /note=\"Proton acceptor\"; /eviden...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BINDING 14..19; /ligand=\"1D-myo-inositol 1,3,4...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Q29548</td>\n",
       "      <td>HEXB</td>\n",
       "      <td>Sus scrofa (Pig)</td>\n",
       "      <td>Beta-hexosaminidase subunit beta (EC 3.2.1.52)...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Glycosyl hydrolase 20 family</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ACT_SITE 329; /note=\"Proton donor\"; /evidence=...</td>\n",
       "      <td>ACTIVITY REGULATION: Addition of GM2A stimulat...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>P74299</td>\n",
       "      <td>ppc</td>\n",
       "      <td>Synechocystis sp. (strain PCC 6803 / Kazusa)</td>\n",
       "      <td>Phosphoenolpyruvate carboxylase (PEPC) (PEPCas...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PEPCase type 1 family</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ACT_SITE 203; /evidence=\"ECO:0000250\"; ACT_SIT...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>P39669</td>\n",
       "      <td>glgC</td>\n",
       "      <td>Rhizobium radiobacter (Agrobacterium tumefacie...</td>\n",
       "      <td>Glucose-1-phosphate adenylyltransferase (EC 2....</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Bacterial/plant glucose-1-phosphate adenylyltr...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BINDING 107; /ligand=\"alpha-D-glucose 1-phosph...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Q8U8L5</td>\n",
       "      <td>glgC</td>\n",
       "      <td>Agrobacterium fabrum (strain C58 / ATCC 33970)...</td>\n",
       "      <td>Glucose-1-phosphate adenylyltransferase (EC 2....</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Bacterial/plant glucose-1-phosphate adenylyltr...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BINDING 107; /ligand=\"alpha-D-glucose 1-phosph...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>B9JAA3</td>\n",
       "      <td>glgC</td>\n",
       "      <td>Agrobacterium radiobacter (strain K84 / ATCC B...</td>\n",
       "      <td>Glucose-1-phosphate adenylyltransferase (EC 2....</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Bacterial/plant glucose-1-phosphate adenylyltr...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BINDING 107; /ligand=\"alpha-D-glucose 1-phosph...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>16088 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entry Gene Names (primary)  \\\n",
       "0   Q7YRC6                AURKB   \n",
       "1   Q32PI1                 VRK1   \n",
       "2   F1MH24                 AAK1   \n",
       "3   P35508               CSNK1D   \n",
       "4   Q01314                 AKT1   \n",
       "..     ...                  ...   \n",
       "0   Q29548                 HEXB   \n",
       "0   P74299                  ppc   \n",
       "0   P39669                 glgC   \n",
       "1   Q8U8L5                 glgC   \n",
       "2   B9JAA3                 glgC   \n",
       "\n",
       "                                             Organism  \\\n",
       "0                                 Bos taurus (Bovine)   \n",
       "1                                 Bos taurus (Bovine)   \n",
       "2                                 Bos taurus (Bovine)   \n",
       "3                                 Bos taurus (Bovine)   \n",
       "4                                 Bos taurus (Bovine)   \n",
       "..                                                ...   \n",
       "0                                    Sus scrofa (Pig)   \n",
       "0        Synechocystis sp. (strain PCC 6803 / Kazusa)   \n",
       "0   Rhizobium radiobacter (Agrobacterium tumefacie...   \n",
       "1   Agrobacterium fabrum (strain C58 / ATCC 33970)...   \n",
       "2   Agrobacterium radiobacter (strain K84 / ATCC B...   \n",
       "\n",
       "                                        Protein names  \\\n",
       "0   Aurora kinase B (EC 2.7.11.1) (Aurora 1) (Auro...   \n",
       "1   Serine/threonine-protein kinase VRK1 (EC 2.7.1...   \n",
       "2   AP2-associated protein kinase 1 (EC 2.7.11.1) ...   \n",
       "3   Casein kinase I isoform delta (CKI-delta) (CKI...   \n",
       "4   RAC-alpha serine/threonine-protein kinase (EC ...   \n",
       "..                                                ...   \n",
       "0   Beta-hexosaminidase subunit beta (EC 3.2.1.52)...   \n",
       "0   Phosphoenolpyruvate carboxylase (PEPC) (PEPCas...   \n",
       "0   Glucose-1-phosphate adenylyltransferase (EC 2....   \n",
       "1   Glucose-1-phosphate adenylyltransferase (EC 2....   \n",
       "2   Glucose-1-phosphate adenylyltransferase (EC 2....   \n",
       "\n",
       "                                          Domain [CC]  \\\n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "3                                                 NaN   \n",
       "4   DOMAIN: Binding of the PH domain to phosphatid...   \n",
       "..                                                ...   \n",
       "0                                                 NaN   \n",
       "0                                                 NaN   \n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "\n",
       "                                          Domain [FT] Motif  \\\n",
       "0   DOMAIN 77..327; /note=\"Protein kinase\"; /evide...   NaN   \n",
       "1   DOMAIN 37..317; /note=\"Protein kinase\"; /evide...   NaN   \n",
       "2   DOMAIN 46..315; /note=\"Protein kinase\"; /evide...   NaN   \n",
       "3   DOMAIN 9..277; /note=\"Protein kinase\"; /eviden...   NaN   \n",
       "4   DOMAIN 5..108; /note=\"PH\"; /evidence=\"ECO:0000...   NaN   \n",
       "..                                                ...   ...   \n",
       "0                                                 NaN   NaN   \n",
       "0                                                 NaN   NaN   \n",
       "0                                                 NaN   NaN   \n",
       "1                                                 NaN   NaN   \n",
       "2                                                 NaN   NaN   \n",
       "\n",
       "                                     Protein families  \\\n",
       "0   Protein kinase superfamily, Ser/Thr protein ki...   \n",
       "1   Protein kinase superfamily, CK1 Ser/Thr protei...   \n",
       "2   Protein kinase superfamily, Ser/Thr protein ki...   \n",
       "3   Protein kinase superfamily, CK1 Ser/Thr protei...   \n",
       "4   Protein kinase superfamily, AGC Ser/Thr protei...   \n",
       "..                                                ...   \n",
       "0                        Glycosyl hydrolase 20 family   \n",
       "0                               PEPCase type 1 family   \n",
       "0   Bacterial/plant glucose-1-phosphate adenylyltr...   \n",
       "1   Bacterial/plant glucose-1-phosphate adenylyltr...   \n",
       "2   Bacterial/plant glucose-1-phosphate adenylyltr...   \n",
       "\n",
       "                                               Region  \\\n",
       "0   REGION 46..65; /note=\"Disordered\"; /evidence=\"...   \n",
       "1   REGION 352..396; /note=\"Disordered\"; /evidence...   \n",
       "2   REGION 1..25; /note=\"Disordered\"; /evidence=\"E...   \n",
       "3   REGION 278..364; /note=\"Centrosomal localizati...   \n",
       "4   REGION 450..480; /note=\"Disordered\"; /evidence...   \n",
       "..                                                ...   \n",
       "0                                                 NaN   \n",
       "0                                                 NaN   \n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "\n",
       "                                          Active site  \\\n",
       "0   ACT_SITE 200; /note=\"Proton acceptor\"; /eviden...   \n",
       "1   ACT_SITE 177; /note=\"Proton acceptor\"; /eviden...   \n",
       "2   ACT_SITE 176; /note=\"Proton acceptor\"; /eviden...   \n",
       "3   ACT_SITE 128; /note=\"Proton acceptor\"; /eviden...   \n",
       "4   ACT_SITE 274; /note=\"Proton acceptor\"; /eviden...   \n",
       "..                                                ...   \n",
       "0   ACT_SITE 329; /note=\"Proton donor\"; /evidence=...   \n",
       "0   ACT_SITE 203; /evidence=\"ECO:0000250\"; ACT_SIT...   \n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "\n",
       "                                  Activity regulation  \\\n",
       "0   ACTIVITY REGULATION: Activity is greatly incre...   \n",
       "1   ACTIVITY REGULATION: Active in presence of Mn(...   \n",
       "2   ACTIVITY REGULATION: Stimulated by clathrin. {...   \n",
       "3   ACTIVITY REGULATION: Drug-mediated inhibition ...   \n",
       "4                                                 NaN   \n",
       "..                                                ...   \n",
       "0   ACTIVITY REGULATION: Addition of GM2A stimulat...   \n",
       "0                                                 NaN   \n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "\n",
       "                                         Binding site  \n",
       "0   BINDING 83..91; /ligand=\"ATP\"; /ligand_id=\"ChE...  \n",
       "1   BINDING 43..51; /ligand=\"ATP\"; /ligand_id=\"ChE...  \n",
       "2   BINDING 52..60; /ligand=\"ATP\"; /ligand_id=\"ChE...  \n",
       "3   BINDING 15..23; /ligand=\"ATP\"; /ligand_id=\"ChE...  \n",
       "4   BINDING 14..19; /ligand=\"1D-myo-inositol 1,3,4...  \n",
       "..                                                ...  \n",
       "0                                                 NaN  \n",
       "0                                                 NaN  \n",
       "0   BINDING 107; /ligand=\"alpha-D-glucose 1-phosph...  \n",
       "1   BINDING 107; /ligand=\"alpha-D-glucose 1-phosph...  \n",
       "2   BINDING 107; /ligand=\"alpha-D-glucose 1-phosph...  \n",
       "\n",
       "[16088 rows x 12 columns]"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_features"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_features.to_csv('protein_features_new.txt')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### Get InterPro ids for every UniProt entry"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Define function for retrieving the InterPro ids\n",
    "def retrieve_interpro_ids(uniprot_entry):\n",
    "    url = \"http://www.uniprot.org/uniprot/{}.xml\".format(uniprot_entry)\n",
    "    try:\n",
    "        handle = urllib.request.urlopen(url)\n",
    "        record = SeqIO.read(handle, \"uniprot-xml\")\n",
    "        interpro_ids = list(filter(lambda ref: 'InterPro' in ref, record.dbxrefs))  \n",
    "    except:\n",
    "        interpro_ids =  [0]\n",
    "    return interpro_ids"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'df_features' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32mc:\\Users\\Elin\\Documents\\GitHub\\predicting-allostery\\Download_features.ipynb Cell 24\u001b[0m in \u001b[0;36m<cell line: 1>\u001b[1;34m()\u001b[0m\n\u001b[1;32m----> <a href='vscode-notebook-cell:/c%3A/Users/Elin/Documents/GitHub/predicting-allostery/Download_features.ipynb#Y115sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m all_entries \u001b[39m=\u001b[39m \u001b[39mset\u001b[39m(df_features[\u001b[39m'\u001b[39m\u001b[39mEntry\u001b[39m\u001b[39m'\u001b[39m])\n\u001b[0;32m      <a href='vscode-notebook-cell:/c%3A/Users/Elin/Documents/GitHub/predicting-allostery/Download_features.ipynb#Y115sZmlsZQ%3D%3D?line=2'>3</a>\u001b[0m f \u001b[39m=\u001b[39m \u001b[39mopen\u001b[39m(\u001b[39m\"\u001b[39m\u001b[39m./all_entries.txt\u001b[39m\u001b[39m\"\u001b[39m, mode\u001b[39m=\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mx\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[0;32m      <a href='vscode-notebook-cell:/c%3A/Users/Elin/Documents/GitHub/predicting-allostery/Download_features.ipynb#Y115sZmlsZQ%3D%3D?line=4'>5</a>\u001b[0m \u001b[39mfor\u001b[39;00m entry \u001b[39min\u001b[39;00m all_entries:\n",
      "\u001b[1;31mNameError\u001b[0m: name 'df_features' is not defined"
     ]
    }
   ],
   "source": [
    "all_entries = set(df_features['Entry'])\n",
    "\n",
    "f = open(\"./all_entries.txt\", mode=\"x\")\n",
    "\n",
    "for entry in all_entries:\n",
    "    f.write(entry + \"\\n\")\n",
    "\n",
    "f.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "import multiprocessing as mp\n",
    "\n",
    "\n",
    "def retrieve_interpro_ids(uniprot_entry):\n",
    "    url = f\"http://www.uniprot.org/uniprot/{uniprot_entry}.xml\"\n",
    "    try:\n",
    "        handle = urllib.request.urlopen(url)\n",
    "        record = SeqIO.read(handle, \"uniprot-xml\")\n",
    "        interpro_ids = [ref for ref in record.dbxrefs if 'InterPro' in ref]\n",
    "    except:\n",
    "        interpro_ids = [0]\n",
    "    return [uniprot_entry, interpro_ids]\n",
    "\n",
    "# main\n",
    "global_list = mp.Manager().list()\n",
    "\n",
    "\n",
    "with mp.Pool(6) as pool:\n",
    "    results = [pool.apply_async(retrieve_interpro_ids, args=(entry, global_list, i)) for i, entry in enumerate(all_entries)]\n",
    "    # wait for all processes to finish\n",
    "    for r in results:\n",
    "        r.wait()\n",
    "return results\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "global_list: [[<entry>, [ids..]]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Apply function to dataframe\n",
    "df_features['Interpro_ids'] = df_features['Entry'].apply(lambda entry: retrieve_interpro_ids(entry))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Define function for extracting the EC number from the protein name\n",
    "def extract_EC(string):\n",
    "    ECs = []\n",
    "    array = string.split('(')\n",
    "    for word in array:\n",
    "        if word.startswith('EC'):\n",
    "            ECs.append(word.strip(') '))\n",
    "    return ECs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\Elin\\AppData\\Local\\Temp\\ipykernel_11976\\3852558869.py:2: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df_features['EC numbers'] = df_features['Protein names'].apply(lambda string: extract_EC(string))\n"
     ]
    }
   ],
   "source": [
    "#Apply function to dataframe to extract EC numbers\n",
    "df_features['EC numbers'] = df_features['Protein names'].apply(lambda string: extract_EC(string))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Keep only relevant columns\n",
    "df_features_small = df_features[['Entry', 'Organism', 'EC numbers', 'Interpro_ids']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Make sure that every element of the EC number and InterPro id lists are evaluated as strings\n",
    "df_features_small['EC numbers'] = df_features_small['EC numbers'].apply(lambda string: ast.literal_eval(string))\n",
    "df_features_small['Interpro_ids'] = df_features_small['Interpro_ids'].apply(lambda string: ast.literal_eval(string))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Explode on EC numbers to get one Entry, Organism, EC number and InterPro id on each row\n",
    "df_features_new = df_features_small.explode('EC numbers')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Remove 'EC ' from EC numbers\n",
    "df_features_new['EC numbers'] = df_features_new['EC numbers'].str.replace('EC ', '')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Extract organism name from Organism columns\n",
    "df_features_new['Org'] = df_features_new['Organism'].apply(lambda x: ' '.join(x.split()[:2]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Save to csv\n",
    "df_features_new.to_csv('protein_domains.txt')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Merge the dataframe of InterPro ids for each EC numbers with interactions dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Merge the dataframes\n",
    "df_merged = pd.merge(df_brenda_top_hundred, df_features_new, left_on=['EC', 'Org'], right_on=['EC numbers', 'Org'], how='left')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Clean up dataframe to get one feature per row"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Drop unnecessary columns\n",
    "df_merged_new = df_merged.drop(['Organism', 'Entry', 'EC numbers', 'Enz'], axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Explode on InterPro ids\n",
    "df_merged_new = df_merged_new.explode('Interpro_ids')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Drop duplicates and reset index\n",
    "df_merged_new.drop_duplicates(inplace=True)\n",
    "df_merged_new.reset_index(drop=True, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Remove 'InterPro' from ids\n",
    "df_merged_new['Interpro_ids'] = df_merged_new['Interpro_ids'].str.replace('InterPro:', '')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Save to csv\n",
    "df_merged_new.to_csv('features_interactions_merged.txt')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Map InterPro ids to entry type and entry name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Get list of entry type and name for InterPro ids, downloaded from the InterPro website\n",
    "df_entry_list = pd.read_csv('entry.list.txt', sep='\\t', header=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Merge dataframe of features and interactions with the entry-list\n",
    "df_merged_type = pd.merge(df_merged_new, df_entry_list, left_on='Interpro_ids', right_on='ENTRY_AC', how='left').drop('ENTRY_AC', axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Save file to csv\n",
    "df_merged_type.to_csv('features_interactions_merged_types.txt')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Get the features/InterPro ids for organisms and EC numbers in phylogenetic tree"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###### This part retrieves the InterPro ids for the Organism-EC number pairs that are mapped in the phylogenetic tree used to identify predicted interactions. The reasoning for doing so is that not all of these EC numbers were documented in BRENDA for all top hundred organisms, and therefore we did not have data on the features for these organisms, resulting in false-negative predictions. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Create list of the EC numbers mapped on the phylogenetic tree\n",
    "top_ten_ECs = ['1.3.5.1', '2.2.1.6', '2.7.1.1', '2.7.1.11', '2.7.1.30', '2.7.1.40', '2.7.2.4', '2.7.7.27', '3.1.3.11', '6.4.1.1']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Make a dataframe of the top hundred organisms \n",
    "df_top_hundred = pd.DataFrame(top_hundred_list, columns=['Org'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Associate every organism with every EC number in the top ten group\n",
    "df_top_hundred['EC'] = [top_ten_ECs]*100\n",
    "df_top_hundred_EC = df_top_hundred.explode('EC').reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Apply string-formula to dataframe to get entry-strings\n",
    "df_top_hundred_EC['Entry_string'] = df_top_hundred_EC.apply(lambda x: entry_formula(x['EC'], x['Org']), axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Get info for every EC number+Organism combination\n",
    "\n",
    "df = pd.DataFrame([])\n",
    "\n",
    "for entry in df_top_hundred_EC['Entry_string']:\n",
    "    df = pd.concat([df,u.get_df(entry)])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Make dataframe with only wanted columns\n",
    "df_features_topEC = df[columns_of_interest]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Get InterPro ids for all features and extract EC numbers\n",
    "df_features_topEC['Interpro_ids'] = df_features_topEC['Entry'].apply(lambda entry: retrieve_interpro_ids(entry))\n",
    "df_features_topEC['EC numbers'] = df_features_topEC['Protein names'].apply(lambda string: extract_EC(string))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Make dataframe with only wanted columns\n",
    "df_features_topEC_small = df_features_topEC[['Entry', 'Organism', 'EC numbers', 'Interpro_ids']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Explode dataframe to get one EC number per row\n",
    "df_features_topEC_exp = df_features_topEC_small.explode('EC numbers')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Clean up EC numbers and organism names\n",
    "df_features_topEC_exp ['EC numbers'] = df_features_topEC_exp ['EC numbers'].str.replace('EC ', '')\n",
    "df_features_topEC_exp['Org'] = df_features_topEC_exp['Organism'].apply(lambda x: ' '.join(x.split()[:2]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Save file to csv\n",
    "df_features_topEC_exp.to_csv('features_for_ECs_in_tree.txt')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.6"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "436b2daae35b1d81a27495251f54690ee226a662a175ea582d303ed01ff4427f"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
